use GenoPubMigrate;

exec sp_MSforeachtable 
 @command1 = 'ALTER table ? NOCHECK CONSTRAINT all';
--
-- Table structure for table dbo.gpUser --

IF  EXISTS (SELECT * FROM sys.objects WHERE object_id = OBJECT_ID(N'dbo.gpUser') AND type in (N'U')) DROP TABLE dbo.gpUser;


CREATE TABLE dbo.gpUser (
  idUser int  NOT NULL identity,
  lastName varchar(200) NOT NULL,
  firstName varchar(200) NOT NULL,
  middleName varchar(100)  NULL,
  UserName varchar(30)  NULL,
  password varchar(200)  NULL,
  email varchar(500)  NULL,
  institute varchar(200)  NULL,
  ucscUrl varchar(250) default 'http://genome.ucsc.edu',
CONSTRAINT [PK_User] PRIMARY KEY CLUSTERED 
(
	[idUser] ASC
),
  CONSTRAINT Index_UserName UNIQUE (UserName)
)  ;


--
-- Dumping data for table dbo.gpUser --


ALTER TABLE dbo.gpUser NOCHECK CONSTRAINT all;
SET IDENTITY_INSERT dbo.gpUser ON;
INSERT INTO dbo.gpUser (  idUser 
  ,lastName
  ,firstName 
  ,middleName
  ,UserName
  ,password 
  ,email 
  ,institute 
  ,ucscUrl )VALUES (1,'','guest','','guest','454326b776dc46d32bb1050efe72df5e','','','http://genome.ucsc.edu'),(3,'Nix','David','','u0028003','57646c8516327eb37ee5aba08e95af57','david.nix@hci.utah.edu','HCI','http://genome.ucsc.edu'),(4,'Di Sera','Tony','','tdisera','a8a4bad48ce922000dd8d48dd5c1cd80','tony.disera@hci.utah.edu','HCI','http://genome.ucsc.edu'),(12,'Courdy','Samir','','scourdy','678d38b5a5e424734eea5fb2e642c9b2','scourdy@hci.utah.edu','Hunstman Cancer dbo.gpInstitute - Research Informatics','http://genome.ucsc.edu'),(14,'Lab','Cairns','','CairnsLab','160e984abad2df38b6594767c7d9887c','','','http://genome.ucsc.edu'),(15,'Lab','Wang','','WangLab','fe9bcb5bfa6c5ae0248b951784bf3076','','','http://genome.ucsc.edu'),(16,'Lab','Lessnick','','LessnickLab','2672162a021f11aa86810dfa923f9806','','','http://genome.ucsc.edu'),(17,'Lab','GenEpi','','GenEpi','516731d3d7dbd22e1915f8af65d8c1b8','','','http://genome.ucsc.edu'),(18,'Lab','Leppert','','LeppertLab','fefbcef3ca08e1a012dc01b638afe048','','','http://genome.ucsc.edu'),(19,'Lab','Graves','','GravesLab','054af8b8aad90ad2252d915965b90ca5','','','http://genome.ucsc.edu'),(20,'Lab','Jones','','JonesLab','3d19649a41bb90fa3349e4403684ed47','','','http://genome.ucsc.edu'),(21,'ARUP','ARUP','','ARUP','be93a26e20edc4d18d4a12814e85710b','','','http://genome.ucsc.edu'),(22,'Lab','Hagedorn','','HagedornLab','6f768a4df2212d5e7d3910e86b8510ec','','','http://genome.ucsc.edu'),(23,'Lab','Schiffman','','SchiffmanLab','3c2d3e84c3253bdff48ac62b11bb5ae0','','','http://genome.ucsc.edu'),(24,'Lab','Weis','','WeisLab','b613d6dd93133c5f5fe33dfdad3cbed2','','','http://genome.ucsc.edu'),(25,'Lab','Tantin','','TantinLab','9b93c3e822e6e1deb3ec43eae7911107','','','http://genome.ucsc.edu'),(26,'Lab','Carrell','','CarrellLab','a29785ca0d8149bf02fce169d8492e5d','','','http://genome.ucsc.edu'),(27,'Lab','Hunt','','HuntLab','ab33eec8fdc9b35ae23c3ee365154971','','','http://genome.ucsc.edu'),(28,'Lab','Orosz','','OroszLab','01b4857805f9fff24d8394289647ce07','','','http://genome.ucsc.edu'),(29,'Lab','Moon','','MoonLab','ce1ff1405a8fb2e4cadd98d89fbf8744','','','http://genome.ucsc.edu'),(30,'Group','UARP','','UARP','ef18b6840f6e83c1294a975993903d5b','','','http://genome.ucsc.edu'),(31,'Lab','Bernard','','BernardLab','2ef9102d101983c000c655e20221ff6f','','','http://genome.ucsc.edu'),(32,'Lab','Bass','','BassLab','b433c284efa77c8992cfa02c4d6a6987','','','http://genome.ucsc.edu'),(33,'Lab','Ayer','','AyerLab','d511968dbffa9b211f2ab79ac274321e','','','http://genome.ucsc.edu'),(34,'Hammoud','Sue','','SueHammoud','0283a1153964eff04dc7b426e3f57bfd','Sue.Hammoud@utah.edu','HCI','http://genome.ucsc.edu'),(35,'Bradley','Diana','','DianaBradley','c1561bef1636f221a76cc208a5c2ea08','','','http://genome.ucsc.edu'),(37,'Milash','Brett','','bmilash','8e1e45e0ff155d29aaf8fe7228d155f8','brett.milash@hci.utah.edu','U of Utah / HCI','http://genome.ucsc.edu'),(38,'Firpo','Matt','','matt.firpo','e070780431ee4d8a0656ffe9ba3a71e4','','','http://genome.ucsc.edu'),(39,'Johnson','Evan','','evan_johnson','f86be5a1e3b95979a78e3b513f99bd12','','','http://genome.ucsc.edu'),(40,'Tolley','Neal','','tolley','ad28bd2dab674ea1e37eab2d1ee23264','','','http://genome.ucsc.edu'),(41,'User','Test','','test','10da2b7d66130f53e1ec4f1fe4e9235b','','','http://genome.ucsc.edu'),(42,'Chandler','Katherine','','Katherine','dffebb2f6107d1ad87c14921b6b7e944','','','http://genome.ucsc.edu'),(43,'Oler','Andrew','','oler','a2117fca023c313a413457b746846ba5','andrew.oler@nih.gov','NIH','https://genome.niaid.nih.gov'),(44,'Clark','Richard','','Clark','18fb92a0c7e84fb8b290bbdf4b6e6299','clark@biology.utah.edu','U of U Biology Dept','http://genome.ucsc.edu'),(45,'Sankar','Savita','','Savita','6733cd1c30c9ae99e11deb6ee0dae127','Savita.Sankar@hci.utah.edu','HCI','http://genome.ucsc.edu'),(46,'Fadul','John','','John.Fadul','0fdc9dca63c476ce57851b1174294605','JohnPhillip.Fadul@hci.utah.edu','HCI','http://genome.ucsc.edu'),(47,'Lessnick','Stephen','','Stephen','b69bae71959c797fba43ef96b0a64674','Stephen.Lessnick@hci.utah.edu','HCI','http://genome.ucsc.edu'),(48,'Yoon','Dong','','YoonLab','fdb5de232f88279074bb61111b36fe85','dong.yoon@hsc.utah.edu','U of Utah Hematology Dept','http://genome.ucsc.edu'),(49,'Jette','Cicely','','cjette','b8d2d215bef8d5ef22c28ad26ebfef20','cicely.jette@hci.utah.edu','HCI, U of Utah','http://genome.ucsc.edu'),(50,'Sun','Ying','','Ying','75d634436d582e61da90d243c7ed8d65','ying.sun@hci.utah.edu','HCI','http://genome.ucsc.edu'),(51,'Bell','Russell','','Russell','f5f108a953e2e7f5e4fbf40736cd3220','russell.bell@hci.utah.edu','','http://genome.ucsc.edu'),(52,'Lyon','Gholson','','Lyon','2619eebbeda2d9a16ce71b6f7f344cb9','Gholson.Lyon@hsc.utah.edu','U of Utah','http://genome.ucsc.edu'),(53,'Robison','Reid','','Reid','0960ac4f5b55624724ba880c4da239c1','Reid.Robison@hsc.utah.edu','U of Utah','http://genome.ucsc.edu'),(54,'Moore','Barry','','Moore','da0e6825412e5b0e2895b0efef603f66','barry.moore@genetics.utah.edu','UofU','http://genome.ucsc.edu'),(55,'Wang','Kai','','kaichop','2bb21782a05daea6afedb93252a3234f','kaichop@gmail.com','CHOP','http://genome.ucsc.edu'),(56,'Yang','Lin','','YangLin','9cc4f3661f6433fec7de8a210a7662ca','yanglin@genomics.org.cn','BGI','http://genome.ucsc.edu'),(57,'Fain','Heidi','','Fain','74c1791b7b5cfc3b7c6af552c41025f6','heidi.fain@utah.edu','','http://genome.ucsc.edu'),(58,'Pahwa','Anuj','','cggroup','844d5878e1d27fab8274be0fb9d5ae83','anuj.pahwa@anu.edu.au','Australian National University','http://genome.ucsc.edu'),(59,'Moon','Anne',NULL,'moon',NULL,NULL,NULL,'http://genome.ucsc.edu'),(60,'Chen','Karin','','karinchen','52a298dff2a33bc0877f1b08d30b51a6','karin.chen@hsc.utah.edu','Eccles dbo.gpInstitute of Human Genetics','http://genome.ucsc.edu'),(61,'Hobbs','Maurine','','maurine.hobbs','fd965e2eb360aba778c77ec754d6a21f','m.hobbs@hsc.utah.edu','University of Utah','http://genome.ucsc.edu'),(62,'Kriesel','John','','john.kriesel','f07f1226777cba25d52674db30013640','john.kriesel@hsc.utah.edu','University of Utah','http://genome.ucsc.edu'),(63,'Purwar','Jahnvi','','jahnvi','aa3b705add218eafb45a739736052a3c','','','http://genome.ucsc.edu'),(65,'Hu','Hao','','hao.hu','65c1db6330804fb575bff31cb842fe00','yunathestar@gmail.com','Eccles dbo.gpInstitute of Human Genetics','http://genome.ucsc.edu'),(66,'Grunwald','David','','david.grunwald','39c612771cf5a547f1a804969a4ae118','dgrunwal@genetics.utah.edu','U of Utah','http://genome.ucsc.edu'),(67,'Jurynec','Mick','','mick.jurynec','d0b221e2c26126854d3efc4d07e63dcd','mjurynec@genetics.utah.edu','U of Utah','http://genome.ucsc.edu'),(68,'Nelson','Adam','','adam','5e08b1301d280321defb31fc60a6fb74','adam.nelson@utah.edu','U of Utah','http://genome.ucsc.edu'),(69,'Potok','Magda','','Magda','3a90315139ef679a18aab2a2b36166f7','Magdalena.Potok@hci.utah.edu','HCI','http://genome.ucsc.edu'),(70,'Neklason','Deb','','deb.neklason','8a743d246936c54a4434fbbce2b93726','deb.neklason@hci.utah.edu','HCI','http://genome.ucsc.edu'),(71,'Wiles','Travis','','Travis','078a9ab9e4ac621e65b1bd7c667241f4','travis.wiles@utah.edu','U of U Pathology Dept','http://genome.ucsc.edu'),(72,'Weeks','Josh','','jweeks','4fdb1c133a0a5eddb7ebb1e10539c52f','','','http://genome.ucsc.edu'),(73,'','Visitor','','','9d9db6877f2837c5e5d5d31102377213',NULL,NULL,'http://genome.ucsc.edu'),(74,'Welm','Bryan','','bryan.welm','4af596fff7411a916a88d265b3f0ceb9','bryan.welm@hci.utah.edu','U of Utah','http://genome.ucsc.edu'),(75,'Basham','Katie','','katie.basham','5dee5ff0b5afe5a05d33392f41bbcb6d','kaitlinbasham@hci.utah.edu','U of Utah','http://genome.ucsc.edu'),(76,'Matsen','Cindy','','cindy.matsen','44f698e98a48a91d7b735826918c9ce7','cindy.matsen@hci.utah.edu','U of Utah','http://genome.ucsc.edu'),(77,'Deininger','Michael','','Deininger','9d007b303ce49f7cd8901cff1087481d','Michael.Deininger@hci.utah.edu','','http://genome.ucsc.edu'),(78,'Ebbert','Mark','','mebbert','469747cec9f47b321872f2c53a2590c7','mark.ebbert@hci.utah.edu','HCI / U of Utah','http://genome.ucsc.edu'),(79,'Toomey','Tish','','tish.toomey','619aac1ca4d7e9619f0c9541e85022e3','elizabeth.toomey@hci.utah.edu','HCI / U of Utah','http://genome.ucsc.edu'),(80,'Ames','Darren','','u0785353','491567f0be11dc9bf5c22cb6521c386a','darren.ames@hci.utah.edu','HCI','http://genome.ucsc.edu'),(81,'Dehghanizadeh','Somaye','','Somaye','ee9d24d753cfce09480bfddaefcc4f4d','Somaye.Dehghanizadeh@hci.utah.edu','','http://genome.ucsc.edu'),(82,'Sloan','Elizabeth','','elizabeth.sloan','bef96512da087c6b8d5278860ccc4492','elizabeth.sloan@hci.utah.edu','HCI / U of Utah','http://genome.ucsc.edu'),(83,'Bend','Renee','','renee.bend','a33ec29bd3317483cc0c79ada9def77d','reneebend@gmail.com','U of Utah','http://genome.ucsc.edu'),(84,'Guccione','Ernesto','','Guccione','25b9a96ae67fa087ed8091e57a82858c','eguccione@imcb.a-star.edu.sg','Sinapore - Cairns lab collaborator','http://genome.ucsc.edu');
ALTER TABLE dbo.gpUser CHECK CONSTRAINT all;
SET IDENTITY_INSERT dbo.gpUser OFF;


--
-- Table structure for table dbo.gpUserGroup --

IF  EXISTS (SELECT * FROM sys.objects WHERE object_id = OBJECT_ID(N'dbo.gpUserGroup') AND type in (N'U')) DROP TABLE dbo.gpUserGroup;


CREATE TABLE dbo.gpUserGroup (
  idUserGroup int  NOT NULL identity,
  name varchar(200) NOT NULL,
  email varchar(500)  NULL,
  institute varchar(200)  NULL,
  contact varchar(500)  NULL,
CONSTRAINT [PK_UserGroup] PRIMARY KEY CLUSTERED 
(
	[idUserGroup] ASC
)
)  ;


--
-- Dumping data for table dbo.gpUserGroup --


ALTER TABLE dbo.gpUserGroup NOCHECK CONSTRAINT all;
SET IDENTITY_INSERT dbo.gpUserGroup ON;
INSERT INTO dbo.gpUserGroup 
( idUserGroup,
  name,
  email,
  institute,
  contact
)
VALUES (7,'Cairns Lab','brad.cairns@hci.utah.edu','Hunstman Cancer Institute','Brad Cairns'),(8,'UARP','','',''),(9,'Wang Lab','l.wang@hsc.utah.edu','University of Utah - Gastroenterology','Li Wang'),(10,'Lessnick Lab','stephen.lessnick@hci.utah.edu','Hunstman Cancer Institute','Stephen Lessnick'),(11,'GenEPI','','',''),(12,'Leppert Lab','mleppert@genetics.utah.edu','University of Utah - Dept. of Human Genetics','Mark Leppert'),(13,'Graves Lab','barbara.graves@hci.utah.edu','Hunstman Cancer Institute','Barbara Graves'),(14,'Jones Lab','david.jones@hci.utah.edu','Hunstman Cancer Institute','David Jones'),(15,'ARUP','','',''),(16,'Hagedorn Lab','curt.hagedorn@hsc.utah.edu','University of Utah - Gastroenterology','Curt Hagedorn'),(17,'Ayer Lab','don.ayer@hci.utah.edu','Hunstman Cancer Institute','Don Ayer'),(18,'Schiffman Lab','','',''),(19,'Weis Lab','','',''),(20,'Tantin Lab','dean.tantin@path.utah.edu','University of Utah - Pathology','Dean Tantin'),(21,'Carrell Lab','douglas.carrell@hsc.utah.edu','University of Utah Andrology','Douglas Carrell'),(22,'Hunt Lab','steve.hunt@utah.edu','University of Utah - Cardiovascular Genetics','Steve Hunt'),(23,'Orosz Lab','','',''),(24,'Moon Lab','anne.moon@genetics.utah.edu','University of Utah - Human Biology and Genetics','Ann Moon'),(25,'Bass Lab','bbass@biochem.utah.edu','University of Utah - Biochemistry Dept.','Brenda Bass'),(26,'Bernard Lab','phil.bernard@hci.utah.edu','Hunstman Cancer Institute','Phil Bernard'),(28,'Admin','','',''),(29,'Trede Lab','nikolaus.trede@hci.utah.edu','HCI','Nick Trede'),(30,'Weyrich Lab','','',''),(31,'Firpo Lab','','','Matt Firpo'),(32,'BYU','','',''),(33,'Test Lab','','',''),(34,'Clark Lab','clark@biology.utah.edu','U of U Biology Dept','Richard Clark'),(35,'Yoon Lab','dong.yoon@hsc.utah.edu','U of Utah Hematology Dept','Dong Yoon'),(36,'Jette Lab','cicely.jette@hci.utah.edu','HCI, U of Utah','Cicely Jette'),(37,'Tavtigian Lab','Sean.Tavtigian@hci.utah.edu','HCI','Sean Tavtigian'),(38,'Lyon Robison Lab','Gholson.Lyon@hsc.utah.edu','U of Utah','Gholson Lyon'),(39,'Australian National University','anuj.pahwa@anu.edu.au','Australian National University','Anuj Pahwa'),(40,'Yandell','barry.moore@genetics.utah.edu','Eccles dbo.gpInstitute of Human Genetics','Barry Moore'),(41,'Chen Lab','','',''),(42,'Kriesel Lab','','',''),(43,'Grunwald Lab','dgrunwal@genetics.utah.edu','U of Utah','David Grunwald'),(44,'Potts Lab','potts@biology.utah.edu','U of U','Wayne Potts'),(45,'Neklason Lab','deb.neklason@hci.utah.edu',NULL,'Deb Neklason'),(46,'Mulvey Lab','Matthew.Mulvey@path.utah.edu',NULL,'Matt Mulvey'),(47,'Bryan Welm Lab','bryan.welm@hci.utah.edu',NULL,'Bryan Welm'),(48,'Deininger Lab','Michael.Deininger@hci.utah.edu',NULL,'Michael Deininger'),(49,'Utah Bioinformatics Shared Resource','',NULL,''),(50,'Lane Lab','robert.lane@hsc.utah.edu',NULL,'Robert Lane'),(51,'Chien Lab','chi-bin.chien@hsc.utah.edu',NULL,'Chi-bin Chien'),(52,'Guccione Lab','eguccione@imcb.a-star.edu.sg',NULL,'Ernesto Guccione');
ALTER TABLE dbo.gpUserGroup CHECK CONSTRAINT all;
SET IDENTITY_INSERT dbo.gpUserGroup OFF;


--
-- Table structure for table dbo.gpUserGroupCollaborator --

IF  EXISTS (SELECT * FROM sys.objects WHERE object_id = OBJECT_ID(N'dbo.gpUserGroupCollaborator') AND type in (N'U')) DROP TABLE dbo.gpUserGroupCollaborator;


CREATE TABLE dbo.gpUserGroupCollaborator (
  idUserGroup int  NOT NULL,
  idUser int  NOT NULL,
 CONSTRAINT [PK_UserGroupCollaborator] PRIMARY KEY CLUSTERED 
(
	[idUserGroup] ASC,
	[idUser] ASC
),
  
  CONSTRAINT FK_UserGroupCollaborator_User FOREIGN KEY (idUser) REFERENCES dbo.gpUser (idUser),
  CONSTRAINT FK_UserGroupCollaborator_UserGroup FOREIGN KEY (idUserGroup) REFERENCES dbo.gpUserGroup (idUserGroup)
) ;


--
-- Dumping data for table dbo.gpUserGroupCollaborator --


ALTER TABLE dbo.gpUserGroupCollaborator NOCHECK CONSTRAINT all;
INSERT INTO dbo.gpUserGroupCollaborator (idUserGroup, idUser)  
 VALUES (13,29),(40,29),(51,37),(47,78),(7,82),(7,84);
ALTER TABLE dbo.gpUserGroupCollaborator CHECK CONSTRAINT all;



--
-- Table structure for table dbo.gpUserGroupManager --

IF  EXISTS (SELECT * FROM sys.objects WHERE object_id = OBJECT_ID(N'dbo.gpUserGroupManager') AND type in (N'U')) DROP TABLE dbo.gpUserGroupManager;


CREATE TABLE dbo.gpUserGroupManager (
  idUserGroup int  NOT NULL,
  idUser int  NOT NULL ,
CONSTRAINT [PK_UserGroupManager] PRIMARY KEY CLUSTERED 
(
	[idUserGroup] ASC,
	[idUser] ASC
),
  
  CONSTRAINT FK_UserGroupManager_User FOREIGN KEY (idUser) REFERENCES dbo.gpUser (idUser),
  CONSTRAINT FK_UserGroupManager_UserGroup FOREIGN KEY (idUserGroup) REFERENCES dbo.gpUserGroup (idUserGroup)
)  ;


--
-- Dumping data for table dbo.gpUserGroupManager --


ALTER TABLE dbo.gpUserGroupManager NOCHECK CONSTRAINT all;
INSERT INTO dbo.gpUserGroupManager (idUserGroup, idUser) VALUES (28,3),(41,3),(28,4),(7,34),(52,34),(28,37),(31,38),(32,39),(30,40),(33,41),(13,42),(7,43),(34,44),(10,45),(10,46),(10,47),(35,48),(28,50),(10,51),(37,51),(38,52),(38,53),(40,54),(39,58),(24,59),(41,60),(48,77),(52,84);
ALTER TABLE dbo.gpUserGroupManager CHECK CONSTRAINT all;


--
-- Table structure for table dbo.gpUserGroupMember --

IF  EXISTS (SELECT * FROM sys.objects WHERE object_id = OBJECT_ID(N'dbo.gpUserGroupMember') AND type in (N'U')) DROP TABLE dbo.gpUserGroupMember;


CREATE TABLE dbo.gpUserGroupMember (
  idUserGroup int  NOT NULL,
  idUser int  NOT NULL,
CONSTRAINT [PK_UserGroupMember] PRIMARY KEY CLUSTERED 
(
	[idUserGroup] ASC,
	[idUser] ASC
),
  
  CONSTRAINT FK_UserGroupMember_User FOREIGN KEY (idUser) REFERENCES dbo.gpUser (idUser),
  CONSTRAINT FK_UserGroupMember_UserGroup FOREIGN KEY (idUserGroup) REFERENCES dbo.gpUserGroup (idUserGroup)
)  ;


--
-- Dumping data for table dbo.gpUserGroupMember --


ALTER TABLE dbo.gpUserGroupMember NOCHECK CONSTRAINT all;
INSERT INTO dbo.gpUserGroupMember  (idUserGroup, idUser) VALUES (7,14),(7,63),(7,69),(8,30),(9,15),(10,16),(10,79),(11,17),(12,18),(13,19),(14,20),(14,81),(15,21),(16,22),(17,33),(17,82),(18,23),(19,24),(20,25),(21,26),(22,27),(23,28),(24,29),(24,54),(24,65),(25,32),(26,31),(26,78),(29,35),(30,43),(36,49),(38,39),(38,55),(38,56),(38,57),(40,65),(42,61),(42,62),(43,66),(43,67),(44,68),(45,70),(46,71),(47,74),(47,75),(47,76),(49,3),(49,37),(49,50),(49,51),(49,80),(51,83);
ALTER TABLE dbo.gpUserGroupMember CHECK CONSTRAINT all;


--
-- Table structure for table dbo.gpUserRole --

IF  EXISTS (SELECT * FROM sys.objects WHERE object_id = OBJECT_ID(N'dbo.gpUserRole') AND type in (N'U')) DROP TABLE dbo.gpUserRole;


CREATE TABLE dbo.gpUserRole (
  UserName varchar(30) NOT NULL,
  roleName varchar(30) NOT NULL,
  idUser int  NOT NULL,
  idUserRole int  NOT NULL identity,
CONSTRAINT [PK_UserRole] PRIMARY KEY CLUSTERED 
(
	[idUserRole] ASC
),
  
  
  CONSTRAINT FK_UserRole_User FOREIGN KEY (idUser) REFERENCES dbo.gpUser (idUser),
  CONSTRAINT FK_UserRole_Username FOREIGN KEY (UserName) REFERENCES dbo.gpUser (UserName)
)   ;


--
-- Dumping data for table dbo.gpUserRole --


ALTER TABLE dbo.gpUserRole NOCHECK CONSTRAINT all;
SET IDENTITY_INSERT dbo.gpUserRole ON;
INSERT INTO dbo.gpUserRole 
(  UserName,
  roleName,
  idUser,
  idUserRole
)
VALUES ('guest','guest',1,1),('scourdy','admin',12,13),('CairnsLab','user',14,15),('WangLab','user',15,16),('LessnickLab','user',16,17),('GenEpi','user',17,18),('LeppertLab','user',18,19),('GravesLab','user',19,20),('JonesLab','user',20,21),('ARUP','user',21,22),('HagedornLab','user',22,23),('SchiffmanLab','user',23,24),('WeisLab','user',24,25),('TantinLab','user',25,26),('CarrellLab','user',26,27),('HuntLab','user',27,28),('OroszLab','user',28,30),('MoonLab','user',29,31),('UARP','user',30,32),('BernardLab','user',31,33),('BassLab','user',32,35),('AyerLab','user',33,36),('SueHammoud','user',34,38),('DianaBradley','user',35,39),('bmilash','admin',37,41),('matt.firpo','user',38,42),('u0028003','admin',3,43),('tdisera','admin',4,45),('evan_johnson','user',39,46),('tolley','user',40,47),('test','user',41,48),('Katherine','user',42,49),('Clark','user',44,51),('Savita','user',45,52),('Stephen','user',47,54),('cjette','user',49,57),('Ying','admin',50,58),('Russell','user',51,59),('Reid','user',53,61),('Moore','user',54,62),('kaichop','user',55,63),('YangLin','user',56,64),('Fain','user',57,65),('cggroup','user',58,66),('moon','user',59,67),('maurine.hobbs','user',61,69),('john.kriesel','user',62,70),('karinchen','user',60,71),('jahnvi','user',63,72),('YoonLab','user',48,74),('hao.hu','user',65,75),('Lyon','user',52,76),('david.grunwald','user',66,77),('mick.jurynec','user',67,78),('John.Fadul','user',46,80),('adam','user',68,81),('Magda','user',69,82),('deb.neklason','user',70,83),('Travis','user',71,84),('jweeks','admin',72,85),('','guest',73,86),('bryan.welm','user',74,87),('katie.basham','user',75,88),('cindy.matsen','user',76,89),('Deininger','user',77,90),('mebbert','user',78,92),('tish.toomey','user',79,93),('oler','user',43,94),('u0785353','admin',80,96),('Somaye','user',81,97),('elizabeth.sloan','user',82,98),('renee.bend','user',83,99),('Guccione','user',84,100);
ALTER TABLE dbo.gpUserRole CHECK CONSTRAINT all;
SET IDENTITY_INSERT dbo.gpUserRole OFF;


--
-- Table structure for table dbo.gpVisibility --

IF  EXISTS (SELECT * FROM sys.objects WHERE object_id = OBJECT_ID(N'dbo.gpVisibility') AND type in (N'U')) DROP TABLE dbo.gpVisibility;


CREATE TABLE dbo.gpVisibility (
  codeVisibility varchar(10) NOT NULL default '',
  name varchar(100) NOT NULL,
CONSTRAINT [PK_Visibility] PRIMARY KEY CLUSTERED 
(
	[codeVisibility] ASC
)
) ;


--
-- Dumping data for table dbo.gpVisibility --


ALTER TABLE dbo.gpVisibility NOCHECK CONSTRAINT all;
INSERT INTO dbo.gpVisibility VALUES ('INST','Institute'),('MEM','Members'),('MEMCOL','Members and Collaborators'),('OWNER','Owner'),('PUBLIC','Public');
ALTER TABLE dbo.gpVisibility CHECK CONSTRAINT all;








--
-- Table structure for table dbo.gpOrganism --

IF  EXISTS (SELECT * FROM sys.objects WHERE object_id = OBJECT_ID(N'dbo.gpOrganism') AND type in (N'U')) DROP TABLE dbo.gpOrganism;


CREATE TABLE dbo.gpOrganism (
  idOrganism int  NOT NULL identity,
  commonName varchar(100) NOT NULL,
  sortOrder int   NULL,
  binomialName varchar(200) NOT NULL,
  NCBITaxID varchar(45)  NULL,
  idUser int   NULL,
  name varchar(200) NOT NULL,
CONSTRAINT [PK_Organism] PRIMARY KEY CLUSTERED 
(
	[idOrganism] ASC
),
  CONSTRAINT Index_OrganismName UNIQUE (name),
  CONSTRAINT Index_OrganismBinomialName UNIQUE (binomialName)
)  ;


--
-- Dumping data for table dbo.gpOrganism --


ALTER TABLE dbo.gpOrganism NOCHECK CONSTRAINT all;
SET IDENTITY_INSERT dbo.gpOrganism ON;
INSERT INTO dbo.gpOrganism 
(idOrganism,
  commonName,
  sortOrder,
  binomialName,
  NCBITaxID,
  idUser,
  name
)
VALUES (400,'A. thaliana',NULL,'Arabidopsis thaliana',NULL,NULL,'A_thaliana'),(1200,'C. elegans',NULL,'Caenorhabditis elegans','6239',NULL,'C_elegans'),(1900,'D. melanogaster',NULL,'Drosophila melanogaster',NULL,NULL,'D_melanogaster'),(2700,'Zebrafish',NULL,'Danio rerio','7955',NULL,'D_rerio'),(2900,'E. coli',NULL,'Escherichia coli',NULL,NULL,'E_coli'),(3100,'Chicken',NULL,'Gallus gallus',NULL,NULL,'G_gallus'),(3400,'Human',NULL,'Homo sapiens','9606',NULL,'H_sapiens'),(3800,'Mouse',NULL,'Mus musculus','10090',NULL,'M_musculus'),(3900,'M. abcessus',NULL,'Mycobacterium abscessus',NULL,NULL,'M_abscessus'),(4100,'M. tuberculosis',NULL,'Mycobacterium tuberculosis',NULL,NULL,'M_tuberculosis'),(5200,'Rat',NULL,'Rattus norvegicus',NULL,NULL,'R_norvegicus'),(5300,'Yeast',NULL,'Saccharomyces cerevisiae',NULL,NULL,'S_cerevisiae'),(5400,'Fission Yeast',NULL,'Schizosaccharomyces pombe','4896',NULL,'S_pombe'),(5500,'S. glossinidius',NULL,'Sodalis glossinidius','343509',NULL,'S_glossinidius'),(6200,'Pipid Frog',NULL,'Xenopus tropicalis','',NULL,'X_tropicalis');
ALTER TABLE dbo.gpOrganism CHECK CONSTRAINT all;
SET IDENTITY_INSERT dbo.gpOrganism OFF;




--
-- Table structure for table dbo.gpGenomeVersion --

IF  EXISTS (SELECT * FROM sys.objects WHERE object_id = OBJECT_ID(N'dbo.gpGenomeVersion') AND type in (N'U')) DROP TABLE dbo.gpGenomeVersion;


CREATE TABLE dbo.gpGenomeVersion (
  idGenomeVersion int  NOT NULL identity,
  name varchar(200) NOT NULL,
  idOrganism int   NULL,
  buildDate datetime  NULL,
  coordURI varchar(2000)  NULL,
  coordVersion varchar(50)  NULL,
  coordSource varchar(50)  NULL,
  coordTestRange varchar(100)  NULL,
  coordAuthority varchar(50)  NULL,
  ucscName varchar(50)  NULL,
  dataPath varchar(500)  NULL,
CONSTRAINT [PK_GenomeVersion] PRIMARY KEY CLUSTERED 
(
	[idGenomeVersion] ASC
),
  CONSTRAINT Index_GenomeVersionName UNIQUE (name),
  
  CONSTRAINT FK_GenomeVersion_Organism FOREIGN KEY (idOrganism) REFERENCES dbo.gpOrganism (idOrganism)
)  ;


--
-- Dumping data for table dbo.gpGenomeVersion --


ALTER TABLE dbo.gpGenomeVersion NOCHECK CONSTRAINT all;
SET IDENTITY_INSERT dbo.gpGenomeVersion ON;
INSERT INTO dbo.gpGenomeVersion 
( idGenomeVersion,
  name,
  idOrganism,
  buildDate,
  coordURI,
  coordVersion,
  coordSource,
  coordTestRange,
  coordAuthority,
  ucscName,
  dataPath
)
VALUES (400,'A_thaliana_Jan_2004',400,'2004-01-01 00:00:00',NULL,NULL,NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),(1204,'C_elegans_May_2008',1200,'2008-05-01 00:00:00','','','','','','ce6','/Data02/GenoPub/GenoPubData/'),(1901,'D_melanogaster_Apr_2006',1900,'2006-04-01 00:00:00',NULL,NULL,NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),(2700,'D_rerio_Mar_2006',2700,'2006-03-01 00:00:00',NULL,NULL,NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),(2701,'D_rerio_Jul_2007',2700,'2007-07-01 00:00:00','http://zfin.org/genome/D_rerio/Zv7/','Zv7','Chromosome','','ZFISH_7','danRer5','/Data02/GenoPub/GenoPubData/'),(3100,'G_gallus_May_2006',3100,'2006-05-01 00:00:00',NULL,NULL,NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),(3404,'H_sapiens_May_2004',3400,'2004-05-01 00:00:00','http://www.ncbi.nlm.nih.gov/genome/H_sapiens/B35.1/','35','Chromosome','','NCBI','hg17','/Data02/GenoPub/GenoPubData/'),(3405,'H_sapiens_Mar_2006',3400,'2006-03-01 00:00:00','http://www.ncbi.nlm.nih.gov/genome/H_sapiens/B36.1/','36','Chromosome','','NCBI','hg18','/Data02/GenoPub/GenoPubData/'),(3804,'M_musculus_Mar_2006',3800,'2006-03-01 00:00:00','','','','','','mm8','/Data02/GenoPub/GenoPubData/'),(3806,'M_musculus_Jul_2007',3800,'2007-07-01 00:00:00','','','','','','mm9','/Data02/GenoPub/GenoPubData/'),(3900,'M_abscessus_Mar_2008',3900,'2008-03-01 00:00:00',NULL,NULL,NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),(4100,'M_tuberculosis_Sep_2008',4100,'2008-09-01 00:00:00','','','','','',NULL,'/Data02/GenoPub/GenoPubData/'),(5202,'R_norvegicus_Nov_2004',5200,'2004-11-01 00:00:00',NULL,NULL,NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),(5207,'S_cerevisiae_May_2008',5300,'2008-05-01 00:00:00','','','','','',NULL,'/Data02/GenoPub/GenoPubData/'),(5400,'S_pombe_Sep_2004',5400,'2004-09-01 00:00:00',NULL,NULL,NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),(5401,'S_pombe_Apr_2007',5400,'2007-04-01 00:00:00','http://www.sanger.ac.uk/Projects/S_pombe/Apr_2007','Apr_2007','Chromosome','','Sanger',NULL,'/Data02/GenoPub/GenoPubData/'),(5500,'S_glossinidius_Jan_2006',5500,'2006-01-01 00:00:00','ftp://ftp.ncbi.nih.gov/genomes/Bacteria/Sodalis_glossinidius_morsitans/Jan_2006','Jan_2006','Chromosome',NULL,'NCBI',NULL,'/Data02/GenoPub/GenoPubData/'),(6201,'X_tropicalis_Aug_2005',6200,'2005-08-01 00:00:00','','','','','',NULL,'/Data02/GenoPub/GenoPubData/'),(6300,'H_sapiens_Feb_2009',3400,'2009-02-01 00:00:00','http://www.ncbi.nlm.nih.gov/genome/H_sapiens/GRCh37/','','','','','hg19','/Data02/GenoPub/GenoPubData/'),(6301,'D_rerio_Dec_2008',2700,'2008-12-01 00:00:00','','','','','','danRer6','/Data02/GenoPub/GenoPubData/'),(6303,'D_rerio_Jul_2010',2700,'2010-07-22 00:00:00','http://www.ncbi.nlm.nih.gov/projects/genome/assembly/grc/zebrafish/index.shtml','','','','','danRer7','/Data02/GenoPub/GenoPubData/'),(6305,'E_coli_F11_Jun_2008',2900,'2008-06-20 00:00:00','http://www.ncbi.nlm.nih.gov/sites/entrez?Db=genome&Cmd=ShowDetailView&TermToSearch=5339','http://www.ncbi.nlm.nih.gov/Traces/wgs/?val=AAJU02','','','NCBI',NULL,'/Data02/GenoPub/GenoPubData/');
ALTER TABLE dbo.gpGenomeVersion CHECK CONSTRAINT all;
SET IDENTITY_INSERT dbo.gpGenomeVersion OFF;


--
-- Table structure for table dbo.gpGenomeVersionAlias --

IF  EXISTS (SELECT * FROM sys.objects WHERE object_id = OBJECT_ID(N'dbo.gpGenomeVersionAlias') AND type in (N'U')) DROP TABLE dbo.gpGenomeVersionAlias;


CREATE TABLE dbo.gpGenomeVersionAlias (
  idGenomeVersionAlias int  NOT NULL identity,
  alias varchar(100) NOT NULL,
  idGenomeVersion int  NOT NULL,
CONSTRAINT [PK_GenomeVersionAlias] PRIMARY KEY CLUSTERED 
(
	[idGenomeVersionAlias] ASC
),
  
  CONSTRAINT FK_GenomeVersionAlias_GenomeVersion FOREIGN KEY (idGenomeVersion) REFERENCES dbo.gpGenomeVersion (idGenomeVersion)
)  ;


--
-- Dumping data for table dbo.gpGenomeVersionAlias --


ALTER TABLE dbo.gpGenomeVersionAlias NOCHECK CONSTRAINT all;
SET IDENTITY_INSERT dbo.gpGenomeVersionAlias ON;
INSERT INTO dbo.gpGenomeVersionAlias 
( idGenomeVersionAlias ,
  alias,
  idGenomeVersion 
)
VALUES (1,'spApr07',6),(4,'sgJan06',7),(5,'danRer5',5),(6,'zv7',5),(7,'Zv7',5),(8,'hg17',9),(9,'hg18',1);
ALTER TABLE dbo.gpGenomeVersionAlias CHECK CONSTRAINT all;
SET IDENTITY_INSERT dbo.gpGenomeVersionAlias OFF;


--
-- Table structure for table dbo.gpInstitute --

IF  EXISTS (SELECT * FROM sys.objects WHERE object_id = OBJECT_ID(N'dbo.gpInstitute') AND type in (N'U')) DROP TABLE dbo.gpInstitute;


CREATE TABLE dbo.gpInstitute (
  idInstitute int  NOT NULL identity,
  name varchar(200) NOT NULL,
  description varchar(500)  NULL,
  isActive char(1) default  'Y',

CONSTRAINT [PK_Institute] PRIMARY KEY CLUSTERED 
(
	[idInstitute] ASC
)
  
)  ;


--
-- Dumping data for table dbo.gpInstitute --


ALTER TABLE dbo.gpInstitute NOCHECK CONSTRAINT all;
SET IDENTITY_INSERT dbo.gpInstitute ON;
INSERT INTO dbo.gpInstitute 
(idInstitute,
name,
description,
isActive
)
VALUES (1,'University of Utah','U of U SLC','Y'),(2,'Huntsman Cancer Institute','@ University of Utah','Y');
ALTER TABLE dbo.gpInstitute CHECK CONSTRAINT all;
SET IDENTITY_INSERT dbo.gpInstitute OFF;


--
-- Table structure for table dbo.gpInstituteUserGroup --

IF  EXISTS (SELECT * FROM sys.objects WHERE object_id = OBJECT_ID(N'dbo.gpInstituteUserGroup') AND type in (N'U')) DROP TABLE dbo.gpInstituteUserGroup;


CREATE TABLE dbo.gpInstituteUserGroup (
  idInstitute int  NOT NULL,
  idUserGroup int  NOT NULL,
CONSTRAINT [PK_InstituteUserGroup] PRIMARY KEY CLUSTERED 
(
	[idInstitute] ASC,
	[idUserGroup] ASC
),
  
  CONSTRAINT FK_InstituteUserGroup_Institute FOREIGN KEY (idInstitute) REFERENCES dbo.gpInstitute (idInstitute) ON DELETE NO ACTION ON UPDATE NO ACTION,
  CONSTRAINT FK_InstituteUserGroup_UserGroup FOREIGN KEY (idUserGroup) REFERENCES dbo.gpUserGroup (idUserGroup) ON DELETE NO ACTION ON UPDATE NO ACTION
)  ;


--
-- Dumping data for table dbo.gpInstituteUserGroup --


ALTER TABLE dbo.gpInstituteUserGroup NOCHECK CONSTRAINT all;
INSERT INTO dbo.gpInstituteUserGroup (idInstitute, idUserGroup) VALUES (1,46),(1,47),(1,48),(2,48),(1,49),(2,49),(1,50);
ALTER TABLE dbo.gpInstituteUserGroup CHECK CONSTRAINT all;


--
-- Table structure for table dbo.gpPropertyType --

IF  EXISTS (SELECT * FROM sys.objects WHERE object_id = OBJECT_ID(N'dbo.gpPropertyType') AND type in (N'U')) DROP TABLE dbo.gpPropertyType;


CREATE TABLE dbo.gpPropertyType (
  codePropertyType varchar(10) NOT NULL,
  name varchar(200) NOT NULL,
  isActive char(1) default 'Y',
CONSTRAINT [PK_PropertyType] PRIMARY KEY CLUSTERED 
(
	[codePropertyType] ASC
)
  
)  ;


--
-- Dumping data for table dbo.gpPropertyType --


ALTER TABLE dbo.gpPropertyType NOCHECK CONSTRAINT all;
INSERT INTO dbo.gpPropertyType VALUES ('CHECK','Checkbox','Y'),('MOPTION','Option (Multiple selection)','Y'),('OPTION','Option (Single selection)','Y'),('TEXT','Text','Y'),('URL','URL','Y');
ALTER TABLE dbo.gpPropertyType CHECK CONSTRAINT all;



--
-- Table structure for table dbo.gpProperty --

IF  EXISTS (SELECT * FROM sys.objects WHERE object_id = OBJECT_ID(N'dbo.gpProperty') AND type in (N'U')) DROP TABLE dbo.gpProperty;


CREATE TABLE dbo.gpProperty (
  idProperty int  NOT NULL identity,
  name varchar(200) NOT NULL,
  isActive char(1) default  'Y',
  codePropertyType varchar(10) NOT NULL,
  idUser int   NULL,
  sortOrder int   NULL,
CONSTRAINT [PK_Property] PRIMARY KEY CLUSTERED 
(
	[idProperty] ASC
),
  
  
  CONSTRAINT FK_Property_PropertyType FOREIGN KEY (codePropertyType) REFERENCES dbo.gpPropertyType (codePropertyType),
  CONSTRAINT FK_Property_User FOREIGN KEY (idUser) REFERENCES dbo.gpUser (idUser)
)  ;


--
-- Dumping data for table dbo.gpProperty --


ALTER TABLE dbo.gpProperty NOCHECK CONSTRAINT all;
SET IDENTITY_INSERT dbo.gpProperty ON;
INSERT INTO dbo.gpProperty 
(idProperty,
name,
isActive,
codePropertyType,
idUser,
sortOrder
)
VALUES (1,'Analysis Type','Y','OPTION',NULL,NULL),(2,'Experiment Platform','Y','OPTION',NULL,NULL),(3,'Experiment Method','Y','OPTION',NULL,NULL),(4,'Analysis URL Links','Y','URL',NULL,NULL);
ALTER TABLE dbo.gpProperty CHECK CONSTRAINT all;
SET IDENTITY_INSERT dbo.gpProperty OFF;


--
-- Table structure for table dbo.gpPropertyOption --

IF  EXISTS (SELECT * FROM sys.objects WHERE object_id = OBJECT_ID(N'dbo.gpPropertyOption') AND type in (N'U')) DROP TABLE dbo.gpPropertyOption;


CREATE TABLE dbo.gpPropertyOption (
  idPropertyOption int  NOT NULL identity,
  name varchar(200) NOT NULL,
  isActive char(1) default 'Y',
  idProperty int  NOT NULL,
  sortOrder int   NULL,
CONSTRAINT [PK_PropertyOption] PRIMARY KEY CLUSTERED 
(
	[idPropertyOption] ASC
),
  
  CONSTRAINT FK_PropertyOption_Property FOREIGN KEY (idProperty) REFERENCES dbo.gpProperty (idProperty)
)  ;


--
-- Dumping data for table dbo.gpPropertyOption --


ALTER TABLE dbo.gpPropertyOption NOCHECK CONSTRAINT all;
SET IDENTITY_INSERT dbo.gpPropertyOption ON;
INSERT INTO dbo.gpPropertyOption 
(idPropertyOption,
name,
isActive,
idProperty,
sortOrder
)VALUES (1,'','Y',1,0),(2,'Dynamic/Differential','Y',1,1),(3,'Static','Y',1,2),(4,'','Y',2,0),(5,'Illumina Genome Analyzer','Y',2,1),(6,'Agilent Microarray','Y',2,2),(7,'Affymetrix Microarray','Y',2,3),(8,'454 Genome Sequencer','Y',2,4),(9,'Applied Biosystems SOLiD','Y',2,5),(10,'Helicos Genomic Signature Sequencing','Y',2,6),(11,'','Y',3,0),(12,'chIP-Seq','Y',3,1),(13,'chIP-chip','Y',3,2),(14,'CGN Microarray','Y',3,3),(15,'SNP Microarray','Y',3,4),(16,'Transcriptome Microarray','Y',3,5),(17,'Gene Expression Microarray','Y',3,6),(18,'Quantitative rtPCR','Y',3,7),(19,'SAGE','Y',3,8),(20,'Read Counts/ Coverage','Y',3,9),(21,'RNA-Seq','Y',3,10),(22,'Digital DNaseI','Y',3,11),(23,'RIP-Seq','Y',3,12),(24,'CAGE','Y',3,13),(25,'Cap-Seq','Y',3,14),(26,'Small RNA-Seq','Y',3,15),(27,'Bis-Seq','Y',3,16),(28,'Whole Genome Sequencing','Y',3,17);
ALTER TABLE dbo.gpPropertyOption CHECK CONSTRAINT all;
SET IDENTITY_INSERT dbo.gpPropertyOption OFF;


--
-- Table structure for table dbo.gpSegment --

IF  EXISTS (SELECT * FROM sys.objects WHERE object_id = OBJECT_ID(N'dbo.gpSegment') AND type in (N'U')) DROP TABLE dbo.gpSegment;


CREATE TABLE dbo.gpSegment (
  idSegment int  NOT NULL identity,
  length int  NOT NULL,
  name varchar(100) NOT NULL,
  idGenomeVersion int  NOT NULL,
  sortOrder int  NOT NULL,
CONSTRAINT [PK_Segment] PRIMARY KEY CLUSTERED 
(
	[idSegment] ASC
),
  
  CONSTRAINT FK_Segment_GenomeVersion FOREIGN KEY (idGenomeVersion) REFERENCES dbo.gpGenomeVersion (idGenomeVersion)
)  


--
-- Dumping data for table dbo.gpSegment --


ALTER TABLE dbo.gpSegment NOCHECK CONSTRAINT all;
SET IDENTITY_INSERT dbo.gpSegment ON;
INSERT INTO dbo.gpSegment 
(idSegment,
length,
name,
idGenomeVersion,
sortOrder
)
VALUES (1,247249719,'chr1',3405,1),(2,242951149,'chr2',3405,2),(3,199501827,'chr3',3405,3),(4,191273063,'chr4',3405,4),(5,180857866,'chr5',3405,5),(6,170899992,'chr6',3405,6),(7,158821424,'chr7',3405,7),(8,146274826,'chr8',3405,8),(9,140273252,'chr9',3405,9),(10,135374737,'chr10',3405,10),(11,134452384,'chr11',3405,11),(12,132349534,'chr12',3405,12),(13,114142980,'chr13',3405,13),(14,106368585,'chr14',3405,14),(15,100338915,'chr15',3405,15),(16,88827254,'chr16',3405,16),(17,78774742,'chr17',3405,17),(18,76117153,'chr18',3405,18),(19,63811651,'chr19',3405,19),(20,62435964,'chr20',3405,20),(21,46944323,'chr21',3405,21),(22,49691432,'chr22',3405,22),(23,154913754,'chrX',3405,23),(24,57772954,'chrY',3405,24),(25,16571,'chrM',3405,25),(26,56204684,'chr1',2701,1),(27,54366722,'chr2',2701,2),(28,62931207,'chr3',2701,3),(29,42602441,'chr4',2701,4),(30,70371393,'chr5',2701,5),(31,59200669,'chr6',2701,6),(32,70262009,'chr7',2701,7),(33,56456705,'chr8',2701,8),(34,51490918,'chr9',2701,9),(35,42379582,'chr10',2701,10),(36,44616367,'chr11',2701,11),(37,47523734,'chr12',2701,12),(38,53547397,'chr13',2701,13),(39,56522864,'chr14',2701,14),(40,46629432,'chr15',2701,15),(41,53070661,'chr16',2701,16),(42,52310423,'chr17',2701,17),(43,49281368,'chr18',2701,18),(45,46181231,'chr19',2701,19),(46,56528676,'chr20',2701,20),(47,46057314,'chr21',2701,21),(48,38981829,'chr22',2701,22),(49,46388020,'chr23',2701,23),(50,40293347,'chr24',2701,24),(51,32876240,'chr25',2701,25),(52,16596,'chrM',2701,26),(53,122532868,'chrNA',2701,27),(54,45965611,'chrScaffold',2701,28),(60,13794,'chrM',5401,6),(62,4171146,'chr',5500,1),(63,52166,'phiSG1',5500,2),(64,83307,'pSG1',5500,3),(65,27241,'pSG2',5500,4),(66,10811,'pSG3',5500,5),(67,5579133,'chr1',5401,1),(68,4539804,'chr2',5401,2),(69,2452883,'chr3',5401,3),(71,35236,'mat',5401,5),(188,247249719,'chr1',3404,1),(189,242951149,'chr2',3404,2),(190,199501827,'chr3',3404,3),(191,191273063,'chr4',3404,4),(192,180857866,'chr5',3404,5),(193,170899992,'chr6',3404,6),(194,158821424,'chr7',3404,7),(195,146274826,'chr8',3404,8),(196,140273252,'chr9',3404,9),(197,135374737,'chr10',3404,10),(198,134452384,'chr11',3404,11),(199,132349534,'chr12',3404,12),(200,114142980,'chr13',3404,13),(201,106368585,'chr14',3404,14),(202,100338915,'chr15',3404,15),(203,88827254,'chr16',3404,16),(204,78774742,'chr17',3404,17),(205,76117153,'chr18',3404,18),(206,63811651,'chr19',3404,19),(207,62435964,'chr20',3404,20),(208,46944323,'chr21',3404,21),(209,49691432,'chr22',3404,22),(210,154913754,'chrX',3404,23),(211,57772954,'chrY',3404,24),(212,16571,'chrM',3404,25),(213,15072421,'chrI',1204,1),(214,15279323,'chrII',1204,2),(215,13783681,'chrIII',1204,3),(216,17493785,'chrIV',1204,4),(217,20919568,'chrV',1204,5),(218,17718854,'chrX',1204,6),(219,13794,'chrM',1204,7),(254,249250621,'chr1',6300,1),(255,243199373,'chr2',6300,2),(256,198022430,'chr3',6300,3),(257,191154276,'chr4',6300,4),(258,180915260,'chr5',6300,5),(259,171115067,'chr6',6300,6),(260,159138663,'chr7',6300,7),(261,146364022,'chr8',6300,8),(262,141213431,'chr9',6300,9),(263,135534747,'chr10',6300,10),(264,135006516,'chr11',6300,11),(265,133851895,'chr12',6300,12),(266,115169878,'chr13',6300,13),(267,107349540,'chr14',6300,14),(268,102531392,'chr15',6300,15),(269,90354753,'chr16',6300,16),(270,81195210,'chr17',6300,17),(271,78077248,'chr18',6300,18),(272,59128983,'chr19',6300,19),(273,63025520,'chr20',6300,20),(274,48129895,'chr21',6300,21),(275,51304566,'chr22',6300,22),(276,155270560,'chrX',6300,23),(277,59373566,'chrY',6300,24),(278,16571,'chrM',6300,25),(279,590426,'chr4_ctg9_hap1',6300,26),(280,4622290,'chr6_apd_hap1',6300,27),(281,4795371,'chr6_cox_hap2',6300,28),(282,4610396,'chr6_dbb_hap3',6300,29),(283,4683263,'chr6_mann_hap4',6300,30),(284,4833398,'chr6_mcf_hap5',6300,31),(285,4611984,'chr6_qbl_hap6',6300,32),(286,4928567,'chr6_ssto_hap7',6300,33),(287,1680828,'chr17_ctg5_hap1',6300,34),(288,106433,'chr1_gl000191_random',6300,35),(289,547496,'chr1_gl000192_random',6300,36),(290,189789,'chr4_gl000193_random',6300,37),(291,191469,'chr4_gl000194_random',6300,38),(292,182896,'chr7_gl000195_random',6300,39),(293,38914,'chr8_gl000196_random',6300,40),(294,37175,'chr8_gl000197_random',6300,41),(295,90085,'chr9_gl000198_random',6300,42),(296,169874,'chr9_gl000199_random',6300,43),(297,187035,'chr9_gl000200_random',6300,44),(298,36148,'chr9_gl000201_random',6300,45),(299,40103,'chr11_gl000202_random',6300,46),(300,37498,'chr17_gl000203_random',6300,47),(301,81310,'chr17_gl000204_random',6300,48),(302,174588,'chr17_gl000205_random',6300,49),(303,41001,'chr17_gl000206_random',6300,50),(304,4262,'chr18_gl000207_random',6300,51),(305,92689,'chr19_gl000208_random',6300,52),(306,159169,'chr19_gl000209_random',6300,53),(307,27682,'chr21_gl000210_random',6300,54),(308,166566,'chrUn_gl000211',6300,55),(309,186858,'chrUn_gl000212',6300,56),(310,164239,'chrUn_gl000213',6300,57),(311,137718,'chrUn_gl000214',6300,58),(312,172545,'chrUn_gl000215',6300,59),(313,172294,'chrUn_gl000216',6300,60),(314,172149,'chrUn_gl000217',6300,61),(315,161147,'chrUn_gl000218',6300,62),(316,179198,'chrUn_gl000219',6300,63),(317,161802,'chrUn_gl000220',6300,64),(318,155397,'chrUn_gl000221',6300,65),(319,186861,'chrUn_gl000222',6300,66),(320,180455,'chrUn_gl000223',6300,67),(321,179693,'chrUn_gl000224',6300,68),(322,211173,'chrUn_gl000225',6300,69),(323,15008,'chrUn_gl000226',6300,70),(324,128374,'chrUn_gl000227',6300,71),(325,129120,'chrUn_gl000228',6300,72),(326,19913,'chrUn_gl000229',6300,73),(327,43691,'chrUn_gl000230',6300,74),(328,27386,'chrUn_gl000231',6300,75),(329,40652,'chrUn_gl000232',6300,76),(330,45941,'chrUn_gl000233',6300,77),(331,40531,'chrUn_gl000234',6300,78),(332,34474,'chrUn_gl000235',6300,79),(333,41934,'chrUn_gl000236',6300,80),(334,45867,'chrUn_gl000237',6300,81),(335,39939,'chrUn_gl000238',6300,82),(336,33824,'chrUn_gl000239',6300,83),(337,41933,'chrUn_gl000240',6300,84),(338,42152,'chrUn_gl000241',6300,85),(339,43523,'chrUn_gl000242',6300,86),(340,43341,'chrUn_gl000243',6300,87),(341,39929,'chrUn_gl000244',6300,88),(342,36651,'chrUn_gl000245',6300,89),(343,38154,'chrUn_gl000246',6300,90),(344,36422,'chrUn_gl000247',6300,91),(345,39786,'chrUn_gl000248',6300,92),(346,38502,'chrUn_gl000249',6300,93),(369,197069962,'chr1',3804,1),(370,181976762,'chr2',3804,2),(371,159872112,'chr3',3804,3);

INSERT INTO dbo.gpSegment 
(idSegment,
length,
name,
idGenomeVersion,
sortOrder
)
VALUES (372,155029701,'chr4',3804,4),(373,152003063,'chr5',3804,5),(374,149525685,'chr6',3804,6),(375,145134094,'chr7',3804,7),(376,132085098,'chr8',3804,8),(377,124000669,'chr9',3804,9),(378,129959148,'chr10',3804,10),(379,121798632,'chr11',3804,11),(380,120463159,'chr12',3804,12),(381,120614378,'chr13',3804,13),(382,123978870,'chr14',3804,14),(383,103492577,'chr15',3804,15),(384,98252459,'chr16',3804,16),(385,95177420,'chr17',3804,17),(386,90736837,'chr18',3804,18),(387,61321190,'chr19',3804,19),(388,165556469,'chrX',3804,20),(389,16029404,'chrY',3804,21),(390,16299,'chrM',3804,22),(391,5067172,'NC_010397',3900,1),(392,197195432,'chr1',3806,1),(393,181748087,'chr2',3806,2),(394,159599783,'chr3',3806,3),(395,155630120,'chr4',3806,4),(396,152537259,'chr5',3806,5),(397,149517037,'chr6',3806,6),(398,152524553,'chr7',3806,7),(399,131738871,'chr8',3806,8),(400,124076172,'chr9',3806,9),(401,129993255,'chr10',3806,10),(402,121843856,'chr11',3806,11),(403,121257530,'chr12',3806,12),(404,120284312,'chr13',3806,13),(405,125194864,'chr14',3806,14),(406,103494974,'chr15',3806,15),(407,98319150,'chr16',3806,16),(408,95272651,'chr17',3806,17),(409,90772031,'chr18',3806,18),(410,61342430,'chr19',3806,19),(411,166650296,'chrX',3806,20),(412,15902555,'chrY',3806,21),(413,16299,'chrM',3806,22),(414,59305620,'chr1',6301,1),(415,58009534,'chr2',6301,2),(416,60907308,'chr3',6301,3),(417,71658100,'chr4',6301,4),(418,74451498,'chr5',6301,5),(419,61647013,'chr6',6301,6),(420,76918211,'chr7',6301,7),(421,55568185,'chr8',6301,8),(422,54736511,'chr9',6301,9),(423,43467561,'chr10',6301,10),(424,44116856,'chr11',6301,11),(425,46853116,'chr12',6301,12),(426,50748729,'chr13',6301,13),(427,52930158,'chr14',6301,14),(428,47237297,'chr15',6301,15),(429,51890894,'chr16',6301,16),(430,49469313,'chr17',6301,17),(431,49271716,'chr18',6301,18),(432,48708673,'chr19',6301,19),(433,51884995,'chr20',6301,20),(434,47572505,'chr21',6301,21),(435,41415389,'chr22',6301,22),(436,44714728,'chr23',6301,23),(437,40403431,'chr24',6301,24),(438,38768535,'chr25',6301,25),(439,16596,'chrM',6301,26),(440,135290047,'chrNA',6301,27),(441,66135787,'chrScaffold',6301,28),(496,60348388,'chr1',6303,1),(497,60300536,'chr2',6303,2),(498,63268876,'chr3',6303,3),(499,62094675,'chr4',6303,4),(500,75682077,'chr5',6303,5),(501,59938731,'chr6',6303,6),(502,77276063,'chr7',6303,7),(503,56184765,'chr8',6303,8),(504,58232459,'chr9',6303,9),(505,46591166,'chr10',6303,10),(506,46661319,'chr11',6303,11),(507,50697278,'chr12',6303,12),(508,54093808,'chr13',6303,13),(509,53733891,'chr14',6303,14),(510,47442429,'chr15',6303,15),(511,58780683,'chr16',6303,16),(512,53984731,'chr17',6303,17),(513,49877488,'chr18',6303,18),(514,50254551,'chr19',6303,19),(515,55952140,'chr20',6303,20),(516,44544065,'chr21',6303,21),(517,42261000,'chr22',6303,22),(518,46386876,'chr23',6303,23),(519,43947580,'chr24',6303,24),(520,38499472,'chr25',6303,25),(521,16596,'chrM',6303,26),(522,31161985,'chrNA',6303,27),(523,25356215,'chrScaffold',6303,28),(524,22128,'Zv9_NA1',6303,29),(525,16058,'Zv9_NA2',6303,30),(526,20749,'Zv9_NA3',6303,31),(527,41486,'Zv9_NA4',6303,32),(528,15341,'Zv9_NA5',6303,33),(529,6378,'Zv9_NA6',6303,34),(530,5401,'Zv9_NA7',6303,35),(531,11297,'Zv9_NA8',6303,36),(532,20238,'Zv9_NA9',6303,37),(533,48062,'Zv9_NA10',6303,38),(534,81751,'Zv9_NA11',6303,39),(535,10334,'Zv9_NA12',6303,40),(536,12144,'Zv9_NA13',6303,41),(537,16517,'Zv9_NA14',6303,42),(538,22730,'Zv9_NA15',6303,43),(539,14579,'Zv9_NA16',6303,44),(540,69829,'Zv9_NA17',6303,45),(541,4371,'Zv9_NA18',6303,46),(542,16275,'Zv9_NA19',6303,47),(543,32121,'Zv9_NA21',6303,48),(544,67349,'Zv9_NA22',6303,49),(545,9515,'Zv9_NA23',6303,50),(546,53048,'Zv9_NA24',6303,51),(547,17273,'Zv9_NA25',6303,52),(548,15782,'Zv9_NA26',6303,53),(549,59009,'Zv9_NA27',6303,54),(550,130903,'Zv9_NA28',6303,55),(551,30195,'Zv9_NA29',6303,56),(552,10762,'Zv9_NA30',6303,57),(553,11944,'Zv9_NA31',6303,58),(554,16487,'Zv9_NA32',6303,59),(555,54475,'Zv9_NA33',6303,60),(556,23395,'Zv9_NA34',6303,61),(557,9230,'Zv9_NA35',6303,62),(558,97240,'Zv9_NA36',6303,63),(559,30922,'Zv9_NA37',6303,64),(560,49389,'Zv9_NA38',6303,65),(561,29772,'Zv9_NA39',6303,66),(562,43073,'Zv9_NA40',6303,67),(563,71685,'Zv9_NA41',6303,68),(564,60856,'Zv9_NA42',6303,69),(565,13714,'Zv9_NA43',6303,70),(566,9871,'Zv9_NA44',6303,71),(567,16781,'Zv9_NA45',6303,72),(568,17430,'Zv9_NA46',6303,73),(569,47767,'Zv9_NA47',6303,74),(570,148772,'Zv9_NA48',6303,75),(571,21348,'Zv9_NA49',6303,76),(572,6674,'Zv9_NA50',6303,77),(573,37662,'Zv9_NA51',6303,78),(574,44649,'Zv9_NA52',6303,79),(575,34431,'Zv9_NA53',6303,80),(576,19310,'Zv9_NA54',6303,81),(577,20389,'Zv9_NA55',6303,82),(578,8697,'Zv9_NA56',6303,83),(579,9714,'Zv9_NA57',6303,84),(580,27350,'Zv9_NA58',6303,85),(581,18756,'Zv9_NA59',6303,86),(582,42458,'Zv9_NA60',6303,87),(583,35842,'Zv9_NA61',6303,88),(584,15451,'Zv9_NA62',6303,89),(585,10966,'Zv9_NA63',6303,90),(586,12831,'Zv9_NA64',6303,91),(587,12735,'Zv9_NA65',6303,92),(588,38884,'Zv9_NA66',6303,93),(589,25796,'Zv9_NA67',6303,94),(590,70538,'Zv9_NA68',6303,95),(591,27326,'Zv9_NA69',6303,96),(592,29907,'Zv9_NA70',6303,97),(593,15078,'Zv9_NA71',6303,98),(594,13297,'Zv9_NA72',6303,99),(595,25381,'Zv9_NA73',6303,100),(596,24343,'Zv9_NA74',6303,101),(597,21196,'Zv9_NA75',6303,102),(598,10220,'Zv9_NA76',6303,103),(599,24494,'Zv9_NA77',6303,104),(600,41459,'Zv9_NA78',6303,105),(601,108195,'Zv9_NA79',6303,106),(602,23586,'Zv9_NA80',6303,107),(603,89657,'Zv9_NA81',6303,108),(604,15371,'Zv9_NA82',6303,109),(605,14644,'Zv9_NA83',6303,110),(606,50229,'Zv9_NA84',6303,111),(607,15739,'Zv9_NA85',6303,112),(608,78076,'Zv9_NA86',6303,113),(609,11002,'Zv9_NA87',6303,114),(610,21856,'Zv9_NA88',6303,115),(611,90601,'Zv9_NA89',6303,116),(612,14065,'Zv9_NA90',6303,117),(613,26145,'Zv9_NA91',6303,118),(614,45130,'Zv9_NA92',6303,119),(615,20402,'Zv9_NA93',6303,120),(616,15103,'Zv9_NA94',6303,121),(617,49544,'Zv9_NA95',6303,122),(618,15815,'Zv9_NA96',6303,123),(619,47727,'Zv9_NA97',6303,124),(620,7902,'Zv9_NA98',6303,125),(621,84903,'Zv9_NA99',6303,126),(622,26026,'Zv9_NA100',6303,127),(623,47589,'Zv9_NA101',6303,128),(624,14832,'Zv9_NA102',6303,129),(625,65247,'Zv9_NA103',6303,130),(626,47531,'Zv9_NA104',6303,131),(627,25462,'Zv9_NA105',6303,132),(628,13397,'Zv9_NA106',6303,133),(629,55784,'Zv9_NA107',6303,134),(630,5702,'Zv9_NA108',6303,135),(631,11016,'Zv9_NA109',6303,136),(632,25708,'Zv9_NA110',6303,137),(633,16298,'Zv9_NA111',6303,138),(634,29992,'Zv9_NA112',6303,139),(635,11849,'Zv9_NA113',6303,140),(636,23645,'Zv9_NA114',6303,141),(637,9442,'Zv9_NA115',6303,142),(638,14804,'Zv9_NA116',6303,143),(639,16041,'Zv9_NA117',6303,144),(640,21637,'Zv9_NA118',6303,145),(641,6288,'Zv9_NA119',6303,146),(642,33766,'Zv9_NA120',6303,147),(643,27700,'Zv9_NA121',6303,148),(644,18352,'Zv9_NA122',6303,149),(645,51610,'Zv9_NA123',6303,150),(646,21941,'Zv9_NA124',6303,151),(647,21023,'Zv9_NA125',6303,152),(648,57500,'Zv9_NA126',6303,153),(649,8150,'Zv9_NA127',6303,154),(650,12948,'Zv9_NA128',6303,155),(651,9079,'Zv9_NA129',6303,156),(652,96570,'Zv9_NA130',6303,157),(653,11127,'Zv9_NA131',6303,158),(654,40089,'Zv9_NA132',6303,159),(655,13928,'Zv9_NA133',6303,160),(656,15848,'Zv9_NA134',6303,161),(657,30249,'Zv9_NA135',6303,162),(658,40214,'Zv9_NA136',6303,163),(659,11941,'Zv9_NA137',6303,164),(660,11785,'Zv9_NA138',6303,165),(661,46605,'Zv9_NA139',6303,166),(662,19644,'Zv9_NA140',6303,167),(663,15732,'Zv9_NA141',6303,168),(664,29197,'Zv9_NA142',6303,169),(665,14664,'Zv9_NA143',6303,170),(666,30874,'Zv9_NA144',6303,171),(667,17671,'Zv9_NA145',6303,172),(668,9278,'Zv9_NA146',6303,173),(669,12046,'Zv9_NA147',6303,174),(670,14151,'Zv9_NA148',6303,175),(671,13273,'Zv9_NA149',6303,176),(672,16306,'Zv9_NA150',6303,177),(673,39831,'Zv9_NA151',6303,178),(674,111598,'Zv9_NA152',6303,179),(675,32345,'Zv9_NA153',6303,180),(676,36097,'Zv9_NA154',6303,181),(677,11892,'Zv9_NA155',6303,182),(678,73829,'Zv9_NA156',6303,183),(679,3679,'Zv9_NA157',6303,184),(680,37156,'Zv9_NA158',6303,185),(681,29163,'Zv9_NA159',6303,186),(682,12990,'Zv9_NA160',6303,187),(683,11232,'Zv9_NA161',6303,188),(684,17517,'Zv9_NA162',6303,189),(685,34615,'Zv9_NA163',6303,190),(686,25635,'Zv9_NA164',6303,191),(687,1990,'Zv9_NA165',6303,192),(688,16018,'Zv9_NA166',6303,193),(689,10148,'Zv9_NA167',6303,194),(690,12432,'Zv9_NA168',6303,195),(691,15454,'Zv9_NA169',6303,196),(692,11900,'Zv9_NA170',6303,197),(693,40464,'Zv9_NA171',6303,198),(694,8720,'Zv9_NA172',6303,199),(695,51419,'Zv9_NA173',6303,200),(696,11116,'Zv9_NA174',6303,201),(697,51667,'Zv9_NA175',6303,202),(698,39965,'Zv9_NA176',6303,203),(699,31785,'Zv9_NA177',6303,204),(700,57586,'Zv9_NA178',6303,205),(701,90482,'Zv9_NA179',6303,206),(702,6675,'Zv9_NA180',6303,207),(703,11414,'Zv9_NA181',6303,208),(704,12483,'Zv9_NA182',6303,209),(705,17343,'Zv9_NA183',6303,210),(706,8007,'Zv9_NA184',6303,211),(707,29025,'Zv9_NA185',6303,212),(708,24180,'Zv9_NA186',6303,213),(709,19609,'Zv9_NA187',6303,214),(710,5676,'Zv9_NA188',6303,215),(711,35705,'Zv9_NA189',6303,216),(712,12998,'Zv9_NA190',6303,217),(713,46864,'Zv9_NA191',6303,218),(714,46519,'Zv9_NA192',6303,219),(715,20710,'Zv9_NA193',6303,220),(716,18016,'Zv9_NA194',6303,221),(717,10201,'Zv9_NA195',6303,222),(718,28664,'Zv9_NA196',6303,223),(719,8328,'Zv9_NA197',6303,224),(720,10341,'Zv9_NA198',6303,225),(721,33401,'Zv9_NA199',6303,226),(722,12094,'Zv9_NA200',6303,227),(723,12817,'Zv9_NA201',6303,228),(724,64403,'Zv9_NA202',6303,229),(725,53306,'Zv9_NA203',6303,230),(726,17408,'Zv9_NA204',6303,231),(727,14336,'Zv9_NA205',6303,232),(728,11987,'Zv9_NA206',6303,233),(729,14752,'Zv9_NA207',6303,234),(730,26893,'Zv9_NA208',6303,235),(731,17192,'Zv9_NA209',6303,236),(732,44680,'Zv9_NA210',6303,237),(733,11298,'Zv9_NA211',6303,238),(734,1347,'Zv9_NA212',6303,239),(735,31386,'Zv9_NA213',6303,240),(736,12938,'Zv9_NA214',6303,241),(737,7091,'Zv9_NA215',6303,242),(738,17117,'Zv9_NA216',6303,243),(739,10166,'Zv9_NA217',6303,244),(740,12029,'Zv9_NA218',6303,245),(741,107910,'Zv9_NA219',6303,246),(742,24638,'Zv9_NA220',6303,247),(743,27995,'Zv9_NA221',6303,248),(744,13102,'Zv9_NA222',6303,249),(745,14044,'Zv9_NA223',6303,250),(746,42463,'Zv9_NA224',6303,251),(747,13632,'Zv9_NA225',6303,252),(748,10512,'Zv9_NA226',6303,253),(749,101587,'Zv9_NA227',6303,254),(750,14056,'Zv9_NA228',6303,255),(751,38987,'Zv9_NA229',6303,256),(752,9870,'Zv9_NA230',6303,257),(753,12800,'Zv9_NA231',6303,258),(754,40056,'Zv9_NA232',6303,259),(755,55805,'Zv9_NA233',6303,260),(756,8389,'Zv9_NA234',6303,261),(757,18241,'Zv9_NA235',6303,262),(758,10395,'Zv9_NA236',6303,263),(759,32983,'Zv9_NA237',6303,264),(760,21073,'Zv9_NA238',6303,265),(761,13053,'Zv9_NA239',6303,266),(762,28254,'Zv9_NA240',6303,267),(763,37908,'Zv9_NA241',6303,268),(764,23619,'Zv9_NA242',6303,269),(765,63587,'Zv9_NA243',6303,270),(766,31367,'Zv9_NA244',6303,271),(767,14297,'Zv9_NA245',6303,272),(768,50906,'Zv9_NA246',6303,273),(769,13470,'Zv9_NA247',6303,274),(770,35264,'Zv9_NA248',6303,275),(771,13433,'Zv9_NA249',6303,276),(772,34033,'Zv9_NA250',6303,277),(773,76611,'Zv9_NA251',6303,278),(774,11887,'Zv9_NA252',6303,279),(775,13607,'Zv9_NA253',6303,280),(776,24909,'Zv9_NA254',6303,281),(777,33071,'Zv9_NA255',6303,282),(778,29092,'Zv9_NA256',6303,283),(779,117499,'Zv9_NA257',6303,284),(780,64126,'Zv9_NA258',6303,285),(781,12046,'Zv9_NA259',6303,286),(782,5952,'Zv9_NA260',6303,287),(783,12074,'Zv9_NA261',6303,288),(784,66365,'Zv9_NA262',6303,289),(785,46218,'Zv9_NA263',6303,290),(786,19033,'Zv9_NA264',6303,291),(787,19758,'Zv9_NA265',6303,292),(788,59274,'Zv9_NA266',6303,293),(789,9624,'Zv9_NA267',6303,294),(790,9235,'Zv9_NA268',6303,295),(791,20600,'Zv9_NA269',6303,296),(792,39283,'Zv9_NA270',6303,297),(793,18539,'Zv9_NA271',6303,298),(794,41819,'Zv9_NA272',6303,299),(795,83881,'Zv9_NA273',6303,300),(796,59701,'Zv9_NA274',6303,301),(797,26969,'Zv9_NA275',6303,302),(798,82990,'Zv9_NA276',6303,303),(799,102386,'Zv9_NA277',6303,304),(800,53295,'Zv9_NA278',6303,305),(801,27489,'Zv9_NA279',6303,306),(802,35555,'Zv9_NA280',6303,307),(803,19596,'Zv9_NA281',6303,308),(804,11858,'Zv9_NA282',6303,309),(805,24131,'Zv9_NA283',6303,310),(806,12811,'Zv9_NA284',6303,311),(807,19619,'Zv9_NA285',6303,312),(808,31814,'Zv9_NA286',6303,313),(809,81645,'Zv9_NA287',6303,314),(810,6556,'Zv9_NA288',6303,315),(811,20220,'Zv9_NA289',6303,316),(812,96477,'Zv9_NA290',6303,317),(813,17202,'Zv9_NA291',6303,318),(814,28756,'Zv9_NA292',6303,319),(815,9944,'Zv9_NA293',6303,320),(816,11702,'Zv9_NA294',6303,321),(817,40063,'Zv9_NA295',6303,322),(818,57651,'Zv9_NA296',6303,323),(819,57912,'Zv9_NA297',6303,324),(820,16921,'Zv9_NA298',6303,325),(821,56304,'Zv9_NA299',6303,326),(822,22954,'Zv9_NA300',6303,327),(823,29537,'Zv9_NA301',6303,328),(824,33528,'Zv9_NA302',6303,329),(825,74724,'Zv9_NA303',6303,330),(826,14071,'Zv9_NA304',6303,331),(827,10046,'Zv9_NA305',6303,332),(828,34409,'Zv9_NA306',6303,333),(829,13949,'Zv9_NA307',6303,334),(830,10518,'Zv9_NA308',6303,335),(831,13072,'Zv9_NA309',6303,336),(832,15572,'Zv9_NA310',6303,337),(833,17727,'Zv9_NA311',6303,338),(834,14703,'Zv9_NA312',6303,339),(835,12995,'Zv9_NA313',6303,340),(836,27068,'Zv9_NA314',6303,341),(837,27237,'Zv9_NA315',6303,342),(838,13588,'Zv9_NA316',6303,343),(839,25636,'Zv9_NA317',6303,344),(840,35750,'Zv9_NA318',6303,345),(841,38162,'Zv9_NA319',6303,346),(842,28998,'Zv9_NA320',6303,347),(843,11925,'Zv9_NA321',6303,348),(844,103967,'Zv9_NA322',6303,349),(845,76538,'Zv9_NA323',6303,350),(846,55820,'Zv9_NA324',6303,351),(847,22153,'Zv9_NA325',6303,352),(848,14527,'Zv9_NA326',6303,353),(849,8813,'Zv9_NA327',6303,354),(850,26343,'Zv9_NA328',6303,355),(851,53190,'Zv9_NA329',6303,356),(852,18401,'Zv9_NA330',6303,357),(853,110945,'Zv9_NA331',6303,358),(854,60899,'Zv9_NA332',6303,359),(855,29546,'Zv9_NA333',6303,360),(856,3679,'Zv9_NA334',6303,361),(857,56448,'Zv9_NA335',6303,362),(858,19927,'Zv9_NA336',6303,363),(859,71326,'Zv9_NA337',6303,364),(860,20631,'Zv9_NA338',6303,365),(861,18649,'Zv9_NA339',6303,366),(862,44913,'Zv9_NA340',6303,367),(863,9130,'Zv9_NA341',6303,368),(864,12023,'Zv9_NA342',6303,369),(865,39438,'Zv9_NA343',6303,370),(866,52186,'Zv9_NA344',6303,371),(867,128109,'Zv9_NA345',6303,372),(868,45008,'Zv9_NA346',6303,373),(869,5663,'Zv9_NA347',6303,374),(870,16143,'Zv9_NA348',6303,375),(871,62665,'Zv9_NA349',6303,376),(872,12545,'Zv9_NA350',6303,377),(873,18882,'Zv9_NA351',6303,378),(874,10736,'Zv9_NA352',6303,379),(875,49005,'Zv9_NA353',6303,380),(876,8909,'Zv9_NA354',6303,381),(877,82545,'Zv9_NA355',6303,382),(878,24048,'Zv9_NA356',6303,383),(879,35038,'Zv9_NA357',6303,384),(880,14598,'Zv9_NA358',6303,385),(881,29537,'Zv9_NA359',6303,386),(882,27325,'Zv9_NA360',6303,387),(883,62725,'Zv9_NA361',6303,388),(884,14960,'Zv9_NA362',6303,389),(885,25595,'Zv9_NA363',6303,390),(886,14425,'Zv9_NA364',6303,391),(887,15053,'Zv9_NA365',6303,392),(888,16690,'Zv9_NA366',6303,393),(889,51360,'Zv9_NA367',6303,394),(890,10626,'Zv9_NA368',6303,395),(891,12019,'Zv9_NA369',6303,396),(892,19437,'Zv9_NA370',6303,397),(893,6328,'Zv9_NA371',6303,398),(894,206081,'Zv9_NA372',6303,399),(895,12836,'Zv9_NA373',6303,400),(896,8788,'Zv9_NA374',6303,401),(897,49514,'Zv9_NA375',6303,402),(898,24192,'Zv9_NA376',6303,403),(899,24652,'Zv9_NA377',6303,404),(900,32319,'Zv9_NA378',6303,405),(901,11020,'Zv9_NA379',6303,406),(902,12028,'Zv9_NA380',6303,407),(903,14544,'Zv9_NA381',6303,408),(904,61361,'Zv9_NA382',6303,409),(905,11690,'Zv9_NA383',6303,410),(906,79937,'Zv9_NA384',6303,411),(907,58062,'Zv9_NA385',6303,412),(908,20039,'Zv9_NA386',6303,413),(909,88343,'Zv9_NA387',6303,414),(910,78223,'Zv9_NA388',6303,415),(911,9804,'Zv9_NA389',6303,416),(912,24178,'Zv9_NA390',6303,417),(913,15497,'Zv9_NA391',6303,418),(914,55498,'Zv9_NA392',6303,419),(915,14260,'Zv9_NA393',6303,420),(916,48543,'Zv9_NA394',6303,421),(917,54494,'Zv9_NA395',6303,422),(918,32330,'Zv9_NA396',6303,423),(919,8447,'Zv9_NA397',6303,424),(920,67534,'Zv9_NA398',6303,425),(921,83315,'Zv9_NA399',6303,426),(922,24847,'Zv9_NA400',6303,427),(923,33302,'Zv9_NA401',6303,428),(924,12262,'Zv9_NA402',6303,429),(925,107891,'Zv9_NA403',6303,430),(926,22375,'Zv9_NA404',6303,431),(927,13999,'Zv9_NA405',6303,432),(928,17476,'Zv9_NA406',6303,433),(929,49373,'Zv9_NA407',6303,434),(930,28725,'Zv9_NA408',6303,435),(931,36779,'Zv9_NA409',6303,436),(932,19733,'Zv9_NA410',6303,437),(933,6273,'Zv9_NA411',6303,438),(934,16251,'Zv9_NA412',6303,439),(935,14974,'Zv9_NA413',6303,440),(936,32451,'Zv9_NA414',6303,441),(937,27317,'Zv9_NA415',6303,442),(938,23078,'Zv9_NA416',6303,443),(939,33247,'Zv9_NA417',6303,444),(940,35229,'Zv9_NA418',6303,445),(941,17824,'Zv9_NA419',6303,446),(942,10930,'Zv9_NA420',6303,447),(943,26178,'Zv9_NA421',6303,448),(944,32739,'Zv9_NA422',6303,449),(945,49457,'Zv9_NA423',6303,450),(946,13466,'Zv9_NA424',6303,451),(947,17027,'Zv9_NA425',6303,452),(948,19583,'Zv9_NA426',6303,453),(949,35047,'Zv9_NA427',6303,454),(950,140649,'Zv9_NA428',6303,455),(951,67766,'Zv9_NA429',6303,456),(952,12500,'Zv9_NA430',6303,457),(953,13387,'Zv9_NA431',6303,458),(954,11996,'Zv9_NA432',6303,459),(955,13259,'Zv9_NA433',6303,460),(956,24612,'Zv9_NA434',6303,461),(957,12108,'Zv9_NA435',6303,462),(958,21670,'Zv9_NA436',6303,463),(959,35759,'Zv9_NA437',6303,464),(960,24002,'Zv9_NA438',6303,465),(961,47473,'Zv9_NA439',6303,466),(962,11735,'Zv9_NA440',6303,467),(963,16312,'Zv9_NA441',6303,468),(964,8651,'Zv9_NA442',6303,469),(965,24376,'Zv9_NA443',6303,470),(966,24654,'Zv9_NA444',6303,471),(967,16544,'Zv9_NA445',6303,472),(968,19179,'Zv9_NA446',6303,473),(969,38868,'Zv9_NA447',6303,474),(970,11399,'Zv9_NA448',6303,475),(971,30599,'Zv9_NA449',6303,476),(972,12580,'Zv9_NA450',6303,477),(973,13629,'Zv9_NA451',6303,478),(974,18921,'Zv9_NA452',6303,479),(975,14532,'Zv9_NA453',6303,480),(976,18196,'Zv9_NA454',6303,481),(977,71322,'Zv9_NA455',6303,482),(978,16564,'Zv9_NA456',6303,483),(979,11494,'Zv9_NA457',6303,484),(980,25470,'Zv9_NA458',6303,485),(981,9480,'Zv9_NA459',6303,486),(982,24004,'Zv9_NA460',6303,487),(983,95149,'Zv9_NA461',6303,488),(984,2052,'Zv9_NA462',6303,489),(985,33163,'Zv9_NA463',6303,490),(986,12550,'Zv9_NA464',6303,491),(987,16386,'Zv9_NA465',6303,492),(988,162877,'Zv9_NA466',6303,493),(989,22488,'Zv9_NA468',6303,494),(990,15354,'Zv9_NA469',6303,495),(991,108843,'Zv9_NA470',6303,496),(992,16672,'Zv9_NA471',6303,497),(993,104196,'Zv9_NA472',6303,498),(994,28122,'Zv9_NA473',6303,499),(995,110669,'Zv9_NA474',6303,500),(996,20248,'Zv9_NA475',6303,501),(997,11398,'Zv9_NA476',6303,502),(998,30201,'Zv9_NA477',6303,503),(999,6919,'Zv9_NA478',6303,504),(1000,87008,'Zv9_NA479',6303,505),(1001,12701,'Zv9_NA480',6303,506),(1002,31979,'Zv9_NA481',6303,507),(1003,17061,'Zv9_NA482',6303,508),(1004,16238,'Zv9_NA483',6303,509),(1005,28415,'Zv9_NA484',6303,510),(1006,24616,'Zv9_NA485',6303,511),(1007,53540,'Zv9_NA486',6303,512),(1008,9340,'Zv9_NA487',6303,513),(1009,18511,'Zv9_NA488',6303,514),(1010,9450,'Zv9_NA489',6303,515),(1011,35271,'Zv9_NA490',6303,516),(1012,60821,'Zv9_NA491',6303,517),(1013,45964,'Zv9_NA492',6303,518),(1014,42860,'Zv9_NA493',6303,519),(1015,18273,'Zv9_NA494',6303,520),(1016,12261,'Zv9_NA495',6303,521),(1017,39310,'Zv9_NA496',6303,522),(1018,23192,'Zv9_NA497',6303,523),(1019,12144,'Zv9_NA498',6303,524),(1020,36661,'Zv9_NA499',6303,525),(1021,17390,'Zv9_NA500',6303,526),(1022,95885,'Zv9_NA501',6303,527),(1023,50098,'Zv9_NA502',6303,528),(1024,202348,'Zv9_NA503',6303,529),(1025,13934,'Zv9_NA504',6303,530),(1026,12318,'Zv9_NA505',6303,531),(1027,31336,'Zv9_NA506',6303,532),(1028,30832,'Zv9_NA507',6303,533),(1029,8540,'Zv9_NA508',6303,534),(1030,77778,'Zv9_NA509',6303,535),(1031,21397,'Zv9_NA510',6303,536),(1032,22285,'Zv9_NA511',6303,537),(1033,12637,'Zv9_NA512',6303,538),(1034,34295,'Zv9_NA513',6303,539),(1035,23414,'Zv9_NA514',6303,540),(1036,12247,'Zv9_NA515',6303,541),(1037,13800,'Zv9_NA516',6303,542),(1038,12942,'Zv9_NA517',6303,543),(1039,30978,'Zv9_NA518',6303,544),(1040,23764,'Zv9_NA519',6303,545),(1041,27067,'Zv9_NA520',6303,546),(1042,45935,'Zv9_NA521',6303,547),(1043,18131,'Zv9_NA522',6303,548),(1044,11856,'Zv9_NA523',6303,549),(1045,22206,'Zv9_NA524',6303,550),(1046,96074,'Zv9_NA525',6303,551),(1047,14376,'Zv9_NA526',6303,552),(1048,18248,'Zv9_NA527',6303,553),(1049,9984,'Zv9_NA528',6303,554),(1050,41505,'Zv9_NA529',6303,555),(1051,37815,'Zv9_NA530',6303,556),(1052,10238,'Zv9_NA531',6303,557),(1053,23472,'Zv9_NA532',6303,558),(1054,23511,'Zv9_NA533',6303,559),(1055,28537,'Zv9_NA534',6303,560),(1056,13512,'Zv9_NA535',6303,561),(1057,9489,'Zv9_NA536',6303,562),(1058,51630,'Zv9_NA537',6303,563),(1059,19784,'Zv9_NA538',6303,564),(1060,5114,'Zv9_NA539',6303,565),(1061,23261,'Zv9_NA540',6303,566),(1062,12472,'Zv9_NA541',6303,567),(1063,36728,'Zv9_NA542',6303,568),(1064,16101,'Zv9_NA543',6303,569),(1065,9167,'Zv9_NA544',6303,570),(1066,196107,'Zv9_NA545',6303,571),(1067,23518,'Zv9_NA546',6303,572),(1068,22983,'Zv9_NA547',6303,573),(1069,31405,'Zv9_NA548',6303,574),(1070,20999,'Zv9_NA549',6303,575),(1071,15747,'Zv9_NA550',6303,576),(1072,47808,'Zv9_NA551',6303,577),(1073,39982,'Zv9_NA552',6303,578),(1074,12455,'Zv9_NA553',6303,579),(1075,20424,'Zv9_NA554',6303,580),(1076,152271,'Zv9_NA555',6303,581),(1077,15807,'Zv9_NA556',6303,582),(1078,11730,'Zv9_NA557',6303,583),(1079,18137,'Zv9_NA558',6303,584),(1080,19525,'Zv9_NA559',6303,585),(1081,13345,'Zv9_NA560',6303,586),(1082,109565,'Zv9_NA561',6303,587),(1083,17410,'Zv9_NA562',6303,588),(1084,13297,'Zv9_NA563',6303,589),(1085,44968,'Zv9_NA564',6303,590),(1086,21958,'Zv9_NA565',6303,591),(1087,12307,'Zv9_NA566',6303,592),(1088,50772,'Zv9_NA567',6303,593),(1089,20260,'Zv9_NA568',6303,594),(1090,31147,'Zv9_NA569',6303,595),(1091,20603,'Zv9_NA570',6303,596),(1092,7952,'Zv9_NA571',6303,597),(1093,13385,'Zv9_NA572',6303,598),(1094,51528,'Zv9_NA573',6303,599),(1095,42704,'Zv9_NA574',6303,600),(1096,16560,'Zv9_NA575',6303,601),(1097,29114,'Zv9_NA576',6303,602),(1098,11234,'Zv9_NA577',6303,603),(1099,28855,'Zv9_NA578',6303,604),(1100,9245,'Zv9_NA579',6303,605),(1101,49608,'Zv9_NA580',6303,606),(1102,7798,'Zv9_NA581',6303,607),(1103,15907,'Zv9_NA582',6303,608),(1104,19631,'Zv9_NA583',6303,609),(1105,71383,'Zv9_NA584',6303,610),(1106,15639,'Zv9_NA585',6303,611),(1107,17145,'Zv9_NA586',6303,612),(1108,9880,'Zv9_NA587',6303,613),(1109,15084,'Zv9_NA588',6303,614),(1110,20976,'Zv9_NA589',6303,615),(1111,11460,'Zv9_NA590',6303,616),(1112,13836,'Zv9_NA591',6303,617),(1113,29560,'Zv9_NA592',6303,618),(1114,13130,'Zv9_NA593',6303,619),(1115,14640,'Zv9_NA594',6303,620),(1116,17431,'Zv9_NA595',6303,621),(1117,19249,'Zv9_NA596',6303,622),(1118,33981,'Zv9_NA597',6303,623),(1119,14074,'Zv9_NA598',6303,624),(1120,13722,'Zv9_NA599',6303,625),(1121,22969,'Zv9_NA600',6303,626),(1122,27290,'Zv9_NA601',6303,627),(1123,35900,'Zv9_NA602',6303,628),(1124,15215,'Zv9_NA603',6303,629),(1125,19118,'Zv9_NA604',6303,630),(1126,31268,'Zv9_NA605',6303,631),(1127,17139,'Zv9_NA606',6303,632),(1128,9792,'Zv9_NA607',6303,633),(1129,40497,'Zv9_NA608',6303,634),(1130,53888,'Zv9_NA609',6303,635),(1131,8176,'Zv9_NA610',6303,636),(1132,17295,'Zv9_NA611',6303,637),(1133,39557,'Zv9_NA612',6303,638),(1134,25191,'Zv9_NA613',6303,639),(1135,19362,'Zv9_NA614',6303,640),(1136,17237,'Zv9_NA615',6303,641),(1137,15977,'Zv9_NA616',6303,642),(1138,10910,'Zv9_NA617',6303,643),(1139,34930,'Zv9_NA618',6303,644),(1140,10081,'Zv9_NA619',6303,645),(1141,34697,'Zv9_NA620',6303,646),(1142,79091,'Zv9_NA621',6303,647),(1143,27587,'Zv9_NA622',6303,648),(1144,31528,'Zv9_NA623',6303,649),(1145,31814,'Zv9_NA624',6303,650),(1146,37191,'Zv9_NA625',6303,651),(1147,15975,'Zv9_NA626',6303,652),(1148,35871,'Zv9_NA627',6303,653),(1149,70574,'Zv9_NA628',6303,654),(1150,33835,'Zv9_NA629',6303,655),(1151,28321,'Zv9_NA630',6303,656),(1152,24320,'Zv9_NA631',6303,657),(1153,14114,'Zv9_NA632',6303,658),(1154,14951,'Zv9_NA633',6303,659),(1155,15053,'Zv9_NA634',6303,660),(1156,25956,'Zv9_NA635',6303,661),(1157,29216,'Zv9_NA636',6303,662),(1158,17683,'Zv9_NA637',6303,663),(1159,16452,'Zv9_NA638',6303,664),(1160,89659,'Zv9_NA639',6303,665),(1161,63628,'Zv9_NA640',6303,666),(1162,28789,'Zv9_NA641',6303,667),(1163,12053,'Zv9_NA642',6303,668),(1164,12561,'Zv9_NA643',6303,669),(1165,15380,'Zv9_NA644',6303,670),(1166,10343,'Zv9_NA645',6303,671),(1167,12439,'Zv9_NA646',6303,672);
INSERT INTO dbo.gpSegment 
(idSegment,
length,
name,
idGenomeVersion,
sortOrder
)
VALUES (1168,13311,'Zv9_NA647',6303,673),(1169,16134,'Zv9_NA648',6303,674),(1170,51503,'Zv9_NA649',6303,675),(1171,62738,'Zv9_NA650',6303,676),(1172,11834,'Zv9_NA651',6303,677),(1173,11830,'Zv9_NA652',6303,678),(1174,47980,'Zv9_NA653',6303,679),(1175,124886,'Zv9_NA654',6303,680),(1176,8345,'Zv9_NA655',6303,681),(1177,18479,'Zv9_NA656',6303,682),(1178,12499,'Zv9_NA657',6303,683),(1179,11539,'Zv9_NA658',6303,684),(1180,73536,'Zv9_NA659',6303,685),(1181,76246,'Zv9_NA660',6303,686),(1182,14997,'Zv9_NA661',6303,687),(1183,17481,'Zv9_NA662',6303,688),(1184,45603,'Zv9_NA663',6303,689),(1185,5580,'Zv9_NA666',6303,690),(1186,15093,'Zv9_NA667',6303,691),(1187,31510,'Zv9_NA668',6303,692),(1188,24865,'Zv9_NA669',6303,693),(1189,9549,'Zv9_NA670',6303,694),(1190,41319,'Zv9_NA671',6303,695),(1191,248473,'Zv9_NA672',6303,696),(1192,14614,'Zv9_NA673',6303,697),(1193,14586,'Zv9_NA674',6303,698),(1194,312116,'Zv9_NA675',6303,699),(1195,21894,'Zv9_NA676',6303,700),(1196,59211,'Zv9_NA677',6303,701),(1197,61693,'Zv9_NA678',6303,702),(1198,25813,'Zv9_NA679',6303,703),(1199,17332,'Zv9_NA680',6303,704),(1200,103999,'Zv9_NA681',6303,705),(1201,7348,'Zv9_NA682',6303,706),(1202,123701,'Zv9_NA683',6303,707),(1203,6489,'Zv9_NA684',6303,708),(1204,17582,'Zv9_NA685',6303,709),(1205,26739,'Zv9_NA686',6303,710),(1206,11552,'Zv9_NA687',6303,711),(1207,31617,'Zv9_NA688',6303,712),(1208,35575,'Zv9_NA689',6303,713),(1209,10411,'Zv9_NA690',6303,714),(1210,10108,'Zv9_NA691',6303,715),(1211,28160,'Zv9_NA692',6303,716),(1212,21062,'Zv9_NA693',6303,717),(1213,46188,'Zv9_NA694',6303,718),(1214,55642,'Zv9_NA695',6303,719),(1215,11840,'Zv9_NA696',6303,720),(1216,19866,'Zv9_NA697',6303,721),(1217,90382,'Zv9_NA698',6303,722),(1218,12094,'Zv9_NA699',6303,723),(1219,14191,'Zv9_NA700',6303,724),(1220,14213,'Zv9_NA701',6303,725),(1221,26814,'Zv9_NA702',6303,726),(1222,13237,'Zv9_NA703',6303,727),(1223,38183,'Zv9_NA704',6303,728),(1224,5946,'Zv9_NA705',6303,729),(1225,56151,'Zv9_NA706',6303,730),(1226,14003,'Zv9_NA707',6303,731),(1227,19391,'Zv9_NA708',6303,732),(1228,21577,'Zv9_NA709',6303,733),(1229,49935,'Zv9_NA710',6303,734),(1230,16171,'Zv9_NA711',6303,735),(1231,2434,'Zv9_NA712',6303,736),(1232,11974,'Zv9_NA713',6303,737),(1233,35226,'Zv9_NA714',6303,738),(1234,20896,'Zv9_NA715',6303,739),(1235,12245,'Zv9_NA716',6303,740),(1236,19004,'Zv9_NA717',6303,741),(1237,15513,'Zv9_NA718',6303,742),(1238,20460,'Zv9_NA719',6303,743),(1239,10750,'Zv9_NA720',6303,744),(1240,47384,'Zv9_NA721',6303,745),(1241,221151,'Zv9_NA722',6303,746),(1242,20880,'Zv9_NA723',6303,747),(1243,13138,'Zv9_NA724',6303,748),(1244,7299,'Zv9_NA725',6303,749),(1245,44130,'Zv9_NA726',6303,750),(1246,50218,'Zv9_NA727',6303,751),(1247,13249,'Zv9_NA728',6303,752),(1248,17661,'Zv9_NA729',6303,753),(1249,17908,'Zv9_NA730',6303,754),(1250,27568,'Zv9_NA731',6303,755),(1251,190002,'Zv9_NA732',6303,756),(1252,12119,'Zv9_NA733',6303,757),(1253,10141,'Zv9_NA734',6303,758),(1254,175146,'Zv9_NA735',6303,759),(1255,31752,'Zv9_NA736',6303,760),(1256,23246,'Zv9_NA737',6303,761),(1257,17517,'Zv9_NA738',6303,762),(1258,12323,'Zv9_NA739',6303,763),(1259,12167,'Zv9_NA740',6303,764),(1260,45335,'Zv9_NA741',6303,765),(1261,42001,'Zv9_NA742',6303,766),(1262,21448,'Zv9_NA743',6303,767),(1263,12932,'Zv9_NA744',6303,768),(1264,1198,'Zv9_NA745',6303,769),(1265,10479,'Zv9_NA746',6303,770),(1266,17546,'Zv9_NA747',6303,771),(1267,37420,'Zv9_NA748',6303,772),(1268,21692,'Zv9_NA749',6303,773),(1269,6274,'Zv9_NA750',6303,774),(1270,9435,'Zv9_NA751',6303,775),(1271,9251,'Zv9_NA752',6303,776),(1272,21302,'Zv9_NA753',6303,777),(1273,9286,'Zv9_NA754',6303,778),(1274,12115,'Zv9_NA755',6303,779),(1275,26580,'Zv9_NA756',6303,780),(1276,6941,'Zv9_NA757',6303,781),(1277,22875,'Zv9_NA758',6303,782),(1278,17709,'Zv9_NA759',6303,783),(1279,20464,'Zv9_NA760',6303,784),(1280,49566,'Zv9_NA761',6303,785),(1281,24359,'Zv9_NA762',6303,786),(1282,16505,'Zv9_NA763',6303,787),(1283,10130,'Zv9_NA764',6303,788),(1284,28871,'Zv9_NA765',6303,789),(1285,26604,'Zv9_NA766',6303,790),(1286,18499,'Zv9_NA767',6303,791),(1287,11084,'Zv9_NA768',6303,792),(1288,36129,'Zv9_NA769',6303,793),(1289,13719,'Zv9_NA770',6303,794),(1290,103698,'Zv9_NA771',6303,795),(1291,32349,'Zv9_NA772',6303,796),(1292,12355,'Zv9_NA773',6303,797),(1293,55488,'Zv9_NA774',6303,798),(1294,34402,'Zv9_NA775',6303,799),(1295,12396,'Zv9_NA776',6303,800),(1296,12430,'Zv9_NA777',6303,801),(1297,14570,'Zv9_NA778',6303,802),(1298,28463,'Zv9_NA779',6303,803),(1299,14103,'Zv9_NA780',6303,804),(1300,8817,'Zv9_NA781',6303,805),(1301,19382,'Zv9_NA782',6303,806),(1302,60847,'Zv9_NA783',6303,807),(1303,12499,'Zv9_NA784',6303,808),(1304,19019,'Zv9_NA785',6303,809),(1305,98280,'Zv9_NA786',6303,810),(1306,102574,'Zv9_NA787',6303,811),(1307,51696,'Zv9_NA788',6303,812),(1308,9531,'Zv9_NA789',6303,813),(1309,9429,'Zv9_NA790',6303,814),(1310,37413,'Zv9_NA791',6303,815),(1311,10565,'Zv9_NA792',6303,816),(1312,11974,'Zv9_NA793',6303,817),(1313,16060,'Zv9_NA794',6303,818),(1314,14844,'Zv9_NA795',6303,819),(1315,3298,'Zv9_NA796',6303,820),(1316,114591,'Zv9_NA797',6303,821),(1317,11546,'Zv9_NA798',6303,822),(1318,20004,'Zv9_NA799',6303,823),(1319,30729,'Zv9_NA800',6303,824),(1320,15811,'Zv9_NA801',6303,825),(1321,1126,'Zv9_NA802',6303,826),(1322,22486,'Zv9_NA803',6303,827),(1323,32850,'Zv9_NA804',6303,828),(1324,25159,'Zv9_NA805',6303,829),(1325,63519,'Zv9_NA806',6303,830),(1326,67541,'Zv9_NA807',6303,831),(1327,86421,'Zv9_NA808',6303,832),(1328,11627,'Zv9_NA809',6303,833),(1329,39727,'Zv9_NA810',6303,834),(1330,32979,'Zv9_NA811',6303,835),(1331,27620,'Zv9_NA812',6303,836),(1332,20890,'Zv9_NA813',6303,837),(1333,19636,'Zv9_NA814',6303,838),(1334,28088,'Zv9_NA815',6303,839),(1335,22839,'Zv9_NA816',6303,840),(1336,81249,'Zv9_NA817',6303,841),(1337,25381,'Zv9_NA818',6303,842),(1338,28018,'Zv9_NA819',6303,843),(1339,18075,'Zv9_NA820',6303,844),(1340,29322,'Zv9_NA821',6303,845),(1341,90387,'Zv9_NA822',6303,846),(1342,17820,'Zv9_NA823',6303,847),(1343,88608,'Zv9_NA824',6303,848),(1344,9590,'Zv9_NA825',6303,849),(1345,11944,'Zv9_NA826',6303,850),(1346,63484,'Zv9_NA827',6303,851),(1347,54201,'Zv9_NA828',6303,852),(1348,15325,'Zv9_NA829',6303,853),(1349,13526,'Zv9_NA830',6303,854),(1350,9332,'Zv9_NA831',6303,855),(1351,13959,'Zv9_NA832',6303,856),(1352,41858,'Zv9_NA833',6303,857),(1353,51674,'Zv9_NA834',6303,858),(1354,7552,'Zv9_NA836',6303,859),(1355,15190,'Zv9_NA837',6303,860),(1356,24053,'Zv9_NA838',6303,861),(1357,13309,'Zv9_NA839',6303,862),(1358,21256,'Zv9_NA840',6303,863),(1359,11947,'Zv9_NA841',6303,864),(1360,19550,'Zv9_NA842',6303,865),(1361,5690,'Zv9_NA843',6303,866),(1362,184625,'Zv9_NA844',6303,867),(1363,60897,'Zv9_NA845',6303,868),(1364,13510,'Zv9_NA846',6303,869),(1365,12709,'Zv9_NA847',6303,870),(1366,25751,'Zv9_NA848',6303,871),(1367,242593,'Zv9_NA849',6303,872),(1368,25985,'Zv9_NA850',6303,873),(1369,48009,'Zv9_NA851',6303,874),(1370,16373,'Zv9_NA852',6303,875),(1371,16938,'Zv9_NA853',6303,876),(1372,19629,'Zv9_NA854',6303,877),(1373,11940,'Zv9_NA855',6303,878),(1374,44612,'Zv9_NA856',6303,879),(1375,23450,'Zv9_NA857',6303,880),(1376,33146,'Zv9_NA858',6303,881),(1377,40050,'Zv9_NA859',6303,882),(1378,13902,'Zv9_NA860',6303,883),(1379,6161,'Zv9_NA861',6303,884),(1380,6736,'Zv9_NA862',6303,885),(1381,27926,'Zv9_NA863',6303,886),(1382,33279,'Zv9_NA864',6303,887),(1383,20567,'Zv9_NA865',6303,888),(1384,29245,'Zv9_NA866',6303,889),(1385,27814,'Zv9_NA867',6303,890),(1386,11473,'Zv9_NA868',6303,891),(1387,16026,'Zv9_NA869',6303,892),(1388,74622,'Zv9_NA870',6303,893),(1389,22550,'Zv9_NA871',6303,894),(1390,17592,'Zv9_NA872',6303,895),(1391,8506,'Zv9_NA873',6303,896),(1392,7316,'Zv9_NA874',6303,897),(1393,22862,'Zv9_NA875',6303,898),(1394,49336,'Zv9_NA876',6303,899),(1395,3381,'Zv9_NA877',6303,900),(1396,20579,'Zv9_NA878',6303,901),(1397,26928,'Zv9_NA879',6303,902),(1398,25201,'Zv9_NA880',6303,903),(1399,9090,'Zv9_NA881',6303,904),(1400,8946,'Zv9_NA882',6303,905),(1401,67738,'Zv9_NA883',6303,906),(1402,4454,'Zv9_NA884',6303,907),(1403,65370,'Zv9_NA885',6303,908),(1404,6767,'Zv9_NA886',6303,909),(1405,62336,'Zv9_NA887',6303,910),(1406,7339,'Zv9_NA888',6303,911),(1407,20886,'Zv9_NA889',6303,912),(1408,6597,'Zv9_NA890',6303,913),(1409,3607,'Zv9_NA891',6303,914),(1410,110593,'Zv9_NA892',6303,915),(1411,15043,'Zv9_NA893',6303,916),(1412,11382,'Zv9_NA894',6303,917),(1413,29524,'Zv9_NA895',6303,918),(1414,33258,'Zv9_NA896',6303,919),(1415,19172,'Zv9_NA897',6303,920),(1416,10149,'Zv9_NA898',6303,921),(1417,92401,'Zv9_NA899',6303,922),(1418,23581,'Zv9_NA900',6303,923),(1419,1786,'Zv9_NA901',6303,924),(1420,3484,'Zv9_NA902',6303,925),(1421,51036,'Zv9_NA903',6303,926),(1422,10439,'Zv9_NA904',6303,927),(1423,58278,'Zv9_NA905',6303,928),(1424,88699,'Zv9_NA906',6303,929),(1425,30410,'Zv9_NA907',6303,930),(1426,130527,'Zv9_NA908',6303,931),(1427,1916,'Zv9_NA909',6303,932),(1428,9279,'Zv9_NA910',6303,933),(1429,16801,'Zv9_NA911',6303,934),(1430,2460,'Zv9_NA912',6303,935),(1431,26843,'Zv9_NA913',6303,936),(1432,9328,'Zv9_NA914',6303,937),(1433,115850,'Zv9_NA915',6303,938),(1434,18284,'Zv9_NA916',6303,939),(1435,25214,'Zv9_NA917',6303,940),(1436,47971,'Zv9_NA918',6303,941),(1437,2001,'Zv9_NA919',6303,942),(1438,3455,'Zv9_NA920',6303,943),(1439,85407,'Zv9_NA921',6303,944),(1440,29672,'Zv9_NA922',6303,945),(1441,33441,'Zv9_NA923',6303,946),(1442,11402,'Zv9_NA924',6303,947),(1443,6296,'Zv9_NA925',6303,948),(1444,44504,'Zv9_NA926',6303,949),(1445,9426,'Zv9_NA927',6303,950),(1446,50629,'Zv9_NA928',6303,951),(1447,1577,'Zv9_NA929',6303,952),(1448,24203,'Zv9_NA930',6303,953),(1449,6731,'Zv9_NA931',6303,954),(1450,2185,'Zv9_NA932',6303,955),(1451,3012,'Zv9_NA933',6303,956),(1452,1266,'Zv9_NA934',6303,957),(1453,1714,'Zv9_NA935',6303,958),(1454,1530,'Zv9_NA936',6303,959),(1455,1137,'Zv9_NA937',6303,960),(1456,14383,'Zv9_NA938',6303,961),(1457,6107,'Zv9_NA939',6303,962),(1458,22542,'Zv9_NA940',6303,963),(1459,1591,'Zv9_NA941',6303,964),(1460,16044,'Zv9_NA942',6303,965),(1461,2065,'Zv9_NA943',6303,966),(1462,1251,'Zv9_NA944',6303,967),(1463,3228,'Zv9_NA945',6303,968),(1464,3086,'Zv9_NA946',6303,969),(1465,197207,'Zv9_NA947',6303,970),(1466,10954,'Zv9_NA948',6303,971),(1467,3033,'Zv9_NA949',6303,972),(1468,1755,'Zv9_NA950',6303,973),(1469,2975,'Zv9_NA951',6303,974),(1470,1326,'Zv9_NA952',6303,975),(1471,1633,'Zv9_NA953',6303,976),(1472,2620,'Zv9_NA954',6303,977),(1473,1989,'Zv9_NA955',6303,978),(1474,1989,'Zv9_NA956',6303,979),(1475,2866,'Zv9_NA957',6303,980),(1476,1428,'Zv9_NA958',6303,981),(1477,1142,'Zv9_NA959',6303,982),(1478,2671,'Zv9_NA960',6303,983),(1479,2357,'Zv9_NA961',6303,984),(1480,1732,'Zv9_NA962',6303,985),(1481,33607,'Zv9_NA963',6303,986),(1482,12227,'Zv9_NA964',6303,987),(1483,10904,'Zv9_NA965',6303,988),(1484,12799,'Zv9_NA966',6303,989),(1485,6697,'Zv9_NA967',6303,990),(1486,1173,'Zv9_NA968',6303,991),(1487,1853,'Zv9_NA969',6303,992),(1488,17707,'Zv9_NA970',6303,993),(1489,2865,'Zv9_NA971',6303,994),(1490,26415,'Zv9_NA972',6303,995),(1491,10057,'Zv9_NA973',6303,996),(1492,7997,'Zv9_NA974',6303,997),(1493,1452,'Zv9_NA975',6303,998),(1494,1692,'Zv9_NA976',6303,999),(1495,2246,'Zv9_NA977',6303,1000),(1496,3784,'Zv9_NA978',6303,1001),(1497,1346,'Zv9_NA979',6303,1002),(1498,1872,'Zv9_NA980',6303,1003),(1499,1813,'Zv9_NA981',6303,1004),(1500,1919,'Zv9_NA982',6303,1005),(1501,1919,'Zv9_NA983',6303,1006),(1502,1745,'Zv9_NA984',6303,1007),(1503,6726,'Zv9_NA985',6303,1008),(1504,1639,'Zv9_NA986',6303,1009),(1505,4206,'Zv9_NA987',6303,1010),(1506,2311,'Zv9_NA988',6303,1011),(1507,1920,'Zv9_NA989',6303,1012),(1508,29963,'Zv9_NA990',6303,1013),(1509,12451,'Zv9_NA991',6303,1014),(1510,10131,'Zv9_NA992',6303,1015),(1511,2100,'Zv9_NA993',6303,1016),(1512,46999,'Zv9_NA994',6303,1017),(1513,152294,'Zv9_NA995',6303,1018),(1514,21489,'Zv9_NA996',6303,1019),(1515,650,'Zv9_NA997',6303,1020),(1516,43558,'Zv9_NA998',6303,1021),(1517,91010,'Zv9_NA999',6303,1022),(1518,942,'Zv9_NA1000',6303,1023),(1519,203829,'Zv9_scaffold3453',6303,1024),(1520,91011,'Zv9_scaffold3454',6303,1025),(1521,543129,'Zv9_scaffold3455',6303,1026),(1522,253302,'Zv9_scaffold3456',6303,1027),(1523,101278,'Zv9_scaffold3457',6303,1028),(1524,223119,'Zv9_scaffold3458',6303,1029),(1525,417943,'Zv9_scaffold3459',6303,1030),(1526,146079,'Zv9_scaffold3460',6303,1031),(1527,302948,'Zv9_scaffold3461',6303,1032),(1528,224830,'Zv9_scaffold3462',6303,1033),(1529,664565,'Zv9_scaffold3463',6303,1034),(1530,182716,'Zv9_scaffold3464',6303,1035),(1531,500345,'Zv9_scaffold3465',6303,1036),(1532,152860,'Zv9_scaffold3466',6303,1037),(1533,195956,'Zv9_scaffold3467',6303,1038),(1534,88922,'Zv9_scaffold3468',6303,1039),(1535,118908,'Zv9_scaffold3469',6303,1040),(1536,425063,'Zv9_scaffold3470',6303,1041),(1537,638129,'Zv9_scaffold3471',6303,1042),(1538,162930,'Zv9_scaffold3472',6303,1043),(1539,352305,'Zv9_scaffold3473',6303,1044),(1540,67913,'Zv9_scaffold3474',6303,1045),(1541,162031,'Zv9_scaffold3475',6303,1046),(1542,146794,'Zv9_scaffold3476',6303,1047),(1543,292984,'Zv9_scaffold3477',6303,1048),(1544,135618,'Zv9_scaffold3478',6303,1049),(1545,80581,'Zv9_scaffold3479',6303,1050),(1546,407754,'Zv9_scaffold3480',6303,1051),(1547,164780,'Zv9_scaffold3481',6303,1052),(1548,283152,'Zv9_scaffold3482',6303,1053),(1549,213343,'Zv9_scaffold3483',6303,1054),(1550,148568,'Zv9_scaffold3484',6303,1055),(1551,341688,'Zv9_scaffold3485',6303,1056),(1552,202621,'Zv9_scaffold3486',6303,1057),(1553,231704,'Zv9_scaffold3487',6303,1058),(1554,217805,'Zv9_scaffold3488',6303,1059),(1555,167085,'Zv9_scaffold3489',6303,1060),(1556,197274,'Zv9_scaffold3490',6303,1061),(1557,284732,'Zv9_scaffold3491',6303,1062),(1558,236140,'Zv9_scaffold3492',6303,1063),(1559,171924,'Zv9_scaffold3493',6303,1064),(1560,233193,'Zv9_scaffold3494',6303,1065),(1561,321770,'Zv9_scaffold3495',6303,1066),(1562,52255,'Zv9_scaffold3496',6303,1067),(1563,110876,'Zv9_scaffold3497',6303,1068),(1564,343018,'Zv9_scaffold3498',6303,1069),(1565,44485,'Zv9_scaffold3499',6303,1070),(1566,271256,'Zv9_scaffold3500',6303,1071),(1567,159057,'Zv9_scaffold3501',6303,1072),(1568,51622,'Zv9_scaffold3502',6303,1073),(1569,965101,'Zv9_scaffold3503',6303,1074),(1570,82506,'Zv9_scaffold3504',6303,1075),(1571,159528,'Zv9_scaffold3505',6303,1076),(1572,210931,'Zv9_scaffold3506',6303,1077),(1573,149509,'Zv9_scaffold3507',6303,1078),(1574,36911,'Zv9_scaffold3508',6303,1079),(1575,266362,'Zv9_scaffold3509',6303,1080),(1576,279262,'Zv9_scaffold3510',6303,1081),(1577,156401,'Zv9_scaffold3511',6303,1082),(1578,86846,'Zv9_scaffold3512',6303,1083),(1579,38724,'Zv9_scaffold3513',6303,1084),(1580,225481,'Zv9_scaffold3514',6303,1085),(1581,276154,'Zv9_scaffold3515',6303,1086),(1582,107959,'Zv9_scaffold3516',6303,1087),(1583,225600,'Zv9_scaffold3517',6303,1088),(1584,504410,'Zv9_scaffold3518',6303,1089),(1585,192420,'Zv9_scaffold3519',6303,1090),(1586,436282,'Zv9_scaffold3520',6303,1091),(1587,192579,'Zv9_scaffold3521',6303,1092),(1588,67935,'Zv9_scaffold3522',6303,1093),(1589,49295,'Zv9_scaffold3523',6303,1094),(1590,123466,'Zv9_scaffold3524',6303,1095),(1591,90105,'Zv9_scaffold3525',6303,1096),(1592,185489,'Zv9_scaffold3526',6303,1097),(1593,22119,'Zv9_scaffold3527',6303,1098),(1594,117557,'Zv9_scaffold3528',6303,1099),(1595,212462,'Zv9_scaffold3529',6303,1100),(1596,1544009,'Zv9_scaffold3530',6303,1101),(1597,200274,'Zv9_scaffold3531',6303,1102),(1598,45094,'Zv9_scaffold3532',6303,1103),(1599,242772,'Zv9_scaffold3533',6303,1104),(1600,206488,'Zv9_scaffold3534',6303,1105),(1601,329156,'Zv9_scaffold3535',6303,1106),(1602,368818,'Zv9_scaffold3536',6303,1107),(1603,35552,'Zv9_scaffold3537',6303,1108),(1604,253640,'Zv9_scaffold3538',6303,1109),(1605,276747,'Zv9_scaffold3539',6303,1110),(1606,281342,'Zv9_scaffold3540',6303,1111),(1607,164497,'Zv9_scaffold3541',6303,1112),(1608,165668,'Zv9_scaffold3542',6303,1113),(1609,33292,'Zv9_scaffold3543',6303,1114),(1610,115763,'Zv9_scaffold3544',6303,1115),(1611,351865,'Zv9_scaffold3545',6303,1116),(1612,150703,'Zv9_scaffold3546',6303,1117),(1613,260465,'Zv9_scaffold3547',6303,1118),(1614,301592,'Zv9_scaffold3548',6303,1119),(1615,111234,'Zv9_scaffold3549',6303,1120),(1616,245711,'Zv9_scaffold3550',6303,1121),(1617,306644,'Zv9_scaffold3551',6303,1122),(1618,137468,'Zv9_scaffold3552',6303,1123),(1619,144917,'Zv9_scaffold3553',6303,1124),(1620,332968,'Zv9_scaffold3554',6303,1125),(1621,184511,'Zv9_scaffold3555',6303,1126),(1622,68961,'Zv9_scaffold3556',6303,1127),(1623,91369,'Zv9_scaffold3557',6303,1128),(1624,102840,'Zv9_scaffold3558',6303,1129),(1625,37822,'Zv9_scaffold3559',6303,1130),(1626,190731,'Zv9_scaffold3560',6303,1131),(1627,204746,'Zv9_scaffold3561',6303,1132),(1628,124569,'Zv9_scaffold3562',6303,1133),(1629,295849,'Zv9_scaffold3563',6303,1134),(1630,143554,'Zv9_scaffold3564',6303,1135),(1869,465383,'AAJU02000001',6305,1),(1870,344438,'AAJU02000002',6305,2),(1871,289304,'AAJU02000003',6305,3),(1872,253726,'AAJU02000004',6305,4),(1873,237566,'AAJU02000005',6305,5),(1874,173985,'AAJU02000006',6305,6),(1875,153371,'AAJU02000007',6305,7),(1876,150530,'AAJU02000008',6305,8),(1877,147417,'AAJU02000009',6305,9),(1878,143479,'AAJU02000010',6305,10),(1879,128336,'AAJU02000011',6305,11),(1880,115847,'AAJU02000012',6305,12),(1881,112832,'AAJU02000013',6305,13),(1882,105996,'AAJU02000014',6305,14),(1883,104275,'AAJU02000015',6305,15),(1884,102560,'AAJU02000016',6305,16),(1885,95928,'AAJU02000017',6305,17),(1886,81604,'AAJU02000018',6305,18),(1887,78823,'AAJU02000019',6305,19),(1888,77245,'AAJU02000020',6305,20),(1889,76183,'AAJU02000021',6305,21),(1890,75602,'AAJU02000022',6305,22),(1891,74681,'AAJU02000023',6305,23),(1892,73178,'AAJU02000024',6305,24),(1893,72466,'AAJU02000025',6305,25),(1894,71751,'AAJU02000026',6305,26),(1895,69134,'AAJU02000027',6305,27),(1896,68908,'AAJU02000028',6305,28),(1897,62414,'AAJU02000029',6305,29),(1898,60897,'AAJU02000030',6305,30),(1899,60810,'AAJU02000031',6305,31),(1900,60761,'AAJU02000032',6305,32),(1901,59487,'AAJU02000033',6305,33),(1902,58423,'AAJU02000034',6305,34),(1903,51753,'AAJU02000035',6305,35),(1904,48846,'AAJU02000036',6305,36),(1905,46532,'AAJU02000037',6305,37),(1906,45299,'AAJU02000038',6305,38),(1907,44040,'AAJU02000039',6305,39),(1908,39954,'AAJU02000040',6305,40),(1909,35527,'AAJU02000041',6305,41),(1910,34518,'AAJU02000042',6305,42),(1911,33268,'AAJU02000043',6305,43),(1912,33257,'AAJU02000044',6305,44),(1913,31379,'AAJU02000045',6305,45),(1914,28104,'AAJU02000046',6305,46),(1915,28024,'AAJU02000047',6305,47),(1916,27376,'AAJU02000048',6305,48),(1917,27177,'AAJU02000049',6305,49),(1918,25632,'AAJU02000050',6305,50),(1919,21207,'AAJU02000051',6305,51),(1920,21034,'AAJU02000052',6305,52),(1921,20752,'AAJU02000053',6305,53),(1922,19747,'AAJU02000054',6305,54),(1923,19655,'AAJU02000055',6305,55),(1924,17693,'AAJU02000056',6305,56),(1925,16850,'AAJU02000057',6305,57),(1926,15760,'AAJU02000058',6305,58),(1927,15635,'AAJU02000059',6305,59),(1928,15044,'AAJU02000060',6305,60),(1929,12838,'AAJU02000061',6305,61),(1930,9914,'AAJU02000062',6305,62),(1931,5955,'AAJU02000063',6305,63),(1932,5356,'AAJU02000064',6305,64),(1933,5283,'AAJU02000065',6305,65),(1934,4653,'AAJU02000066',6305,66),(1935,4558,'AAJU02000067',6305,67),(1936,4389,'AAJU02000068',6305,68),(1937,4382,'AAJU02000069',6305,69),(1938,3768,'AAJU02000070',6305,70),(1939,3480,'AAJU02000071',6305,71),(1940,3267,'AAJU02000072',6305,72),(1941,3078,'AAJU02000073',6305,73),(1942,2990,'AAJU02000074',6305,74),(1943,2913,'AAJU02000075',6305,75),(1944,2712,'AAJU02000076',6305,76),(1945,2639,'AAJU02000077',6305,77),(1946,2495,'AAJU02000078',6305,78),(1947,2339,'AAJU02000079',6305,79),(1948,1979,'AAJU02000080',6305,80),(1949,1969,'AAJU02000081',6305,81),(1950,1810,'AAJU02000082',6305,82),(1951,1738,'AAJU02000083',6305,83),(1952,1720,'AAJU02000084',6305,84),(1953,1712,'AAJU02000085',6305,85),(1954,1707,'AAJU02000086',6305,86),(1955,1690,'AAJU02000087',6305,87),(1956,1661,'AAJU02000088',6305,88),(1957,1648,'AAJU02000089',6305,89),(1958,1609,'AAJU02000090',6305,90),(1959,1595,'AAJU02000091',6305,91),(1960,1591,'AAJU02000092',6305,92),(1961,1567,'AAJU02000093',6305,93),(1962,1565,'AAJU02000094',6305,94),(1963,1550,'AAJU02000095',6305,95),(1964,1539,'AAJU02000096',6305,96),(1965,1471,'AAJU02000097',6305,97),(1966,1465,'AAJU02000098',6305,98),(1967,1460,'AAJU02000099',6305,99),(1968,1456,'AAJU02000100',6305,100),(1969,1432,'AAJU02000101',6305,101),(1970,1384,'AAJU02000102',6305,102),(1971,1326,'AAJU02000103',6305,103),(1972,1323,'AAJU02000104',6305,104),(1973,1301,'AAJU02000105',6305,105),(1974,1251,'AAJU02000106',6305,106),(1975,1227,'AAJU02000107',6305,107),(1976,1217,'AAJU02000108',6305,108),(1977,1211,'AAJU02000109',6305,109),(1978,1210,'AAJU02000110',6305,110),(1979,1192,'AAJU02000111',6305,111),(1980,1182,'AAJU02000112',6305,112),(1981,1180,'AAJU02000113',6305,113),(1982,1165,'AAJU02000114',6305,114),(1983,1161,'AAJU02000115',6305,115),(1984,1125,'AAJU02000116',6305,116),(1985,1067,'AAJU02000117',6305,117),(1986,1061,'AAJU02000118',6305,118),(1987,1032,'AAJU02000119',6305,119);
ALTER TABLE dbo.gpSegment CHECK CONSTRAINT all;
SET IDENTITY_INSERT dbo.gpSegment OFF;









--
-- Table structure for table dbo.gpAnnotation 
--

IF  EXISTS (SELECT * FROM sys.objects WHERE object_id = OBJECT_ID(N'dbo.gpAnnotation') AND type in (N'U')) DROP TABLE dbo.gpAnnotation;


CREATE TABLE dbo.gpAnnotation (
  idAnnotation int  NOT NULL identity,
  name varchar(2000) NOT NULL,
  description varchar(max)  NULL,
  fileName varchar(2000)  NULL,
  idGenomeVersion int  NOT NULL,
  codeVisibility varchar(10) NOT NULL,
  idUser int   NULL,
  idUserGroup int   NULL,
  summary varchar(5000)  NULL,
  createdBy varchar(200)  NULL,
  createDate datetime  NULL,
  isLoaded char(1) default 'N',
  idInstitute int   NULL,
  dataPath varchar(500)  NULL,
CONSTRAINT [PK_Annotation] PRIMARY KEY CLUSTERED 
(
	[idAnnotation] ASC
),
  
  
  
  
  
  CONSTRAINT FK_Annotation_GenomeVersion FOREIGN KEY (idGenomeVersion) REFERENCES dbo.gpGenomeVersion (idGenomeVersion),
  CONSTRAINT FK_Annotation_Institute FOREIGN KEY (idInstitute) REFERENCES dbo.gpInstitute (idInstitute) ON DELETE NO ACTION ON UPDATE NO ACTION,
  CONSTRAINT FK_Annotation_User FOREIGN KEY (idUser) REFERENCES dbo.gpUser (idUser),
  CONSTRAINT FK_Annotation_UserGroup FOREIGN KEY (idUserGroup) REFERENCES dbo.gpUserGroup (idUserGroup),
  CONSTRAINT FK_Annotation_Visibility FOREIGN KEY (codeVisibility) REFERENCES dbo.gpVisibility (codeVisibility)
)  ;


--
-- Dumping data for table dbo.gpAnnotation --


ALTER TABLE dbo.gpAnnotation NOCHECK CONSTRAINT all;
SET IDENTITY_INSERT dbo.gpAnnotation ON;
INSERT INTO dbo.gpAnnotation 
(
idAnnotation,
name,
description,
fileName,
idGenomeVersion,
codeVisibility,
idUser,
idUserGroup,
summary,
createdBy,
createDate,
isLoaded,
idInstitute,
dataPath
)
VALUES (2,'MiRNAs','','A2',1204,'PUBLIC',NULL,NULL,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3,'PseudoGenes','','A3',1204,'PUBLIC',NULL,NULL,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4,'Max1 Minus','','A4',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(5,'Max1 Plus','','A5',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(6,'Max10 Minus','','A6',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(7,'Max10 Plus','','A7',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(8,'Max1 Minus','','A8',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(9,'Max1 Plus','','A9',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(10,'Max10 Minus','','A10',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(11,'Max10 Plus','','A11',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(12,'Max1 Minus','','A12',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(13,'Max1 Plus','','A13',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(14,'Max10 Minus','','A14',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(15,'Max10 Plus','','A15',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(16,'Max1 - Norm Diffs','','A16',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(17,'Max1 - QValFDR','','A17',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(18,'Max1 Minus - NormDiff','','A18',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(19,'Max1 Minus - QValFDR','','A19',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(20,'Max1 Plus - NormDiff','','A20',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(21,'Max1 Plus - QValFDR','','A21',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(22,'Max10 - Norm Diff','','A22',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(23,'Max10 - QValFDR','','A23',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(24,'Max10 Minus - NormDiff','','A24',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(25,'Max10 Minus - QValFDR','','A25',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(26,'Max10 Plus - NormDiff','','A26',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(27,'Max10 Plus - QValFDR','','A27',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(28,'Max1','','A28',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(29,'Max1 Minus','','A29',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(31,'Max1 Plus','','A31',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(32,'Max10','','A32',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(33,'Max10 Minus','','A33',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(34,'Max10 Plus','','A34',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(35,'Max1','','A35',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(36,'Max1 Minus','','A36',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(37,'Max1 Plus','','A37',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(38,'Max10','','A38',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(39,'Max10 Plus','','A39',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(40,'Max10 Minus','','A40',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(41,'Max1 - BinPVal','','A41',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(42,'Max1 - QValFDR','','A42',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(43,'Max1 Minus - BinPVal','','A43',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(44,'Max1 Minus - QValFDR','','A44',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(45,'Max1 Plus - BinPVal','','A45',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(46,'Max1 Plus - QValFDR','','A46',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(47,'Max10 - BinPVal','','A47',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(48,'Max10 - QValFDR','','A48',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(49,'Max10 Minus - BinPVal','','A49',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(50,'Max10 Minus - QValFDR','','A50',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(51,'Max10 Plus - BinPVal','','A51',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(52,'Max10 Plus - QValFDR','','A52',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(53,'Minus','','A53',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(54,'Plus','','A54',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(55,'Minus','','A55',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(56,'Plus','','A56',1204,'MEM',NULL,25,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(58,'Minus','','A58',1204,'PUBLIC',NULL,NULL,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(59,'Plus','','A59',1204,'PUBLIC',NULL,NULL,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(61,'Non coding RNAs','All variety of noncoding RNAs from UCSC annotations.','A61',2701,'PUBLIC',NULL,NULL,'miRNAs, snRNAs, snoRNAs, tRNAs, U RNAs, from UCSC',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(63,'refseq','','A63',3900,'PUBLIC',NULL,NULL,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(64,'A','','A64',3900,'MEM',NULL,15,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(65,'C','','A65',3900,'MEM',NULL,15,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(66,'G','','A66',3900,'MEM',NULL,15,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(67,'T','','A67',3900,'MEM',NULL,15,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(68,'A','','A68',3900,'MEM',NULL,15,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(69,'C','','A69',3900,'MEM',NULL,15,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(70,'G','','A70',3900,'MEM',NULL,15,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(71,'T','','A71',3900,'MEM',NULL,15,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(72,'A','','A72',3900,'MEM',NULL,15,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(73,'C','','A73',3900,'MEM',NULL,15,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(74,'G','','A74',3900,'MEM',NULL,15,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(75,'T','','A75',3900,'MEM',NULL,15,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(76,'A','','A76',3900,'MEM',NULL,15,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(77,'C','','A77',3900,'MEM',NULL,15,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(78,'G','','A78',3900,'MEM',NULL,15,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(79,'T','','A79',3900,'MEM',NULL,15,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(80,'A','','A80',3900,'MEM',NULL,15,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(81,'C','','A81',3900,'MEM',NULL,15,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(82,'G','','A82',3900,'MEM',NULL,15,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(83,'T','','A83',3900,'MEM',NULL,15,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(84,'A','','A84',3900,'MEM',NULL,15,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(85,'C','','A85',3900,'MEM',NULL,15,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(86,'G','','A86',3900,'MEM',NULL,15,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(87,'T','','A87',3900,'MEM',NULL,15,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(88,'A','','A88',3900,'MEM',NULL,15,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(89,'C','','A89',3900,'MEM',NULL,15,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(90,'G','','A90',3900,'MEM',NULL,15,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(91,'T','','A91',3900,'MEM',NULL,15,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(92,'BinPVal','','A92',3806,'MEM',NULL,16,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(93,'QValFDR','','A93',3806,'MEM',NULL,16,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(94,'enriched All','','A94',3806,'MEM',NULL,16,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(95,'enriched Genes','','A95',3806,'MEM',NULL,16,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(96,'enriched Novels','','A96',3806,'MEM',NULL,16,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(97,'reduced All','','A97',3806,'MEM',NULL,16,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(98,'reduced Genes','','A98',3806,'MEM',NULL,16,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(99,'reduced Novels','','A99',3806,'MEM',NULL,16,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(100,'ctrl','','A100',3806,'MEM',NULL,16,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(101,'dss','','A101',3806,'MEM',NULL,16,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(102,'1 Lane Lymphoma','','A102',3806,'MEM',NULL,14,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(103,'4 Lanes Normal','','A103',3806,'MEM',NULL,14,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(105,'Combine','','A105',3806,'MEM',NULL,23,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(106,'Left Side','','A106',3806,'MEM',NULL,23,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(107,'Right Side','','A107',3806,'MEM',NULL,23,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(108,'Oligos','','A108',3806,'MEM',NULL,9,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(109,'PseHM','','A109',3806,'MEM',NULL,9,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(110,'All Oligos','','A110',3806,'MEM',NULL,9,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(111,'All_PseHM','','A111',3806,'MEM',NULL,9,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(112,'Partial Oligos','','A112',3806,'MEM',NULL,9,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(113,'Partial PseHM','','A113',3806,'MEM',NULL,9,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(114,'tiledOligosRepMsked','','A114',3806,'MEM',NULL,14,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(116,'refseq','','A116',5500,'PUBLIC',NULL,NULL,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(117,'centromeres','','A117',5401,'PUBLIC',NULL,NULL,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(118,'refseq','','A118',5401,'PUBLIC',NULL,NULL,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(119,'telomeres','','A119',5401,'PUBLIC',NULL,NULL,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(120,'UTRs','','A120',5401,'PUBLIC',NULL,NULL,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(121,'Histone','','A121',5401,'PUBLIC',NULL,NULL,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(122,'K36Me3','','A122',5401,'PUBLIC',NULL,NULL,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(123,'PolII','','A123',5401,'PUBLIC',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(124,'Hs_Ye_AS','','A124',5401,'PUBLIC',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(125,'Hs_Ye_S','','A125',5401,'PUBLIC',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(126,'Mm_Ye_AS','','A126',5401,'PUBLIC',NULL,NULL,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(127,'Mm_Ye_S','','A127',5401,'PUBLIC',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(128,'Mms_Ye_AS','','A128',5401,'PUBLIC',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(129,'Mms_Ye_S','','A129',5401,'PUBLIC',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(130,'PolyA_Total_AS','','A130',5401,'PUBLIC',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(131,'PolyA_Total_S','','A131',5401,'PUBLIC',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(132,'Small_PolyA_AS','','A132',5401,'PUBLIC',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(133,'Small_PolyA_S','','A133',5401,'PUBLIC',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(134,'Small_Total_AS','','A134',5401,'PUBLIC',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(135,'Small_Total_S','','A135',5401,'PUBLIC',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(136,'PolyA_AS','','A136',5401,'PUBLIC',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(137,'PolyA_S','','A137',5401,'PUBLIC',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(138,'Small_AS','','A138',5401,'PUBLIC',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(139,'Small_S','','A139',5401,'PUBLIC',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(140,'Hs_AS','','A140',5401,'PUBLIC',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(141,'Hs_S','','A141',5401,'PUBLIC',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(142,'Mm_AS','','A142',5401,'PUBLIC',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(143,'Mm_S','','A143',5401,'PUBLIC',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(144,'Mms_AS','','A144',5401,'PUBLIC',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(145,'Mms_S','','A145',5401,'PUBLIC',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(146,'Ye_AS','','A146',5401,'PUBLIC',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(147,'Ye_S','','A147',5401,'PUBLIC',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(148,'PolyA_AS','','A148',5401,'PUBLIC',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(149,'PolyA_S','','A149',5401,'PUBLIC',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(150,'Small_AS','','A150',5401,'PUBLIC',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(151,'Small_S','','A151',5401,'PUBLIC',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(152,'Hs_AS','','A152',5401,'PUBLIC',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(153,'Hs_S','','A153',5401,'PUBLIC',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(154,'Mm_AS','','A154',5401,'PUBLIC',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(155,'Mm_S','','A155',5401,'PUBLIC',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(156,'Mms_AS','','A156',5401,'PUBLIC',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(157,'Mms_S','','A157',5401,'PUBLIC',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(158,'Ye_AS','','A158',5401,'PUBLIC',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(159,'Ye_S','','A159',5401,'PUBLIC',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(160,'Dcr_Ye_AS','','A160',5401,'MEM',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(161,'Dcr_Ye_S','','A161',5401,'MEM',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(162,'Rrp6_Ye_AS','','A162',5401,'MEM',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(163,'Rrp6_Ye_S','','A163',5401,'MEM',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(164,'Spo_Ye_AS','','A164',5401,'MEM',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(165,'Spo_Ye_S','','A165',5401,'MEM',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(166,'Cap_AS','','A166',5401,'MEM',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(167,'Cap_S','','A167',5401,'MEM',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(168,'Dcr_AS','','A168',5401,'MEM',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(169,'Dcr_S','','A169',5401,'MEM',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(170,'Rrp6_AS','','A170',5401,'MEM',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(171,'Rrp6_S','','A171',5401,'MEM',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(172,'Spo_AS','','A172',5401,'MEM',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(173,'Spo_S','','A173',5401,'MEM',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(174,'Dcr_AS','','A174',5401,'MEM',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(175,'Dcr_S','','A175',5401,'MEM',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(176,'Rrp6_AS','','A176',5401,'MEM',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(177,'Rrp6_S','','A177',5401,'MEM',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(178,'Spo_AS','','A178',5401,'MEM',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(179,'Spo_S','','A179',5401,'MEM',NULL,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(234,'ERs_QValFDR_20_SubWin','','A234',3804,'MEM',NULL,19,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(235,'NormDiff','','A235',3804,'MEM',NULL,19,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(236,'QValFDR','','A236',3804,'MEM',NULL,19,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(237,'refseq','','A237',6300,'PUBLIC',3,28,'From UCSC hg19 table',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(241,'A673_Ewing','','A241',3404,'MEMCOL',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(242,'EWS502_Ewing','','A242',3404,'MEMCOL',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(243,'JS_NrmBld','','A243',3404,'MEMCOL',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(244,'KG_NrmBld','','A244',3404,'MEMCOL',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(245,'MK_NrmBld','','A245',3404,'MEMCOL',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(246,'MS_NrmBld','','A246',3404,'MEMCOL',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(247,'SKNMC_Ewing','','A247',3404,'MEMCOL',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(248,'SS_NrmBld','','A248',3404,'MEMCOL',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(249,'TC32_Ewing','','A249',3404,'MEMCOL',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(250,'TC71_Ewing','','A250',3404,'MEMCOL',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(251,'WL_NrmBld','','A251',3404,'MEMCOL',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(252,'A673_Ewing','','A252',3404,'MEMCOL',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(253,'EWS502_Ewing','','A253',3404,'MEMCOL',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(254,'JS_NrmBld','','A254',3404,'MEMCOL',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(255,'KG_NrmBld','','A255',3404,'MEMCOL',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(256,'MK_NrmBld.graphs','','A256',3404,'MEMCOL',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(257,'MS_NrmBld','','A257',3404,'MEMCOL',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(258,'SKNMC_Ewing','','A258',3404,'MEMCOL',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(259,'SS_NrmBld','','A259',3404,'MEMCOL',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(260,'TC32_Ewing','','A260',3404,'MEMCOL',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(261,'TC71_Ewing','','A261',3404,'MEMCOL',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(262,'WL_NrmBld','','A262',3404,'MEMCOL',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(263,'A673','','A263',3404,'MEMCOL',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(264,'All','','A264',3404,'MEMCOL',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(265,'EWS502','','A265',3404,'MEMCOL',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(266,'SKNMC','','A266',3404,'MEMCOL',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(267,'TC32','','A267',3404,'MEMCOL',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(268,'TC71','','A268',3404,'MEMCOL',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(269,'A673','','A269',3404,'MEMCOL',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(270,'All','','A270',3404,'MEMCOL',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(271,'EWS502','','A271',3404,'MEMCOL',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(272,'SKNMC','','A272',3404,'MEMCOL',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(273,'TC32','','A273',3404,'MEMCOL',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(274,'TC71','','A274',3404,'MEMCOL',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(275,'1D','','A275',3404,'MEM',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(276,'1G','','A276',3404,'MEM',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(277,'1R','','A277',3404,'MEM',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(278,'2D','','A278',3404,'MEM',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(279,'2G','','A279',3404,'MEM',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(280,'2R','','A280',3404,'MEM',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(281,'3D','','A281',3404,'MEM',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(282,'3G','','A282',3404,'MEM',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(283,'3R','','A283',3404,'MEM',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(284,'4D','','A284',3404,'MEM',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(285,'4G','','A285',3404,'MEM',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(286,'4R','','A286',3404,'MEM',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(287,'5D','','A287',3404,'MEM',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(288,'5G','','A288',3404,'MEM',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(289,'5R','','A289',3404,'MEM',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(290,'X348','','A290',3404,'MEM',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(291,'X416','','A291',3404,'MEM',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(292,'X8780_JustNsp','','A292',3404,'MEM',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(293,'X21058','','A293',3404,'MEM',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(294,'X128633','','A294',3404,'MEM',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(296,'348_Tumor','','A296',3404,'MEM',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(297,'416_GuthrieCard',NULL,'A297',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(298,'416_Tumor',NULL,'A298',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(299,'1058_GuthrieCard',NULL,'A299',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(301,'8633_GuthrieCard',NULL,'A301',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(302,'8633_Tumor',NULL,'A302',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(303,'8780_GuthrieCard',NULL,'A303',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(304,'8780_Tumor',NULL,'A304',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(305,'348_GuthrieCard',NULL,'A305',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(306,'348_Tumor',NULL,'A306',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(307,'416_GuthrieCard',NULL,'A307',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(308,'416_Tumor',NULL,'A308',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(309,'1058_GuthrieCard',NULL,'A309',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(310,'1058_Tumor',NULL,'A310',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(311,'8633_GuthrieCard',NULL,'A311',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(312,'8633_Tumor',NULL,'A312',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(313,'8780_GuthrieCard.graphs',NULL,'A313',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(314,'8780_Tumor',NULL,'A314',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(315,'2676_Astrocytoma',NULL,'A315',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(316,'2995_Astrocytoma',NULL,'A316',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(317,'3094_Astrocytoma',NULL,'A317',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(318,'DN26_25',NULL,'A318',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(319,'DN26_26',NULL,'A319',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(320,'DN26_27',NULL,'A320',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(321,'DN26_28',NULL,'A321',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(322,'DN26_29',NULL,'A322',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(323,'DN26_30',NULL,'A323',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(324,'DN26_31',NULL,'A324',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(325,'DN26_32',NULL,'A325',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(326,'DN26_33',NULL,'A326',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(327,'DN26_34',NULL,'A327',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(328,'DN26_35',NULL,'A328',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(329,'DN26_36',NULL,'A329',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(330,'DN26_37',NULL,'A330',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(331,'DN26_38',NULL,'A331',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(332,'DN26_39',NULL,'A332',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(333,'DN26_40','','A333',3404,'MEM',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(334,'DN26_42',NULL,'A334',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(335,'DN26_43',NULL,'A335',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(336,'DN26_44',NULL,'A336',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(337,'DN26_45',NULL,'A337',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(338,'DN26_46',NULL,'A338',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(339,'DN26_47',NULL,'A339',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(340,'DN26_48',NULL,'A340',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(341,'yw423_04','','A341',3404,'MEM',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(342,'yw423_05',NULL,'A342',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(343,'yw423_06',NULL,'A343',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(344,'yw423_07',NULL,'A344',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(345,'yw423_08',NULL,'A345',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(346,'yw423_09',NULL,'A346',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(347,'yw423_10',NULL,'A347',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(348,'yw423_11',NULL,'A348',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(349,'yw423_12',NULL,'A349',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(350,'yw423_13',NULL,'A350',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(351,'yw423_14',NULL,'A351',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(352,'yw423_15',NULL,'A352',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(353,'yw423_16',NULL,'A353',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(354,'yw423_17','','A354',3404,'MEM',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(355,'yw423_18',NULL,'A355',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(356,'yw423_19',NULL,'A356',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(357,'yw423_20',NULL,'A357',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(358,'yw423_21',NULL,'A358',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(359,'yw423_22',NULL,'A359',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(360,'yw423_23',NULL,'A360',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(361,'2676_Astrocytoma','','A361',3404,'MEM',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(362,'2995_Astrocytoma','','A362',3404,'MEM',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(363,'3094_Astrocytoma','','A363',3404,'MEM',NULL,18,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(364,'DN26_25',NULL,'A364',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(365,'DN26_26',NULL,'A365',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(366,'DN26_27',NULL,'A366',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(367,'DN26_28',NULL,'A367',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(368,'DN26_29',NULL,'A368',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(369,'DN26_30',NULL,'A369',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(370,'DN26_31',NULL,'A370',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(371,'DN26_32',NULL,'A371',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(372,'DN26_33',NULL,'A372',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(373,'DN26_34',NULL,'A373',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(374,'DN26_35',NULL,'A374',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(375,'DN26_36',NULL,'A375',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(376,'DN26_37',NULL,'A376',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(377,'DN26_38',NULL,'A377',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(378,'DN26_39',NULL,'A378',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(379,'DN26_40',NULL,'A379',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(380,'DN26_42',NULL,'A380',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(381,'DN26_43',NULL,'A381',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(382,'DN26_44',NULL,'A382',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(383,'DN26_45',NULL,'A383',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(384,'DN26_46',NULL,'A384',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(385,'DN26_47',NULL,'A385',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(386,'DN26_48',NULL,'A386',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(387,'yw423_04',NULL,'A387',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(388,'yw423_05',NULL,'A388',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(389,'yw423_06',NULL,'A389',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(390,'yw423_07',NULL,'A390',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(391,'yw423_08',NULL,'A391',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(392,'yw423_09',NULL,'A392',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(393,'yw423_10',NULL,'A393',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(394,'yw423_11',NULL,'A394',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(395,'yw423_12',NULL,'A395',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(397,'yw423_13',NULL,'A397',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(398,'yw423_14',NULL,'A398',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(399,'yw423_15',NULL,'A399',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(400,'yw423_16',NULL,'A400',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(401,'yw423_17',NULL,'A401',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(402,'yw423_18',NULL,'A402',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(403,'yw423_19',NULL,'A403',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(404,'yw423_20',NULL,'A404',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(405,'yw423_21',NULL,'A405',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(406,'yw423_22',NULL,'A406',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(407,'yw423_23',NULL,'A407',3404,'MEM',NULL,18,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(409,'Oligos',NULL,'A409',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(410,'PseHM',NULL,'A410',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(411,'Oligos',NULL,'A411',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(412,'PseHM',NULL,'A412',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(413,'Minus',NULL,'A413',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(414,'Plus',NULL,'A414',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(415,'Minus',NULL,'A415',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(416,'Plus',NULL,'A416',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(417,'Minus',NULL,'A417',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(418,'Plus',NULL,'A418',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(419,'Minus',NULL,'A419',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(420,'Plus',NULL,'A420',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(421,'BinPVal',NULL,'A421',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(422,'QValFDR',NULL,'A422',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(423,'BinPVal',NULL,'A423',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(424,'QValFDR',NULL,'A424',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(425,'BinPVal',NULL,'A425',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(426,'QValFDR',NULL,'A426',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(427,'QValFDR13Lg2Rto1',NULL,'A427',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(428,'QValFDR13Lg2Rto1',NULL,'A428',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(429,'BinPVal',NULL,'A429',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(430,'QValFDR',NULL,'A430',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(431,'Cap_Oligos',NULL,'A431',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(432,'Cap_PseHM',NULL,'A432',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(433,'Cap_RndFDRHM',NULL,'A433',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(434,'CapVsPolyA_HCV_Oligos',NULL,'A434',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(435,'CapVsPolyA_HCV_PseHM',NULL,'A435',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(436,'CapVsPolyA_HCV_RndFDRHM',NULL,'A436',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(437,'CapVsPolyA_Norm_Oligos',NULL,'A437',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(438,'CapVsPolyA_Norm_PseHM',NULL,'A438',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(439,'CapVsPolyA_Norm_RndFDRHM',NULL,'A439',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(440,'PolyA_Oligos',NULL,'A440',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(441,'PolyA_PseHM',NULL,'A441',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(442,'PolyA_RndFDRHM',NULL,'A442',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(443,'Cap_HCV_Oligos',NULL,'A443',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(444,'Cap_HCV_PseHM',NULL,'A444',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(445,'Cap_Norm_Oligos',NULL,'A445',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(446,'Cap_Norm_PseHM',NULL,'A446',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(447,'PolyA_HCV_Oligos',NULL,'A447',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(448,'PolyA_HCV_PseHM',NULL,'A448',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(449,'PolyA_Norm_Oligos',NULL,'A449',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(450,'PolyA_Norm_PseHM',NULL,'A450',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(451,'0Min',NULL,'A451',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(452,'5Min',NULL,'A452',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(453,'10Min',NULL,'A453',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(454,'15Min','','A454',3405,'MEM',NULL,16,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(455,'BinPVal',NULL,'A455',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(456,'QValFDR',NULL,'A456',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(457,'BinPVal','','A457',3405,'MEM',NULL,16,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(458,'QValFDR',NULL,'A458',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(459,'BinPVal',NULL,'A459',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(460,'QValFDR',NULL,'A460',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(461,'HCV6hrs',NULL,'A461',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(462,'HCV24hrs',NULL,'A462',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(463,'Mock6hrs',NULL,'A463',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(464,'Mock24hrs',NULL,'A464',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(466,'BinPVal',NULL,'A466',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(467,'QValFDR',NULL,'A467',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(468,'BinPVal',NULL,'A468',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(469,'QValFDR',NULL,'A469',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(470,'CappAll',NULL,'A470',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(471,'CapNovel',NULL,'A471',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(472,'PolyAAll',NULL,'A472',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(473,'PolyANovel',NULL,'A473',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(474,'HCV',NULL,'A474',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(475,'Normal',NULL,'A475',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(476,'HCV',NULL,'A476',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(477,'Normal',NULL,'A477',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(478,'BinPVal',NULL,'A478',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(479,'QValFDR',NULL,'A479',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(480,'BinPVal',NULL,'A480',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(481,'QValFDR',NULL,'A481',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(482,'HCVCap',NULL,'A482',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(483,'HCVPolyA',NULL,'A483',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(484,'NormCap',NULL,'A484',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(485,'NormPolyA',NULL,'A485',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(486,'CapAll',NULL,'A486',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(487,'CapNovel',NULL,'A487',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(488,'PolyAAll',NULL,'A488',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(489,'PolyANovel',NULL,'A489',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(490,'HCVCap',NULL,'A490',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(491,'HCVPolyA',NULL,'A491',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(492,'NormCap',NULL,'A492',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(493,'NormPolyA',NULL,'A493',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(494,'Cap',NULL,'A494',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(495,'PolyA',NULL,'A495',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(496,'Cap','','A496',3405,'MEM',NULL,16,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(497,'PolyA',NULL,'A497',3405,'MEM',NULL,16,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(498,'CTCF',NULL,'A498',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(499,'H2AZ',NULL,'A499',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(500,'H2BK5me1',NULL,'A500',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(501,'H3K4me1',NULL,'A501',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(502,'H3K4me2',NULL,'A502',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(503,'H3K4me3',NULL,'A503',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(504,'H3K9me1',NULL,'A504',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(505,'H3K9me2',NULL,'A505',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(506,'H3K9me3',NULL,'A506',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(507,'H3K27me1',NULL,'A507',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(508,'H3K27me2',NULL,'A508',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(509,'H3K27me3',NULL,'A509',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(510,'H3K36me1',NULL,'A510',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(511,'H3K36me3',NULL,'A511',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(512,'H3K79me1',NULL,'A512',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(513,'H3K79me2',NULL,'A513',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(514,'H3K79me3',NULL,'A514',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(515,'H3R2me1',NULL,'A515',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(516,'H3R2me2',NULL,'A516',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(517,'H4K20me1',NULL,'A517',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(518,'H4K20me3',NULL,'A518',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(519,'H4R3me2',NULL,'A519',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(520,'PolII',NULL,'A520',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(521,'CTCF',NULL,'A521',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(522,'H2AZ',NULL,'A522',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(523,'H2BK5me1',NULL,'A523',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(524,'H3K4me1',NULL,'A524',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(525,'H3K4me2',NULL,'A525',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(526,'H3K4me3',NULL,'A526',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(527,'H3K9me1',NULL,'A527',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(528,'H3K9me2',NULL,'A528',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(529,'H3K9me3',NULL,'A529',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(530,'H3K27me1',NULL,'A530',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(531,'H3K27me2',NULL,'A531',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(532,'H3K27me3',NULL,'A532',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(533,'H3K36me1',NULL,'A533',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(534,'H3K36me3',NULL,'A534',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(535,'H3K79me1',NULL,'A535',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(536,'H3K79me2',NULL,'A536',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(537,'H3K79me3',NULL,'A537',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(538,'H3R2me1',NULL,'A538',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(539,'H3R2me2',NULL,'A539',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(540,'H4K20me1',NULL,'A540',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(541,'H4K20me3',NULL,'A541',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(542,'H4R3me2',NULL,'A542',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(543,'PolII',NULL,'A543',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(544,'CTCF',NULL,'A544',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(545,'H2AZ',NULL,'A545',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(546,'H2BK5me1',NULL,'A546',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(547,'H3K4me1',NULL,'A547',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(548,'H3K4me2',NULL,'A548',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(549,'H3K4me3',NULL,'A549',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(550,'H3K9me1',NULL,'A550',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(551,'H3K9me2',NULL,'A551',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(552,'H3K9me3',NULL,'A552',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(553,'H3K27me1',NULL,'A553',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(554,'H3K27me2',NULL,'A554',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(555,'H3K27me3',NULL,'A555',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(556,'H3K36me1',NULL,'A556',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(557,'H3K36me3',NULL,'A557',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(558,'H3K79me1',NULL,'A558',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(559,'H3K79me2',NULL,'A559',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(560,'H3K79me3',NULL,'A560',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(561,'H3R2me1',NULL,'A561',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(562,'H3R2me2',NULL,'A562',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(563,'H4K20me1',NULL,'A563',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(564,'H4K20me3',NULL,'A564',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(565,'H4R3me2',NULL,'A565',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(566,'PolII',NULL,'A566',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(567,'E1',NULL,'A567',3405,'MEM',NULL,22,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(568,'E2',NULL,'A568',3405,'MEM',NULL,22,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(569,'E3',NULL,'A569',3405,'MEM',NULL,22,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(570,'E4',NULL,'A570',3405,'MEM',NULL,22,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(571,'E5',NULL,'A571',3405,'MEM',NULL,22,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(572,'E6',NULL,'A572',3405,'MEM',NULL,22,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(573,'E7',NULL,'A573',3405,'MEM',NULL,22,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(574,'E8',NULL,'A574',3405,'MEM',NULL,22,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(575,'E9',NULL,'A575',3405,'MEM',NULL,22,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(576,'E10',NULL,'A576',3405,'MEM',NULL,22,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(577,'E11',NULL,'A577',3405,'MEM',NULL,22,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(578,'E12',NULL,'A578',3405,'MEM',NULL,22,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(579,'E13',NULL,'A579',3405,'MEM',NULL,22,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(580,'E14',NULL,'A580',3405,'MEM',NULL,22,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(581,'E15',NULL,'A581',3405,'MEM',NULL,22,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(582,'E16',NULL,'A582',3405,'MEM',NULL,22,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(583,'E1',NULL,'A583',3405,'MEM',NULL,22,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(584,'E2',NULL,'A584',3405,'MEM',NULL,22,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(585,'E3',NULL,'A585',3405,'MEM',NULL,22,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(586,'E4',NULL,'A586',3405,'MEM',NULL,22,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(587,'E5',NULL,'A587',3405,'MEM',NULL,22,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(588,'E6',NULL,'A588',3405,'MEM',NULL,22,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(589,'E7',NULL,'A589',3405,'MEM',NULL,22,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(590,'E8',NULL,'A590',3405,'MEM',NULL,22,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(591,'E9',NULL,'A591',3405,'MEM',NULL,22,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(592,'E10',NULL,'A592',3405,'MEM',NULL,22,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(593,'E11',NULL,'A593',3405,'MEM',NULL,22,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(594,'E12',NULL,'A594',3405,'MEM',NULL,22,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(595,'E13',NULL,'A595',3405,'MEM',NULL,22,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(596,'E14',NULL,'A596',3405,'MEM',NULL,22,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(597,'E15',NULL,'A597',3405,'MEM',NULL,22,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(598,'E16',NULL,'A598',3405,'MEM',NULL,22,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(599,'Norm_Inp',NULL,'A599',3405,'MEM',NULL,14,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(600,'Polyp_Inp',NULL,'A600',3405,'MEM',NULL,14,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(601,'Polyp_Norm',NULL,'A601',3405,'MEM',NULL,14,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(602,'Norm_Inp',NULL,'A602',3405,'MEM',NULL,14,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(603,'Polyp_Inp',NULL,'A603',3405,'MEM',NULL,14,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(604,'Polyp_Norm',NULL,'A604',3405,'MEM',NULL,14,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(605,'6516',NULL,'A605',3405,'MEM',NULL,14,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(606,'6522',NULL,'A606',3405,'MEM',NULL,14,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(607,'8161',NULL,'A607',3405,'MEM',NULL,14,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(608,'8712',NULL,'A608',3405,'MEM',NULL,14,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(609,'All',NULL,'A609',3405,'MEM',NULL,14,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(610,'6516',NULL,'A610',3405,'MEM',NULL,14,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(611,'6522',NULL,'A611',3405,'MEM',NULL,14,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(612,'8161',NULL,'A612',3405,'MEM',NULL,14,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(613,'8712',NULL,'A613',3405,'MEM',NULL,14,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(614,'All',NULL,'A614',3405,'MEM',NULL,14,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(615,'All_FDR',NULL,'A615',3405,'MEM',NULL,14,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(616,'All_PVal',NULL,'A616',3405,'MEM',NULL,14,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(624,'Oligos',NULL,'A624',3405,'MEM',NULL,10,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(625,'PseHM',NULL,'A625',3405,'MEM',NULL,10,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(626,'QValHM',NULL,'A626',3405,'MEM',NULL,10,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(627,'Oligos',NULL,'A627',3405,'MEM',NULL,10,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(628,'PseHM',NULL,'A628',3405,'MEM',NULL,10,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(629,'QValHM',NULL,'A629',3405,'MEM',NULL,10,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(630,'Oligos',NULL,'A630',3405,'MEM',NULL,10,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(631,'PseHM',NULL,'A631',3405,'MEM',NULL,10,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(632,'QValHM',NULL,'A632',3405,'MEM',NULL,10,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(633,'Oligos',NULL,'A633',3405,'MEM',NULL,10,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(634,'PseHM',NULL,'A634',3405,'MEM',NULL,10,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(635,'QValHM',NULL,'A635',3405,'MEM',NULL,10,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(636,'RndFDRHM',NULL,'A636',3405,'MEM',NULL,10,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(637,'RndPValHM',NULL,'A637',3405,'MEM',NULL,10,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(638,'SumHM',NULL,'A638',3405,'MEM',NULL,10,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(639,'Oligos',NULL,'A639',3405,'MEM',NULL,10,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(640,'PseHM',NULL,'A640',3405,'MEM',NULL,10,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(641,'Oligos',NULL,'A641',3405,'MEM',NULL,10,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(642,'PseHM',NULL,'A642',3405,'MEM',NULL,10,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(643,'Oligos',NULL,'A643',3405,'MEM',NULL,10,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(644,'PseHM',NULL,'A644',3405,'MEM',NULL,10,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(645,'Oligos',NULL,'A645',3405,'MEM',NULL,10,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(646,'PseHM',NULL,'A646',3405,'MEM',NULL,10,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(647,'QValHM',NULL,'A647',3405,'MEM',NULL,10,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(648,'SumHM',NULL,'A648',3405,'MEM',NULL,10,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(649,'Oligos',NULL,'A649',3405,'MEM',NULL,10,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(650,'PseHM',NULL,'A650',3405,'MEM',NULL,10,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(651,'H2O2',NULL,'A651',3405,'MEM',NULL,20,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(652,'Input',NULL,'A652',3405,'MEM',NULL,20,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(653,'IR',NULL,'A653',3405,'MEM',NULL,20,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(654,'NoTreat',NULL,'A654',3405,'MEM',NULL,20,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(655,'ERs_254_SubWin50',NULL,'A655',3405,'MEM',NULL,20,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(656,'NormDiff',NULL,'A656',3405,'MEM',NULL,20,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(657,'QValFDR',NULL,'A657',3405,'MEM',NULL,20,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(658,'ERs_269_SubWin50',NULL,'A658',3405,'MEM',NULL,20,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(659,'NormDiff',NULL,'A659',3405,'MEM',NULL,20,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(660,'QValFDR',NULL,'A660',3405,'MEM',NULL,20,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(661,'EmpFDR',NULL,'A661',3405,'MEM',NULL,20,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(662,'ERs_248_SubWin',NULL,'A662',3405,'MEM',NULL,20,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(663,'NormDiff',NULL,'A663',3405,'MEM',NULL,20,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(664,'QValFDR',NULL,'A664',3405,'MEM',NULL,20,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(665,'EmpFDR',NULL,'A665',3405,'MEM',NULL,20,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(666,'NormDiff',NULL,'A666',3405,'MEM',NULL,20,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(667,'QValFDR',NULL,'A667',3405,'MEM',NULL,20,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(668,'EmpFDR',NULL,'A668',3405,'MEM',NULL,20,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(669,'AllSubWindowStairStepBarFiles',NULL,'A669',3405,'MEM',NULL,20,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(670,'NormDiff',NULL,'A670',3405,'MEM',NULL,20,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(671,'QValFDR',NULL,'A671',3405,'MEM',NULL,20,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(759,'Array1NimbleGenTiledRegions','','A759',3405,'MEM',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(760,'Array1RequestedRegions','','A760',3405,'MEM',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(761,'Array2NimbleGenTiledRegions','','A761',3405,'MEM',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(762,'Array2RequestedRegions','','A762',3405,'MEM',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(765,'Array1_29957','','A765',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(766,'Array1_30085','','A766',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(767,'Array1_45817','','A767',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(768,'Array1_46544','','A768',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(769,'Array1_64541','','A769',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(770,'Array1_66395','','A770',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(771,'Array1_73197','','A771',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(772,'Array1_76591','','A772',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(773,'Array1_77580','','A773',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(774,'Array1_77935','','A774',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(775,'Array1_77937','','A775',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(776,'Array1_78117','','A776',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(777,'Array1_78118','','A777',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(778,'Array1_78174','','A778',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(779,'Array2_41968','','A779',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(780,'Array2_41968Combine','','A780',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(781,'Array2_45817','','A781',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(782,'Array2_59473','','A782',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(783,'Array2_59475','','A783',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(784,'Array2_59489','','A784',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(785,'Array2_59491','','A785',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(786,'Array2_67594','','A786',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(787,'Array2_71430','','A787',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(788,'Array2_73197','','A788',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(789,'Array2_76204','','A789',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(790,'Array2_78114','','A790',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(791,'Array2_79406','','A791',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(792,'Array2_79407','','A792',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(793,'Array2_80388','','A793',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(794,'Array1_29957','','A794',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(795,'Array1_30085','','A795',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(796,'Array1_45817','','A796',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(797,'Array1_46544','','A797',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(798,'Array1_64541','','A798',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(799,'Array1_66395','','A799',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(800,'Array1_73197','','A800',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(801,'Array1_76591','','A801',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(802,'Array1_77580','','A802',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(803,'Array1_77935','','A803',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(804,'Array1_77937','','A804',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(805,'Array1_78117','','A805',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(806,'Array1_78118','','A806',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(807,'Array1_78174','','A807',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(808,'Array2_41968','','A808',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(809,'Array2_41968Combine','','A809',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(810,'Array2_45817','','A810',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(811,'Array2_59473','','A811',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(812,'Array2_59475','','A812',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(813,'Array2_59489','','A813',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(814,'Array2_59491','','A814',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(815,'Array2_67594','','A815',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(816,'Array2_71430','','A816',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(817,'Array2_73197','','A817',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(818,'Array2_76204','','A818',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(819,'Array2_78114','','A819',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(820,'Array2_79406','','A820',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(821,'Array2_79407','','A821',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(822,'Array2_80388','','A822',3405,'MEMCOL',NULL,8,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(823,'Test','','A823',3405,'MEM',3,8,'','u0028003','2010-04-13 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(881,'BinPVal','<P ALIGN=""LEFT"">Comparison of the simulated chIP vs Input ChIPSeqChallenge 1.0 datasets</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">These datasets, user submissions, and results have been archived on Sourceforge under the USeq project under <A HREF=""https://sourceforge.net/project/showfiles.php?group_id=222996&package_id=317544"" TARGET=""_blank""><B><U>CommunityChIPSeqChallenge</U></B></A>.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B>Windows</B>: several different measurements related to the overlapping window scanning summary statistics created by ScanSeqs. These are best used for visualization in the IGB. For each summary score, two types of window representations maybe generated. HeatMap is a window representation best viewed in IGB as a stair-step or heatmap. Point window summaries are where the window score is assigned to the center position in the window. Both use the xxx.bar.zip binary format.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B>BinPVal</B>: -10Log10(p-values) - Bionomial p-values for each window calculated by comparing the number of treatment reads to the number of control reads. Not multiple test corrected. Treat this as a variance corrected score for ranking purposes.</P>','A881',3405,'PUBLIC',3,28,'USeq''s ScanSeqs window level -10Log10(binomial pvalue) score from the ChIPSeqChallenge 1.0',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(882,'EmpFDR','<P ALIGN=""LEFT"">Comparison of the simulated chIP vs Input ChIPSeqChallenge 1.0 datasets</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">These datasets, user submissions, and results have been archived on Sourceforge under the USeq project under <A HREF=""https://sourceforge.net/project/showfiles.php?group_id=222996&package_id=317544"" TARGET=""_blank""><B><U>CommunityChIPSeqChallenge</U></B></A>.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B>Windows</B>: several different measurements related to the overlapping window scanning summary statistics created by ScanSeqs. These are best used for visualization in the IGB. For each summary score, two types of window representations maybe generated. HeatMap is a window representation best viewed in IGB as a stair-step or heatmap. Point window summaries are where the window score is assigned to the center position in the window. Both use the xxx.bar.zip binary format.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B>QValFDR</B>: -10Log10(q-values) - Window level bionomial p-values converted into q-value FDRs using John Storey&apos;s R package. </P>','A882',3405,'PUBLIC',3,28,'USeq''s ScanSeqs window level -10Log10(qValueFDR) score from the ChIPSeqChallenge 1.0',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(883,'QValFDR','<P ALIGN=""LEFT"">Comparison of the simulated chIP vs Input ChIPSeqChallenge 1.0 datasets</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">These datasets, user submissions, and results have been archived on Sourceforge under the USeq project under <A HREF=""https://sourceforge.net/project/showfiles.php?group_id=222996&package_id=317544/"" TARGET=""_blank""><B><U>CommunityChIPSeqChallenge</U></B></A>.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B>Windows</B>: several different measurements related to the overlapping window scanning summary statistics created by ScanSeqs. These are best used for visualization in the IGB. For each summary score, two types of window representations maybe generated. HeatMap is a window representation best viewed in IGB as a stair-step or heatmap. Point window summaries are where the window score is assigned to the center position in the window. Both use the xxx.bar.zip binary format.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B>EmpFDR</B>: -10Log10(FDR) - Empirical false discovery rates based on a control vs control null distribution.</P>','A883',3405,'PUBLIC',3,28,'USeq''s ScanSeqs window level -10Log10(empFDR) FDR from the ChIPSeqChallenge 1.0',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/');

INSERT INTO dbo.gpAnnotation 
(
idAnnotation,
name,
description,
fileName,
idGenomeVersion,
codeVisibility,
idUser,
idUserGroup,
summary,
createdBy,
createDate,
isLoaded,
idInstitute,
dataPath
)
VALUES 
(1158,'CompleteFDR10',NULL,'A1158',3405,'MEM',NULL,17,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1159,'BinPVal',NULL,'A1159',3405,'MEM',NULL,17,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1160,'EmpFDR',NULL,'A1160',3405,'MEM',NULL,17,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1161,'QValFDR',NULL,'A1161',3405,'MEM',NULL,17,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1162,'StarveFDR10',NULL,'A1162',3405,'MEM',NULL,17,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1163,'BinPVal',NULL,'A1163',3405,'MEM',NULL,17,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1164,'EmpFDR',NULL,'A1164',3405,'MEM',NULL,17,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1165,'QValFDR',NULL,'A1165',3405,'MEM',NULL,17,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1166,'ReadCoverage_adjusted',NULL,'A1166',3405,'MEM',NULL,26,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1167,'ReadCoverage_adjusted',NULL,'A1167',3405,'MEM',NULL,26,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1168,'EmpFDR',NULL,'A1168',3405,'MEM',NULL,7,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1169,'ERs_EmpFDR_20_SubWin',NULL,'A1169',3405,'MEM',NULL,7,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1170,'NormDiff',NULL,'A1170',3405,'MEM',NULL,7,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1171,'QValFDR',NULL,'A1171',3405,'MEM',NULL,7,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1259,'EmpFDR',NULL,'A1259',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1260,'ERs_EmpFDR20_SubWin',NULL,'A1260',3405,'PUBLIC',NULL,NULL,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1262,'QValFDR','<P ALIGN=""LEFT"">A reanalysis of Valoue et.al.&apos;s <A HREF=""http://mendel.stanford.edu/SidowLab/downloads/quest/index.html"" TARGET=""_blank""><B><U>GABP Nature Genetics chIP-seq dataset</U></B></A> using <A HREF=""http://useq.sourceforge.net"" TARGET=""_blank""><B><U>USeq</U></B></A><B><U>.</U></B></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B># Convert to PointData </B></P><P ALIGN=""LEFT"">jj ~/USeqApps/Tag2Point -v H_sapiens_Mar_2006 -i 3 -f /Users/nix/HCI/PIs/Graves/Seq/GABP_ValouevNatMeth08/Jurkat_RX_noIP.align_25.hg18.txt.bed </P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B># Run ScanSeqs to determine shift</B></P><P ALIGN=""LEFT"">jj ~/USeqApps/ScanSeqs -s /Users/nix/HCI/PIs/Graves/Seq/GABP_ValouevNatMeth08/NoCenterResults/Plus -t /Users/nix/HCI/PIs/Graves/Seq/GABP_ValouevNatMeth08/GABP+ -l 2927734026 -r /usr/bin/R -w 50 -m 1 -j -a</P><P ALIGN=""LEFT"">jj ~/USeqApps/ScanSeqs -s /Users/nix/HCI/PIs/Graves/Seq/GABP_ValouevNatMeth08/NoCenterResults/Min -t /Users/nix/HCI/PIs/Graves/Seq/GABP_ValouevNatMeth08/GABP- -l 2927734026 -r /usr/bin/R -w 50 -m 1 -j -a </P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B>#Filtering agaist satellite repeats and shifting 39bp using corrected filter</B></P><P ALIGN=""LEFT"">jj ~/USeqApps/FilterPointData -p /Users/nix/HCI/PIs/Graves/Seq/GABP_ValouevNatMeth08/PointData/GABP_Jurkat_SL12.align_25.hg18.txt_Point -r /Users/nix/Desktop/hg18Satellite.bed -s 39</P><P ALIGN=""LEFT"">jj ~/USeqApps/FilterPointData -p /Users/nix/HCI/PIs/Graves/Seq/GABP_ValouevNatMeth08/PointData/Jurkat_RX_noIP.align_25.hg18.txt_Point -r /Users/nix/Desktop/hg18Satellite.bed -s 39</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Fraction removed:</P><P ALIGN=""LEFT"">	0.003 (26738/7835493) GABP_Jurkat_SL12.align_25.hg18.txt_Point</P><P ALIGN=""LEFT"">	0.006 (107996/17296926) Jurkat_RX_noIP.align_25.hg18.txt_Point</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B>#Subsampled the control data to match the treatment</B></P><P ALIGN=""LEFT"">java -Xmx1000M edu/utah/seq/data/SubSamplePointData -f /Users/nix/HCI/PIs/Graves/Seq/GABP_ValouevNatMeth08/PointData/Jurkat_RX_noIP.align_25.hg18.txt_Point_hg18Satellite_Filtered_39bp/ -n 7835493 -s /Users/nix/HCI/PIs/Graves/Seq/GABP_ValouevNatMeth08/PointData/Jurkat_RX_noIP_Filtered_39_SubSampled</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B># Launch USeq Win 500bp, Min 10 reads, max control 300</B></P><P ALIGN=""LEFT"">jj ~/USeqApps/ScanSeqs \\</P><P ALIGN=""LEFT"">-s /Users/nix/HCI/PIs/Graves/Seq/GABP_ValouevNatMeth08/ResultsMin10_500bp \\</P><P ALIGN=""LEFT"">-t /Users/nix/HCI/PIs/Graves/Seq/GABP_ValouevNatMeth08/PointData/GABP_Jurkat_SL12.align_25.hg18.txt_Point_hg18Satellite_Filtered_39bp \\</P><P ALIGN=""LEFT"">-c /Users/nix/HCI/PIs/Graves/Seq/GABP_ValouevNatMeth08/PointData/Jurkat_RX_noIP.align_25.hg18.txt_Point_hg18Satellite_Filtered_39bp \\</P><P ALIGN=""LEFT"">-l 2927734026 -r /usr/bin/R -w 500 -m 10 -j -b -e -f -g 300</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Minimum number window reads 10</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Merging controls...</P><P ALIGN=""LEFT"">Merging treatments...</P><P ALIGN=""LEFT"">Random sub sampling data...</P><P ALIGN=""LEFT"">	#Ts 7835493  #Cs 17296926</P><P ALIGN=""LEFT"">	Reducing number control observations to match 2x treatments, 15670986</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Processing window size 500</P><P ALIGN=""LEFT"">	Calculating and saving normalized difference window bar files.........................</P><P ALIGN=""LEFT"">	Filtering windows based on control data window read count...</P><P ALIGN=""LEFT"">		7885(0.3%) windows removed. Minimum window sum of filtered window, 300.0</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B>#Make Enriched Regions, thresholding at 20 and 30, QValFDR and EmpFDR</B></P><P ALIGN=""LEFT"">jj ~/USeqApps/EnrichedRegionMaker \\</P><P ALIGN=""LEFT"">-t /Users/nix/HCI/PIs/Graves/Seq/GABP_ValouevNatMeth08/PointData/GABP_Jurkat_SL12.align_25.hg18.txt_Point_hg18Satellite_Filtered_39bp \\</P><P ALIGN=""LEFT"">-c /Users/nix/HCI/PIs/Graves/Seq/GABP_ValouevNatMeth08/PointData/Jurkat_RX_noIP.align_25.hg18.txt_Point_hg18Satellite_Filtered_39bp \\</P><P ALIGN=""LEFT"">-f . \\</P><P ALIGN=""LEFT"">-s 20,30 \\</P><P ALIGN=""LEFT"">-g 300 \\</P><P ALIGN=""LEFT"">-i 4 </P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">300	Max Gap</P><P ALIGN=""LEFT"">4	Score Index</P><P ALIGN=""LEFT"">50	Sub Window Size</P><P ALIGN=""LEFT"">Processing gabp.swi...</P><P ALIGN=""LEFT"">20.0	Score Threshold</P><P ALIGN=""LEFT"">	9640	EnrichedRegions created</P><P ALIGN=""LEFT"">30.0	Score Threshold</P><P ALIGN=""LEFT"">	8164	EnrichedRegions created</P><P ALIGN=""LEFT"">737263	Total number of windows</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">300	Max Gap</P><P ALIGN=""LEFT"">5	Score Index</P><P ALIGN=""LEFT"">50	Sub Window Size</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Processing gabp.swi...</P><P ALIGN=""LEFT"">20.0	Score Threshold</P><P ALIGN=""LEFT"">	8587	EnrichedRegions created</P><P ALIGN=""LEFT"">30.0	Score Threshold</P><P ALIGN=""LEFT"">	7619	EnrichedRegions created</P>','A1262',3405,'PUBLIC',3,13,'QValue False Discovery Rate Graph based on fuzzy binomial p-values',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1263,'EmpFDR',NULL,'A1263',3405,'MEM',NULL,13,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1264,'NormDiff',NULL,'A1264',3405,'MEM',NULL,13,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1265,'EmpFDR',NULL,'A1265',3405,'MEM',NULL,13,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1266,'ERs_EmpFDR30_SubWin',NULL,'A1266',3405,'MEM',NULL,13,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1267,'NormDiff',NULL,'A1267',3405,'MEM',NULL,13,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1268,'QValFDR',NULL,'A1268',3405,'MEM',NULL,13,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1269,'EmpFDR','','A1269',3405,'PUBLIC',NULL,13,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1270,'ERs_EmpFDR20_SubWin','','A1270',3405,'PUBLIC',NULL,13,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1271,'NormDiff','','A1271',3405,'PUBLIC',NULL,13,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1272,'QValFDR',NULL,'A1272',3405,'MEM',NULL,13,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1273,'EmpFDR',NULL,'A1273',3405,'MEM',NULL,13,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1274,'ERs_EmpFDR30_SubWin',NULL,'A1274',3405,'MEM',NULL,13,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1275,'NormDiff',NULL,'A1275',3405,'MEM',NULL,13,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1276,'NormDiff',NULL,'A1276',3405,'MEM',NULL,13,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1277,'QValFDR',NULL,'A1277',3405,'MEM',NULL,13,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1278,'SubWin50bp',NULL,'A1278',3405,'MEM',NULL,13,NULL,NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1285,'H3K4Me2','<P ALIGN=""LEFT"">ChIP-chip oligo level log2(mean chIP/ mean input) normalized intensity ratio from sperm.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">From: Distinctive chromatin in human sperm packages genes for embryo development. Hammoud SS, Nix DA, Zhang H, Purwar J, Carrell DT, Cairns BR. Nature. 2009 Jul 23;460(7254):473-8. Epub 2009 Jun 14.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">See <A HREF=""http://timat2.sourceforge.net/outputFileTypeDescriptions.html"" TARGET=""_blank""><B><U>TiMAT2</U></B></A> for a description of the analysis file types.</P>','A1285',3405,'PUBLIC',3,7,'ChIP-chip oligo level log2(mean chIP/ mean input) from sperm using Agilent''s Promoter Array','dnix','2009-10-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1286,'H3K4Me3','<P ALIGN=""LEFT"">ChIP-chip oligo level log2(mean chIP/ mean input) from sperm.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><I>From: Distinctive chromatin in human sperm packages genes for embryo development. Hammoud SS, Nix DA, Zhang H, Purwar J, Carrell DT, Cairns BR. Nature. 2009 Jul 23;460(7254):473-8. Epub 2009 Jun 14.</I></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">See <A HREF=""http://timat2.sourceforge.net/outputFileTypeDescriptions.html"" TARGET=""_blank""><B><U>TiMAT2</U></B></A> for a description of the analysis file types.</P>','A1286',3405,'PUBLIC',3,7,'ChIP-chip oligo level log2(mean chIP/ mean input) normalized intensity ratio from sperm.','dnix','2009-10-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1287,'H3K27Me3','<P ALIGN=""LEFT"">ChIP-chip oligo level log2(mean chIP/ mean input) from sperm.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">From: <I>Distinctive chromatin in human sperm packages genes for embryo development. Hammoud SS, Nix DA, Zhang H, Purwar J, Carrell DT, Cairns BR. Nature. 2009 Jul 23;460(7254):473-8. Epub 2009 Jun 14.</I></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">See <A HREF=""http://timat2.sourceforge.net/outputFileTypeDescriptions.html"" TARGET=""_blank""><B><U>TiMat2 File Format</U></B></A><B> </B>for a description of the analysis file types.</P>','A1287',3405,'PUBLIC',3,NULL,'ChIP-chip oligo level log2(mean chIP/ mean input) from sperm using Agilent''s Promoter Array','u0028003','2009-12-29 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1288,'Histone','<P ALIGN=""LEFT"">ChIP-chip oligo level log2(mean chIP/ mean input) from sperm.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><I>Part of Nix DA, Zhang H, Purwar J, Carrell DT, Cairns BR. Nature. 2009 Jul 23;460(7254):473-8. Epub 2009 Jun 14.</I></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">See <A HREF=""http://timat2.sourceforge.net/outputFileTypeDescriptions.html"" TARGET=""_blank""><B><U>TiMAT2</U></B></A> for a description of the analysis file types.</P>','A1288',3405,'PUBLIC',3,7,'ChIP-chip oligo level log2(mean chIP/ mean input) normalized intensity ratio from sperm.','dnix','2009-10-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1289,'MeDIPFibro',NULL,'A1289',3405,'PUBLIC',NULL,NULL,NULL,'dnix','2009-10-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1290,'MeDIPSperm',NULL,'A1290',3405,'PUBLIC',NULL,NULL,NULL,'dnix','2009-10-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1291,'MeDIPSprmVsFibro',NULL,'A1291',3405,'PUBLIC',NULL,NULL,NULL,'dnix','2009-10-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1292,'Protamine',NULL,'A1292',3405,'PUBLIC',NULL,NULL,NULL,'dnix','2009-10-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1293,'TH2B',NULL,'A1293',3405,'PUBLIC',NULL,NULL,NULL,'dnix','2009-10-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1294,'H3K4Me2','<P ALIGN=""LEFT"">ChIP-chip window level pseudomedian summary score converted to log2(chIP/input).</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><I>From: Distinctive chromatin in human sperm packages genes for embryo development. Hammoud SS, Nix DA, Zhang H, Purwar J, Carrell DT, Cairns BR. Nature. 2009 Jul 23;460(7254):473-8. Epub 2009 Jun 14.</I></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">See <A HREF=""http://timat2.sourceforge.net/outputFileTypeDescriptions.html"" TARGET=""_blank""><B><U>TiMAT2</U></B></A> for a description of the analysis file types.</P>','A1294',3405,'PUBLIC',3,7,'ChIP-chip log2(chIP/input) window score from sperm using Agilent''s Promoter Array','dnix','2009-10-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1295,'H3K4Me3','<P ALIGN=""LEFT"">ChIP-chip window level pseudomedian summary score converted to log2(chIP/input).</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><I>From: Distinctive chromatin in human sperm packages genes for embryo development. Hammoud SS, Nix DA, Zhang H, Purwar J, Carrell DT, Cairns BR. Nature. 2009 Jul 23;460(7254):473-8. Epub 2009 Jun 14.</I></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">See <A HREF=""http://timat2.sourceforge.net/outputFileTypeDescriptions.html"" TARGET=""_blank""><B><U>TiMAT2</U></B></A> for a description of the analysis file types.</P>','A1295',3405,'PUBLIC',3,7,'ChIP-chip log2(chIP/input) window score from sperm using Agilent''s Promoter Array','dnix','2009-10-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1296,'H3K27Me3','<P ALIGN=""LEFT"">ChIP-chip window level pseudomedian summary score converted to log2(chIP/input).</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><I>From: Distinctive chromatin in human sperm packages genes for embryo development. Hammoud SS, Nix DA, Zhang H, Purwar J, Carrell DT, Cairns BR. Nature. 2009 Jul 23;460(7254):473-8. Epub 2009 Jun 14.</I></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">See <A HREF=""http://timat2.sourceforge.net/outputFileTypeDescriptions.html"" TARGET=""_blank""><B><U>TiMAT2</U></B></A> for a description of the analysis file types.</P>','A1296',3405,'PUBLIC',3,7,'ChIP-chip log2(chIP/input) window score from sperm using Agilent''s Promoter Array','dnix','2009-10-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1297,'Histone','<P ALIGN=""LEFT"">ChIP-chip window level pseudomedian summary score converted to log2(chIP/input).</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><I>From: Distinctive chromatin in human sperm packages genes for embryo development. Hammoud SS, Nix DA, Zhang H, Purwar J, Carrell DT, Cairns BR. Nature. 2009 Jul 23;460(7254):473-8. Epub 2009 Jun 14.</I></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">See <A HREF=""http://timat2.sourceforge.net/outputFileTypeDescriptions.html"" TARGET=""_blank""><B><U>TiMAT2</U></B></A> for a description of the analysis file types.</P>','A1297',3405,'PUBLIC',3,7,'ChIP-chip log2(chIP/input) window score from sperm using Agilent''s Promoter Array','dnix','2009-10-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1298,'MeDIPFibro','<P ALIGN=""LEFT"">ChIP-chip window level pseudomedian summary score converted to log2(chIP/input).</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><I>From: Distinctive chromatin in human sperm packages genes for embryo development. Hammoud SS, Nix DA, Zhang H, Purwar J, Carrell DT, Cairns BR. Nature. 2009 Jul 23;460(7254):473-8. Epub 2009 Jun 14.</I></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">See <A HREF=""http://timat2.sourceforge.net/outputFileTypeDescriptions.html"" TARGET=""_blank""><B><U>TiMAT2</U></B></A> for a description of the analysis file types.</P>','A1298',3405,'PUBLIC',3,7,'ChIP-chip log2(chIP/input) window score from fibroblasts using Agilent''s Promoter Array','dnix','2009-10-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1299,'MeDIPSperm','<P ALIGN=""LEFT"">ChIP-chip window level pseudomedian summary score converted to log2(chIP/input).</P><P ALIGN=""LEFT""><I></I></P><P ALIGN=""LEFT""><I>From: Distinctive chromatin in human sperm packages genes for embryo development. Hammoud SS, Nix DA, Zhang H, Purwar J, Carrell DT, Cairns BR. Nature. 2009 Jul 23;460(7254):473-8. Epub 2009 Jun 14.</I></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">See <A HREF=""http://timat2.sourceforge.net/outputFileTypeDescriptions.html"" TARGET=""_blank""><B><U>TiMAT2</U></B></A> for a description of the analysis file types.</P>','A1299',3405,'PUBLIC',3,7,'ChIP-chip log2(chIP/input) window score from sperm using Agilent''s Promoter Array','dnix','2009-10-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1300,'MeDIPSprmVsFibro','<P ALIGN=""LEFT"">ChIP-chip window level pseudomedian summary score converted to log2(sperm/fibro).</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><I>From: Distinctive chromatin in human sperm packages genes for embryo development. Hammoud SS, Nix DA, Zhang H, Purwar J, Carrell DT, Cairns BR. Nature. 2009 Jul 23;460(7254):473-8. Epub 2009 Jun 14.</I></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">See <A HREF=""http://timat2.sourceforge.net/outputFileTypeDescriptions.html"" TARGET=""_blank""><B><U>TiMAT2</U></B></A> for a description of the analysis file types.</P>','A1300',3405,'PUBLIC',3,7,'ChIP-chip log2(sperm/fibro) window score from sperm using Agilent''s Promoter Array','dnix','2009-10-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1301,'Protamine','<P ALIGN=""LEFT"">ChIP-chip window level pseudomedian summary score converted to log2(chIP/input).</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><I>From: Distinctive chromatin in human sperm packages genes for embryo development. Hammoud SS, Nix DA, Zhang H, Purwar J, Carrell DT, Cairns BR. Nature. 2009 Jul 23;460(7254):473-8. Epub 2009 Jun 14.</I></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">See <A HREF=""http://timat2.sourceforge.net/outputFileTypeDescriptions.html"" TARGET=""_blank""><B><U>TiMAT2</U></B></A> for a description of the analysis file types.</P>','A1301',3405,'PUBLIC',3,7,'ChIP-chip log2(chIP/input) window score from sperm using Agilent''s Promoter Array','dnix','2009-10-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1302,'enH3K27Me3__100',NULL,'A1302',3405,'PUBLIC',NULL,NULL,NULL,'dnix','2009-10-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1303,'enH3K27Me3__400',NULL,'A1303',3405,'PUBLIC',NULL,NULL,NULL,'dnix','2009-10-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1305,'Not Assembled Contigs','Contigs from Zv7_NA1- Zv7_NA7971 were concatinated with 1KB on N''s as spacers to for an artificial chromosome called chrNA.  Annotation (sequence and refseq gene models) coordinates were shifted to match.','A1305',2701,'PUBLIC',3,28,'Location of concatinated contigs','dnix','2009-10-14 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1306,'Scaffolds','Scaffolds from Zv7_scaffold2485- Zv7_scaffold2650 were concatinated with 1KB on N''s as spacers to for an artificial chromosome called chrScaffold.  Annotation (sequence and refseq gene models) coordinates were shifted to match.','A1306',2701,'PUBLIC',3,28,'Locations of concatinated scaffolds','dnix','2009-10-14 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1307,'RefSeq With chrNA chrScaffold','Refseq annotation from UCSC with shifted chrScaffold and chrNA  gene models.','A1307',2701,'PUBLIC',3,28,'Gene annotation with shifted models for chrNA and chrScaffold','dnix','2009-10-14 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1329,'H2Az','<P ALIGN=""LEFT"">A window level comparison graph showing enrichment as a function of FDR for H2Az ChIP vs Input from healthy donor sperm</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">EmpFDR: -10Log10(FDR) - Empirical false discovery rates based on a control vs control null distribution.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">http://useq.sourceforge.net</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">From: <A HREF=""http://dx.doi.org/10.1038/nature08162"" TARGET=""_blank""><U>Hammoud SS, Nix DA, Zhang H, Purwar J, Carrell DT, Cairns BR. Distinctive chromatin in human sperm packages genes for embryo development. Nature. 2009 Jul 23;460(7254):473-8</U></A></P>','A1329',3405,'PUBLIC',3,7,'USeq window level empirical false discovery rate graph','dnix','2009-10-30 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1330,'H3K4me3','<P ALIGN=""LEFT"">A window level comparison graph showing enrichment as a function of FDR for H3K4me3 ChIP vs Input from healthy donor sperm</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">EmpFDR: -10Log10(FDR) - Empirical false discovery rates based on a control vs control null distribution.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><A HREF=""http://useq.sourceforge.net"" TARGET=""_blank""><U>http://useq.sourceforge.net</U></A></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">From: <A HREF=""http://dx.doi.org/10.1038/nature08162"" TARGET=""_blank""><U>Hammoud SS, Nix DA, Zhang H, Purwar J, Carrell DT, Cairns BR. Distinctive chromatin in human sperm packages genes for embryo development. Nature. 2009 Jul 23;460(7254):473-8</U></A></P>','A1330',3405,'PUBLIC',3,7,'USeq window level empirical false discovery rate graph','dnix','2009-10-30 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1331,'H3K27me3','<P ALIGN=""LEFT"">A window level comparison graph showing enrichment as a function of FDR for H3K27me3 ChIP vs Input from healthy donor sperm</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">EmpFDR: -10Log10(FDR) - Empirical false discovery rates based on a control vs control null distribution.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><A HREF=""http://useq.sourceforge.net"" TARGET=""_blank""><U>http://useq.sourceforge.net</U></A></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">From: <A HREF=""http://dx.doi.org/10.1038/nature08162"" TARGET=""_blank""><U>Hammoud SS, Nix DA, Zhang H, Purwar J, Carrell DT, Cairns BR. Distinctive chromatin in human sperm packages genes for embryo development. Nature. 2009 Jul 23;460(7254):473-8</U></A></P>','A1331',3405,'PUBLIC',3,7,'USeq window level empirical false discovery rate graph','dnix','2009-10-30 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1332,'Histone','<P ALIGN=""LEFT"">A window level comparison graph showing enrichment as a function of FDR for Histone vs Input from healthy donor sperm</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">EmpFDR: -10Log10(FDR) - Empirical false discovery rates based on a control vs control null distribution.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><A HREF=""http://useq.sourceforge.net"" TARGET=""_blank""><U>http://useq.sourceforge.net</U></A></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">From: <A HREF=""http://dx.doi.org/10.1038/nature08162"" TARGET=""_blank""><U>Hammoud SS, Nix DA, Zhang H, Purwar J, Carrell DT, Cairns BR. Distinctive chromatin in human sperm packages genes for embryo development. Nature. 2009 Jul 23;460(7254):473-8</U></A></P>','A1332',3405,'PUBLIC',3,7,'USeq window level empirical false discovery rate graph','dnix','2009-10-30 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1333,'Histone','<P ALIGN=""LEFT"">A window level comparison graph showing enrichment as for Histone vs Input from healthy donor sperm</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">NormDiff enrichment score (Sum T - Sum C)/ Sqrt (Sum T + Sum C)</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><A HREF=""http://useq.sourceforge.net"" TARGET=""_blank""><U>http://useq.sourceforge.net</U></A></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">From: <A HREF=""http://dx.doi.org/10.1038/nature08162"" TARGET=""_blank""><U>Hammoud SS, Nix DA, Zhang H, Purwar J, Carrell DT, Cairns BR. Distinctive chromatin in human sperm packages genes for embryo development. Nature. 2009 Jul 23;460(7254):473-8</U></A></P>','A1333',3405,'PUBLIC',3,7,'USeq window level normalized difference enrichment','dnix','2009-10-30 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1334,'H3K27me3','<P ALIGN=""LEFT"">A window level comparison graph showing enrichment for H3K27me3 ChIP vs Input from healthy donor sperm</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">NormDiff enrichment score (Sum T - Sum C)/ Sqrt (Sum T + Sum C)</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><A HREF=""http://useq.sourceforge.net"" TARGET=""_blank""><U>http://useq.sourceforge.net</U></A></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">From: <A HREF=""http://dx.doi.org/10.1038/nature08162"" TARGET=""_blank""><U>Hammoud SS, Nix DA, Zhang H, Purwar J, Carrell DT, Cairns BR. Distinctive chromatin in human sperm packages genes for embryo development. Nature. 2009 Jul 23;460(7254):473-8</U></A></P>','A1334',3405,'PUBLIC',3,7,'USeq window level normalized difference enrichment',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1335,'H3K4me3','<P ALIGN=""LEFT"">A window level comparison graph showing enrichment for H3K4me3 ChIP vs Input from healthy donor sperm</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">NormDiff enrichment score (Sum T - Sum C)/ Sqrt (Sum T + Sum C)</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><A HREF=""http://useq.sourceforge.net"" TARGET=""_blank""><U>http://useq.sourceforge.net</U></A></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">From: <A HREF=""http://dx.doi.org/10.1038/nature08162"" TARGET=""_blank""><U>Hammoud SS, Nix DA, Zhang H, Purwar J, Carrell DT, Cairns BR. Distinctive chromatin in human sperm packages genes for embryo development. Nature. 2009 Jul 23;460(7254):473-8</U></A></P>','A1335',3405,'PUBLIC',3,7,'USeq window level normalized difference enrichment',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1336,'H2Az','<P ALIGN=""LEFT"">A window level comparison graph showing enrichment for H2Az ChIP vs Input from healthy donor sperm</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">NormDiff enrichment score (Sum T - Sum C)/ Sqrt (Sum T + Sum C)</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><A HREF=""http://useq.sourceforge.net"" TARGET=""_blank""><U>http://useq.sourceforge.net</U></A></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">From: <A HREF=""http://dx.doi.org/10.1038/nature08162"" TARGET=""_blank""><U>Hammoud SS, Nix DA, Zhang H, Purwar J, Carrell DT, Cairns BR. Distinctive chromatin in human sperm packages genes for embryo development. Nature. 2009 Jul 23;460(7254):473-8</U></A></P>','A1336',3405,'PUBLIC',3,7,'USeq window level normalized difference enrichment',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1337,'Histone','<P ALIGN=""LEFT"">A window level comparison graph showing q-value FDRs for Histone vs Input from healthy donor sperm</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">QValFDR: -10Log10(q-values) - Window level bionomial p-values converted into q-value FDRs using John Storey&apos;s R package. </P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><A HREF=""http://useq.sourceforge.net"" TARGET=""_blank""><U>http://useq.sourceforge.net</U></A></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">From: <A HREF=""http://dx.doi.org/10.1038/nature08162"" TARGET=""_blank""><U>Hammoud SS, Nix DA, Zhang H, Purwar J, Carrell DT, Cairns BR. Distinctive chromatin in human sperm packages genes for embryo development. Nature. 2009 Jul 23;460(7254):473-8</U></A></P>','A1337',3405,'PUBLIC',3,7,'USeq window level QValue False Discovery Rates','dnix','2009-10-30 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1338,'H2Az','<P ALIGN=""LEFT"">A window level comparison graph showing q-value FDRs for H2Az ChIP vs Input from healthy donor sperm</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">QValFDR: -10Log10(q-values) - Window level bionomial p-values converted into q-value FDRs using John Storey&apos;s R package. </P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><A HREF=""http://useq.sourceforge.net"" TARGET=""_blank""><U>http://useq.sourceforge.net</U></A></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">From: <A HREF=""http://dx.doi.org/10.1038/nature08162"" TARGET=""_blank""><U>Hammoud SS, Nix DA, Zhang H, Purwar J, Carrell DT, Cairns BR. Distinctive chromatin in human sperm packages genes for embryo development. Nature. 2009 Jul 23;460(7254):473-8</U></A></P>','A1338',3405,'PUBLIC',3,7,'USeq window level QValue False Discovery Rates','dnix','2009-10-30 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1339,'H3K4me3','<P ALIGN=""LEFT"">A window level comparison graph showing q-value FDRs for H3K4me3 ChIP vs Input from healthy donor sperm</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">QValFDR: -10Log10(q-values) - Window level bionomial p-values converted into q-value FDRs using John Storey&apos;s R package. </P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><A HREF=""http://useq.sourceforge.net"" TARGET=""_blank""><U>http://useq.sourceforge.net</U></A></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">From: <A HREF=""http://dx.doi.org/10.1038/nature08162"" TARGET=""_blank""><U>Hammoud SS, Nix DA, Zhang H, Purwar J, Carrell DT, Cairns BR. Distinctive chromatin in human sperm packages genes for embryo development. Nature. 2009 Jul 23;460(7254):473-8</U></A></P>','A1339',3405,'PUBLIC',3,7,'USeq window level QValue False Discovery Rates','dnix','2009-10-30 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1340,'H3K27me3','<P ALIGN=""LEFT"">A window level comparison graph showing q-value FDRs for H3K27me3 ChIP vs Input from healthy donor sperm</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">QValFDR: -10Log10(q-values) - Window level bionomial p-values converted into q-value FDRs using John Storey&apos;s R package. </P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><A HREF=""http://useq.sourceforge.net"" TARGET=""_blank""><U>http://useq.sourceforge.net</U></A></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">From: <A HREF=""http://dx.doi.org/10.1038/nature08162"" TARGET=""_blank""><U>Hammoud SS, Nix DA, Zhang H, Purwar J, Carrell DT, Cairns BR. Distinctive chromatin in human sperm packages genes for embryo development. Nature. 2009 Jul 23;460(7254):473-8</U></A></P>','A1340',3405,'PUBLIC',3,7,'USeq window level QValue False Discovery Rates',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1341,'Ultra Conserved Elements','<P ALIGN=""LEFT"">Charleston Chiang used the UCSC liftover utility to effect the conversion to hg18 from hg17.  From Harvard Med.</P>','A1341',3405,'PUBLIC',3,28,'','dnix','2009-11-02 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1345,'ensembl','<P ALIGN=""LEFT"">Downloaded from UCSC who derived the table from ensembl for hg19</P>','A1345',6300,'PUBLIC',3,28,'gene transcript annotation','dnix','2009-11-30 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1354,'Gm12878CfosV2','','A1354',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1355,'Gm12878JundV2','','A1355',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1356,'Gm12878MaxV2','','A1356',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1357,'Gm12878NfkbTnfa','','A1357',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1358,'Gm12878Pol2V2','','A1358',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1359,'Gm12878Pol3','','A1359',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1360,'Hct116Pol2','','A1360',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1361,'Hct116Tcf4V2','','A1361',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1362,'Hek293Pol2V2','','A1362',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1363,'Helas3Ap2alpha','','A1363',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1364,'Helas3Ap2gamma','','A1364',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1365,'Helas3Cfos','','A1365',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1366,'Helas3Cmyc','','A1366',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1367,'Helas3E2f4','','A1367',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1368,'Helas3Cmyc_copy','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&amp;g=wgEncodeYaleChIPseq</U></A></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Converted to xxx.useq binary format using the <A HREF=""http://useq.sourceforge.net/cmdLnMenus.html#Wig2USeq"" TARGET=""_blank""><U>Wig2USeq app</U></A> (assuming column 8 is qvalue, java -jar -Xmx10G ~/USeqApps/Text2USeq -f /home/u0028003/ENCODE/YaleTFBS/NarrowPeak -g H_sapiens_Mar_2006 -c 0 -b 1 -e 2 -v 8)</P>','A1368',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'N',NULL,'/Data02/GenoPub/GenoPubData/'),
(1369,'Helas3Ha','','A1369',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1370,'Helas3ifngStat1','','A1370',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1371,'Helas3Largefragment','','A1371',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1372,'Helas3Max','','A1372',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1373,'Helas3Mouseigg','','A1373',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1374,'Helas3Pol2V2','','A1374',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1375,'Helas3Tr4','','A1375',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1376,'Hepg2bTr4','','A1376',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1377,'Hepg2Pol2','','A1377',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1378,'Hepg2Srebp1a','','A1378',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1379,'Hepg2Srebp2','','A1379',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1380,'K562Bdp1','','A1380',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1381,'K562bE2f4','','A1381',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1382,'K562bE2f6','','A1382',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1383,'K562bGata1V2','','A1383',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1384,'K562bGata2','','A1384',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1385,'K562Brf1','','A1385',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1386,'K562bSetdb1','','A1386',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1387,'K562bTr4','','A1387',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1388,'K562bYy1','','A1388',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1389,'K562bZnf263V2','','A1389',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1390,'K562CfosV2','','A1390',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1391,'K562CjunV2','','A1391',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1392,'K562CmycV2','','A1392',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1393,'K562Ifna6hCjun','','A1393',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1394,'K562Ifna6hCmyc','','A1394',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1395,'K562Ifna6hPol2','','A1395',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1396,'K562Ifna6hStat1','','A1396',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1397,'K562Ifna6hStat2','','A1397',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1398,'K562Ifna30Cmyc','','A1398',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1399,'K562Ifna30Pol2','','A1399',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1400,'K562Ifna30Stat1','','A1400',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1401,'K562Ifna30Stat2','','A1401',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1402,'K562Ifng6hCjun','','A1402',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1403,'K562Ifng6hCmyc','','A1403',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1404,'K562Ifng6hPol2','','A1404',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1405,'K562Ifng30Cjun','','A1405',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1406,'K562JundV2','','A1406',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1407,'K562MaxV2','','A1407',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1408,'K562Nfe2V2','','A1408',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1409,'K562Nfya','','A1409',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1410,'K562Nfyb','','A1410',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1411,'K562Pol2V2','','A1411',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1412,'K562Sirt6','','A1412',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1413,'K562Xrcc4','','A1413',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1414,'Nb4Pol2V2','','A1414',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1415,'Nt2d1Suz12','','A1415',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1416,'Nt2d1Yy1','','A1416',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1424,'EnrichedRegionsAll','','A1424',6300,'MEM',3,30,'Enriched regions with a < 1% FDR and 2x enrichment','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1425,'EnrichedRegions0.5kb','','A1425',6300,'MEM',3,30,'Enriched regions with a < 1% FDR, 2x enrichment, and further than 0.5kb away from a known ensembl exon','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1426,'EnrichedRegions1kb','','A1426',6300,'MEM',3,30,'Enriched regions with a < 1% FDR, 2x enrichment, and further than 1kb away from a known ensembl exon',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1427,'ReducedRegionsAll','','A1427',6300,'MEM',3,30,'Reduced regions with a < 1% FDR and 2x reduction',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1428,'ReducedRegions0.5kb','','A1428',6300,'MEM',3,30,'Reduced regions with a < 1% FDR, 2x reduction and further than 0.5kb away from a known ensembl exon','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1429,'ReducedRegions1kb','','A1429',6300,'MEM',3,30,'Reduced regions with a < 1% FDR, 2x reduction and further than 1kb away from a known ensembl exon',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1430,'BinomialPValue','','A1430',6300,'MEM',3,30,'200bp window -10Log10(binomialPValue) summary statistic for enrichment or -1 for reduction','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1431,'QValueFDR','','A1431',6300,'MEM',3,30,'200bp window -10Log10(qValueFDR) summary statistic for enrichment or -1 for reduction',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1432,'Megakaryocytes','','A1432',6300,'MEM',3,30,'Single read splice junctions derived from the NovoalignParser using Ensembl transcript gene models','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1433,'Platelets','','A1433',6300,'MEM',3,30,'Single read splice junctions derived from the NovoalignParser using Ensembl transcript gene models',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1434,'MegakaryocytesNM','','A1434',6300,'MEM',3,30,'','dnix','2009-12-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1435,'PlateletsNT','','A1435',6300,'MEM',3,30,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1449,'Gm12878Ctcf','<P ALIGN=""LEFT"">See <A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><B><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&amp;g=wgEncodeBroadChipSeq</U></B></A></P>','A1449',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1450,'Hepg2H3k4me3','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1450',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1451,'HuvecH3k4me3','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1451',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1452,'NhekH3k27me3','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1452',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1453,'Gm12878H3k27ac','<P ALIGN=""LEFT"">See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&amp;g=wgEncodeBroadChipSeq</P>','A1453',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1454,'Hepg2H3k9ac','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1454',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1455,'HuvecH3k9ac','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1455',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1456,'NhekH3k36me3','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1456',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1457,'Gm12878H3k27me3','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1457',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1458,'Hepg2H4k20me1','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1458',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1459,'HuvecH3k9me1','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1459',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1460,'NhekH3k4me1','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1460',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1461,'Gm12878H3k36me3','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1461',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1462,'HmecCtcf','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1462',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1463,'HuvecH4k20me1','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1463',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1464,'NhekH3k4me2','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1464',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1465,'Gm12878H3k4me1','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1465',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1466,'HmecH3k27ac','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1466',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1467,'HuvecPol2b','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1467',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1468,'NhekH3k4me3','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1468',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1469,'Gm12878H3k4me2','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1469',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1470,'HmecH3k27me3','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1470',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1471,'K562Ctcf','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1471',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1472,'NhekH3k9ac','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1472',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1473,'Gm12878H3k4me3','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1473',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1474,'HmecH3k36me3','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1474',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1475,'K562H3k27ac','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1475',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1476,'NhekH3k9me1','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1476',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1477,'Gm12878H3k9ac','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1477',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1478,'HmecH3k4me1','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1478',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1479,'K562H3k27me3','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1479',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1480,'NhekH4k20me1','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1480',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1481,'Gm12878H4k20me1','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1481',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1482,'HmecH3k4me2','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1482',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1483,'K562H3k36me3','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1483',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1484,'NhekPol2b','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1484',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1485,'H1hescCtcf','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1485',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1486,'HmecH3k4me3','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1486',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1487,'K562H3k4me1','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1487',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1488,'NhlfH3k27ac','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1488',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1489,'H1hescH3k27me3','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1489',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1490,'HmecH3k9ac','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1490',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1491,'K562H3k4me2','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1491',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1492,'NhlfH3k27me3','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1492',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1493,'H1hescH3k4me1','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1493',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1494,'HmecH4k20me1','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1494',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1495,'K562H3k4me3','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1495',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1496,'NhlfH3k36me3','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1496',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1497,'H1hescH3k4me3','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1497',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1498,'HuvecCtcf','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1498',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1499,'K562H3k9ac','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1499',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1500,'NhlfH3k4me1','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1500',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1501,'H1hescH4k20me1','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1501',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1502,'HuvecH3k27ac','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1502',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1503,'K562H3k9me1','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1503',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1504,'NhlfH3k4me2','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1504',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1505,'Hepg2Ctcf','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1505',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1506,'HuvecH3k27me3','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1506',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1507,'K562H4k20me1','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1507',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1508,'NhlfH3k4me3','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1508',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1509,'Hepg2H3k27ac','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1509',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1510,'HuvecH3k36me3','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1510',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1511,'K562Pol2b','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1511',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1512,'NhlfH3k9ac','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1512',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1513,'Hepg2H3k36me3','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1513',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1514,'HuvecH3k4me1','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1514',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1515,'NhekCtcf','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1515',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1516,'NhlfH4k20me1','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1516',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1517,'Hepg2H3k4me2','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1517',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1518,'HuvecH3k4me2','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1518',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1519,'NhekH3k27ac','See http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq','A1519',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1521,'BjtertRep1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&amp;g=wgEncodeUwDnaseSeq</U></A></P>','A1521',3405,'PUBLIC',3,28,'Hot Spot calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1522,'Hepg2Rep1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&amp;g=wgEncodeUwDnaseSeq</U></A></P>','A1522',3405,'PUBLIC',3,28,'Hot Spot calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1523,'K562Rep1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&amp;g=wgEncodeUwDnaseSeq</U></A></P>','A1523',3405,'PUBLIC',3,28,'Hot Spot calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1524,'BjtertRep2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&amp;g=wgEncodeUwDnaseSeq</U></A></P>','A1524',3405,'PUBLIC',3,28,'Hot Spot calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1525,'Hepg2Rep2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&amp;g=wgEncodeUwDnaseSeq</U></A></P>','A1525',3405,'PUBLIC',3,28,'Hot Spot calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1526,'Caco2Rep1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&amp;g=wgEncodeUwDnaseSeq</U></A></P>','A1526',3405,'PUBLIC',3,28,'Hot Spot calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1527,'Hl60Rep1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&amp;g=wgEncodeUwDnaseSeq</U></A></P>','A1527',3405,'PUBLIC',3,28,'Hot Spot calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1528,'SkmcRep1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&amp;g=wgEncodeUwDnaseSeq</U></A></P>','A1528',3405,'PUBLIC',3,28,'Hot Spot calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1529,'Caco2Rep2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&amp;g=wgEncodeUwDnaseSeq</U></A></P>','A1529',3405,'PUBLIC',3,28,'Hot Spot calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1530,'HrcepicRep2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&amp;g=wgEncodeUwDnaseSeq</U></A></P>','A1530',3405,'PUBLIC',3,28,'Hot Spot calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1531,'Gm06990Rep1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&amp;g=wgEncodeUwDnaseSeq</U></A></P>','A1531',3405,'PUBLIC',3,28,'Hot Spot calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1532,'HrcepicRep1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&amp;g=wgEncodeUwDnaseSeq</U></A></P>','A1532',3405,'PUBLIC',3,28,'Hot Spot calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1533,'SknshraRep1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&amp;g=wgEncodeUwDnaseSeq</U></A></P>','A1533',3405,'PUBLIC',3,28,'Hot Spot calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1534,'Gm06990Rep2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&amp;g=wgEncodeUwDnaseSeq</U></A></P>','A1534',3405,'PUBLIC',3,28,'Hot Spot calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1535,'HreRep2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&amp;g=wgEncodeUwDnaseSeq</U></A></P>','A1535',3405,'PUBLIC',3,28,'Hot Spot calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1536,'Gm12878Rep1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&amp;g=wgEncodeUwDnaseSeq</U></A></P>','A1536',3405,'PUBLIC',3,28,'Hot Spot calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1537,'HreRep1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&amp;g=wgEncodeUwDnaseSeq</U></A></P>','A1537',3405,'PUBLIC',3,28,'Hot Spot calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1538,'Th1Rep1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&amp;g=wgEncodeUwDnaseSeq</U></A></P>','A1538',3405,'PUBLIC',3,28,'Hot Spot calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1539,'Gm12878Rep2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&amp;g=wgEncodeUwDnaseSeq</U></A></P>','A1539',3405,'PUBLIC',3,28,'Hot Spot calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1540,'K562Rep2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&amp;g=wgEncodeUwDnaseSeq</U></A></P>','A1540',3405,'PUBLIC',3,28,'Hot Spot calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1541,'Helas3Rep1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&amp;g=wgEncodeUwDnaseSeq</U></A></P>','A1541',3405,'PUBLIC',3,28,'Hot Spot calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1542,'HuvecRep1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&amp;g=wgEncodeUwDnaseSeq</U></A></P>','A1542',3405,'PUBLIC',3,28,'Hot Spot calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1543,'Th2Rep1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&amp;g=wgEncodeUwDnaseSeq</U></A></P>','A1543',3405,'PUBLIC',3,28,'Hot Spot calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1544,'Helas3Rep2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&amp;g=wgEncodeUwDnaseSeq</U></A></P>','A1544',3405,'PUBLIC',3,28,'Hot Spot calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1545,'SknshraRep2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&amp;g=wgEncodeUwDnaseSeq</U></A></P>','A1545',3405,'PUBLIC',3,28,'Hot Spot calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1546,'BjtertRep1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq</U></A></P>','A1546',3405,'PUBLIC',3,28,'Peak calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1547,'Caco2Rep1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq</U></A></P>','A1547',3405,'PUBLIC',3,28,'Peak calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1548,'Gm06990Rep1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq</U></A></P>','A1548',3405,'PUBLIC',3,28,'Peak calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1549,'Gm12878Rep1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq</U></A></P>','A1549',3405,'PUBLIC',3,28,'Peak calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1550,'Helas3Rep1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq</U></A></P>','A1550',3405,'PUBLIC',3,28,'Peak calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1551,'Hepg2Rep1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq</U></A></P>','A1551',3405,'PUBLIC',3,28,'Peak calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1552,'Hl60Rep1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq</U></A></P>','A1552',3405,'PUBLIC',3,28,'Peak calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1553,'HrcepicRep1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq</U></A></P>','A1553',3405,'PUBLIC',3,28,'Peak calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1554,'HreRep1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq</U></A></P>','A1554',3405,'PUBLIC',3,28,'Peak calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1555,'HuvecRep1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq</U></A></P>','A1555',3405,'PUBLIC',3,28,'Peak calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1556,'K562Rep1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq</U></A></P>','A1556',3405,'PUBLIC',3,28,'Peak calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1557,'SkmcRep1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq</U></A></P>','A1557',3405,'PUBLIC',3,28,'Peak calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1558,'SknshraRep1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq</U></A></P>','A1558',3405,'PUBLIC',3,28,'Peak calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1559,'Th1Rep1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq</U></A></P>','A1559',3405,'PUBLIC',3,28,'Peak calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1560,'Th2Rep1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq</U></A></P>','A1560',3405,'PUBLIC',3,28,'Peak calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1561,'BjtertRep2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq</U></A></P>','A1561',3405,'PUBLIC',3,28,'Peak calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1562,'Caco2Rep2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq</U></A></P>','A1562',3405,'PUBLIC',3,28,'Peak calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1563,'Gm06990Rep2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq</U></A></P>','A1563',3405,'PUBLIC',3,28,'Peak calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1564,'Gm12878Rep2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq</U></A></P>','A1564',3405,'PUBLIC',3,28,'Peak calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1565,'Helas3Rep2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq</U></A></P>','A1565',3405,'PUBLIC',3,28,'Peak calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1566,'Hepg2Rep2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq</U></A></P>','A1566',3405,'PUBLIC',3,28,'Peak calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1567,'HrcepicRep2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq</U></A></P>','A1567',3405,'PUBLIC',3,28,'Peak calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1568,'HreRep2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq</U></A></P>','A1568',3405,'PUBLIC',3,28,'Peak calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1569,'K562Rep2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq</U></A></P>','A1569',3405,'PUBLIC',3,28,'Peak calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1570,'SknshraRep2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq</U></A></P>','A1570',3405,'PUBLIC',3,28,'Peak calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1571,'BjtertRep1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq</U></A></P>','A1571',3405,'PUBLIC',3,28,'Signal calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1572,'Caco2Rep1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq</U></A></P>','A1572',3405,'PUBLIC',3,28,'Signal calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1573,'Gm06990Rep1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq</U></A></P>','A1573',3405,'PUBLIC',3,28,'Signal calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1574,'Gm12878Rep1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq</U></A></P>','A1574',3405,'PUBLIC',3,28,'Signal calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1575,'Helas3Rep1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq</U></A></P>','A1575',3405,'PUBLIC',3,28,'Signal calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1576,'Hepg2Rep1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq</U></A></P>','A1576',3405,'PUBLIC',3,28,'Signal calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1577,'Hl60Rep1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq</U></A></P>','A1577',3405,'PUBLIC',3,28,'Signal calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1578,'HrcepicRep1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq</U></A></P>','A1578',3405,'PUBLIC',3,28,'Signal calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1579,'HreRep1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq</U></A></P>','A1579',3405,'PUBLIC',3,28,'Signal calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1580,'HuvecRep1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq</U></A></P>','A1580',3405,'PUBLIC',3,28,'Signal calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1581,'K562Rep1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq</U></A></P>','A1581',3405,'PUBLIC',3,28,'Signal calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1582,'SkmcRep1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq</U></A></P>','A1582',3405,'PUBLIC',3,28,'Signal calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1583,'SknshraRep1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq</U></A></P>','A1583',3405,'PUBLIC',3,28,'Signal calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1584,'Th1Rep1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq</U></A></P>','A1584',3405,'PUBLIC',3,28,'Signal calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1585,'Th2Rep1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq</U></A></P>','A1585',3405,'PUBLIC',3,28,'Signal calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1586,'BjtertRep2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq</U></A></P>','A1586',3405,'PUBLIC',3,28,'Signal calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1587,'Caco2Rep2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq</U></A></P>','A1587',3405,'PUBLIC',3,28,'Signal calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1588,'Gm06990Rep2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq</U></A></P>','A1588',3405,'PUBLIC',3,28,'Signal calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1589,'Gm12878Rep2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq</U></A></P>','A1589',3405,'PUBLIC',3,28,'Signal calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1590,'Helas3Rep2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq</U></A></P>','A1590',3405,'PUBLIC',3,28,'Signal calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1591,'Hepg2Rep2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq</U></A></P>','A1591',3405,'PUBLIC',3,28,'Signal calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1592,'HrcepicRep2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq</U></A></P>','A1592',3405,'PUBLIC',3,28,'Signal calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1593,'HreRep2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq</U></A></P>','A1593',3405,'PUBLIC',3,28,'Signal calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1594,'K562Rep2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq</U></A></P>','A1594',3405,'PUBLIC',3,28,'Signal calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1595,'SknshraRep2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq</U></A></P>','A1595',3405,'PUBLIC',3,28,'Signal calls from the ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI group',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1598,'Gm12878Control','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1598',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1599,'Gm12878Ctcf','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1599',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1600,'Gm12878H3k4me1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1600',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1601,'Gm12878H3k4me2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1601',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1602,'Gm12878H3k4me3','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1602',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1603,'Gm12878H3k9ac','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1603',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1604,'Gm12878H3k27ac','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1604',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1605,'Gm12878H3k27me3','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1605',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1606,'Gm12878H3k36me3','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1606',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1607,'Gm12878H4k20me1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1607',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1608,'H1hescControl','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1608',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1609,'H1hescCtcf','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1609',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1610,'H1hescH3k4me1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1610',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1611,'H1hescH3k4me3','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1611',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1612,'H1hescH3k27me3','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1612',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1613,'H1hescH4k20me1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1613',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1614,'Hepg2Control','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1614',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1615,'Hepg2Ctcf','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1615',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1616,'Hepg2H3k4me2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1616',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1617,'Hepg2H3k4me3','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1617',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1618,'Hepg2H3k9ac','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1618',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1619,'Hepg2H3k27ac','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1619',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1620,'Hepg2H3k36me3','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1620',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1621,'Hepg2H4k20me1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1621',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1622,'HmecControl','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1622',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1623,'HmecCtcf','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1623',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1624,'HmecH3k4me1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1624',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1625,'HmecH3k4me2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1625',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1626,'HmecH3k4me3','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1626',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1627,'HmecH3k9ac','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1627',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1628,'HmecH3k27ac','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1628',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1629,'HmecH3k27me3','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1629',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1630,'HmecH3k36me3','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1630',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1631,'HmecH4k20me1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1631',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1632,'HuvecControl','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1632',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1633,'HuvecCtcf','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1633',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1634,'HuvecH3k4me1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1634',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1635,'HuvecH3k4me2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1635',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1636,'HuvecH3k4me3','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1636',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1637,'HuvecH3k9ac','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1637',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1638,'HuvecH3k9me1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1638',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1639,'HuvecH3k27ac','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1639',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1640,'HuvecH3k27me3','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1640',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1641,'HuvecH3k36me3','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1641',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1642,'HuvecH4k20me1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1642',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1643,'HuvecPol2b','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1643',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1644,'K562Control','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1644',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1645,'K562Ctcf','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1645',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1646,'K562H3k4me1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1646',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1647,'K562H3k4me2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1647',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1648,'K562H3k4me3','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1648',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1649,'K562H3k9ac','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1649',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1650,'K562H3k9me1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1650',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1651,'K562H3k27ac','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1651',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1652,'K562H3k27me3','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1652',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1653,'K562H3k36me3','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1653',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1654,'K562H4k20me1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1654',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1655,'K562Pol2b','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1655',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1656,'NhekControl','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1656',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1657,'NhekCtcf','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1657',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1658,'NhekH3k4me1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1658',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1659,'NhekH3k4me2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1659',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1660,'NhekH3k4me3','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1660',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1661,'NhekH3k9ac','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1661',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1662,'NhekH3k9me1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1662',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1663,'NhekH3k27ac','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1663',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1664,'NhekH3k27me3','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1664',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1665,'NhekH3k36me3','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1665',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1666,'NhekH4k20me1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1666',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1667,'NhekPol2b','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1667',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1668,'NhlfControl','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1668',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1669,'NhlfH3k4me1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1669',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1670,'NhlfH3k4me2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1670',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1671,'NhlfH3k4me3','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1671',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1672,'NhlfH3k9ac','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1672',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1673,'NhlfH3k27ac','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1673',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1674,'NhlfH3k27me3','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1674',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1675,'NhlfH3k36me3','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1675',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1676,'NhlfH4k20me1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq</U></A></P>','A1676',3405,'PUBLIC',3,28,'ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1677,'Gm12878Cfos','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1677',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1678,'Gm12878Input','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1678',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1679,'Gm12878Jund','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1679',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1680,'Gm12878Max','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1680',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1681,'Gm12878NfkbTnfa','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1681',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1682,'Gm12878Pol2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1682',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1683,'Gm12878Pol3','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1683',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1684,'Hct116Input','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1684',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1685,'Hct116Pol2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1685',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1686,'Hct116Tcf4','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1686',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1687,'Hek293Pol2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1687',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1688,'Helas3Ap2alpha','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1688',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1689,'Helas3Ap2gamma','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1689',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1690,'Helas3Cfos','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1690',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1691,'Helas3Cmyc','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1691',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1692,'Helas3E2f4','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1692',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1693,'Helas3Ha','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1693',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1694,'Helas3ifngInput','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1694',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1695,'Helas3ifngStat1V2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1695',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1696,'Helas3Input','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1696',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1697,'Helas3LargefragmentV2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1697',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1698,'Helas3Max','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1698',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1699,'Helas3MnaseV2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1699',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1700,'Helas3MouseiggV2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1700',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1701,'Helas3NakeddnaV2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1701',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1702,'Helas3Pol2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1702',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1703,'Helas3Tr4','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1703',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1704,'Hepg2bInput','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1704',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1705,'Hepg2bTr4','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1705',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1706,'Hepg2Input','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1706',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1707,'Hepg2Pol2V2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1707',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1708,'Hepg2Srebp1aV2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1708',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1709,'Hepg2Srebp2V2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1709',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1710,'K562Bdp1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1710',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1711,'K562bE2f4','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1711',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1712,'K562bE2f6','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1712',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1713,'K562bGata1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1713',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1714,'K562bGata2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1714',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1715,'K562bInput','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1715',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1716,'K562Brf1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1716',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1717,'K562bSetdb1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1717',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1718,'K562bTr4','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1718',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1719,'K562bYy1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1719',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1720,'K562bZnf263','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1720',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1721,'K562Cfos','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1721',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1722,'K562Cjun','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1722',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1723,'K562Cmyc','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1723',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1724,'K562Ifna6hCjun','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1724',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1725,'K562Ifna6hCmyc','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1725',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1726,'K562Ifna6hInput','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1726',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1727,'K562Ifna6hPol2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1727',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1728,'K562Ifna6hStat1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1728',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1729,'K562Ifna6hStat2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1729',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1730,'K562Ifna30Cmyc','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1730',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1731,'K562Ifna30Input','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1731',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1732,'K562Ifna30Pol2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1732',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1733,'K562Ifna30Stat1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1733',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1734,'K562Ifna30Stat2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1734',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1735,'K562Ifng6hCjun','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1735',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1736,'K562Ifng6hCmyc','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1736',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1737,'K562Ifng6hInput','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1737',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1738,'K562Ifng6hPol2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1738',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1739,'K562Ifng30Cjun','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1739',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1740,'K562Ifng30Input','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1740',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1741,'K562InputV2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1741',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1742,'K562Jund','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1742',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1743,'K562Max','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1743',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1744,'K562Nfe2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1744',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1745,'K562Nfya','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1745',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1746,'K562Nfyb','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1746',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1747,'K562Pol2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1747',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1748,'K562Rad21','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1748',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1749,'K562Rpc155','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1749',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1750,'K562Sirt6','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1750',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1751,'K562Xrcc4','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1751',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1752,'Nb4Input','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1752',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1753,'Nb4Pol2','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1753',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1754,'Nt2d1Input','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1754',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1755,'Nt2d1Suz12','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1755',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1756,'Nt2d1Yy1','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq</U></A></P>','A1756',3405,'PUBLIC',3,28,'UCSC -  ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1758,'Gm12878Nov69522T7tag','<P ALIGN=""LEFT""><A HREF=""http://hgdownload.cse.ucsc.edu/goldenPath/hg18/encodeDCC/wgEncodeSunyalbanyRnaGeneChip/"" TARGET=""_blank""><U>http://hgdownload.cse.ucsc.edu/goldenPath/hg18/encodeDCC/wgEncodeSunyalbanyRnaGeneChip/</U></A></P>','A1758',3405,'PUBLIC',3,28,'ENCODE SUNY Albany RNA Binding Proteins by RIP-chip',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1759,'K562Sigp6246Pabpc1','<P ALIGN=""LEFT""><A HREF=""http://hgdownload.cse.ucsc.edu/goldenPath/hg18/encodeDCC/wgEncodeSunyalbanyRnaGeneChip/"" TARGET=""_blank""><U>http://hgdownload.cse.ucsc.edu/goldenPath/hg18/encodeDCC/wgEncodeSunyalbanyRnaGeneChip/</U></A></P>','A1759',3405,'PUBLIC',3,28,'ENCODE SUNY Albany RNA Binding Proteins by RIP-chip',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1760,'Gm12878Sc5261Elavl1','<P ALIGN=""LEFT""><A HREF=""http://hgdownload.cse.ucsc.edu/goldenPath/hg18/encodeDCC/wgEncodeSunyalbanyRnaGeneChip/"" TARGET=""_blank""><U>http://hgdownload.cse.ucsc.edu/goldenPath/hg18/encodeDCC/wgEncodeSunyalbanyRnaGeneChip/</U></A></P>','A1760',3405,'PUBLIC',3,28,'ENCODE SUNY Albany RNA Binding Proteins by RIP-chip',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1761,'Gm12878Sigp6246Pabpc1','<P ALIGN=""LEFT""><A HREF=""http://hgdownload.cse.ucsc.edu/goldenPath/hg18/encodeDCC/wgEncodeSunyalbanyRnaGeneChip/"" TARGET=""_blank""><U>http://hgdownload.cse.ucsc.edu/goldenPath/hg18/encodeDCC/wgEncodeSunyalbanyRnaGeneChip/</U></A></P>','A1761',3405,'PUBLIC',3,28,'ENCODE SUNY Albany RNA Binding Proteins by RIP-chip',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1762,'K562Nov69522T7tag','<P ALIGN=""LEFT""><A HREF=""http://hgdownload.cse.ucsc.edu/goldenPath/hg18/encodeDCC/wgEncodeSunyalbanyRnaGeneChip/"" TARGET=""_blank""><U>http://hgdownload.cse.ucsc.edu/goldenPath/hg18/encodeDCC/wgEncodeSunyalbanyRnaGeneChip/</U></A></P>','A1762',3405,'PUBLIC',3,28,'ENCODE SUNY Albany RNA Binding Proteins by RIP-chip',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1763,'K562Sc5261Elavl1','<P ALIGN=""LEFT""><A HREF=""http://hgdownload.cse.ucsc.edu/goldenPath/hg18/encodeDCC/wgEncodeSunyalbanyRnaGeneChip/"" TARGET=""_blank""><U>http://hgdownload.cse.ucsc.edu/goldenPath/hg18/encodeDCC/wgEncodeSunyalbanyRnaGeneChip/</U></A></P>','A1763',3405,'PUBLIC',3,28,'ENCODE SUNY Albany RNA Binding Proteins by RIP-chip',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1764,'K562Sc21027Igf2bp1','<P ALIGN=""LEFT""><A HREF=""http://hgdownload.cse.ucsc.edu/goldenPath/hg18/encodeDCC/wgEncodeSunyalbanyRnaGeneChip/"" TARGET=""_blank""><U>http://hgdownload.cse.ucsc.edu/goldenPath/hg18/encodeDCC/wgEncodeSunyalbanyRnaGeneChip/</U></A></P>','A1764',3405,'PUBLIC',3,28,'ENCODE SUNY Albany RNA Binding Proteins by RIP-chip',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1765,'delme','','A1765',3405,'PUBLIC',3,28,'','u0028003','2010-01-05 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1766,'Gm12878CytosolLongnonpolya-','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeRikenCage"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeRikenCage</U></A></P>','A1766',3405,'PUBLIC',3,28,'ENCODE RIKEN RNA Subcellular Localization by CAGE Tags',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1767,'ProstateCellLongnonpolya+','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeRikenCage"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeRikenCage</U></A></P>','A1767',3405,'PUBLIC',3,28,'ENCODE RIKEN RNA Subcellular Localization by CAGE Tags',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1768,'Gm12878CytosolLongnonpolya+','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeRikenCage"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeRikenCage</U></A></P>','A1768',3405,'PUBLIC',3,28,'ENCODE RIKEN RNA Subcellular Localization by CAGE Tags',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1769,'Gm12878NucleusLongnonpolya-','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeRikenCage"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeRikenCage</U></A></P>','A1769',3405,'PUBLIC',3,28,'ENCODE RIKEN RNA Subcellular Localization by CAGE Tags',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1770,'Gm12878NucleusLongnonpolya+','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeRikenCage"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeRikenCage</U></A></P>','A1770',3405,'PUBLIC',3,28,'ENCODE RIKEN RNA Subcellular Localization by CAGE Tags',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1771,'K562ChromatinTotal-','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeRikenCage"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeRikenCage</U></A></P>','A1771',3405,'PUBLIC',3,28,'ENCODE RIKEN RNA Subcellular Localization by CAGE Tags',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1772,'K562ChromatinTotal+','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeRikenCage"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeRikenCage</U></A></P>','A1772',3405,'PUBLIC',3,28,'ENCODE RIKEN RNA Subcellular Localization by CAGE Tags',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1773,'K562CytosolLongnonpolya-','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeRikenCage"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeRikenCage</U></A></P>','A1773',3405,'PUBLIC',3,28,'ENCODE RIKEN RNA Subcellular Localization by CAGE Tags',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1774,'K562CytosolLongnonpolya+','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeRikenCage"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeRikenCage</U></A></P>','A1774',3405,'PUBLIC',3,28,'ENCODE RIKEN RNA Subcellular Localization by CAGE Tags',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1775,'K562CytosolLongpolya-','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeRikenCage"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeRikenCage</U></A></P>','A1775',3405,'PUBLIC',3,28,'ENCODE RIKEN RNA Subcellular Localization by CAGE Tags',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1776,'K562CytosolLongpolya+','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeRikenCage"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeRikenCage</U></A></P>','A1776',3405,'PUBLIC',3,28,'ENCODE RIKEN RNA Subcellular Localization by CAGE Tags',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1777,'K562NucleolusTotal-','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeRikenCage"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeRikenCage</U></A></P>','A1777',3405,'PUBLIC',3,28,'ENCODE RIKEN RNA Subcellular Localization by CAGE Tags',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1778,'K562NucleolusTotal+','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeRikenCage"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeRikenCage</U></A></P>','A1778',3405,'PUBLIC',3,28,'ENCODE RIKEN RNA Subcellular Localization by CAGE Tags',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1779,'K562NucleoplasmTotal-','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeRikenCage"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeRikenCage</U></A></P>','A1779',3405,'PUBLIC',3,28,'ENCODE RIKEN RNA Subcellular Localization by CAGE Tags',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1780,'K562NucleoplasmTotal+','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeRikenCage"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeRikenCage</U></A></P>','A1780',3405,'PUBLIC',3,28,'ENCODE RIKEN RNA Subcellular Localization by CAGE Tags',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1781,'K562NucleusLongnonpolya-','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeRikenCage"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeRikenCage</U></A></P>','A1781',3405,'PUBLIC',3,28,'ENCODE RIKEN RNA Subcellular Localization by CAGE Tags',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1782,'K562NucleusLongnonpolya+','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeRikenCage"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeRikenCage</U></A></P>','A1782',3405,'PUBLIC',3,28,'ENCODE RIKEN RNA Subcellular Localization by CAGE Tags',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1783,'K562NucleusLongpolya-','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeRikenCage"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeRikenCage</U></A></P>','A1783',3405,'PUBLIC',3,28,'ENCODE RIKEN RNA Subcellular Localization by CAGE Tags',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1784,'K562NucleusLongpolya+','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeRikenCage"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeRikenCage</U></A></P>','A1784',3405,'PUBLIC',3,28,'ENCODE RIKEN RNA Subcellular Localization by CAGE Tags',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1785,'K562PolysomeLongnonpolya-','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeRikenCage"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeRikenCage</U></A></P>','A1785',3405,'PUBLIC',3,28,'ENCODE RIKEN RNA Subcellular Localization by CAGE Tags',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1786,'K562PolysomeLongnonpolya+','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeRikenCage"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeRikenCage</U></A></P>','A1786',3405,'PUBLIC',3,28,'ENCODE RIKEN RNA Subcellular Localization by CAGE Tags',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1787,'ProstateCellLongnonpolya-','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeRikenCage"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeRikenCage</U></A></P>','A1787',3405,'PUBLIC',3,28,'ENCODE RIKEN RNA Subcellular Localization by CAGE Tags',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1792,'Ln7Meg6903x2','<P ALIGN=""LEFT"">Converted to useq format with: java -jar -Xmx4G ~/USeqApps/Text2USeq -g H_sapiens_Feb_2009 -c 0 -b 1 -e 2 -t 3 -v 4 -s 5 -f Ln7Meg6903x2_NPP.bed.gz</P>','A1792',6300,'MEM',40,30,'Novoalign Paired Reads from Megakaryocytes',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1793,'Human Platelet Paired Reads','<P ALIGN=""LEFT"">Converted to useq format with: java -jar -Xmx4G ~/USeqApps/Text2USeq -g H_sapiens_Feb_2009 -c 0 -b 1 -e 2 -t 3 -v 4 -s 5 -f Ln8Plat6914x1_NPP.bed.gz</P>','A1793',6300,'PUBLIC',40,30,'Paired Reads from Human Platelets',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1799,'CapCapture','','A1799',3405,'MEM',40,30,'','tolley','2010-02-17 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1800,'HumanBaseline','','A1800',3405,'MEM',40,30,'','tolley','2010-02-17 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1801,'HumanThrombin\r','','A1801',3405,'MEM',40,30,'','tolley','2010-02-17 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1802,'MonoBaseline','','A1802',3405,'MEM',40,30,'','tolley','2010-02-17 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1803,'MonoClot','','A1803',3405,'MEM',40,30,'','tolley','2010-02-17 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1805,'MouseBaseline','','A1805',3806,'MEM',40,30,'','tolley','2010-02-17 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1806,'PltBaseline','','A1806',3405,'MEM',40,30,'','tolley','2010-02-17 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1807,'Megs14Day','','A1807',3405,'MEM',40,30,'','tolley','2010-02-17 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1808,'CBP Activated','<P ALIGN=""LEFT"">Jurkat</P>','A1808',3405,'MEM',42,13,'','u0028003','2010-02-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1809,'CBP Resting','','A1809',3405,'MEM',42,13,'','u0028003','2010-02-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1810,'CHD7','','A1810',3405,'MEM',42,13,'','u0028003','2010-02-22 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1811,'Input','','A1811',3405,'MEM',42,13,'','u0028003','2010-02-22 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1812,'CBP Activated','','A1812',3405,'MEM',42,13,'','u0028003','2010-02-22 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1813,'CBP Resting','','A1813',3405,'MEM',42,13,'','u0028003','2010-02-22 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1814,'CHD7','','A1814',3405,'MEM',42,13,'','u0028003','2010-02-22 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1815,'CBP Active Vs Resting','','A1815',3405,'MEM',42,13,'','u0028003','2010-02-22 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1816,'CBP Activated','','A1816',3405,'MEM',42,13,'','u0028003','2010-02-22 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1817,'CBP Resting','','A1817',3405,'MEM',42,13,'','u0028003','2010-02-22 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1818,'CBP Activated Vs Resting','','A1818',3405,'MEM',42,13,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1819,'CHD7','','A1819',3405,'MEM',42,13,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1820,'1st PEA3 Exp','','A1820',3806,'MEM',3,24,'Top 100 peaks from 1st experiment 6408x1 and 6415x1','u0028003','2010-02-24 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1821,'2st PEA3 Exp','','A1821',3806,'MEM',3,24,'Top 100 peaks from 2st experiment 7022x1','u0028003','2010-02-24 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1822,'1st PEA3 ChIP','','A1822',3806,'MEM',3,24,'','u0028003','2010-02-24 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1823,'Petes PEA3 Exp','<P ALIGN=""LEFT"">These thresholds generated 2900 peaks from Pete Hollenhorst&apos;s (Graves Lab) human PEA3 chIP 6652R.  Used the UCSC liftover tool to convert hg18 to mm9, lost 600 peaks.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">&quot;ETV4 IP from PC3 cells (experimental) and two lanes of PC3 input (control).&quot;</P>','A1823',3806,'MEM',3,24,'Thresholded peaks (1% FDR, 2x ratio) lifted over from human',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1824,'2nd PEA3 ChIP','','A1824',3806,'MEM',3,24,'','u0028003','2010-02-24 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1825,'2nd Input','','A1825',3806,'MEM',3,24,'','u0028003','2010-02-24 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1826,'1st Input','','A1826',3806,'MEM',3,24,'','u0028003','2010-02-24 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1827,'1st PEA3 Exp BinPVal','','A1827',3806,'MEM',3,24,'','u0028003','2010-02-24 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1828,'2st PEA3 Exp BinPVal','','A1828',3806,'MEM',3,24,'','u0028003','2010-02-24 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1829,'Combine PEA3 Exps BinPVal','','A1829',3806,'MEM',3,24,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1830,'Binomial PValue','','A1830',3405,'MEMCOL',3,13,'','u0028003','2010-02-24 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1831,'QValue FDR','','A1831',3405,'MEMCOL',3,13,'','u0028003','2010-02-24 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1832,'Empirical FDR','','A1832',3405,'MEMCOL',3,13,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1833,'PEA3 ChIP','','A1833',3405,'MEMCOL',3,13,'','u0028003','2010-02-24 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1834,'Input','','A1834',3405,'MEMCOL',3,13,'','u0028003','2010-02-24 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1835,'1% FDR, 2x','','A1835',3405,'MEMCOL',3,13,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1837,'Human Platelets Column','','A1837',6300,'PUBLIC',40,30,'Poly A isolated RNA from male human platelets.','tolley','2010-03-01 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1838,'PltThrombinJR','','A1838',6300,'MEM',40,30,'','tolley','2010-03-01 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1839,'Plt5PrimeCaptureNT','','A1839',6300,'MEM',40,30,'','tolley','2010-03-01 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1840,'MonoBaselineRC','','A1840',6300,'MEM',40,30,'','tolley','2010-03-01 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1841,'MonoClotRC','','A1841',6300,'MEM',40,30,'','tolley','2010-03-01 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1842,'Mouse Platelets Column','','A1842',3806,'PUBLIC',40,30,'Spin column isolated poly A selected RNA from female c57bl/6 mice (pooled platelets)','tolley','2010-03-01 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1844,'Primate DNA Conservation','<P ALIGN=""LEFT""><B>Conservation (cons44way) Track Description</B></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">This track shows multiple alignments of 44 vertebrate species and measurements of evolutionary conservation using two methods (phastCons and phyloP) from the PHAST package, for all species (vertebrate) and two subsets (primate and placental mammal). The multiple alignments were generated using multiz and other tools in the UCSC/Penn State Bioinformatics comparative genomics alignment pipeline. Conserved elements identified by phastCons are also displayed in this track.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">PhastCons (which has been used in previous Conservation tracks) is a hidden Markov model-based method that estimates the probability that each nucleotide belongs to a conserved element, based on the multiple alignment. It considers not just each individual alignment column, but also its flanking columns. By contrast, phyloP separately measures conservation at individual columns, ignoring the effects of their neighbors. As a consequence, the phyloP plots have a less smooth appearance than the phastCons plots, with more &quot;texture&quot; at individual sites. The two methods have different strengths and weaknesses. PhastCons is sensitive to &quot;runs&quot; of conserved sites, and is therefore effective for picking out conserved elements. PhyloP, on the other hand, is more appropriate for evaluating signatures of selection at particular nucleotides or classes of nucleotides (e.g., third codon positions, or first positions of miRNA target sites).</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Another important difference is that phyloP can measure acceleration (faster evolution than expected under neutral drift) as well as conservation (slower than expected evolution). In the phyloP plots, sites predicted to be conserved are assigned positive scores (and shown in blue), while sites predicted to be fast-evolving are assigned negative scores (and shown in red). The absolute values of the scores represent -log p-values under a null hypothesis of neutral evolution. The phastCons scores, by contrast, represent probabilities of negative selection and range between 0 and 1.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Both phastCons and phyloP treat alignment gaps and unaligned nucleotides as missing data, and both were run with the same parameters for each species set (vertebrates, placental mammals, and primates). Thus, in regions in which only primates appear in the alignment, all three sets of scores will be the same, but in regions in which additional species are available, the mammalian and/or vertebrate scores may differ from the primate scores. The alternative plots help to identify sequences that are under different evolutionary pressures in, say, primates and non-primates, or mammals and non-mammals.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">The species aligned for this track include the reptile, amphibian, bird, and fish clades, as well as marsupial, monotreme (platypus), and placental mammals. Compared to the previous 28-vertebrate alignment, this track includes 16 new species and 8 species with updated sequence assemblies (Table 1). The new species consist of two high-coverage (5-8.5X) assemblies (orangutan, zebra finch), low-coverage draft assemblies of gorilla, marmoset, tarsier, mouse lemur, kangaroo rat, squirrel, pika, megabat, microbat, dolphin, alpaca, sloth, rock hyrax and lamprey. The mouse, cow, guinea pig, horse, elephant, zebrafish, and medaka assemblies have been updated from those used in the previous 28-species alignment.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">UCSC has repeatmasked and aligned the low-coverage genome assemblies, and provides the sequence for download; however, we do not construct genome browsers for them. Missing sequence in the low-coverage assemblies is highlighted in the track display by regions of yellow when zoomed out and Ns displayed at base level (see Gap Annotation, below).</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Downloaded from <A HREF=""http://hgdownload.cse.ucsc.edu/goldenPath/hg18/phastCons44way/primates/"" TARGET=""_blank""><B><U>http://hgdownload.cse.ucsc.edu/goldenPath/hg18/phastCons44way/primates/</U></B></A></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Credits</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">This track was created using the following programs:</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Alignment tools: blastz and multiz by Minmei Hou, Scott Schwartz and Webb Miller of the Penn State Bioinformatics Group</P><P ALIGN=""LEFT"">Chaining and Netting: axtChain, chainNet by Jim Kent at UCSC</P><P ALIGN=""LEFT"">Conservation scoring: phastCons, phyloP, phyloFit, tree_doctor, msa_view and other programs in PHAST by Adam Siepel at Cornell University (original development done at the Haussler lab at UCSC).</P><P ALIGN=""LEFT"">MAF Annotation tools: mafAddIRows by Brian Raney, UCSC; mafAddQRows by Richard Burhans, Penn State; genePredToMafFrames by Mark Diekhans, UCSC</P><P ALIGN=""LEFT"">Tree image generator: phyloGif by Galt Barber, UCSC</P><P ALIGN=""LEFT"">Conservation track display: Kate Rosenbloom, Hiram Clawson (wiggle display), and Brian Raney (gap annotation and codon framing) at UCSC</P><P ALIGN=""LEFT"">The phylogenetic tree is based on Murphy et al. (2001) and general consensus in the vertebrate phylogeny community as of March 2007.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">References</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Phylo-HMMs, phastCons, and phyloP:</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Pollard KS, Hubisz MJ, Siepel A. Detection of non-neutral substitution rates on mammalian phylogenies. Genome Res. 2009 Oct 26. [Epub ahead of print]</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Siepel A, Bejerano G, Pedersen JS, Hinrichs AS, Hou M, Rosenbloom K, Clawson H, Spieth J, Hillier LW, Richards S, et al. Evolutionarily conserved elements in vertebrate, insect, worm, and yeast genomes. Genome Res. 2005 Aug;15(8):1034-50.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Chain/Net:</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Kent WJ, Baertsch R, Hinrichs A, Miller W, Haussler D. Evolution&apos;s cauldron: duplication, deletion, and rearrangement in the mouse and human genomes. Proc Natl Acad Sci U S A. 2003 Sep 30;100(20):11484-9.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Multiz:</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Blanchette M, Kent WJ, Riemer C, Elnitski L, Smit AF, Roskin KM, Baertsch R, Rosenbloom K, Clawson H, Green ED, et al. Aligning multiple genomic sequences with the threaded blockset aligner. Genome Res. 2004 Apr;14(4):708-15.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Blastz:</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Chiaromonte F, Yap VB, Miller W. Scoring pairwise genomic sequence alignments. Pac Symp Biocomput. 2002;:115-26.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Schwartz S, Kent WJ, Smit A, Zhang Z, Baertsch R, Hardison RC, Haussler D, Miller W. Human-mouse alignments with BLASTZ. Genome Res. 2003 Jan;13(1):103-7.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Phylogenetic Tree:</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Murphy WJ, Eizirik E, O&apos;Brien SJ, Madsen O, Scally M, Douady CJ, Teeling E, Ryder OA, Stanhope MJ, de Jong WW, Springer MS. Resolution of the early placental mammal radiation using Bayesian phylogenetics. Science. 2001 Dec 14;294(5550):2348-51</P>','A1844',3405,'PUBLIC',NULL,28,'Phast Con 44 Way Primates from UCSC',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1850,'ChIP-array_Bound','<P ALIGN=""LEFT"">Pol3 bound regions greater than 2-fold plus probes within 500bp enriched less than but approx 2-fold.  I put the score (fold*100) after the probe name (connected with a comma).</P>','A1850',3405,'PUBLIC',43,7,'','Andrew','2010-03-02 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1852,'refseq','<P ALIGN=""LEFT"">This contains refseq gene annotation downloaded from UCSC and converted to brs format.  The scaffold and na annotations have been shifted to match a composite concatinated chrScaffold or chrNA artificial chromosomes.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">See <A HREF=""https://hci-as1.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A88&isGuest=Y"" TARGET=""_blank""><B><U>https://hci-as1.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A88&amp;isGuest=Y</U></B></A></P>','A1852',6301,'PUBLIC',3,28,'RefSeq gene annotation from UCSC','u0028003','2010-03-03 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1853,'ensembl','<P ALIGN=""LEFT"">This contains Ensembl gene annotation downloaded from UCSC and converted to brs format.  The scaffold and na annotations have been shifted to match a composite concatinated chrScaffold or chrNA artificial chromosomes.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">See <A HREF=""https://hci-as1.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A88&isGuest=Y"" TARGET=""_blank""><B><U>https://hci-as1.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A88&amp;isGuest=Y</U></B></A></P>','A1853',6301,'PUBLIC',3,28,'Ensembl transcript annotation from UCSC',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1854,'chrNA contigs','','A1854',6301,'PUBLIC',3,28,'Location of the NA contings in the chrNA artificial chromosome','u0028003','2010-03-03 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1855,'chrScaffold contigs','','A1855',6301,'PUBLIC',3,28,'Location of the scaffold contings in the chrScaffold artificial chromosome','u0028003','2010-03-03 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1856,'chrScaffold spacers','','A1856',6301,'PUBLIC',3,28,'Location of the ''N'' spacers between scaffolds in the chrScaffold artificial chromosome','u0028003','2010-03-03 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1857,'chrN spacers','','A1857',6301,'PUBLIC',3,28,'Location of the ''N'' spacers between NAs in the chrNA artificial chromosome',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1859,'CCDS Target Regions','<P ALIGN=""LEFT"">See <A HREF=""https://hci-as1.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A67&isGuest=Y"" TARGET=""_blank""><B><U>https://hci-as1.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A67&amp;isGuest=Y</U></B></A></P>','A1859',6300,'PUBLIC',3,8,'','u0028003','2010-03-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1860,'Oligos','<P ALIGN=""LEFT"">See <A HREF=""https://hci-as1.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A67&isGuest=Y"" TARGET=""_blank""><B><U>https://hci-as1.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A67&amp;isGuest=Y</U></B></A></P>','A1860',6300,'PUBLIC',3,8,'','u0028003','2010-03-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1861,'OMin','','A1861',3405,'MEM',3,16,'','u0028003','2010-03-05 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1862,'5Min','','A1862',3405,'MEM',3,16,'','u0028003','2010-03-05 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1863,'10Min','','A1863',3405,'MEM',3,16,'','u0028003','2010-03-05 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1864,'15Min','','A1864',3405,'MEM',3,16,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1865,'5'' 3'' 200bp Ends','<P ALIGN=""LEFT""></P>','A1865',3405,'MEM',3,16,'Abundantly expressed genes with > 25 read counts on each 5'' and 3'' 200bp ends','u0028003','2010-03-05 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1866,'5 Vs 0Min QValFDR','','A1866',3405,'MEM',3,16,'200 bp window summary track showing -10Log10(FDR)','u0028003','2010-03-05 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1867,'10 Vs 0Min QValFDR','','A1867',3405,'MEM',3,16,'200 bp window summary track showing -10Log10(FDR)','u0028003','2010-03-05 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1868,'15 Vs 0Min QValFDR','','A1868',3405,'MEM',3,16,'200 bp window summary track showing -10Log10(FDR)',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1869,'7178X1 KVexome A','','A1869',6300,'MEM',3,15,'','u0028003','2010-03-10 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1870,'7178X2 KVexome B','','A1870',6300,'MEM',3,15,'','u0028003','2010-03-10 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1871,'7178X3 YOexome A','','A1871',6300,'MEM',3,15,'','u0028003','2010-03-10 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1872,'7178X4 YOexome B','','A1872',6300,'MEM',3,15,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1873,'g1p6 coverage','','A1873',6300,'MEM',38,31,'','bmilash','2010-03-24 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1874,'g1p16 coverage','','A1874',6300,'MEM',38,31,'','bmilash','2010-03-24 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1875,'g2p5 coverage','','A1875',6300,'MEM',38,31,'','bmilash','2010-03-24 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1876,'g2p35 coverage','','A1876',6300,'MEM',38,31,'','bmilash','2010-03-24 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1877,'g3p5 coverage','','A1877',6300,'MEM',38,31,'','bmilash','2010-03-24 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1878,'g3p26 coverage','','A1878',6300,'MEM',38,31,'','bmilash','2010-03-24 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1879,'ILUC','<P ALIGN=""LEFT"">## Sub sample to match sample x7</P><P ALIGN=""LEFT"">java -jar -Xmx2G /Applications/BioApps/USeq/Apps/SubSamplePointData -n 17307070 -f /Users/davidnix/Desktop/Lessnick/7190R/PointData/6PointData -s /Users/davidnix/Desktop/Lessnick/7190R/PointData/6PointData_Sub</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">## ReadCoverage for visualization</P><P ALIGN=""LEFT"">java -jar -Xmx2G /Applications/BioApps/USeq/Apps/ReadCoverage -p 3PointData_Sub/ -s ReadCoverage/3</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Unstranded</P>','A1879',6300,'MEM',3,10,'Displays the number of reads overlapping each base','u0028003','2010-03-29 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1880,'IGLI5','<P ALIGN=""LEFT"">## Sub sample to match sample x7</P><P ALIGN=""LEFT"">java -jar -Xmx2G /Applications/BioApps/USeq/Apps/SubSamplePointData -n 17307070 -f /Users/davidnix/Desktop/Lessnick/7190R/PointData/6PointData -s /Users/davidnix/Desktop/Lessnick/7190R/PointData/6PointData_Sub</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">## ReadCoverage for visualization</P><P ALIGN=""LEFT"">java -jar -Xmx2G /Applications/BioApps/USeq/Apps/ReadCoverage -p 3PointData_Sub/ -s ReadCoverage/3</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Unstranded</P>','A1880',6300,'MEM',3,10,'Displays the number of reads overlapping each base','u0028003','2010-03-29 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1881,'IGLI6','<P ALIGN=""LEFT"">## Sub sample to match sample x7</P><P ALIGN=""LEFT"">java -jar -Xmx2G /Applications/BioApps/USeq/Apps/SubSamplePointData -n 17307070 -f /Users/davidnix/Desktop/Lessnick/7190R/PointData/6PointData -s /Users/davidnix/Desktop/Lessnick/7190R/PointData/6PointData_Sub</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">## ReadCoverage for visualization</P><P ALIGN=""LEFT"">java -jar -Xmx2G /Applications/BioApps/USeq/Apps/ReadCoverage -p 3PointData_Sub/ -s ReadCoverage/3</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Unstranded</P>','A1881',6300,'MEM',3,10,'Displays the number of reads overlapping each base','u0028003','2010-03-29 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1882,'ILUC Empty Vector','<P ALIGN=""LEFT"">## Sub sample to match sample x7</P><P ALIGN=""LEFT"">java -jar -Xmx2G /Applications/BioApps/USeq/Apps/SubSamplePointData -n 17307070 -f /Users/davidnix/Desktop/Lessnick/7190R/PointData/6PointData -s /Users/davidnix/Desktop/Lessnick/7190R/PointData/6PointData_Sub</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">## ReadCoverage for visualization</P><P ALIGN=""LEFT"">java -jar -Xmx2G /Applications/BioApps/USeq/Apps/ReadCoverage -p 3PointData_Sub/ -s ReadCoverage/3</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Unstranded</P>','A1882',6300,'MEM',3,10,'Displays the number of reads overlapping each base','u0028003','2010-03-29 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1883,'IGLI5 Empty Vector','<P ALIGN=""LEFT"">## Sub sample to match sample x7</P><P ALIGN=""LEFT"">java -jar -Xmx2G /Applications/BioApps/USeq/Apps/SubSamplePointData -n 17307070 -f /Users/davidnix/Desktop/Lessnick/7190R/PointData/6PointData -s /Users/davidnix/Desktop/Lessnick/7190R/PointData/6PointData_Sub</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">## ReadCoverage for visualization</P><P ALIGN=""LEFT"">java -jar -Xmx2G /Applications/BioApps/USeq/Apps/ReadCoverage -p 3PointData_Sub/ -s ReadCoverage/3</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Unstranded</P>','A1883',6300,'MEM',3,10,'Displays the number of reads overlapping each base','u0028003','2010-03-29 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1884,'IGLI5 GLi1 cDNA','<P ALIGN=""LEFT"">## Sub sample to match sample x7</P><P ALIGN=""LEFT"">java -jar -Xmx2G /Applications/BioApps/USeq/Apps/SubSamplePointData -n 17307070 -f /Users/davidnix/Desktop/Lessnick/7190R/PointData/6PointData -s /Users/davidnix/Desktop/Lessnick/7190R/PointData/6PointData_Sub</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">## ReadCoverage for visualization</P><P ALIGN=""LEFT"">java -jar -Xmx2G /Applications/BioApps/USeq/Apps/ReadCoverage -p 3PointData_Sub/ -s ReadCoverage/3</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Unstranded</P>','A1884',6300,'MEM',3,10,'Displays the number of reads overlapping each base','u0028003','2010-03-29 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1885,'IGLI6 Empty Vector','<P ALIGN=""LEFT"">## Sub sample to match sample x7</P><P ALIGN=""LEFT"">java -jar -Xmx2G /Applications/BioApps/USeq/Apps/SubSamplePointData -n 17307070 -f /Users/davidnix/Desktop/Lessnick/7190R/PointData/6PointData -s /Users/davidnix/Desktop/Lessnick/7190R/PointData/6PointData_Sub</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">## ReadCoverage for visualization</P><P ALIGN=""LEFT"">java -jar -Xmx2G /Applications/BioApps/USeq/Apps/ReadCoverage -p 3PointData_Sub/ -s ReadCoverage/3</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Unstranded</P>','A1885',6300,'MEM',3,10,'Displays the number of reads overlapping each base',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1886,'NKX2.2 ChIP','<P ALIGN=""LEFT"">Data was duplicate and satellite repeat filtered.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">java -jar -Xmx4G /Applications/BioApps/USeq/Apps/FilterDuplicateAlignments -f /Users/davidnix/Desktop/Lessnick/7180R/GAII/100312_7180X1_s_7_sorted.txt.gz -r /Users/davidnix/Desktop/Lessnick/7180R/DupFiltered -c 10 -p 12 -s 8 -t 13 -b 9</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">java -jar -Xmx4G /Applications/BioApps/USeq/Apps/ElandParser -v H_sapiens_Feb_2009 -f /Users/davidnix/Desktop/Lessnick/7180R/DupFiltered/ChIP_NKX2.2/filteredAlignments.txt</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">java -jar -Xmx4G /Applications/BioApps/USeq/Apps/FilterPointData -p /Users/davidnix/Desktop/Lessnick/7180R/DupFiltered/ChIP_NKX2.2/PointData0bp -r /Users/davidnix/Desktop/Lessnick/hg19SatelliteRepeats.txt.gz</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">java -jar -Xmx4G /Applications/BioApps/USeq/Apps/ReadCoverage -p /Users/davidnix/Desktop/Lessnick/7180R/DupFiltered/ChIP_NKX2.2/PointData0bp_hg19SatelliteRepeats.txt_Filt0bp -s /Users/davidnix/Desktop/Lessnick/7180R/DupFiltered/ChIP_NKX2.2/ReadCoverage</P>','A1886',6300,'MEM',3,10,'Displays the number of reads overlapping each base','u0028003','2010-03-29 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1887,'Input','<P ALIGN=""LEFT"">Data was duplicate and satellite repeat filtered and subsampled to match the chIP sample</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">java -jar -Xmx4G /Applications/BioApps/USeq/Apps/FilterDuplicateAlignments -f /Users/davidnix/Desktop/Lessnick/7180R/GAII/100312_7180X2_s_8_sorted.txt.gz  -r /Users/davidnix/Desktop/Lessnick/7180R/DupFiltered/Input/ -c 10 -p 12 -s 8 -t 13 -b 9</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">java -jar -Xmx4G /Applications/BioApps/USeq/Apps/ElandParser -v H_sapiens_Feb_2009 -f /Users/davidnix/Desktop/Lessnick/7180R/DupFiltered/Input/filteredAlignments.txt</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">java -jar -Xmx4G /Applications/BioApps/USeq/Apps/FilterPointData -p /Users/davidnix/Desktop/Lessnick/7180R/DupFiltered/Input/PointData0bp -r /Users/davidnix/Desktop/Lessnick/hg19SatelliteRepeats.txt.gz</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">java -jar -Xmx4G /Applications/BioApps/USeq/Apps/SubSamplePointData -f /Users/davidnix/Desktop/Lessnick/7180R/DupFiltered/Input/PointData0bp_hg19SatelliteRepeats.txt_Filt0bp -n 16742701 -s /Users/davidnix/Desktop/Lessnick/7180R/DupFiltered/Input/PointData_SubSampledForVisONLY</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">java -jar -Xmx4G /Applications/BioApps/USeq/Apps/ReadCoverage -p /Users/davidnix/Desktop/Lessnick/7180R/DupFiltered/Input/PointData_SubSampledForVisONLY -s /Users/davidnix/Desktop/Lessnick/7180R/DupFiltered/Input/ReadCoverage</P>','A1887',6300,'MEM',3,10,'Displays the number of reads overlapping each base',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1888,'Binomial PValue','','A1888',6300,'MEM',3,10,'300bp window level enrichment score based on the -10Log10(p-val)','u0028003','2010-03-29 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1889,'QValue FDR','','A1889',6300,'MEM',3,10,'300bp window level enrichment score based on the -10Log10(qvalue FDR)','u0028003','2010-03-29 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1890,'Empirical FDR','','A1890',6300,'MEM',3,10,'300bp window level enrichment score based on the -10Log10(emp FDR)',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1891,'Top 100 Peaks','','A1891',6300,'MEM',3,10,'','u0028003','2010-03-29 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1893,'HuvecBL_7108X1','','A1893',6300,'MEM',40,30,'','tolley','2010-04-07 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1894,'QValFDR','','A1894',3405,'PUBLIC',43,7,'','Andrew','2010-04-07 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1895,'QValFDR','','A1895',3405,'PUBLIC',43,7,'','Andrew','2010-04-07 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1896,'QValFDR','','A1896',3405,'PUBLIC',43,7,'','Andrew','2010-04-07 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1897,'QValFDR','','A1897',3405,'PUBLIC',43,7,'','Andrew','2010-04-08 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1898,'EmpFDR','','A1898',3405,'PUBLIC',43,7,'','Andrew','2010-04-07 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1899,'EmpFDR','','A1899',3405,'PUBLIC',43,7,'','Andrew','2010-04-07 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1900,'EmpFDR','','A1900',3405,'PUBLIC',43,7,'','Andrew','2010-04-07 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1901,'EmpFDR','','A1901',3405,'PUBLIC',43,7,'','Andrew','2010-04-08 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1902,'BinPVal','','A1902',3405,'PUBLIC',43,7,'','Andrew','2010-04-07 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1903,'BinPVal','','A1903',3405,'PUBLIC',43,7,'','Andrew','2010-04-07 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1904,'BinPVal','','A1904',3405,'PUBLIC',43,7,'','Andrew','2010-04-07 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1905,'BinPVal','','A1905',3405,'PUBLIC',43,7,'','Andrew','2010-04-08 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1921,'BinPVal','','A1921',3405,'PUBLIC',43,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1922,'BinPVal','','A1922',3405,'PUBLIC',43,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1923,'BinPVal','','A1923',3405,'PUBLIC',43,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1924,'QValFDR','','A1924',3405,'PUBLIC',43,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1925,'EmpFDR','','A1925',3405,'PUBLIC',43,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1926,'EmpFDR','','A1926',3405,'PUBLIC',43,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1927,'QValFDR','','A1927',3405,'PUBLIC',43,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1928,'EmpFDR','','A1928',3405,'PUBLIC',43,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1929,'QValFDR','','A1929',3405,'PUBLIC',43,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1930,'Sum_window25bp','','A1930',3405,'PUBLIC',43,7,'','Andrew','2010-04-08 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1931,'Sum_window25bp','','A1931',3405,'PUBLIC',43,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1932,'HuvecPlt_7108X2','','A1932',6300,'MEM',40,30,'','tolley','2010-04-08 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1933,'Peaks_FDR_0.10','','A1933',3405,'PUBLIC',43,7,'','Andrew','2010-04-08 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1934,'Peaks_FDR_0.01','','A1934',3405,'PUBLIC',43,7,'','Andrew','2010-04-08 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1935,'Peaks_FDR_0.01','','A1935',3405,'PUBLIC',43,7,'','Andrew','2010-04-08 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1936,'Peaks_FDR_0.10','','A1936',3405,'PUBLIC',43,7,'','Andrew','2010-04-08 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1937,'Top400','','A1937',3405,'PUBLIC',43,7,'','Andrew','2010-04-08 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1938,'Peaks_FDR_0.01','','A1938',3405,'PUBLIC',43,7,'','Andrew','2010-04-08 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1939,'Peaks_FDR_0.05','','A1939',3405,'PUBLIC',43,7,'','Andrew','2010-04-08 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1940,'Top400','','A1940',3405,'PUBLIC',43,7,'','Andrew','2010-04-08 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1941,'Peaks_FDR_0.01','','A1941',3405,'PUBLIC',43,7,'','Andrew','2010-04-08 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1942,'Peaks_FDR_0.05','','A1942',3405,'PUBLIC',43,7,'','Andrew','2010-04-08 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1943,'Top400','','A1943',3405,'PUBLIC',43,7,'','Andrew','2010-04-08 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1944,'Peaks_FDR_0.01','','A1944',3405,'PUBLIC',43,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1945,'Peaks_FDR_0.10','','A1945',3405,'PUBLIC',43,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1946,'Top400','','A1946',3405,'PUBLIC',43,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1947,'Peaks_FDR_0.01','','A1947',3405,'PUBLIC',43,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1948,'Peaks_FDR_0.10','','A1948',3405,'PUBLIC',43,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1949,'Peaks_FDR_0.01','','A1949',3405,'PUBLIC',43,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1950,'Peaks_FDR_0.10','','A1950',3405,'PUBLIC',43,7,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1954,'HuvecPltAD_7108X3','','A1954',6300,'MEM',40,30,'','tolley','2010-04-08 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1958,'HuvecPltClot_7108X4','','A1958',6300,'MEM',40,30,'','tolley','2010-04-08 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1959,'HuvecPltADClot_7108X5','','A1959',6300,'MEM',40,30,'','tolley','2010-04-08 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1960,'MonoClotRXRC','','A1960',6300,'MEM',40,30,'','tolley','2010-04-08 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1961,'RefSeq Genes','','A1961',3405,'PUBLIC',3,28,'RefSeq annotation downloaded from UCSC','u0028003','2010-04-13 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1962,'Ensembl Gene Transcripts','','A1962',3405,'PUBLIC',NULL,28,'Ensembl transcripts downloaded from UCSC','u0028003','2010-04-13 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1963,'UCSC Known Genes','','A1963',3405,'PUBLIC',3,28,'UCSC''s assembly of all known genes','u0028003','2010-04-13 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1966,'Array1_29957.useq','All SNPs from Maq','A1966',3405,'MEM',3,8,'All SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1967,'Array1_77580.useq','All SNPs from Maq','A1967',3405,'MEM',3,8,'All SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1968,'Array2_45817.useq','All SNPs from Maq','A1968',3405,'MEM',3,8,'All SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1969,'Array2_76204.useq','All SNPs from Maq','A1969',3405,'MEM',3,8,'All SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1970,'Array1_30085.useq','All SNPs from Maq','A1970',3405,'MEM',3,8,'All SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1971,'Array1_77935.useq','All SNPs from Maq','A1971',3405,'MEM',3,8,'All SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1972,'Array2_59473.useq','All SNPs from Maq','A1972',3405,'MEM',3,8,'All SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1973,'Array2_78114.useq','All SNPs from Maq','A1973',3405,'MEM',3,8,'All SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1974,'Array1_45817.useq','All SNPs from Maq','A1974',3405,'MEM',3,8,'All SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1975,'Array1_77937.useq','All SNPs from Maq','A1975',3405,'MEM',3,8,'All SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1976,'Array2_59475.useq','All SNPs from Maq','A1976',3405,'MEM',3,8,'All SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1977,'Array2_79406.useq','All SNPs from Maq','A1977',3405,'MEM',3,8,'All SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1978,'Array1_46544.useq','All SNPs from Maq','A1978',3405,'MEM',3,8,'All SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1979,'Array1_78117.useq','All SNPs from Maq','A1979',3405,'MEM',3,8,'All SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1980,'Array2_59489.useq','All SNPs from Maq','A1980',3405,'MEM',3,8,'All SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1981,'Array2_79407.useq','All SNPs from Maq','A1981',3405,'MEM',3,8,'All SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1982,'Array1_64541.useq','All SNPs from Maq','A1982',3405,'MEM',3,8,'All SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1983,'Array1_78118.useq','All SNPs from Maq','A1983',3405,'MEM',3,8,'All SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1984,'Array2_59491.useq','All SNPs from Maq','A1984',3405,'MEM',3,8,'All SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1985,'Array2_80388.useq','All SNPs from Maq','A1985',3405,'MEM',3,8,'All SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1986,'Array1_66395.useq','All SNPs from Maq','A1986',3405,'MEM',3,8,'All SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1987,'Array1_78174.useq','All SNPs from Maq','A1987',3405,'MEM',3,8,'All SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1988,'Array2_67594.useq','All SNPs from Maq','A1988',3405,'MEM',3,8,'All SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1989,'Array1_73197.useq','All SNPs from Maq','A1989',3405,'MEM',3,8,'All SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1990,'Array2_41968Combine.useq','All SNPs from Maq','A1990',3405,'MEM',3,8,'All SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1991,'Array2_71430.useq','All SNPs from Maq','A1991',3405,'MEM',3,8,'All SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1992,'Array1_76591.useq','All SNPs from Maq','A1992',3405,'MEM',3,8,'All SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1993,'Array2_41968.useq','All SNPs from Maq','A1993',3405,'MEM',3,8,'All SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1994,'Array2_73197.useq','All SNPs from Maq','A1994',3405,'MEM',3,8,'All SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1995,'Array1_29957.useq','Unique duplicate read SNPs from Maq','A1995',3405,'MEM',3,8,'Unique duplicate read SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1996,'Array1_30085.useq','Unique duplicate read SNPs from Maq','A1996',3405,'MEM',3,8,'Unique duplicate read SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1997,'Array1_45817.useq','Unique duplicate read SNPs from Maq','A1997',3405,'MEM',3,8,'Unique duplicate read SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1998,'Array1_46544.useq','Unique duplicate read SNPs from Maq','A1998',3405,'MEM',3,8,'Unique duplicate read SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(1999,'Array1_64541.useq','Unique duplicate read SNPs from Maq','A1999',3405,'MEM',3,8,'Unique duplicate read SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/');

INSERT INTO dbo.gpAnnotation 
(
idAnnotation,
name,
description,
fileName,
idGenomeVersion,
codeVisibility,
idUser,
idUserGroup,
summary,
createdBy,
createDate,
isLoaded,
idInstitute,
dataPath
)
VALUES 
(2000,'Array1_66395.useq','Unique duplicate read SNPs from Maq','A2000',3405,'MEM',3,8,'Unique duplicate read SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2001,'Array1_73197.useq','Unique duplicate read SNPs from Maq','A2001',3405,'MEM',3,8,'Unique duplicate read SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2002,'Array1_76591.useq','Unique duplicate read SNPs from Maq','A2002',3405,'MEM',3,8,'Unique duplicate read SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2003,'Array1_77580.useq','Unique duplicate read SNPs from Maq','A2003',3405,'MEM',3,8,'Unique duplicate read SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2004,'Array1_77935.useq','Unique duplicate read SNPs from Maq','A2004',3405,'MEM',3,8,'Unique duplicate read SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2005,'Array1_77937.useq','Unique duplicate read SNPs from Maq','A2005',3405,'MEM',3,8,'Unique duplicate read SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2006,'Array1_78117.useq','Unique duplicate read SNPs from Maq','A2006',3405,'MEM',3,8,'Unique duplicate read SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2007,'Array1_78118.useq','Unique duplicate read SNPs from Maq','A2007',3405,'MEM',3,8,'Unique duplicate read SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2008,'Array1_78174.useq','Unique duplicate read SNPs from Maq','A2008',3405,'MEM',3,8,'Unique duplicate read SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2009,'Array2_41968.useq','Unique duplicate read SNPs from Maq','A2009',3405,'MEM',3,8,'Unique duplicate read SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2010,'Array2_41968Combine.useq','Unique duplicate read SNPs from Maq','A2010',3405,'MEM',3,8,'Unique duplicate read SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2011,'Array2_45817.useq','Unique duplicate read SNPs from Maq','A2011',3405,'MEM',3,8,'Unique duplicate read SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2012,'Array2_59473.useq','Unique duplicate read SNPs from Maq','A2012',3405,'MEM',3,8,'Unique duplicate read SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2013,'Array2_59475.useq','Unique duplicate read SNPs from Maq','A2013',3405,'MEM',3,8,'Unique duplicate read SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2014,'Array2_59489.useq','Unique duplicate read SNPs from Maq','A2014',3405,'MEM',3,8,'Unique duplicate read SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2015,'Array2_59491.useq','Unique duplicate read SNPs from Maq','A2015',3405,'MEM',3,8,'Unique duplicate read SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2016,'Array2_67594.useq','Unique duplicate read SNPs from Maq','A2016',3405,'MEM',3,8,'Unique duplicate read SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2017,'Array2_71430.useq','Unique duplicate read SNPs from Maq','A2017',3405,'MEM',3,8,'Unique duplicate read SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2018,'Array2_73197.useq','Unique duplicate read SNPs from Maq','A2018',3405,'MEM',3,8,'Unique duplicate read SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2019,'Array2_76204.useq','Unique duplicate read SNPs from Maq','A2019',3405,'MEM',3,8,'Unique duplicate read SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2020,'Array2_78114.useq','Unique duplicate read SNPs from Maq','A2020',3405,'MEM',3,8,'Unique duplicate read SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2021,'Array2_79406.useq','Unique duplicate read SNPs from Maq','A2021',3405,'MEM',3,8,'Unique duplicate read SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2022,'Array2_79407.useq','Unique duplicate read SNPs from Maq','A2022',3405,'MEM',3,8,'Unique duplicate read SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2023,'Array2_80388.useq','Unique duplicate read SNPs from Maq','A2023',3405,'MEM',3,8,'Unique duplicate read SNPs from Maq',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2024,'Array1_29957','INDELS from Novoalign','A2024',3405,'MEM',3,8,'INDELS from Novoalign',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2025,'Array1_66395','INDELS from Novoalign','A2025',3405,'MEM',3,8,'INDELS from Novoalign',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2026,'Array1_77937','INDELS from Novoalign','A2026',3405,'MEM',3,8,'INDELS from Novoalign',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2027,'Array2_41968','INDELS from Novoalign','A2027',3405,'MEM',3,8,'INDELS from Novoalign',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2028,'Array2_59491','INDELS from Novoalign','A2028',3405,'MEM',3,8,'INDELS from Novoalign',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2029,'Array2_78114','INDELS from Novoalign','A2029',3405,'MEM',3,8,'INDELS from Novoalign',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2030,'Array1_30085','INDELS from Novoalign','A2030',3405,'MEM',3,8,'INDELS from Novoalign',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2031,'Array1_73197','INDELS from Novoalign','A2031',3405,'MEM',3,8,'INDELS from Novoalign',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2032,'Array1_78117','INDELS from Novoalign','A2032',3405,'MEM',3,8,'INDELS from Novoalign',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2033,'Array2_45817','INDELS from Novoalign','A2033',3405,'MEM',3,8,'INDELS from Novoalign',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2034,'Array2_67594','INDELS from Novoalign','A2034',3405,'MEM',3,8,'INDELS from Novoalign',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2035,'Array2_79406','INDELS from Novoalign','A2035',3405,'MEM',3,8,'INDELS from Novoalign',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2036,'Array1_45817','INDELS from Novoalign','A2036',3405,'MEM',3,8,'INDELS from Novoalign',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2037,'Array1_76591','INDELS from Novoalign','A2037',3405,'MEM',3,8,'INDELS from Novoalign',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2038,'Array1_78118','INDELS from Novoalign','A2038',3405,'MEM',3,8,'INDELS from Novoalign',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2039,'Array2_59473','INDELS from Novoalign','A2039',3405,'MEM',3,8,'INDELS from Novoalign',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2040,'Array2_71430','INDELS from Novoalign','A2040',3405,'MEM',3,8,'INDELS from Novoalign',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2041,'Array2_79407','INDELS from Novoalign','A2041',3405,'MEM',3,8,'INDELS from Novoalign',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2042,'Array1_46544','INDELS from Novoalign','A2042',3405,'MEM',3,8,'INDELS from Novoalign',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2043,'Array1_77580','INDELS from Novoalign','A2043',3405,'MEM',3,8,'INDELS from Novoalign',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2044,'Array1_78174','INDELS from Novoalign','A2044',3405,'MEM',3,8,'INDELS from Novoalign',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2045,'Array2_59475','INDELS from Novoalign','A2045',3405,'MEM',3,8,'INDELS from Novoalign',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2046,'Array2_73197','INDELS from Novoalign','A2046',3405,'MEM',3,8,'INDELS from Novoalign',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2047,'Array2_80388','INDELS from Novoalign','A2047',3405,'MEM',3,8,'INDELS from Novoalign',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2048,'Array1_64541','INDELS from Novoalign','A2048',3405,'MEM',3,8,'INDELS from Novoalign',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2049,'Array1_77935','INDELS from Novoalign','A2049',3405,'MEM',3,8,'INDELS from Novoalign',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2050,'Array2_41968Combine','INDELS from Novoalign','A2050',3405,'MEM',3,8,'INDELS from Novoalign',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2051,'Array2_59489','INDELS from Novoalign','A2051',3405,'MEM',3,8,'INDELS from Novoalign',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2052,'Array2_76204','INDELS from Novoalign','A2052',3405,'MEM',3,8,'INDELS from Novoalign',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2074,'Array1_29957','Alleler calls from all SNPs','A2074',3405,'MEM',3,8,'Alleler calls from all SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2075,'Array1_66395','Alleler calls from all SNPs','A2075',3405,'MEM',3,8,'Alleler calls from all SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2076,'Array1_77937','Alleler calls from all SNPs','A2076',3405,'MEM',3,8,'Alleler calls from all SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2077,'Array2_41968','Alleler calls from all SNPs','A2077',3405,'MEM',3,8,'Alleler calls from all SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2078,'Array2_59491','Alleler calls from all SNPs','A2078',3405,'MEM',3,8,'Alleler calls from all SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2079,'Array2_78114','Alleler calls from all SNPs','A2079',3405,'MEM',3,8,'Alleler calls from all SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2080,'Array1_30085','Alleler calls from all SNPs','A2080',3405,'MEM',3,8,'Alleler calls from all SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2081,'Array1_73197','Alleler calls from all SNPs','A2081',3405,'MEM',3,8,'Alleler calls from all SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2082,'Array1_78117','Alleler calls from all SNPs','A2082',3405,'MEM',3,8,'Alleler calls from all SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2083,'Array2_45817','Alleler calls from all SNPs','A2083',3405,'MEM',3,8,'Alleler calls from all SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2084,'Array2_67594','Alleler calls from all SNPs','A2084',3405,'MEM',3,8,'Alleler calls from all SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2085,'Array2_79406','Alleler calls from all SNPs','A2085',3405,'MEM',3,8,'Alleler calls from all SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2086,'Array1_45817','Alleler calls from all SNPs','A2086',3405,'MEM',3,8,'Alleler calls from all SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2087,'Array1_76591','Alleler calls from all SNPs','A2087',3405,'MEM',3,8,'Alleler calls from all SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2088,'Array1_78118','Alleler calls from all SNPs','A2088',3405,'MEM',3,8,'Alleler calls from all SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2089,'Array2_59473','Alleler calls from all SNPs','A2089',3405,'MEM',3,8,'Alleler calls from all SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2090,'Array2_71430','Alleler calls from all SNPs','A2090',3405,'MEM',3,8,'Alleler calls from all SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2091,'Array2_79407','Alleler calls from all SNPs','A2091',3405,'MEM',3,8,'Alleler calls from all SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2092,'Array1_46544','Alleler calls from all SNPs','A2092',3405,'MEM',3,8,'Alleler calls from all SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2093,'Array1_77580','Alleler calls from all SNPs','A2093',3405,'MEM',3,8,'Alleler calls from all SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2094,'Array1_78174','Alleler calls from all SNPs','A2094',3405,'MEM',3,8,'Alleler calls from all SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2095,'Array2_59475','Alleler calls from all SNPs','A2095',3405,'MEM',3,8,'Alleler calls from all SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2096,'Array2_73197','Alleler calls from all SNPs','A2096',3405,'MEM',3,8,'Alleler calls from all SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2097,'Array2_80388','Alleler calls from all SNPs','A2097',3405,'MEM',3,8,'Alleler calls from all SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2098,'Array1_64541','Alleler calls from all SNPs','A2098',3405,'MEM',3,8,'Alleler calls from all SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2099,'Array1_77935','Alleler calls from all SNPs','A2099',3405,'MEM',3,8,'Alleler calls from all SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2100,'Array2_41968Combine','Alleler calls from all SNPs','A2100',3405,'MEM',3,8,'Alleler calls from all SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2101,'Array2_59489','Alleler calls from all SNPs','A2101',3405,'MEM',3,8,'Alleler calls from all SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2102,'Array2_76204','Alleler calls from all SNPs','A2102',3405,'MEM',3,8,'Alleler calls from all SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2103,'Array1_29957.useq','Alleler calls from unique duplicate SNPs','A2103',3405,'MEM',3,8,'Alleler calls from unique duplicate SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2104,'Array1_66395.useq','Alleler calls from unique duplicate SNPs','A2104',3405,'MEM',3,8,'Alleler calls from unique duplicate SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2105,'Array1_77937.useq','Alleler calls from unique duplicate SNPs','A2105',3405,'MEM',3,8,'Alleler calls from unique duplicate SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2106,'Array2_41968.useq','Alleler calls from unique duplicate SNPs','A2106',3405,'MEM',3,8,'Alleler calls from unique duplicate SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2107,'Array2_59491.useq','Alleler calls from unique duplicate SNPs','A2107',3405,'MEM',3,8,'Alleler calls from unique duplicate SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2108,'Array2_78114.useq','Alleler calls from unique duplicate SNPs','A2108',3405,'MEM',3,8,'Alleler calls from unique duplicate SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2109,'Array1_30085.useq','Alleler calls from unique duplicate SNPs','A2109',3405,'MEM',3,8,'Alleler calls from unique duplicate SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2110,'Array1_73197.useq','Alleler calls from unique duplicate SNPs','A2110',3405,'MEM',3,8,'Alleler calls from unique duplicate SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2111,'Array1_78117.useq','Alleler calls from unique duplicate SNPs','A2111',3405,'MEM',3,8,'Alleler calls from unique duplicate SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2112,'Array2_45817.useq','Alleler calls from unique duplicate SNPs','A2112',3405,'MEM',3,8,'Alleler calls from unique duplicate SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2113,'Array2_67594.useq','Alleler calls from unique duplicate SNPs','A2113',3405,'MEM',3,8,'Alleler calls from unique duplicate SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2114,'Array2_79406.useq','Alleler calls from unique duplicate SNPs','A2114',3405,'MEM',3,8,'Alleler calls from unique duplicate SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2115,'Array1_45817.useq','Alleler calls from unique duplicate SNPs','A2115',3405,'MEM',3,8,'Alleler calls from unique duplicate SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2116,'Array1_76591.useq','Alleler calls from unique duplicate SNPs','A2116',3405,'MEM',3,8,'Alleler calls from unique duplicate SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2117,'Array1_78118.useq','Alleler calls from unique duplicate SNPs','A2117',3405,'MEM',3,8,'Alleler calls from unique duplicate SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2118,'Array2_59473.useq','Alleler calls from unique duplicate SNPs','A2118',3405,'MEM',3,8,'Alleler calls from unique duplicate SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2119,'Array2_71430.useq','Alleler calls from unique duplicate SNPs','A2119',3405,'MEM',3,8,'Alleler calls from unique duplicate SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2120,'Array2_79407.useq','Alleler calls from unique duplicate SNPs','A2120',3405,'MEM',3,8,'Alleler calls from unique duplicate SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2121,'Array1_46544.useq','Alleler calls from unique duplicate SNPs','A2121',3405,'MEM',3,8,'Alleler calls from unique duplicate SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2122,'Array1_77580.useq','Alleler calls from unique duplicate SNPs','A2122',3405,'MEM',3,8,'Alleler calls from unique duplicate SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2123,'Array1_78174.useq','Alleler calls from unique duplicate SNPs','A2123',3405,'MEM',3,8,'Alleler calls from unique duplicate SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2124,'Array2_59475.useq','Alleler calls from unique duplicate SNPs','A2124',3405,'MEM',3,8,'Alleler calls from unique duplicate SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2125,'Array2_73197.useq','Alleler calls from unique duplicate SNPs','A2125',3405,'MEM',3,8,'Alleler calls from unique duplicate SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2126,'Array2_80388.useq','Alleler calls from unique duplicate SNPs','A2126',3405,'MEM',3,8,'Alleler calls from unique duplicate SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2127,'Array1_64541.useq','Alleler calls from unique duplicate SNPs','A2127',3405,'MEM',3,8,'Alleler calls from unique duplicate SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2128,'Array1_77935.useq','Alleler calls from unique duplicate SNPs','A2128',3405,'MEM',3,8,'Alleler calls from unique duplicate SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2129,'Array2_41968Combine.useq','Alleler calls from unique duplicate SNPs','A2129',3405,'MEM',3,8,'Alleler calls from unique duplicate SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2130,'Array2_59489.useq','Alleler calls from unique duplicate SNPs','A2130',3405,'MEM',3,8,'Alleler calls from unique duplicate SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2131,'Array2_76204.useq','Alleler calls from unique duplicate SNPs','A2131',3405,'MEM',3,8,'Alleler calls from unique duplicate SNPs',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2132,'Unique Read Coverage','<P ALIGN=""LEFT"">See <A HREF=""https://hci-as1.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A96"" TARGET=""_blank""><B><U>https://hci-as1.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A96</U></B></A></P>','A2132',3405,'MEM',3,7,'Unique alignment read coverage track','u0028003','2010-04-20 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2134,'Binomial PValue','','A2134',3806,'PUBLIC',3,13,'','u0028003','2010-04-16 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2135,'QValue FDR','','A2135',3806,'PUBLIC',3,13,'','u0028003','2010-04-16 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2136,'Emprical FDR','','A2136',3806,'PUBLIC',3,13,'','u0028003','2010-04-16 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2137,'Peak pick sub windows','','A2137',3806,'PUBLIC',3,13,'Binomial pvalue track of peak picks at 2x, FDR 5%',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2138,'5% FDR, 2x Enriched\r','','A2138',3806,'PUBLIC',3,13,'Scores are scaled ratios of enrichment','u0028003','2010-04-16 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2139,'LiftOver Peaks from mm9','<P ALIGN=""LEFT""><A HREF=""https://hci-as1.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A98&isGuest=Y"" TARGET=""_blank""><B><U>https://hci-as1.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A98&amp;isGuest=Y</U></B></A></P>','A2139',3405,'PUBLIC',3,13,'5% FDR, 2x Enriched','u0028003','2010-04-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2140,'Peaks FDR QVal and Emp 0.1%, 2x Enrichment','','A2140',3405,'PUBLIC',NULL,13,'','u0028003','2010-04-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2141,'Valouev et.al. Picks','','A2141',3806,'PUBLIC',3,13,'','u0028003','2010-04-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2142,'Read Coverage Minus Strand','<P ALIGN=""LEFT"">See <A HREF=""https://hci-as1.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A99"" TARGET=""_blank""><B><U>https://hci-as1.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A99</U></B></A></P>','A2142',3405,'MEM',3,7,'Primarily uniquely aligning reads plus up to 10 repeats provided they fall within 1kb ','u0028003','2010-04-20 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2143,'Read Coverage Plus Strand','<P ALIGN=""LEFT"">See <A HREF=""https://hci-as1.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A99"" TARGET=""_blank""><B><U>https://hci-as1.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A99</U></B></A></P>','A2143',3405,'MEM',3,7,'Primarily uniquely aligning reads plus up to 10 repeats provided they fall within 1kb ',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2144,'ChIP Hypo Alpha1','','A2144',3806,'MEM',3,35,'','u0028003','2010-04-22 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2145,'ChIP Norm Alpha1','','A2145',3806,'MEM',3,35,'','u0028003','2010-04-22 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2146,'ChIP Norm Alpha2','','A2146',3806,'MEM',3,35,'','u0028003','2010-04-22 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2147,'Igg Hypo Alpha1','','A2147',3806,'MEM',3,35,'','u0028003','2010-04-22 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2148,'Igg Hypo Alpha2','','A2148',3806,'MEM',3,35,'','u0028003','2010-04-22 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2149,'Input','','A2149',3806,'MEM',3,35,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2150,'ChIP Hypo Alpha1 Vs ChIP Norm Alpha1','','A2150',3806,'MEM',3,35,'','u0028003','2010-04-22 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2151,'ChIP Hypo Alpha1 Vs IgGs Input','','A2151',3806,'MEM',3,35,'','u0028003','2010-04-22 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2152,'ChIP Normal Alpha1 Vs IgGs Input','','A2152',3806,'MEM',3,35,'','u0028003','2010-04-22 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2153,'ChIP Normal Alpha1 Vs IgGs Input','','A2153',3806,'MEM',3,35,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2154,'ChIP Hypo Alpha1 Vs IgGs Input','','A2154',3806,'MEM',3,35,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2155,'ChIP Hypo Alpha1 Vs ChIP Norm Alpha1','','A2155',3806,'MEM',3,35,'',NULL,NULL,'Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2160,'Input','','A2160',3806,'PUBLIC',3,13,'Reprocessed from GEO GSM480163','u0028003','2010-04-26 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2161,'Jarid1a','','A2161',3806,'PUBLIC',3,13,'Reprocessed from GEO GSM480163','u0028003','2010-04-26 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2162,'ChIP',NULL,'A2162',3405,'PUBLIC',NULL,28,NULL,'u0028003','2010-04-26 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2163,'Input',NULL,'A2163',3405,'PUBLIC',NULL,28,NULL,'u0028003','2010-04-26 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2167,'Read Coverage',NULL,'A2167',3405,'MEM',NULL,7,NULL,'u0028003','2010-05-03 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2168,'Splice Junctions',NULL,'A2168',3405,'MEM',NULL,7,NULL,'u0028003','2010-05-03 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2169,'H1 RNA-Seq','','A2169',3405,'PUBLIC',3,7,'ES stem cell line, standard unstranded RNA-Seq','u0028003','2010-05-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2170,'IMR90 RNA-Seq','','A2170',3405,'PUBLIC',3,7,'Normal fibroblast cell line, standard unstranded RNA-Seq','u0028003','2010-05-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2171,'H1 Small RNA-Seq','','A2171',3405,'PUBLIC',3,7,'ES stem cell line','u0028003','2010-05-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2172,'IMR90 Small RNA-Seq','','A2172',3405,'PUBLIC',3,7,'Normal fibroblast cell line','u0028003','2010-05-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2173,'Gm12878 Cell','','A2173',3405,'PUBLIC',3,28,'','u0028003','2010-05-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2174,'Gm12878 Cytosol','','A2174',3405,'PUBLIC',3,28,'','u0028003','2010-05-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2175,'Gm12878 Nucleus','','A2175',3405,'PUBLIC',3,28,'','u0028003','2010-05-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2176,'K562 Chromatin','','A2176',3405,'PUBLIC',3,28,'','u0028003','2010-05-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2177,'K562 Cell','','A2177',3405,'PUBLIC',3,28,'','u0028003','2010-05-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2178,'K562 Cytosol','','A2178',3405,'PUBLIC',3,28,'','u0028003','2010-05-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2179,'K562 Nucleolus','','A2179',3405,'PUBLIC',3,28,'','u0028003','2010-05-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2180,'K562 Nucleoplasm','','A2180',3405,'PUBLIC',3,28,'','u0028003','2010-05-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2181,'K562 Nucleus','','A2181',3405,'PUBLIC',3,28,'','u0028003','2010-05-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2182,'K562 Polysome','','A2182',3405,'PUBLIC',3,28,'','u0028003','2010-05-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2183,'Prostate Cell','','A2183',3405,'PUBLIC',3,28,'','u0028003','2010-05-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2185,'HPltBlUCSCJR','','A2185',6300,'MEM',40,30,'','tolley','2010-05-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2188,'WholeBloodBaseline','','A2188',6300,'MEM',40,30,'','tolley','2010-05-07 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2193,'WholeBloodClot',NULL,'A2193',6300,'MEM',40,30,NULL,'tolley','2010-05-07 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2194,'WholeBloodClotRX',NULL,'A2194',6300,'MEM',40,30,NULL,'tolley','2010-05-07 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2200,'JR MM9 plt BL UCSC','<P ALIGN=""LEFT"">these were reparsed to mouse genome using a ucsc table in parsing</P>','A2200',3806,'MEM',40,30,'','tolley','2010-05-11 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2211,'Donor H3k27me3_QVAL','','A2211',3405,'MEM',14,7,'','CairnsLab','2010-05-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2212,'Donor H3k4me3_QVAL','','A2212',3405,'MEM',14,7,'','CairnsLab','2010-05-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2213,'GI36 H3k4me3_QVAL','','A2213',3405,'MEM',14,7,'','CairnsLab','2010-05-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2214,'GI36 H3k27me3_QVAL','','A2214',3405,'MEM',14,7,'','CairnsLab','2010-05-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2215,'GI36 Monos_QVal','','A2215',3405,'MEM',14,7,'','CairnsLab','2010-05-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2216,'PTO4 H3k4me3_QVAL','','A2216',3405,'MEM',14,7,'','CairnsLab','2010-05-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2217,'PTO4 H3k27me3_QVAL','','A2217',3405,'MEM',14,7,'','CairnsLab','2010-05-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2218,'PTO4 Monos_QVal','','A2218',3405,'MEM',14,7,'','CairnsLab','2010-05-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2219,'JQ81 H3k4me3_QVAL','','A2219',3405,'MEM',14,7,'','CairnsLab','2010-05-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2220,'JQ81 H3k27me3_QVAL','','A2220',3405,'MEM',14,7,'','CairnsLab','2010-05-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2221,'JQ81 Histone_QVAL','','A2221',3405,'MEM',14,7,'','CairnsLab','2010-05-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2222,'ODN2 H3K4me3_QVal','','A2222',3405,'MEM',14,7,'','CairnsLab','2010-05-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2223,'ODN2 H3K27me3_QVal','','A2223',3405,'MEM',14,7,'','CairnsLab','2010-05-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2224,'ODN2 Monos_QVal','','A2224',3405,'MEM',14,7,'','CairnsLab','2010-05-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2225,'WA1V_H3k4me3_Qval','','A2225',3405,'MEM',14,7,'','CairnsLab','2010-05-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2226,'WA1V_H3k27me3_Qval','','A2226',3405,'MEM',14,7,'','CairnsLab','2010-05-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2228,'WA1V_Monos_Qval','','A2228',3405,'MEM',14,7,'','CairnsLab','2010-05-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2229,'X7FC_H3k4me3_Qval','','A2229',3405,'MEM',14,7,'','CairnsLab','2010-05-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2230,'X7FC_H3k27me3_Qval','','A2230',3405,'MEM',14,7,'','CairnsLab','2010-05-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2231,'X7FC_Monos_Qval','','A2231',3405,'MEM',14,7,'','CairnsLab','2010-05-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2234,'JR hg19 plt bl pair parse (7101X3)_copy','<P ALIGN=""LEFT"">these were pair parsed for splice junction visualization</P>','A2234',6300,'MEM',40,30,'human bl platelets','tolley','2010-05-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2235,'JR MM9 plt bl pair parse_copy','<P ALIGN=""LEFT"">these were pair parsed for splice junction visualization</P>','A2235',3806,'MEM',40,30,'mouse bl platelets','tolley','2010-05-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2236,'Filtered150bpwindow_10PolAT','','A2236',3405,'MEM',14,7,'','CairnsLab','2010-05-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2239,'Filtered200bpwindow_20PolAT_copy','','A2239',3405,'MEM',14,7,'','CairnsLab','2010-05-13 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2240,'rs25 sibs chr19','<P ALIGN=""LEFT"">Read coverage on chromosome 19, data aligned with soap aligner.</P>','A2240',6301,'MEM',49,36,'RNA seq read coverage','bmilash','2010-05-13 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2241,'rs25 mut chr19','','A2241',6301,'MEM',49,36,'Read coverage of rs25 mutant data','bmilash','2010-05-13 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2263,'H3K4me1','','A2263',3806,'PUBLIC',19,13,'H3K4me1 in macrophages untreated with LPS','GravesLab','2010-05-17 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2264,'PU.1_UT','','A2264',3806,'PUBLIC',19,13,'PU.1 in macrophages untreated with LPS','GravesLab','2010-05-17 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2265,'PU.1_2H','','A2265',3806,'PUBLIC',19,13,'PU.1 in macrophages treated with LPS 2H','GravesLab','2010-05-17 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2267,'PU.1 UT','','A2267',3806,'MEM',19,13,'','GravesLab','2010-05-17 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2268,'ReadCoverage',NULL,'A2268',6300,'MEM',43,7,NULL,'Andrew','2010-05-20 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2269,'EnsemblFlatGenes','<P ALIGN=""LEFT"">Ensembl transcripts table downloaded from UCSC.  </P><P ALIGN=""LEFT"">Downloaded table of Ensembl transcript id to Ensembl gene id from Biomart.</P><P ALIGN=""LEFT"">Transcripts were grouped based on gene id and exons were flattened inclusively using script FlattenEnsemblTable.pl.</P><P ALIGN=""LEFT""></P>','A2269',6300,'PUBLIC',43,7,'Ensembl transcripts merged based on Ensembl gene name.','Andrew','2010-05-20 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2271,'ReadCoverage',NULL,'A2271',3806,'MEM',43,7,NULL,'Andrew','2010-05-20 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2272,'EnsemblFlatGenes','<P ALIGN=""LEFT"">Ensembl transcripts table downloaded from UCSC.  </P><P ALIGN=""LEFT"">Downloaded table of Ensembl transcript id to Ensembl gene id from Biomart.</P><P ALIGN=""LEFT"">Transcripts were grouped based on gene id and exons were flattened inclusively using script FlattenEnsemblTable.pl.</P>','A2272',3806,'PUBLIC',43,7,'Ensembl transcripts merged based on Ensembl gene name.','Andrew','2010-05-20 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2273,'Converted Cs 7259X1','','A2273',3405,'MEM',3,7,'250bp window level sum from ScanSeqs, unstranded','u0028003','2010-05-21 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2274,'Converted Cs 7259X2','','A2274',3405,'MEM',3,7,'250bp window level sum from ScanSeqs, unstranded','u0028003','2010-05-21 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2275,'Non Converted Cs 7259X2','','A2275',3405,'MEM',3,7,'250bp window level sum from ScanSeqs, unstranded','u0028003','2010-05-21 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2276,'Non Converted Cs 7259X1','','A2276',3405,'MEM',3,7,'250bp window level sum from ScanSeqs, unstranded','u0028003','2010-05-21 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2277,'Plus Strand','','A2277',3405,'MEM',3,7,'Single bp resolution, y axis is the number of reads with this call','u0028003','2010-05-21 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2278,'Minus Strand','','A2278',3405,'MEM',3,7,'Single bp resolution, y axis is the number of reads with this call','u0028003','2010-05-21 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2279,'Plus Strand','','A2279',3405,'MEM',3,7,'Single bp resolution, y axis is the number of reads with this call','u0028003','2010-05-21 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2280,'Minus Strand','','A2280',3405,'MEM',3,7,'Single bp resolution, y axis is the number of reads with this call','u0028003','2010-05-21 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2281,'Plus Strand','','A2281',3405,'MEM',3,7,'Single bp resolution, y axis is the number of reads with this call','u0028003','2010-05-21 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2282,'Minus Strand','','A2282',3405,'MEM',3,7,'Single bp resolution, y axis is the number of reads with this call','u0028003','2010-05-21 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2283,'Plus Strand','','A2283',3405,'MEM',3,7,'Single bp resolution, y axis is the number of reads with this call','u0028003','2010-05-21 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2284,'Minus Strand','','A2284',3405,'MEM',3,7,'Single bp resolution, y axis is the number of reads with this call','u0028003','2010-05-21 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2285,'NonredundantKnownGeneTableDRSSGuided','','A2285',6300,'PUBLIC',43,7,'','Andrew','2010-05-21 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2286,'NonredundantKnownGeneTableExonCoverageGuided',NULL,'A2286',6300,'PUBLIC',43,7,NULL,'Andrew','2010-05-21 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2287,'NonredundantKnownGeneTableDRSSGuided','','A2287',3806,'PUBLIC',43,7,'','Andrew','2010-05-21 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2288,'NonredundantKnownGeneTableExonCoverageGuided',NULL,'A2288',3806,'PUBLIC',43,7,NULL,'Andrew','2010-05-21 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2289,'PltALI','','A2289',6300,'MEM',40,30,'','tolley','2010-05-24 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2290,'PredictedPol3Genes','','A2290',3405,'PUBLIC',43,7,'','Andrew','2010-05-24 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2291,'KnownPol3GenesHeLa','','A2291',3405,'PUBLIC',43,7,'','Andrew','2010-05-24 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2292,'Predicted_tRNAs_513','','A2292',3405,'PUBLIC',43,7,'','Andrew','2010-05-24 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2293,'MappabletRNAs_467',NULL,'A2293',3405,'PUBLIC',43,7,NULL,'Andrew','2010-05-24 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2294,'tRNA-pseudogenes_172','','A2294',3405,'PUBLIC',43,7,'','Andrew','2010-05-25 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2295,'ensembl genes','','A2295',3806,'PUBLIC',3,28,'Merged transcripts with same ensemble gene name maximizing exonic sequence','u0028003','2010-05-26 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2296,'ensembl transcripts','','A2296',3806,'PUBLIC',3,28,'','u0028003','2010-05-26 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2297,'ensemble genes no overlaps','','A2297',3806,'PUBLIC',3,28,'','u0028003','2010-05-26 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2298,'Read Coverage SureSelect','<P ALIGN=""LEFT"">Show the number of reads overlapping each base pair</P>','A2298',6300,'MEM',3,8,'Duplicate filtered read coverage track','u0028003','2010-05-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2299,'Read Coverage Custom Array','<P ALIGN=""LEFT"">Show the number of reads overlapping each base pair</P>','A2299',6300,'MEM',3,8,'Duplicate filtered read coverage track','u0028003','2010-05-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2306,'PMN_Adult_BL','','A2306',6300,'MEM',40,30,'','tolley','2010-05-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2307,'PMN_Premature_BL','','A2307',6300,'MEM',40,30,'','tolley','2010-05-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2328,'PMN_preterm_PAF','','A2328',6300,'MEM',40,30,'','tolley','2010-06-01 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2349,'PMN_Adult_PAF','','A2349',6300,'MEM',40,30,'','tolley','2010-06-01 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2350,'Read Coverage','<P ALIGN=""LEFT"">From one lane of sequencing</P>','A2350',6301,'MEM',3,7,'NovoalignParser PointData converted to relative ReadCoverage','u0028003','2010-06-02 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2353,'Plus','','A2353',6301,'MEM',3,7,'','u0028003','2010-06-02 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2354,'Minus','','A2354',6301,'MEM',3,7,'','u0028003','2010-06-02 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2355,'Plus','','A2355',6301,'MEM',3,7,'','u0028003','2010-06-02 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2356,'Minus','','A2356',6301,'MEM',3,7,'','u0028003','2010-06-02 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2357,'T0','','A2357',6300,'PUBLIC',3,28,'Relative Read Coverage','u0028003','2010-06-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2358,'T1','','A2358',6300,'PUBLIC',3,28,'Relative Read Coverage','u0028003','2010-06-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2359,'T2','','A2359',6300,'PUBLIC',3,28,'Relative Read Coverage','u0028003','2010-06-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2360,'T_All','','A2360',6300,'PUBLIC',3,28,'Relative Read Coverage','u0028003','2010-06-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2361,'C0','','A2361',6300,'PUBLIC',3,28,'Relative Read Coverage','u0028003','2010-06-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2362,'C1','','A2362',6300,'PUBLIC',3,28,'Relative Read Coverage','u0028003','2010-06-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2363,'C2','','A2363',6300,'PUBLIC',3,28,'Relative Read Coverage','u0028003','2010-06-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2364,'C_All','','A2364',6300,'PUBLIC',3,28,'Relative Read Coverage','u0028003','2010-06-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2365,'IXO1_Monos_Qval','','A2365',3405,'MEM',14,7,'','CairnsLab','2010-06-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2366,'Eri4 reads','','A2366',1204,'MEM',32,25,'Simple coversion of bed file from novo','BassLab','2010-06-10 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2367,'Eri4','','A2367',1204,'MEM',32,25,'Relative read coverage, one lane seq','BassLab','2010-06-10 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2368,'IXO1_H3K4me3_Qval','','A2368',3405,'MEM',14,7,'','CairnsLab','2010-06-10 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2371,'Read Coverage Unique and Repeat Reads Plus Strand',NULL,'A2371',3405,'MEM',43,7,NULL,'Andrew','2010-06-10 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2372,'Read Coverage Unique and Repeat Reads Minus Strand',NULL,'A2372',3405,'MEM',43,7,NULL,'Andrew','2010-06-10 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2373,'Read Coverage Repeat Reads Plus Strand',NULL,'A2373',3405,'MEM',43,7,NULL,'Andrew','2010-06-10 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2374,'Read Coverage Repeat Reads Minus Strand',NULL,'A2374',3405,'MEM',43,7,NULL,'Andrew','2010-06-10 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2375,'Read Coverage Unique Reads Plus Strand',NULL,'A2375',3405,'MEM',43,7,NULL,'Andrew','2010-06-10 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2376,'Read Coverage Unique Reads Minus Strand',NULL,'A2376',3405,'MEM',43,7,NULL,'Andrew','2010-06-10 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2377,'H3K27me1','','A2377',3405,'MEM',14,7,'','CairnsLab','2010-06-11 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2387,'H3K36me3','','A2387',3405,'MEM',14,7,'','CairnsLab','2010-06-13 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2388,'Gm12878','','A2388',3405,'PUBLIC',3,28,'FDR < 1%, 2x','u0028003','2010-06-14 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2389,'H1hesc','','A2389',3405,'PUBLIC',3,28,'FDR < 1%, 2x','u0028003','2010-06-14 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2390,'Hepg2','','A2390',3405,'PUBLIC',3,28,'FDR < 1%, 2x','u0028003','2010-06-14 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2391,'Hmec','','A2391',3405,'PUBLIC',3,28,'FDR < 1%, 2x','u0028003','2010-06-14 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2392,'Hsmm','','A2392',3405,'PUBLIC',3,28,'FDR < 1%, 2x','u0028003','2010-06-14 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2393,'Huvec','','A2393',3405,'PUBLIC',3,28,'FDR < 1%, 2x','u0028003','2010-06-14 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2394,'Keratinocyte','','A2394',3405,'PUBLIC',3,28,'FDR < 1%, 2x','u0028003','2010-06-14 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2395,'K562','','A2395',3405,'PUBLIC',3,28,'FDR < 1%, 2x','u0028003','2010-06-14 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2396,'NhlfC','','A2396',3405,'PUBLIC',3,28,'FDR < 1%, 2x','u0028003','2010-06-14 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2397,'All Peaks Summed Scores','<P ALIGN=""LEFT"">Overlapping peaks from all of the datasets were graphed as the sum of their associated normalized log2ratio scores.</P>','A2397',3405,'PUBLIC',3,28,'FDR < 1%, 2x','u0028003','2010-06-15 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2398,'All Peaks Summed','<P ALIGN=""LEFT"">Overlapping peaks from all of the datasets were graphed as the count of the number of overlaps</P>','A2398',3405,'PUBLIC',3,28,'FDR < 1%, 2x','u0028003','2010-06-15 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2401,'Rep1Gm12878ControlV2','Reprocessed from http://hgdownload.cse.ucsc.edu/goldenPath/hg18/encodeDCC/wgEncodeBroadChipSeq/','A2401',3405,'PUBLIC',3,28,'# reads per base/ # mapped reads/ million',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(2402,'Rep1NhlfCtcf','Reprocessed from http://hgdownload.cse.ucsc.edu/goldenPath/hg18/encodeDCC/wgEncodeBroadChipSeq/','A2402',3405,'PUBLIC',3,28,'# reads per base/ # mapped reads/ million',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(2403,'Rep1Gm12878CtcfV2','Reprocessed from http://hgdownload.cse.ucsc.edu/goldenPath/hg18/encodeDCC/wgEncodeBroadChipSeq/','A2403',3405,'PUBLIC',3,28,'# reads per base/ # mapped reads/ million',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(2404,'Rep2Gm12878ControlV2','Reprocessed from http://hgdownload.cse.ucsc.edu/goldenPath/hg18/encodeDCC/wgEncodeBroadChipSeq/','A2404',3405,'PUBLIC',3,28,'# reads per base/ # mapped reads/ million',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(2405,'Rep1Hepg2Control','Reprocessed from http://hgdownload.cse.ucsc.edu/goldenPath/hg18/encodeDCC/wgEncodeBroadChipSeq/','A2405',3405,'PUBLIC',3,28,'# reads per base/ # mapped reads/ million',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(2406,'Rep2Gm12878CtcfV2','Reprocessed from http://hgdownload.cse.ucsc.edu/goldenPath/hg18/encodeDCC/wgEncodeBroadChipSeq/','A2406',3405,'PUBLIC',3,28,'# reads per base/ # mapped reads/ million',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(2407,'Rep1Hepg2Ctcf','Reprocessed from http://hgdownload.cse.ucsc.edu/goldenPath/hg18/encodeDCC/wgEncodeBroadChipSeq/','A2407',3405,'PUBLIC',3,28,'# reads per base/ # mapped reads/ million',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(2408,'Rep2Hepg2Control','Reprocessed from http://hgdownload.cse.ucsc.edu/goldenPath/hg18/encodeDCC/wgEncodeBroadChipSeq/','A2408',3405,'PUBLIC',3,28,'# reads per base/ # mapped reads/ million',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(2409,'Rep1HmecControl','Reprocessed from http://hgdownload.cse.ucsc.edu/goldenPath/hg18/encodeDCC/wgEncodeBroadChipSeq/','A2409',3405,'PUBLIC',3,28,'# reads per base/ # mapped reads/ million',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(2410,'Rep2HmecCtcf','Reprocessed from http://hgdownload.cse.ucsc.edu/goldenPath/hg18/encodeDCC/wgEncodeBroadChipSeq/','A2410',3405,'PUBLIC',3,28,'# reads per base/ # mapped reads/ million',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(2411,'Rep1HsmmControl','Reprocessed from http://hgdownload.cse.ucsc.edu/goldenPath/hg18/encodeDCC/wgEncodeBroadChipSeq/','A2411',3405,'PUBLIC',3,28,'# reads per base/ # mapped reads/ million',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(2412,'Rep2HsmmControl','Reprocessed from http://hgdownload.cse.ucsc.edu/goldenPath/hg18/encodeDCC/wgEncodeBroadChipSeq/','A2412',3405,'PUBLIC',3,28,'# reads per base/ # mapped reads/ million',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(2413,'Rep1HuvecControlV2','Reprocessed from http://hgdownload.cse.ucsc.edu/goldenPath/hg18/encodeDCC/wgEncodeBroadChipSeq/','A2413',3405,'PUBLIC',3,28,'# reads per base/ # mapped reads/ million',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(2414,'Rep2HsmmCtcf','Reprocessed from http://hgdownload.cse.ucsc.edu/goldenPath/hg18/encodeDCC/wgEncodeBroadChipSeq/','A2414',3405,'PUBLIC',3,28,'# reads per base/ # mapped reads/ million',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(2415,'Rep1HuvecCtcfV2','Reprocessed from http://hgdownload.cse.ucsc.edu/goldenPath/hg18/encodeDCC/wgEncodeBroadChipSeq/','A2415',3405,'PUBLIC',3,28,'# reads per base/ # mapped reads/ million',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(2416,'Rep2HuvecControlV2','Reprocessed from http://hgdownload.cse.ucsc.edu/goldenPath/hg18/encodeDCC/wgEncodeBroadChipSeq/','A2416',3405,'PUBLIC',3,28,'# reads per base/ # mapped reads/ million',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(2417,'Rep1K562CtcfV2','Reprocessed from http://hgdownload.cse.ucsc.edu/goldenPath/hg18/encodeDCC/wgEncodeBroadChipSeq/','A2417',3405,'PUBLIC',3,28,'# reads per base/ # mapped reads/ million',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(2418,'Rep2K562ControlV2','Reprocessed from http://hgdownload.cse.ucsc.edu/goldenPath/hg18/encodeDCC/wgEncodeBroadChipSeq/','A2418',3405,'PUBLIC',3,28,'# reads per base/ # mapped reads/ million',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(2419,'Rep1KeratinocyteControlV2','Reprocessed from http://hgdownload.cse.ucsc.edu/goldenPath/hg18/encodeDCC/wgEncodeBroadChipSeq/','A2419',3405,'PUBLIC',3,28,'# reads per base/ # mapped reads/ million',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(2420,'Rep2KeratinocyteControlV2','Reprocessed from http://hgdownload.cse.ucsc.edu/goldenPath/hg18/encodeDCC/wgEncodeBroadChipSeq/','A2420',3405,'PUBLIC',3,28,'# reads per base/ # mapped reads/ million',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(2421,'Rep1KeratinocyteCtcfV2','Reprocessed from http://hgdownload.cse.ucsc.edu/goldenPath/hg18/encodeDCC/wgEncodeBroadChipSeq/','A2421',3405,'PUBLIC',3,28,'# reads per base/ # mapped reads/ million',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(2422,'Rep2NhlfControl','Reprocessed from http://hgdownload.cse.ucsc.edu/goldenPath/hg18/encodeDCC/wgEncodeBroadChipSeq/','A2422',3405,'PUBLIC',3,28,'# reads per base/ # mapped reads/ million',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(2423,'Gm12878','<P ALIGN=""LEFT"">200bp Window level -10Log10(FDR)</P>','A2423',3405,'PUBLIC',3,28,'DESeq negative binomial p-values -> -10Log10(BH FDRs)','u0028003','2010-06-15 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2426,'Hepg2','200bp Window level -10Log10(FDR)','A2426',3405,'PUBLIC',3,28,'DESeq negative binomial p-values -> -10Log10(BH FDRs)',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(2427,'Hsmm','200bp Window level -10Log10(FDR)','A2427',3405,'PUBLIC',3,28,'DESeq negative binomial p-values -> -10Log10(BH FDRs)',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(2428,'K562','200bp Window level -10Log10(FDR)','A2428',3405,'PUBLIC',3,28,'DESeq negative binomial p-values -> -10Log10(BH FDRs)',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(2429,'NhlfC','200bp Window level -10Log10(FDR)','A2429',3405,'PUBLIC',3,28,'DESeq negative binomial p-values -> -10Log10(BH FDRs)',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(2430,'H1hesc','200bp Window level -10Log10(FDR)','A2430',3405,'PUBLIC',3,28,'DESeq negative binomial p-values -> -10Log10(BH FDRs)',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(2431,'Hmec','200bp Window level -10Log10(FDR)','A2431',3405,'PUBLIC',3,28,'DESeq negative binomial p-values -> -10Log10(BH FDRs)',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(2432,'Huvec','200bp Window level -10Log10(FDR)','A2432',3405,'PUBLIC',3,28,'DESeq negative binomial p-values -> -10Log10(BH FDRs)',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(2433,'Keratinocyte','200bp Window level -10Log10(FDR)','A2433',3405,'PUBLIC',3,28,'DESeq negative binomial p-values -> -10Log10(BH FDRs)',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(2435,'Gm12878','200bp Window level log2 ratio','A2435',3405,'PUBLIC',3,28,'Normalized Log2Ratio',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(2436,'Hepg2','200bp Window level log2 ratio','A2436',3405,'PUBLIC',3,28,'Normalized Log2Ratio',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(2437,'Hsmm','200bp Window level log2 ratio','A2437',3405,'PUBLIC',3,28,'Normalized Log2Ratio',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(2438,'K562','200bp Window level log2 ratio','A2438',3405,'PUBLIC',3,28,'Normalized Log2Ratio',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(2439,'NhlfC','200bp Window level log2 ratio','A2439',3405,'PUBLIC',3,28,'Normalized Log2Ratio',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(2440,'H1hesc','200bp Window level log2 ratio','A2440',3405,'PUBLIC',3,28,'Normalized Log2Ratio',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(2441,'Hmec','200bp Window level log2 ratio','A2441',3405,'PUBLIC',3,28,'Normalized Log2Ratio',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(2442,'Huvec','200bp Window level log2 ratio','A2442',3405,'PUBLIC',3,28,'Normalized Log2Ratio',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(2443,'Keratinocyte','200bp Window level log2 ratio','A2443',3405,'PUBLIC',3,28,'Normalized Log2Ratio',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(2444,'Psuedogenes','<P ALIGN=""LEFT"">Data was retrieved from psuedogene.org and combined with what has been submitted to UCSC encode project.</P>','A2444',3405,'MEM',14,7,'','CairnsLab','2010-06-21 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2446,'Hg18_EST','','A2446',3405,'MEM',14,7,'','CairnsLab','2010-06-21 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2453,'Agilent SureSelect Exome Baits','<P ALIGN=""LEFT"">Downloaded and combine bed files from eArray</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">PGRID157583nonHDBED.bed</P><P ALIGN=""LEFT"">PGRID157586nonHDBED.bed</P><P ALIGN=""LEFT"">PGRID157588nonHDBED.bed</P><P ALIGN=""LEFT"">PGRID157590nonHDBED.bed</P><P ALIGN=""LEFT"">PGRID157593nonHDBED.bed</P><P ALIGN=""LEFT"">PGRID157594nonHDBED.bed</P>','A2453',6300,'PUBLIC',3,8,'','u0028003','2010-06-29 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2459,'84060','','A2459',6300,'MEM',3,38,'All Filtered Maq SNP calls','u0028003','2010-07-01 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2460,'84615','','A2460',6300,'MEM',3,38,'All Filtered Maq SNP calls','u0028003','2010-07-01 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2461,'crb030','','A2461',6300,'MEM',3,38,'All Filtered Maq SNP calls','u0028003','2010-07-01 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2462,'84060','','A2462',6300,'MEM',3,38,'Non-synonymous indels called by Alleler using Ensembl transcripts, min 2 unique reads','u0028003','2010-07-02 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2463,'84615','','A2463',6300,'MEM',3,38,'Non-synonymous indels called by Alleler using Ensembl transcripts, min 2 unique reads','u0028003','2010-07-02 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2464,'crb030','','A2464',6300,'MEM',3,38,'Non-synonymous indels called by Alleler using Ensembl transcripts, min 2 unique reads','u0028003','2010-07-02 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2465,'crb030','','A2465',6300,'MEM',3,38,'Non-synonymous SNPs called by Alleler using Ensembl transcripts from Maq SNPs','u0028003','2010-07-02 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2466,'84060','','A2466',6300,'MEM',3,38,'Non-synonymous SNPs called by Alleler using Ensembl transcripts from Maq SNPs','u0028003','2010-07-02 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2467,'84615','','A2467',6300,'MEM',3,38,'Non-synonymous SNPs called by Alleler using Ensembl transcripts from Maq SNPs','u0028003','2010-07-02 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2468,'IP','','A2468',3806,'MEM',3,19,'','u0028003','2010-07-15 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2469,'Input','','A2469',3806,'MEM',3,19,'','u0028003','2010-07-15 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2470,'Binomial P-Value','','A2470',3806,'MEM',3,19,'','u0028003','2010-07-15 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2471,'Empirical FDR','','A2471',3806,'MEM',3,19,'','u0028003','2010-07-15 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2472,'QValue FDR','','A2472',3806,'MEM',3,19,'','u0028003','2010-07-15 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2473,'IP','','A2473',3806,'MEM',3,19,'','u0028003','2010-07-15 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2474,'Input','','A2474',3806,'MEM',3,19,'','u0028003','2010-07-15 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2475,'Binomial P-Value','','A2475',3806,'MEM',3,19,'','u0028003','2010-07-15 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2476,'Empirical FDR','','A2476',3806,'MEM',3,19,'','u0028003','2010-07-15 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2477,'QValue FDR','','A2477',3806,'MEM',3,19,'','u0028003','2010-07-15 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2478,'1% FDR, 2x diff','','A2478',3806,'MEM',3,19,'','u0028003','2010-07-15 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2488,'Top 50 SNPs',NULL,'A2488',6300,'MEM',NULL,38,NULL,'u0028003','2010-07-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2494,'RNASeq_control','','A2494',6300,'PUBLIC',NULL,28,'','Ying','2010-07-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2495,'binomial p-value','','A2495',6300,'MEM',51,10,'ScanSeqs'' binomial p-values. window 200 -p 300','Russell','2010-08-02 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2496,'emperical FDR','','A2496',6300,'MEM',51,10,'ScanSeqs'' emperical window 200 -p 300','Russell','2010-08-02 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2497,'Q-Values FDR','','A2497',6300,'MEM',51,10,'ScanSeqs'' Q-Values using window 200 -p 300','Russell','2010-08-02 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2498,'Sum','','A2498',6300,'MEM',51,10,'','Russell','2010-08-02 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2499,'Sum minus','','A2499',6300,'MEM',51,10,'','Russell','2010-08-02 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2501,'Sum plus','','A2501',6300,'MEM',51,10,'','Russell','2010-08-02 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2502,'Read Coverage unique alignments Plus Strand',NULL,'A2502',3405,'MEM',43,7,NULL,'Andrew','2010-08-03 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2503,'Read Coverage 2-10 repeat alignments Minus Strand',NULL,'A2503',3405,'MEM',43,7,NULL,'Andrew','2010-08-03 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2504,'Read Coverage 2-10 repeat alignments Plus Strand',NULL,'A2504',3405,'MEM',43,7,NULL,'Andrew','2010-08-03 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2505,'Read Coverage unique alignments Minus Strand',NULL,'A2505',3405,'MEM',43,7,NULL,'Andrew','2010-08-03 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2511,'Read Coverage 2-10 repeat alignments Minus Strand',NULL,'A2511',3405,'MEM',43,7,NULL,'Andrew','2010-08-05 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2513,'Read Coverage 2-10 repeat alignments Plus Strand',NULL,'A2513',3405,'MEM',43,7,NULL,'Andrew','2010-08-05 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2516,'Read Coverage unique alignments Minus Strand',NULL,'A2516',3405,'MEM',43,7,NULL,'Andrew','2010-08-05 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2518,'Read Coverage unique alignments Plus Strand',NULL,'A2518',3405,'MEM',43,7,NULL,'Andrew','2010-08-05 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2525,'Predicted novel miRNAs','','A2525',3405,'MEM',14,7,'','CairnsLab','2010-08-07 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2527,'Read Coverage 2-10 repeat alignments Minus Strand',NULL,'A2527',3405,'MEM',43,7,NULL,'Andrew','2010-08-10 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2528,'Read Coverage 2-10 repeat alignments Plus Strand',NULL,'A2528',3405,'MEM',43,7,NULL,'Andrew','2010-08-10 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2529,'Read Coverage unique alignments Minus Strand',NULL,'A2529',3405,'MEM',43,7,NULL,'Andrew','2010-08-10 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2530,'Read Coverage unique alignments Plus Strand',NULL,'A2530',3405,'MEM',43,7,NULL,'Andrew','2010-08-10 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2531,'Read Coverage 2-10 repeat alignments Minus Strand',NULL,'A2531',3405,'MEM',43,7,NULL,'Andrew','2010-08-10 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2532,'Read Coverage 2-10 repeat alignments Plus Strand',NULL,'A2532',3405,'MEM',43,7,NULL,'Andrew','2010-08-10 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2533,'Read Coverage unique alignments Minus Strand',NULL,'A2533',3405,'MEM',43,7,NULL,'Andrew','2010-08-10 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2534,'Read Coverage unique alignments Plus Strand',NULL,'A2534',3405,'MEM',43,7,NULL,'Andrew','2010-08-10 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2536,'2.1M_Human_exome','','A2536',3405,'MEM',51,37,'design target regions','Russell','2010-08-10 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2562,'7410X1_DIO_CONT','','A2562',3806,'MEM',40,30,'poly A selected RNA from age and gender matched mice to diet induced obese mice','tolley','2010-08-17 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2592,'7410X2_DIO','','A2592',3806,'MEM',40,30,'poly A selected RNA from platelets of diet induced obese mice','tolley','2010-08-17 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2593,'Affected 30085','','A2593',6300,'MEM',3,8,'Reads per million mapped that overlap a given base','u0028003','2010-08-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2594,'Affected 76591','','A2594',6300,'MEM',3,8,'Reads per million mapped that overlap a given base','u0028003','2010-08-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2595,'Unaffected 30088','','A2595',6300,'MEM',3,8,'Reads per million mapped that overlap a given base','u0028003','2010-08-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2596,'Unaffected 30090','','A2596',6300,'MEM',3,8,'Reads per million mapped that overlap a given base','u0028003','2010-08-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2597,'FDR','','A2597',6300,'MEM',3,8,'Affected vs Unaffected negative binomial p-values converted to B&H FDRs','u0028003','2010-08-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2598,'Log2 Ratio','','A2598',6300,'MEM',3,8,'Affected vs Unaffected normalized log2 ratios','u0028003','2010-08-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2599,'Gene Exons','','A2599',6300,'MEM',3,8,'Max FDR 10, min ratio 1.5x','u0028003','2010-08-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2600,'Gene Introns','','A2600',6300,'MEM',3,8,'Max FDR 10, min ratio 1.5x','u0028003','2010-08-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2601,'Read Coverage 2-10 repeat alignments Minus Strand',NULL,'A2601',3405,'MEM',43,7,NULL,'Andrew','2010-08-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2602,'Read Coverage 2-10 repeat alignments Plus Strand',NULL,'A2602',3405,'MEM',43,7,NULL,'Andrew','2010-08-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2603,'Read Coverage unique alignments Minus Strand',NULL,'A2603',3405,'MEM',43,7,NULL,'Andrew','2010-08-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2604,'Read Coverage unique alignments Plus Strand',NULL,'A2604',3405,'MEM',43,7,NULL,'Andrew','2010-08-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2605,'Read Coverage 2-10 repeat alignments Minus Strand',NULL,'A2605',3405,'MEM',43,7,NULL,'Andrew','2010-08-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2606,'Read Coverage 2-10 repeat alignments Plus Strand',NULL,'A2606',3405,'MEM',43,7,NULL,'Andrew','2010-08-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2607,'Read Coverage unique alignments Minus Strand',NULL,'A2607',3405,'MEM',43,7,NULL,'Andrew','2010-08-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2608,'Read Coverage unique alignments Plus Strand',NULL,'A2608',3405,'MEM',43,7,NULL,'Andrew','2010-08-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2609,'Read Coverage 2-10 repeat alignments Minus Strand','','A2609',3405,'MEM',43,7,'','Andrew','2010-08-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2610,'Read Coverage 2-10 repeat alignments Plus Strand',NULL,'A2610',3405,'MEM',43,7,NULL,'Andrew','2010-08-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2611,'Read Coverage unique repeat alignments Minus Strand',NULL,'A2611',3405,'MEM',43,7,NULL,'Andrew','2010-08-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2612,'Read Coverage unique alignments Plus Strand',NULL,'A2612',3405,'MEM',43,7,NULL,'Andrew','2010-08-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2613,'Read Coverage 2-10 repeat alignments Minus Strand',NULL,'A2613',3405,'MEM',43,7,NULL,'Andrew','2010-08-23 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2614,'Read Coverage 2-10 repeat alignments Plus Strand',NULL,'A2614',3405,'MEM',43,7,NULL,'Andrew','2010-08-23 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2615,'Read Coverage unique alignments Minus Strand',NULL,'A2615',3405,'MEM',43,7,NULL,'Andrew','2010-08-23 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2616,'Read Coverage unique alignments Plus Strand',NULL,'A2616',3405,'MEM',43,7,NULL,'Andrew','2010-08-23 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2617,'FDR','<P ALIGN=""LEFT"">No 30090 sample</P>','A2617',6300,'MEM',3,8,'Affected vs Unaffected negative binomial p-values converted to B&H FDRs','u0028003','2010-08-24 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2618,'Log2Ratio','<P ALIGN=""LEFT"">No 30090 sample</P>','A2618',6300,'MEM',3,8,'Affected vs Unaffected normalized log2 ratios','u0028003','2010-08-24 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2631,'7476X1','<P ALIGN=""LEFT"">platelets from male c57bl6 mice with floxed allele from dicer deleted.  paired end from trizol isolated RNA.</P>','A2631',3806,'MEM',40,30,'Dicer KO','tolley','2010-09-07 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2640,'7476X2','<P ALIGN=""LEFT"">platelet rNA (trizol) from littermate controls for dicer KO mice</P>','A2640',3806,'MEM',40,30,'dicer KO control','tolley','2010-09-07 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2778,'Problem file upload','','A2778',3806,'MEM',40,30,'','u0028003','2010-09-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2779,'Read Coverage 2-10 repeat alignments Minus Strand','','A2779',3806,'MEM',43,7,'','Andrew','2010-09-15 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2780,'Read Coverage 2-10 repeat alignments Plus Strand','','A2780',3806,'MEM',43,7,'','Andrew','2010-09-15 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2781,'Read Coverage unique alignments Minus Strand','','A2781',3806,'MEM',43,7,'','Andrew','2010-09-15 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2782,'Read Coverage unique alignments Plus Strand','','A2782',3806,'MEM',43,7,'','Andrew','2010-09-15 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2783,'Read Coverage 2-10 repeat alignments Minus Strand','','A2783',3806,'MEM',43,7,'','Andrew','2010-09-15 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2784,'Read Coverage 2-10 repeat alignments Plus Strand','','A2784',3806,'MEM',43,7,'','Andrew','2010-09-15 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2785,'Read Coverage unique alignments Minus Strand','','A2785',3806,'MEM',43,7,'','Andrew','2010-09-15 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2786,'Read Coverage unique alignments Plus Strand','','A2786',3806,'MEM',43,7,'','Andrew','2010-09-15 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2787,'Bdp1HeLaStruhl','','A2787',3405,'MEM',43,7,'','Andrew','2010-10-07 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2788,'Brf1HeLaStruhl','','A2788',3405,'MEM',43,7,'','Andrew','2010-10-07 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2789,'Brf2HeLaStruhl','','A2789',3405,'MEM',43,7,'','Andrew','2010-10-07 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2790,'Rpc155HeLaStruhl','','A2790',3405,'MEM',43,7,'','Andrew','2010-10-07 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2791,'TFIIICHeLaStruhl','','A2791',3405,'MEM',43,7,'','Andrew','2010-10-07 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2792,'Bdp1IMR90','','A2792',3405,'MEM',43,7,'','Andrew','2010-10-07 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2793,'Brf1IMR90',NULL,'A2793',3405,'MEM',43,7,NULL,'Andrew','2010-10-07 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2794,'Rpc4IMR90',NULL,'A2794',3405,'MEM',43,7,NULL,'Andrew','2010-10-07 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2795,'Brf1K562','','A2795',3405,'MEM',43,7,'','Andrew','2010-10-07 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2796,'Bdp1K562',NULL,'A2796',3405,'MEM',43,7,NULL,'Andrew','2010-10-07 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2797,'Rpc155K562','','A2797',3405,'MEM',43,7,'','Andrew','2010-10-07 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2798,'TFIIICK562','','A2798',3405,'MEM',43,7,'','Andrew','2010-10-07 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2799,'FosK562','','A2799',3405,'MEM',43,7,'','Andrew','2010-10-07 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2800,'Brf2K562','','A2800',3405,'MEM',43,7,'','Andrew','2010-10-07 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2801,'JunK562','','A2801',3405,'MEM',43,7,'','Andrew','2010-10-07 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2802,'Rpc32GM12878Snyder','','A2802',3405,'MEM',43,7,'','Andrew','2010-10-07 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2816,'IXO1_H3k27me3_QVAL','','A2816',3405,'MEM',14,7,'','CairnsLab','2010-10-15 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2817,'7396_7397X1LanesCombined_ReadCoverage1-10Minus','','A2817',3405,'MEM',14,7,'','CairnsLab','2010-10-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2818,'7396_7397X1LanesCombined_ReadCoverage1-10Positive','','A2818',3405,'MEM',14,7,'','CairnsLab','2010-10-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2819,'7396_7397X2LanesCombined_ReadCoverage1-10Positive_copy','','A2819',3405,'MEM',14,7,'','CairnsLab','2010-10-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2820,'7396_7397X2LanesCombined_ReadCoverage1-10Minus','','A2820',3405,'MEM',14,7,'','CairnsLab','2010-10-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2821,'Read Coverage 1-10 alignments Minus Strand',NULL,'A2821',3405,'MEM',43,7,NULL,'Andrew','2010-10-20 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2822,'Read Coverage 1-10 alignments Plus Strand',NULL,'A2822',3405,'MEM',43,7,NULL,'Andrew','2010-10-20 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2823,'Read Coverage 2-10 alignments Minus Strand','','A2823',3405,'MEM',43,7,'','Andrew','2010-10-20 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2824,'Read Coverage 2-10 alignments Plus Strand',NULL,'A2824',3405,'MEM',43,7,NULL,'Andrew','2010-10-20 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2825,'Read Coverage unique alignments Minus Strand',NULL,'A2825',3405,'MEM',43,7,NULL,'Andrew','2010-10-20 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2826,'Read Coverage unique alignments Plus Strand',NULL,'A2826',3405,'MEM',43,7,NULL,'Andrew','2010-10-20 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2827,'Read Coverage 1-10 alignments Minus Strand',NULL,'A2827',3405,'MEM',43,7,NULL,'Andrew','2010-10-20 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2828,'Read Coverage 1-10 alignments Plus Strand',NULL,'A2828',3405,'MEM',43,7,NULL,'Andrew','2010-10-20 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2829,'Read Coverage 2-10 alignments Minus Strand',NULL,'A2829',3405,'MEM',43,7,NULL,'Andrew','2010-10-20 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2830,'Read Coverage 2-10 alignments Plus Strand','','A2830',3405,'MEM',43,7,'','Andrew','2010-10-20 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2831,'Read Coverage unique alignments Minus Strand',NULL,'A2831',3405,'MEM',43,7,NULL,'Andrew','2010-10-20 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2832,'Read Coverage unique alignments Plus Strand',NULL,'A2832',3405,'MEM',43,7,NULL,'Andrew','2010-10-20 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2833,'Read Coverage 1-10 alignments Minus Strand','','A2833',3405,'MEM',43,7,'','Andrew','2010-10-20 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2834,'Read Coverage 1-10 alignments Plus Strand',NULL,'A2834',3405,'MEM',43,7,NULL,'Andrew','2010-10-20 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2835,'Read Coverage 2-10 alignments Minus Strand',NULL,'A2835',3405,'MEM',43,7,NULL,'Andrew','2010-10-20 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2836,'Read Coverage 2-10 alignments Plus Strand',NULL,'A2836',3405,'MEM',43,7,NULL,'Andrew','2010-10-20 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2837,'Read Coverage unique alignments Minus Strand',NULL,'A2837',3405,'MEM',43,7,NULL,'Andrew','2010-10-20 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2838,'Read Coverage unique alignments Plus Strand',NULL,'A2838',3405,'MEM',43,7,NULL,'Andrew','2010-10-20 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2839,'7001_28818 All SNPs','<P ALIGN=""LEFT"">Patient IDS = 7001, 1023001, 28818 </P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""></P>','A2839',6300,'MEM',30,8,'Unfiltered SNP calls from Schellenberg lab','u0028003','2010-10-21 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2840,'7001_28818 Novel SNPs No QC','<P ALIGN=""LEFT"">Patient IDS = 7001, 1023001, 28818</P><P ALIGN=""LEFT"">Only unique Het and Homo. ALT SNPs for this individual = NO dbSNPs </P><P ALIGN=""LEFT"">From Schellenberg lab </P>','A2840',6300,'MEM',30,8,'SNP filtered for dbSNP ','u0028003','2010-10-21 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2841,'7001_30085 All SNPs','<P ALIGN=""LEFT"">Patient IDS = 2_7001_1054001_30085 </P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""></P>','A2841',6300,'MEM',30,8,'Unfiltered SNP calls from Schellenberg lab','UARP','2010-10-21 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2842,'7001_30085 Novel SNPs No QC','<P ALIGN=""LEFT"">Patient IDS = 2_7001_1054001_30085</P><P ALIGN=""LEFT"">Only unique Het and Homo. ALT SNPs for this individual = NO dbSNPs </P><P ALIGN=""LEFT"">From Schellenberg lab </P>','A2842',6300,'MEM',30,8,'SNP filtered for dbSNP ','UARP','2010-10-21 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2843,'7001_63678 All SNPs','<P ALIGN=""LEFT"">Subject IDS = 3_7001_1306001_63678 </P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""></P>','A2843',6300,'MEM',30,8,'Unfiltered SNP calls from Schellenberg lab','UARP','2010-10-21 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2844,'7001_63678 Novel SNPs No QC','<P ALIGN=""LEFT"">Subject IDS = 3_7001_1306001_63678</P><P ALIGN=""LEFT"">Only unique Het and Homo. ALT SNPs for this individual = NO dbSNPs </P><P ALIGN=""LEFT"">From Schellenberg lab </P>','A2844',6300,'MEM',30,8,'SNP filtered for dbSNP ','UARP','2010-10-21 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2845,'Ets1 QValFDR',NULL,'A2845',3405,'MEM',43,7,NULL,'Andrew','2010-10-22 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2846,'CBP Jurkat',NULL,'A2846',3405,'MEM',43,7,NULL,'Andrew','2010-10-22 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2847,'7001_76591 All SNPs','<P ALIGN=""LEFT"">Subject IDS = 4_7001_1595001_76591 </P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""></P>','A2847',6300,'MEM',30,8,'Unfiltered SNP calls from Schellenberg lab','UARP','2010-10-26 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2849,'7001_76591 Novel SNPs No QC','<P ALIGN=""LEFT"">Subject IDS = 4_7001_1595001_76591</P><P ALIGN=""LEFT"">Only unique Het and Homo. ALT SNPs for this individual = NO dbSNPs </P><P ALIGN=""LEFT"">From Bernie&apos;s lab </P>','A2849',6300,'MEM',30,8,'SNP filtered for dbSNP ','UARP','2010-10-26 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2850,'10001_36913 All SNPs','<P ALIGN=""LEFT"">Subject IDS = 5_10001_4007001_36913</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""> </P>','A2850',6300,'MEM',30,8,'Unfiltered SNP calls from Schellenberg lab','UARP','2010-10-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2851,'10001_36913 Novel SNPs No QC','<P ALIGN=""LEFT"">Patient IDS = 10001_4007001_36913</P><P ALIGN=""LEFT"">Only unique Het and Homo. ALT SNPs for this individual = NO dbSNPs </P><P ALIGN=""LEFT"">From Schellenberg lab </P>','A2851',6300,'MEM',30,8,'SNP filtered for dbSNP ','UARP','2010-10-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2852,'10001_63577 Novel SNPs No QC','<P ALIGN=""LEFT"">Patient IDS = 6_10001_1298001_63577</P><P ALIGN=""LEFT"">Only unique Het and Homo. ALT SNPs for this individual = NO dbSNPs </P><P ALIGN=""LEFT"">From Schellenberg lab </P>','A2852',6300,'MEM',30,8,'SNP filtered for dbSNP ','UARP','2010-10-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2853,'10001_64049 Novel SNPs No QC','<P ALIGN=""LEFT"">Patient IDS = 7_10001_1329001_64049</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">From Schellenberg lab </P>','A2853',6300,'MEM',30,8,'SNP filtered for dbSNP ','UARP','2010-10-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2854,'10001_75506 Novel SNPs No QC','<P ALIGN=""LEFT"">Patient IDS = 8_10001_1581001_75506</P><P ALIGN=""LEFT"">Only unique Het and Homo. ALT SNPs for this individual = NO dbSNPs </P><P ALIGN=""LEFT"">From Schellenberg lab </P>','A2854',6300,'MEM',30,8,'SNP filtered for dbSNP ','UARP','2010-10-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2855,'10001_63577 All SNPs','<P ALIGN=""LEFT"">Subject IDS = 6_10001_1298001_63577</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""> </P>','A2855',6300,'MEM',30,8,'Unfiltered SNP calls from Schellenberg lab','UARP','2010-10-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2856,'10001_64049 All SNPs','<P ALIGN=""LEFT"">Subject IDS = 7_10001_1329001_64049</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""> </P>','A2856',6300,'MEM',30,8,'Unfiltered SNP calls from Schellenberg lab','UARP','2010-10-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2857,'10001_75506 All SNPs','<P ALIGN=""LEFT"">Subject IDS = 8_10001_1581001_75506</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""> </P>','A2857',6300,'MEM',30,8,'Unfiltered SNP calls from Schellenberg lab','UARP','2010-10-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2858,'H3K4me3 HeLa ','<P ALIGN=""LEFT"">From Robertson and Bilenki</P>','A2858',3405,'PUBLIC',43,7,'','Andrew','2010-10-29 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2859,'H3K4me1 HeLa','<P ALIGN=""LEFT"">Robertson and Bilenki</P>','A2859',3405,'PUBLIC',43,7,'','Andrew','2010-10-29 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2860,'Pol II HeLa','<P ALIGN=""LEFT"">PeakSeq enables systematic scoring of ChIP-seq experiments relative to controls.</P><P ALIGN=""LEFT"">    J Rozowsky, G Euskirchen, RK Auerbach, ZD Zhang, T Gibson, R Bjornson, N Carriero, M Snyder, MB Gerstein (2009) Nat Biotechnol 27: 66-75. </P>','A2860',3405,'PUBLIC',43,7,'','Andrew','2010-10-29 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2882,'7410x1_paired','','A2882',3806,'MEM',40,30,'mouse bl platelets -DIO cont - paired reads','tolley','2010-11-01 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2883,'X7FC_Histone_Score','','A2883',3405,'MEM',14,7,'','CairnsLab','2010-11-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2886,'IXO1_Histone_Score','','A2886',3405,'MEM',14,7,'','CairnsLab','2010-11-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2887,'WA1V_Histone_Score','','A2887',3405,'MEM',14,7,'','CairnsLab','2010-11-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2888,'JQ81_Histone_Score','','A2888',3405,'MEM',14,7,'','CairnsLab','2010-11-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2889,'ODN2_Histone_Score','','A2889',3405,'MEM',14,7,'','CairnsLab','2010-11-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2890,'GI36_Histone_Score','','A2890',3405,'MEM',14,7,'','CairnsLab','2010-11-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2891,'PTO4_Histone_Score','','A2891',3405,'MEM',14,7,'','CairnsLab','2010-11-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2892,'7410X6_proplt_megs','','A2892',3806,'MEM',40,30,'from mouse fetal liver megs - italiano lab','tolley','2010-11-17 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2893,'7410X7_round_megs','','A2893',3806,'MEM',40,30,'mouse fetal liver megs from italiano - round fraction','tolley','2010-11-17 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2895,'Mouse chr1 UCSC kg',NULL,'A2895',3806,'MEM',54,40,NULL,'Moore','2010-11-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2896,'HiSeq_7741X1_BL_Platelet','','A2896',6300,'MEM',40,30,'Male human platelets 50 cycle single end sequencing on HiSeq','tolley','2010-11-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2897,'HiSeq_7741X2_THR_Platelet','<P ALIGN=""LEFT"">thrombin stimulated</P>','A2897',6300,'MEM',40,30,'Male platelets 50 cycle sequencing on hi seq','tolley','2010-11-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2898,'HiSeq_7623X1_BL_Platelet','<P ALIGN=""LEFT"">baseline platelets</P>','A2898',6300,'MEM',40,30,'male platelets 50 cycle seq on hiseq','tolley','2010-11-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2899,'HiSeq_7623X6_THR_Platelet','<P ALIGN=""LEFT"">thrombin stim</P>','A2899',6300,'MEM',40,30,'male 50 cycle on hiseq ','tolley','2010-11-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2902,'novoalign',NULL,'A2902',6300,'MEM',43,7,NULL,'Andrew','2010-11-30 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2903,'tophat_seg_18bp',NULL,'A2903',6300,'MEM',43,7,NULL,'Andrew','2010-11-30 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2904,'tophat_seg_16bp',NULL,'A2904',6300,'MEM',43,7,NULL,'Andrew','2010-11-30 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2905,'novoalign',NULL,'A2905',3806,'MEM',43,7,NULL,'Andrew','2010-11-30 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2906,'tophat_seg_18bp',NULL,'A2906',3806,'MEM',43,7,NULL,'Andrew','2010-11-30 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2907,'tophat_seg_16bp',NULL,'A2907',3806,'MEM',43,7,NULL,'Andrew','2010-11-30 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2908,'DNaseSeq_usingScanSeq1000bp','','A2908',6300,'MEM',14,7,'','CairnsLab','2010-12-06 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2909,'DNase5000bp','','A2909',6300,'MEM',14,7,'','CairnsLab','2010-12-06 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2910,'chrNA Spacers','','A2910',6303,'PUBLIC',3,28,'1kb N spacers between concatinated NA sequences','u0028003','2010-12-06 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2911,'chrScaffold Spacers','','A2911',6303,'PUBLIC',3,28,'1kb N spacers between concatinated NA scaffolds','u0028003','2010-12-06 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2912,'chrNA Segments','','A2912',6303,'PUBLIC',NULL,28,'Concatinated NA Segments','u0028003','2010-12-06 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2913,'chrScaffold Segments','','A2913',6303,'PUBLIC',NULL,28,'Concatinated Scaffold Segments','u0028003','2010-12-06 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2917,'chr1','','A2917',6300,'MEM',14,7,'','CairnsLab','2010-12-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2918,'chr2','','A2918',6300,'MEM',14,7,'','CairnsLab','2010-12-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2919,'chr3','','A2919',6300,'MEM',14,7,'','CairnsLab','2010-12-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2920,'chr4','','A2920',6300,'MEM',14,7,'','CairnsLab','2010-12-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2921,'chr5','','A2921',6300,'MEM',14,7,'','CairnsLab','2010-12-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2922,'chr6','','A2922',6300,'MEM',14,7,'','CairnsLab','2010-12-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2923,'chr7','','A2923',6300,'MEM',14,7,'','CairnsLab','2010-12-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2924,'chr8','','A2924',6300,'MEM',14,7,'','CairnsLab','2010-12-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2925,'chr9','','A2925',6300,'MEM',14,7,'','CairnsLab','2010-12-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2926,'chr10','','A2926',6300,'MEM',14,7,'','CairnsLab','2010-12-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2927,'chr11','','A2927',6300,'MEM',14,7,'','CairnsLab','2010-12-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2928,'chr12','','A2928',6300,'MEM',14,7,'','CairnsLab','2010-12-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2929,'chr13','','A2929',6300,'MEM',14,7,'','CairnsLab','2010-12-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2930,'chr14','','A2930',6300,'MEM',14,7,'','CairnsLab','2010-12-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2931,'chr15','','A2931',6300,'MEM',14,7,'','CairnsLab','2010-12-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2932,'chr16',NULL,'A2932',6300,'MEM',14,7,NULL,'CairnsLab','2010-12-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2933,'chr17',NULL,'A2933',6300,'MEM',14,7,NULL,'CairnsLab','2010-12-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2934,'chr18',NULL,'A2934',6300,'MEM',14,7,NULL,'CairnsLab','2010-12-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2935,'chr19',NULL,'A2935',6300,'MEM',14,7,NULL,'CairnsLab','2010-12-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2936,'chr20',NULL,'A2936',6300,'MEM',14,7,NULL,'CairnsLab','2010-12-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2937,'chr21',NULL,'A2937',6300,'MEM',14,7,NULL,'CairnsLab','2010-12-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2938,'chr22\r',NULL,'A2938',6300,'MEM',14,7,NULL,'CairnsLab','2010-12-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2939,'chrX',NULL,'A2939',6300,'MEM',14,7,NULL,'CairnsLab','2010-12-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2940,'chrY',NULL,'A2940',6300,'MEM',14,7,NULL,'CairnsLab','2010-12-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2941,'DukeDNAseNormInput1000bp(QVAL)','','A2941',3405,'MEM',14,7,'','CairnsLab','2010-12-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2942,'DukeDNaseNoInput1000(Sum)',NULL,'A2942',3405,'MEM',14,7,NULL,'CairnsLab','2010-12-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2943,'FAIRE_D264_1000(SUM)',NULL,'A2943',3405,'MEM',14,7,NULL,'CairnsLab','2010-12-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2944,'FAIRE_D265_1000(SUM)',NULL,'A2944',3405,'MEM',14,7,NULL,'CairnsLab','2010-12-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2946,'7550X1_B20AAHABXX','','A2946',6300,'MEM',3,41,'1st HiSeq paired 101bp alignments','u0028003','2010-12-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2947,'2x75 ReadCoverage','','A2947',3405,'MEM',43,7,'','Andrew','2010-12-13 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2949,'1x75d ReadCoverage Plus',NULL,'A2949',3405,'MEM',43,7,NULL,'Andrew','2010-12-13 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2950,'1x75d ReadCoverage Minus',NULL,'A2950',3405,'MEM',43,7,NULL,'Andrew','2010-12-13 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2952,'7550X2_B20AAHABXX','','A2952',6300,'MEM',3,41,'1st HiSeq paired 101bp alignments','u0028003','2010-12-14 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2953,'7550X3_B20AAHABXX','','A2953',6300,'MEM',3,41,'1st HiSeq paired 101bp alignment run','u0028003','2010-12-14 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2954,'7550X4_B20AAHABXX','','A2954',6300,'MEM',3,41,'1st HiSeq paired 101bp alignment run','u0028003','2010-12-15 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2957,'D264_FAIRE_NormalizedtoInput','','A2957',6300,'MEM',14,7,'','CairnsLab','2010-12-17 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2958,'D265_FAIRE_NormalizedtoInput','','A2958',6300,'MEM',14,7,'','CairnsLab','2010-12-17 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2959,'D264_FAIRE_NOInput_1000bp','','A2959',6300,'MEM',14,7,'','CairnsLab','2010-12-17 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2960,'D265_NoINput_1000bp','','A2960',6300,'MEM',14,7,'','CairnsLab','2010-12-17 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2962,'7550X5_B20AAHABXX','','A2962',6300,'MEM',3,41,'1st HiSeq paired 101bp alignment run','u0028003','2010-12-17 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2967,'DukeDNAseNormInput','','A2967',6300,'MEM',14,7,'','CairnsLab','2010-12-23 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2968,'wEmbH3925a','','A2968',1204,'MEM',NULL,7,'.bam / .bam.bai files for sample 7761X1.','bmilash','2010-12-30 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2969,'wEmbHyb1107','','A2969',1204,'MEM',NULL,7,'.bam / .bam.bai files for sample 7776X1','bmilash','2010-12-30 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2970,'zEmb4hHs106','','A2970',6301,'MEM',NULL,7,'.bam / .bam.bai files for sample 7728X2.','bmilash','2010-12-30 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2978,'BinPVal','','A2978',3405,'MEM',19,13,'','GravesLab','2011-01-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2979,'QValFDR','','A2979',3405,'MEM',19,13,'','GravesLab','2011-01-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2980,'EmpFDR','','A2980',3405,'MEM',19,13,'','GravesLab','2011-01-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2981,'BinPValDR',NULL,'A2981',3405,'MEM',19,13,NULL,'GravesLab','2011-01-13 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2982,'EmpFDRDR',NULL,'A2982',3405,'MEM',19,13,NULL,'GravesLab','2011-01-13 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2983,'QValFDRDR',NULL,'A2983',3405,'MEM',19,13,NULL,'GravesLab','2011-01-13 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2987,'pitx1 fdr','','A2987',3806,'MEM',29,24,'','MoonLab','2011-01-13 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(2988,'pitx1  log2','','A2988',3806,'MEM',29,24,'','MoonLab','2011-01-13 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/');

INSERT INTO dbo.gpAnnotation 
(
idAnnotation,
name,
description,
fileName,
idGenomeVersion,
codeVisibility,
idUser,
idUserGroup,
summary,
createdBy,
createDate,
isLoaded,
idInstitute,
dataPath
)
VALUES 
(2989,'top peak coordinates',NULL,'A2989',3806,'MEM',29,24,NULL,'MoonLab','2011-01-13 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3001,'Ensembl protein genes','<P ALIGN=""LEFT"">From Tim Parnel who extracted the UCSC table formatted file from an ensembl gff3 download.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">chrMT annotations are renamed chrM according to UCSC standards</P>','A3001',6303,'PUBLIC',3,28,'Protein coding genes from Ensembl','u0028003','2011-01-14 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3002,'Ensembl non coding genes','<P ALIGN=""LEFT"">From Tim Parnel who extracted the UCSC table formatted file from an ensembl gff3 download.</P>','A3002',6303,'PUBLIC',3,28,'Non coding genes from Ensembl','u0028003','2011-01-14 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3014,'ES Cells Non Methylated','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A201</P>','A3014',3405,'PUBLIC',3,7,'Number of Converted Cs Sequenced at each genomic C','u0028003','2011-01-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3015,'ES Cells Methylated','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A201</P>','A3015',3405,'PUBLIC',3,7,'Number of Non Converted Cs Sequenced at each genomic C','u0028003','2011-01-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3016,'ES Fibroblasts Non Methylated','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A201</P>','A3016',3405,'PUBLIC',3,7,'Number of Converted Cs Sequenced at each genomic C','u0028003','2011-01-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3017,'ES Fibroblasts Methylated','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A201</P>','A3017',3405,'PUBLIC',3,7,'Number of Non Converted Cs Sequenced at each genomic C','u0028003','2011-01-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3018,'Foreskin Fibroblasts Methylated','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A201</P>','A3018',3405,'PUBLIC',3,7,'Number of Non Converted Cs Sequenced at each genomic C','u0028003','2011-01-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3019,'Foreskin Fibroblasts Non Methylated','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A201</P>','A3019',3405,'PUBLIC',3,7,'Number of Converted Cs Sequenced at each genomic C','u0028003','2011-01-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3041,'concatinated PseHM','<P ALIGN=""LEFT"">Pseudo median relative difference window score</P>','A3041',2701,'PUBLIC',14,7,'','CairnsLab','2011-01-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3042,'concatinated Oligo','','A3042',2701,'PUBLIC',14,7,'','CairnsLab','2011-01-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3043,'concatinated Oligo','','A3043',2701,'PUBLIC',14,7,'','CairnsLab','2011-01-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3044,'concatinated PseHM','<P ALIGN=""LEFT"">Pseudo median relative difference window score</P>','A3044',2701,'PUBLIC',14,7,'','CairnsLab','2011-01-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3045,'concatinated Oligo','','A3045',2701,'PUBLIC',14,7,'','CairnsLab','2011-01-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3046,'concatinated PseHM','<P ALIGN=""LEFT"">Pseudo median relative difference window score</P>','A3046',2701,'PUBLIC',14,7,'','CairnsLab','2011-01-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3047,'concatinated Oligo','','A3047',2701,'PUBLIC',14,7,'','CairnsLab','2011-01-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3048,'concatinated PseHM','<P ALIGN=""LEFT"">Pseudo median relative difference window score</P>','A3048',2701,'PUBLIC',14,7,'','CairnsLab','2011-01-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3052,'concatinated Oligo','','A3052',2701,'PUBLIC',14,7,'','CairnsLab','2011-01-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3053,'concatinated PseHM','<P ALIGN=""LEFT"">Pseudo median relative difference window score</P>','A3053',2701,'PUBLIC',14,7,'','CairnsLab','2011-01-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3054,'concatinated Oligo','','A3054',2701,'PUBLIC',14,7,'','CairnsLab','2011-01-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3055,'concatinated PseHM','<P ALIGN=""LEFT"">Pseudo median relative difference window score</P>','A3055',2701,'PUBLIC',14,7,'','CairnsLab','2011-01-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3056,'concatinated Oligo','','A3056',2701,'PUBLIC',14,7,'','CairnsLab','2011-01-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3057,'concatinated PseHM','<P ALIGN=""LEFT"">Pseudo median relative difference window score</P>','A3057',2701,'PUBLIC',14,7,'','CairnsLab','2011-01-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3060,'concatinated Oligo','','A3060',2701,'PUBLIC',14,7,'','CairnsLab','2011-01-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3061,'concatinated PseHM','<P ALIGN=""LEFT"">Pseudo median relative difference window score</P>','A3061',2701,'PUBLIC',14,7,'','CairnsLab','2011-01-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3062,'concatinated Oligo','','A3062',2701,'PUBLIC',14,7,'','CairnsLab','2011-01-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3063,'concatinated PseHM','<P ALIGN=""LEFT"">Pseudo median relative difference window score</P>','A3063',2701,'PUBLIC',14,7,'','CairnsLab','2011-01-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3076,'Binomial PValue','','A3076',3806,'MEM',3,24,'','u0028003','2011-01-21 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3077,'Input',NULL,'A3077',3806,'MEM',NULL,24,NULL,'u0028003','2011-01-21 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3078,'ChIP',NULL,'A3078',3806,'MEM',NULL,24,NULL,'u0028003','2011-01-21 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3079,'QValue FDR','','A3079',3806,'MEM',3,24,'','u0028003','2011-01-21 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3080,'Empirical FDR','','A3080',3806,'MEM',3,24,'','u0028003','2011-01-21 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3081,'Top 48 Peaks',NULL,'A3081',3806,'MEM',NULL,24,NULL,'u0028003','2011-01-21 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3082,'Input',NULL,'A3082',3806,'MEM',NULL,24,NULL,'u0028003','2011-01-21 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3083,'ChIP',NULL,'A3083',3806,'MEM',NULL,24,NULL,'u0028003','2011-01-21 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3098,'Chr15 H3M ReadCoverage',NULL,'A3098',3806,'MEM',NULL,24,NULL,'u0028003','2011-01-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3099,'Chr15 Input ReadCoverage',NULL,'A3099',3806,'MEM',NULL,24,NULL,'u0028003','2011-01-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3100,'Chr15 ETV5 ReadCoverage',NULL,'A3100',3806,'MEM',NULL,24,NULL,'u0028003','2011-01-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3101,'Early Embryo',NULL,'A3101',1204,'MEM',NULL,7,NULL,'bmilash','2011-01-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3102,'Input','','A3102',3806,'MEM',3,24,'','u0028003','2011-01-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3103,'ETV5','','A3103',3806,'MEM',3,24,'','u0028003','2011-01-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3104,'H3M','','A3104',3806,'MEM',3,24,'','u0028003','2011-01-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3105,'H3M Binomisl PValue',NULL,'A3105',3806,'MEM',NULL,24,NULL,'u0028003','2011-01-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3106,'H3M Empirical FDR',NULL,'A3106',3806,'MEM',NULL,24,NULL,'u0028003','2011-01-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3107,'H3M QValue FDR',NULL,'A3107',3806,'MEM',NULL,24,NULL,'u0028003','2011-01-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3108,'ETV5 Binomial PValue',NULL,'A3108',3806,'MEM',NULL,24,NULL,'u0028003','2011-01-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3109,'ETV5 Empirical FDR',NULL,'A3109',3806,'MEM',NULL,24,NULL,'u0028003','2011-01-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3110,'ETV5 QValue FDR',NULL,'A3110',3806,'MEM',NULL,24,NULL,'u0028003','2011-01-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3111,'H3M top 43 peaks ',NULL,'A3111',3806,'MEM',NULL,24,NULL,'u0028003','2011-01-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3112,'ETV5 top 43 peaks','','A3112',3806,'MEM',NULL,24,'','u0028003','2011-01-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3115,'FibFore_IRM90','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3115',3405,'MEM',3,7,'Enriched methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3116,'X1_FibFore','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3116',3405,'MEM',3,7,'Enriched methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3117,'X1X2_H1','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3117',3405,'MEM',3,7,'Enriched methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3118,'X2_H1','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3118',3405,'MEM',3,7,'Enriched methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3119,'ES_FibFore','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3119',3405,'MEM',3,7,'Enriched methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3120,'H1_FibES','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3120',3405,'MEM',3,7,'Enriched methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3121,'X1_H1','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3121',3405,'MEM',3,7,'Enriched methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3122,'X1X2__IRM90FibFore','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3122',3405,'MEM',3,7,'Enriched methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3123,'X2_IRM90FibFore','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3123',3405,'MEM',3,7,'Enriched methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3124,'ES_H1','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3124',3405,'MEM',3,7,'Enriched methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3125,'H1_FibFore','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3125',3405,'MEM',3,7,'Enriched methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3126,'X1_IRM90FibFore','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3126',3405,'MEM',3,7,'Enriched methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3127,'X1X2_IRM90','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3127',3405,'MEM',3,7,'Enriched methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3128,'X2_IRM90','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3128',3405,'MEM',3,7,'Enriched methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3129,'ES_IRM90FibFore','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3129',3405,'MEM',3,7,'Enriched methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3130,'H1_IRM90FibFore','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3130',3405,'MEM',3,7,'Enriched methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3131,'X1_IRM90','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3131',3405,'MEM',3,7,'Enriched methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3132,'X1_X2','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3132',3405,'MEM',3,7,'Enriched methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3133,'ES_IRM90','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3133',3405,'MEM',3,7,'Enriched methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3134,'H1_IRM90','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3134',3405,'MEM',3,7,'Enriched methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3135,'X1X2_ES','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3135',3405,'MEM',3,7,'Enriched methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3136,'X2_ES','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3136',3405,'MEM',3,7,'Enriched methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3137,'FibES_FibFore','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3137',3405,'MEM',3,7,'Enriched methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3138,'X1_ES','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3138',3405,'MEM',3,7,'Enriched methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3139,'X1X2_FibES','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3139',3405,'MEM',3,7,'Enriched methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3140,'X2_FibES','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3140',3405,'MEM',3,7,'Enriched methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3141,'FibES_IRM90','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3141',3405,'MEM',3,7,'Enriched methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3142,'X1_FibES','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3142',3405,'MEM',3,7,'Enriched methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3143,'X1X2_FibFore','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3143',3405,'MEM',3,7,'Enriched methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3144,'X2_FibFore','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3144',3405,'MEM',3,7,'Enriched methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3145,'ES_FibES','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3145',3405,'MEM',3,7,'Log2Ratio for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3146,'FibFore_IRM90','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3146',3405,'MEM',3,7,'Log2Ratio for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3147,'X1_FibFore','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3147',3405,'MEM',3,7,'Log2Ratio for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3148,'X1X2_H1','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3148',3405,'MEM',3,7,'Log2Ratio for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3149,'X2_H1','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3149',3405,'MEM',3,7,'Log2Ratio for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3150,'ES_FibFore','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3150',3405,'MEM',3,7,'Log2Ratio for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3151,'H1_FibES','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3151',3405,'MEM',3,7,'Log2Ratio for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3152,'X1_H1','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3152',3405,'MEM',3,7,'Log2Ratio for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3153,'X1X2__IRM90FibFore','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3153',3405,'MEM',3,7,'Log2Ratio for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3154,'X2_IRM90FibFore','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3154',3405,'MEM',3,7,'Log2Ratio for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3155,'ES_H1','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3155',3405,'MEM',3,7,'Log2Ratio for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3156,'H1_FibFore','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3156',3405,'MEM',3,7,'Log2Ratio for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3157,'X1_IRM90FibFore','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3157',3405,'MEM',3,7,'Log2Ratio for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3158,'X1X2_IRM90','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3158',3405,'MEM',3,7,'Log2Ratio for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3159,'X2_IRM90','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3159',3405,'MEM',3,7,'Log2Ratio for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3160,'ES_IRM90FibFore','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3160',3405,'MEM',3,7,'Log2Ratio for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3161,'H1_IRM90FibFore','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3161',3405,'MEM',3,7,'Log2Ratio for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3162,'X1_IRM90','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3162',3405,'MEM',3,7,'Log2Ratio for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3163,'X1_X2','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3163',3405,'MEM',3,7,'Log2Ratio for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3164,'ES_IRM90','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3164',3405,'MEM',3,7,'Log2Ratio for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3165,'H1_IRM90','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3165',3405,'MEM',3,7,'Log2Ratio for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3166,'X1X2_ES','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3166',3405,'MEM',3,7,'Log2Ratio for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3167,'X2_ES','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3167',3405,'MEM',3,7,'Log2Ratio for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3168,'FibES_FibFore','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3168',3405,'MEM',3,7,'Log2Ratio for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3169,'X1_ES','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3169',3405,'MEM',3,7,'Log2Ratio for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3170,'X1X2_FibES','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3170',3405,'MEM',3,7,'Log2Ratio for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3171,'X2_FibES','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3171',3405,'MEM',3,7,'Log2Ratio for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3172,'FibES_IRM90','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3172',3405,'MEM',3,7,'Log2Ratio for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3173,'X1_FibES','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3173',3405,'MEM',3,7,'Log2Ratio for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3174,'X1X2_FibFore','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3174',3405,'MEM',3,7,'Log2Ratio for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3175,'X2_FibFore','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3175',3405,'MEM',3,7,'Log2Ratio for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3176,'ES_FibES','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3176',3405,'MEM',3,7,'Uncorrected -10Log10(Pvalue) for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3177,'FibFore_IRM90','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3177',3405,'MEM',3,7,'Uncorrected -10Log10(Pvalue) for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3178,'X1_FibFore','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3178',3405,'MEM',3,7,'Uncorrected -10Log10(Pvalue) for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3179,'X1X2_H1','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3179',3405,'MEM',3,7,'Uncorrected -10Log10(Pvalue) for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3180,'X2_H1','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3180',3405,'MEM',3,7,'Uncorrected -10Log10(Pvalue) for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3181,'ES_FibFore','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3181',3405,'MEM',3,7,'Uncorrected -10Log10(Pvalue) for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3182,'H1_FibES','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3182',3405,'MEM',3,7,'Uncorrected -10Log10(Pvalue) for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3183,'X1_H1','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3183',3405,'MEM',3,7,'Uncorrected -10Log10(Pvalue) for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3184,'X1X2__IRM90FibFore','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3184',3405,'MEM',3,7,'Uncorrected -10Log10(Pvalue) for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3185,'X2_IRM90FibFore','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3185',3405,'MEM',3,7,'Uncorrected -10Log10(Pvalue) for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3186,'ES_H1','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3186',3405,'MEM',3,7,'Uncorrected -10Log10(Pvalue) for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3187,'H1_FibFore','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3187',3405,'MEM',3,7,'Uncorrected -10Log10(Pvalue) for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3188,'X1_IRM90FibFore','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3188',3405,'MEM',3,7,'Uncorrected -10Log10(Pvalue) for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3189,'X1X2_IRM90','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3189',3405,'MEM',3,7,'Uncorrected -10Log10(Pvalue) for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3190,'X2_IRM90','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3190',3405,'MEM',3,7,'Uncorrected -10Log10(Pvalue) for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3191,'ES_IRM90FibFore','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3191',3405,'MEM',3,7,'Uncorrected -10Log10(Pvalue) for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3192,'H1_IRM90FibFore','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3192',3405,'MEM',3,7,'Uncorrected -10Log10(Pvalue) for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3193,'X1_IRM90','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3193',3405,'MEM',3,7,'Uncorrected -10Log10(Pvalue) for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3194,'X1_X2','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3194',3405,'MEM',3,7,'Uncorrected -10Log10(Pvalue) for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3195,'ES_IRM90','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3195',3405,'MEM',3,7,'Uncorrected -10Log10(Pvalue) for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3196,'H1_IRM90','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3196',3405,'MEM',3,7,'Uncorrected -10Log10(Pvalue) for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3197,'X1X2_ES','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3197',3405,'MEM',3,7,'Uncorrected -10Log10(Pvalue) for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3198,'X2_ES','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3198',3405,'MEM',3,7,'Uncorrected -10Log10(Pvalue) for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3199,'FibES_FibFore','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3199',3405,'MEM',3,7,'Uncorrected -10Log10(Pvalue) for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3200,'X1_ES','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3200',3405,'MEM',3,7,'Uncorrected -10Log10(Pvalue) for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3201,'X1X2_FibES','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3201',3405,'MEM',3,7,'Uncorrected -10Log10(Pvalue) for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3202,'X2_FibES','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3202',3405,'MEM',3,7,'Uncorrected -10Log10(Pvalue) for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3203,'FibES_IRM90','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3203',3405,'MEM',3,7,'Uncorrected -10Log10(Pvalue) for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3204,'X1_FibES','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3204',3405,'MEM',3,7,'Uncorrected -10Log10(Pvalue) for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3205,'X1X2_FibFore','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3205',3405,'MEM',3,7,'Uncorrected -10Log10(Pvalue) for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3206,'X2_FibFore','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3206',3405,'MEM',3,7,'Uncorrected -10Log10(Pvalue) for differential methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3207,'ES_FibES','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3207',3405,'MEM',3,7,'Reduced methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3208,'FibFore_IRM90','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3208',3405,'MEM',3,7,'Reduced methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3209,'X1_FibFore','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3209',3405,'MEM',3,7,'Reduced methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3210,'X1X2_H1','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3210',3405,'MEM',3,7,'Reduced methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3211,'X2_H1','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3211',3405,'MEM',3,7,'Reduced methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3212,'ES_FibFore','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3212',3405,'MEM',3,7,'Reduced methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3213,'H1_FibES','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3213',3405,'MEM',3,7,'Reduced methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3214,'X1_H1','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3214',3405,'MEM',3,7,'Reduced methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3215,'X1X2__IRM90FibFore','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3215',3405,'MEM',3,7,'Reduced methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3216,'X2_IRM90FibFore','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3216',3405,'MEM',3,7,'Reduced methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3217,'ES_H1','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3217',3405,'MEM',3,7,'Reduced methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3218,'H1_FibFore','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3218',3405,'MEM',3,7,'Reduced methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3219,'X1_IRM90FibFore','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3219',3405,'MEM',3,7,'Reduced methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3220,'X1X2_IRM90','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3220',3405,'MEM',3,7,'Reduced methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3221,'X2_IRM90','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3221',3405,'MEM',3,7,'Reduced methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3222,'ES_IRM90FibFore','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3222',3405,'MEM',3,7,'Reduced methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3223,'H1_IRM90FibFore','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3223',3405,'MEM',3,7,'Reduced methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3224,'X1_IRM90','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3224',3405,'MEM',3,7,'Reduced methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3225,'X1_X2','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3225',3405,'MEM',3,7,'Reduced methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3226,'ES_IRM90','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3226',3405,'MEM',3,7,'Reduced methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3227,'H1_IRM90','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3227',3405,'MEM',3,7,'Reduced methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3228,'X1X2_ES','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3228',3405,'MEM',3,7,'Reduced methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3229,'X2_ES','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3229',3405,'MEM',3,7,'Reduced methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3230,'FibES_FibFore','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3230',3405,'MEM',3,7,'Reduced methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3231,'X1_ES','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3231',3405,'MEM',3,7,'Reduced methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3232,'X1X2_FibES','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3232',3405,'MEM',3,7,'Reduced methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3233,'X2_FibES','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3233',3405,'MEM',3,7,'Reduced methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3234,'FibES_IRM90','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3234',3405,'MEM',3,7,'Reduced methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3235,'X1_FibES','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3235',3405,'MEM',3,7,'Reduced methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3236,'X1X2_FibFore','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3236',3405,'MEM',3,7,'Reduced methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3237,'X2_FibFore','BisSeq differential methylation analysis, 1.5kb windows, 3x diff, 0.001 FDR, see GNomEx Analysis Reports A201, A203, A207','A3237',3405,'MEM',3,7,'Reduced methylated regions, 3x diff, 0.001 FDR',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3250,'7795X1 Methylated','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A207</P>','A3250',3405,'MEM',3,7,'Number of Non Converted Cs Sequenced at each genomic C','u0028003','2011-01-31 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3251,'7795X1 Non Methylated','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A207</P>','A3251',3405,'MEM',3,7,'Number of Converted Cs Sequenced at each genomic C','u0028003','2011-01-31 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3252,'7795X2 Methylated','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A207</P>','A3252',3405,'MEM',3,7,'Number of Non Converted Cs Sequenced at each genomic C','u0028003','2011-01-31 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3253,'7795X2 Non Methylated','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A207</P>','A3253',3405,'MEM',3,7,'Number of Converted Cs Sequenced at each genomic C','u0028003','2011-01-31 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3254,'H1 Methylated','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A203</P>','A3254',3405,'PUBLIC',3,7,'Number of Non Converted Cs Sequenced at each genomic C','u0028003','2011-01-31 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3257,'IRM90 Methylated','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A203</P>','A3257',3405,'PUBLIC',3,7,'Number of Non Converted Cs Sequenced at each genomic C','u0028003','2011-01-31 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3258,'Hypo 1 Alpha','','A3258',3806,'MEM',3,35,'Relative read coverage, 4.7m reads','u0028003','2011-02-01 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3259,'Hypo 2 Alpha','','A3259',3806,'MEM',3,35,'Relative read coverage, 7.1m reads','u0028003','2011-02-01 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3260,'Input','','A3260',3806,'MEM',3,35,'Relative read coverage, 5.8m reads','u0028003','2011-02-01 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3261,'Hypo 1 Alpha vs Input Binomial PValue','','A3261',3806,'MEM',3,35,'','u0028003','2011-02-01 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3262,'Hypo 2 Alpha vs Input Binomial PValue','','A3262',3806,'MEM',3,35,'','u0028003','2011-02-01 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3263,'Hypo 1 Alpha vs Hypo 2 Alpha Binomial PValue','','A3263',3806,'MEM',3,35,'','u0028003','2011-02-01 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3264,'Hypo 1 Alpha vs Input QValue FDR','','A3264',3806,'MEM',3,35,'','u0028003','2011-02-01 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3265,'Hypo 1 Alpha vs Input Emp FDR','','A3265',3806,'MEM',3,35,'','u0028003','2011-02-01 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3266,'Hypo 2 Alpha vs Input QValue FDR','','A3266',3806,'MEM',3,35,'','u0028003','2011-02-01 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3267,'Hypo 2 Alpha vs Input Emp FDR','','A3267',3806,'MEM',3,35,'','u0028003','2011-02-01 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3268,'Hypo 1 Alpha vs Hypo 2 Alpha QValue FDR','','A3268',3806,'MEM',3,35,'','u0028003','2011-02-01 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3269,'Hypo 1 Alpha vs Input Top 100','','A3269',3806,'MEM',3,35,'','u0028003','2011-02-01 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3270,'Hypo 2 Alpha vs Input Top 100','','A3270',3806,'MEM',3,35,'','u0028003','2011-02-01 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3271,'Hypo 1 Alpha vs Hypo 2 Alpha Enriched','','A3271',3806,'MEM',3,35,'','u0028003','2011-02-01 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3272,'Hypo 1 Alpha vs Hypo 2 Alpha Reduced','','A3272',3806,'MEM',3,35,'','u0028003','2011-02-01 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3275,'H1 NonMethylated','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A203</P>','A3275',3405,'PUBLIC',3,7,'Number of Converted Cs Sequenced at each genomic C',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3276,'IRM90 NonMethylated','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A203</P>','A3276',3405,'PUBLIC',3,7,'Number of Converted Cs Sequenced at each genomic C',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3286,'Log2Ratio','<P ALIGN=""LEFT"">From StrandedBisSeq</P>','A3286',3405,'PUBLIC',3,7,'Fraction plus strand methylation/ fraction minus',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3287,'PValue','<P ALIGN=""LEFT"">From StrandedBisSeq</P>','A3287',3405,'PUBLIC',3,7,'-10Log10(fischer/chi-square pvalue), uncorrected',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3288,'Minus Regions, 2x, 0.01 FDR','<P ALIGN=""LEFT"">From StrandedBisSeq</P>','A3288',3405,'PUBLIC',3,7,'Regions enriched for minus stranded methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3289,'Plus Regions, 2x, 0.01 FDR','<P ALIGN=""LEFT"">From StrandedBisSeq</P>','A3289',3405,'PUBLIC',3,7,'Regions enriched for plus stranded methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3290,'Minus Regions, 2x, 0.01 FDR','<P ALIGN=""LEFT"">From StrandedBisSeq</P>','A3290',3405,'PUBLIC',3,7,'Regions enriched for minus stranded methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3291,'Plus Regions, 2x, 0.01 FDR','<P ALIGN=""LEFT"">From StrandedBisSeq</P>','A3291',3405,'PUBLIC',3,7,'Regions enriched for plus stranded methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3292,'Log2Ratio','<P ALIGN=""LEFT"">From StrandedBisSeq</P>','A3292',3405,'PUBLIC',3,7,'Fraction plus strand methylation/ fraction minus',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3293,'PValue','<P ALIGN=""LEFT"">From StrandedBisSeq</P>','A3293',3405,'PUBLIC',3,7,'-10Log10(fischer/chi-square pvalue), uncorrected',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3294,'Log2Ratio','<P ALIGN=""LEFT"">From StrandedBisSeq, Neonatal foreskin fibroblasts</P>','A3294',3405,'PUBLIC',3,7,'Fraction plus strand methylation/ fraction minus',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3295,'PValue','<P ALIGN=""LEFT"">From StrandedBisSeq, Neonatal foreskin fibroblasts</P>','A3295',3405,'PUBLIC',3,7,'-10Log10(fischer/chi-square pvalue), uncorrected',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3296,'Minus Regions, 2x, 0.01 FDR','<P ALIGN=""LEFT"">From StrandedBisSeq, Neonatal foreskin fibroblasts</P>','A3296',3405,'PUBLIC',3,7,'Regions enriched for minus stranded methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3297,'Plus Regions, 2x, 0.01 FDR','<P ALIGN=""LEFT"">From StrandedBisSeq, Neonatal foreskin fibroblasts</P>','A3297',3405,'PUBLIC',3,7,'Regions enriched for plus stranded methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3298,'Log2Ratio','<P ALIGN=""LEFT"">From StrandedBisSeq</P>','A3298',3405,'PUBLIC',3,7,'Fraction plus strand methylation/ fraction minus',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3299,'PValue','<P ALIGN=""LEFT"">From StrandedBisSeq</P>','A3299',3405,'PUBLIC',3,7,'-10Log10(fischer/chi-square pvalue), uncorrected',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3300,'Minus Regions, 2x, 0.01 FDR','<P ALIGN=""LEFT"">From StrandedBisSeq</P>','A3300',3405,'PUBLIC',3,7,'Regions enriched for minus stranded methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3301,'Plus Regions, 2x, 0.01 FDR','<P ALIGN=""LEFT"">From StrandedBisSeq</P>','A3301',3405,'PUBLIC',3,7,'Regions enriched for plus stranded methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3302,'Log2Ratio','<P ALIGN=""LEFT"">From StrandedBisSeq</P>','A3302',3405,'PUBLIC',3,7,'Fraction plus strand methylation/ fraction minus',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3303,'PValue','<P ALIGN=""LEFT"">From StrandedBisSeq</P>','A3303',3405,'PUBLIC',3,7,'-10Log10(fischer/chi-square pvalue), uncorrected',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3304,'Minus Regions, 2x, 0.01 FDR','<P ALIGN=""LEFT"">From StrandedBisSeq</P>','A3304',3405,'PUBLIC',3,7,'Regions enriched for minus stranded methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3305,'Plus Regions, 2x, 0.01 FDR','<P ALIGN=""LEFT"">From StrandedBisSeq</P>','A3305',3405,'PUBLIC',3,7,'Regions enriched for plus stranded methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3306,'Log2Ratio','From StrandedBisSeq','A3306',3405,'MEM',3,7,'Fraction plus strand methylation/ fraction minus',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3307,'PValue','From StrandedBisSeq','A3307',3405,'MEM',3,7,'-10Log10(fischer/chi-square pvalue), uncorrected',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3308,'Minus Regions, 2x, 0.01 FDR','From StrandedBisSeq','A3308',3405,'MEM',3,7,'Regions enriched for minus stranded methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3309,'Plus Regions, 2x, 0.01 FDR','From StrandedBisSeq','A3309',3405,'MEM',3,7,'Regions enriched for plus stranded methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3310,'Log2Ratio','From StrandedBisSeq','A3310',3405,'MEM',3,7,'Fraction plus strand methylation/ fraction minus',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3311,'PValue','From StrandedBisSeq','A3311',3405,'MEM',3,7,'-10Log10(fischer/chi-square pvalue), uncorrected',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3312,'Minus Regions, 2x, 0.01 FDR','From StrandedBisSeq','A3312',3405,'MEM',3,7,'Regions enriched for minus stranded methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3313,'Plus Regions, 2x, 0.01 FDR','From StrandedBisSeq','A3313',3405,'MEM',3,7,'Regions enriched for plus stranded methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3314,'Log2Ratio','From StrandedBisSeq','A3314',3405,'MEM',3,7,'Fraction plus strand methylation/ fraction minus',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3315,'PValue','From StrandedBisSeq','A3315',3405,'MEM',3,7,'-10Log10(fischer/chi-square pvalue), uncorrected',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3316,'Minus Regions, 2x, 0.01 FDR','From StrandedBisSeq','A3316',3405,'MEM',3,7,'Regions enriched for minus stranded methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3317,'Plus Regions, 2x, 0.01 FDR','From StrandedBisSeq','A3317',3405,'MEM',3,7,'Regions enriched for plus stranded methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3318,'H3K27ac',NULL,'A3318',3405,'MEM',43,7,NULL,'Andrew','2011-02-08 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3319,'H3K27me3',NULL,'A3319',3405,'MEM',43,7,NULL,'Andrew','2011-02-08 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3320,'H3K36me3',NULL,'A3320',3405,'MEM',43,7,NULL,'Andrew','2011-02-08 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3321,'H3K4me1',NULL,'A3321',3405,'MEM',43,7,NULL,'Andrew','2011-02-08 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3322,'H3K4me3',NULL,'A3322',3405,'MEM',43,7,NULL,'Andrew','2011-02-08 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3324,'ESCell 0.25kb Log2Ratios','<P ALIGN=""LEFT"">Data derived from running the USeq StrandedBisSeq application to identify regions exhibiting biased stranded CG methylation.</P>','A3324',3405,'PUBLIC',3,7,'Log2Ratio of +/- strand methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3325,'ESCell 0.25kb PValues','<P ALIGN=""LEFT"">Data derived from running the USeq StrandedBisSeq application to identify regions exhibiting biased stranded CG methylation.</P>','A3325',3405,'PUBLIC',3,7,'Uncorrected p-value for differential stranded methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3326,'ESFibro 0.25kb Log2Ratios','<P ALIGN=""LEFT"">Data derived from running the USeq StrandedBisSeq application to identify regions exhibiting biased stranded CG methylation.</P>','A3326',3405,'PUBLIC',3,7,'Log2Ratio of +/- strand methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3327,'ESFibro 0.25kb PValues','<P ALIGN=""LEFT"">Data derived from running the USeq StrandedBisSeq application to identify regions exhibiting biased stranded CG methylation.</P>','A3327',3405,'PUBLIC',3,7,'Uncorrected p-value for differential stranded methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3328,'ForeFibro 0.25kb Log2Ratios','<P ALIGN=""LEFT"">Data derived from running the USeq StrandedBisSeq application to identify regions exhibiting biased stranded CG methylation.</P>','A3328',3405,'PUBLIC',3,7,'Log2Ratio of +/- strand methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3329,'ForeFibro 0.25kb PValues','<P ALIGN=""LEFT"">Data derived from running the USeq StrandedBisSeq application to identify regions exhibiting biased stranded CG methylation.</P>','A3329',3405,'PUBLIC',3,7,'Uncorrected p-value for differential stranded methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3330,'ESCell 0.25kb Diff Methyl Regions','<P ALIGN=""LEFT"">Data derived from running the USeq StrandedBisSeq application to identify regions exhibiting biased stranded CG methylation.</P>','A3330',3405,'PUBLIC',3,7,'Differentially methylated regions, FDR 0.001, 3x',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3331,'ESFibro 0.25kb Diff Methyl Regions','<P ALIGN=""LEFT"">Data derived from running the USeq StrandedBisSeq application to identify regions exhibiting biased stranded CG methylation.</P>','A3331',3405,'PUBLIC',3,7,'Differentially methylated regions, FDR 0.001, 3x',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3332,'ForeFibro 0.25kb Diff Methyl Regions','<P ALIGN=""LEFT"">Data derived from running the USeq StrandedBisSeq application to identify regions exhibiting biased stranded CG methylation.</P>','A3332',3405,'PUBLIC',3,7,'Differentially methylated regions, FDR 0.001, 3x',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3333,'ESCell 1.5kb Log2Ratios','<P ALIGN=""LEFT"">Data derived from running the USeq StrandedBisSeq application to identify regions exhibiting biased stranded CG methylation.</P>','A3333',3405,'PUBLIC',3,7,'Log2Ratio of +/- strand methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3334,'ESCell 1.5kb PValues','<P ALIGN=""LEFT"">Data derived from running the USeq StrandedBisSeq application to identify regions exhibiting biased stranded CG methylation.</P>','A3334',3405,'PUBLIC',3,7,'Uncorrected p-value for differential stranded methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3335,'ESFibro 1.5kb Log2Ratios','<P ALIGN=""LEFT"">Data derived from running the USeq StrandedBisSeq application to identify regions exhibiting biased stranded CG methylation.</P>','A3335',3405,'PUBLIC',3,7,'Log2Ratio of +/- strand methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3336,'ESFibro 1.5kb PValues','<P ALIGN=""LEFT"">Data derived from running the USeq StrandedBisSeq application to identify regions exhibiting biased stranded CG methylation.</P>','A3336',3405,'PUBLIC',3,7,'Uncorrected p-value for differential stranded methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3337,'ForeFibro 1.5kb Log2Ratios','<P ALIGN=""LEFT"">Data derived from running the USeq StrandedBisSeq application to identify regions exhibiting biased stranded CG methylation.</P>','A3337',3405,'PUBLIC',3,7,'Log2Ratio of +/- strand methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3338,'ForeFibro 1.5kb PValues','<P ALIGN=""LEFT"">Data derived from running the USeq StrandedBisSeq application to identify regions exhibiting biased stranded CG methylation.</P>','A3338',3405,'PUBLIC',3,7,'Uncorrected p-value for differential stranded methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3339,'ESCell 1.5kb Diff Methyl Regions','<P ALIGN=""LEFT"">Data derived from running the USeq StrandedBisSeq application to identify regions exhibiting biased stranded CG methylation.</P>','A3339',3405,'PUBLIC',3,7,'Differentially methylated regions, FDR 0.001, 3x',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3340,'ESFibro 1.5kb Diff Methyl Regions','<P ALIGN=""LEFT"">Data derived from running the USeq StrandedBisSeq application to identify regions exhibiting biased stranded CG methylation.</P>','A3340',3405,'PUBLIC',3,7,'Differentially methylated regions, FDR 0.001, 3x',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3341,'ForeFibro 1.5kb Diff Methyl Regions','<P ALIGN=""LEFT"">Data derived from running the USeq StrandedBisSeq application to identify regions exhibiting biased stranded CG methylation.</P>','A3341',3405,'PUBLIC',3,7,'Differentially methylated regions, FDR 0.001, 3x',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3342,'H1 0.25kb Log2Ratios','<P ALIGN=""LEFT"">Data derived from running the USeq StrandedBisSeq application to identify regions exhibiting biased stranded CG methylation.</P>','A3342',3405,'PUBLIC',3,7,'Log2Ratio of +/- strand methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3343,'H1 0.25kb PValues','<P ALIGN=""LEFT"">Data derived from running the USeq StrandedBisSeq application to identify regions exhibiting biased stranded CG methylation.</P>','A3343',3405,'PUBLIC',3,7,'Uncorrected p-value for differential stranded methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3344,'IRM90 0.25kb Log2Ratios','<P ALIGN=""LEFT"">Data derived from running the USeq StrandedBisSeq application to identify regions exhibiting biased stranded CG methylation.</P>','A3344',3405,'PUBLIC',3,7,'Log2Ratio of +/- strand methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3345,'IRM90 0.25kb PValues','<P ALIGN=""LEFT"">Data derived from running the USeq StrandedBisSeq application to identify regions exhibiting biased stranded CG methylation.</P>','A3345',3405,'PUBLIC',3,7,'Uncorrected p-value for differential stranded methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3346,'H1 0.25kb Diff Methyl Regions','<P ALIGN=""LEFT"">Data derived from running the USeq StrandedBisSeq application to identify regions exhibiting biased stranded CG methylation.</P>','A3346',3405,'PUBLIC',3,7,'Differentially methylated regions, FDR 0.001, 3x',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3347,'IRM90 0.25kb Diff Methyl Regions','<P ALIGN=""LEFT"">Data derived from running the USeq StrandedBisSeq application to identify regions exhibiting biased stranded CG methylation.</P>','A3347',3405,'PUBLIC',3,7,'Differentially methylated regions, FDR 0.001, 3x',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3348,'H1 1.5kb Log2Ratios','<P ALIGN=""LEFT"">Data derived from running the USeq StrandedBisSeq application to identify regions exhibiting biased stranded CG methylation.</P>','A3348',3405,'PUBLIC',3,7,'Log2Ratio of +/- strand methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3349,'H1 1.5kb PValues','<P ALIGN=""LEFT"">Data derived from running the USeq StrandedBisSeq application to identify regions exhibiting biased stranded CG methylation.</P>','A3349',3405,'PUBLIC',3,7,'Uncorrected p-value for differential stranded methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3350,'IRM90 1.5kb Log2Ratios','<P ALIGN=""LEFT"">Data derived from running the USeq StrandedBisSeq application to identify regions exhibiting biased stranded CG methylation.</P>','A3350',3405,'PUBLIC',3,7,'Log2Ratio of +/- strand methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3351,'IRM90 1.5kb PValues','<P ALIGN=""LEFT"">Data derived from running the USeq StrandedBisSeq application to identify regions exhibiting biased stranded CG methylation.</P>','A3351',3405,'PUBLIC',3,7,'Uncorrected p-value for differential stranded methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3352,'H1 1.5kb Diff Methyl Regions','<P ALIGN=""LEFT"">Data derived from running the USeq StrandedBisSeq application to identify regions exhibiting biased stranded CG methylation.</P>','A3352',3405,'PUBLIC',3,7,'Differentially methylated regions, FDR 0.001, 3x',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3353,'IRM90 1.5kb Diff Methyl Regions','<P ALIGN=""LEFT"">Data derived from running the USeq StrandedBisSeq application to identify regions exhibiting biased stranded CG methylation.</P>','A3353',3405,'PUBLIC',3,7,'Differentially methylated regions, FDR 0.001, 3x',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3354,'X1 0.25kb Log2Ratios','Data derived from running the USeq StrandedBisSeq application to identify regions exhibiting biased stranded CG methylation.','A3354',3405,'MEM',3,7,'Log2Ratio of +/- strand methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3355,'X1 0.25kb PValues','Data derived from running the USeq StrandedBisSeq application to identify regions exhibiting biased stranded CG methylation.','A3355',3405,'MEM',3,7,'Uncorrected p-value for differential stranded methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3356,'X2 0.25kb Log2Ratios','Data derived from running the USeq StrandedBisSeq application to identify regions exhibiting biased stranded CG methylation.','A3356',3405,'MEM',3,7,'Log2Ratio of +/- strand methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3357,'X2 0.25kb PValues','Data derived from running the USeq StrandedBisSeq application to identify regions exhibiting biased stranded CG methylation.','A3357',3405,'MEM',3,7,'Uncorrected p-value for differential stranded methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3358,'X1X2 0.25kb Log2Ratios','Data derived from running the USeq StrandedBisSeq application to identify regions exhibiting biased stranded CG methylation.','A3358',3405,'MEM',3,7,'Log2Ratio of +/- strand methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3359,'X1X2 0.25kb PValues','Data derived from running the USeq StrandedBisSeq application to identify regions exhibiting biased stranded CG methylation.','A3359',3405,'MEM',3,7,'Uncorrected p-value for differential stranded methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3360,'X1 0.25kb Diff Methyl Regions','Data derived from running the USeq StrandedBisSeq application to identify regions exhibiting biased stranded CG methylation.','A3360',3405,'MEM',3,7,'Differentially methylated regions, FDR 0.001, 3x',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3361,'X2 0.25kb Diff Methyl Regions','Data derived from running the USeq StrandedBisSeq application to identify regions exhibiting biased stranded CG methylation.','A3361',3405,'MEM',3,7,'Differentially methylated regions, FDR 0.001, 3x',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3362,'X1X2 0.25kb Diff Methyl Regions','Data derived from running the USeq StrandedBisSeq application to identify regions exhibiting biased stranded CG methylation.','A3362',3405,'MEM',3,7,'Differentially methylated regions, FDR 0.001, 3x',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3363,'X1 1.5kb Log2Ratios','Data derived from running the USeq StrandedBisSeq application to identify regions exhibiting biased stranded CG methylation.','A3363',3405,'MEM',3,7,'Log2Ratio of +/- strand methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3364,'X1 1.5kb PValues','Data derived from running the USeq StrandedBisSeq application to identify regions exhibiting biased stranded CG methylation.','A3364',3405,'MEM',3,7,'Uncorrected p-value for differential stranded methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3365,'X2 1.5kb Log2Ratios','Data derived from running the USeq StrandedBisSeq application to identify regions exhibiting biased stranded CG methylation.','A3365',3405,'MEM',3,7,'Log2Ratio of +/- strand methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3366,'X2 1.5kb PValues','Data derived from running the USeq StrandedBisSeq application to identify regions exhibiting biased stranded CG methylation.','A3366',3405,'MEM',3,7,'Uncorrected p-value for differential stranded methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3367,'X1X2 1.5kb Log2Ratios','Data derived from running the USeq StrandedBisSeq application to identify regions exhibiting biased stranded CG methylation.','A3367',3405,'MEM',3,7,'Log2Ratio of +/- strand methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3368,'X1X2 1.5kb PValues','Data derived from running the USeq StrandedBisSeq application to identify regions exhibiting biased stranded CG methylation.','A3368',3405,'MEM',3,7,'Uncorrected p-value for differential stranded methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3369,'X1 1.5kb Diff Methyl Regions','Data derived from running the USeq StrandedBisSeq application to identify regions exhibiting biased stranded CG methylation.','A3369',3405,'MEM',3,7,'Differentially methylated regions, FDR 0.001, 3x',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3370,'X2 1.5kb Diff Methyl Regions','Data derived from running the USeq StrandedBisSeq application to identify regions exhibiting biased stranded CG methylation.','A3370',3405,'MEM',3,7,'Differentially methylated regions, FDR 0.001, 3x',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3371,'X1X2 1.5kb Diff Methyl Regions','Data derived from running the USeq StrandedBisSeq application to identify regions exhibiting biased stranded CG methylation.','A3371',3405,'MEM',3,7,'Differentially methylated regions, FDR 0.001, 3x',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3372,'H3K4me3',NULL,'A3372',3405,'MEM',43,7,NULL,'Andrew','2011-02-16 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3373,'H3K4me1',NULL,'A3373',3405,'MEM',43,7,NULL,'Andrew','2011-02-16 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3374,'H3K27me3',NULL,'A3374',3405,'MEM',43,7,NULL,'Andrew','2011-02-16 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3375,'H3K27ac',NULL,'A3375',3405,'MEM',43,7,NULL,'Andrew','2011-02-16 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3376,'H3K36me3',NULL,'A3376',3405,'MEM',43,7,NULL,'Andrew','2011-02-16 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3380,'smRNA and mRNA 2-10 alignments','<P ALIGN=""LEFT"">110216</P><P ALIGN=""LEFT"">Downloaded mRNA and smRNA datasets in .sra format for H1 from SRA, UCSD ENCODE.</P><P ALIGN=""LEFT"">Converted to .fastq</P><P ALIGN=""LEFT"">Aligned with novoalign.  Default adapter sequence (checked on GEO).  Standard format.  </P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">cd /MacHome/andrew/temp/H1_sra/ftp-trace.ncbi.nlm.nih.gov/sra/sra-instant/reads/ByExp/sra/SRX</P><P ALIGN=""LEFT"">smRNA</P><P ALIGN=""LEFT"">nohup novoalign -FSTDFQ -m -l17 -h90 -t30 -rA 15 -q 3 -c 10 -a -d /usr/local/share/novo_indices/hg18WithSplicesAdaptersk13s3 -f SRR020285.fastq &gt;&amp; /MacHome/andrew/BioAnalysis/Sue/H1_RNA/110216_smRNA_H1_SRR020285.novo.txt &amp;	</P><P ALIGN=""LEFT"">mRNA</P><P ALIGN=""LEFT"">put this in a shell script, looping through each file.</P><P ALIGN=""LEFT"">nohup novoalign -FSTDFQ -m -l17 -t60 -rA 10 -q 3 -c 10 -a -d /usr/local/share/novo_indices/hg18WithSplicesAdaptersk13s3 -f .fastq &gt;&amp; /MacHome/andrew/BioAnalysis/Sue/H1_RNA/101008_am_7396X2_s5.txt &amp;	</P><P ALIGN=""LEFT"">Started at about 11:45pm. Finished at 2am.  ~2 hrs.</P><P ALIGN=""LEFT"">Converted with Novo2Bed.pl --zerobase --gzip.</P><P ALIGN=""LEFT"">BedAlignFilter.pl for smRNA to get unique, 2-10 and 1-10.  For mRNA to get unique and 2-10 and 1-10 as well.</P><P ALIGN=""LEFT"">Took about 3.5 hours.  Used run_bedalignfilter.sh (ad hoc).  </P><P ALIGN=""LEFT"">Converted to Point with jj ~/USeqApps/Tag2Point -v H_sapiens_Mar_2006 -f &lt;folder&gt;</P><P ALIGN=""LEFT"">Made ReadCoverage files with this type of command: jj ~/USeqApps/ReadCoverage -p 1to10 -s 1to10_ReadCov -k -r</P><P ALIGN=""LEFT"">for i in *ReadCov; do jj ~/USeqApps/Bar2USeq -r 1 -f $i; done</P>','A3380',3405,'MEM',43,7,'','Andrew','2011-02-17 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3381,'smRNA and mRNA unique alignments','<P ALIGN=""LEFT"">110216</P><P ALIGN=""LEFT"">Downloaded mRNA and smRNA datasets in .sra format for H1 from SRA, UCSD ENCODE.</P><P ALIGN=""LEFT"">Converted to .fastq</P><P ALIGN=""LEFT"">Aligned with novoalign.  Default adapter sequence (checked on GEO).  Standard format.  </P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">cd /MacHome/andrew/temp/H1_sra/ftp-trace.ncbi.nlm.nih.gov/sra/sra-instant/reads/ByExp/sra/SRX</P><P ALIGN=""LEFT"">smRNA</P><P ALIGN=""LEFT"">nohup novoalign -FSTDFQ -m -l17 -h90 -t30 -rA 15 -q 3 -c 10 -a -d /usr/local/share/novo_indices/hg18WithSplicesAdaptersk13s3 -f SRR020285.fastq &gt;&amp; /MacHome/andrew/BioAnalysis/Sue/H1_RNA/110216_smRNA_H1_SRR020285.novo.txt &amp;	</P><P ALIGN=""LEFT"">mRNA</P><P ALIGN=""LEFT"">put this in a shell script, looping through each file.</P><P ALIGN=""LEFT"">nohup novoalign -FSTDFQ -m -l17 -t60 -rA 10 -q 3 -c 10 -a -d /usr/local/share/novo_indices/hg18WithSplicesAdaptersk13s3 -f .fastq &gt;&amp; /MacHome/andrew/BioAnalysis/Sue/H1_RNA/101008_am_7396X2_s5.txt &amp;	</P><P ALIGN=""LEFT"">Started at about 11:45pm. Finished at 2am.  ~2 hrs.</P><P ALIGN=""LEFT"">Converted with Novo2Bed.pl --zerobase --gzip.</P><P ALIGN=""LEFT"">BedAlignFilter.pl for smRNA to get unique, 2-10 and 1-10.  For mRNA to get unique and 2-10 and 1-10 as well.</P><P ALIGN=""LEFT"">Took about 3.5 hours.  Used run_bedalignfilter.sh (ad hoc).  </P><P ALIGN=""LEFT"">Converted to Point with jj ~/USeqApps/Tag2Point -v H_sapiens_Mar_2006 -f &lt;folder&gt;</P><P ALIGN=""LEFT"">Made ReadCoverage files with this type of command: jj ~/USeqApps/ReadCoverage -p 1to10 -s 1to10_ReadCov -k -r</P><P ALIGN=""LEFT"">for i in *ReadCov; do jj ~/USeqApps/Bar2USeq -r 1 -f $i; done</P>','A3381',3405,'MEM',43,7,'','Andrew','2011-02-17 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3382,'smRNA and mRNA 1-10 alignments','<P ALIGN=""LEFT"">110216</P><P ALIGN=""LEFT"">Downloaded mRNA and smRNA datasets in .sra format for H1 from SRA, UCSD ENCODE.</P><P ALIGN=""LEFT"">Converted to .fastq</P><P ALIGN=""LEFT"">Aligned with novoalign.  Default adapter sequence (checked on GEO).  Standard format.  </P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">cd /MacHome/andrew/temp/H1_sra/ftp-trace.ncbi.nlm.nih.gov/sra/sra-instant/reads/ByExp/sra/SRX</P><P ALIGN=""LEFT"">smRNA</P><P ALIGN=""LEFT"">nohup novoalign -FSTDFQ -m -l17 -h90 -t30 -rA 15 -q 3 -c 10 -a -d /usr/local/share/novo_indices/hg18WithSplicesAdaptersk13s3 -f SRR020285.fastq &gt;&amp; /MacHome/andrew/BioAnalysis/Sue/H1_RNA/110216_smRNA_H1_SRR020285.novo.txt &amp;	</P><P ALIGN=""LEFT"">mRNA</P><P ALIGN=""LEFT"">put this in a shell script, looping through each file.</P><P ALIGN=""LEFT"">nohup novoalign -FSTDFQ -m -l17 -t60 -rA 10 -q 3 -c 10 -a -d /usr/local/share/novo_indices/hg18WithSplicesAdaptersk13s3 -f .fastq &gt;&amp; /MacHome/andrew/BioAnalysis/Sue/H1_RNA/101008_am_7396X2_s5.txt &amp;	</P><P ALIGN=""LEFT"">Started at about 11:45pm. Finished at 2am.  ~2 hrs.</P><P ALIGN=""LEFT"">Converted with Novo2Bed.pl --zerobase --gzip.</P><P ALIGN=""LEFT"">BedAlignFilter.pl for smRNA to get unique, 2-10 and 1-10.  For mRNA to get unique and 2-10 and 1-10 as well.</P><P ALIGN=""LEFT"">Took about 3.5 hours.  Used run_bedalignfilter.sh (ad hoc).  </P><P ALIGN=""LEFT"">Converted to Point with jj ~/USeqApps/Tag2Point -v H_sapiens_Mar_2006 -f &lt;folder&gt;</P><P ALIGN=""LEFT"">Made ReadCoverage files with this type of command: jj ~/USeqApps/ReadCoverage -p 1to10 -s 1to10_ReadCov -k -r</P><P ALIGN=""LEFT"">for i in *ReadCov; do jj ~/USeqApps/Bar2USeq -r 1 -f $i; done</P>','A3382',3405,'MEM',43,7,'','Andrew','2011-02-17 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3383,'PartialX1X2 Stranded Read Coverage',NULL,'A3383',3405,'MEM',NULL,7,NULL,'u0028003','2011-02-22 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3385,'ESCell','Fraction stranded +/- mCG, from StandedBisSeq','A3385',3405,'MEM',NULL,7,'BP fraction stranded mCG',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3386,'ESFibro','Fraction stranded +/- mCG, from StandedBisSeq','A3386',3405,'MEM',NULL,7,'BP fraction stranded mCG',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3387,'ForeFibro','Fraction stranded +/- mCG, from StandedBisSeq','A3387',3405,'MEM',NULL,7,'BP fraction stranded mCG',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3388,'H1','Fraction stranded +/- mCG, from StandedBisSeq','A3388',3405,'MEM',NULL,7,'BP fraction stranded mCG',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3389,'IRM90','Fraction stranded +/- mCG, from StandedBisSeq','A3389',3405,'MEM',NULL,7,'BP fraction stranded mCG',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3390,'X1','Fraction stranded +/- mCG, from StandedBisSeq','A3390',3405,'MEM',NULL,7,'BP fraction stranded mCG',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3391,'X1X2','Fraction stranded +/- mCG, from StandedBisSeq','A3391',3405,'MEM',NULL,7,'BP fraction stranded mCG',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3392,'X2','Fraction stranded +/- mCG, from StandedBisSeq','A3392',3405,'MEM',NULL,7,'BP fraction stranded mCG',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3393,'ESCell','<P ALIGN=""LEFT"">Uncorrected fischer p-val for asymetric mCG, from StrandedBisSeq</P>','A3393',3405,'MEM',NULL,7,'BP uncorr fischer p-val stranded mCG',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3394,'ESFibro','Uncorrected fischer p-val for asymetric mCG, from StrandedBisSeq','A3394',3405,'MEM',NULL,7,'BP uncorr fischer p-val stranded mCG',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3395,'ForeFibro','Uncorrected fischer p-val for asymetric mCG, from StrandedBisSeq','A3395',3405,'MEM',NULL,7,'BP uncorr fischer p-val stranded mCG',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3396,'H1','Uncorrected fischer p-val for asymetric mCG, from StrandedBisSeq','A3396',3405,'MEM',NULL,7,'BP uncorr fischer p-val stranded mCG',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3397,'IRM90','Uncorrected fischer p-val for asymetric mCG, from StrandedBisSeq','A3397',3405,'MEM',NULL,7,'BP uncorr fischer p-val stranded mCG',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3398,'X1','Uncorrected fischer p-val for asymetric mCG, from StrandedBisSeq','A3398',3405,'MEM',NULL,7,'BP uncorr fischer p-val stranded mCG',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3399,'X1X2','Uncorrected fischer p-val for asymetric mCG, from StrandedBisSeq','A3399',3405,'MEM',NULL,7,'BP uncorr fischer p-val stranded mCG',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3400,'X2','Uncorrected fischer p-val for asymetric mCG, from StrandedBisSeq','A3400',3405,'MEM',NULL,7,'BP uncorr fischer p-val stranded mCG',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3401,'X2','<P ALIGN=""LEFT"">Regions defined from StrandedBisSeq of asymetric stranded methylation, min FDR 0.05, 2x, max gap 500bp</P>','A3401',3405,'MEM',NULL,7,'Regions of asymetric stranded methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3402,'X1X2','<P ALIGN=""LEFT"">Regions defined from StrandedBisSeq of asymetric stranded methylation, min FDR 0.05, 2x, max gap 500bp</P>','A3402',3405,'MEM',NULL,7,'Regions of asymetric stranded methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3403,'X1','<P ALIGN=""LEFT"">Regions defined from StrandedBisSeq of asymetric stranded methylation, min FDR 0.05, 2x, max gap 500bp</P>','A3403',3405,'MEM',NULL,7,'Regions of asymetric stranded methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3404,'IRM90','<P ALIGN=""LEFT"">Regions defined from StrandedBisSeq of asymetric stranded methylation, min FDR 0.05, 2x, max gap 500bp</P>','A3404',3405,'MEM',NULL,7,'Regions of asymetric stranded methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3405,'H1','<P ALIGN=""LEFT"">Regions defined from StrandedBisSeq of asymetric stranded methylation, min FDR 0.05, 2x, max gap 500bp</P>','A3405',3405,'MEM',NULL,7,'Regions of asymetric stranded methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3406,'ForeFibro','<P ALIGN=""LEFT"">Regions defined from StrandedBisSeq of asymetric stranded methylation, min FDR 0.05, 2x, max gap 500bp</P>','A3406',3405,'MEM',NULL,7,'Regions of asymetric stranded methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3407,'ESFibro','<P ALIGN=""LEFT"">Regions defined from StrandedBisSeq of asymetric stranded methylation, min FDR 0.05, 2x, max gap 500bp</P>','A3407',3405,'MEM',NULL,7,'Regions of asymetric stranded methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3408,'ESCell','<P ALIGN=""LEFT"">Regions defined from StrandedBisSeq of asymetric stranded methylation, min FDR 0.05, 2x, max gap 500bp</P>','A3408',3405,'MEM',NULL,7,'Regions of asymetric stranded methylation',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3409,'IS-High-28818','','A3409',6300,'MEM',30,8,'','UARP','2011-02-24 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3410,'IS-High-64049','','A3410',6300,'MEM',30,8,'','UARP','2011-02-24 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3411,'IS-High-63678','','A3411',6300,'MEM',30,8,'','UARP','2011-02-24 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3412,'IS-Low-30058','','A3412',6300,'MEM',30,8,'','UARP','2011-02-24 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3413,'IS-Low-63577','','A3413',6300,'MEM',30,8,'','UARP','2011-02-24 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3414,'IS-Low-76591','','A3414',6300,'MEM',30,8,'','UARP','2011-02-24 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3418,'HiSeq_7900X1_Sepsis_Plt',NULL,'A3418',6300,'MEM',40,30,NULL,'tolley','2011-02-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3419,'HiSeq_7900X7_PreDiabetic_Plt','','A3419',6300,'MEM',40,30,'','tolley','2011-02-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3420,'HiSeq_7900X2_HealthyElderly_Plt','','A3420',6300,'MEM',40,30,'','tolley','2011-03-01 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3421,'HiSeq_7900X3_HealthyElderly_Plt','','A3421',6300,'MEM',40,30,'','tolley','2011-03-01 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3422,'HiSeq_7900X4_Sepsis_Plt','','A3422',6300,'MEM',40,30,'','tolley','2011-03-01 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3423,'HiSeq_7900X5_Diabetic_Plt','','A3423',6300,'MEM',40,30,'','tolley','2011-03-01 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3424,'HiSeq_7900X6_Diabetic_Plt','','A3424',6300,'MEM',40,30,'','tolley','2011-03-01 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3425,'HiSeq_7900X8_PreDiabetic_Plt','','A3425',6300,'MEM',40,30,'','tolley','2011-03-01 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3430,'H3K4me1','<P ALIGN=""LEFT"">Series GSE24165	Histone H3K27ac separates active from poised enhancers and predicts developmental state</P><P ALIGN=""LEFT"">GSM594586 	ChIP-Seq for H3K4me1 in whole liver</P><P ALIGN=""LEFT"">SRX027/SRX027339</P><P ALIGN=""LEFT"">Aligned with bowtie -m 40 -a --best --strata to mm9.</P><P ALIGN=""LEFT"">ScanSeqs using input from GSM594590 and GSM427096.</P>','A3430',3806,'MEM',43,7,'H3K4me1 in liver','Andrew','2011-03-01 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3431,'H3K4me3','<P ALIGN=""LEFT"">Series GSE24165	Histone H3K27ac separates active from poised enhancers and predicts developmental state</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">GSM594589 	ChIP-Seq for H3K4me3 in whole liver</P><P ALIGN=""LEFT"">SRX027/SRX027342</P><P ALIGN=""LEFT"">Aligned with bowtie -m 40 -a --best --strata to mm9.</P><P ALIGN=""LEFT"">ScanSeqs using input from GSM594590 and GSM427096.</P>','A3431',3806,'MEM',43,7,'','Andrew','2011-03-01 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3432,'H3K27ac','<P ALIGN=""LEFT"">Series GSE24165	Histone H3K27ac separates active from poised enhancers and predicts developmental state</P><P ALIGN=""LEFT"">GSM594587 	ChIP-Seq for H3K27ac in whole liver</P><P ALIGN=""LEFT"">SRX027/SRX027340</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Aligned with bowtie -m 40 -a --best --strata to mm9.</P><P ALIGN=""LEFT"">ScanSeqs using input from GSM594590 and GSM427096.</P>','A3432',3806,'MEM',43,7,'','Andrew','2011-03-01 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3433,'H3','<P ALIGN=""LEFT"">Series GSE24165	Histone H3K27ac separates active from poised enhancers and predicts developmental state</P><P ALIGN=""LEFT"">GSM594588 	ChIP-Seq for H3 in whole liver</P><P ALIGN=""LEFT"">SRX027/SRX027341</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Aligned with bowtie -m 40 -a --best --strata to mm9.</P><P ALIGN=""LEFT"">ScanSeqs using input from GSM594590 and GSM427096.</P>','A3433',3806,'MEM',43,7,'','Andrew','2011-03-01 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3434,'PolII','<P ALIGN=""LEFT"">Genome-wide mapping of RNA Pol-II promoter usage in mouse tissues by ChIP-seq</P><P ALIGN=""LEFT"">	liver_RNAP-II_ChIP-Seq</P><P ALIGN=""LEFT"">	SRX020/SRX020251 		</P><P ALIGN=""LEFT"">Aligned with bowtie -m 40 -a --best --strata to mm9.</P><P ALIGN=""LEFT"">ScanSeqs using input from GSM594590 and GSM427096.</P>','A3434',3806,'MEM',43,7,'','Andrew','2011-03-01 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3435,'PolII','<P ALIGN=""LEFT"">Genome-wide mapping of RNA Pol-II promoter usage in mouse tissues by ChIP-seq</P><P ALIGN=""LEFT"">GSM542590 	spleen_RNAP-II_ChIP-Seq</P><P ALIGN=""LEFT"">SRX020/SRX020253 	</P><P ALIGN=""LEFT"">Aligned with bowtie -m 40 -a --best --strata to mm9.</P><P ALIGN=""LEFT"">ScanSeqs using input from GSM594590 and GSM427096.</P>','A3435',3806,'MEM',43,7,'','Andrew','2011-03-01 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3439,'FoxA1','<P ALIGN=""LEFT"">GSE17067 record: A quantitative model of transcription regulation reveals the role of non-conserved enhancers [Mus musculus] </P><P ALIGN=""LEFT"">GSM427090 	FOXA1 liver ChIPseq</P><P ALIGN=""LEFT"">Aligned with bowtie -m 40 -a --best --strata to mm9.</P><P ALIGN=""LEFT"">ScanSeqs using input from GSM594590 and GSM427096.</P>','A3439',3806,'MEM',43,7,'','Andrew','2011-03-01 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3440,'FoxA2','<P ALIGN=""LEFT"">GSE17067 record: A quantitative model of transcription regulation reveals the role of non-conserved enhancers [Mus musculus] </P><P ALIGN=""LEFT"">GSM427089 	FOXA2 liver ChIPseq</P><P ALIGN=""LEFT"">Aligned with bowtie -m 40 -a --best --strata to mm9.</P><P ALIGN=""LEFT"">ScanSeqs using input from GSM594590 and GSM427096.</P>','A3440',3806,'MEM',43,7,'','Andrew','2011-03-01 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3443,'p300','<P ALIGN=""LEFT"">GSE17067 record: A quantitative model of transcription regulation reveals the role of non-conserved enhancers [Mus musculus] </P><P ALIGN=""LEFT"">GSM427087 	p300 liver ChIPseq		SRX019/SRX019006</P><P ALIGN=""LEFT"">Aligned with bowtie -m 40 -a --best --strata to mm9.</P><P ALIGN=""LEFT"">ScanSeqs using input from GSM594590 and GSM427096.</P>','A3443',3806,'MEM',43,7,'','Andrew','2011-03-01 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3444,'E2F4','<P ALIGN=""LEFT"">GSE17067 record: A quantitative model of transcription regulation reveals the role of non-conserved enhancers [Mus musculus] </P><P ALIGN=""LEFT"">GSM427091 	E2F4 liver ChIPseq</P><P ALIGN=""LEFT"">Aligned with bowtie -m 40 -a --best --strata to mm9.</P><P ALIGN=""LEFT"">ScanSeqs using input from GSM594590 and GSM427096.</P>','A3444',3806,'MEM',43,7,'','Andrew','2011-03-01 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3445,'CEBPa','<P ALIGN=""LEFT"">GSE17067 record: A quantitative model of transcription regulation reveals the role of non-conserved enhancers [Mus musculus] </P><P ALIGN=""LEFT"">GSM427088 	CEBPa liver ChIPseq</P><P ALIGN=""LEFT"">Aligned with bowtie -m 40 -a --best --strata to mm9.</P><P ALIGN=""LEFT"">ScanSeqs using input from GSM594590 and GSM427096.</P>','A3445',3806,'MEM',43,7,'','Andrew','2011-03-01 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3446,'IS high 28818','','A3446',6300,'MEM',30,8,'','UARP','2011-03-02 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3447,'IS high 63678','','A3447',6300,'MEM',30,8,'','UARP','2011-03-02 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3448,'IS high 64049','','A3448',6300,'MEM',30,8,'','UARP','2011-03-02 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3449,'IS low 30085','','A3449',6300,'MEM',30,8,'','UARP','2011-03-02 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3450,'IS low 76591','','A3450',6300,'MEM',30,8,'','UARP','2011-03-02 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3451,'IS low 63577','','A3451',6300,'MEM',30,8,'','UARP','2011-03-02 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3452,'IS med 36913','','A3452',6300,'MEM',30,8,'','UARP','2011-03-02 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3453,'IS med 75506','','A3453',6300,'MEM',30,8,'','UARP','2011-03-02 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3461,'WT','<P ALIGN=""LEFT"">RNA-seq data from wt sample 7800X1.</P><P ALIGN=""LEFT"">Aligned to Zv9 genome (including splice junctions).</P><P ALIGN=""LEFT"">Aligned with novoalign, -r Random option, SAM format.</P><P ALIGN=""LEFT"">SAM file corrected with SamFixer.</P><P ALIGN=""LEFT"">Converted to .bam/.bai files with PicardTools SortSam.jar.</P><P ALIGN=""LEFT"">Bam file validated with PicardTools ValidateSamFile.jar.</P>','A3461',6303,'MEM',66,43,'','bmilash','2011-03-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3462,'Polysome tags','','A3462',1204,'PUBLIC',NULL,28,'A polysome tags track that displays observed tags from a polysome-enriched RNA pool.','u0028003','2011-03-10 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3463,'Novel transcripts','','A3463',1204,'PUBLIC',NULL,28,'RNA sequences from regions with no existing gene predictions (title: ""additional (non-annotated) transcript regions"", orange). In this custom track the bar height represents the number of sequences that align to each position.','u0028003','2011-03-10 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3464,'Novel exon junctions','<P ALIGN=""LEFT"">&quot;Potential exon-junctions&quot; track (blue), which displays potential novel exon-exon junctions that are supported by RNAseq reads by two bars, each for the 23bp of the adjacent exons, with a connecting arrow that indicates the exon-junction directionality. The bar shade indicates a strength score, calculated from the number of aligned tags to the exon-junction and number of bases from each junction that are included in the sequence tag, with darker shades representing higher scores</P>','A3464',1204,'PUBLIC',NULL,28,'','u0028003','2011-03-10 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3465,'PolyA tags','','A3465',1204,'PUBLIC',NULL,28,'Displays polyadenylation junctions identified by the RNA-seq.','u0028003','2011-03-10 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3466,'Splice leader 1','','A3466',1204,'PUBLIC',NULL,28,'Displays trans-splice leader sites identified by the RNA-seq','u0028003','2011-03-10 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3467,'Splice leader 2','','A3467',1204,'PUBLIC',NULL,28,'Displays trans-splice leader sites identified by the RNA-seq','u0028003','2011-03-10 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3476,'Combined 1 to 10 Read Coverage','<P ALIGN=""LEFT"">Combined 7506X1 and 7506X2 (one lane of each, one is smRNA and the other is mRNA) with 1 to 10 alignments.</P><P ALIGN=""LEFT"">Made ReadCoverage tracks with jj ~/USeqApps/ReadCoverage -p 7506X1_7506X2_1_to_10_Point/ -s 7506_ReadCov -k -r</P>','A3476',3806,'MEM',43,7,'','Andrew','2011-03-11 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3477,'Hypo1Alpha','','A3477',3806,'MEM',3,35,'','u0028003','2011-03-11 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3478,'Hypo2Alpha','','A3478',3806,'MEM',3,35,'','u0028003','2011-03-11 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3479,'Input','','A3479',3806,'MEM',3,35,'','u0028003','2011-03-11 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3480,'Normal','','A3480',3806,'MEM',3,35,'','u0028003','2011-03-11 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3481,'Hypo1AlphaVsInput','','A3481',3806,'MEM',3,35,'','u0028003','2011-03-11 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3482,'NormalVsInput','','A3482',3806,'MEM',3,35,'','u0028003','2011-03-11 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3483,'Hypo2AlphaVsInput','','A3483',3806,'MEM',3,35,'','u0028003','2011-03-11 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3484,'Donors 1 and 2','<P ALIGN=""LEFT"">Combined smRNA-seq and mRNA-seq from both donors, 1 to 10 alignments.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">jj ~/USeqApps/ReadCoverage -p DonorX1/7396X1_7397X1_1_to_10_Point/,DonorX2/7397X2_7396X2_1_to_10_Point/ -s BothDonors_ReadCov -r -k </P>','A3484',3405,'MEM',43,7,'Read Coverage','Andrew','2011-03-11 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3486,'Deletions Paired 101bp','<P ALIGN=""LEFT"">Sequencing of human sperm, unpublished, Brad Cairns laboratory, University of Utah</P>','A3486',6300,'PUBLIC',3,28,'','u0028003','2011-03-16 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3487,'Insertions Paired 101bp','<P ALIGN=""LEFT"">Sequencing of human sperm, unpublished, Brad Cairns laboratory, University of Utah</P>','A3487',6300,'PUBLIC',3,28,'','u0028003','2011-03-16 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3527,'HiSeq_7975X1_ITP','','A3527',6300,'MEM',40,30,'','tolley','2011-03-16 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3531,'HiSeq_7975X2_GPS','','A3531',6300,'MEM',40,30,'','tolley','2011-03-16 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3532,'Test Track','','A3532',6300,'MEM',14,7,'','CairnsLab','2011-03-16 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3533,'DNMT2-5AZA-IP.bam','','A3533',6300,'MEM',14,7,'','CairnsLab','2011-03-17 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3534,'RepeatMasker','<P ALIGN=""LEFT"">Downloaded from UCSC as bed file, removed comment lines, converted to USeq format:</P><P ALIGN=""LEFT"">jj ~/USeqApps/Text2USeq -f rmsk_hg18_sorted_nocomment.bed.gz -g H_sapiens_Mar_2006 -c 0 -b 1 -e 2 -t 3,4 -s 5</P>','A3534',3405,'PUBLIC',43,7,'','Andrew','2011-03-22 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3537,'D1','','A3537',3405,'MEM',14,7,'','CairnsLab','2011-03-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3538,'D2','','A3538',3405,'MEM',14,7,'','CairnsLab','2011-03-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3539,'D3','','A3539',3405,'MEM',14,7,'','CairnsLab','2011-03-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3540,'D4','','A3540',3405,'MEM',14,7,'','CairnsLab','2011-03-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3541,'D1','','A3541',3405,'MEM',14,7,'','CairnsLab','2011-03-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3542,'D2_Pval','','A3542',3405,'MEM',14,7,'','CairnsLab','2011-03-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3543,'D2_Pval','','A3543',3405,'MEM',14,7,'','CairnsLab','2011-03-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3544,'D3_P_Val','','A3544',3405,'MEM',14,7,'','CairnsLab','2011-03-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3545,'D4_Pval','','A3545',3405,'MEM',14,7,'','CairnsLab','2011-03-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3546,'GI36','','A3546',3405,'MEM',14,7,'','CairnsLab','2011-03-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3547,'PTO4','','A3547',3405,'MEM',14,7,'','CairnsLab','2011-03-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3548,'X7FC_Pval','','A3548',3405,'MEM',14,7,'','CairnsLab','2011-03-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3549,'ODN2','','A3549',3405,'MEM',14,7,'','CairnsLab','2011-03-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3552,'All Paired 101bp Fixed','','A3552',6300,'PUBLIC',NULL,28,'','u0028003','2011-03-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3554,'10001_36913 All SNPs','<P ALIGN=""LEFT"">Subject IDS = 5_10001_4007001_36913</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">SNPs filtered for Het or Homozygous genotype in sample, and no duplicate locations (overlapping splice variants not included in SNP list).</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""> </P>','A3554',6300,'MEM',30,8,'Unfiltered SNP calls from Schellenberg lab','UARP','2011-03-29 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3555,'10001_63577 All SNPs','<P ALIGN=""LEFT"">Subject IDS = 6_10001_1298001_63577</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">SNPs filtered for Het or Homozygous genotype in sample, and no duplicate locations (overlapping splice variants not included in SNP list).</P><P ALIGN=""LEFT""> </P>','A3555',6300,'MEM',30,8,'Unfiltered SNP calls from Schellenberg lab','UARP','2011-03-29 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3556,'10001_64049 All SNPs','<P ALIGN=""LEFT"">Subject IDS = 7_10001_1329001_64049</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">SNPs filtered for Het or Homozygous genotype in sample, and no duplicate locations (overlapping splice variants not included in SNP list).</P><P ALIGN=""LEFT""> </P>','A3556',6300,'MEM',30,8,'Unfiltered SNP calls from Schellenberg lab','UARP','2011-03-29 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3557,'10001_75506 All SNPs','<P ALIGN=""LEFT"">Subject IDS = 8_10001_1581001_75506</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""> </P>','A3557',6300,'MEM',30,8,'Unfiltered SNP calls from Schellenberg lab','UARP','2011-03-29 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3558,'7001_28818 All SNPs','<P ALIGN=""LEFT"">Patient IDS = 7001, 1023001, 28818</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""></P>','A3558',6300,'MEM',30,8,'Unfiltered SNP calls from Schellenberg lab','UARP','2011-03-29 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3559,'7001_76591 All SNPs','<P ALIGN=""LEFT"">Subject IDS = 4_7001_1595001_76591</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""> </P>','A3559',6300,'MEM',30,8,'Unfiltered SNP calls from Schellenberg lab','UARP','2011-03-29 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3560,'7001_30085 All SNPs','<P ALIGN=""LEFT"">Patient IDS = 2_7001_1054001_30085</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""> </P>','A3560',6300,'MEM',30,8,'Unfiltered SNP calls from Schellenberg lab','UARP','2011-03-29 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3561,'7001_63678 All SNPs','<P ALIGN=""LEFT"">Subject IDS = 3_7001_1306001_63678</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""> </P>','A3561',6300,'MEM',30,8,'Unfiltered SNP calls from Schellenberg lab','UARP','2011-03-29 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3566,'H1 FDR Base Methyl','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A203</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">BisStat Full Coverage</P>','A3566',3405,'PUBLIC',3,7,'FDR for methylation at each sequenced C','u0028003','2011-04-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3567,'H1 Fraction Base Methyl','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A203</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">BisStat full coverage</P>','A3567',3405,'PUBLIC',3,7,'Fraction of methylation at each sequenced C','u0028003','2011-04-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3568,'IRM90 Fraction Base Methyl','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A203</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">BisStat full coverage</P>','A3568',3405,'PUBLIC',3,7,'Fraction of methylation at each sequenced C','u0028003','2011-04-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3569,'IRM90 FDR Base Methyl','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A203</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">BisStat full coverage</P>','A3569',3405,'PUBLIC',3,7,'FDR for methylation at each sequenced C','u0028003','2011-04-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3570,'H1 Fraction Window Methyl','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A203</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">BisStat full coverage</P><P ALIGN=""LEFT""></P>','A3570',3405,'PUBLIC',3,7,'Fraction of methylation at each 1KB window','u0028003','2011-04-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3571,'IRM90 Fraction Window Methyl','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A203</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">BisStat full coverage</P>','A3571',3405,'PUBLIC',3,7,'Fraction of methylation at each 1KB window','u0028003','2011-04-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3572,'ES Cells FDR Base Methyl','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A201</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">BisStat</P>','A3572',3405,'PUBLIC',3,7,'FDR for methylation at each sequenced C','u0028003','2011-04-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3573,'ES Cells Fraction Base Methyl','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A201</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">BisStat</P>','A3573',3405,'PUBLIC',3,7,'Fraction of methylation at each sequenced C','u0028003','2011-04-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3574,'ES Fibroblasts FDR Base Methyl','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A201</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">BisStat</P>','A3574',3405,'PUBLIC',3,7,'FDR for methylation at each sequenced C','u0028003','2011-04-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3575,'ES Fibroblasts Fraction Base Methyl','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A201</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">BisStat</P>','A3575',3405,'PUBLIC',3,7,'Fraction of methylation at each sequenced C','u0028003','2011-04-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3576,'Foreskin Fibroblast FDR Base Methyl','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A201</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">BisStat</P>','A3576',3405,'PUBLIC',3,7,'FDR for methylation at each sequenced C','u0028003','2011-04-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3577,'Foreskin Fibroblast Fraction Base Methyl','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A201</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">BisStat</P>','A3577',3405,'PUBLIC',3,7,'Fraction of methylation at each sequenced C','u0028003','2011-04-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3578,'ES Cell Fraction Window Methyl','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A201</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">BisStat</P>','A3578',3405,'PUBLIC',3,7,'Fraction of methylation at each 1KB window','u0028003','2011-04-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3579,'ES Fibroblast Fraction Window Methyl','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A201</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">BisStat</P>','A3579',3405,'PUBLIC',3,7,'Fraction of methylation at each 1KB window','u0028003','2011-04-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3580,'Foreskin Fibroblast Fraction Window Methyl','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A201</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">BisStat</P>','A3580',3405,'PUBLIC',3,7,'Fraction of methylation at each 1KB window','u0028003','2011-04-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3581,'7795X1 FDR Base Methyl','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A207</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">BisStat</P>','A3581',3405,'MEM',3,7,'FDR for methylation at each sequenced C','u0028003','2011-04-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3582,'7795X1 Fraction Base Methyl','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A207</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">BisStat</P>','A3582',3405,'MEM',3,7,'Fraction of methylation at each sequenced C','u0028003','2011-04-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3583,'7795X2 FDR Base Methyl','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A207</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">BisStat</P>','A3583',3405,'MEM',3,7,'FDR for methylation at each sequenced C','u0028003','2011-04-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3584,'7795X2 Fraction Base Methyl','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A207</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">BisStat</P>','A3584',3405,'MEM',3,7,'Fraction of methylation at each sequenced C','u0028003','2011-04-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3585,'7795X1-X2 FDR Base Methyl','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A207</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">BisStat</P>','A3585',3405,'MEM',3,7,'FDR for methylation at each sequenced C','u0028003','2011-04-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3586,'7795X1-X2 Fraction Base Methyl','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A207</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">BisStat</P>','A3586',3405,'MEM',3,7,'Fraction of methylation at each sequenced C','u0028003','2011-04-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3587,'7795X1 Fraction Window Methyl','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A207</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">BisStat</P>','A3587',3405,'MEM',3,7,'Fraction of methylation at each 1KB window','u0028003','2011-04-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3588,'7795X2 Fraction Window Methyl','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A207</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">BisStat</P>','A3588',3405,'MEM',3,7,'Fraction of methylation at each 1KB window','u0028003','2011-04-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3589,'7795X1-X2 Fraction Window Methyl','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A207</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">BisStat</P>','A3589',3405,'MEM',3,7,'Fraction of methylation at each 1KB window','u0028003','2011-04-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3596,'ESCell FDR Base Methyl mCG','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A201</P>','A3596',3405,'PUBLIC',3,7,'FDR for methylation at each sequenced CG',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3597,'ESCell Fraction Base Methyl mCG','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A201</P>','A3597',3405,'PUBLIC',3,7,'Fraction of methylation at each sequenced CG',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3598,'ESCell Fraction Window Methyl mCG','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A201</P>','A3598',3405,'PUBLIC',3,7,'Fraction of CG methylation at each 1KB window',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3599,'ESFibro FDR Base Methyl mCG','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A201</P>','A3599',3405,'PUBLIC',3,7,'FDR for methylation at each sequenced CG',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3600,'ESFibro Fraction Base Methyl mCG','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A201</P>','A3600',3405,'PUBLIC',3,7,'Fraction of methylation at each sequenced CG',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3601,'ESFibro Fraction Window Methyl mCG','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A201</P>','A3601',3405,'PUBLIC',3,7,'Fraction of CG methylation at each 1KB window',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3602,'ForeFibro FDR Base Methyl mCG','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A201</P>','A3602',3405,'PUBLIC',3,7,'FDR for methylation at each sequenced CG',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3603,'ForeFibro Fraction Base Methyl mCG','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A201</P>','A3603',3405,'PUBLIC',3,7,'Fraction of methylation at each sequenced CG',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3604,'ForeFibro Fraction Window Methyl mCG','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A201</P>','A3604',3405,'PUBLIC',3,7,'Fraction of CG methylation at each 1KB window',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3605,'H1 FDR Base Methyl mCG','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A203</P>','A3605',3405,'PUBLIC',3,7,'FDR for methylation at each sequenced CG',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3606,'H1 Fraction Base Methyl mCG','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A203</P>','A3606',3405,'PUBLIC',3,7,'Fraction of methylation at each sequenced CG',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3607,'H1 Fraction Window mCG','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A203</P>','A3607',3405,'PUBLIC',3,7,'Fraction of CG methylation at each 1KB window',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3608,'IRM90 FDR Base Methyl mCG','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A203</P>','A3608',3405,'PUBLIC',3,7,'FDR for methylation at each sequenced CG',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3609,'IRM90 Fraction Base Methyl mCG','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A203</P>','A3609',3405,'PUBLIC',3,7,'Fraction of methylation at each sequenced CG',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3610,'IRM90 Fraction Window mCG','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A203</P>','A3610',3405,'PUBLIC',3,7,'Fraction of CG methylation at each 1KB window',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3611,'7795X1 FDR Base Methyl mCG','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A207</P>','A3611',3405,'MEM',3,7,'FDR for methylation at each sequenced CG',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3612,'7795X1 Fraction Base Methyl mCG','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A207</P>','A3612',3405,'MEM',3,7,'Fraction of methylation at each sequenced CG',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3613,'7795X1 Fraction Window Methyl mCG','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A207</P>','A3613',3405,'MEM',3,7,'Fraction of CG methylation at each 1KB window',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3614,'7795X1-X2 FDR Base Methyl mCG','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A207</P>','A3614',3405,'MEM',3,7,'FDR for methylation at each sequenced CG',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3615,'7795X1-X2 Fraction Base Methyl mCG','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A207</P>','A3615',3405,'MEM',3,7,'Fraction of methylation at each sequenced CG',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3616,'7795X1-X2 Fraction Window Methyl mCG','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A207</P>','A3616',3405,'MEM',3,7,'Fraction of CG methylation at each 1KB window',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3617,'7795X2 FDR Base Methyl mCG','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A207</P>','A3617',3405,'MEM',3,7,'FDR for methylation at each sequenced CG',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3618,'7795X2 Fraction Base Methyl mCG','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A207</P>','A3618',3405,'MEM',3,7,'Fraction of methylation at each sequenced CG',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3619,'7795X2 Fraction Window Methyl mCG','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A207</P>','A3619',3405,'MEM',3,7,'Fraction of CG methylation at each 1KB window',NULL,NULL,NULL,NULL,'/Data02/GenoPub/GenoPubData/'),
(3620,'H3K27ac','','A3620',3405,'MEM',14,7,'','CairnsLab','2011-04-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3621,'H3k4me1','','A3621',3405,'MEM',14,7,'','CairnsLab','2011-04-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3622,'H3K14ac','','A3622',3405,'MEM',14,7,'','CairnsLab','2011-04-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3623,'CombinedDonor_FAIRE_DNAse_300bp','','A3623',3405,'MEM',14,7,'DNAse HS Sites Processed InJason Liebs Lab','CairnsLab','2011-04-11 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3624,'COmbined_DNAse_HS_FAIRE_1000bp','','A3624',3405,'MEM',14,7,'DNAse_Data From Jason Liebs Lab','CairnsLab','2011-04-11 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3625,'5002_35793_HH_NoDups2','<P ALIGN=""LEFT"">Subject IDS = 5002_35793</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">SNPs filtered for Het or Homozygous genotype in sample, and no duplicate locations (overlapping splice variants not included in SNP list).</P><P ALIGN=""LEFT""> </P>','A3625',6300,'MEM',30,8,'SNPs filtered for Het or Homozygous genotype in sample','UARP','2011-04-11 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3626,'5002_41704_HH_NoDups2','<P ALIGN=""LEFT"">Patient IDS = 5002, 41704</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">SNPs filtered for Het or Homozygous genotype in sample, and no duplicate locations (overlapping splice variants not included in SNP list).</P><P ALIGN=""LEFT""> </P>','A3626',6300,'MEM',30,8,'SNPs filtered for Het or Homozygous genotype in sample','UARP','2011-04-11 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3627,'5002_64021_HH_NoDups2','<P ALIGN=""LEFT"">Patient IDS = 5002, 64021</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">SNPs filtered for Het or Homozygous genotype in sample, and no duplicate locations (overlapping splice variants not included in SNP list).</P><P ALIGN=""LEFT""> </P>','A3627',6300,'MEM',30,8,'SNPs filtered for Het or Homozygous genotype in sample','UARP','2011-04-11 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3628,'5002_66830_HH_NoDups2','<P ALIGN=""LEFT"">Patient IDS = 5002, 66830</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">SNPs filtered for Het or Homozygous genotype in sample, and no duplicate locations (overlapping splice variants not included in SNP list).</P><P ALIGN=""LEFT""> </P>','A3628',6300,'MEM',30,8,'SNPs filtered for Het or Homozygous genotype in sample','UARP','2011-04-11 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3629,'5002_35793_ALL_SNPs2','<P ALIGN=""LEFT"">Subject IDS = aut25_5002_35793</P><P ALIGN=""LEFT"">All Het and Homo. ALL SNPs for this individual = dbSNPs + unique SNPs, all transcripts.</P><P ALIGN=""LEFT""></P>','A3629',6300,'MEM',30,8,'Unfiltered SNP calls ','UARP','2011-04-11 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3631,'5002_64021_ALL_SNPs2','<P ALIGN=""LEFT"">Subject IDS = aut27_5002_64021_ALL_SNPs</P><P ALIGN=""LEFT"">All Het and Homo. ALL SNPs for this individual = dbSNPs + unique SNPs, all transcripts.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""></P>','A3631',6300,'MEM',30,8,'All SNP calls','UARP','2011-04-11 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3632,'5002_66830_ALL_SNPs2','<P ALIGN=""LEFT"">Subject IDS = aut28_5002_66830_ALL_SNPs</P><P ALIGN=""LEFT"">All Het and Homo. ALL SNPs for this individual = dbSNPs + unique SNPs, all transcripts.</P><P ALIGN=""LEFT""> </P><P ALIGN=""LEFT""></P>','A3632',6300,'MEM',30,8,'All SNP calls ','UARP','2011-04-11 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3633,'5002_41704_ALL_SNPs2','<P ALIGN=""LEFT"">Subject IDS = Aut26_5002_41704_ALL</P><P ALIGN=""LEFT"">All Het and Homo. ALL SNPs for this individual = dbSNPs + unique SNPs, all transcripts.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""></P>','A3633',6300,'MEM',30,8,'All SNP calls','UARP','2011-04-11 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3635,'5002_41704_NOVEL_SNPS2','<P ALIGN=""LEFT"">Subject IDS = aut26_5002_41704_NOVEL_SNPS</P><P ALIGN=""LEFT"">Only unique Het and Homo. ALT SNPs for this individual = NO dbSNPs </P><P ALIGN=""LEFT""></P>','A3635',6300,'MEM',30,8,'SNP filtered for dbSNP ','UARP','2011-04-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3636,'5002_64021_NOVEL_SNPS2','<P ALIGN=""LEFT"">Subject IDS = aut27_5002_64021_NOVEL_SNPS</P><P ALIGN=""LEFT"">Only unique Het and Homo. ALT SNPs for this individual = NO dbSNPs </P><P ALIGN=""LEFT""></P>','A3636',6300,'MEM',30,8,'SNP filtered for dbSNP ','UARP','2011-04-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3637,'5002_66830_NOVEL_SNPs2','<P ALIGN=""LEFT"">Subject IDS = aut28_5002_66830_NOVEL_SNPs</P><P ALIGN=""LEFT"">Only unique Het and Homo. ALT SNPs for this individual = NO dbSNPs </P><P ALIGN=""LEFT""></P>','A3637',6300,'MEM',30,8,'SNP filtered for dbSNP ','UARP','2011-04-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3638,'5002_35793_Novel_SNPs2','<P ALIGN=""LEFT"">Subject IDS = Aut25_5002_35793_Novel_SNPs</P><P ALIGN=""LEFT"">Only unique Het and Homo. ALT SNPs for this individual = NO dbSNPs </P><P ALIGN=""LEFT""></P>','A3638',6300,'MEM',30,8,'SNP filtered for dbSNP ','UARP','2011-04-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3639,'WT','','A3639',6301,'MEM',49,36,'','bmilash','2011-04-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3641,'Mouse Platelets Paired Reads','','A3641',3806,'PUBLIC',40,30,'','tolley','2011-04-14 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3642,'Mouse Platelets Trizol','','A3642',3806,'PUBLIC',40,30,'Trizol isolated Poly A selected RNA from male c57bl/6 mice (pooled from multiple mice).','tolley','2011-04-14 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3643,'Human Platelets Trizol','','A3643',6300,'PUBLIC',3,30,'Poly-A isolated RNA from female human platelets','tolley','2011-04-14 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3644,'TestBamFromSOLiD',NULL,'A3644',6300,'MEM',NULL,28,NULL,'u0028003','2011-04-14 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3645,'Capture Design',NULL,'A3645',6300,'MEM',NULL,28,NULL,'u0028003','2011-04-14 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3648,'7795X2 Non Methyl ChrXY','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A207</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">These datasets were derived from alignments that were aligned to indexes lacking chrY and chrX then merged.</P>','A3648',3405,'MEM',3,7,'Number of Converted Cs Sequenced at each genomic C','u0028003','2011-04-20 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3649,'7795X2 Methyl ChrXY','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A207</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">These datasets were derived from alignments that were aligned to indexes lacking chrY and chrX then merged.</P>','A3649',3405,'MEM',3,7,'Number of Non Converted Cs Sequenced at each genomic C','u0028003','2011-04-20 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3650,'7795X1 Non Methyl ChrXY','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A207</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">These datasets were derived from alignments that were aligned to indexes lacking chrY and chrX then merged.</P>','A3650',3405,'MEM',3,7,'Number of Converted Cs Sequenced at each genomic C','u0028003','2011-04-20 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3651,'7795X1 Methyl ChrXY','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A207</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">These datasets were derived from alignments that were aligned to indexes lacking chrY and chrX then merged.</P>','A3651',3405,'MEM',3,7,'Number of Non Converted Cs Sequenced at each genomic C','u0028003','2011-04-20 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3653,'Genbank annotation','<P ALIGN=""LEFT"">F11 genome project url: http://www.ncbi.nlm.nih.gov/sites/entrez?Db=genome&amp;Cmd=ShowDetailView&amp;TermToSearch=5339</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Downloaded annotation and fasta files from http://www.ncbi.nlm.nih.gov/Traces/wgs/?val=AAJU02</P>','A3653',6305,'PUBLIC',NULL,28,'','u0028003','2011-04-20 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3660,'1104max1-1_00000025',NULL,'A3660',6300,'MEM',NULL,45,NULL,'bmilash','2011-04-22 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3661,'1104max1-2_00000026',NULL,'A3661',6300,'MEM',NULL,45,NULL,'bmilash','2011-04-22 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3662,'1104max1-3_00000027',NULL,'A3662',6300,'MEM',NULL,45,NULL,'bmilash','2011-04-22 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3663,'1104max1-4_00000028',NULL,'A3663',6300,'MEM',NULL,45,NULL,'bmilash','2011-04-22 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3664,'1104max1-5_00000029',NULL,'A3664',6300,'MEM',NULL,45,NULL,'bmilash','2011-04-22 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3665,'1104max1-6_00000030',NULL,'A3665',6300,'MEM',NULL,45,NULL,'bmilash','2011-04-22 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3666,'1104max1-7_00000031',NULL,'A3666',6300,'MEM',NULL,45,NULL,'bmilash','2011-04-22 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3667,'CpG Contexts',NULL,'A3667',3405,'PUBLIC',NULL,28,NULL,'u0028003','2011-04-25 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3677,'Fixed 25',NULL,'A3677',6300,'MEM',NULL,45,NULL,'bmilash','2011-04-26 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3678,'Blood 0','','A3678',6305,'MEM',NULL,46,'','u0028003','2011-04-26 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3679,'Blood 1','','A3679',6305,'MEM',NULL,46,'','u0028003','2011-04-26 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3680,'Blood 2','','A3680',6305,'MEM',NULL,46,'','u0028003','2011-04-26 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3681,'Input 0','','A3681',6305,'MEM',NULL,46,'','u0028003','2011-04-26 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3682,'Input 1','','A3682',6305,'MEM',NULL,46,'','u0028003','2011-04-26 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3683,'Input 2','<P ALIGN=""LEFT"">test desc</P>','A3683',6305,'MEM',NULL,46,'test summ','u0028003','2011-04-26 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3684,'PC 0','<P ALIGN=""LEFT""></P>','A3684',6305,'MEM',NULL,46,'','u0028003','2011-04-26 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3685,'PC 1','<P ALIGN=""LEFT""></P>','A3685',6305,'MEM',NULL,46,'','u0028003','2011-04-26 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3686,'PC 2','<P ALIGN=""LEFT""></P>','A3686',6305,'MEM',NULL,46,'','u0028003','2011-04-26 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3695,'Blood vs Input BinPVal',NULL,'A3695',6305,'MEM',NULL,46,NULL,'u0028003','2011-04-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3696,'Blood vs Input QVal FDR',NULL,'A3696',6305,'MEM',NULL,46,NULL,'u0028003','2011-04-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3697,'Blood vs Input Log2Ratio',NULL,'A3697',6305,'MEM',NULL,46,NULL,'u0028003','2011-04-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3698,'PC vs Input QVal FDR','','A3698',6305,'MEM',NULL,46,'','u0028003','2011-04-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3699,'PC vs Input Log2Ratio','','A3699',6305,'MEM',NULL,46,'','u0028003','2011-04-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3700,'PC vs Input BinPVal','','A3700',6305,'MEM',NULL,46,'','u0028003','2011-04-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3701,'Blood vs Input Enriched Regions',NULL,'A3701',6305,'MEM',NULL,46,NULL,'u0028003','2011-04-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3702,'Blood vs Input Reduced Regions',NULL,'A3702',6305,'MEM',NULL,46,NULL,'u0028003','2011-04-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3703,'PC vs Input Enriched Regions','','A3703',6305,'MEM',NULL,46,'','u0028003','2011-04-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3704,'PC vs Input Reduced Regions','','A3704',6305,'MEM',NULL,46,'','u0028003','2011-04-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3705,'Enriched Regions Blood 4x FDR 30 0.5kb','','A3705',6305,'MEM',NULL,46,'Common to all 3 Replicas','u0028003','2011-05-02 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3706,'Enriched Regions PC 4x FDR 30 0.5kb','','A3706',6305,'MEM',NULL,46,'Common to all 3 Replicas','u0028003','2011-05-02 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3707,'Enriched Regions Blood PC 4x FDR 30 0.5kb','','A3707',6305,'MEM',NULL,46,'Common to all 3 Replicas','u0028003','2011-05-02 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3708,'Reduced Regions PC 4x FDR 30 0.5kb','','A3708',6305,'MEM',NULL,46,'Common to all 3 Replicas','u0028003','2011-05-02 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3709,'Reduced Regions Blood PC 4x FDR 30 0.5kb','','A3709',6305,'MEM',NULL,46,'Common to all 3 Replicas','u0028003','2011-05-02 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3710,'Reduced Regions Blood 4x FDR 30 0.5kb','','A3710',6305,'MEM',NULL,46,'Common to all 3 Replicas','u0028003','2011-05-02 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3711,'7795X1 with ChrXY',NULL,'A3711',3405,'MEM',NULL,7,NULL,'u0028003','2011-05-03 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3712,'7795X2 with ChrXY','<P ALIGN=""LEFT"">Independent alignments to genomes without chrX or chrY were merged.</P>','A3712',3405,'MEM',NULL,7,'','u0028003','2011-05-03 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3725,'Ensembl Transcripts','<P ALIGN=""LEFT"">Ensembl transcripts downloaded from UCSC.</P>','A3725',6303,'PUBLIC',NULL,28,'','u0028003','2011-05-06 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3726,'FDR','','A3726',6303,'MEM',NULL,14,'MultipleReplicaScanSeqs window scan 150bp','u0028003','2011-05-06 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3727,'Log2Ratio','','A3727',6303,'MEM',NULL,14,'MultipleReplicaScanSeqs window scan 150bp','u0028003','2011-05-06 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3728,'DMSO Replica 1','','A3728',6303,'MEM',NULL,14,'Relative Read Coverage','u0028003','2011-05-06 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3729,'DMSO Replica 2','','A3729',6303,'MEM',NULL,14,'Relative Read Coverage','u0028003','2011-05-06 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3730,'DMSO Replica 3','','A3730',6303,'MEM',NULL,14,'Relative Read Coverage','u0028003','2011-05-06 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3731,'DMSO Replica 4','','A3731',6303,'MEM',NULL,14,'Relative Read Coverage','u0028003','2011-05-06 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3732,'DMSO Combine','','A3732',6303,'MEM',NULL,14,'Relative Read Coverage','u0028003','2011-05-06 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3733,'KalF Combine','','A3733',6303,'MEM',NULL,14,'Relative Read Coverage','u0028003','2011-05-06 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3734,'KalF Replica 1','','A3734',6303,'MEM',NULL,14,'Relative Read Coverage','u0028003','2011-05-06 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3735,'KalF Replica 2','','A3735',6303,'MEM',NULL,14,'Relative Read Coverage','u0028003','2011-05-06 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3736,'KalF Replica 3','','A3736',6303,'MEM',NULL,14,'Relative Read Coverage','u0028003','2011-05-06 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3737,'KalF Replica 4','','A3737',6303,'MEM',NULL,14,'Relative Read Coverage','u0028003','2011-05-06 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3738,'Diff Expressed Genes','<P ALIGN=""LEFT"">From MultipleReplicaDefinedRegionScanSeqs using the DESeq package</P>','A3738',6303,'MEM',NULL,14,'Min fold diff 1.5x, 10% FDR','u0028003','2011-05-06 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3741,'piRNA precursors for bMBT','','A3741',6303,'MEM',69,7,'','Magda','2011-05-07 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3744,'wt','<P ALIGN=""LEFT"">Reads aligned using Novoalign, then processed with USeq SamParser, USeq ReadCoverage, and USeq Bar2Useq programs.</P>','A3744',6301,'MEM',NULL,36,'Read coverage in wt sample','bmilash','2011-05-11 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3745,'rs7_muts',NULL,'A3745',6301,'MEM',NULL,36,NULL,'bmilash','2011-05-11 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3747,'rs7_sibs',NULL,'A3747',6301,'MEM',NULL,36,NULL,'bmilash','2011-05-17 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3749,'rs25_muts',NULL,'A3749',6301,'MEM',NULL,36,NULL,'bmilash','2011-05-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3750,'rs25_sibs',NULL,'A3750',6301,'MEM',NULL,36,NULL,'bmilash','2011-05-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3751,'Input reads scaled','','A3751',3405,'MEM',43,7,'','Andrew','2011-05-25 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3754,'Rpc155 QVal','','A3754',3405,'MEM',43,7,'','Andrew','2011-05-26 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3755,'Rpc155 reads scaled','','A3755',3405,'MEM',43,7,'','Andrew','2011-05-26 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3756,'Input reads unscaled',NULL,'A3756',3405,'MEM',43,7,NULL,'Andrew','2011-05-26 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3757,'Rpc155 reads unscaled','','A3757',3405,'MEM',43,7,'','Andrew','2011-05-26 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3758,'Rpc155 reads unscaled stranded',NULL,'A3758',3405,'MEM',43,7,NULL,'Andrew','2011-05-26 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3759,'Input reads unscaled stranded',NULL,'A3759',3405,'MEM',43,7,NULL,'Andrew','2011-05-26 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3764,'H1 Methy ChrXY','These datasets were derived from alignments that were aligned to indexes lacking chrY and chrX then merged.','A3764',3405,'PUBLIC',3,7,'Number of Non Converted Cs Sequenced at each genomic C.','u0028003','2011-06-02 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3765,'H1 Non Methyl ChrXY','These datasets were derived from alignments that were aligned to indexes lacking chrY and chrX then merged.','A3765',3405,'PUBLIC',3,7,'Number of Converted Cs Sequenced at each genomic C.','u0028003','2011-06-02 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3766,'IRM90 Methy ChrXY','These datasets were derived from alignments that were aligned to indexes lacking chrY and chrX then merged.','A3766',3405,'PUBLIC',3,7,'Number of Non Converted Cs Sequenced at each genomic C.','u0028003','2011-06-02 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3767,'IRM90 Non Methyl ChrXY','These datasets were derived from alignments that were aligned to indexes lacking chrY and chrX then merged.','A3767',3405,'PUBLIC',3,7,'Number of Converted Cs Sequenced at each genomic C.','u0028003','2011-06-02 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3768,'Rpc155_ChIP_meC',NULL,'A3768',3405,'MEM',43,7,NULL,'Andrew','2011-06-02 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3769,'Rpc155_ChIP_UnmeC',NULL,'A3769',3405,'MEM',43,7,NULL,'Andrew','2011-06-02 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3771,'Rpc155 Base FDRs NonConverted',NULL,'A3771',3405,'MEM',43,7,NULL,'Andrew','2011-06-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3772,'Rpc155 BaseFractionNonConverted',NULL,'A3772',3405,'MEM',43,7,NULL,'Andrew','2011-06-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3773,'Rpc155 WindowFractionNonConverted Window1kb',NULL,'A3773',3405,'MEM',43,7,NULL,'Andrew','2011-06-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3774,'Top4thGenes',NULL,'A3774',3405,'MEM',43,7,NULL,'Andrew','2011-06-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3775,'SecondTop4th Genes',NULL,'A3775',3405,'MEM',43,7,NULL,'Andrew','2011-06-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3776,'Rpc155 to Input BisSeq PValues','<P ALIGN=""LEFT"">	nohup java -Xmx30G -jar ~/USeqApps/BisSeq -s BisSeq_Rpc155_110618 \\</P><P ALIGN=""LEFT"">	-c NBP_Rpc155_ThreeLanes/MergedHitCountPointData/ConvertedC/ \\</P><P ALIGN=""LEFT"">	-C NBP_HeLaBisInput110615/MergedHitCountPointData/ConvertedC,NBP/MergedHitCountPointData/ConvertedC/ \\</P><P ALIGN=""LEFT"">	-n NBP_Rpc155_ThreeLanes/MergedHitCountPointData/NonConvertedC/ \\</P><P ALIGN=""LEFT"">	-N NBP_HeLaBisInput110615/MergedHitCountPointData/NonConvertedC,NBP/MergedHitCountPointData/NonConvertedC/ &gt;&amp; bisseq_output_110618.txt &amp;</P>','A3776',3405,'MEM',43,7,'','Andrew','2011-06-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3777,'Rpc155 to Input Enriched',NULL,'A3777',3405,'MEM',43,7,NULL,'Andrew','2011-06-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3778,'Rpc155 to Input Reduced',NULL,'A3778',3405,'MEM',43,7,NULL,'Andrew','2011-06-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3779,'7741X1 Tophat','<P ALIGN=""LEFT"">#Trying Novoalign on the same junctions found by tophat (plus knownGene and ensGene).	</P><P ALIGN=""LEFT"">	#Downloaded knownGene and ensGene tables from hg19 (UCSC).  Combined with junctions from 7741X1 and 7623X1 (after input 7741).  Removed lines with random, hap and chrUn.</P><P ALIGN=""LEFT"">		cat hg19_knownGene_ensGene_7741_7623_Splices.ucsc | grep -v &quot;hap&quot; | grep -v &quot;random&quot; | grep -v &quot;chrUn&quot; &gt; hg19_knownGene_ensGene_7741_7623_Splices_norandomhapUn.ucsc</P><P ALIGN=""LEFT"">	#Made fasta file on updraft qsub simple_MSJF.sh (not showing PBS options):</P><P ALIGN=""LEFT"">		java -Xmx20G -jar /uufs/chpc.utah.edu/common/home/u0491180/Apps/USeq/Apps/MakeSpliceJunctionFasta \\</P><P ALIGN=""LEFT"">		-f /uufs/chpc.utah.edu/common/home/u0491180/sc/Genomes/Human/hg19 \\</P><P ALIGN=""LEFT"">		-u /scratch/ibrix/chpc_gen/u0491180/hg19_knownGene_ensGene_7741_7623_Splices_norandomhapUn.ucsc \\</P><P ALIGN=""LEFT"">		-s 46 \\</P><P ALIGN=""LEFT"">		-r /scratch/ibrix/chpc_gen/u0491180/hg19_kg_ens_tophat7741_7623_Splices.fa</P><P ALIGN=""LEFT"">	#novoindex on updraft</P><P ALIGN=""LEFT"">		~/Apps/novocraft/novoindex -k13 -s3 ~/sc/Genomes/hg19LambdaAdaptersTophatSplicesk13s3 *.fa</P><P ALIGN=""LEFT"">   #novoalign</P><P ALIGN=""LEFT"">          /uufs/chpc.utah.edu/common/home/u0491180/Apps/novocraft/novoalign -FILMFQ -t90 -rA 10 -d hg19LambdaAdaptersTophat7741Junctionsk13s3 -f 7741X1_60files_00.txt.gz -a  | gzip &gt; 7741X1_60files_00_novo.gz </P><P ALIGN=""LEFT""></P>','A3779',6300,'MEM',43,7,'','Andrew','2011-06-20 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3780,'0570 read coverage','','A3780',6300,'MEM',NULL,47,'','bmilash','2011-06-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3781,'basal read coverage','','A3781',6300,'MEM',NULL,47,'','bmilash','2011-06-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3782,'luminal read coverage','','A3782',6300,'MEM',NULL,47,'','bmilash','2011-06-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3783,'other tumor read coverage','','A3783',6300,'MEM',NULL,47,'','bmilash','2011-06-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3784,'0570 read coverage','','A3784',3806,'MEM',NULL,47,'','bmilash','2011-06-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3785,'0642 read coverage','','A3785',3806,'MEM',NULL,47,'','bmilash','2011-06-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3786,'basal read coverage','','A3786',3806,'MEM',NULL,47,'','bmilash','2011-06-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3787,'luminal read coverage','','A3787',3806,'MEM',NULL,47,'','bmilash','2011-06-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3788,'other tumor read coverage','','A3788',3806,'MEM',NULL,47,'','bmilash','2011-06-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3789,'3293 read coverage','','A3789',6300,'MEM',NULL,47,'','bmilash','2011-06-29 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3790,'0642 read coverage','','A3790',6300,'MEM',NULL,47,'','bmilash','2011-06-29 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3791,'pe1 read coverage','','A3791',6300,'MEM',NULL,47,'','bmilash','2011-06-29 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3792,'3293 read coverage','','A3792',3806,'MEM',NULL,47,'','bmilash','2011-06-29 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3793,'pe1 read coverage','','A3793',3806,'MEM',NULL,47,'','bmilash','2011-06-29 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3794,'EggTotalRNAFPDrRNA2lanesReadCoverage','','A3794',6303,'MEM',69,7,'','Magda','2011-07-17 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3795,'64TotalRNAFPDrRNA1laneReadCoverage',NULL,'A3795',6303,'MEM',69,7,NULL,'Magda','2011-07-17 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3796,'256TotalRNAFPDrRNA2lanesReadCoverage',NULL,'A3796',6303,'MEM',69,7,NULL,'Magda','2011-07-17 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3797,'sphereTotalRNAFPDrRNA2lanesReadCoverage',NULL,'A3797',6303,'MEM',69,7,NULL,'Magda','2011-07-17 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3798,'egg_smallRNA_ReadCoverage_scaled_2lanes','','A3798',6303,'MEM',69,7,'','Magda','2011-07-17 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3799,'64_smallRNA_readcoverage_scaled','','A3799',6303,'MEM',69,7,'','Magda','2011-07-17 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3800,'256_smallRNA_readcoverag_scaled','','A3800',6303,'MEM',69,7,'','Magda','2011-07-17 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3801,'sphere_smallRNA_ReadCoverage_scaled','','A3801',6303,'MEM',69,7,'','Magda','2011-07-17 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3802,'eggvs64_ER_QvalFDR20',NULL,'A3802',6303,'MEM',69,7,NULL,'Magda','2011-07-17 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3803,'64vs256_ER_QvalFDR20',NULL,'A3803',6303,'MEM',69,7,NULL,'Magda','2011-07-17 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3804,'256vssphere_ER_QvalFDR20',NULL,'A3804',6303,'MEM',69,7,NULL,'Magda','2011-07-17 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3805,'DeleteMe','<P ALIGN=""LEFT"">X</P>','A3805',3405,'MEM',77,48,'X','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3806,'MDSNP07','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3806',3405,'MEM',77,48,'Normal CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3807,'MDSNP11','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3807',3405,'MEM',77,48,'Normal CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3808,'MDSNP15','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3808',3405,'MEM',77,48,'Normal CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3809,'CMML07','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3809',3405,'MEM',77,48,'Normal CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3810,'CMML13','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3810',3405,'MEM',77,48,'Normal CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3811,'CMML19','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3811',3405,'MEM',77,48,'Normal CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3812,'CMML23','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3812',3405,'MEM',77,48,'Normal CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3813,'CMML31','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3813',3405,'MEM',77,48,'Normal CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3814,'CMML33','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3814',3405,'MEM',77,48,'Normal CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3815,'CMML35','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3815',3405,'MEM',77,48,'Normal CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3816,'CMML37','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3816',3405,'MEM',77,48,'Normal CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3817,'CMML01','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3817',3405,'MEM',77,48,'Normal CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3818,'CMML03','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3818',3405,'MEM',77,48,'Normal CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3819,'CMML05','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3819',3405,'MEM',77,48,'Normal CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3820,'CMML09','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3820',3405,'MEM',77,48,'Normal CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3821,'CMML11','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3821',3405,'MEM',77,48,'Normal CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3822,'CMML18','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3822',3405,'MEM',77,48,'Normal CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3823,'MDSNP07','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3823',3405,'MEM',77,48,'Normal LOH','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3824,'MDSNP11','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3824',3405,'MEM',77,48,'Normal LOH','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3825,'MDSNP15','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3825',3405,'MEM',77,48,'Normal LOH','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3826,'CMML07','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3826',3405,'MEM',77,48,'Normal LOH','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3827,'CMML13','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3827',3405,'MEM',77,48,'Normal LOH','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3828,'CMML19','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3828',3405,'MEM',77,48,'Normal LOH','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3829,'CMML23','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3829',3405,'MEM',77,48,'Normal LOH','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3830,'CMML31','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3830',3405,'MEM',77,48,'Normal LOH','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3831,'CMML33','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3831',3405,'MEM',77,48,'Normal LOH','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3832,'CMML35','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3832',3405,'MEM',77,48,'Normal LOH','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3833,'CMML37','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3833',3405,'MEM',77,48,'Normal LOH','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3834,'CMML01','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3834',3405,'MEM',77,48,'Normal LOH','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3835,'CMML03','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3835',3405,'MEM',77,48,'Normal LOH','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3836,'CMML05','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3836',3405,'MEM',77,48,'Normal LOH','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3837,'CMML09','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3837',3405,'MEM',77,48,'Normal LOH','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3838,'CMML11','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3838',3405,'MEM',77,48,'Normal LOH','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3839,'CMML18','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3839',3405,'MEM',77,48,'Normal LOH','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3840,'MDSNP04','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3840',3405,'MEM',77,48,'Tumor CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3841,'MDSNP06','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3841',3405,'MEM',77,48,'Tumor CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3842,'MDSNP10','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3842',3405,'MEM',77,48,'Tumor CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3843,'MDSNP12','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3843',3405,'MEM',77,48,'Tumor CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3844,'CMML02','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3844',3405,'MEM',77,48,'Tumor CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3845,'CMML04','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3845',3405,'MEM',77,48,'Tumor CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3846,'CMML06','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3846',3405,'MEM',77,48,'Tumor CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3847,'CMML08','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3847',3405,'MEM',77,48,'Tumor CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3848,'CMML10','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3848',3405,'MEM',77,48,'Tumor CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3849,'CMML17','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3849',3405,'MEM',77,48,'Tumor CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3850,'CMML14','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3850',3405,'MEM',77,48,'Tumor CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3851,'CMML20.x','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3851',3405,'MEM',77,48,'Tumor CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3852,'CMML20.y','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3852',3405,'MEM',77,48,'Tumor CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3853,'CMML22','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3853',3405,'MEM',77,48,'Tumor CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3854,'CMML24','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3854',3405,'MEM',77,48,'Tumor CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3855,'CMML26','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3855',3405,'MEM',77,48,'Tumor CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3856,'CMML28','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3856',3405,'MEM',77,48,'Tumor CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3857,'CMML32','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3857',3405,'MEM',77,48,'Tumor CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3858,'CMML34','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3858',3405,'MEM',77,48,'Tumor CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3859,'CMML36','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3859',3405,'MEM',77,48,'Tumor CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3860,'CMML38.x','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3860',3405,'MEM',77,48,'Tumor CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3861,'CMML38.y','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3861',3405,'MEM',77,48,'Tumor CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3862,'CMML40','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3862',3405,'MEM',77,48,'Tumor CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3863,'CMML42','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3863',3405,'MEM',77,48,'Tumor CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3864,'CMML44','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3864',3405,'MEM',77,48,'Tumor CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3865,'CMML46','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3865',3405,'MEM',77,48,'Tumor CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3866,'CMML48','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3866',3405,'MEM',77,48,'Tumor CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3867,'CMML50','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3867',3405,'MEM',77,48,'Tumor CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3868,'CMML52','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3868',3405,'MEM',77,48,'Tumor CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3869,'CMML54','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3869',3405,'MEM',77,48,'Tumor CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3870,'MDSNP04','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3870',3405,'MEM',77,48,'Tumor LOH','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3871,'MDSNP06','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3871',3405,'MEM',77,48,'Tumor LOH','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3872,'MDSNP10','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3872',3405,'MEM',77,48,'Tumor LOH','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3873,'MDSNP12','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3873',3405,'MEM',77,48,'Tumor LOH','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3874,'CMML08','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3874',3405,'MEM',77,48,'Tumor LOH','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3875,'CMML14','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3875',3405,'MEM',77,48,'Tumor LOH','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3876,'CMML20.x','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3876',3405,'MEM',77,48,'Tumor LOH','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3877,'CMML20.y','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3877',3405,'MEM',77,48,'Tumor LOH','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3878,'CMML22','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3878',3405,'MEM',77,48,'Tumor LOH','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3879,'CMML24','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3879',3405,'MEM',77,48,'Tumor LOH','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3880,'CMML26','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3880',3405,'MEM',77,48,'Tumor LOH','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3881,'CMML28','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3881',3405,'MEM',77,48,'Tumor LOH','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3882,'CMML32','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3882',3405,'MEM',77,48,'Tumor LOH','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3883,'CMML34','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3883',3405,'MEM',77,48,'Tumor LOH','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3884,'CMML36','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3884',3405,'MEM',77,48,'Tumor LOH','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3885,'CMML38.x','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3885',3405,'MEM',77,48,'Tumor LOH','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3886,'CMML38.y','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3886',3405,'MEM',77,48,'Tumor LOH','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3887,'CMML40','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3887',3405,'MEM',77,48,'Tumor LOH','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3888,'CMML42','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3888',3405,'MEM',77,48,'Tumor LOH','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3889,'CMML44','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3889',3405,'MEM',77,48,'Tumor LOH','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3890,'CMML46','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3890',3405,'MEM',77,48,'Tumor LOH','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3891,'CMML48','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3891',3405,'MEM',77,48,'Tumor LOH','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3892,'CMML50','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3892',3405,'MEM',77,48,'Tumor LOH','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3893,'CMML52','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3893',3405,'MEM',77,48,'Tumor LOH','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3894,'CMML54','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3894',3405,'MEM',77,48,'Tumor LOH','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3895,'CMML02','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3895',3405,'MEM',77,48,'Tumor LOH','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3896,'CMML04','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3896',3405,'MEM',77,48,'Tumor LOH','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3897,'CMML06','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3897',3405,'MEM',77,48,'Tumor LOH','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3898,'CMML10','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3898',3405,'MEM',77,48,'Tumor LOH','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3899,'CMML17','Processed data converted from Beth Wilmot''s xxx.cn graph files. See Analysis Report A237','A3899',3405,'MEM',77,48,'Tumor LOH','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3901,'Merged Leukemic CNVs','','A3901',3405,'MEM',77,48,'','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3902,'Merged Constitutional CNVs','','A3902',3405,'MEM',77,48,'','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3903,'Merged Leukemic - Const CNVs','','A3903',3405,'MEM',77,48,'','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3905,'CMML08','<P ALIGN=""LEFT"">Processed data converted from Beth Wilmot&apos;sSpreadsheet files. See Analysis Report A237</P>','A3905',3405,'MEM',NULL,48,'Tumor CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3906,'CMML14','Processed data converted from Beth Wilmot''sSpreadsheet files. See Analysis Report A237','A3906',3405,'MEM',NULL,48,'Tumor CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3907,'CMML20','Processed data converted from Beth Wilmot''sSpreadsheet files. See Analysis Report A237','A3907',3405,'MEM',NULL,48,'Tumor CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3908,'CMML22','Processed data converted from Beth Wilmot''sSpreadsheet files. See Analysis Report A237','A3908',3405,'MEM',NULL,48,'Tumor CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3909,'CMML24','Processed data converted from Beth Wilmot''sSpreadsheet files. See Analysis Report A237','A3909',3405,'MEM',NULL,48,'Tumor CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3910,'CMML26','Processed data converted from Beth Wilmot''sSpreadsheet files. See Analysis Report A237','A3910',3405,'MEM',NULL,48,'Tumor CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3911,'CMML28','Processed data converted from Beth Wilmot''sSpreadsheet files. See Analysis Report A237','A3911',3405,'MEM',NULL,48,'Tumor CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3912,'CMML32','Processed data converted from Beth Wilmot''sSpreadsheet files. See Analysis Report A237','A3912',3405,'MEM',NULL,48,'Tumor CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3913,'CMML34','Processed data converted from Beth Wilmot''sSpreadsheet files. See Analysis Report A237','A3913',3405,'MEM',NULL,48,'Tumor CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3914,'CMML36','Processed data converted from Beth Wilmot''sSpreadsheet files. See Analysis Report A237','A3914',3405,'MEM',NULL,48,'Tumor CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3915,'CMML38','Processed data converted from Beth Wilmot''sSpreadsheet files. See Analysis Report A237','A3915',3405,'MEM',NULL,48,'Tumor CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3916,'CMML40','Processed data converted from Beth Wilmot''sSpreadsheet files. See Analysis Report A237','A3916',3405,'MEM',NULL,48,'Tumor CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3917,'CMML42','Processed data converted from Beth Wilmot''sSpreadsheet files. See Analysis Report A237','A3917',3405,'MEM',NULL,48,'Tumor CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3918,'CMML44','Processed data converted from Beth Wilmot''sSpreadsheet files. See Analysis Report A237','A3918',3405,'MEM',NULL,48,'Tumor CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3919,'CMML46','Processed data converted from Beth Wilmot''sSpreadsheet files. See Analysis Report A237','A3919',3405,'MEM',NULL,48,'Tumor CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3920,'CMML48','Processed data converted from Beth Wilmot''sSpreadsheet files. See Analysis Report A237','A3920',3405,'MEM',NULL,48,'Tumor CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3921,'CMML52','Processed data converted from Beth Wilmot''sSpreadsheet files. See Analysis Report A237','A3921',3405,'MEM',NULL,48,'Tumor CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3922,'CMML54','Processed data converted from Beth Wilmot''sSpreadsheet files. See Analysis Report A237','A3922',3405,'MEM',NULL,48,'Tumor CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3923,'CMML07','Processed data converted from Beth Wilmot''sSpreadsheet files. See Analysis Report A237','A3923',3405,'MEM',NULL,48,'Normal CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3924,'CMML13','Processed data converted from Beth Wilmot''sSpreadsheet files. See Analysis Report A237','A3924',3405,'MEM',NULL,48,'Normal CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3925,'CMML19','Processed data converted from Beth Wilmot''sSpreadsheet files. See Analysis Report A237','A3925',3405,'MEM',NULL,48,'Normal CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3926,'CMML23','Processed data converted from Beth Wilmot''sSpreadsheet files. See Analysis Report A237','A3926',3405,'MEM',NULL,48,'Normal CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3927,'CMML31','Processed data converted from Beth Wilmot''sSpreadsheet files. See Analysis Report A237','A3927',3405,'MEM',NULL,48,'Normal CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3928,'CMML33','Processed data converted from Beth Wilmot''sSpreadsheet files. See Analysis Report A237','A3928',3405,'MEM',NULL,48,'Normal CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3929,'CMML35','Processed data converted from Beth Wilmot''sSpreadsheet files. See Analysis Report A237','A3929',3405,'MEM',NULL,48,'Normal CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3930,'CMML37','Processed data converted from Beth Wilmot''sSpreadsheet files. See Analysis Report A237','A3930',3405,'MEM',NULL,48,'Normal CNVs','u0028003','2011-07-18 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3931,'egg_totalRNAFPDrRNA2lanes_readcoverage_notscaled','','A3931',6303,'MEM',69,7,'','Magda','2011-07-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3932,'sperm_smallRNA_ReadCoverage_scaled','','A3932',6303,'MEM',69,7,'','Magda','2011-08-02 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3934,'7741X1_7623X1 Read Coverage','','A3934',6300,'MEM',43,7,'','Andrew','2011-08-04 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3935,'Novel_Transcripts_7741X1_7623X1_x','','A3935',6300,'MEM',43,7,'','Andrew','2011-08-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3940,'Novel_Transcripts_7741X1_7623X1_gtf','','A3940',6300,'MEM',43,7,'','Andrew','2011-08-10 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3941,'Novel_Transcripts_7741X1_7623X1','','A3941',6300,'MEM',43,7,'','Andrew','2011-08-10 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3942,'Novel_Genes','','A3942',6300,'INST',43,7,'','Andrew','2011-08-10 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3943,'Novel_Isoforms','','A3943',6300,'MEM',43,7,'','Andrew','2011-08-10 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3944,'Novel_Genes_gtf',NULL,'A3944',6300,'MEM',43,7,NULL,'Andrew','2011-08-10 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3945,'Novel_Isoforms_gtf',NULL,'A3945',6300,'MEM',43,7,NULL,'Andrew','2011-08-10 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3948,'7256X2_Sepsis_PLT','','A3948',6300,'MEM',40,30,'','tolley','2011-08-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3949,'7280X1_Adult_BL_PMN','','A3949',6300,'MEM',40,30,'','tolley','2011-08-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3958,'sperm_unmethylatedC','<P ALIGN=""LEFT""># 7781X3 sperm converted data </P><P ALIGN=""LEFT"">java -jar -Xmx30G /Volumes/MacHome/magda/Documents/USeq_7.8.2/Apps/MergePointData -p /Volumes/MacHome/magda/Documents/Bis-seq/Bs-Seq_zv9/7781X3sperm/NovoalignBisulfiteParser/trimmedadapters/MergedHitCountPointData/ConvertedC/ -m -c -s /Volumes/MacHome/magda/Documents/Bis-seq/Bs-Seq_zv9/7781X3sperm/NovoalignBisulfiteParser/trimmedadapters/MergePointData_converted_sperm/</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">java -jar -Xmx30G /Volumes/MacHome/magda/Documents/USeq_7.8.2/Apps/Bar2USeq -f /Volumes/MacHome/magda/Documents/Bis-seq/Bs-Seq_zv9/7781X3sperm/NovoalignBisulfiteParser/trimmedadapters/MergePointData_converted_sperm/</P>','A3958',6303,'MEM',69,7,'','Magda','2011-08-25 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3959,'sperm_methylatedC','<P ALIGN=""LEFT"">java -jar -Xmx30G /Volumes/MacHome/magda/Documents/USeq_7.8.2/Apps/MergePointData -p /Volumes/MacHome/magda/Documents/Bis-seq/Bs-Seq_zv9/7781X3sperm/NovoalignBisulfiteParser/trimmedadapters/MergedHitCountPointData/NonConvertedC/ -m -c -s /Volumes/MacHome/magda/Documents/Bis-seq/Bs-Seq_zv9/7781X3sperm/NovoalignBisulfiteParser/trimmedadapters/MergePointData_nonconverted_sperm/</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">java -jar -Xmx30G /Volumes/MacHome/magda/Documents/USeq_7.8.2/Apps/Bar2USeq -f /Volumes/MacHome/magda/Documents/Bis-seq/Bs-Seq_zv9/7781X3sperm/NovoalignBisulfiteParser/trimmedadapters/MergePointData_nonconverted_sperm/</P>','A3959',6303,'MEM',69,7,'','Magda','2011-08-25 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3960,'64_methylatedC','<P ALIGN=""LEFT"">java -jar -Xmx30G /Volumes/MacHome/magda/Documents/USeq_7.8.2/Apps/MergePointData -p /Volumes/MacHome/magda/Documents/Bis-seq/Bs-Seq_zv9/7781X464/NBP/MergedHitCountPointData/NonConvertedC/ -m -c -s /Volumes/MacHome/magda/Documents/Bis-seq/Bs-Seq_zv9/7781X464/NBP/MergePointData_nonconverted_64/</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">java -jar -Xmx30G /Volumes/MacHome/magda/Documents/USeq_7.8.2/Apps/Bar2USeq -f /Volumes/MacHome/magda/Documents/Bis-seq/Bs-Seq_zv9/7781X464/NBP/MergePointData_nonconverted_64/</P>','A3960',6303,'MEM',69,7,'','Magda','2011-08-25 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3961,'64_unmethylatedC','<P ALIGN=""LEFT"">java -jar -Xmx30G /Volumes/MacHome/magda/Documents/USeq_7.8.2/Apps/MergePointData -p /Volumes/MacHome/magda/Documents/Bis-seq/Bs-Seq_zv9/7781X464/NBP/MergedHitCountPointData/ConvertedC/ -m -c -s /Volumes/MacHome/magda/Documents/Bis-seq/Bs-Seq_zv9/7781X464/NBP/MergePointData_converted_64/</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">java -jar -Xmx30G /Volumes/MacHome/magda/Documents/USeq_7.8.2/Apps/Bar2USeq -f /Volumes/MacHome/magda/Documents/Bis-seq/Bs-Seq_zv9/7781X464/NBP/MergePointData_converted_64/</P>','A3961',6303,'MEM',69,7,'','Magda','2011-08-25 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3962,'256_methylatedC','<P ALIGN=""LEFT"">java -jar -Xmx30G /Volumes/MacHome/magda/Documents/USeq_7.8.2/Apps/MergePointData -p /Volumes/MacHome/magda/Documents/Bis-seq/Bs-Seq_zv9/7781X5256/NovoalignBisulfiteParser/trimmedadaptors/MergedHitCountPointData/NonConvertedC/ -m -c -s /Volumes/MacHome/magda/Documents/Bis-seq/Bs-Seq_zv9/7781X5256/NovoalignBisulfiteParser/trimmedadaptors/MergePointData_nonconverted_256/</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">java -jar -Xmx30G /Volumes/MacHome/magda/Documents/USeq_7.8.2/Apps/Bar2USeq -f /Volumes/MacHome/magda/Documents/Bis-seq/Bs-Seq_zv9/7781X5256/NovoalignBisulfiteParser/trimmedadaptors/MergePointData_nonconverted_256/</P>','A3962',6303,'MEM',69,7,'','Magda','2011-08-25 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3963,'256_unmethylatedC','<P ALIGN=""LEFT"">java -jar -Xmx30G /Volumes/MacHome/magda/Documents/USeq_7.8.2/Apps/MergePointData -p /Volumes/MacHome/magda/Documents/Bis-seq/Bs-Seq_zv9/7781X5256/NovoalignBisulfiteParser/trimmedadaptors/MergedHitCountPointData/ConvertedC/ -m -c -s /Volumes/MacHome/magda/Documents/Bis-seq/Bs-Seq_zv9/7781X5256/NovoalignBisulfiteParser/trimmedadaptors/MergePointData_converted_256/</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">java -jar -Xmx30G /Volumes/MacHome/magda/Documents/USeq_7.8.2/Apps/Bar2USeq -f /Volumes/MacHome/magda/Documents/Bis-seq/Bs-Seq_zv9/7781X5256/NovoalignBisulfiteParser/trimmedadaptors/MergePointData_converted_256/</P>','A3963',6303,'MEM',69,7,'','Magda','2011-08-25 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3964,'sphere_methylatedC','<P ALIGN=""LEFT"">java -jar -Xmx30G /Volumes/MacHome/magda/Documents/USeq_7.8.2/Apps/MergePointData -p /Volumes/MacHome/magda/Documents/Bis-seq/Bs-Seq_zv9/7781X6sphere/NovoalignBisulfiteParser/trimmedadaptors/MergedHitCountPointData/NonConvertedC/ -m -c -s /Volumes/MacHome/magda/Documents/Bis-seq/Bs-Seq_zv9/7781X6sphere/NovoalignBisulfiteParser/trimmedadaptors/MergePointData_nonconverted_sphere/</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">java -jar -Xmx30G /Volumes/MacHome/magda/Documents/USeq_7.8.2/Apps/Bar2USeq -f /Volumes/MacHome/magda/Documents/Bis-seq/Bs-Seq_zv9/7781X6sphere/NovoalignBisulfiteParser/trimmedadaptors/MergePointData_nonconverted_sphere/</P>','A3964',6303,'MEM',69,7,'','Magda','2011-08-25 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3965,'sphere_unmethylated','<P ALIGN=""LEFT"">java -jar -Xmx30G /Volumes/MacHome/magda/Documents/USeq_7.8.2/Apps/MergePointData -p /Volumes/MacHome/magda/Documents/Bis-seq/Bs-Seq_zv9/7781X6sphere/NovoalignBisulfiteParser/trimmedadaptors/MergedHitCountPointData/ConvertedC/ -m -c -s /Volumes/MacHome/magda/Documents/Bis-seq/Bs-Seq_zv9/7781X6sphere/NovoalignBisulfiteParser/trimmedadaptors/MergePointData_converted_sphere/</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">java -jar -Xmx30G /Volumes/MacHome/magda/Documents/USeq_7.8.2/Apps/Bar2USeq -f /Volumes/MacHome/magda/Documents/Bis-seq/Bs-Seq_zv9/7781X6sphere/NovoalignBisulfiteParser/trimmedadaptors/MergePointData_converted_sphere/</P>','A3965',6303,'MEM',69,7,'','Magda','2011-08-25 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3972,'sphere_basefractionnonconverted','','A3972',6303,'MEM',69,7,'','Magda','2011-09-06 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3973,'sperm_BaseFractionNonConverted','','A3973',6303,'MEM',69,7,'','Magda','2011-09-06 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3974,'256_BaseFractionNonConverted',NULL,'A3974',6303,'MEM',69,7,NULL,'Magda','2011-09-06 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3975,'64_BaseFractionNonConverted','','A3975',6303,'MEM',69,7,'','Magda','2011-09-06 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3976,'256_BaseFractionNonConverted_CG',NULL,'A3976',6303,'MEM',69,7,NULL,'Magda','2011-09-07 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3981,'7741X1 7623X1 merged',NULL,'A3981',6300,'MEM',43,30,NULL,'oler','2011-09-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3982,'Novel Genes',NULL,'A3982',6300,'MEM',43,30,NULL,'oler','2011-09-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3984,'H3K4me1 megakaryocytes','<P ALIGN=""LEFT"">GSM610328 	CMK-K4ME1</P><P ALIGN=""LEFT"">	</P><P ALIGN=""LEFT"">    * Pencovich N, Jaschek R, Tanay A, Groner Y. Dynamic combinatorial interactions of RUNX1 and cooperating partners regulates megakaryocytic differentiation in cell line models. Blood 2011 Jan 6;117(1):e1-14. PMID: 20959602</P><P ALIGN=""LEFT""></P>','A3984',6300,'MEM',43,30,'','oler','2011-09-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3985,'RUNX1 megakaryocytes\r','<P ALIGN=""LEFT"">GSM610330 	CMK-RUNX1</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">    * Pencovich N, Jaschek R, Tanay A, Groner Y. Dynamic combinatorial interactions of RUNX1 and cooperating partners regulates megakaryocytic differentiation in cell line models. Blood 2011 Jan 6;117(1):e1-14. PMID: 20959602</P><P ALIGN=""LEFT""></P>','A3985',6300,'MEM',43,30,'','oler','2011-09-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3986,'RUNX1 Meg','','A3986',6300,'MEM',43,30,'','oler','2011-09-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3987,'GATA1 Meg',NULL,'A3987',6300,'MEM',43,30,NULL,'oler','2011-09-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3988,'GATA2 Meg',NULL,'A3988',6300,'MEM',43,30,NULL,'oler','2011-09-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3989,'FLI1 Meg',NULL,'A3989',6300,'MEM',43,30,NULL,'oler','2011-09-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3990,'SCL Meg',NULL,'A3990',6300,'MEM',43,30,NULL,'oler','2011-09-09 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(3999,'PMN_Term_BL','','A3999',6300,'MEM',40,30,'PMNs from term infant','tolley','2011-09-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/');

INSERT INTO dbo.gpAnnotation 
(
idAnnotation,
name,
description,
fileName,
idGenomeVersion,
codeVisibility,
idUser,
idUserGroup,
summary,
createdBy,
createDate,
isLoaded,
idInstitute,
dataPath
)
VALUES 
(4000,'PMN_Term_PAF','','A4000',6300,'MEM',40,30,'','tolley','2011-09-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4002,'T0 2ndRnd','','A4002',6300,'MEM',NULL,28,'','u0028003','2011-09-13 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4003,'T1',NULL,'A4003',6300,'MEM',NULL,28,NULL,'u0028003','2011-09-13 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4004,'T2',NULL,'A4004',6300,'MEM',NULL,28,NULL,'u0028003','2011-09-13 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4005,'T3',NULL,'A4005',6300,'MEM',NULL,28,NULL,'u0028003','2011-09-13 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4006,'C0 2ndRnd',NULL,'A4006',6300,'MEM',NULL,28,NULL,'u0028003','2011-09-13 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4007,'C1',NULL,'A4007',6300,'MEM',NULL,28,NULL,'u0028003','2011-09-13 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4008,'C2',NULL,'A4008',6300,'MEM',NULL,28,NULL,'u0028003','2011-09-13 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4009,'C3',NULL,'A4009',6300,'MEM',NULL,28,NULL,'u0028003','2011-09-13 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4082,'adult female no head ovary-1','Relative RNA-Seq read/depth coverage tracks from whole female adult zebrafish (zv9) without head or ovary (37bp paired-end reads)','A4082',6303,'MEM',NULL,49,'read/depth coverage','u0785353','2011-09-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4083,'adult male no head testis-1','Relative RNA-Seq read/depth coverage tracks from whole male adult zebrafish (zv9) without head or testis (37bp paired-end reads)','A4083',6303,'MEM',NULL,49,'read/depth coverage','u0785353','2011-09-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4084,'adult female no head ovary-2','Relative RNA-Seq read/depth coverage tracks from whole female adult zebrafish (zv9) without head or ovary (76bp paired-end reads)','A4084',6303,'MEM',NULL,49,'read/depth coverage','u0785353','2011-09-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4085,'adult male no head testis-2','Relative RNA-Seq read/depth coverage tracks from whole male adult zebrafish (zv9) without head or testis (76bp paired-end reads)','A4085',6303,'MEM',NULL,49,'read/depth coverage','u0785353','2011-09-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4086,'embryo 3dpf-1','Relative RNA-Seq read/depth coverage tracks from zebrafish (zv9) embryo at 3 dpf (36bp paired-end reads)','A4086',6303,'MEM',NULL,49,'read/depth coverage','u0785353','2011-09-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4087,'embryo 3dpf-2','Relative RNA-Seq read/depth coverage tracks from zebrafish (zv9) embryo at 3 dpf (36bp paired-end reads)','A4087',6303,'MEM',NULL,49,'read/depth coverage','u0785353','2011-09-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4088,'adult male head-1','Relative RNA-Seq read/depth coverage tracks from male adult zebrafish (zv9) head (36bp paired-end reads)','A4088',6303,'MEM',NULL,49,'read/depth coverage','u0785353','2011-09-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4089,'adult male head-2','Relative RNA-Seq read/depth coverage tracks from male adult zebrafish (zv9) head (36bp paired-end reads)','A4089',6303,'MEM',NULL,49,'read/depth coverage','u0785353','2011-09-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4090,'adult ovary-1','Relative RNA-Seq read/depth coverage tracks from adult zebrafish (zv9) ovary (36bp paired-end reads)','A4090',6303,'MEM',NULL,49,'read/depth coverage','u0785353','2011-09-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4091,'adult ovary-2','Relative RNA-Seq read/depth coverage tracks from adult zebrafish (zv9) ovary (36bp paired-end reads)','A4091',6303,'MEM',NULL,49,'read/depth coverage','u0785353','2011-09-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4092,'adult ovary-3','Relative RNA-Seq read/depth coverage tracks from adult zebrafish (zv9) ovary (36bp paired-end reads)','A4092',6303,'MEM',NULL,49,'read/depth coverage','u0785353','2011-09-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4093,'embryo 1dpf-1','Relative RNA-Seq read/depth coverage tracks from zebrafish (zv9) embryo at 1 dpf (37bp paired-end reads)','A4093',6303,'MEM',NULL,49,'read/depth coverage','u0785353','2011-09-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4094,'embryo 1dpf-2','Relative RNA-Seq read/depth coverage tracks from zebrafish (zv9) embryo at 1 dpf (37bp paired-end reads)','A4094',6303,'MEM',NULL,49,'read/depth coverage','u0785353','2011-09-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4095,'embryo 1dpf-3','Relative RNA-Seq read/depth coverage tracks from zebrafish (zv9) embryo at 1 dpf (37bp paired-end reads)','A4095',6303,'MEM',NULL,49,'read/depth coverage','u0785353','2011-09-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4096,'embryo 2dpf-1','Relative RNA-Seq read/depth coverage tracks from zebrafish (zv9) embryo at 2 dpf (37bp paired-end reads)','A4096',6303,'MEM',NULL,49,'read/depth coverage','u0785353','2011-09-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4097,'embryo 2dpf-2','Relative RNA-Seq read/depth coverage tracks from zebrafish (zv9) embryo at 2 dpf (37bp paired-end reads)','A4097',6303,'MEM',NULL,49,'read/depth coverage','u0785353','2011-09-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4098,'embryo 2dpf-3','Relative RNA-Seq read/depth coverage tracks from zebrafish (zv9) embryo at 2 dpf (37bp paired-end reads)','A4098',6303,'MEM',NULL,49,'read/depth coverage','u0785353','2011-09-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4099,'embryo 3dpf-3','Relative RNA-Seq read/depth coverage tracks from zebrafish (zv9) embryo at 3 dpf (37bp paired-end reads)','A4099',6303,'MEM',NULL,49,'read/depth coverage','u0785353','2011-09-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4100,'embryo 5dpf-1','Relative RNA-Seq read/depth coverage tracks from zebrafish (zv9) embryo at 5 dpf (37bp paired-end reads)','A4100',6303,'MEM',NULL,49,'read/depth coverage','u0785353','2011-09-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4101,'embryo 14dpf-1','Relative RNA-Seq read/depth coverage tracks from zebrafish (zv9) embryo at 14 dpf (37bp paired-end reads)','A4101',6303,'MEM',NULL,49,'read/depth coverage','u0785353','2011-09-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4102,'adult female head-1','Relative RNA-Seq read/depth coverage tracks from female adult zebrafish (zv9) head (37bp paired-end reads)','A4102',6303,'MEM',NULL,49,'read/depth coverage','u0785353','2011-09-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4103,'adult ovary-4','Relative RNA-Seq read/depth coverage tracks from adult zebrafish (zv9) ovary (37bp paired-end reads)','A4103',6303,'MEM',NULL,49,'read/depth coverage','u0785353','2011-09-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4104,'adult ovary-5','Relative RNA-Seq read/depth coverage tracks from adult zebrafish (zv9) ovary (37bp paired-end reads)','A4104',6303,'MEM',NULL,49,'read/depth coverage','u0785353','2011-09-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4105,'embryo 5dpf-2','Relative RNA-Seq read/depth coverage tracks from zebrafish (zv9) embryo at 5 dpf (54bp paired-end reads)','A4105',6303,'MEM',NULL,49,'read/depth coverage','u0785353','2011-09-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4106,'embryo 5dpf-3','Relative RNA-Seq read/depth coverage tracks from zebrafish (zv9) embryo at 5 dpf (54bp paired-end reads)','A4106',6303,'MEM',NULL,49,'read/depth coverage','u0785353','2011-09-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4107,'adult female head-2','Relative RNA-Seq read/depth coverage tracks from female adult zebrafish (zv9) head (54bp paired-end reads)','A4107',6303,'MEM',NULL,49,'read/depth coverage','u0785353','2011-09-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4108,'adult female head-3','Relative RNA-Seq read/depth coverage tracks from female adult zebrafish (zv9) head (54bp paired-end reads)','A4108',6303,'MEM',NULL,49,'read/depth coverage','u0785353','2011-09-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4109,'adult male head-3','Relative RNA-Seq read/depth coverage tracks from male adult zebrafish (zv9) head (54bp paired-end reads)','A4109',6303,'MEM',NULL,49,'read/depth coverage','u0785353','2011-09-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4110,'adult male head-4','Relative RNA-Seq read/depth coverage tracks from male adult zebrafish (zv9) head (54bp paired-end reads)','A4110',6303,'MEM',NULL,49,'read/depth coverage','u0785353','2011-09-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4111,'embryo 1dpf-4','Relative RNA-Seq read/depth coverage tracks from zebrafish (zv9) embryo at 1 dpf (76bp paired-end reads)','A4111',6303,'MEM',NULL,49,'read/depth coverage','u0785353','2011-09-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4112,'embryo 1dpf-5','Relative RNA-Seq read/depth coverage tracks from zebrafish (zv9) embryo at 1 dpf (76bp paired-end reads)','A4112',6303,'MEM',NULL,49,'read/depth coverage','u0785353','2011-09-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4113,'embryo 2dpf-4','Relative RNA-Seq read/depth coverage tracks from zebrafish (zv9) embryo at 2 dpf (76bp paired-end reads)','A4113',6303,'MEM',NULL,49,'read/depth coverage','u0785353','2011-09-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4114,'embryo 3dpf-4','Relative RNA-Seq read/depth coverage tracks from zebrafish (zv9) embryo at 3 dpf (76bp paired-end reads)','A4114',6303,'MEM',NULL,49,'read/depth coverage','u0785353','2011-09-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4115,'embryo 5dpf-4','Relative RNA-Seq read/depth coverage tracks from zebrafish (zv9) embryo at 5 dpf (76bp paired-end reads)','A4115',6303,'MEM',NULL,49,'read/depth coverage','u0785353','2011-09-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4116,'embryo 5dpf-5','Relative RNA-Seq read/depth coverage tracks from zebrafish (zv9) embryo at 5 dpf (76bp paired-end reads)','A4116',6303,'MEM',NULL,49,'read/depth coverage','u0785353','2011-09-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4117,'adult female head-4','Relative RNA-Seq read/depth coverage tracks from female adult zebrafish (zv9) head (76bp paired-end reads)','A4117',6303,'MEM',NULL,49,'read/depth coverage','u0785353','2011-09-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4118,'adult female head-5','Relative RNA-Seq read/depth coverage tracks from female adult zebrafish (zv9) head (76bp paired-end reads)','A4118',6303,'MEM',NULL,49,'read/depth coverage','u0785353','2011-09-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4119,'adult male head-5','Relative RNA-Seq read/depth coverage tracks from male adult zebrafish (zv9) head (76bp paired-end reads)','A4119',6303,'MEM',NULL,49,'read/depth coverage','u0785353','2011-09-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4120,'adult ovary-6','Relative RNA-Seq read/depth coverage tracks from adult zebrafish (zv9) ovary (76bp paired-end reads)','A4120',6303,'MEM',NULL,49,'read/depth coverage','u0785353','2011-09-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4121,'adult ovary-7','Relative RNA-Seq read/depth coverage tracks from adult zebrafish (zv9) ovary (76bp paired-end reads)','A4121',6303,'MEM',NULL,49,'read/depth coverage','u0785353','2011-09-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4122,'1f1',NULL,'A4122',3806,'MEM',NULL,17,NULL,'bmilash','2011-09-30 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4123,'1f2',NULL,'A4123',3806,'MEM',NULL,17,NULL,'bmilash','2011-09-30 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4124,'1f3',NULL,'A4124',3806,'MEM',NULL,17,NULL,'bmilash','2011-09-30 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4125,'1f4',NULL,'A4125',3806,'MEM',NULL,17,NULL,'bmilash','2011-09-30 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4126,'4f1',NULL,'A4126',3806,'MEM',NULL,17,NULL,'bmilash','2011-09-30 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4127,'4f2',NULL,'A4127',3806,'MEM',NULL,17,NULL,'bmilash','2011-09-30 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4128,'4f3',NULL,'A4128',3806,'MEM',NULL,17,NULL,'bmilash','2011-09-30 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4129,'4f4',NULL,'A4129',3806,'MEM',NULL,17,NULL,'bmilash','2011-09-30 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4130,'1s1',NULL,'A4130',3806,'MEM',NULL,17,NULL,'bmilash','2011-09-30 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4131,'1s2',NULL,'A4131',3806,'MEM',NULL,17,NULL,'bmilash','2011-09-30 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4132,'1s3',NULL,'A4132',3806,'MEM',NULL,17,NULL,'bmilash','2011-09-30 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4133,'1s4',NULL,'A4133',3806,'MEM',NULL,17,NULL,'bmilash','2011-09-30 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4134,'4s1',NULL,'A4134',3806,'MEM',NULL,17,NULL,'bmilash','2011-09-30 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4135,'4s2',NULL,'A4135',3806,'MEM',NULL,17,NULL,'bmilash','2011-09-30 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4136,'4s3',NULL,'A4136',3806,'MEM',NULL,17,NULL,'bmilash','2011-09-30 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4137,'4s4',NULL,'A4137',3806,'MEM',NULL,17,NULL,'bmilash','2011-09-30 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4138,'BaseFractionNonconverted','','A4138',3806,'MEM',14,7,'','CairnsLab','2011-10-05 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4139,'BaseFDRNonConverted','','A4139',3806,'MEM',14,7,'','CairnsLab','2011-10-05 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4140,'WindowFractionNonConverted','','A4140',3806,'MEM',14,7,'','CairnsLab','2011-10-05 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4141,'BaseFractionNonconverted','','A4141',3806,'MEM',14,7,'','CairnsLab','2011-10-05 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4142,'BaseFDRNonConverted','','A4142',3806,'MEM',14,7,'','CairnsLab','2011-10-05 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4143,'WindowFractionNonConverted','','A4143',3806,'MEM',14,7,'','CairnsLab','2011-10-05 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4144,'BaseFractionNonConverted',NULL,'A4144',3806,'MEM',14,7,NULL,'CairnsLab','2011-10-05 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4145,'BaseFDRNonConverted',NULL,'A4145',3806,'MEM',14,7,NULL,'CairnsLab','2011-10-05 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4146,'WindowFractionNonconverted',NULL,'A4146',3806,'MEM',14,7,NULL,'CairnsLab','2011-10-05 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4147,'X4 Brain4, Glu+, Gla1','','A4147',3806,'MEM',NULL,50,'','u0028003','2011-10-10 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4148,'X6 Brain6, Glu-, Gla1','','A4148',3806,'MEM',NULL,50,'','u0028003','2011-10-10 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4149,'BinPVal',NULL,'A4149',3806,'MEM',NULL,50,NULL,'u0028003','2011-10-10 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4150,'Log2Ratio',NULL,'A4150',3806,'MEM',NULL,50,NULL,'u0028003','2011-10-10 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4151,'GalI 500',NULL,'A4151',3806,'MEM',NULL,50,NULL,'u0028003','2011-10-10 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4152,'MspI 500',NULL,'A4152',3806,'MEM',NULL,50,NULL,'u0028003','2011-10-10 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4153,'MspJI 500',NULL,'A4153',3806,'MEM',NULL,50,NULL,'u0028003','2011-10-10 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4154,'TaqAlphaI 500',NULL,'A4154',3806,'MEM',NULL,50,NULL,'u0028003','2011-10-10 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4157,'muts_8474X2','','A4157',6303,'MEM',NULL,51,'','bmilash','2011-10-11 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4158,'sibs_8474X1','','A4158',6303,'MEM',NULL,51,'','bmilash','2011-10-11 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4166,'1 cell embryo','Relative RNA-Seq read depth coverage tracks from 1 cell embryo','A4166',6303,'MEM',NULL,49,'read depth coverage','u0785353','2011-10-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4167,'16 cell embryo','Relative RNA-Seq read depth coverage tracks from 16 cell embryo','A4167',6303,'MEM',NULL,49,'read depth coverage','u0785353','2011-10-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4168,'512 cell embryo','Relative RNA-Seq read depth coverage tracks from 512 cell embryo','A4168',6303,'MEM',NULL,49,'read depth coverage','u0785353','2011-10-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4169,'50% epiboly embryo','Relative RNA-Seq read depth coverage tracks from 50% embryo','A4169',6303,'MEM',NULL,49,'read depth coverage','u0785353','2011-10-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4171,'unfertilized egg','Relative RNA-Seq read depth coverage tracks from unfertilized egg','A4171',6303,'MEM',NULL,49,'read depth coverage','u0785353','2011-10-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4172,'1 cell embryo','Relative RNA-Seq read depth coverage tracks from 1 cell embryo','A4172',6303,'MEM',NULL,49,'read depth coverage','u0785353','2011-10-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4173,'16-32 cell embryo','Relative RNA-Seq read depth coverage tracks from 16-32 cell embryo','A4173',6303,'MEM',NULL,49,'read depth coverage','u0785353','2011-10-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4174,'128-256 cell embryo','Relative RNA-Seq read depth coverage tracks from 128/256 cell embryo','A4174',6303,'MEM',NULL,49,'read depth coverage','u0785353','2011-10-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4175,'3.5hpf embryo','Relative RNA-Seq read depth coverage tracks from 3.5hpf embryo','A4175',6303,'MEM',NULL,49,'read depth coverage','u0785353','2011-10-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4176,'5.3hpf embryo','Relative RNA-Seq read depth coverage tracks from 5.3hpf embryo','A4176',6303,'MEM',NULL,49,'read depth coverage','u0785353','2011-10-12 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4196,'X1 Brain 1, Glu+, Msp1','','A4196',3806,'MEM',NULL,50,'','u0028003','2011-10-13 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4197,'X2 Brain 2, Glu+, Taqa1','','A4197',3806,'MEM',NULL,50,'','u0028003','2011-10-13 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4198,'X3 Brain3, Glu+, Mspj1','','A4198',3806,'MEM',NULL,50,'','u0028003','2011-10-13 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4199,'X5 Brain5, Glu-, Msp1','','A4199',3806,'MEM',NULL,50,'','u0028003','2011-10-13 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4200,'X7 Liver 1, Glu+, Msp1','','A4200',3806,'MEM',NULL,50,'','u0028003','2011-10-13 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4201,'X8 Liver 2, Glu+, Taqa1','','A4201',3806,'MEM',NULL,50,'','u0028003','2011-10-13 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4202,'X9 Liver 3, Glu+, Mspj1','','A4202',3806,'MEM',NULL,50,'','u0028003','2011-10-13 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4203,'X10 Liver 4, Glu+, Gla1','','A4203',3806,'MEM',NULL,50,'','u0028003','2011-10-13 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4204,'X11 Liver 5, Glu-, Msp1','','A4204',3806,'MEM',NULL,50,'','u0028003','2011-10-13 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4205,'X12 Liver 6, Glu-, Gla1','','A4205',3806,'MEM',NULL,50,'','u0028003','2011-10-13 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4207,'X1',NULL,'A4207',3806,'MEM',NULL,50,NULL,'u0028003','2011-10-13 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4208,'X2',NULL,'A4208',3806,'MEM',NULL,50,NULL,'u0028003','2011-10-13 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4209,'X3',NULL,'A4209',3806,'MEM',NULL,50,NULL,'u0028003','2011-10-13 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4210,'BinPVal',NULL,'A4210',3806,'MEM',NULL,50,NULL,'u0028003','2011-10-13 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4211,'Log2Ratio',NULL,'A4211',3806,'MEM',NULL,50,NULL,'u0028003','2011-10-13 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4214,'X4',NULL,'A4214',3806,'MEM',NULL,50,NULL,'u0028003','2011-10-13 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4215,'X5',NULL,'A4215',3806,'MEM',NULL,50,NULL,'u0028003','2011-10-13 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4216,'X6',NULL,'A4216',3806,'MEM',NULL,50,NULL,'u0028003','2011-10-13 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4217,'X7',NULL,'A4217',3806,'MEM',NULL,50,NULL,'u0028003','2011-10-13 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4222,'X8','','A4222',3806,'MEM',NULL,50,'','u0028003','2011-10-13 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4223,'X9',NULL,'A4223',3806,'MEM',NULL,50,NULL,'u0028003','2011-10-13 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4224,'X10','','A4224',3806,'MEM',NULL,50,'','u0028003','2011-10-13 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4225,'X11',NULL,'A4225',3806,'MEM',NULL,50,NULL,'u0028003','2011-10-13 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4226,'X12','','A4226',3806,'MEM',NULL,50,'','u0028003','2011-10-13 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4228,'SuperStrong_CGI','<P ALIGN=""LEFT"">O/E &gt;1 </P>','A4228',6303,'MEM',69,7,'','Magda','2011-10-13 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4258,'treatment rep0','','A4258',6303,'PUBLIC',NULL,49,'','u0785353','2011-10-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4259,'treatment rep1','','A4259',6303,'PUBLIC',NULL,49,'','u0785353','2011-10-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4260,'control rep0','','A4260',6303,'PUBLIC',NULL,49,'','u0785353','2011-10-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4261,'control rep1','','A4261',6303,'PUBLIC',NULL,49,'','u0785353','2011-10-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4262,'control combine','','A4262',6303,'PUBLIC',NULL,49,'','u0785353','2011-10-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4263,'treatment combine','','A4263',6303,'PUBLIC',NULL,49,'','u0785353','2011-10-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4264,'control combine','','A4264',6303,'MEM',NULL,49,'','u0785353','2011-10-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4265,'control rep0','','A4265',6303,'MEM',NULL,49,'','u0785353','2011-10-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4266,'control rep1','','A4266',6303,'MEM',NULL,49,'','u0785353','2011-10-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4267,'treatment rep0','','A4267',6303,'MEM',NULL,49,'','u0785353','2011-10-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4268,'treatment rep1','','A4268',6303,'MEM',NULL,49,'','u0785353','2011-10-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4269,'treatment combine','','A4269',6303,'MEM',NULL,49,'','u0785353','2011-10-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4271,'single cell vs 16 cell embryo','','A4271',6303,'MEM',NULL,49,'','u0785353','2011-10-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4275,'hg19 test bam',NULL,'A4275',6300,'MEM',43,7,NULL,'oler','2011-10-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4276,'test bb',NULL,'A4276',6300,'MEM',43,7,NULL,'oler','2011-10-19 00:00:00','N',NULL,'/Data02/GenoPub/GenoPubData/'),
(4278,'CKIT(+)_8132_X3_Unique','','A4278',3806,'MEM',14,7,'','CairnsLab','2011-10-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4279,'CKIT(+)_Repeated_1-10Reads','','A4279',3806,'MEM',14,7,'','CairnsLab','2011-10-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4280,'CKIT(-)_8132X4_Unique','','A4280',3806,'MEM',14,7,'','CairnsLab','2011-10-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4281,'CKIT(1)_8132X4_Repeats','','A4281',3806,'MEM',14,7,'','CairnsLab','2011-10-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4282,'Spermatid_8132X2_Unique','','A4282',3806,'MEM',14,7,'','CairnsLab','2011-10-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4283,'Spermatid_8132X2_Repeats',NULL,'A4283',3806,'MEM',14,7,NULL,'CairnsLab','2011-10-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4284,'Spermatocyte_8132X1_Unique','','A4284',3806,'MEM',14,7,'','CairnsLab','2011-10-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4285,'Spermatocyte_8132X1_Repeats','','A4285',3806,'MEM',14,7,'','CairnsLab','2011-10-19 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4286,'Adult male wo head or testis vs 1dpf embryo','','A4286',6303,'MEM',NULL,49,'List of FDR-adjusted genes exhibiting differential expression in this comparison','u0785353','2011-10-20 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4287,'Adult male wo head or testis vs 2dpf embryo',NULL,'A4287',6303,'MEM',NULL,49,NULL,'u0785353','2011-10-20 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4288,'Adult male wo head or testis vs 3dpf embryo',NULL,'A4288',6303,'MEM',NULL,49,NULL,'u0785353','2011-10-20 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4289,'Adult male wo head or testis vs 5 dpf embryo',NULL,'A4289',6303,'MEM',NULL,49,NULL,'u0785353','2011-10-20 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4290,'Adult male wo head or testis vs 14dpf embryo',NULL,'A4290',6303,'MEM',NULL,49,NULL,'u0785353','2011-10-20 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4291,'Adult male wo head or testis vs adult male head',NULL,'A4291',6303,'MEM',NULL,49,NULL,'u0785353','2011-10-20 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4292,'Adult male wo head or testis vs adult female wo head or ovary',NULL,'A4292',6303,'MEM',NULL,49,NULL,'u0785353','2011-10-20 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4293,'Adult female wo head or ovary vs adult female head',NULL,'A4293',6303,'MEM',NULL,49,NULL,'u0785353','2011-10-20 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4294,'Adult female wo head or ovary vs adult female ovary',NULL,'A4294',6303,'MEM',NULL,49,NULL,'u0785353','2011-10-20 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4295,'Adult female head vs adult male head',NULL,'A4295',6303,'MEM',NULL,49,NULL,'u0785353','2011-10-20 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4296,'Adult female wo head or ovary vs 1dpf embryo',NULL,'A4296',6303,'MEM',NULL,49,NULL,'u0785353','2011-10-20 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4297,'Adult female wo head or ovary vs 2dpf embryo',NULL,'A4297',6303,'MEM',NULL,49,NULL,'u0785353','2011-10-20 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4298,'Adult female wo head or ovary vs 3 dpf embryo',NULL,'A4298',6303,'MEM',NULL,49,NULL,'u0785353','2011-10-20 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4299,'Adult female wo head or ovary vs 5dpf embryo',NULL,'A4299',6303,'MEM',NULL,49,NULL,'u0785353','2011-10-20 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4300,'Adult female wo head or ovary vs 14dpf embryo',NULL,'A4300',6303,'MEM',NULL,49,NULL,'u0785353','2011-10-20 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4301,'single cell vs 512 cell embryo',NULL,'A4301',6303,'MEM',NULL,49,NULL,'u0785353','2011-10-24 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4302,'single cell vs 50% epiboly embryo','','A4302',6303,'MEM',NULL,49,'','u0785353','2011-10-24 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4303,'16 cell vs 512 cell embryo',NULL,'A4303',6303,'MEM',NULL,49,NULL,'u0785353','2011-10-24 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4304,'16 cell vs 50% epibody embryo',NULL,'A4304',6303,'MEM',NULL,49,NULL,'u0785353','2011-10-24 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4305,'512 cell vs 50% epiboly embryo',NULL,'A4305',6303,'MEM',NULL,49,NULL,'u0785353','2011-10-24 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4306,'unfertilized egg vs single cell embryo',NULL,'A4306',6303,'MEM',NULL,49,NULL,'u0785353','2011-10-24 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4307,'16-32 cell vs 128-256 cell embryo',NULL,'A4307',6303,'MEM',NULL,49,NULL,'u0785353','2011-10-24 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4308,'16-32 cell vs 3.5 hpf embryo','','A4308',6303,'MEM',NULL,49,'','u0785353','2011-10-24 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4309,'16-32 cell vs 5.3 hpf embryo',NULL,'A4309',6303,'MEM',NULL,49,NULL,'u0785353','2011-10-24 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4310,'128-256 cell vs 3.5 hpf embryo',NULL,'A4310',6303,'MEM',NULL,49,NULL,'u0785353','2011-10-24 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4311,'128-256 cell vs 5.3 hpf embryo',NULL,'A4311',6303,'MEM',NULL,49,NULL,'u0785353','2011-10-24 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4312,'3.5 hpf vs 5.3 hpf embryo',NULL,'A4312',6303,'MEM',NULL,49,NULL,'u0785353','2011-10-24 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4315,'BinPVal',NULL,'A4315',3806,'MEM',NULL,50,NULL,'u0028003','2011-10-26 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4316,'Log2Ratio',NULL,'A4316',3806,'MEM',NULL,50,NULL,'u0028003','2011-10-26 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4317,'Log2Ratios',NULL,'A4317',3806,'MEM',NULL,50,NULL,'u0028003','2011-10-26 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4318,'BinPValues',NULL,'A4318',3806,'MEM',NULL,50,NULL,'u0028003','2011-10-26 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4319,'egg_predictedpiRNA_readcoverage_notscaled',NULL,'A4319',6303,'MEM',69,7,NULL,'Magda','2011-10-26 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4320,'Top 100 Enriched Msp1 Regions',NULL,'A4320',3806,'MEM',NULL,50,NULL,'u0028003','2011-10-26 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4321,'Top 100 Enriched Gla1 Regions',NULL,'A4321',3806,'MEM',NULL,50,NULL,'u0028003','2011-10-26 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4322,'Top 100 Enriched Msp1 Regions',NULL,'A4322',3806,'MEM',NULL,50,NULL,'u0028003','2011-10-26 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4338,'sperm_predictedpiRNA_readcoverage_notscaled',NULL,'A4338',6303,'MEM',69,7,NULL,'Magda','2011-10-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4339,'64_predictedpiRNA_readcoverage_notscaled',NULL,'A4339',6303,'MEM',69,7,NULL,'Magda','2011-10-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4340,'256_predictedpiRNA_readcoverage',NULL,'A4340',6303,'MEM',69,7,NULL,'Magda','2011-10-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4341,'sphere_predictedpiRNA_readcoverage_notscaled',NULL,'A4341',6303,'MEM',69,7,NULL,'Magda','2011-10-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4342,'Strong_CGI',NULL,'A4342',6303,'MEM',69,7,NULL,'Magda','2011-10-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4343,'Intermediate_CGI',NULL,'A4343',6303,'MEM',69,7,NULL,'Magda','2011-10-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4344,'Weak_CGI',NULL,'A4344',6303,'MEM',69,7,NULL,'Magda','2011-10-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4345,'Reapeats_DNA',NULL,'A4345',6303,'MEM',69,7,NULL,'Magda','2011-10-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4346,'Repeats_LINE',NULL,'A4346',6303,'MEM',69,7,NULL,'Magda','2011-10-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4347,'Repeats_lowcomplexity',NULL,'A4347',6303,'MEM',69,7,NULL,'Magda','2011-10-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4348,'Repeats_LTR',NULL,'A4348',6303,'MEM',69,7,NULL,'Magda','2011-10-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4349,'Repeats_Satellite',NULL,'A4349',6303,'MEM',69,7,NULL,'Magda','2011-10-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4350,'Repeats_simplerepeat',NULL,'A4350',6303,'MEM',69,7,NULL,'Magda','2011-10-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4351,'Repeats_SINE',NULL,'A4351',6303,'MEM',69,7,NULL,'Magda','2011-10-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4352,'Repeats_unknown',NULL,'A4352',6303,'MEM',69,7,NULL,'Magda','2011-10-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4353,'Repeats','','A4353',6303,'MEM',69,7,'tim''s repeat masker on zv9','Magda','2011-10-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4354,'Top 100 Enriched Gla1 Regions',NULL,'A4354',3806,'MEM',NULL,50,NULL,'u0028003','2011-10-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4355,'egg_predictedpiRNA_readcoverage_notscaled_stairstep','','A4355',6303,'MEM',69,7,'','Magda','2011-10-27 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4357,'Sperm long RNA x1',NULL,'A4357',3806,'MEM',82,17,NULL,'elizabeth.sloan','2011-10-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4358,'Sperm long RNA x2\r',NULL,'A4358',3806,'MEM',82,17,NULL,'elizabeth.sloan','2011-10-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4359,'Sperm long RNA x3',NULL,'A4359',3806,'MEM',82,17,NULL,'elizabeth.sloan','2011-10-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4360,'Sperm long RNA x4',NULL,'A4360',3806,'MEM',82,17,NULL,'elizabeth.sloan','2011-10-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4361,'H3.3_Collapsed','','A4361',3405,'MEM',14,7,'','CairnsLab','2011-10-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4362,'H3.3AllReads','','A4362',3405,'MEM',14,7,'','CairnsLab','2011-10-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4364,'H3K9me3_Collapsed\r','','A4364',3405,'MEM',14,7,'','CairnsLab','2011-10-28 00:00:00','Y',NULL,'/Data02/GenoPub/GenoPubData/'),
(4377,'wbc',NULL,'A4377',6300,'MEM',43,7,NULL,'oler','2011-10-31 00:00:00','Y',NULL,'/mnt/Bio3Data03/GenoPub/GenoPubData/'),
(4378,'hela pol2 qval',NULL,'A4378',6300,'MEM',43,7,NULL,'oler','2011-10-31 00:00:00','Y',NULL,'/mnt/Bio3Data03/GenoPub/GenoPubData/'),
(4379,'T_Rep0 useq (Pol2)',NULL,'A4379',6300,'MEM',43,7,NULL,'oler','2011-10-31 00:00:00','Y',NULL,'/mnt/Bio3Data03/GenoPub/GenoPubData/'),
(4380,'C_Rep0 input',NULL,'A4380',6300,'MEM',43,7,NULL,'oler','2011-10-31 00:00:00','Y',NULL,'/mnt/Bio3Data03/GenoPub/GenoPubData/'),
(4383,'C_Rep0 useq from bar',NULL,'A4383',6300,'MEM',43,7,NULL,'oler','2011-10-31 00:00:00','Y',NULL,'/mnt/Bio3Data03/GenoPub/GenoPubData/'),
(4384,'T_Rep0 useq from bar',NULL,'A4384',6300,'MEM',43,7,NULL,'oler','2011-10-31 00:00:00','Y',NULL,'/mnt/Bio3Data03/GenoPub/GenoPubData/'),
(4387,'Novel Isoforms',NULL,'A4387',6300,'MEM',43,30,NULL,'oler','2011-10-31 00:00:00','Y',NULL,'/mnt/Bio3Data03/GenoPub/GenoPubData/'),
(4388,'piRNA_zili_ziwi_germline_regions','','A4388',6303,'MEM',69,7,'aligned all piRNA_zili_ziwi bound in germline from 2008 paper from Ketting lab using novoalign rAll option to get all locations','Magda','2011-11-01 00:00:00','Y',NULL,'/mnt/Bio3Data03/GenoPub/GenoPubData/'),
(4389,'egg_smallRNA_ReadCoverage_notscaled','','A4389',6303,'MEM',69,7,'','Magda','2011-11-01 00:00:00','Y',NULL,'/mnt/Bio3Data03/GenoPub/GenoPubData/'),
(4390,'64_smallRNA_ReadCoverage_notscaled',NULL,'A4390',6303,'MEM',69,7,NULL,'Magda','2011-11-01 00:00:00','Y',NULL,'/mnt/Bio3Data03/GenoPub/GenoPubData/'),
(4391,'256_smallRNA_ReadCoverage_notscaled',NULL,'A4391',6303,'MEM',69,7,NULL,'Magda','2011-11-01 00:00:00','Y',NULL,'/mnt/Bio3Data03/GenoPub/GenoPubData/'),
(4392,'sphere_smallRNA_ReadCoverage_notscaled',NULL,'A4392',6303,'MEM',69,7,NULL,'Magda','2011-11-01 00:00:00','Y',NULL,'/mnt/Bio3Data03/GenoPub/GenoPubData/'),
(4393,'sperm_smallRNA_ReadCoverage_notscaled',NULL,'A4393',6303,'MEM',69,7,NULL,'Magda','2011-11-01 00:00:00','Y',NULL,'/mnt/Bio3Data03/GenoPub/GenoPubData/'),
(4394,'egg_predictedpiRNA_uniqueregions',NULL,'A4394',6303,'MEM',69,7,NULL,'Magda','2011-11-01 00:00:00','Y',NULL,'/mnt/Bio3Data03/GenoPub/GenoPubData/'),
(4395,'256_predictedpiRNA_uniqueregions','','A4395',6303,'MEM',69,7,'','Magda','2011-11-01 00:00:00','Y',NULL,'/mnt/Bio3Data03/GenoPub/GenoPubData/'),
(4396,'sphere_predictedpiRNA_uniqueregions','','A4396',6303,'MEM',69,7,'','Magda','2011-11-01 00:00:00','Y',NULL,'/mnt/Bio3Data03/GenoPub/GenoPubData/'),
(4397,'sperm_predictedpiRNA_uniqueregions','','A4397',6303,'MEM',69,7,'','Magda','2011-11-01 00:00:00','Y',NULL,'/mnt/Bio3Data03/GenoPub/GenoPubData/'),
(4398,'64_predictedpiRNA_uniqueregions',NULL,'A4398',6303,'MEM',69,7,NULL,'Magda','2011-11-01 00:00:00','Y',NULL,'/mnt/Bio3Data03/GenoPub/GenoPubData/'),
(4399,'piRNA_predictedregions','','A4399',6303,'MEM',69,7,'combined all unique regions form all stages sequenced into unique piRNA locations from bed files, used MergeRegiosn to collapse into regions, these were used for DRSS for all stages ','Magda','2011-11-01 00:00:00','Y',NULL,'/mnt/Bio3Data03/GenoPub/GenoPubData/'),
(4400,'rRNA_ensemble_UCSC',NULL,'A4400',6303,'MEM',69,7,NULL,'Magda','2011-11-07 00:00:00','Y',NULL,'/mnt/Bio3Data03/GenoPub/GenoPubData/'),
(4401,'tRNA_UCSC',NULL,'A4401',6303,'MEM',69,7,NULL,'Magda','2011-11-07 00:00:00','Y',NULL,'/mnt/Bio3Data03/GenoPub/GenoPubData/'),
(4402,'pseudogenes_ensemble',NULL,'A4402',6303,'MEM',69,7,NULL,'Magda','2011-11-07 00:00:00','Y',NULL,'/mnt/Bio3Data03/GenoPub/GenoPubData/'),
(4403,'Sperm_exons_BaseFractionNonConverted',NULL,'A4403',6303,'MEM',69,7,NULL,'Magda','2011-11-08 00:00:00','Y',NULL,'/mnt/Bio3Data03/GenoPub/GenoPubData/'),
(4404,'64_exons_BaseFractionNonConverted',NULL,'A4404',6303,'MEM',69,7,NULL,'Magda','2011-11-08 00:00:00','Y',NULL,'/mnt/Bio3Data03/GenoPub/GenoPubData/'),
(4405,'256_exons_BaseFractionNonConverted',NULL,'A4405',6303,'MEM',69,7,NULL,'Magda','2011-11-08 00:00:00','Y',NULL,'/mnt/Bio3Data03/GenoPub/GenoPubData/'),
(4406,'Sphere_BaseFractionNonConverted',NULL,'A4406',6303,'MEM',69,7,NULL,'Magda','2011-11-08 00:00:00','Y',NULL,'/mnt/Bio3Data03/GenoPub/GenoPubData/'),
(4408,'8374X1_N2total_1Rx Unique',NULL,'A4408',1204,'MEM',NULL,25,NULL,'u0028003','2011-11-09 00:00:00','Y',NULL,'/mnt/Bio3Data03/GenoPub/GenoPubData/'),
(4409,'8374X1_N2total_1Rx  Duplicates',NULL,'A4409',1204,'MEM',NULL,25,NULL,'u0028003','2011-11-09 00:00:00','Y',NULL,'/mnt/Bio3Data03/GenoPub/GenoPubData/'),
(4410,'25bpSubWinData','','A4410',6303,'MEM',NULL,49,'','u0785353','2011-11-09 00:00:00','Y',NULL,'/mnt/Bio3Data03/GenoPub/GenoPubData/'),
(4412,'25bpSubWinData',NULL,'A4412',6303,'MEM',NULL,49,NULL,'u0785353','2011-11-09 00:00:00','Y',NULL,'/mnt/Bio3Data03/GenoPub/GenoPubData/'),
(4413,'FDR\r','','A4413',6303,'PUBLIC',NULL,49,'','u0785353','2011-11-09 00:00:00','Y',NULL,'/mnt/Bio3Data03/GenoPub/GenoPubData/'),
(4414,'Log2Ratio','','A4414',6303,'PUBLIC',NULL,49,'','u0785353','2011-11-09 00:00:00','Y',NULL,'/mnt/Bio3Data03/GenoPub/GenoPubData/'),
(4415,'8374X1_N2total_1Rx  4 Copies','','A4415',1204,'MEM',NULL,25,'','u0028003','2011-11-09 00:00:00','Y',NULL,NULL),
(4417,'sperm_methylatedC_NBP_x240_q0',NULL,'A4417',6303,'MEM',69,7,NULL,'Magda','2011-11-16 00:00:00','Y',NULL,'/mnt/Bio3Data03/GenoPub/GenoPubData/'),
(4418,'sperm_unmethylated_NBP_x240_q0',NULL,'A4418',6303,'MEM',69,7,NULL,'Magda','2011-11-16 00:00:00','Y',NULL,'/mnt/Bio3Data03/GenoPub/GenoPubData/'),
(4419,'sperm_readcoverage_NP_p13_q60',NULL,'A4419',6303,'MEM',69,7,NULL,'Magda','2011-11-16 00:00:00','Y',NULL,'/mnt/Bio3Data03/GenoPub/GenoPubData/'),
(4420,'sperm _readcovergae_NP_p0_q240',NULL,'A4420',6303,'MEM',69,7,NULL,'Magda','2011-11-16 00:00:00','Y',NULL,'/mnt/Bio3Data03/GenoPub/GenoPubData/'),
(4422,'EnrichedRegions_w500_spermvssphere','<P ALIGN=""LEFT"">Params:</P><P ALIGN=""LEFT"">30.0    FDR threshold</P><P ALIGN=""LEFT"">1.585   Log2Ratio threshold</P><P ALIGN=""LEFT"">500     Window size</P><P ALIGN=""LEFT"">20      Minimum # obs in window</P><P ALIGN=""LEFT"">0.8     Window merge multiplier</P><P ALIGN=""LEFT"">true    Print graphs</P><P ALIGN=""LEFT"">true    Merge windows</P>','A4422',6303,'MEM',69,7,'','Magda','2011-11-21 00:00:00','Y',NULL,'/mnt/Bio3Data03/GenoPub/GenoPubData/'),
(4423,'EnrichedRegions_w1500_spermvssphere','<P ALIGN=""LEFT"">Params:</P><P ALIGN=""LEFT"">30.0	FDR threshold</P><P ALIGN=""LEFT"">1.585	Log2Ratio threshold</P><P ALIGN=""LEFT"">1500	Window size</P><P ALIGN=""LEFT"">20	Minimum # obs in window</P><P ALIGN=""LEFT"">0.8	Window merge multiplier</P><P ALIGN=""LEFT"">true	Print graphs</P><P ALIGN=""LEFT"">true	Merge windows</P>','A4423',6303,'MEM',69,7,'','Magda','2011-11-21 00:00:00','Y',NULL,'/mnt/Bio3Data03/GenoPub/GenoPubData/'),
(4424,'Converted_CG_context_sperm','','A4424',6303,'MEM',69,7,'','Magda','2011-11-21 00:00:00','Y',NULL,'/mnt/Bio3Data03/GenoPub/GenoPubData/'),
(4425,'NonConverted_CG_sperm_NBP',NULL,'A4425',6303,'MEM',69,7,NULL,'Magda','2011-11-21 00:00:00','Y',NULL,'/mnt/Bio3Data03/GenoPub/GenoPubData/'),
(4426,'Converted_CG_context_sphere',NULL,'A4426',6303,'MEM',69,7,NULL,'Magda','2011-11-21 00:00:00','Y',NULL,'/mnt/Bio3Data03/GenoPub/GenoPubData/'),
(4427,'NonConverted_CG_sphere',NULL,'A4427',6303,'MEM',69,7,NULL,'Magda','2011-11-21 00:00:00','Y',NULL,'/mnt/Bio3Data03/GenoPub/GenoPubData/'),
(4428,'Log2Val_w1500',NULL,'A4428',6303,'MEM',69,7,NULL,'Magda','2011-11-21 00:00:00','Y',NULL,'/mnt/Bio3Data03/GenoPub/GenoPubData/');
ALTER TABLE dbo.gpAnnotation CHECK CONSTRAINT all;
SET IDENTITY_INSERT dbo.gpAnnotation OFF;


--
-- Table structure for table dbo.gpAnnotationCollaborator --

IF  EXISTS (SELECT * FROM sys.objects WHERE object_id = OBJECT_ID(N'dbo.gpAnnotationCollaborator') AND type in (N'U')) DROP TABLE dbo.gpAnnotationCollaborator;


CREATE TABLE dbo.gpAnnotationCollaborator (
  idAnnotation int  NOT NULL,
  idUser int  NOT NULL,
CONSTRAINT [PK_AnnotationCollaborator] PRIMARY KEY CLUSTERED 
(
	[idAnnotation] ASC,
	[idUser] ASC
),
  
  CONSTRAINT FK_AnnotationCollaborator_Annotation FOREIGN KEY (idAnnotation) REFERENCES dbo.gpAnnotation (idAnnotation) ON DELETE NO ACTION ON UPDATE NO ACTION,
  CONSTRAINT FK_AnnotationCollaborator_User FOREIGN KEY (idUser) REFERENCES dbo.gpUser (idUser) ON DELETE NO ACTION ON UPDATE NO ACTION
)  ;


--
-- Dumping data for table dbo.gpAnnotationCollaborator --


ALTER TABLE dbo.gpAnnotationCollaborator NOCHECK CONSTRAINT all;
INSERT INTO dbo.gpAnnotationCollaborator (idAnnotation, idUser) VALUES (3780,78),(3781,78),(3782,78),(3783,78),(3789,78),(3790,78),(3791,78),(3611,84);
ALTER TABLE dbo.gpAnnotationCollaborator CHECK CONSTRAINT all;


--
-- Table structure for table dbo.gpAnnotationGrouping --

IF  EXISTS (SELECT * FROM sys.objects WHERE object_id = OBJECT_ID(N'dbo.gpAnnotationGrouping') AND type in (N'U')) DROP TABLE dbo.gpAnnotationGrouping;


CREATE TABLE dbo.gpAnnotationGrouping (
  idAnnotationGrouping int  NOT NULL identity,
  name varchar(2000) NOT NULL,
  description varchar(max)  NULL,
  idParentAnnotationGrouping int   NULL,
  idGenomeVersion int   NULL,
  idUserGroup int   NULL,
  createdBy varchar(200)  NULL,
  createDate datetime  NULL,
CONSTRAINT [PK_AnnotationGrouping] PRIMARY KEY CLUSTERED 
(
	[idAnnotationGrouping] ASC
),
  
  
  
  CONSTRAINT FK_AnnotationGrouping_GenomeVersion FOREIGN KEY (idGenomeVersion) REFERENCES dbo.gpGenomeVersion (idGenomeVersion),
  CONSTRAINT FK_AnnotationGrouping_parentAnnotationGrouping FOREIGN KEY (idParentAnnotationGrouping) REFERENCES dbo.gpAnnotationGrouping (idAnnotationGrouping),
  CONSTRAINT FK_AnnotationGrouping_UserGroup FOREIGN KEY (idUserGroup) REFERENCES dbo.gpUserGroup (idUserGroup)
)  ;


--
-- Dumping data for table dbo.gpAnnotationGrouping --

ALTER TABLE dbo.gpAnnotationGrouping NOCHECK CONSTRAINT all;
SET IDENTITY_INSERT dbo.gpAnnotationGrouping ON;
INSERT INTO dbo.gpAnnotationGrouping 
(idAnnotationGrouping,
name,
description,
idParentAnnotationGrouping,
idGenomeVersion,
idUserGroup,
createdBy,
createDate
)VALUES (400,'A_thaliana_Jan_2004','A_thaliana_Jan_2004',NULL,400,NULL,NULL,NULL),
(1204,'C_elegans_May_2008','C_elegans_May_2008',NULL,1204,NULL,NULL,NULL),
(1901,'D_melanogaster_Apr_2006','D_melanogaster_Apr_2006',NULL,1901,NULL,NULL,NULL),
(2401,'H_sapiens_Nov_2002','H_sapiens_Nov_2002',NULL,2401,NULL,NULL,NULL),
(2403,'H_sapiens_Jul_2003','H_sapiens_Jul_2003',NULL,2403,NULL,NULL,NULL),
(2700,'D_rerio_Mar_2006','D_rerio_Mar_2006',NULL,2700,NULL,NULL,NULL),
(2701,'D_rerio_Jul_2007','D_rerio_Jul_2007',NULL,2701,NULL,NULL,NULL),
(3100,'G_gallus_May_2006','G_gallus_May_2006',NULL,3100,NULL,NULL,NULL),
(3404,'H_sapiens_May_2004','H_sapiens_May_2004',NULL,3404,NULL,NULL,NULL),
(3405,'H_sapiens_Mar_2006','H_sapiens_Mar_2006',NULL,3405,NULL,NULL,NULL),
(3804,'M_musculus_Mar_2006','M_musculus_Mar_2006',NULL,3804,NULL,NULL,NULL),
(3806,'M_musculus_Jul_2007','M_musculus_Jul_2007',NULL,3806,NULL,NULL,NULL),
(3900,'M_abscessus_Mar_2008','M_abscessus_Mar_2008',NULL,3900,NULL,NULL,NULL),
(4100,'M_tuberculosis_Sep_2008','M_tuberculosis_Sep_2008',NULL,4100,NULL,NULL,NULL),
(5202,'R_norvegicus_Nov_2004','R_norvegicus_Nov_2004',NULL,5202,NULL,NULL,NULL),
(5207,'S_cerevisiae_May_2008','S_cerevisiae_May_2008',NULL,5207,NULL,NULL,NULL),
(5400,'S_pombe_Sep_2004','S_pombe_Sep_2004',NULL,5400,NULL,NULL,NULL),
(5401,'S_pombe_Apr_2007','S_pombe_Apr_2007',NULL,5401,NULL,NULL,NULL),
(6201,'X_tropicalis_Aug_2005','X_tropicalis_Aug_2005',NULL,6201,NULL,NULL,NULL),
(6300,'Chip Seq Challenge 1.0','<P ALIGN=""LEFT""><B>ChIP-Seq Community Challenge 1.0 Report</B></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">The chIP-seq community challenge is an effort to benchmark chIP-seq detection algorithms using a common dataset containing simulated chIP-seq peaks.  The advantages to using simulated peaks are numerous. Known truth can be used to unambiguously evaluate different algorithms, simulations allow one to model specific types of chIP-seq datasets, and simulations enable rapid prototyping under a variety of conditions.  Simulations also come with a variety of pitfalls, namely, over simplification of actual data, idealized random distributions, and a potential to miss uncharacterized experimental bias.  By combining simulated chIP-seq reads with real experimentally derived input sequencing data we hoped to provide hybrid datasets that can be used in a larger testing framework that includes real chIP-seq datasets for evaluating novel chIP-seq detection algorithms. See below for a description of the simulated chIP-seq datasets.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">As a first pass, the chIP-seq community challenge, was a resounding success with 13 submissions representing 12 different peak detection packages from both academic and industry groups. See peakDetectorPackages.xls. We would like to thank each of the submitters and both Illumina and Applied Biosciences for sponsoring this contest.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">The contest focused on two key aspects of chIP-seq analysis: 1) discrimination between true positives and false positives, and 2) accurate confidence estimation in a given list of candidate binding peaks. The discrimination test is based on a simple summary statistic, the total number of recovered chIP-seq peaks from lists containing the top 500, 1000, and 1500 candidate regions.  See the topHitListResults.xls file.  For a more comprehensive analysis see the rocResults.xls file.  This contains plots of the true positive rate versus the false discovery rate (FDR) for a ranked intersection from the top 1500 region lists. Both analysis show MACS, USeq, Partek, SWEMBL, and BPC algorithms as the best discriminators with prizes going to Tao Liu (iPod Shuffle), Mali Salmon, Justin Brown, Steven Wilder, and Shouyong Peng. (Note, as organizers of the contest, the USeq submission is excluded from any prizes.)</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">The second key aspect of chIP-seq analysis address in this contest is the ability of a chIP-seq detection package to accurately estimate confidence in a given list of candidate binding peaks.  It is of only marginal help to an investigator to provide a ranked list of putative binding peaks without an estimation of where in the rankings the regions become primarily false.  Ideally, a peak detection package should be able to predict what the FDR is for a particular list.  Basically, what % of the regions are false.  FDRs have an unambiguous interpretation and are more useful than p-values for setting confidence thresholds in this type of list based genomic analysis.  The confidence estimation statistic used here is based on the absolute fold difference between the estimated FDR and the actual FDR for lists predicted to contain 1%, 5%, 10%, and 20% false discoveries.  USeq''s and Cisgenome''s empirical FDR estimations scored quite well with an average fold difference of </P>',3405,3405,28,NULL,NULL),
(6301,'CntrdReadCounts',NULL,6302,1204,25,NULL,NULL),(6302,'Bass Lab','',1204,1204,25,NULL,NULL),
(6305,'Dcr1_Vs_Wt',NULL,6302,1204,25,NULL,NULL),
(6306,'Depreciated_DcrOrWt',NULL,6302,1204,25,NULL,NULL),
(6307,'K39A_2_Vs_Wt',NULL,6302,1204,25,NULL,NULL),
(6308,'Novo',NULL,6302,1204,25,NULL,NULL),
(6315,'DcrSum',NULL,6306,1204,25,NULL,NULL),
(6316,'Dcr',NULL,6301,1204,25,NULL,NULL),
(6317,'K39A_2',NULL,6301,1204,25,NULL,NULL),
(6318,'Wt',NULL,6301,1204,25,NULL,NULL),
(6319,'WtSum',NULL,6306,1204,25,NULL,NULL),
(6320,'K39A_pp10_point',NULL,6308,1204,25,NULL,NULL),
(6321,'WT_pp10_point',NULL,6308,1204,25,NULL,NULL),
(6323,'Ruby_Bartel_Cell06ReadCounts',NULL,1204,1204,NULL,NULL,NULL),
(6324,'ARUP',NULL,3900,3900,15,NULL,NULL),
(6325,'1st Run',NULL,6324,3900,15,NULL,NULL),
(6326,'GATC',NULL,6325,3900,15,NULL,NULL),
(6327,'Lane 1',NULL,6326,3900,15,NULL,NULL),
(6328,'Lane 2',NULL,6326,3900,15,NULL,NULL)
,(6329,'Lane 3',NULL,6326,3900,15,NULL,NULL)
,(6330,'Lane 4','',6326,3900,15,NULL,NULL)
,(6331,'Lane 7','',6326,3900,15,NULL,NULL)
,(6332,'Lane 6',NULL,6326,3900,15,NULL,NULL)
,(6333,'Lane 8',NULL,6326,3900,15,NULL,NULL)
,(6335,'Hagedorn',NULL,3806,3806,16,NULL,NULL)
,(6336,'DSS',NULL,6335,3806,16,NULL,NULL)
,(6337,'Graphs',NULL,6336,3806,16,NULL,NULL)
,(6338,'Regions',NULL,6336,3806,16,NULL,NULL)
,(6339,'Splice Junctions',NULL,6336,3806,16,NULL,NULL)
,(6340,'Jones',NULL,3806,3806,14,NULL,NULL)
,(6341,'Promoter Cap Seq','',6340,3806,14,NULL,NULL)
,(6342,'Read Coverage',NULL,6341,3806,14,NULL,NULL)
,(6343,'Moon',NULL,3806,3806,24,NULL,NULL)
,(6345,'Orosz',NULL,3806,3806,23,NULL,NULL)
,(6346,'HSF1',NULL,6345,3806,23,NULL,NULL)
,(6348,'Wang',NULL,3806,3806,9,NULL,NULL)
,(6349,'6149_SHP_KOvsWT',NULL,6348,3806,9,NULL,NULL)
,(6350,'6202_SHPvsGFP',NULL,6348,3806,9,NULL,NULL)
,(6351,'S_glossinidius_Jan_2006','S_glossinidius_Jan_2006',NULL,5500,NULL,NULL,NULL)
,(6352,'Cairns Public','',5401,5401,7,NULL,NULL)
,(6353,'ChIPChip',NULL,6352,5401,7,NULL,NULL)
,(6354,'RNA',NULL,6352,5401,7,NULL,NULL)
,(6355,'Diff',NULL,6354,5401,7,NULL,NULL)
,(6356,'Static',NULL,6354,5401,7,NULL,NULL)
,(6357,'IntergenicNorm',NULL,6356,5401,7,NULL,NULL)
,(6358,'Total',NULL,6357,5401,7,NULL,NULL)
,(6359,'NonHybNorm',NULL,6356,5401,7,NULL,NULL)
,(6360,'Total',NULL,6359,5401,7,NULL,NULL)
,(6361,'Cairns Private',NULL,5401,5401,7,NULL,NULL)
,(6362,'RNA',NULL,6361,5401,7,NULL,NULL)
,(6363,'Diff',NULL,6362,5401,7,NULL,NULL)
,(6364,'Static',NULL,6362,5401,7,NULL,NULL)
,(6365,'IntergenicNorm','',6364,5401,7,NULL,NULL)
,(6366,'Total',NULL,6365,5401,7,NULL,NULL)
,(6367,'NonHybNorm',NULL,6364,5401,7,NULL,NULL)
,(6368,'Total',NULL,6367,5401,7,NULL,NULL)
,(6370,'Weis Private',NULL,3804,3804,19,NULL,NULL)
,(6371,'H_sapiens_Feb_2009',NULL,NULL,6300,NULL,NULL,NULL)
,(6373,'Schiffman and Lessnick Private',NULL,3404,3404,18,NULL,NULL)
,(6374,'MIP',NULL,6373,3404,18,NULL,NULL)
,(6375,'AlleleRatio',NULL,6374,3404,18,NULL,NULL)
,(6376,'CopyNumber',NULL,6374,3404,18,NULL,NULL)
,(6377,'T2Analysis',NULL,6374,3404,18,NULL,NULL)
,(6378,'Oligos',NULL,6377,3404,18,NULL,NULL)
,(6379,'Windows',NULL,6377,3404,18,NULL,NULL)
,(6380,'Schiffman Private',NULL,3404,3404,18,NULL,NULL)
,(6381,'Brain',NULL,6380,3404,18,NULL,NULL)
,(6382,'Leukemia',NULL,6380,3404,18,NULL,NULL)
,(6383,'PairedGDRLeukemia',NULL,6380,3404,18,NULL,NULL)
,(6384,'AlleleRatio',NULL,6381,3404,18,NULL,NULL)
,(6385,'CopyNumber',NULL,6381,3404,18,NULL,NULL)
,(6386,'Affy',NULL,6382,3404,18,NULL,NULL)
,(6387,'MIPs',NULL,6382,3404,18,NULL,NULL)
,(6388,'AlleleRatio',NULL,6387,3404,18,NULL,NULL)
,(6389,'CopyNumber',NULL,6387,3404,18,NULL,NULL)
,(6390,'Hagedorn Lab','',3405,3405,16,NULL,NULL)
,(6391,'6404R_NegVsPos',NULL,6390,3405,16,NULL,NULL)
,(6392,'6404R_PosVsNeg',NULL,6390,3405,16,NULL,NULL)
,(6393,'DGE',NULL,6390,3405,16,NULL,NULL)
,(6394,'HitCounts',NULL,6393,3405,16,NULL,NULL)
,(6395,'Old',NULL,6393,3405,16,NULL,NULL)
,(6396,'Regions',NULL,6393,3405,16,NULL,NULL)
,(6397,'WindowTracks',NULL,6393,3405,16,NULL,NULL)
,(6398,'HCV',NULL,6394,3405,16,NULL,NULL)
,(6399,'Normal',NULL,6394,3405,16,NULL,NULL)
,(6400,'HitTracks',NULL,6395,3405,16,NULL,NULL)
,(6401,'HCV',NULL,6400,3405,16,NULL,NULL)
,(6402,'Norm',NULL,6400,3405,16,NULL,NULL)
,(6403,'StrdScanSeqs',NULL,6395,3405,16,NULL,NULL)
,(6404,'HCV Norm',NULL,6403,3405,16,NULL,NULL)
,(6405,'HCV_Norm_Minus',NULL,6403,3405,16,NULL,NULL)
,(6406,'HCV_Norm_Plus',NULL,6403,3405,16,NULL,NULL)
,(6407,'Enriched',NULL,6396,3405,16,NULL,NULL)
,(6408,'Reduced',NULL,6396,3405,16,NULL,NULL)
,(6409,'ENCODE',NULL,6390,3405,16,NULL,NULL)
,(6410,'Diff',NULL,6409,3405,16,NULL,NULL)
,(6411,'Static',NULL,6409,3405,16,NULL,NULL)
,(6412,'RNASeq',NULL,6390,3405,16,NULL,NULL)
,(6413,'Degradation',NULL,6412,3405,16,NULL,NULL)
,(6414,'HCVHuH7.5',NULL,6412,3405,16,NULL,NULL)
,(6415,'ReadCoverage',NULL,6414,3405,16,NULL,NULL)
,(6416,'WindowTracks',NULL,6414,3405,16,NULL,NULL)
,(6417,'HitTracks',NULL,6673,3405,16,NULL,NULL)
,(6418,'ScanSeqs',NULL,6673,3405,16,NULL,NULL)
,(6419,'HCVLiver',NULL,6412,3405,16,NULL,NULL)
,(6420,'EnrichedRegions',NULL,6419,3405,16,NULL,NULL)
,(6421,'Old',NULL,6419,3405,16,NULL,NULL)
,(6422,'ReadCoverage',NULL,6419,3405,16,NULL,NULL)
,(6423,'ReducedRegions',NULL,6419,3405,16,NULL,NULL)
,(6424,'SpliceJunctions',NULL,6419,3405,16,NULL,NULL)
,(6425,'WindowTracks',NULL,6419,3405,16,NULL,NULL)
,(6426,'5Min',NULL,6418,3405,16,NULL,NULL)
,(6427,'10Min',NULL,6418,3405,16,NULL,NULL)
,(6428,'15Min',NULL,6418,3405,16,NULL,NULL)
,(6429,'HCVMock6hrs',NULL,6416,3405,16,NULL,NULL)
,(6430,'HCVMock24hrs',NULL,6416,3405,16,NULL,NULL)
,(6431,'CoverageTracks',NULL,6421,3405,16,NULL,NULL)
,(6432,'Cap',NULL,6431,3405,16,NULL,NULL)
,(6433,'PolyA',NULL,6431,3405,16,NULL,NULL)
,(6434,'HCV_vs_Normal',NULL,6421,3405,16,NULL,NULL)
,(6435,'Cap',NULL,6434,3405,16,NULL,NULL)
,(6436,'PolyA',NULL,6434,3405,16,NULL,NULL)
,(6437,'BinPVal','',6425,3405,16,NULL,NULL)
,(6438,'QValFDR',NULL,6425,3405,16,NULL,NULL)
,(6439,'HighResProfHis_Barski2007Cell',NULL,3405,3405,NULL,NULL,NULL)
,(6440,'BinPVal',NULL,6439,3405,NULL,NULL,NULL)
,(6441,'EmpFDR',NULL,6439,3405,NULL,NULL,NULL)
,(6442,'QValFDR',NULL,6439,3405,NULL,NULL,NULL)
,(6443,'Hunt Lab','',3405,3405,22,NULL,NULL)
,(6444,'Oligos',NULL,6443,3405,22,NULL,NULL)
,(6445,'Windows',NULL,6443,3405,22,NULL,NULL)
,(6446,'Jones Lab','',3405,3405,14,NULL,NULL)
,(6447,'APC_MeDIP',NULL,6446,3405,14,NULL,NULL)
,(6448,'NimPolypsAll',NULL,6446,3405,14,NULL,NULL)
,(6449,'Oligo','',6447,3405,14,NULL,NULL)
,(6450,'Windows',NULL,6447,3405,14,NULL,NULL)
,(6451,'Oligos',NULL,6448,3405,14,NULL,NULL)
,(6452,'Windows',NULL,6448,3405,14,NULL,NULL)
,(6456,'Lessnick Lab','',3405,3405,10,NULL,NULL)
,(6457,'6077_EWS',NULL,6456,3405,10,NULL,NULL)
,(6458,'6108_EWS',NULL,6456,3405,10,NULL,NULL)
,(6459,'6138_NKX2.2',NULL,6456,3405,10,NULL,NULL)
,(6460,'6143_6194_NRO',NULL,6456,3405,10,NULL,NULL)
,(6461,'6143_NRO',NULL,6456,3405,10,NULL,NULL)
,(6462,'6157_E3-E4_NKX2.2',NULL,6456,3405,10,NULL,NULL)
,(6463,'6157_NKX2.2',NULL,6456,3405,10,NULL,NULL)
,(6464,'6194_NRO',NULL,6456,3405,10,NULL,NULL)
,(6465,'6266_Flg',NULL,6456,3405,10,NULL,NULL)
,(6466,'Tantin Lab','',3405,3405,20,NULL,NULL)
,(6467,'Alone',NULL,6466,3405,20,NULL,NULL)
,(6468,'Diff',NULL,6466,3405,20,NULL,NULL)
,(6469,'Static',NULL,6466,3405,20,NULL,NULL)
,(6470,'H2O2VsNoTreat',NULL,6468,3405,20,NULL,NULL)
,(6471,'NoTreatVsH2O2',NULL,6468,3405,20,NULL,NULL)
,(6472,'H2O2',NULL,6469,3405,20,NULL,NULL)
,(6473,'IR',NULL,6469,3405,20,NULL,NULL)
,(6474,'NoTreat',NULL,6469,3405,20,NULL,NULL)
,(6475,'ERs_236_SubWin','This directory H_sapiens_Mar_2006\\TantinPvt\\Static\\NoTreat\\ERs_236_SubWin.graphs.seqs doesn\''t have any .bar files; however, a its subdirectory AllSubWindowStairStepBarFiles contains zipped up bar files.  I\''ve created an annotation for the subdirectory and uploaded the zip files.  \r-Tony Di Sera Sep 23, 2009.',6474,3405,20,NULL,NULL)
,(6476,'Utah Autism Research Project','',3405,3405,8,NULL,NULL)
,(6477,'CapSeq28Patients',NULL,6476,3405,8,NULL,NULL)
,(6478,'Alleler',NULL,6477,3405,8,NULL,NULL)
,(6479,'ArrayDesign',NULL,6477,3405,8,NULL,NULL)
,(6481,'ReadCoverage',NULL,6477,3405,8,NULL,NULL)
,(6482,'SNPs',NULL,6477,3405,8,NULL,NULL)
,(6484,'SNPs',NULL,6478,3405,8,NULL,NULL)
,(6489,'All',NULL,6481,3405,8,NULL,NULL)
,(6490,'UniDup',NULL,6481,3405,8,NULL,NULL)
,(6491,'All',NULL,6482,3405,8,NULL,NULL)
,(6492,'UniDup',NULL,6482,3405,8,NULL,NULL)
,(6493,'Read Coverage','<P ALIGN=""LEFT"">These contain a count of the number of overlapping reads at each base.</P>',6300,3405,28,NULL,NULL)
,(6505,'Ayer Lab','Don Ayer\''s group',3405,3405,17,NULL,NULL)
,(6506,'CompleteMedia',NULL,6505,3405,17,NULL,NULL)
,(6507,'Starve',NULL,6505,3405,17,NULL,NULL)
,(6508,'ERMData',NULL,6506,3405,17,NULL,NULL)
,(6509,'WindowData',NULL,6506,3405,17,NULL,NULL)
,(6510,'ERMData',NULL,6507,3405,17,NULL,NULL)
,(6511,'WindowData',NULL,6507,3405,17,NULL,NULL)
,(6512,'Bernard Lab','',3405,3405,26,NULL,NULL)
,(6513,'Breast Samples',NULL,6512,3405,26,NULL,NULL)
,(6514,'ReadCoverage',NULL,6513,3405,26,NULL,NULL)
,(6515,'cc08-01058',NULL,6514,3405,26,NULL,NULL)
,(6516,'cc08-06157',NULL,6514,3405,26,NULL,NULL)
,(6517,'PolIII2LnsMixInp','',6538,3405,7,NULL,NULL)
,(6526,'ENCODE','<P ALIGN=""LEFT"">The <A HREF=""http://www.genome.gov/10005107"" TARGET=""_blank""><B><U>Encyclopedia of DNA Elements (ENCODE) Consortium</U></B></A> is an international collaboration of research groups funded by the National Human Genome Research dbo.gpInstitute (NHGRI). The goal of the consortium is to build a comprehensive parts list of the functional elements of the human genome, including elements that act at the protein level (coding genes) and RNA level (non-coding genes), and regulatory elements that control the cells and circumstances in which a gene is active. The discovery and annotation of gene elements is accomplished primarily by sequencing RNA from a diverse range of sources, comparative genomics, integrative bioinformatic methods, and human curation. Regulatory elements are typically investigated through DNA hypersensitivity assays, assays of DNA methylation, and chromatin immunoprecipitation (ChIP) of proteins that interact with DNA, including modified histones and transcription factors, followed by sequencing (ChIP-Seq). </P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Enclosed datasets were downloaded from the UCSC web site and reformatted, October 2009</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">UCSC&apos;s and ENCODE&apos;s data release policy:</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Please observe the following guidelines when using ENCODE data:</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Data users may freely use ENCODE data, but may not, without prior consent, submit publications that use an unpublished ENCODE dataset until nine months following the release of the dataset (see time stamp for release date).</P><P ALIGN=""LEFT"">Data users should properly acknowledge the ENCODE Project and resource producer(s) as the source of the data in any publication.</P><P ALIGN=""LEFT"">See the full <B><U>E</U></B><A HREF=""http://www.genome.gov/Pages/Research/ENCODE/ENCODEDataReleasePolicyFinal2008.pdf"" TARGET=""_blank""><B><U>NCODE-modENCODE Data Release Policy (2008-Present)</U></B></A><A HREF=""http://www.genome.gov/Pages/Research/ENCODE/ENCODEDataReleasePolicyFinal2008.pdf"" TARGET=""_blank""> document</A> for further details, and the <A HREF=""http://www.genome.gov/27528022"" TARGET=""_blank""><B><U>ENCODE-modENCODE Data Release Policy</U></B></A><A HREF=""http://www.genome.gov/27528022"" TARGET=""_blank""> </A>description page at NHGRI for background.</P>',3405,3405,28,NULL,NULL)
,(6527,'GABP_Valouev2008NatGen','',3405,3405,NULL,NULL,NULL)
,(6528,'Graves Lab','',3405,3405,13,NULL,NULL)
,(6529,'CBP1Ln',NULL,6528,3405,13,NULL,NULL)
,(6530,'CBP2Ln',NULL,6528,3405,13,NULL,NULL)
,(6531,'Ets1',NULL,6528,3405,13,NULL,NULL)
,(6532,'ETV1_1Ln',NULL,6528,3405,13,NULL,NULL)
,(6533,'p300_1Ln',NULL,6528,3405,13,NULL,NULL)
,(6538,'Cairns Lab','Brad Cairns Laboratory at the University of Utah ',3405,3405,7,'dnix','2009-10-12 00:00:00')
,(6539,'SpermEpiProject','Project data related to sperm epigenetics',6538,3405,7,'dnix','2009-10-12 00:00:00')
,(6540,'ChIPPromoterArray','Agilent promoter array chIP-chip data',6539,3405,7,'dnix','2009-10-12 00:00:00')
,(6541,'Oligo','Oligos: a simple log2(mean T/ mean C) ratio for each oligo in the dataset, the value is assigned to the center of the oligo. Oligo data is typically broken down by chromosome and stored as binary xxx.bar files for viewing in IGB.',6540,3405,7,'dnix','2009-10-12 00:00:00')
,(6542,'TopChIPPeaks','Intervals: Intervals are collapsed overlapping windows and best used for downstream analysis. Several different sets of intervals are created by lowering the pse score threshold to produce the top 100, 200, 400 ... intervals. For each interval set, the following are generated.\r\rBed: a text file representing each interval in the xxx.bed format. Simple start stop. Ranked most significant to least.',6540,3405,7,'dnix','2009-10-12 00:00:00')
,(6543,'WindowPseHM','Windows: several different measurements related to the overlapping window scanning summary statistics. These are best used for visualization in a browser. For each summary score, two types of window representations are given. xxxHM is a window representation best viewed in IGB as a stair step or black and white heatmap. The xxx folder are point window summaries where the window score is assigned to the center position in the window.\r\rPse: all layer pseudomedian relative difference, this is a robust signal estimation and is recommended for ranking and viewing your results. A relative difference is ratio between the difference in intensities and the average intensity (t-c) / ((t+c)/2). Relative differences have proven to be a better transformation than log2.',6540,3405,7,'dnix','2009-10-12 00:00:00')
,(6548,'MeDIPFibro',NULL,6542,3405,7,'dnix','2009-10-12 00:00:00')
,(6549,'MeDIPSperm',NULL,6542,3405,7,'dnix','2009-10-12 00:00:00')
,(6550,'MeDIPSprmVsFibro',NULL,6542,3405,7,'dnix','2009-10-12 00:00:00')
,(6551,'Protamine',NULL,6542,3405,7,'dnix','2009-10-12 00:00:00')
,(6553,'TH2B',NULL,6542,3405,7,'dnix','2009-10-12 00:00:00')
,(6564,'H3K4Me3',NULL,6542,3405,7,'dnix','2009-10-12 00:00:00')
,(6566,'H3K4Me2',NULL,6542,3405,7,'dnix','2009-10-12 00:00:00')
,(6567,'H3K27Me3',NULL,6542,3405,7,'dnix','2009-10-12 00:00:00')
,(6568,'Histone',NULL,6542,3405,7,'dnix','2009-10-12 00:00:00')
,(6573,'Yale TFBS','<P ALIGN=""LEFT"">All information and data were obtained from UCSC <A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeYaleChIPseq"" TARGET=""_blank""><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&amp;g=wgEncodeYaleChIPseq</U></A></P><P ALIGN=""CENTER""></P><P ALIGN=""CENTER""></P><P ALIGN=""CENTER""><B><U>ENCODE Transcription Factor Binding Sites by ChIP-seq from Yale/UC-Davis/Harvard</U></B></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B>Description</B></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">This track shows probable binding sites of the specified transcription factors (TFs) in the given cell types as determined by chromatin immunoprecipitation followed by high throughput sequencing (ChIP-Seq). Included for each cell type is the input signal, which represents the control condition where no antibody targeting was performed. For each experiment (cell type vs. antibody) this track shows a graph of enrichment for TF binding (Signal), along with sites that have the greatest evidence of transcription factor binding (Peaks).</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B>Methods</B></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Cells were grown according to the approved ENCODE cell culture protocols. Further preparations were similar to those previously published (Euskirchen et al., 2007) with the exceptions that the cells were unstimulated and sodium orthovanadate was omitted from the buffers. For details on the chromatin immunoprecipitation protocol used, see Euskirchen et al. (2007) and Rozowsky et al. (2009).</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">DNA recovered from the precipitated chromatin was sequenced on the Illumina (Solexa) sequencing platform and mapped to the genome using the Eland alignment program. ChIP-seq data was scored based on sequence reads (length ~30 bps) that align uniquely to the human genome. From the mapped tags a signal map of ChIP DNA fragments (average fragment length ~ 200 bp) was constructed where the signal height is the number of overlapping fragments at each nucleotide position in the genome.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">For each 1 Mb segment of each chromosome a peak height threshold was determined by requiring a false discovery rate &lt;= 0.05 when comparing the number of peaks above threshold as compared the number obtained from multiple simulations of a random null background with the same number of mapped reads (also accounting for the fraction of mapable bases for sequence tags in that 1 Mb segment). The number of mapped tags in a putative binding region is compared to the normalized (normalized by correlating tag counts in genomic 10 kb windows) number of mapped tags in the same region from an input DNA control. Using a binomial test, only regions that have a p-value &lt;= 0.05 are considered to be significantly enriched compared to the input DNA control.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B>Release Notes</B></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">This is Release 2 (Oct 2009) of this track, which includes data from additional experiments, and the changes described below.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">A number of previously released datasets have been replaced by updated versions. The affected database tables and files include &apos;V2&apos; (or alternatively, &apos;Rel2&apos;) in the name, and metadata is marked with &quot;submittedDataVersion=V2&quot;, followed by the reason for replacement. Specific changes are:</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><I>Peaks</I>	 The previously released &quot;Peaks&quot; datasets had an off-by-one start location error and did not include pValues and point peaks. All Peaks data now have these problems corrected.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><I>Signal</I>	 The majority of the previously released &quot;Signal&quot; datasets contained a sparse representation of the signal. These have been corrected to be the continuous signal intended.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">This track contains experiments and data from several labs. During review, it was noticed that the some cell cultures were different enough between the labs that they should be analyzed separately???using Input specific for the lab. Specifically:</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">K562 vs. K562b	 The Farnham lab (UC Davis) K562 cell line variant differs from the Snyder (Yale) and Struhl (Harvard) labs and is now labeled: K562b. Experiments on K562b should be contrasted with Input (UCDavis). The K562 Input previously released has been replaced by two versions marked K562 and K562b. The K562 version used by the Snyder and Struhl groups directly replaces the previous version, and is versioned at V2. The K562b used by the Farnham group is now provided as K562b.</P><P ALIGN=""LEFT"">HepG2 vs. HepG2b	 The Farnham lab HepG2 cell line is now labeled HepG2b. Experiments on HepG2b should be contrasted with Input (UCDavis).</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B>Credits</B></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">These data were generated and analyzed by the labs of Michael Snyder, Mark Gerstein and Sherman Weissman at Yale University; Peggy Farnham at UC Davis; and Kevin Struhl at Harvard.    Contact: datasubmission@gersteinlab.org.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B>References</B></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Euskirchen G, Royce TE, Bertone P, Martone R, Rinn JL, Nelson FK, Sayward F, Luscombe NM, Miller P, Gerstein, M et al. CREB binds to multiple loci on human chromosome 22. Mol Cell Biol. 2004 May;24(9):3804-14.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Euskirchen GM, Rozowsky JS, Wei C, Lee WH, Zhang ZD, Hartman S, Emanuelsson O, Stolc V, Weissman S, Gerstein MB et al. Mapping of transcription factor binding regions in mammalian cells by ChIP: comparison of array- and sequencing-based technologies. Genome Res. 2007;17:898-909.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Martone R, Euskirchen G, Bertone P, Hartman S, Royce TE, Luscombe NM, Rinn JL, Nelson FK, Miller P, Gerstein M et al. Distribution of NF-kappaB-binding sites across human chromosome 22. Proc Natl Acad Sci U S A. 2003 Oct;100(21):12247-52.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Robertson G, Hirst M, Bainbridge M, Bilenky M, Zhao Y, Zeng T, Euskirchen G, Bernier B, Varhol R, Delaney A et al. Genome-wide profiles of STAT1 DNA association using chromatin immunoprecipitation and massively parallel sequencing. Nat. Methods 2007;4:651-7.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Rozowsky J, Euskirchen G, Auerbach RK, Zhang ZD, Gibson T, Bjornson R, Carriero N, Snyder M, Gerstein MB. PeakSeq enables systematic scoring of ChIP-seq experiments relative to controls. Nature Biotech. 2009;27:66-75.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B>Data Release Policy</B></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Data users may freely use ENCODE data, but may not, without prior consent, submit publications that use an unpublished ENCODE dataset until nine months following the release of the dataset. This date is listed in the Restricted Until column on the track configuration page and the download page. The full data release policy for ENCODE is available here.</P><P ALIGN=""LEFT""></P>',6584,3405,28,'dnix','2009-10-19 00:00:00')
,(6574,'Broad Histone','<P ALIGN=""LEFT"">See -&gt; <A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeBroadChipSeq"" TARGET=""_blank""><B><U>ENCODE Histone Modifications by Broad dbo.gpInstitute ChIP-seq</U></B></A></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B>Data version:</B> ENCODE Feb 2009 Freeze</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B>Description:</B></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">These track displays maps of chromatin state generated by the Broad/MGH ENCODE group using ChIP-seq. Chemical modifications (methylation, acylation) to the histone proteins present in chromatin influence gene expression by changing how accessible the chromatin is to transcription.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">The ChIP-seq method involves cross-linking histones and other DNA associated proteins to genomic DNA within cells using formaldehyde. The cross-linked chromatin is subsequently extracted, mechanically sheared, and immunoprecipitated using specific antibodies. After reversal of cross-links, the immunoprecipitated DNA is sequenced and mapped to the human reference genome. The relative enrichment of each antibody-target (epitope) across the genome is inferred from the density of mapped fragments.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B>Credits:</B></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">The ChIP-seq data were generated at the Broad dbo.gpInstitute and in the Bradley E. Bernstein lab at the Massachusetts General Hospital/Harvard Medical School.    Contact: nshoresh@broad.mit.edu.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Data generation and analysis was supported by funds from the NHGRI, the Burroughs Wellcome Fund, Massachusetts General Hospital and the Broad Institute.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B>References:</B></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Bernstein BE, Kamal M, Lindblad-Toh K, Bekiranov S, Bailey DK, Huebert DJ, McMahon S, Karlsson EK, Kulbokas EJ, Gingeras TR et al. Genomic maps and comparative analysis of histone modifications in human and mouse. Cell. 2005 Jan 28;120(2):169-81.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Bernstein BE, Mikkelsen TS, Xie X, Kamal M, Huebert DJ, Cuff J, Fry B, Meissner A, Wernig M, Plath K et al. A bivalent chromatin structure marks key developmental genes in embryonic stem cells. Cell. 2006 Apr 21;125(2):315-26.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Mikkelsen TS, Ku M, Jaffe DB, Issac B, Lieberman E, Giannoukos G, Alvarez P, Brockman W, Kim T, Koche RP et al. Genome-wide maps of chromatin state in pluripotent and lineage-committed cells. Nature. 2007 Aug 2;448:553-60.</P><P ALIGN=""LEFT""></P>',6584,3405,28,'dnix','2009-10-19 00:00:00')
,(6575,'SUNY RBP','<P ALIGN=""LEFT""><A HREF=""http://hgdownload.cse.ucsc.edu/goldenPath/hg18/encodeDCC/wgEncodeSunyalbanyRnaGeneChip/"" TARGET=""_blank""><B><U>ENCODE SUNY Albany RNA Binding Proteins by RIP-chip</U></B></A></P>',6584,3405,28,'dnix','2009-10-19 00:00:00')
,(6578,'RIKEN CAGE Loc','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeRikenCage"" TARGET=""_blank""><B><U>ENCODE RIKEN RNA Subcellular Localization by CAGE Tags</U></B></A></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B>Description</B></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">This track shows 5&apos; cap analysis gene expression (CAGE) tags and clusters in RNA extracts from different sub-cellular localizations in multiple cell lines. A CAGE cluster is a region of overlapping tags with an assigned value that represents the expression level. The data in this track were produced as part of the ENCODE Transcriptome Project.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B>These tracks contains the following views:</B></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><I>Plus and Minus Clusters</I> - These views display clusters of overlapping read mappings on the forward and reverse genomic strands.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B>Methods</B></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Cells were grown according to the approved ENCODE cell culture protocols. RNA molecules longer than 200 nt and present in the RNA population isolated from each subcellular compartment were fractionated into polyA+ and polyA- fractions as described in these protocols. The CAGE tags were sequenced from the 5&apos; ends of cap-trapped cDNAs produced using RIKEN CAGE technology (Kodzius et al. 2006; Valen et al. 2009). To create the tag, a linker was attached to the 5&apos; end of polyA+ or polyA- reverse-transcribed cDNAs which were selected by cap trapping (Carninci et al. 1996). The first 27 bp of the cDNA were cleaved using class II restriction enzymes. A linker was then attached to the 3&apos; end of the cDNA.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">After PCR amplification, the tags were sequenced (36 bp single reads) using ABI SOLiD technology (polyA- RNA from the cytosol and nucleus of K562 cell lines, and from whole cell in prostate cells) or Illumina/Solexa GA (all other data). Tags were mapped to the human genome (NCBI Build36, hg18) using the program nexalign (T. Lassmann manuscript in preparation). SOlid CAGE sequences were mapped with up to 3 mismatches; 2 mismatches were allowed for Solexa CAGE. Alignments of sequences mapping 10 times or fewer were retained. The expression level was computed as the number of reads making up the cluster, divided by the total number of reads sequenced, times 1 million.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B>Credits</B></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">These data were generated and analyzed by Timo Lassmann, Phil Kapranov, Hazuki Takahashi, Yoshihide Hayashizaki, Carrie Davis, Tom Gingeras, and Piero Carninci.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B>Contact:</B> Piero Carninci at RIKEN Omics Science Center</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B>References</B></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Kodzius R, Kojima M, Nishiyori H, Nakamura M, Fukuda S, Tagami M, Sasaki D, Imamura K, Kai C, Harbers M, et al. CAGE: cap analysis of gene expression. Nat Methods. 2006 March 1; 3(3):211-222.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Valen E, Pascarella G, Chalk A, Maeda N, Kojima M, Kawazu C, Murata M, Nishiyori H, Lazarevic D, Motti D, et al. Genome-wide detection and analysis of hippocampus core promoters using DeepCAGE. Genome Res. 2009 February; 19(2):255-265.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Carninci P, Kvam C, Kitamura A, Ohsumi T, Okazaki Y, Itoh M, Kamiya M, Shibata K, Sasaki N, Izawa M, et al. High-efficiency full-length cDNA cloning by biotinylated CAP trapper. Genomics. 1996 November 1; 37(3):327-336.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B>Data Release Policy</B></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Data users may freely use ENCODE data, but may not, without prior consent, submit publications that use an unpublished ENCODE dataset until nine months following the release of the dataset. This date is listed in the Restricted Until column on the track configuration page and the download page. The full data release policy for ENCODE is available <A HREF=""http://genome.cse.ucsc.edu/ENCODE/terms.html"" TARGET=""_blank""><U>here.</U></A></P>',6584,3405,28,'dnix','2009-10-19 00:00:00')
,(6579,'Open Chromatin','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeChromatinMap"" TARGET=""_blank""><B><U>ENCODE Open Chromatin Map by the Crawford/Duke, Leib/UNC and Iyer/UT-Austin labs</U></B></A></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B>Description</B></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">These tracks display evidence of open chromatin in multiple cell types from the Duke/UNC/UT-Austin/EBI ENCODE group. Open chromatin was identified using two independent and complementary methods: DNaseI hypersensitivity (HS) and Formaldehyde-Assisted Isolation of Regulatory Elements (FAIRE), combined with chromatin immunoprecipitation (ChIP) for select regulatory factors. Each method was verified by two detection platforms: Illumina (formerly Solexa) sequencing by synthesis, and high-resolution 1% ENCODE tiled microarrays supplied by NimbleGen.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B>DNaseI HS data:</B> DNaseI is an enzyme that has long been used to map general chromatin accessibility, and DNaseI &quot;hyperaccessibility&quot; or &quot;hypersensitivity&quot; is a feature of active cis-regulatory sequences. The use of this method has led to the discovery of functional regulatory elements that include enhancers, silencers, insulators, promotors, locus control regions and novel elements. DNaseI hypersensitivity signifies chromatin accessibility following binding of trans-acting factors in place of a canonical nucleosome.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B>FAIRE data:</B> FAIRE (Formaldehyde Assisted Isolation of Regulatory Elements) is a method to isolate and identify nucleosome-depleted regions of the genome. FAIRE was initially discovered in yeast and subsequently shown to identify active regulatory elements in human cells (Giresi et al., 2007). Although less well-characterized than DNase, FAIRE also appears to identify functional regulatory elements that include enhancers, silencers, insulators, promotors, locus control regions and novel elements. DNA fragments isolated by FAIRE are 100-200 bp in length, with the average length being 140 bp.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B>ChIP data:</B> ChIP (Chromatin Immunoprecipitation) is a method to identify the specific location of proteins that are directly or indirectly bound to genomic DNA. By identifying the binding location of sequence-specific transcription factors, general transcription machinery components, and chromatin factors, ChIP can help in the functional annotation of the open chromatin regions identified by DNaseI HS mapping and FAIRE.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B>Display Conventions and Configuration</B></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B>Peaks</B></P><P ALIGN=""LEFT"">Regions of enriched signal in either DNaseI HS, FAIRE, or ChIP experiments. Peaks were called based on signals created using F-Seq, a software program developed at Duke (Boyle et al., 2008b). Significant regions were determined by performing ROC analysis of sequence data using data from the 1% ENCODE arrays, and determining a cut-off value at approximately the 95% sensitivity level. The solid vertical line in the peak represents the point with highest signal. ENCODE Peaks tables contain a p-value for statistical significance. For these data, this was determined by fitting the data to a gamma distribution.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B>Signal (F-Seq Density)</B></P><P ALIGN=""LEFT"">Density graph (wiggle) of signal enrichment calculated using F-Seq for the combined set of sequences from all replicates. F-Seq employs Parzen kernel density estimation to create base pair scores (Boyle et al., 2008b). This method does not look at fixed-length windows but rather weights contributions of nearby sequences in proportion to their distance from that base. It only considers sequences aligned 4 or less times in the genome, and uses an alignability background model to try to correct for regions where sequences cannot be aligned. For the K562, HepG2 and HelaS3 cell types, where there is an abnormal karyotype, a model to try to correct for amplifications and deletions was also used. No control data were used in the creation of these annotations.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B>Signal (Base Overlap)</B></P><P ALIGN=""LEFT"">An alternative version of the Signal (F-Seq Density) track annotation that provides a higher resolution view of the raw sequence data. This track also includes the combined set of sequences from all replicates. For each sequence, the aligned read is extended in the following way: for DNase, the read is extended 5 bp in both directions from its 5&apos; aligned end where DNase cut the DNA; for FAIRE and ChIP, the sequence is extend to a fragment length of 134 bp from the 5&apos; aligned end representing the approximate average fragment length. The score at each base pair represents the number of extended fragments that overlap the base pair.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B>Uniqueness</B></P><P ALIGN=""LEFT"">The Duke uniqueness tracks were used in identify regions of unique sequence for different tag lengths. The tracks also identify regions where high-throughput sequence tags cannot be mapped.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B>Excluded Regions</B></P><P ALIGN=""LEFT"">The Duke excluded regions track was used to identify problematic regions for short sequence tag signal detection (such as satellites and rRNA genes). These regions of the genome were excluded from the Open Chromatin tracks.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B>Methods</B></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Cells were grown according to the approved ENCODE cell culture protocols.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">DNaseI hypersensitive sites were isolated using methods called DNase-seq or DNase-chip (Boyle et al., 2008a, Crawford et al., 2006). Briefly, cells were lysed with NP40, and intact nuclei were digested with optimal levels of DNaseI enzyme. DNaseI digested ends were captured from three different DNase concentrations, and material was sequenced using Illumina (Solexa) sequencing. DNase-seq data were verified using material that was hybridized to NimbleGen Human ENCODE tiling arrays (1% of the genome). Multiple independent growths (replicates) were compared to verify the reproducibility of the data. A more detailed protocol is available here.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">FAIRE was performed (Giresi et al., 2007) by cross-linking proteins to DNA using 1% formaldehyde solution, and the complex was sheared using sonication. Phenol/chloroform extractions were performed to remove DNA fragments cross-linked to protein. The DNA recovered in the aqueous phase was hybridized to NimbleGen Human ENCODE tiling arrays (1% of the genome) and sequenced using a Solexa sequencing system. The ENCODE array data were used to verify the accuracy of the sequencing data, and multiple independent growths (replicates) were compared to assess the reproducibility of the data. A more detailed protocol is available here. Also see Giresi et al., 2009.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">To perform ChIP, proteins were cross-linked to DNA in vivo using 1% formaldehyde solution (Bhinge et al., 2007, ENCODE Project Consortium., 2007). Cross-linked chromatin was sheared by sonication and immunoprecipitated using a specific antibody against the protein of interest. After reversal of the cross-links, the immunoprecipitated DNA was used to identify the genomic location of transcription factor binding. This was accomplished by Solexa sequencing of the ends of the immunoprecipitated DNA (ChIP-seq), as well as labeling and hybridization of the immunoprecipitated DNA to NimbleGen Human ENCODE tiling arrays (1% of the genome) along with the input DNA as reference (ChIP-chip). The ENCODE array data were used to verify the accuracy of the sequencing data, and multiple independent growths (replicates) were compared to assess the reproducibility of the data. A more detailed protocol is available here.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">ENCODE Array data were normalized using the Tukey biweight normalization, and peaks were called using ChIPOTle (Buck, et al., 2005) at multiple levels of significance. Regions matched on size to these peaks that were devoid of any significant signal were also created to allow for ROC analysis.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Sequences from each experiment were aligned to the genome using Maq (Li et al., 2008) and those that aligned to 4 or fewer locations were retained. Other sequences were also filtered based on their alignment to problematic regions (such as satellites and rRNA genes). The resulting digital signal was converted to a continuous wiggle track using F-Seq that employs Parzen kernel density estimation to create base pair scores (Boyle et al., 2008b). Discrete DNase HS, FAIRE, and ChIP sites (peaks) were identified from DNase/FAIRE/ChIP-seq using F-Seq by setting a Parzen cutoff based on ROC curve analysis using peaks and non-peaks identified from DNase/FAIRE/ChIP-chip using NimbleGen Human ENCODE tiling arrays (1% of the genome).</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Input data was generated for GM12878, K562, HeLa-S3, HepG2, and HUVEC. These were used directly to create a control/background model used for F-Seq when generating signal annotations and subsequenntly peaks for these cell lines. These models are meant to correct for sequencing biases, alignment artifacts, and copy number changes in these cell lines. Input data is not being generated directly for other cell lines. Instead, a general background model was derived from the five Input data sets. This should provide corrections for sequencing biases and alignment artifacts, but obviously not for cell type specific copy number changes.</P>',6584,3405,28,'dnix','2009-10-19 00:00:00')
,(6580,'Affy RNA Loc','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeAffyRnaChip"" TARGET=""_blank""><B><U>ENCODE Affymetrix/CSHL Subcellular RNA Localization by Tiling Array</U></B></A></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B>Description</B></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">This track is produced as part of the ENCODE Transcriptome Project. Transcription of different RNA extracts from different sub-cellular localizations in different cell lines is compared in companion experiments using three different technologies: tiling arrays, RNA-seq using Solexa, and RNA-seq using SOLiD. The tiling array data are shown in this track. The Raw Signal view shows an estimate of abundance of RNA molecules and the Transfrags view shows the locations of sites corresponding to these molecules.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B>Display Conventions and Configuration</B></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Transfrags</P><P ALIGN=""LEFT"">The Transfrags view includes all transfrags before filtering.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Raw Signal</P><P ALIGN=""LEFT"">The Raw Signal view shows the probe intensity based on all probes.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B>Methods</B></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Cells were grown according to the approved ENCODE cell culture protocols. RNA molecules longer than 200 nt and present in RNA population isolated from different subcellular compartments (such as cytosol, nucleus, polysomes and others) were fractionated into polyA+ and polyA- fractions as described in these protocols. Each RNA fraction was converted into double-stranded cDNA using random hexamers, labeled and hybridized to a tiling 91-array set containing probes against the non-repetitive portion of the human genome tiled on average every 5 bp (center-to-center of each consecutive 25-mers).</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">All arrays were scaled to a median array intensity of 330. Within a sliding 61 bp window centered on each probe, an estimate of RNA abundance (signal; see the Raw Signal view) was found by calculating the median of all pairwise average PM-MM values, where PM is a perfect match and MM is a mismatch. Kapranov et al. (2002), Cheng et al. (2005) , Kapranov et al. (2007), and Cawley et al. (2004) are good references for the experimental methods. Cawley et al. also describes the analytical methods.</P><P ALIGN=""LEFT""><B></B></P><P ALIGN=""LEFT""><B>Verification</B></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">The reproducibility of the labeling method was assessed separately. Three independent technical replicates were generated from the same RNA pool for each RNA preparation and hybridized to duplicate arrays (two technical replicates) that contain the ENCODE regions. Labeled RNA samples were then pooled and hybridized to the tiling 91-array set spanning the whole genome. Transcribed regions (transfrags; see the Transfrags view) were generated from the Raw Signal by merging genomic positions to which probes are mapped. This merging was based on a 5% false positive rate cutoff in negative bacterial controls, a maximum gap (MaxGap) of 40 base-pairs and minimum run (MinRun) of 40 base-pairs.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B>Credits</B></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">These data were generated and analyzed by the transcriptome group at Affymetrix and Cold Spring Harbor Laboratories: P. Kapranov, I. Bell, E. Dumais, J. Drenkow, J. Dumais, N. Garg, M. Lubinsky, Carrie A. Davis, Huaien Wang, Kimberly Bell, Jorg Drenkow, Chris Zaleski, and Thomas R. Gingeras. </P><P ALIGN=""LEFT"">Contact: gingeras@cshl. edu (Tom Gingeras)</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B>References</B></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Cawley S, Bekiranov S, Ng HH, Kapranov P, Sekinger EA, Kampa D, Piccolboni A, Sementchenko V, Cheng J, Williams AJ et al. Unbiased mapping of transcription factor binding sites along human chromosomes 21 and 22 points to widespread regulation of noncoding RNAs. Cell. 2004 Feb 20;116(4):499-509.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Cheng J, Kapranov P, Drenkow J, Dike S, Brubaker S, Patel S, Long J, Stern D, Tammana H, Helt G et al. Transcriptional Maps of 10 Human Chromosomes at 5 Nucleotide Resolution. Science. 2005 May 20;308(5725):1149-1154.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Kapranov P, Cawley SE, Drenkow J, Bekiranov S, Strausberg RL, Fodor SP, Gingeras TR. Large-scale transcriptional activity in chromosomes 21 and 22. Science. 2002 May 3;296(5569):916-9.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Kapranov P, Cheng J, Dike S, Nix DA, Duttagupta R, Willingham AT, Stadler PF, Hertel J, Hackerm???ller J, Hofacker IL, et al. RNA maps reveal new RNA classes and a possible function for pervasive transcription. Science. 2007 Jun 8;316(5830):1484-8.</P><P ALIGN=""LEFT""></P>',6584,3405,28,'dnix','2009-10-19 00:00:00')
,(6583,'UW DNaseI HS','<P ALIGN=""LEFT""><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeUwDnaseSeq"" TARGET=""_blank""><B><U>ENCODE Univ. Washington DNaseI Hypersensitivity by Digital DNaseI</U></B></A><B><U></U></B></P><P ALIGN=""LEFT""><B><U></U></B></P><P ALIGN=""LEFT""><B>Description</B></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">This track is produced as part of the ENCODE Project. This track shows DNaseI sensitivity measured genome-wide in different cell lines using the Digital DNaseI methodology (see below), and DNaseI hypersensitive sites. DNaseI has long been used to map general chromatin accessibility and DNaseI hypersensitivity is a universal feature of active cis-regulatory sequences. The use of this method has led to the discovery of functional regulatory elements that include enhancers, insulators, promotors, locus control regions and novel elements. For each experiment (cell type) this track shows DNaseI hypersensitive zones (HotSpots) and hypersensitive sites (Peaks) based on the sequencing tag density (Signal).</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B>Display Conventions and Configuration</B></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">For each cell type, this track contains the following views:</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><I>HotSpots</I> - DNaseI hypersensitive zones identified using the HotSpot algorithm.</P><P ALIGN=""LEFT""><I>Peaks</I> - DNaseI hypersensitive sites (DHSs) identified as signal peaks within FDR 0.5% hypersensitive zones.</P><P ALIGN=""LEFT""><I>Raw Signal</I> - Density graph of signal enrichment based on aligned read density.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">DNaseI sensitivity is shown as the absolute density of in vivo cleavage sites across the genome mapped using the Digital DNaseI methodology (see below). Data have been normalized to 25 million reads per cell type.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B>Methods</B></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Cells were grown according to the approved ENCODE cell culture protocols. Digital DNaseI was performed by performing DNaseI digestion of intact nuclei, isolating DNaseI &apos;double-hit&apos; fragments as described in Sabo et al. (2006), and direct sequencing of fragment ends (which correspond to in vivo DNaseI cleavage sites) using the Solexa platform (27 bp reads). Uniquely mapping high-quality reads were mapped to the genome. DNaseI sensitivity is directly reflected in raw tag density (Signal), which is shown in the track as density of tags mapping within a 150 bp sliding window (at a 20 bp step across the genome). DNaseI hypersensitive zones (HotSpots) were identified using the HotSpot algorithm described in Sabo et al. (2004). 0.5% false discovery rate thresholds (FDR 0.005) were computed for each cell type by applying the HotSpot algorithm to an equivalent number of random uniquely mapping 27mers. DNaseI hypersensitive sites (DHSs or Peaks) were identified as signal peaks within FDR 0.5% hypersensitive zones using a peak-finding algorithm.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B>Verification</B></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Data were verified by sequencing biological replicates displaying correlation coefficient &gt;0.9. Results are extensively validated by conventional DNaseI hypersensitivity assays using end-labeling/Southern blotting methods.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B>Credits</B></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">These data were generated by the UW ENCODE group.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Contact: Richard Sandstrom</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B>References</B></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Sabo PJ, Kuehn MS, Thurman R, Johnson BE, Johnson EM, Cao H, Yu M, Rosenzweig E, Goldy J, Haydock A et al. Genome-scale mapping of DNase I sensitivity in vivo using tiling DNA microarrays Nature Methods. 2006 Jul;3(7):511-8.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Sabo PJ, Hawrylycz M, Wallace JC, Humbert R, Yu M, Shafer A, Kawamoto J, Hall R, Mack J, Dorschner M et al. Discovery of functional noncoding elements by digital analysis of chromatin structure. PNAS. 2004 Nov 30;101(48):16837-42.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><B>Data Release Policy</B></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Data users may freely use ENCODE data, but may not, without prior consent, submit publications that use an unpublished ENCODE dataset until nine months following the release of the dataset. This date is listed in the Restricted Until column, above. The full data release policy for ENCODE is available <A HREF=""http://genome.ucsc.edu/ENCODE/terms.html"" TARGET=""_blank""><B><U>here.</U></B></A></P>',6584,3405,28,'dnix','2009-10-19 00:00:00')
,(6584,'UCSC','<P ALIGN=""LEFT"">Downloaded from <A HREF=""http://genome.ucsc.edu/ENCODE/downloads.html"" TARGET=""_blank""><B><U>http://genome.ucsc.edu/ENCODE/downloads.html</U></B></A></P>',6526,3405,28,'dnix','2009-10-19 00:00:00')
,(6585,'Common Cell CNV','ENCODE Common Cell Type Copy Number Variation, by Illumina 1M and CBS',6584,3405,28,'dnix','2009-10-19 00:00:00')
,(6588,'NHGRI NRE','ENCODE NHGRI Elnitski Negative Regulatory Elements',6584,3405,28,'dnix','2009-10-19 00:00:00')
,(6598,'ChIP-Seq','',6539,3405,7,'dnix','2009-10-30 00:00:00')
,(6599,'EmpFDR','<P ALIGN=""LEFT"">EmpFDR: -10Log10(FDR) - Empirical false discovery rates based on a control vs control null distribution.</P>',6598,3405,7,'dnix','2009-10-30 00:00:00')
,(6600,'NormDiff','<P ALIGN=""LEFT"">NormDiff: (Sum T - Sum C)/ Sqrt (Sum T + Sum C) enrichment score</P>',6598,3405,7,'dnix','2009-10-30 00:00:00')
,(6601,'QValueFDR','<P ALIGN=""LEFT"">QValFDR: -10Log10(q-values) - Window level bionomial p-values converted into q-value FDRs using John Storey&apos;s R package. </P>',6598,3405,7,'dnix','2009-10-30 00:00:00')
,(6602,'WeyrichLab','',6371,6300,30,'dnix','2009-11-25 00:00:00')
,(6603,'FirpoLab',NULL,6371,6300,31,'bmilash','2009-12-22 00:00:00')
,(6604,'Peaks','<P ALIGN=""LEFT"">Regions of signal enrichment based on processed data (usually normalized data from pooled replicates). ENCODE Peaks tables contain fields for statistical significance, including FDR (qValue).</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Converted to <A HREF=""http://useq.sourceforge.net/useqArchiveFormat.html"" TARGET=""_blank""><U>xxx.useq</U></A> binary format using the <A HREF=""http://useq.sourceforge.net/cmdLnMenus.html#Text2USeq"" TARGET=""_blank""><U>Text2USeq app</U></A> (assuming column 8 is qvalue. little documentation, java -jar -Xmx10G ~/USeqApps/Text2USeq -f /home/u0028003/ENCODE/YaleTFBS/NarrowPeak -g H_sapiens_Mar_2006 -c 0 -b 1 -e 2 -v 8)</P>',6573,3405,28,'dnix','2009-12-28 00:00:00')
,(6605,'Signal','<P ALIGN=""LEFT"">Density graph of signal enrichment based on processed data.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">No documentation on what the signa actually represents.  Converted to <A HREF=""http://useq.sourceforge.net/useqArchiveFormat.html"" TARGET=""_blank""><U>xxx.useq</U></A> format using the <A HREF=""http://useq.sourceforge.net/cmdLnMenus.html#Wig2USeq"" TARGET=""_blank""><U>Wig2USeq app</U></A> (java -jar -Xmx4G ~/USeqApps/Wig2USeq -f . -v H_sapiens_Mar_2006) </P>',6573,3405,28,'dnix','2009-12-28 00:00:00')
,(6606,'RNA-Seq','',6602,6300,30,'dnix','2009-12-28 00:00:00')
,(6607,'Plat_Meg1st Pass1Ln','<P ALIGN=""LEFT"">RNA-Seq analysis comparing platelets vs megakaryocytes using the <A HREF=""http://useq.sourceforge.net/"" TARGET=""_blank""><U>USeq analysis package</U></A> </P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">6914x1 and 6903x2</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">See https://hci-as1.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A80</P>',6606,6300,30,'dnix','2009-12-28 00:00:00')
,(6608,'ScanSeqs',NULL,6607,6300,30,'dnix','2009-12-28 00:00:00')
,(6609,'Regions',NULL,6608,6300,30,'dnix','2009-12-28 00:00:00')
,(6610,'Signal','',6608,6300,30,'dnix','2009-12-28 00:00:00')
,(6611,'SpliceJunctions',NULL,6607,6300,30,'dnix','2009-12-28 00:00:00')
,(6612,'ReadCoverage','<P ALIGN=""LEFT"">Basically a hit track of the number of reads that cover a given base</P>',6606,6300,30,'dnix','2009-12-28 00:00:00')
,(6614,'Peaks','<P ALIGN=""LEFT"">ChIP-Seq Peak calls</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Regions of signal enrichment based on processed data (usually normalized data from pooled replicates). ENCODE Peaks tables contain fields for statistical significance. Peaks for this track include a signalValue and pValue. The signalValue represents the fold enrichment of reads across the length of the interval, relative to random expectation. The pValue reflects the likelihood of observing an interval of the given length and signalValue at random. A long interval with a moderate signalValue and a short interval with a high signalValue can therefore have the same pValue.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Converted from broadPeak format to useq format using the <A HREF=""http://useq.sourceforge.net/cmdLnMenus.html#Text2USeq"" TARGET=""_blank""><U>Text2USeq</U></A> app (java -jar -Xmx4G ~/USeqApps/Text2USeq -g H_sapiens_Mar_2006 -f . -c 0 -b 1 -e 2 -v 7)</P>',6574,3405,28,'u0028003','2009-12-30 00:00:00')
,(6615,'Signal','<P ALIGN=""LEFT"">Density graph of signal enrichment based on processed data.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">No documentation on what the signa actually represents. Converting variable step wig signal files using the <A HREF=""http://useq.sourceforge.net/cmdLnMenus.html#Wig2USeq"" TARGET=""_blank""><B><U>Wig2USeq</U></B></A> app ( java -jar -Xmx4G  ~/USeqApps/Wig2USeq -f . -v H_sapiens_Mar_2006)</P>',6574,3405,28,'u0028003','2009-12-30 00:00:00')
,(6620,'Peaks','<P ALIGN=""LEFT"">DNaseI hypersensitive sites (DHSs) identified as signal peaks within FDR 0.5% hypersensitive zones.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Coverted to useq format using the <A HREF=""http://useq.sourceforge.net/cmdLnMenus.html#Text2USeq"" TARGET=""_blank""><B><U>Text2USeq app</U></B></A> (no documentation on what the score columns represent, going with column 7, p-val?: java -jar -Xmx4G ~/USeqApps/Text2USeq -g H_sapiens_Mar_2006 -f . -c 0 -b 1 -e 2 -v 7)</P>',6583,3405,28,'u0028003','2010-01-04 00:00:00')
,(6621,'Hot Spots','<P ALIGN=""LEFT"">DNaseI hypersensitive zones identified using the HotSpot algorithm.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Coverted to useq format using the <A HREF=""http://useq.sourceforge.net/cmdLnMenus.html#Text2USeq"" TARGET=""_blank""><B><U>Text2USeq app</U></B></A> (no documentation on what the score columns represent, going with column 7, p-val?: java -jar -Xmx4G ~/USeqApps/Text2USeq -g H_sapiens_Mar_2006 -f . -c 0 -b 1 -e 2 -v 7)</P>',6583,3405,28,'u0028003','2010-01-04 00:00:00')
,(6622,'Signal','<P ALIGN=""LEFT"">Density graph of signal enrichment based on aligned read density.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Coverted to useq format using the <A HREF=""http://useq.sourceforge.net/cmdLnMenus.html#Wig2USeq"" TARGET=""_blank""><B><U>Wig2USeq</U></B></A> (no documentation on what the score column represents: java -jar -Xmx8G ~/USeqApps/Wig2USeq -v H_sapiens_Mar_2006 -f .)</P>',6583,3405,28,'u0028003','2010-01-04 00:00:00')
,(6624,'Peaks','<P ALIGN=""LEFT"">No information available.</P>',6575,3405,28,'u0028003','2010-01-05 00:00:00')
,(6625,'Clusters','<P ALIGN=""LEFT""><I>Plus and Minus Clusters</I> - These views display clusters of overlapping read mappings on the forward and reverse genomic strands.</P>',6578,3405,28,'u0028003','2010-01-05 00:00:00')
,(6626,'Paired Reads',NULL,6607,6300,30,'u0028003','2010-01-11 00:00:00')
,(6627,'Hyperplastic Polyposis',NULL,6412,3405,16,'HagedornLab','2010-01-13 00:00:00')
,(6629,'Mammalian Conservation',NULL,6390,3405,16,'HagedornLab','2010-01-21 00:00:00')
,(6630,'Primate Conservation Chr14',NULL,6629,3405,16,'HagedornLab','2010-01-21 00:00:00')
,(6632,'Vertebrate Conservation Chr14',NULL,6629,3405,16,'HagedornLab','2010-01-21 00:00:00')
,(6633,'HAIB Methyl27',NULL,6584,3405,28,'u0028003','2010-01-22 00:00:00')
,(6634,'HAIB Methyl-seq',NULL,6584,3405,28,'u0028003','2010-01-22 00:00:00')
,(6636,'Transfrags',NULL,6580,3405,28,'u0028003','2010-01-22 00:00:00')
,(6637,'Raw Signal',NULL,6580,3405,28,'u0028003','2010-01-22 00:00:00')
,(6638,'WeyrichLab',NULL,3405,3405,30,'tolley','2010-02-17 00:00:00')
,(6639,'ElandParseSamples','',6638,3405,30,'tolley','2010-02-17 00:00:00')
,(6640,'WeyrichLab',NULL,3806,3806,30,'tolley','2010-02-17 00:00:00')
,(6641,'ElandParse',NULL,6640,3806,30,'tolley','2010-02-17 00:00:00')
,(6642,'NixReDo',NULL,6528,3405,13,'u0028003','2010-02-19 00:00:00')
,(6643,'ReadCoverage',NULL,6642,3405,13,'u0028003','2010-02-19 00:00:00')
,(6644,'WindowTracks',NULL,6642,3405,13,'u0028003','2010-02-19 00:00:00')
,(6645,'BinomialPValues',NULL,6644,3405,13,'u0028003','2010-02-19 00:00:00')
,(6646,'Top 100 Regions',NULL,6642,3405,13,'u0028003','2010-02-22 00:00:00')
,(6647,'Window Tracks',NULL,6929,3806,24,'u0028003','2010-02-24 00:00:00')
,(6648,'Peaks',NULL,6929,3806,24,'u0028003','2010-02-24 00:00:00')
,(6649,'Read Coverage Tracks',NULL,6929,3806,24,'u0028003','2010-02-24 00:00:00')
,(6650,'ChIP-Seq',NULL,6343,3806,24,'u0028003','2010-02-24 00:00:00')
,(6651,'PEA3','<P ALIGN=""LEFT"">Processing of Pete&apos;s human data hg18 6652R by Nix</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">ETV4 IP from PC3 cells (experimental) and two lanes of PC3 input (control).</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">java -jar -Xmx4G ~/USeqApps/ElandParser -v H_sapiens_Mar_2006 -f Input </P><P ALIGN=""LEFT"">java -jar -Xmx4G ~/USeqApps/ElandParser -v H_sapiens_Mar_2006 -f ChIP</P><P ALIGN=""LEFT"">java -jar -Xmx4G ~/USeqApps/FilterPointData -p Input/PointData0bp/,ChIP/PointData0bp/ -r /home/Annotation/Human/Hg18/Repeats/hg18Satellite.bed.gz</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">jj ~/USeqApps/PointDataManipulator -i -o -p PointData0bp_hg18Satellite.bed_Filt0bp/ -s DupSatFilt_ChIP_PointData</P><P ALIGN=""LEFT"">8044384	Number starting reads</P><P ALIGN=""LEFT"">7118901	Number collapsed reads</P><P ALIGN=""LEFT"">0.885	Fraction unique reads</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">jj ~/USeqApps/PointDataManipulator -i -o -p PointData0bp_hg18Satellite.bed_Filt0bp/ -s DupSatFilt_Input_PointData</P><P ALIGN=""LEFT"">14747797	Number starting reads</P><P ALIGN=""LEFT"">13612598	Number collapsed reads</P><P ALIGN=""LEFT"">0.923	Fraction unique reads</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">jj ~/USeqApps/PeakShiftFinder -s ../PeakShiftFinder -t ../ChIP/DupSatFilt_ChIP_PointData/ -c DupSatFilt_Input_PointData/</P><P ALIGN=""LEFT"">Calculating composite shift...</P><P ALIGN=""LEFT"">	499	5&apos; Peak</P><P ALIGN=""LEFT"">	752	3&apos; Peak</P><P ALIGN=""LEFT"">	253	Composite peak shift</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Calculating individual peak shifts for top 81...</P><P ALIGN=""LEFT"">	286	Median peak shift</P><P ALIGN=""LEFT"">	294	Mean peak shift</P><P ALIGN=""LEFT"">	75.0	Stnd dev peak shifts</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Going with 253 + 75 = 328 -&gt; 330, kind of on the large size?!</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">jj ~/USeqApps/ScanSeqs -p 330 -q -s PetesPEA3 -t ../../SeqData/PetesData/ChIP/DupSatFilt_ChIP_PointData/ -c ../../SeqData/PetesData/Input/DupSatFilt_Input_PointData/ </P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Arguments: -p 330 -q -s PetesPEA3 -t ../../SeqData/PetesData/ChIP/DupSatFilt_ChIP_PointData/ -c ../../SeqData/PetesData/Input/DupSatFilt_Input_PointData/</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Scanning for enriched regions...</P><P ALIGN=""LEFT"">	2	Minium number reads in window</P><P ALIGN=""LEFT"">	330	Peak shift</P><P ALIGN=""LEFT"">	330	Window size</P><P ALIGN=""LEFT"">Calculating read count stats and binomial p-value look up tables...</P><P ALIGN=""LEFT"">	7118893	Treatment Observations</P><P ALIGN=""LEFT"">	13612589	Control Observations</P><P ALIGN=""LEFT"">	</P><P ALIGN=""LEFT"">jj ~/USeqApps/EnrichedRegionMaker -f PetesPEA3/ -s 20,20,1 -i 1,2,4</P><P ALIGN=""LEFT"">Processing windowData330bp.swi...</P><P ALIGN=""LEFT"">	330	Max gap </P><P ALIGN=""LEFT"">	Thresholds:</P><P ALIGN=""LEFT"">	20.0	QValueFDR score threshold</P><P ALIGN=""LEFT"">	20.0	EmpFDR score threshold</P><P ALIGN=""LEFT"">	1.0	Log2((sumT+1)/(sumC+1)) score threshold</P><P ALIGN=""LEFT"">	2934	Enriched Regions created</P><P ALIGN=""LEFT"">	</P><P ALIGN=""LEFT"">	</P><P ALIGN=""LEFT"">jj ~/USeqApps/ReadCoverage -s ReadCoverage/PEA3ChIP -p ChIP/DupSatFilt_ChIP_PointData/</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Used UCSC&apos;s liftover tool to convert 2934 ers to mm9 coordinates, lost 600.</P>',6528,3405,13,'u0028003','2010-02-24 00:00:00')
,(6652,'Window Tracks',NULL,6651,3405,13,'u0028003','2010-02-24 00:00:00')
,(6654,'Read Coverage Tracks',NULL,6651,3405,13,'u0028003','2010-02-24 00:00:00')
,(6655,'Peaks',NULL,6651,3405,13,'u0028003','2010-02-24 00:00:00')
,(6656,'Oler_2010','',6538,3405,7,'Andrew','2010-02-26 00:00:00')
,(6657,'Mouse_platelets','<P ALIGN=""LEFT""></P>',6640,3806,30,'tolley','2010-03-01 00:00:00')
,(6658,'Brf1',NULL,6662,3405,7,'Andrew','2010-03-02 00:00:00')
,(6659,'Pol3',NULL,6662,3405,7,'Andrew','2010-03-02 00:00:00')
,(6660,'Brf2',NULL,6662,3405,7,'Andrew','2010-03-02 00:00:00')
,(6661,'TFIIIC63',NULL,6662,3405,7,'Andrew','2010-03-02 00:00:00')
,(6662,'HeLa-S3','',6656,3405,7,'Andrew','2010-03-02 00:00:00')
,(6663,'Other',NULL,6656,3405,7,'Andrew','2010-03-02 00:00:00')
,(6664,'Pol3_HEK293T','',6663,3405,7,'Andrew','2010-03-02 00:00:00')
,(6665,'Pol3_HFF','',6663,3405,7,'Andrew','2010-03-02 00:00:00')
,(6666,'Pol3_Jurkat',NULL,6663,3405,7,'Andrew','2010-03-02 00:00:00')
,(6668,'D_rerio_Dec_2008',NULL,NULL,6301,NULL,NULL,NULL)
,(6669,'Scaffold NA Info','<P ALIGN=""LEFT"">This folder contains region files describing where the scaffold and na contings were joined with 1kb N spacers.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">See <A HREF=""https://hci-as1.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A88&isGuest=Y"" TARGET=""_blank""><B><U>https://hci-as1.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A88&amp;isGuest=Y</U></B></A></P>',6668,6301,28,'u0028003','2010-03-03 00:00:00')
,(6670,'UARP','',6371,6300,8,'u0028003','2010-03-04 00:00:00')
,(6671,'6980X1',NULL,6670,6300,8,'u0028003','2010-03-04 00:00:00')
,(6672,'Microarray Exome Design','<P ALIGN=""LEFT"">See <A HREF=""https://hci-as1.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A67&isGuest=Y"" TARGET=""_blank""><B><U>https://hci-as1.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A67&amp;isGuest=Y</U></B></A></P>',6670,6300,8,'u0028003','2010-03-04 00:00:00')
,(6673,'Single Lane Data',NULL,6413,3405,16,'u0028003','2010-03-05 00:00:00')
,(6674,'Two Lane Data',NULL,6413,3405,16,'u0028003','2010-03-05 00:00:00')
,(6675,'Read Coverage',NULL,6674,3405,16,'u0028003','2010-03-05 00:00:00')
,(6676,'ScanSeqs',NULL,6674,3405,16,'u0028003','2010-03-05 00:00:00')
,(6677,'ARUP',NULL,6371,6300,15,'u0028003','2010-03-10 00:00:00')
,(6678,'Read Coverage 1Ln FC66','<P ALIGN=""LEFT"">Number of alignments overlapping every bp. See <A HREF=""http://useq.sourceforge.net/cmdLnMenus.html#ReadCoverage"" TARGET=""_blank""><B><U>http://useq.sourceforge.net/cmdLnMenus.html#ReadCoverage</U></B></A></P>',6677,6300,15,'u0028003','2010-03-10 00:00:00')
,(6679,'Helicose RNA Seq','<P ALIGN=""LEFT""><B>See </B><A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeHelicosRnaSeq"" TARGET=""_blank""><B><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&amp;g=wgEncodeHelicosRnaSeq</U></B></A><B></B></P><P ALIGN=""LEFT""><B></B></P><P ALIGN=""LEFT""><B>Description</B></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">This track depicts high throughput sequencing of long RNAs (&gt;200 nt) from whole cell RNA samples from tissues or sub cellular compartments from cell lines included in the ENCODE Transcriptome subproject. The overall goal of the ENCODE project is to identify and characterize all functional elements in the sequence of the human genome. RNA-Seq was performed by reverse-transcribing an RNA sample into cDNA, followed by high throughput DNA sequencing of the cDNA, which was done here on Helicos? Genetic Analysis System (Harris et al; http://www.helicosbio.com/).</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Display Conventions and Configuration</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Raw Signal</P><P ALIGN=""LEFT"">Density graph (wiggle) of the number of reads overlapping a nucleotide in the genome.</P><P ALIGN=""LEFT"">To show only selected subtracks, uncheck the boxes next to the tracks that you wish to hide. Color differences among the views are arbitrary. They provide a visual cue for distinguishing between the different cell types and compartments.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Note that the strand of the RNA is not displayed in the track in the genome browser. The strand can be found in the download file.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Methods</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Cells were grown according to the approved ENCODE cell culture protocols. RNA molecules longer than 200 nt and present in RNA population isolated from different subcellular compartments (such as cytosol, nucleus, polysomes and others) were fractionated into polyA+ and polyA- fractions as described in these protocols.</P><P ALIGN=""LEFT"">RNA was converted into first strand cDNA using a high excess of random hexamers without prior fragmentation. Spurious second-strand cDNA synthesis could occur under these conditions. The first strand cDNA molecules were tailed at the 3? ends with polyA residues using terminal transferase and used directly for sequencing.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Filtered reads were aligned to the human genome using in-house and freely available Helicos Alignment software indexDPgenomic (http://open.helicosbio.com/mwiki/index.php/Docs/Software/Bioinformatics#Executables, requires registration (free)) with a minimum normalized alignment score of 4.5. The normalized score was defined as following:</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Score=(#matches*5-#mismatches*4)/length_read</P><P ALIGN=""LEFT"">For example, in the following alignment:</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Tag Sequence CCTCCGTGTTGTTCCAGCC-CAGTGCTCGCAGG </P><P ALIGN=""LEFT"">Ref Sequence C-TCCGTGTTGTTCCAGCCACAGTGCTCGCAGG</P><P ALIGN=""LEFT"">Length of alignment block: 33</P><P ALIGN=""LEFT"">Length of tag sequence: 32</P><P ALIGN=""LEFT"">Number of matches: 31</P><P ALIGN=""LEFT"">Number of errors: 2</P><P ALIGN=""LEFT"">Score: (31*5) - (2*4) = 155 - 8 = 147</P><P ALIGN=""LEFT"">Normalized score = 147/32 = 4.59375</P><P ALIGN=""LEFT"">Raw data can be found at Helicos (requires registration (free)).</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Verification</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Known exon maps as displayed on the genome browser are confirmed by the alignment of sequence reads.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Credits</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Helicos BioSciences: Philipp Kapranov, Eldar Giladi, Steve Roels, Chris Hart, Stan Letovsky, Patrice Milos.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Cold Spring Harbor Laboratory: Carrie Davis, Kim Bell, Huaien Wang, Tom Gingeras.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Contacts: Philipp Kapranov ; Patrice Milos</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">References</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Harris TD, Buzby PR, Babcock H, Beer E, Bowers J, Braslavsky I, Causey M, Colonell J, Dimeo J, Efcavitch JW, Giladi E, Gill J, Healy J, Jarosz M, Lapen D, Moulton K, Quake SR, Steinmann K, Thayer E, Tyurina A, Ward R, Weiss H, Xie Z. Single-molecule DNA sequencing of a viral genome Science. 2008 Apr 4;320(5872):106-9</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Data Release Policy</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Data users may freely use ENCODE data, but may not, without prior consent, submit publications that use an unpublished ENCODE dataset until nine months following the release of the dataset. This date is listed in the Restricted Until column, above. The full data release policy for ENCODE is available here.</P><P ALIGN=""LEFT""></P>',6584,3405,28,'u0028003','2010-03-24 00:00:00')
,(6680,'Gemcitibine',NULL,6603,6300,31,'bmilash','2010-03-24 00:00:00')
,(6681,'Lessnick Lab',NULL,6371,6300,10,'u0028003','2010-03-29 00:00:00')
,(6682,'RNA-Seq',NULL,6681,6300,10,'u0028003','2010-03-29 00:00:00')
,(6683,'7190R',NULL,6682,6300,10,'u0028003','2010-03-29 00:00:00')
,(6684,'Read Coverage',NULL,6683,6300,10,'u0028003','2010-03-29 00:00:00')
,(6685,'ChIP-Seq',NULL,6681,6300,10,'u0028003','2010-03-29 00:00:00')
,(6686,'7180R',NULL,6685,6300,10,'u0028003','2010-03-29 00:00:00')
,(6687,'ReadCoverage',NULL,6686,6300,10,'u0028003','2010-03-29 00:00:00')
,(6688,'Window Enrichment Tracks','',6686,6300,10,'u0028003','2010-03-29 00:00:00')
,(6689,'Binding Peaks',NULL,6686,6300,10,'u0028003','2010-03-29 00:00:00')
,(6691,'RNATotal_RNA-seq','',6662,3405,7,'Andrew','2010-04-08 00:00:00')
,(6692,'RNACapped_RNA-seq','',6662,3405,7,'Andrew','2010-04-08 00:00:00')
,(6693,'INDELS',NULL,6478,3405,8,'u0028003','2010-04-13 00:00:00')
,(6694,'All',NULL,6484,3405,8,'u0028003','2010-04-13 00:00:00')
,(6695,'UniDup',NULL,6484,3405,8,'u0028003','2010-04-13 00:00:00')
,(6697,'7202X1','<P ALIGN=""LEFT"">See <A HREF=""https://hci-as1.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A96"" TARGET=""_blank""><B><U>https://hci-as1.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A96</U></B></A></P>',6716,3405,7,'u0028003','2010-04-13 00:00:00')
,(6699,'Graves Lab',NULL,3806,3806,13,'u0028003','2010-04-16 00:00:00')
,(6700,'Jarid1a ChIP-Seq','<P ALIGN=""LEFT"">Reprocessing of Valouev&apos;s chIP-seq data for Jarid1a, http://www.ncbi.nlm.nih.gov/pubmed/20064375.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">See <A HREF=""https://hci-as1.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A000000&isGuest=Y"" TARGET=""_blank""><B><U>https://hci-as1.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A97&amp;isGuest=Y</U></B></A></P>',6699,3806,13,'u0028003','2010-04-16 00:00:00')
,(6701,'Enrichment Tracks',NULL,6700,3806,13,'u0028003','2010-04-16 00:00:00')
,(6702,'Peak Picks',NULL,6700,3806,13,'u0028003','2010-04-16 00:00:00')
,(6703,'Jarid1a',NULL,6528,3405,13,'u0028003','2010-04-19 00:00:00')
,(6705,'Yoon Lab',NULL,3806,3806,35,'u0028003','2010-04-22 00:00:00')
,(6706,'Read Coverage','',6952,3806,35,'u0028003','2010-04-22 00:00:00')
,(6707,'Enrichment Tracks',NULL,6952,3806,35,'u0028003','2010-04-22 00:00:00')
,(6708,'Binomial P-Value',NULL,6707,3806,35,'u0028003','2010-04-22 00:00:00')
,(6709,'Q-Value FDR',NULL,6707,3806,35,'u0028003','2010-04-22 00:00:00')
,(6712,'Read Coverage','',6700,3806,13,'u0028003','2010-04-26 00:00:00')
,(6713,'Window Enrichment',NULL,6300,3405,28,'u0028003','2010-04-26 00:00:00')
,(6714,'RNA-Seq Challenge','<P ALIGN=""LEFT"">Source of RNA seq data:</P><P ALIGN=""LEFT"">	50M aligned reads from a human platelet and megakaryocyte unstranded paired RNA-Seq experiment (Weyrich Lab)</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Spike-in generation:</P><P ALIGN=""LEFT"">	Randomly selected 500 non-intersecting genes with &gt; 20 reads </P><P ALIGN=""LEFT"">	Randomized and split the 50M reads into 3rds</P><P ALIGN=""LEFT"">	For each of the 3rds, split the reads into a treatment and control group</P><P ALIGN=""LEFT"">		For the selected 500 genes, skewed the read distribution between 0.2-0.45 and 0.55-0.8 (the skew was selected once and applied to all 3rds)</P><P ALIGN=""LEFT"">		For other mapped reads those falling within each gene were assigned to either group using a randomly selected skewed distribution 0.45-0.55</P><P ALIGN=""LEFT"">	Yields an Overdispersed dataset with 3 treatment and 3 control replicas</P>',6371,6300,28,'u0028003','2010-04-27 00:00:00')
,(6715,'Read Coverage','<P ALIGN=""LEFT"">Relative Read Coverage, #Reads overlapping each base/ million mapped reads in dataset</P>',6714,6300,28,'u0028003','2010-04-27 00:00:00')
,(6716,'RNA-Seq',NULL,6539,3405,7,'u0028003','2010-05-03 00:00:00')
,(6717,'7203X1 Donor 264',NULL,6716,3405,7,'u0028003','2010-05-03 00:00:00')
,(6718,'Jette Lab','',6668,6301,36,'bmilash','2010-05-04 00:00:00')
,(6719,'San Diego Epigenome Center','<P ALIGN=""LEFT"">Data downloaded and in some cases reprocessed from <A HREF=""http://neomorph.salk.edu/human_methylome/data.html"" TARGET=""_blank""><B><U>http://neomorph.salk.edu/human_methylome/data.html</U></B></A></P>',3405,3405,7,'u0028003','2010-05-04 00:00:00')
,(6720,'Read Coverage','<P ALIGN=""LEFT"">Reprocessed data tracks</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">#for small RNA-seq gave funky format, first had to strip out reads with more than three matches</P><P ALIGN=""LEFT"">gunzip -c smrna_h1.gz | awk &apos;$8 &lt;= 3&apos; &gt; smrna_h1_Max3.txt</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">#convert to point data</P><P ALIGN=""LEFT"">for x in $(ls); do java -jar -Xmx2G ~/Software/USeq_6.2/Apps/Tag2Point -v H_sapiens_Mar_2006 -i 3 -b -c -f $x; done</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">#read coverage</P><P ALIGN=""LEFT"">for x in mRNA_H1_Point mRNA_IMR90_Point smRNA_H1_Point smRNA_IMR90_Point; do java -jar -Xmx2G ~/Software/USeq_6.2/Apps/ReadCoverage -p $x -s ReadCoverage/$x ; done</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">See <A HREF=""https://hci-as1.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A103&isGuest=Y"" TARGET=""_blank""><B><U>https://hci-as1.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A103&amp;isGuest=Y</U></B></A></P>',6719,3405,7,'u0028003','2010-05-04 00:00:00')
,(6721,'CSHL Sm RNA-seq','<P ALIGN=""LEFT"">Data was downloaded and in some cases reprocessed from <A HREF=""http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeCshlShortRnaSeq"" TARGET=""_blank""><B><U>http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg18&amp;g=wgEncodeCshlShortRnaSeq</U></B></A> and <A HREF=""http://hgdownload.cse.ucsc.edu/goldenPath/hg18/encodeDCC/wgEncodeCshlShortRnaSeq/"" TARGET=""_blank""><B><U>http://hgdownload.cse.ucsc.edu/goldenPath/hg18/encodeDCC/wgEncodeCshlShortRnaSeq/</U></B></A></P>',6584,3405,28,'u0028003','2010-05-04 00:00:00')
,(6722,'Read Coverage','<P ALIGN=""LEFT"">xxx.tagAlign.gz files were downloaded from UCSC and reprocessed into ReadCoverage tracks using the USeq Tag2Point and ReadCoverage applications</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">for x in $(ls *gz); do echo $x; java -jar -Xmx2G ~/Software/USeq_6.2/Apps/Tag2Point -v H_sapiens_Mar_2006 -f $x; done</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">for x in $(ls); do echo $x; java -jar -Xmx2G ~/Software/USeq_6.2/Apps/ReadCoverage -p $x -s ../ReadCoverage/$x; done</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">java -jar -Xmx2G ~/Software/USeq_6.2/Apps/Bar2USeq -f ReadCoverage/ -e</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">See <A HREF=""https://hci-as1.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A103&isGuest=Y"" TARGET=""_blank""><B><U>https://hci-as1.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A103&amp;isGuest=Y</U></B></A></P>',6721,3405,28,'u0028003','2010-05-04 00:00:00')
,(6723,'SubSample12.5M',NULL,6612,6300,30,'tolley','2010-05-07 00:00:00')
,(6726,'InfertilePatients',NULL,6539,3405,7,'CairnsLab','2010-05-10 00:00:00')
,(6727,'ReprocessedFilteredRNA',NULL,6716,3405,7,'CairnsLab','2010-05-12 00:00:00')
,(6731,'PU.1 LPS Data','<P ALIGN=""LEFT"">GSE19553</P>',6699,3806,13,'GravesLab','2010-05-17 00:00:00')
,(6732,'MACS_PeakCalls','<P ALIGN=""LEFT"">Peak Calls downloaded from GEO</P>',6731,3806,13,'GravesLab','2010-05-17 00:00:00')
,(6733,'Read Coverage',NULL,6731,3806,13,'GravesLab','2010-05-17 00:00:00')
,(6735,'Andrew',NULL,6737,6300,7,'Andrew','2010-05-20 00:00:00')
,(6736,'BL',NULL,6735,6300,7,'Andrew','2010-05-20 00:00:00')
,(6737,'CairnsLab','',6371,6300,7,'Andrew','2010-05-20 00:00:00')
,(6738,'CairnsLab',NULL,3806,3806,7,'Andrew','2010-05-20 00:00:00')
,(6739,'Andrew',NULL,6738,3806,7,'Andrew','2010-05-20 00:00:00')
,(6740,'BL',NULL,6739,3806,7,'Andrew','2010-05-20 00:00:00')
,(6741,'Bis-Seq',NULL,6539,3405,7,'u0028003','2010-05-21 00:00:00')
,(6742,'7259R',NULL,6741,3405,7,'u0028003','2010-05-21 00:00:00')
,(6743,'Window Sums','<P ALIGN=""LEFT"">250bp window sums from single sample ScanSeqs</P>',6742,3405,7,'u0028003','2010-05-21 00:00:00')
,(6744,'BP Parsed Cs',NULL,6742,3405,7,'u0028003','2010-05-21 00:00:00')
,(6745,'Converted Cs 7259X1',NULL,6744,3405,7,'u0028003','2010-05-21 00:00:00')
,(6746,'Converted Cs 7259X2',NULL,6744,3405,7,'u0028003','2010-05-21 00:00:00')
,(6747,'Non Converted Cs 7259X1',NULL,6744,3405,7,'u0028003','2010-05-21 00:00:00')
,(6748,'Non Converted Cs 7259X2',NULL,6744,3405,7,'u0028003','2010-05-21 00:00:00')
,(6749,'AnnotationTracks',NULL,6656,3405,7,'Andrew','2010-05-24 00:00:00')
,(6750,'Cairns Lab',NULL,6668,6301,7,'u0028003','2010-06-02 00:00:00')
,(6751,'7302R',NULL,6750,6301,7,'u0028003','2010-06-02 00:00:00')
,(6752,'Converted C\''s',NULL,6751,6301,7,'u0028003','2010-06-02 00:00:00')
,(6753,'Unconverted C\''s',NULL,6751,6301,7,'u0028003','2010-06-02 00:00:00')
,(6754,'Treatment',NULL,6715,6300,28,'u0028003','2010-06-04 00:00:00')
,(6755,'Control',NULL,6715,6300,28,'u0028003','2010-06-04 00:00:00')
,(6756,'Osama',NULL,6302,1204,25,'BassLab','2010-06-10 00:00:00')
,(6757,'Raw Reads',NULL,6756,1204,25,'BassLab','2010-06-10 00:00:00')
,(6758,'Read Coverage',NULL,6756,1204,25,'BassLab','2010-06-10 00:00:00')
,(6759,'7202X1 With Repeats',NULL,6716,3405,7,'Andrew','2010-06-10 00:00:00')
,(6760,'USeq','<P ALIGN=""LEFT"">This contains reprocessing of ENCODE datasets using USeq/ DESeq</P>',6526,3405,28,'u0028003','2010-06-14 00:00:00')
,(6761,'CTCF Broad','<P ALIGN=""LEFT"">Reprocessing of the Broad&apos;s CTCF data, two replicas each for 9 different cell lines.</P>',6760,3405,28,'u0028003','2010-06-14 00:00:00')
,(6762,'Peak Calls',NULL,6761,3405,28,'u0028003','2010-06-14 00:00:00')
,(6763,'Window Tracks','',6761,3405,28,'u0028003','2010-06-15 00:00:00')
,(6764,'FDR',NULL,6763,3405,28,'u0028003','2010-06-15 00:00:00')
,(6765,'Log2Ratio',NULL,6763,3405,28,'u0028003','2010-06-15 00:00:00')
,(6766,'ReadCoverage','<P ALIGN=""LEFT"">Relataive read coverage, #reads/base/million mapped reads</P>',6761,3405,28,'u0028003','2010-06-15 00:00:00')
,(6767,'Hg18Downloads',NULL,6539,3405,7,'CairnsLab','2010-06-21 00:00:00')
,(6768,'BGI','',6790,6300,38,'u0028003','2010-06-25 00:00:00')
,(6769,'USeq Analysis','',6768,6300,38,'u0028003','2010-06-25 00:00:00')
,(6775,'SNP','',6769,6300,38,'u0028003','2010-07-01 00:00:00')
,(6776,'Maq','',6775,6300,38,'u0028003','2010-07-01 00:00:00')
,(6777,'Non-Synonymous','',6775,6300,38,'u0028003','2010-07-02 00:00:00')
,(6778,'SNP','',6777,6300,38,'u0028003','2010-07-02 00:00:00')
,(6779,'INDEL','',6777,6300,38,'u0028003','2010-07-02 00:00:00')
,(6780,'Weis, John Lab',NULL,3806,3806,19,'u0028003','2010-07-15 00:00:00')
,(6781,'ChIP-Seq',NULL,6780,3806,19,'u0028003','2010-07-15 00:00:00')
,(6782,'Smuc 6264R','<P ALIGN=""LEFT"">See the analysis report https://hci-as1.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A131</P>',6781,3806,19,'u0028003','2010-07-15 00:00:00')
,(6783,'Snai3 6491R','<P ALIGN=""LEFT"">See the analysis report https://hci-as1.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A131</P>',6781,3806,19,'u0028003','2010-07-15 00:00:00')
,(6784,'Read Coverage',NULL,6782,3806,19,'u0028003','2010-07-15 00:00:00')
,(6785,'Window Tracks',NULL,6782,3806,19,'u0028003','2010-07-15 00:00:00')
,(6786,'Read Coverage',NULL,6783,3806,19,'u0028003','2010-07-15 00:00:00')
,(6787,'Window Tracks',NULL,6783,3806,19,'u0028003','2010-07-15 00:00:00')
,(6788,'Enriched Regions',NULL,6782,3806,19,'u0028003','2010-07-15 00:00:00')
,(6790,'Lyon Robison Lab',NULL,6371,6300,38,'u0028003','2010-07-26 00:00:00')
,(6791,'7397X1',NULL,6814,3405,7,'Andrew','2010-07-27 00:00:00')
,(6792,'7397X2',NULL,6814,3405,7,'Andrew','2010-07-27 00:00:00')
,(6793,'BGI Analysis',NULL,6768,6300,38,'u0028003','2010-07-28 00:00:00')
,(6794,'7396X1',NULL,6814,3405,7,'Andrew','2010-07-28 00:00:00')
,(6795,'Ying','<P ALIGN=""LEFT"">This is for test only</P>',6371,6300,28,'Ying','2010-07-28 00:00:00')
,(6796,'TEST',NULL,6795,6300,28,'Ying','2010-07-29 00:00:00')
,(6797,'7406R',NULL,6685,6300,10,'Russell','2010-08-02 00:00:00')
,(6798,'Window Tracks',NULL,6797,6300,10,'Russell','2010-08-02 00:00:00')
,(6799,'shiftless',NULL,6798,6300,10,'Russell','2010-08-02 00:00:00')
,(6806,'MethylationSINEs','',6538,3405,7,'Andrew','2010-08-06 00:00:00')
,(6807,'PredictedmiRNAs','<P ALIGN=""LEFT"">Evan Johnson Analysis</P>',6716,3405,7,'CairnsLab','2010-08-07 00:00:00')
,(6808,'7396X2',NULL,6814,3405,7,'Andrew','2010-08-10 00:00:00')
,(6809,'Tavtigian Lab',NULL,3405,3405,37,'Russell','2010-08-10 00:00:00')
,(6810,'Exome',NULL,6809,3405,37,'Russell','2010-08-10 00:00:00')
,(6811,'whole exome',NULL,6810,3405,37,'Russell','2010-08-10 00:00:00')
,(6812,'capture reagents',NULL,6811,3405,37,'Russell','2010-08-10 00:00:00')
,(6813,'Nimblegen 2.1M HG18',NULL,6812,3405,37,'Russell','2010-08-10 00:00:00')
,(6814,'RelaxedAlignment-t60s1noqualfilt',NULL,6716,3405,7,'Andrew','2010-08-10 00:00:00')
,(6815,'7397X1','<P ALIGN=""LEFT"">100819</P><P ALIGN=""LEFT"">#Ran novoalign for smRNA</P><P ALIGN=""LEFT"">nohup /home/BioApps/novocraft/novoalign -FILMFQ -m -l17 -h90 -t30 -rA 15 -q 3 -d /home/Genomes/Human/Hg18/novo_index/hg18WithSplicesAdaptersk13s3 -a ATCTCGTATGCCGTCTTCTGCTTG -f ~/BioAnalysis/Sue/7397_smRNA/100720_7397X1_s_7_sequence.txt.gz -c 4 &gt;&amp; ~/BioAnalysis/Sue/7397_smRNA/100818_4pm_7397X1.txt &amp;</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">#Ran Novo2Bed.pl on the output file with settings --zerobase and --miRNA</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">#Ran BedAlignFilter.pl on the output of Novo2Bed.pl with settings </P><P ALIGN=""LEFT"">  1)for unique:  --maxAlign 1</P><P ALIGN=""LEFT"">  2)for 2-10: --minAlign 2 --maxAlign 10</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">#Ran BedAlignStats.pl to get statistics of the filtered output files.  Here is a summary:</P><P ALIGN=""LEFT"">	7397X1_unique.bed</P><P ALIGN=""LEFT"">		1604599 alignments (from 1602622 reads, including 1977 reads that have two bed lines because spliced.)</P><P ALIGN=""LEFT"">		Size range 17-36.  Quality scores range between 0 and 118.  Alignment scores range between 0 and 30 (with one at 33).</P><P ALIGN=""LEFT"">	7397X1_2_to_10.bed</P><P ALIGN=""LEFT"">		Repeat_alignments: 3858311		Repeat_reads: 926817</P><P ALIGN=""LEFT"">		Size range 17-36.  Quality score range 0-6 (most 0-1).  Alignment score range 0-35.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">#Made stranded ReadCoverage files.  </P><P ALIGN=""LEFT"">	for i in *Point; do mkdir -p ${i/_Point/_ReadCov}/Plus ${i/_Point/_ReadCov}/Minus; done; for i in *Point; do mv ${i}/*-* ${i/_Point/_ReadCov}/Minus; mv ${i}/*+* ${i/_Point/_ReadCov}/Plus; done</P><P ALIGN=""LEFT"">	for i in *ReadCov/*; do java -Xmx4G -jar ~/USeqApps/ReadCoverage -p $i -s ${i}_ReadCov; done; </P><P ALIGN=""LEFT"">	for i in *ReadCov/*us*ReadCov*; do java -Xmx3G -jar ~/USeqApps/Bar2USeq -f $i -r 1; done</P><P ALIGN=""LEFT"">	for i in *ReadCov; do for s in Minus Plus; do mv ${i}/${s}_ReadCov.useq ${i/_ReadCov/}_${s}_ReadCov.useq; done; done; for i in *Point; do mv ${i/_Point/_ReadCov}/Minus/* $i; mv ${i/_Point/_ReadCov}/Plus/* $i; done; rm -r *ReadCov;</P>',6716,3405,7,'Andrew','2010-08-10 00:00:00')
,(6816,'7397X2','<P ALIGN=""LEFT"">100819</P><P ALIGN=""LEFT"">#Ran novoalign for smRNA</P><P ALIGN=""LEFT"">nohup /home/BioApps/novocraft/novoalign -FILMFQ -m -l17 -h90 -t30 -rA 15 -q 3 -d /home/Genomes/Human/Hg18/novo_index/hg18WithSplicesAdaptersk13s3 -a ATCTCGTATGCCGTCTTCTGCTTG -f ~/BioAnalysis/Sue/7397_smRNA/100720_7397X2_s_8_sequence.txt.gz -c 4 &gt;&amp; ~/BioAnalysis/Sue/7397_smRNA/100819_12am_7397X2.txt &amp;</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">#Ran Novo2Bed.pl on the output file with settings --zerobase and --miRNA</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">#Ran BedAlignFilter.pl on the output of Novo2Bed.pl with settings </P><P ALIGN=""LEFT"">  1)for unique:  --maxAlign 1</P><P ALIGN=""LEFT"">  2)for 2-10: --minAlign 2 --maxAlign 10</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">#Ran BedAlignStats.pl to get statistics of the filtered output files.  Here is a summary:</P><P ALIGN=""LEFT"">	7397X2_unique.bed</P><P ALIGN=""LEFT"">		1135519 alignments (from 1134925 reads, including 594 reads that have two bed lines because spliced.)</P><P ALIGN=""LEFT"">		Size range 17-36.  Quality score range 0-117.  Alignment score range 0-34.</P><P ALIGN=""LEFT"">	7397X2_2_to_10.bed</P><P ALIGN=""LEFT"">		Repeat_alignments: 3041005	Repeat_reads: 762290</P><P ALIGN=""LEFT"">		Size range 17-36.  Quality score range 0-6 (most 0-1).  Alignment score range 0-35.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">#Made stranded ReadCoverage files.  </P><P ALIGN=""LEFT"">	for i in *Point; do mkdir -p ${i/_Point/_ReadCov}/Plus ${i/_Point/_ReadCov}/Minus; done; for i in *Point; do mv ${i}/*-* ${i/_Point/_ReadCov}/Minus; mv ${i}/*+* ${i/_Point/_ReadCov}/Plus; done</P><P ALIGN=""LEFT"">	for i in *ReadCov/*; do java -Xmx4G -jar ~/USeqApps/ReadCoverage -p $i -s ${i}_ReadCov; done; </P><P ALIGN=""LEFT"">	for i in *ReadCov/*us*ReadCov*; do java -Xmx3G -jar ~/USeqApps/Bar2USeq -f $i -r 1; done</P><P ALIGN=""LEFT"">	for i in *ReadCov; do for s in Minus Plus; do mv ${i}/${s}_ReadCov.useq ${i/_ReadCov/}_${s}_ReadCov.useq; done; done; for i in *Point; do mv ${i/_Point/_ReadCov}/Minus/* $i; mv ${i/_Point/_ReadCov}/Plus/* $i; done; rm -r *ReadCov;</P><P ALIGN=""LEFT""></P>',6716,3405,7,'Andrew','2010-08-10 00:00:00')
,(6817,'7396X1','<P ALIGN=""LEFT"">100819</P><P ALIGN=""LEFT"">#Ran novoalign for mRNA</P><P ALIGN=""LEFT"">nohup /home/BioApps/novocraft/novoalign -FILMFQ -m -l17 -t60 -rA 15 -q 3 -d /home/Genomes/Human/Hg18/novo_index/hg18WithSplicesAdaptersk13s3 -a ATCTCGTATGCCGTCTTCTGCTTG -f ~/BioAnalysis/Sue/7396_mRNA/100720_7396X1_s_6_sequence.txt.gz -c 4 &gt;&amp; ~/BioAnalysis/Sue/7396_mRNA/100819_12am_7396X1.txt &amp;</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">#Ran Novo2Bed.pl on the output file with settings --zerobase and --miRNA</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">#Ran BedAlignFilter.pl on the output of Novo2Bed.pl with settings </P><P ALIGN=""LEFT"">  1)for unique:  --maxAlign 1</P><P ALIGN=""LEFT"">  2)for 2-10: --minAlign 2 --maxAlign 10</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">#Ran BedAlignStats.pl to get statistics of the filtered output files.  Here is a summary:</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">#Made stranded ReadCoverage files.  </P><P ALIGN=""LEFT"">	for i in *Point; do mkdir -p ${i/_Point/_ReadCov}/Plus ${i/_Point/_ReadCov}/Minus; done; for i in *Point; do mv ${i}/*-* ${i/_Point/_ReadCov}/Minus; mv ${i}/*+* ${i/_Point/_ReadCov}/Plus; done</P><P ALIGN=""LEFT"">	for i in *ReadCov/*; do java -Xmx4G -jar ~/USeqApps/ReadCoverage -p $i -s ${i}_ReadCov; done; </P><P ALIGN=""LEFT"">	for i in *ReadCov/*us*ReadCov*; do java -Xmx3G -jar ~/USeqApps/Bar2USeq -f $i -r 1; done</P><P ALIGN=""LEFT"">	for i in *ReadCov; do for s in Minus Plus; do mv ${i}/${s}_ReadCov.useq ${i/_ReadCov/}_${s}_ReadCov.useq; done; done; for i in *Point; do mv ${i/_Point/_ReadCov}/Minus/* $i; mv ${i/_Point/_ReadCov}/Plus/* $i; done; rm -r *ReadCov;</P>',6716,3405,7,'Andrew','2010-08-10 00:00:00')
,(6818,'Methylome Project','<P ALIGN=""LEFT"">Bisulfite sequencing of various cell types.</P>',6538,3405,7,'u0028003','2010-08-17 00:00:00')
,(6820,'Lister et. al.','<P ALIGN=""LEFT"">Reprocessing of Lister et. al. 2009 Nature paper &quot;Human DNA methylomes at base resolution show widespread epigenomic differences&quot; with USeq apps.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><A HREF=""http://www.nature.com/nature/journal/v462/n7271/full/nature08514.html"" TARGET=""_blank""><B><U>http://www.nature.com/nature/journal/v462/n7271/full/nature08514.html</U></B></A></P>',6818,3405,7,'u0028003','2010-08-17 00:00:00')
,(6821,'SOLiD RNA-Seq','<P ALIGN=""LEFT"">RNA-Seq analysis of several affected and unaffected autism individuals.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Sort   Sample    Status          Sex  </P><P ALIGN=""LEFT""> 1      30088       Unaffected      Female  </P><P ALIGN=""LEFT""> 2      30090       Unaffected      Male  </P><P ALIGN=""LEFT""> 3      30085       Affected        Male  </P><P ALIGN=""LEFT""> 4      76591       Affected        Male</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">The first three are siblings, and the fourth one is a distant cousin, he shares a set of great-great-great-grandparents with the other three.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">See the following for details:</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Novoalignments at Amazon EC2</P><P ALIGN=""LEFT"">https://hci-as1.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A166</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">RNA-Seq analysis comparing affected vs unaffected</P><P ALIGN=""LEFT"">https://hci-as1.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A170</P>',6670,6300,8,'u0028003','2010-08-19 00:00:00')
,(6822,'Window Transfrag Tracks',NULL,6826,6300,8,'u0028003','2010-08-19 00:00:00')
,(6823,'ReadCoverage',NULL,6821,6300,8,'u0028003','2010-08-19 00:00:00')
,(6824,'Differentially Expressed Genes',NULL,6826,6300,8,'u0028003','2010-08-19 00:00:00')
,(6825,'7396X2','<P ALIGN=""LEFT"">100819</P><P ALIGN=""LEFT"">#Ran novoalign for mRNA</P><P ALIGN=""LEFT"">nohup /home/BioApps/novocraft/novoalign -FILMFQ -m -l17 -t60 -rA 15 -q 3 -d /home/Genomes/Human/Hg18/novo_index/hg18WithSplicesAdaptersk13s3 -a ATCTCGTATGCCGTCTTCTGCTTG -f ~/BioAnalysis/Sue/7396_mRNA/100803_7396X2_s_8_sequence.txt.gz -c 4 &gt;&amp; ~/BioAnalysis/Sue/7396_mRNA/100819_10am_7396X2.txt &amp;</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">#Ran Novo2Bed.pl on the output file with settings --zerobase and --miRNA</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">#Ran BedAlignFilter.pl on the output of Novo2Bed.pl with settings </P><P ALIGN=""LEFT"">  1)for unique:  --maxAlign 1</P><P ALIGN=""LEFT"">  2)for 2-10: --minAlign 2 --maxAlign 10</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">#Ran BedAlignStats.pl to get statistics of the filtered output files.  Here is a summary:</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">#Made stranded ReadCoverage files.  </P><P ALIGN=""LEFT"">	for i in *Point; do mkdir -p ${i/_Point/_ReadCov}/Plus ${i/_Point/_ReadCov}/Minus; done; for i in *Point; do mv ${i}/*-* ${i/_Point/_ReadCov}/Minus; mv ${i}/*+* ${i/_Point/_ReadCov}/Plus; done</P><P ALIGN=""LEFT"">	for i in *ReadCov/*; do java -Xmx4G -jar ~/USeqApps/ReadCoverage -p $i -s ${i}_ReadCov; done; </P><P ALIGN=""LEFT"">	for i in *ReadCov/*us*ReadCov*; do java -Xmx3G -jar ~/USeqApps/Bar2USeq -f $i -r 1; done</P><P ALIGN=""LEFT"">	for i in *ReadCov; do for s in Minus Plus; do mv ${i}/${s}_ReadCov.useq ${i/_ReadCov/}_${s}_ReadCov.useq; done; done; for i in *Point; do mv ${i/_Point/_ReadCov}/Minus/* $i; mv ${i/_Point/_ReadCov}/Plus/* $i; done; rm -r *ReadCov;</P>',6716,3405,7,'Andrew','2010-08-23 00:00:00')
,(6826,'Old With Bad 30090',NULL,6821,6300,8,'u0028003','2010-08-24 00:00:00')
,(6827,'Window Transfrag Tracks',NULL,6821,6300,8,'u0028003','2010-08-24 00:00:00')
,(6828,'TestDeleteMe',NULL,6640,3806,30,'u0028003','2010-09-09 00:00:00')
,(6829,'SpermEpiProject',NULL,6738,3806,7,'Andrew','2010-09-15 00:00:00')
,(6830,'RNA-Seq','',6829,3806,7,'Andrew','2010-09-15 00:00:00')
,(6832,'7506X1','<P ALIGN=""LEFT"">#Ran novoalign for small RNA</P><P ALIGN=""LEFT"">nohup /home/BioApps/novocraft/novoalign -FILMFQ -m -l17 -h90 -t30 -rA 15 -q 3 -d /home/Genomes/Mouse/Mm9/novoindex/mm9WithSplices34Adapterk13s3 -a ATCTCGTATGCCGTCTTCTGCTTG -f ~/BioAnalysis/Sue/7506_mouse/100903_7506X1_s_7_sequence.txt.gz -c 4 &gt;&amp; ~/BioAnalysis/Sue/7506_mouse/100913_12pm_7506X1.txt &amp;</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">#Parsed with Novo2Bed.pl --zerobase --gzip</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">#Filtered with BedAlignFilter.pl --maxAlign 1 --gzip</P><P ALIGN=""LEFT"">   and BedAlignFilter.pl --minAlign 2 --maxAlign 10 --gzip</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">#Converted to PointData</P><P ALIGN=""LEFT"">for i in 7506X1_2_to_10_aln.bed.gz 7506X1_unique.bed.gz; do jj ~/USeqApps/Tag2Point -v M_musculus_Jul_2007 -f $i; done &gt;&gt; Stats_7506X1/wc_tag2point_files.txt </P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">#Made stranded ReadCoverage files</P><P ALIGN=""LEFT"">	for i in *Point; do mkdir -p ${i/_Point/_ReadCov}/Plus ${i/_Point/_ReadCov}/Minus; done; for i in *Point; do mv ${i}/*-* ${i/_Point/_ReadCov}/Minus; mv ${i}/*+* ${i/_Point/_ReadCov}/Plus; done</P><P ALIGN=""LEFT"">	for i in *ReadCov/*; do java -Xmx4G -jar ~/USeqApps/ReadCoverage -p $i -s ${i}_ReadCov; done; </P><P ALIGN=""LEFT"">	for i in *ReadCov/*us*ReadCov*; do java -Xmx3G -jar ~/USeqApps/Bar2USeq -f $i -r 1; done</P><P ALIGN=""LEFT"">	for i in *ReadCov; do for s in Minus Plus; do mv ${i}/${s}_ReadCov.useq ${i/_ReadCov/}_${s}_ReadCov.useq; done; done; for i in *Point; do mv ${i/_Point/_ReadCov}/Minus/* $i; mv ${i/_Point/_ReadCov}/Plus/* $i; done; rm -r *ReadCov;</P>',6830,3806,7,'Andrew','2010-09-15 00:00:00')
,(6833,'7506X2','<P ALIGN=""LEFT"">#Ran novoalign for mRNA</P><P ALIGN=""LEFT"">nohup /home/BioApps/novocraft/novoalign -FILMFQ -m -l17 -t60 -rA 15 -q 3 -d /home/Genomes/Mouse/Mm9/novoindex/mm9WithSplices34Adapterk13s3 -a ATCTCGTATGCCGTCTTCTGCTTG -f ~/BioAnalysis/Sue/7506_mouse/100903_7506X2_s_8_sequence.txt.gz -c 4 2&gt;&amp;1 | nohup gzip &gt; ~/BioAnalysis/Sue/7506_mouse/100913_9pm_7506X2.txt.gz &amp;</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">#Parsed with Novo2Bed.pl --zerobase --gzip</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">#Filtered with BedAlignFilter.pl --maxAlign 1 --gzip</P><P ALIGN=""LEFT"">   and BedAlignFilter.pl --minAlign 2 --maxAlign 10 --gzip</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">#Converted to PointData</P><P ALIGN=""LEFT"">for i in 7506X2_2_to_10_aln.bed.gz 7506X2_unique.bed.gz; do jj ~/USeqApps/Tag2Point -v M_musculus_Jul_2007 -f $i; done &gt;&gt; Stats_7506X2/wc_tag2point_files.txt </P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">#Made stranded ReadCoverage files</P><P ALIGN=""LEFT"">	for i in *Point; do mkdir -p ${i/_Point/_ReadCov}/Plus ${i/_Point/_ReadCov}/Minus; done; for i in *Point; do mv ${i}/*-* ${i/_Point/_ReadCov}/Minus; mv ${i}/*+* ${i/_Point/_ReadCov}/Plus; done</P><P ALIGN=""LEFT"">	for i in *ReadCov/*; do java -Xmx4G -jar ~/USeqApps/ReadCoverage -p $i -s ${i}_ReadCov; done; </P><P ALIGN=""LEFT"">	for i in *ReadCov/*us*ReadCov*; do java -Xmx3G -jar ~/USeqApps/Bar2USeq -f $i -r 1; done</P><P ALIGN=""LEFT"">	for i in *ReadCov; do for s in Minus Plus; do mv ${i}/${s}_ReadCov.useq ${i/_ReadCov/}_${s}_ReadCov.useq; done; done; for i in *Point; do mv ${i/_Point/_ReadCov}/Minus/* $i; mv ${i/_Point/_ReadCov}/Plus/* $i; done; rm -r *ReadCov;</P>',6830,3806,7,'Andrew','2010-09-15 00:00:00')
,(6834,'OtherPol3',NULL,6538,3405,7,'Andrew','2010-10-07 00:00:00')
,(6835,'HeLa (Struhl and Zhao)','',6834,3405,7,'Andrew','2010-10-07 00:00:00')
,(6836,'K562 (Struhl)','',6834,3405,7,'Andrew','2010-10-07 00:00:00')
,(6837,'IMR90 (Hernandez)','',6834,3405,7,'Andrew','2010-10-07 00:00:00')
,(6838,'GM12878 (Snyder)','',6834,3405,7,'Andrew','2010-10-07 00:00:00')
,(6844,'DonorX1',NULL,6848,3405,7,'Andrew','2010-10-15 00:00:00')
,(6846,'DonorX2','',6848,3405,7,'Andrew','2010-10-15 00:00:00')
,(6847,'IMR90',NULL,7037,3405,7,'Andrew','2010-10-15 00:00:00')
,(6848,'Combined',NULL,6716,3405,7,'Andrew','2010-10-15 00:00:00')
,(6849,'SpermRNA_AllLanesCombined',NULL,6716,3405,7,'CairnsLab','2010-10-18 00:00:00')
,(6850,'Exome Sequencing','<P ALIGN=""LEFT""> </P>',6670,6300,8,'u0028003','2010-10-21 00:00:00')
,(6851,'Novel SNP Calls = not in dbSNP','<P ALIGN=""LEFT"">The SNPs in these files are not vound in dbSNP (they do not have an rs number). </P>',6850,6300,8,'u0028003','2010-10-21 00:00:00')
,(6853,'Ets1_Jurkat',NULL,6663,3405,7,'Andrew','2010-10-22 00:00:00')
,(6854,'CBP_Jurkat',NULL,6663,3405,7,'Andrew','2010-10-22 00:00:00')
,(6855,'Chromatin, Pol II','',6662,3405,7,'Andrew','2010-10-29 00:00:00')
,(6857,'HeLa',NULL,6806,3405,7,'Andrew','2010-10-29 00:00:00')
,(6859,'InfertileScore',NULL,6726,3405,7,'CairnsLab','2010-11-09 00:00:00')
,(6860,'Mouse_Fetal_Liver_Megs',NULL,6640,3806,30,'tolley','2010-11-17 00:00:00')
,(6863,'Yandell Lab',NULL,3806,3806,40,'Moore','2010-11-18 00:00:00')
,(6864,'Moon_ChipSeq','<P ALIGN=""LEFT"">ChipSeq Data Nov 2010</P>',6863,3806,40,'Moore','2010-11-18 00:00:00')
,(6865,'BamTest',NULL,6371,6300,28,'u0028003','2010-11-23 00:00:00')
,(6869,'Tophat vs Novoalign',NULL,6735,6300,7,'Andrew','2010-11-30 00:00:00')
,(6870,'Tophat vs. Novoalign',NULL,6739,3806,7,'Andrew','2010-11-30 00:00:00')
,(6871,'D_rerio_Jul_2010',NULL,NULL,6303,NULL,NULL,NULL)
,(6872,'Sue',NULL,6737,6300,7,'CairnsLab','2010-12-06 00:00:00')
,(6873,'DNAseSeq-Duke',NULL,6872,6300,7,'CairnsLab','2010-12-06 00:00:00')
,(6874,'Concatinated NA Scaffold Annotations','<P ALIGN=""LEFT"">Artificial chromosomes &quot;chrNA&quot; and &quot;chrScaffold&quot;  were created to facilitate analysis from the thousands of unassembled Zv9 fasta files:</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">#Merged NA fastas into a concatinate chromosome with 1kb N spacers</P><P ALIGN=""LEFT"">jj ~/Software/USeq/Apps/ConcatinateFastas -d NACon -f NA/ -c chrNA</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">#Merged scaffold fastas into a concatinate chromosome with 1kb N spacers</P><P ALIGN=""LEFT"">jj ~/Software/USeq/Apps/ConcatinateFastas -d ScaffoldConcatinate -f Scaffold/ -c chrScaffold</P>',7084,6303,28,'u0028003','2010-12-06 00:00:00')
,(6875,'Duke_F-seqOut',NULL,6872,6300,7,'CairnsLab','2010-12-09 00:00:00')
,(6877,'DNAseHS',NULL,6539,3405,7,'CairnsLab','2010-12-09 00:00:00')
,(6878,'DukeDNAseNormInput',NULL,6877,3405,7,'CairnsLab','2010-12-09 00:00:00')
,(6879,'DukeDNAse_NoInput',NULL,6877,3405,7,'CairnsLab','2010-12-09 00:00:00')
,(6880,'FAIRE_NormInput',NULL,6877,3405,7,'CairnsLab','2010-12-09 00:00:00')
,(6881,'FAIRE_NoInput',NULL,6877,3405,7,'CairnsLab','2010-12-09 00:00:00')
,(6882,'Chen Lab',NULL,6371,6300,41,'u0028003','2010-12-09 00:00:00')
,(6883,'Novoalignments',NULL,6882,6300,41,'u0028003','2010-12-09 00:00:00')
,(6884,'CEPH data','<P ALIGN=""LEFT"">RNA seq data from ArrayExpress experiment E-MTAB-197:</P><P ALIGN=""LEFT"">Citation: Stephen B. Montgomery, Micha Sammeth, Maria Gutierrez-Arcelus, Radoslaw P. Lach, Catherine Ingle, James Nisbett, Roderic Guigo, Emmanouil T. Dermitzakis. &quot;Transcriptome genetics using second generation sequencing in a Caucasian population&quot;.</P><P ALIGN=""LEFT"">SAM-format alignment data downloaded from http://jungle.unige.ch/rnaseq_CEU60/</P>',6887,3405,42,'bmilash','2010-12-13 00:00:00')
,(6885,'Coverage','<P ALIGN=""LEFT"">Depth of coverage data calculated using USeq version 10.5 SamParser and ReadCoverage applications. Chromosome namess in original SAM files where converted to be compatible with hg18 chromosome names.</P>',6884,3405,42,'bmilash','2010-12-13 00:00:00')
,(6886,'BAM files','<P ALIGN=""LEFT"">.bam and .bam.bai files created from .sam files, after chromosome names changed to be compatible with hg18 chromosome names.</P><P ALIGN=""LEFT"">.bam and .bam.bai files created using samtools 1.10.</P>',6884,3405,42,'bmilash','2010-12-13 00:00:00')
,(6887,'Kriesel lab',NULL,3405,3405,42,'bmilash','2010-12-13 00:00:00')
,(6889,'Caltech Rna-seq','',6662,3405,7,'Andrew','2010-12-13 00:00:00')
,(6890,'FAIRE_NormalizedToInput',NULL,6872,6300,7,'CairnsLab','2010-12-17 00:00:00')
,(6891,'Wu_2011','',2701,2701,7,'u0028003','2010-12-20 00:00:00')
,(6892,'H3K4me3 in zebrafish sperm','',6891,2701,7,'u0028003','2010-12-20 00:00:00')
,(6893,'Cairns Lab',NULL,1204,1204,7,'bmilash','2010-12-30 00:00:00')
,(6894,'Chromatin bound RNAs','<P ALIGN=""LEFT"">Alignments and .bam/.bam.bai files for small RNA samples from C. elegans.</P>',6893,1204,7,'bmilash','2010-12-30 00:00:00')
,(6895,'Chromatin bound RNAs',NULL,6750,6301,7,'bmilash','2010-12-30 00:00:00')
,(6898,'CHD8',NULL,6528,3405,13,'GravesLab','2011-01-12 00:00:00')
,(6899,'Jurkat',NULL,6898,3405,13,'GravesLab','2011-01-12 00:00:00')
,(6900,'window tracks',NULL,6899,3405,13,'GravesLab','2011-01-12 00:00:00')
,(6901,'window tracks new',NULL,6899,3405,13,'GravesLab','2011-01-13 00:00:00')
,(6902,'7711R','',6650,3806,24,'MoonLab','2011-01-13 00:00:00')
,(6903,'read coverage',NULL,6902,3806,24,'MoonLab','2011-01-13 00:00:00')
,(6904,'window tracks',NULL,6902,3806,24,'MoonLab','2011-01-13 00:00:00')
,(6905,'peak picks',NULL,6902,3806,24,'MoonLab','2011-01-13 00:00:00')
,(6907,'Nix Analysis A206','<P ALIGN=""LEFT"">See A206 in GNomEx for details.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><A HREF=""https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A206"" TARGET=""_blank""><B><U>https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A206</U></B></A></P>',6902,3806,24,'u0028003','2011-01-18 00:00:00')
,(6908,'ReadCoverage',NULL,6907,3806,24,'u0028003','2011-01-18 00:00:00')
,(6909,'WindowEnrichmentTracks',NULL,6907,3806,24,'u0028003','2011-01-18 00:00:00')
,(6910,'PeakPicks',NULL,6907,3806,24,'u0028003','2011-01-18 00:00:00')
,(6911,'Laurent et. al.','<P ALIGN=""LEFT"">Reprocessing of Laurent et. al. 2010 Genome Research paper &quot;Dynamic changes in the human methylome during differentiation&quot; with USeq apps.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""><A HREF=""http://genome.cshlp.org/content/20/3/320.full"" TARGET=""_blank""><B><U>http://genome.cshlp.org/content/20/3/320.full</U></B></A></P>',6818,3405,7,'u0028003','2011-01-19 00:00:00')
,(6912,'Base Counts','<P ALIGN=""LEFT"">From the NovoalignBisulfiteParser</P>',7128,3405,7,'u0028003','2011-01-19 00:00:00')
,(6913,'Base Calls','<P ALIGN=""LEFT"">From BisStat</P>',7128,3405,7,'u0028003','2011-01-19 00:00:00')
,(6914,'Window Calls','<P ALIGN=""LEFT"">BisStat analysis</P>',7128,3405,7,'u0028003','2011-01-19 00:00:00')
,(6916,'H3K4me2 in zebrafish sperm',NULL,6891,2701,7,'CairnsLab','2011-01-19 00:00:00')
,(6917,'H3K27me3 in zebrafish sperm',NULL,6891,2701,7,'CairnsLab','2011-01-19 00:00:00')
,(6918,'H3K9me3 in zebrafish sperm',NULL,6891,2701,7,'CairnsLab','2011-01-19 00:00:00')
,(6919,'H3K14ac in zebrafish sperm',NULL,6891,2701,7,'CairnsLab','2011-01-19 00:00:00')
,(6920,'H2AFV in zebrafish sperm',NULL,6891,2701,7,'CairnsLab','2011-01-19 00:00:00')
,(6921,'H3K36me3 in zebrafish sperm','',6891,2701,7,'CairnsLab','2011-01-19 00:00:00')
,(6922,'DNA methylation in zebrafish sperm',NULL,6891,2701,7,'CairnsLab','2011-01-19 00:00:00')
,(6923,'DNA methylation in zebrafish muscle',NULL,6891,2701,7,'CairnsLab','2011-01-19 00:00:00')
,(6928,'PointData',NULL,6907,3806,24,'u0028003','2011-01-21 00:00:00')
,(6929,'Old PEA3','',6650,3806,24,'u0028003','2011-01-25 00:00:00')
,(6932,'Test',NULL,6343,3806,24,'u0028003','2011-01-27 00:00:00')
,(6933,'7789R',NULL,6650,3806,24,'u0028003','2011-01-27 00:00:00')
,(6934,'Relative Read Coverage','',6933,3806,24,'u0028003','2011-01-27 00:00:00')
,(6935,'Window Tracks',NULL,6933,3806,24,'u0028003','2011-01-27 00:00:00')
,(6936,'Peak Picks',NULL,6933,3806,24,'u0028003','2011-01-27 00:00:00')
,(6937,'Differential Methylation','<P ALIGN=""LEFT"">Code for annotations:</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">X1 - Sperm sample 1</P><P ALIGN=""LEFT"">X2 - Sperm sample 2</P><P ALIGN=""LEFT"">X1X2 - Combine Sperm samples 1 and 2</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">ES - Laurent et al. Embryonic Cell Line WA09</P><P ALIGN=""LEFT"">FibES - Laurent et al. Fibroblasts spontaniously derived from Embryonic Cell Line WA09</P><P ALIGN=""LEFT"">FibFore - Laurent et al. Neonatal foreskin fibroblasts</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">H1 - Lister et al. Embryonic Cell Line H1</P><P ALIGN=""LEFT"">IRM90 - Lister et al. Fibroblast cell line IRM90</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">IRM90FibFore - Combine analysis using FibFore and IRM90</P>',6818,3405,7,'u0028003','2011-01-28 00:00:00')
,(6940,'EnrichedRegions','',7006,3405,7,NULL,NULL)
,(6941,'Log2Ratios','',7006,3405,7,NULL,NULL)
,(6942,'PValues','',7006,3405,7,NULL,NULL)
,(6943,'ReducedRegions','',7006,3405,7,NULL,NULL)
,(6944,'Window Tacks',NULL,7125,3405,7,'u0028003','2011-01-31 00:00:00')
,(6945,'Hammoud et. al.',NULL,6818,3405,7,'u0028003','2011-01-31 00:00:00')
,(6946,'Base Calls',NULL,7127,3405,7,'u0028003','2011-01-31 00:00:00')
,(6947,'Base Counts',NULL,7127,3405,7,'u0028003','2011-01-31 00:00:00')
,(6948,'Window Calls',NULL,7127,3405,7,'u0028003','2011-01-31 00:00:00')
,(6949,'Base Calls',NULL,7125,3405,7,'u0028003','2011-01-31 00:00:00')
,(6950,'Base Counts',NULL,7125,3405,7,'u0028003','2011-01-31 00:00:00')
,(6951,'7836R','<P ALIGN=""LEFT"">See https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&amp;analysisNumber=A208 for details</P>',6705,3806,35,'u0028003','2011-02-01 00:00:00')
,(6952,'6722R',NULL,6705,3806,35,'u0028003','2011-02-01 00:00:00')
,(6953,'Read Coverage',NULL,7032,3806,35,'u0028003','2011-02-01 00:00:00')
,(6954,'Enrichment Tracks',NULL,7032,3806,35,'u0028003','2011-02-01 00:00:00')
,(6955,'Peak Picks',NULL,7032,3806,35,'u0028003','2011-02-01 00:00:00')
,(6956,'Stranded Methylation','<P ALIGN=""LEFT"">StrandedBisSeq</P>',7128,3405,7,'u0028003','2011-02-03 00:00:00')
,(6958,'ES Fibroblasts',NULL,6997,3405,7,'u0028003','2011-02-04 00:00:00')
,(6967,'WA09 ES Cells',NULL,6997,3405,7,NULL,NULL)
,(6968,'Fore Fibroblasts',NULL,6997,3405,7,NULL,NULL)
,(6970,'Stranded Methylation','<P ALIGN=""LEFT"">WARNING: we have found substantial bias in which stand is sequenced after bisulfite conversion so the results contained within are suspect!</P>',7125,3405,7,NULL,NULL)
,(6971,'IRM90 Fibroblasts',NULL,6996,3405,7,NULL,NULL)
,(6972,'H1 ES Cells',NULL,6996,3405,7,NULL,NULL)
,(6974,'Stranded Methylation',NULL,7127,3405,7,NULL,NULL)
,(6975,'X1 Sperm',NULL,6998,3405,7,NULL,NULL)
,(6976,'X2 Sperm',NULL,6998,3405,7,NULL,NULL)
,(6977,'X1X2 Sperm',NULL,6998,3405,7,NULL,NULL)
,(6980,'IRM90 QValFDR','',7124,3405,7,'Andrew','2011-02-08 00:00:00')
,(6982,'Window',NULL,6956,3405,7,NULL,NULL)
,(6983,'Log2Ratios',NULL,6982,3405,7,NULL,NULL)
,(6984,'PValues',NULL,6982,3405,7,NULL,NULL)
,(6985,'Regions',NULL,6982,3405,7,NULL,NULL)
,(6987,'Window',NULL,6970,3405,7,NULL,NULL)
,(6988,'Log2Ratios',NULL,6987,3405,7,NULL,NULL)
,(6989,'PValues',NULL,6987,3405,7,NULL,NULL)
,(6990,'Regions',NULL,6987,3405,7,NULL,NULL)
,(6992,'Window',NULL,6974,3405,7,NULL,NULL)
,(6993,'Log2Ratios',NULL,6992,3405,7,NULL,NULL)
,(6994,'PValues',NULL,6992,3405,7,NULL,NULL)
,(6995,'Regions',NULL,6992,3405,7,NULL,NULL)
,(6996,'Base Pair',NULL,6970,3405,7,'u0028003','2011-02-09 00:00:00')
,(6997,'Base Pair',NULL,6956,3405,7,'u0028003','2011-02-09 00:00:00')
,(6998,'Base Pair',NULL,6974,3405,7,'u0028003','2011-02-09 00:00:00')
,(7001,'H1 QValFDR','',7124,3405,7,'Andrew','2011-02-16 00:00:00')
,(7002,'H1','',7037,3405,7,'Andrew','2011-02-17 00:00:00')
,(7003,'Hao',NULL,6863,3806,40,'hao.hu','2011-02-19 00:00:00')
,(7004,'Junji','<P ALIGN=""LEFT"">Chipseq results</P>',7003,3806,40,'hao.hu','2011-02-19 00:00:00')
,(7005,'SequenceBias',NULL,6974,3405,7,'u0028003','2011-02-22 00:00:00')
,(7006,'Two Sample Comparison',NULL,6937,3405,7,'u0028003','2011-02-23 00:00:00')
,(7007,'One Sample Stranded Comparison',NULL,6937,3405,7,'u0028003','2011-02-23 00:00:00')
,(7008,'Log2Ratios',NULL,7007,3405,7,NULL,NULL)
,(7009,'PValues',NULL,7007,3405,7,NULL,NULL)
,(7010,'Regions Not Satellite Filtered','',7007,3405,7,NULL,NULL)
,(7011,'Chromosome 2 files of 6 sujects IS matched SNPs',NULL,6851,6300,8,'UARP','2011-02-24 00:00:00')
,(7012,'Read Coverage',NULL,7127,3405,7,'u0028003','2011-02-25 00:00:00')
,(7017,'ChIPseq',NULL,6739,3806,7,'Andrew','2011-03-01 00:00:00')
,(7018,'QValFDR',NULL,7019,3806,7,'Andrew','2011-03-01 00:00:00')
,(7019,'Liver',NULL,7017,3806,7,'Andrew','2011-03-01 00:00:00')
,(7020,'Spleen',NULL,7017,3806,7,'Andrew','2011-03-01 00:00:00')
,(7021,'QValFDR',NULL,7020,3806,7,'Andrew','2011-03-01 00:00:00')
,(7022,'chr2 is high low med with 8 subjects',NULL,6851,6300,8,'UARP','2011-03-02 00:00:00')
,(7024,'Grunwald Lab','',6871,6303,43,'bmilash','2011-03-03 00:00:00')
,(7025,'RNA-seq',NULL,7024,6303,43,'bmilash','2011-03-03 00:00:00')
,(7026,'Test 5002',NULL,6851,6300,8,'UARP','2011-03-03 00:00:00')
,(7027,'7837',NULL,6650,3806,24,'Moore','2011-03-04 00:00:00')
,(7028,'Moore Analysis',NULL,7027,3806,24,'Moore','2011-03-04 00:00:00')
,(7029,'BAM_bwa','<P ALIGN=""LEFT"">BAM files from bwa alignment, Picard deduplicate, samtools sort</P><P ALIGN=""LEFT"">/Users/bmoore/Projects/11-01-24_Moon_ChIPSeq_7837/alignments</P>',7028,3806,24,'Moore','2011-03-04 00:00:00')
,(7030,'Fire Lab',NULL,1204,1204,28,'u0028003','2011-03-10 00:00:00')
,(7031,'Multimodal RNA-seq, Lamm AT et. al. 2010','<P ALIGN=""LEFT"">Downloaded data from GEO: GSE22410</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Status	Public on Oct 20, 2010</P><P ALIGN=""LEFT"">Title	Multimodal RNA-seq using single-strand, double-strand, and circligase-based capture yields a refined and extended description of the C. elegans transcriptome</P><P ALIGN=""LEFT"">Organism	Caenorhabditis elegans</P><P ALIGN=""LEFT"">Experiment type	Expression profiling by high throughput sequencing</P><P ALIGN=""LEFT"">Summary	We have used a combination of three high-throughput RNA capture and sequencing methods to refine and augment the transcriptome map of a well studied genetic model, Caenorhabditis elegans. The three methods include a standard (non-directional) library preparation protocol relying on cDNA priming and foldback that has been used in several previous studies for transcriptome characterization in this species, and two directional protocols, one involving direct capture of single stranded RNA fragments and one involving circular-template PCR (circligase). We find that each RNA-seq approach shows specific limitations and biases, with the application of multiple methods providing a more complete map than was obtained from any single method. Of particular note in the analysis were substantial advantages of circligase-based and ssRNA-based capture for defining sequences and structures of the precise 5&apos; ends (which were lost using the double strand cDNA capture method). Of the three methods, ssRNA capture was most effective in defining sequences to the polyA junction. Using datasets from a spectrum of C. elegans strains and stages and the UCSC Genome Browser, we provide a series of tools, which facilitate rapid visualization and assignment of gene structures.</P><P ALIGN=""LEFT""> 	</P><P ALIGN=""LEFT"">Overall design	single-strand-capture, double-strand-capture, and circligase-based RNA-seq</P><P ALIGN=""LEFT""> 	</P><P ALIGN=""LEFT"">Contributor(s)	Fire AZ, Lamm AT</P><P ALIGN=""LEFT"">Citation(s)	</P><P ALIGN=""LEFT"">Lamm AT, Stadler MR, Zhang H, Gent JI et al. Multimodal RNA-seq using single-strand, double-strand, and CircLigase-based capture yields a refined and extended description of the C. elegans transcriptome. Genome Res 2011 Feb;21(2):265-75. PMID: 21177965</P><P ALIGN=""LEFT"">Submission date	Jun 17, 2010</P><P ALIGN=""LEFT"">Last update date	Jan 19, 2011</P><P ALIGN=""LEFT"">Contact name	Ayelet Lamm</P><P ALIGN=""LEFT"">E-mail(s)	ayeletla@stanford.edu</P><P ALIGN=""LEFT"">Phone	650-725-1609</P><P ALIGN=""LEFT"">Organization name	Stanford University School of Medicine</P><P ALIGN=""LEFT"">Department	Pathology</P><P ALIGN=""LEFT"">Lab	Fire Lab</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""></P>',7030,1204,28,'u0028003','2011-03-10 00:00:00')
,(7032,'SingleLaneAnalysis',NULL,6951,3806,35,'u0028003','2011-03-11 00:00:00')
,(7033,'MultiLaneAnalysis',NULL,6951,3806,35,'u0028003','2011-03-11 00:00:00')
,(7034,'ReadCoverage',NULL,7033,3806,35,'u0028003','2011-03-11 00:00:00')
,(7036,'Top100Peaks',NULL,7033,3806,35,'u0028003','2011-03-11 00:00:00')
,(7037,'RNA-seq',NULL,6820,3405,7,'Andrew','2011-03-11 00:00:00')
,(7038,'ChrY',NULL,6865,6300,28,'u0028003','2011-03-16 00:00:00')
,(7039,'Vahid',NULL,6737,6300,7,'CairnsLab','2011-03-16 00:00:00')
,(7040,'ReadCoverage',NULL,7039,6300,7,'CairnsLab','2011-03-16 00:00:00')
,(7041,'BamFiles',NULL,7039,6300,7,'CairnsLab','2011-03-17 00:00:00')
,(7042,'DonorSep',NULL,6598,3405,7,'CairnsLab','2011-03-27 00:00:00')
,(7043,'QVal',NULL,7042,3405,7,'CairnsLab','2011-03-27 00:00:00')
,(7044,'P-VAl',NULL,7042,3405,7,'CairnsLab','2011-03-27 00:00:00')
,(7045,'InfertilePatientsSubSampled',NULL,6598,3405,7,'CairnsLab','2011-03-27 00:00:00')
,(7046,'BinP-Val',NULL,7045,3405,7,'CairnsLab','2011-03-27 00:00:00')
,(7049,'All SNPs in raw vcf files','<P ALIGN=""LEFT""> </P>',6850,6300,8,'UARP','2011-03-29 00:00:00')
,(7050,'All Het-Homozygous SNP calls per subject','<P ALIGN=""LEFT"">These files contain all dbSNP AND novel SNPS for each subject.</P><P ALIGN=""LEFT"">A SNP in these files is listed only if the person is heterozygous or homozygous for that SNP.</P><P ALIGN=""LEFT""> </P>',6850,6300,8,'UARP','2011-03-29 00:00:00')
,(7057,'All Contexts',NULL,6946,3405,7,'u0028003','2011-04-05 00:00:00')
,(7058,'All Contexts',NULL,6948,3405,7,'u0028003','2011-04-05 00:00:00')
,(7059,'All Contexts',NULL,6913,3405,7,'u0028003','2011-04-05 00:00:00')
,(7060,'All Contexts',NULL,6914,3405,7,'u0028003','2011-04-05 00:00:00')
,(7061,'All Contexts',NULL,6949,3405,7,'u0028003','2011-04-05 00:00:00')
,(7062,'All Contexts',NULL,6944,3405,7,'u0028003','2011-04-05 00:00:00')
,(7063,'Platelet RNA-seq',NULL,6606,6300,30,'tolley','2011-04-14 00:00:00')
,(7064,'CampLab',NULL,6371,6300,28,'u0028003','2011-04-14 00:00:00')
,(7066,'ChrXY Proc Indep','<P ALIGN=""LEFT"">These datasets were derived from alignments that were aligned to indexes lacking chrY and chrX then merged.</P>',6947,3405,7,'u0028003','2011-04-20 00:00:00')
,(7067,'E_coli_F11_Jun_2008',NULL,NULL,6305,NULL,NULL,NULL)
,(7068,'Magda\''s small and total RNA','',7086,6303,7,'Magda','2011-04-21 00:00:00')
,(7069,'Neklason Lab',NULL,6371,6300,45,'bmilash','2011-04-22 00:00:00')
,(7070,'Resequencing',NULL,7069,6300,45,'bmilash','2011-04-22 00:00:00')
,(7071,'Fastq files',NULL,7070,6300,45,'bmilash','2011-04-22 00:00:00')
,(7072,'Alignments',NULL,7070,6300,45,'bmilash','2011-04-22 00:00:00')
,(7074,'Mulvey Lab','',7067,6305,46,'u0028003','2011-04-26 00:00:00')
,(7075,'7930R TnSeq','',7074,6305,46,'u0028003','2011-04-26 00:00:00')
,(7076,'Read Coverage',NULL,7075,6305,46,'u0028003','2011-04-26 00:00:00')
,(7077,'Window Sumaries',NULL,7080,6305,46,'u0028003','2011-04-26 00:00:00')
,(7079,'Regions',NULL,7080,6305,46,'u0028003','2011-04-27 00:00:00')
,(7080,'Old Combine Analysis',NULL,7075,6305,46,'u0028003','2011-05-02 00:00:00')
,(7081,'Regions',NULL,7075,6305,46,'u0028003','2011-05-02 00:00:00')
,(7083,'Magda\''s Bs-seq data',NULL,7086,6303,7,'Magda','2011-05-04 00:00:00')
,(7084,'Annotations',NULL,6871,6303,28,'u0028003','2011-05-06 00:00:00')
,(7085,'Jones Lab',NULL,6871,6303,14,'u0028003','2011-05-06 00:00:00')
,(7086,'Cairns Lab',NULL,6871,6303,7,'u0028003','2011-05-06 00:00:00')
,(7087,'KalF RNASeq Pilot',NULL,7085,6303,14,'u0028003','2011-05-06 00:00:00')
,(7088,'Window Summaries',NULL,7087,6303,14,'u0028003','2011-05-06 00:00:00')
,(7089,'Read Coverage',NULL,7087,6303,14,'u0028003','2011-05-06 00:00:00')
,(7090,'Coverage',NULL,6718,6301,36,'bmilash','2011-05-11 00:00:00')
,(7091,'Alignments',NULL,6718,6301,36,'bmilash','2011-05-11 00:00:00')
,(7092,'BisSeq',NULL,6857,3405,7,'Andrew','2011-05-25 00:00:00')
,(7093,'ChrXY Proc Indep',NULL,6950,3405,7,'u0028003','2011-05-31 00:00:00')
,(7094,'Differential Methylation','<P ALIGN=""LEFT"">	nohup java -Xmx30G -jar ~/USeqApps/BisSeq -s BisSeq_Rpc155_110618 \\</P><P ALIGN=""LEFT"">	-c NBP_Rpc155_ThreeLanes/MergedHitCountPointData/ConvertedC/ \\</P><P ALIGN=""LEFT"">	-C NBP_HeLaBisInput110615/MergedHitCountPointData/ConvertedC,NBP/MergedHitCountPointData/ConvertedC/ \\</P><P ALIGN=""LEFT"">	-n NBP_Rpc155_ThreeLanes/MergedHitCountPointData/NonConvertedC/ \\</P><P ALIGN=""LEFT"">	-N NBP_HeLaBisInput110615/MergedHitCountPointData/NonConvertedC,NBP/MergedHitCountPointData/NonConvertedC/ &gt;&amp; bisseq_output_110618.txt &amp;</P>',7092,3405,7,'Andrew','2011-06-18 00:00:00')
,(7095,'ChIPseq',NULL,7092,3405,7,'Andrew','2011-06-18 00:00:00')
,(7096,'Bryan Welm Lab',NULL,6371,6300,47,'bmilash','2011-06-28 00:00:00')
,(7097,'Bryan Welm Lab',NULL,3806,3806,47,'bmilash','2011-06-28 00:00:00')
,(7098,'Deininger Lab',NULL,3405,3405,48,'u0028003','2011-07-18 00:00:00')
,(7099,'Ignore','',7098,3405,48,'u0028003','2011-07-18 00:00:00')
,(7100,'Normal',NULL,7107,3405,NULL,NULL,NULL)
,(7101,'CNV',NULL,7100,3405,NULL,NULL,NULL)
,(7102,'LOH',NULL,7100,3405,NULL,NULL,NULL)
,(7103,'Tumor',NULL,7107,3405,NULL,NULL,NULL)
,(7104,'CNV',NULL,7103,3405,NULL,NULL,NULL)
,(7105,'LOH',NULL,7103,3405,NULL,NULL,NULL)
,(7106,'CMML',NULL,7098,3405,48,'u0028003','2011-07-18 00:00:00')
,(7107,'Graphs',NULL,7106,3405,48,'u0028003','2011-07-18 00:00:00')
,(7108,'Regions',NULL,7106,3405,48,'u0028003','2011-07-18 00:00:00')
,(7109,'Tumor',NULL,7108,3405,NULL,NULL,NULL)
,(7110,'Normal',NULL,7108,3405,NULL,NULL,NULL)
,(7111,'Novels',NULL,6736,6300,7,'Andrew','2011-08-09 00:00:00')
,(7112,'Bed',NULL,7111,6300,7,'Andrew','2011-08-10 00:00:00')
,(7113,'Gtf',NULL,7111,6300,7,'Andrew','2011-08-10 00:00:00')
,(7117,'Novels',NULL,6606,6300,30,'oler','2011-09-09 00:00:00')
,(7118,'ChIP-seq',NULL,6602,6300,30,'oler','2011-09-09 00:00:00')
,(7119,'Large Dataset',NULL,6714,6300,28,'u0028003','2011-09-12 00:00:00')
,(7121,'Read Coverage Counts',NULL,7119,6300,28,'u0028003','2011-09-13 00:00:00')
,(7122,'Utah Bioinformatics Shared Resource','',6871,6303,49,'u0785353','2011-09-13 00:00:00')
,(7124,'ChIPSeq',NULL,6820,3405,7,'u0028003','2011-09-13 00:00:00')
,(7125,'BisSeq',NULL,6820,3405,7,'u0028003','2011-09-13 00:00:00')
,(7126,'ChIPSeq',NULL,6945,3405,7,'u0028003','2011-09-13 00:00:00')
,(7127,'BisSeq',NULL,6945,3405,7,'u0028003','2011-09-13 00:00:00')
,(7128,'BisSeq',NULL,6911,3405,7,'u0028003','2011-09-13 00:00:00')
,(7134,'Test folder',NULL,6340,3806,14,'bmilash','2011-09-14 00:00:00')
,(7135,'Jones Lab',NULL,6371,6300,14,'bmilash','2011-09-14 00:00:00')
,(7141,'GNomEx A280','<P ALIGN=""LEFT"">Analysis of pre-publication data from http://www.ebi.ac.uk/ena/data/view/display=html&amp;ERP000016</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Read coverage depth tracks generated using primarily the USeq RNA-Seq analysis pipeline</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Differential expression analysis performed using USeq&apos;s OverDispersedRegionScanSeqs app, which makes use of the DESeq package in R. </P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">These data are associated with analysis A280 (U of U Bioinformatics Shared Resource) in GNomEx</P>',7122,6303,49,NULL,NULL)
,(7161,'Relative read depth coverage','',7141,6303,49,NULL,NULL)
,(7162,'adult female no head ovary',NULL,7161,6303,49,NULL,NULL)
,(7163,'adult male no head testis',NULL,7161,6303,49,NULL,NULL)
,(7164,'embryo 3dpf',NULL,7161,6303,49,NULL,NULL)
,(7165,'adult male head',NULL,7161,6303,49,NULL,NULL)
,(7166,'adult ovary',NULL,7161,6303,49,NULL,NULL)
,(7167,'embryo 1dpf',NULL,7161,6303,49,NULL,NULL)
,(7168,'embryo 2dpf',NULL,7161,6303,49,NULL,NULL)
,(7169,'embryo 5dpf',NULL,7161,6303,49,NULL,NULL)
,(7170,'adult female head',NULL,7161,6303,49,NULL,NULL)
,(7171,'embryo 14dpf',NULL,7161,6303,49,'u0785353','2011-09-19 00:00:00')
,(7172,'BisSeq_8186X1_over8186x3_w1500_FDR30_m20_2x',NULL,7135,6300,14,'Somaye','2011-09-21 00:00:00')
,(7173,'Ayer Lab',NULL,3806,3806,17,'bmilash','2011-09-30 00:00:00')
,(7174,'Small RNA',NULL,7173,3806,17,'bmilash','2011-09-30 00:00:00')
,(7175,'Alignments',NULL,7174,3806,17,'bmilash','2011-09-30 00:00:00')
,(7176,'Coverage',NULL,7174,3806,17,'bmilash','2011-09-30 00:00:00')
,(7177,'MouseStagedSpermatogenesis',NULL,6738,3806,7,'CairnsLab','2011-10-05 00:00:00')
,(7178,'Spermatocyte',NULL,7265,3806,7,'CairnsLab','2011-10-05 00:00:00')
,(7179,'RoundSpermatid',NULL,7265,3806,7,'CairnsLab','2011-10-05 00:00:00')
,(7180,'MatureSperm',NULL,7265,3806,7,'CairnsLab','2011-10-05 00:00:00')
,(7181,'CKit+',NULL,7265,3806,7,'CairnsLab','2011-10-05 00:00:00')
,(7182,'CKIT-',NULL,7265,3806,7,'CairnsLab','2011-10-05 00:00:00')
,(7183,'CG_BisStatResults','',7180,3806,7,'CairnsLab','2011-10-05 00:00:00')
,(7184,'CG_BiStat',NULL,7178,3806,7,'CairnsLab','2011-10-05 00:00:00')
,(7185,'CG_BisStat',NULL,7179,3806,7,'CairnsLab','2011-10-05 00:00:00')
,(7186,'Lane Lab','',3806,3806,50,'u0028003','2011-10-10 00:00:00')
,(7187,'Methyl',NULL,7186,3806,50,'u0028003','2011-10-10 00:00:00')
,(7188,'8326R',NULL,7187,3806,50,'u0028003','2011-10-10 00:00:00')
,(7189,'ReadCoverage',NULL,7188,3806,50,'u0028003','2011-10-10 00:00:00')
,(7190,'ScanSeqs',NULL,7188,3806,50,'u0028003','2011-10-10 00:00:00')
,(7191,'X4_X6 Brain Gla1 + vs - Glu','',7190,3806,50,'u0028003','2011-10-10 00:00:00')
,(7192,'CutRegions',NULL,7187,3806,50,'u0028003','2011-10-10 00:00:00')
,(7193,'Chien Lab',NULL,6871,6303,51,'bmilash','2011-10-11 00:00:00')
,(7194,'Alignments',NULL,7193,6303,51,'bmilash','2011-10-11 00:00:00')
,(7195,'GNomEx A290','<P ALIGN=""LEFT"">Post-publication data from Vesterlund et al. (2011) The zebrafish transcriptome during early development. BMC Dev Biol 11:30</P><P ALIGN=""LEFT"">http://www.biomedcentral.com/1471-213X/11/30</P><P ALIGN=""LEFT"">http://www.ebi.ac.uk/ena/data/view/ERP000635</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Read coverage depth tracks generated using primarily the USeq RNA-Seq analysis pipeline</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Differential expression analysis performed using USeq&apos;s OverDispersedRegionScanSeqs app, which makes use of the DESeq package in R. </P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">These data are associated with analysis A290 (U of U Bioinformatics Shared Resource) in GNomEx</P>',7122,6303,49,NULL,NULL)
,(7196,'GNomEx A303','<P ALIGN=""LEFT"">Post-publication data from Aanes et al. (2011) Zebrafish mRNA sequencing deciphers novelties in transcriptome dynamics during maternal to zygotic transition. Genome Res 21:1328-1338</P><P ALIGN=""LEFT"">http://genome.cshlp.org/content/21/8/1328.full</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Raw data housed here: http://www.ncbi.nlm.nih.gov/gds          GEO DataSets: 22830</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Read coverage depth tracks generated using primarily the USeq RNA-Seq analysis pipeline</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Differential expression analysis performed using USeq&apos;s OverDispersedRegionScanSeqs app, which makes use of the DESeq package in R. </P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">These data are associated with analysis A303 (U of U Bioinformatics Shared Resource) in GNomEx</P>',7122,6303,49,NULL,NULL)
,(7197,'1 cell embryo','<P ALIGN=""LEFT"">relative read depth coverage tracks</P>',7213,6303,49,'u0785353','2011-10-12 00:00:00')
,(7198,'16 cell embryo','<P ALIGN=""LEFT"">relative read depth coverage tracks</P>',7213,6303,49,'u0785353','2011-10-12 00:00:00')
,(7199,'512 cell embryo','<P ALIGN=""LEFT"">relative read depth coverage tracks</P>',7213,6303,49,'u0785353','2011-10-12 00:00:00')
,(7200,'50% epiboly embryo','<P ALIGN=""LEFT"">relative read depth coverage tracks</P>',7213,6303,49,'u0785353','2011-10-12 00:00:00')
,(7201,'unfertilized egg','',7214,6303,49,'u0785353','2011-10-12 00:00:00')
,(7202,'1 cell embryo',NULL,7214,6303,49,'u0785353','2011-10-12 00:00:00')
,(7203,'16-32 cell embryo',NULL,7214,6303,49,'u0785353','2011-10-12 00:00:00')
,(7204,'128-256 cell embryo',NULL,7214,6303,49,'u0785353','2011-10-12 00:00:00')
,(7205,'3.5hpf embryo',NULL,7214,6303,49,'u0785353','2011-10-12 00:00:00')
,(7206,'5.3hpf embryo',NULL,7214,6303,49,'u0785353','2011-10-12 00:00:00')
,(7213,'relative read coverage',NULL,7195,6303,49,'u0785353','2011-10-12 00:00:00')
,(7214,'relative read coverage',NULL,7196,6303,49,'u0785353','2011-10-12 00:00:00')
,(7216,'Differential gene expression-1','<P ALIGN=""LEFT"">Analysis of differential expression using USeq&apos;s OverdispersedRegionScanSeqs app, which utilized R&apos;s DESeq package.</P>',7195,6303,49,'u0785353','2011-10-12 00:00:00')
,(7217,'Differential gene expression-1','<P ALIGN=""LEFT"">Analysis of differential expression using USeq&apos;s OverdispersedRegionScanSeqs app, which utilized R&apos;s DESeq package.</P>',7196,6303,49,'u0785353','2011-10-12 00:00:00')
,(7248,'Alignments',NULL,7188,3806,50,'u0028003','2011-10-13 00:00:00')
,(7249,'X1_X5 Brain Msp1 + vs - Glu','',7190,3806,50,'u0028003','2011-10-13 00:00:00')
,(7250,'X7_X11 Liver Msp1 + vs - Glu',NULL,7190,3806,50,'u0028003','2011-10-13 00:00:00')
,(7251,'Magda\''s annotations',NULL,7086,6303,7,'Magda','2011-10-13 00:00:00')
,(7255,'Differential gene expression-1','<P ALIGN=""LEFT"">Analysis of differential expression using USeq&apos;s OverdispersedRegionScanSeqs app, which utilized R&apos;s DESeq package.</P>',7141,6303,49,'u0785353','2011-10-17 00:00:00')
,(7257,'GNomEx A342','<P ALIGN=""LEFT"">Post-publication analysis of: http://www.sciencedirect.com/science/article/pii/S0012160611001564</P><P ALIGN=""LEFT"">http://www.ncbi.nlm.nih.gov/gds?term=GSE20600</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Extracted fastq files using fastq-dump option in NCBI&apos;S SRA Toolbox: http://trace.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?cmd=show&amp;f=software&amp;m=software&amp;s=software</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">First created novoindex without including a splice junction file.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Aligned against zebrafish genome (Zv9) using novoalign: novoalign -F STDFQ -o SAM $&apos;@RG\\tID:unknownReadgroup\\tSM:unknownSample&apos; -r All 50 -t 60 -d zv9ChrPhiXAdaptrChIPSeq.nix -f SRR038567.fastq | grep -v ^@SQ | grep chr | gzip &gt; SRR038567.sam.gz</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Rep 1 for input, H3K4me1 and H3K4me3 samples consist of fastq files from all six sequencing lanes. After aligning individually with novoalign, I appended a sam header onto each sam file (USeq&apos;s FileJoiner), merged the six sams using Picard&apos;s MergeSamFiles, and passed to USeq&apos;s ChipSeq.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">MergeSamFiles params:</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">java -jar -Xmx2G /home/BioApps/picard-tools-1.43/MergeSamFiles.jar INPUT=SRR038573.sam.gz INPUT=SRR038574.sam.gz... OUTPUT=H3K4me3rep1-1.sam</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Run the ChIPSeq app (USeq) to parse raw alignments, remove duplicate reads, generate read coverage tracks, estimate peak shift, and identify ChIP-Seq peaks.</P>',7122,6303,49,'u0785353','2011-10-19 00:00:00')
,(7261,'bam and bai files',NULL,7141,6303,49,'u0785353','2011-10-19 00:00:00')
,(7262,'bam and bai files',NULL,7195,6303,49,'u0785353','2011-10-19 00:00:00')
,(7263,'bam and bai files',NULL,7196,6303,49,'u0785353','2011-10-19 00:00:00')
,(7264,'test',NULL,6735,6300,7,'oler','2011-10-19 00:00:00')
,(7265,'Bisulfite',NULL,7177,3806,7,'CairnsLab','2011-10-19 00:00:00')
,(7266,'RNASeq','',7177,3806,7,'CairnsLab','2011-10-19 00:00:00')
,(7267,'Spermatocyte',NULL,7266,3806,7,'CairnsLab','2011-10-19 00:00:00')
,(7268,'Spermatid',NULL,7266,3806,7,'CairnsLab','2011-10-19 00:00:00')
,(7269,'CKit+',NULL,7266,3806,7,'CairnsLab','2011-10-19 00:00:00')
,(7270,'CKit-',NULL,7266,3806,7,'CairnsLab','2011-10-19 00:00:00')
,(7271,'Mature',NULL,7266,3806,7,'CairnsLab','2011-10-19 00:00:00')
,(7272,'Differential gene expression-2','<P ALIGN=""LEFT"">Analysis of differential gene expression using USeq&apos;s DefinedRegionScanSeqs (DRSS) app. </P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">DRSS takes chromosome specific PointData xxx.bar.zip files and extracts scores under each region to calculate several statistics including a binomial p-value, Storeyq-value FDR, an empirical FDR, a p-value for strand skew, </P><P ALIGN=""LEFT"">and a chi-square test of independence between the exon read count distributions between treatment and control data (a test for alternative splicing). Several measures of read counts are provided including counts for each strand, a normalized log2 ratio, and RPKMs (# reads per kbof interrogated region per total million mapped reads). If a gene table is provided, scores under each exon are summed to give a whole gene summary. It is also recommended to run a gene table of introns (see the ExportIntronicRegions app) to look for intronic retention and novel transfrags/ exons.  If one provides splice junction bed files for treatment and control RNA-Seq data, </P><P ALIGN=""LEFT"">see the NovoalignParser, splice junctions will be scored for differential expression. This is an additional calculation unrelated to the chi-square independance test. Lastly, if control data is not provided, simple region sums are calculated.</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT"">Params: java -jar -Xmx4G /home/USeq_7.9.4/Apps/DefinedRegionScanSeqs -s /ERP000016/results/1/ -t /ERP000016/pointData/1dpf/PointData/ -c /ERP000016/pointData/malNoHeadTestis/PointData/ -p 0 -u ../ucsc/zv9EnsemblTranscripts.PSC.orig.ucsc</P><P ALIGN=""LEFT""></P><P ALIGN=""LEFT""></P>',7141,6303,49,'u0785353','2011-10-25 00:00:00')
,(7273,'MitoSeqs',NULL,6670,6300,8,'UARP','2011-10-25 00:00:00')
,(7274,'X10_X12 Liver Gla1 + vs - Glu',NULL,7190,3806,50,'u0028003','2011-10-26 00:00:00')
,(7275,'Useq_test',NULL,6735,6300,7,'oler','2011-10-26 00:00:00')
,(7276,'Repeats',NULL,7251,6303,7,'Magda','2011-10-27 00:00:00')
,(7277,'CG_Islands',NULL,7251,6303,7,'Magda','2011-10-27 00:00:00')
,(7278,'piRNAs',NULL,7251,6303,7,'Magda','2011-10-27 00:00:00')
,(7279,'TotalRNA',NULL,7068,6303,7,'Magda','2011-10-27 00:00:00')
,(7280,'smallRNA',NULL,7068,6303,7,'Magda','2011-10-27 00:00:00')
,(7281,'predictedpiRNA',NULL,7068,6303,7,'Magda','2011-10-27 00:00:00')
,(7282,'Cairns Lab data',NULL,7173,3806,17,'elizabeth.sloan','2011-10-28 00:00:00')
,(7288,'methylationinexons','',7083,6303,7,'Magda','2011-11-08 00:00:00')
,(7289,'RegionMaker',NULL,7083,6303,7,'Magda','2011-11-08 00:00:00')
,(7291,'8374R',NULL,6756,1204,25,'u0028003','2011-11-09 00:00:00')
,(7292,'BAM',NULL,7291,1204,25,'u0028003','2011-11-09 00:00:00')
,(7297,'H3K4me1 vs Input',NULL,7257,6303,49,'u0785353','2011-11-09 00:00:00')
,(7298,'H3K4me3 vs Input',NULL,7257,6303,49,'u0785353','2011-11-09 00:00:00')
,(7299,'relative read coverage','',7297,6303,49,'u0785353','2011-11-09 00:00:00')
,(7300,'relative read coverage',NULL,7298,6303,49,'u0785353','2011-11-09 00:00:00')
,(7301,'MultipleReplicaScanSeqs',NULL,7298,6303,49,'u0785353','2011-11-09 00:00:00')
,(7302,'MultipleReplicaScanSeqs',NULL,7297,6303,49,'u0785353','2011-11-09 00:00:00')
,(7303,'Enriched regions',NULL,7302,6303,49,'u0785353','2011-11-09 00:00:00')
,(7304,'Reduced regions',NULL,7302,6303,49,'u0785353','2011-11-09 00:00:00')
,(7305,'Window summary tracks',NULL,7302,6303,49,'u0785353','2011-11-09 00:00:00')
,(7306,'Enriched regions',NULL,7301,6303,49,'u0785353','2011-11-09 00:00:00')
,(7308,'BisSeq',NULL,7083,6303,7,'Magda','2011-11-21 00:00:00')
,(7309,'CG_context_NBP',NULL,7083,6303,7,'Magda','2011-11-21 00:00:00');
ALTER TABLE dbo.gpAnnotationGrouping CHECK CONSTRAINT all;
SET IDENTITY_INSERT dbo.gpAnnotationGrouping OFF;



--
-- Table structure for table dbo.gpAnnotationProperty --

IF  EXISTS (SELECT * FROM sys.objects WHERE object_id = OBJECT_ID(N'dbo.gpAnnotationProperty') AND type in (N'U')) DROP TABLE dbo.gpAnnotationProperty;


CREATE TABLE dbo.gpAnnotationProperty (
  idAnnotationProperty int  NOT NULL identity,
  name varchar(200) NOT NULL,
  value varchar(200)  NULL,
  idProperty int  NOT NULL,
  idAnnotation int  NOT NULL,
CONSTRAINT [PK_AnnotationProperty] PRIMARY KEY CLUSTERED 
(
	[idAnnotationProperty] ASC
),
  
  
  CONSTRAINT FK_AnnotationProperty_Annotation FOREIGN KEY (idAnnotation) REFERENCES dbo.gpAnnotation (idAnnotation),
  CONSTRAINT FK_AnnotationProperty_Property FOREIGN KEY (idProperty) REFERENCES dbo.gpProperty (idProperty)
)  ;


--
-- Dumping data for table dbo.gpAnnotationProperty --


ALTER TABLE dbo.gpAnnotationProperty NOCHECK CONSTRAINT all;
SET IDENTITY_INSERT dbo.gpAnnotationProperty ON;
INSERT INTO dbo.gpAnnotationProperty
(idAnnotationProperty,
name,
value,
idProperty,
idAnnotation
) VALUES (1,'Analysis Type','Dynamic/Differential',1,1300),(2,'Analysis Type','Dynamic/Differential',1,1424),(3,'Analysis Type','Dynamic/Differential',1,1425),(4,'Analysis Type','Dynamic/Differential',1,1426),(5,'Analysis Type','Dynamic/Differential',1,1427),(6,'Analysis Type','Dynamic/Differential',1,1428),(7,'Analysis Type','Dynamic/Differential',1,1429),(8,'Analysis Type','Dynamic/Differential',1,1430),(9,'Analysis Type','Dynamic/Differential',1,1431),(10,'Analysis Type','Dynamic/Differential',1,1432),(11,'Analysis Type','Dynamic/Differential',1,1433),(12,'Analysis Type','Dynamic/Differential',1,1815),(13,'Analysis Type','Dynamic/Differential',1,1818),(14,'Analysis Type','Dynamic/Differential',1,1866),(15,'Analysis Type','Dynamic/Differential',1,1867),(16,'Analysis Type','Dynamic/Differential',1,1868),(17,'Analysis Type','Dynamic/Differential',1,2150),(18,'Analysis Type','Dynamic/Differential',1,2155),(19,'Analysis Type','Dynamic/Differential',1,2597),(20,'Analysis Type','Dynamic/Differential',1,2598),(21,'Analysis Type','Dynamic/Differential',1,2599),(22,'Analysis Type','Dynamic/Differential',1,2600),(23,'Analysis Type','Dynamic/Differential',1,2617),(24,'Analysis Type','Dynamic/Differential',1,2618),(25,'Analysis Type','Dynamic/Differential',1,3115),(26,'Analysis Type','Dynamic/Differential',1,3116),(27,'Analysis Type','Dynamic/Differential',1,3117),(28,'Analysis Type','Dynamic/Differential',1,3118),(29,'Analysis Type','Dynamic/Differential',1,3119),(30,'Analysis Type','Dynamic/Differential',1,3120),(31,'Analysis Type','Dynamic/Differential',1,3121),(32,'Analysis Type','Dynamic/Differential',1,3122),(33,'Analysis Type','Dynamic/Differential',1,3123),(34,'Analysis Type','Dynamic/Differential',1,3124),(35,'Analysis Type','Dynamic/Differential',1,3125),(36,'Analysis Type','Dynamic/Differential',1,3126),(37,'Analysis Type','Dynamic/Differential',1,3127),(38,'Analysis Type','Dynamic/Differential',1,3128),(39,'Analysis Type','Dynamic/Differential',1,3129),(40,'Analysis Type','Dynamic/Differential',1,3130),(41,'Analysis Type','Dynamic/Differential',1,3131),(42,'Analysis Type','Dynamic/Differential',1,3132),(43,'Analysis Type','Dynamic/Differential',1,3133),(44,'Analysis Type','Dynamic/Differential',1,3134),(45,'Analysis Type','Dynamic/Differential',1,3135),(46,'Analysis Type','Dynamic/Differential',1,3136),(47,'Analysis Type','Dynamic/Differential',1,3137),(48,'Analysis Type','Dynamic/Differential',1,3138),(49,'Analysis Type','Dynamic/Differential',1,3139),(50,'Analysis Type','Dynamic/Differential',1,3140),(51,'Analysis Type','Dynamic/Differential',1,3141),(52,'Analysis Type','Dynamic/Differential',1,3142),(53,'Analysis Type','Dynamic/Differential',1,3143),(54,'Analysis Type','Dynamic/Differential',1,3144),(55,'Analysis Type','Dynamic/Differential',1,3145),(56,'Analysis Type','Dynamic/Differential',1,3146),(57,'Analysis Type','Dynamic/Differential',1,3147),(58,'Analysis Type','Dynamic/Differential',1,3148),(59,'Analysis Type','Dynamic/Differential',1,3149),(60,'Analysis Type','Dynamic/Differential',1,3150),(61,'Analysis Type','Dynamic/Differential',1,3151),(62,'Analysis Type','Dynamic/Differential',1,3152),(63,'Analysis Type','Dynamic/Differential',1,3153),(64,'Analysis Type','Dynamic/Differential',1,3154),(65,'Analysis Type','Dynamic/Differential',1,3155),(66,'Analysis Type','Dynamic/Differential',1,3156),(67,'Analysis Type','Dynamic/Differential',1,3157),(68,'Analysis Type','Dynamic/Differential',1,3158),(69,'Analysis Type','Dynamic/Differential',1,3159),(70,'Analysis Type','Dynamic/Differential',1,3160),(71,'Analysis Type','Dynamic/Differential',1,3161),(72,'Analysis Type','Dynamic/Differential',1,3162),(73,'Analysis Type','Dynamic/Differential',1,3163),(74,'Analysis Type','Dynamic/Differential',1,3164),(75,'Analysis Type','Dynamic/Differential',1,3165),(76,'Analysis Type','Dynamic/Differential',1,3166),(77,'Analysis Type','Dynamic/Differential',1,3167),(78,'Analysis Type','Dynamic/Differential',1,3168),(79,'Analysis Type','Dynamic/Differential',1,3169),(80,'Analysis Type','Dynamic/Differential',1,3170),(81,'Analysis Type','Dynamic/Differential',1,3171),(82,'Analysis Type','Dynamic/Differential',1,3172),(83,'Analysis Type','Dynamic/Differential',1,3173),(84,'Analysis Type','Dynamic/Differential',1,3174),(85,'Analysis Type','Dynamic/Differential',1,3175),(86,'Analysis Type','Dynamic/Differential',1,3176),(87,'Analysis Type','Dynamic/Differential',1,3177),(88,'Analysis Type','Dynamic/Differential',1,3178),(89,'Analysis Type','Dynamic/Differential',1,3179),(90,'Analysis Type','Dynamic/Differential',1,3180),(91,'Analysis Type','Dynamic/Differential',1,3181),(92,'Analysis Type','Dynamic/Differential',1,3182),(93,'Analysis Type','Dynamic/Differential',1,3183),(94,'Analysis Type','Dynamic/Differential',1,3184),(95,'Analysis Type','Dynamic/Differential',1,3185),(96,'Analysis Type','Dynamic/Differential',1,3186),(97,'Analysis Type','Dynamic/Differential',1,3187),(98,'Analysis Type','Dynamic/Differential',1,3188),(99,'Analysis Type','Dynamic/Differential',1,3189),(100,'Analysis Type','Dynamic/Differential',1,3190),(101,'Analysis Type','Dynamic/Differential',1,3191),(102,'Analysis Type','Dynamic/Differential',1,3192),(103,'Analysis Type','Dynamic/Differential',1,3193),(104,'Analysis Type','Dynamic/Differential',1,3194),(105,'Analysis Type','Dynamic/Differential',1,3195),(106,'Analysis Type','Dynamic/Differential',1,3196),(107,'Analysis Type','Dynamic/Differential',1,3197),(108,'Analysis Type','Dynamic/Differential',1,3198),(109,'Analysis Type','Dynamic/Differential',1,3199),(110,'Analysis Type','Dynamic/Differential',1,3200),(111,'Analysis Type','Dynamic/Differential',1,3201),(112,'Analysis Type','Dynamic/Differential',1,3202),(113,'Analysis Type','Dynamic/Differential',1,3203),(114,'Analysis Type','Dynamic/Differential',1,3204),(115,'Analysis Type','Dynamic/Differential',1,3205),(116,'Analysis Type','Dynamic/Differential',1,3206),(117,'Analysis Type','Dynamic/Differential',1,3207),(118,'Analysis Type','Dynamic/Differential',1,3208),(119,'Analysis Type','Dynamic/Differential',1,3209),(120,'Analysis Type','Dynamic/Differential',1,3210),(121,'Analysis Type','Dynamic/Differential',1,3211),(122,'Analysis Type','Dynamic/Differential',1,3212),(123,'Analysis Type','Dynamic/Differential',1,3213),(124,'Analysis Type','Dynamic/Differential',1,3214),(125,'Analysis Type','Dynamic/Differential',1,3215),(126,'Analysis Type','Dynamic/Differential',1,3216),(127,'Analysis Type','Dynamic/Differential',1,3217),(128,'Analysis Type','Dynamic/Differential',1,3218),(129,'Analysis Type','Dynamic/Differential',1,3219),(130,'Analysis Type','Dynamic/Differential',1,3220),(131,'Analysis Type','Dynamic/Differential',1,3221),(132,'Analysis Type','Dynamic/Differential',1,3222),(133,'Analysis Type','Dynamic/Differential',1,3223),(134,'Analysis Type','Dynamic/Differential',1,3224),(135,'Analysis Type','Dynamic/Differential',1,3225),(136,'Analysis Type','Dynamic/Differential',1,3226),(137,'Analysis Type','Dynamic/Differential',1,3227),(138,'Analysis Type','Dynamic/Differential',1,3228),(139,'Analysis Type','Dynamic/Differential',1,3229),(140,'Analysis Type','Dynamic/Differential',1,3230),(141,'Analysis Type','Dynamic/Differential',1,3231),(142,'Analysis Type','Dynamic/Differential',1,3232),(143,'Analysis Type','Dynamic/Differential',1,3233),(144,'Analysis Type','Dynamic/Differential',1,3234),(145,'Analysis Type','Dynamic/Differential',1,3235),(146,'Analysis Type','Dynamic/Differential',1,3236),(147,'Analysis Type','Dynamic/Differential',1,3237),(148,'Analysis Type','Dynamic/Differential',1,3263),(149,'Analysis Type','Dynamic/Differential',1,3268),(150,'Analysis Type','Static',1,823),(151,'Analysis Type','Static',1,881),(152,'Analysis Type','Static',1,882),(153,'Analysis Type','Static',1,883),(154,'Analysis Type','Static',1,1262),(155,'Analysis Type','Static',1,1285),(156,'Analysis Type','Static',1,1286),(157,'Analysis Type','Static',1,1287),(158,'Analysis Type','Static',1,1288),(159,'Analysis Type','Static',1,1294),(160,'Analysis Type','Static',1,1295),(161,'Analysis Type','Static',1,1296),(162,'Analysis Type','Static',1,1297),(163,'Analysis Type','Static',1,1298),(164,'Analysis Type','Static',1,1299),(165,'Analysis Type','Static',1,1301),(166,'Analysis Type','Static',1,1329),(167,'Analysis Type','Static',1,1330),(168,'Analysis Type','Static',1,1331),(169,'Analysis Type','Static',1,1332),(170,'Analysis Type','Static',1,1333),(171,'Analysis Type','Static',1,1334),(172,'Analysis Type','Static',1,1335),(173,'Analysis Type','Static',1,1336),(174,'Analysis Type','Static',1,1337),(175,'Analysis Type','Static',1,1338),(176,'Analysis Type','Static',1,1339),(177,'Analysis Type','Static',1,1340),(178,'Analysis Type','Static',1,1354),(179,'Analysis Type','Static',1,1355),(180,'Analysis Type','Static',1,1356),(181,'Analysis Type','Static',1,1357),(182,'Analysis Type','Static',1,1358),(183,'Analysis Type','Static',1,1359),(184,'Analysis Type','Static',1,1360),(185,'Analysis Type','Static',1,1361),(186,'Analysis Type','Static',1,1362),(187,'Analysis Type','Static',1,1363),(188,'Analysis Type','Static',1,1364),(189,'Analysis Type','Static',1,1365),(190,'Analysis Type','Static',1,1366),(191,'Analysis Type','Static',1,1367),(192,'Analysis Type','Static',1,1368),(193,'Analysis Type','Static',1,1369),(194,'Analysis Type','Static',1,1370),(195,'Analysis Type','Static',1,1371),(196,'Analysis Type','Static',1,1372),(197,'Analysis Type','Static',1,1373),(198,'Analysis Type','Static',1,1374),(199,'Analysis Type','Static',1,1375),(200,'Analysis Type','Static',1,1376),(201,'Analysis Type','Static',1,1377),(202,'Analysis Type','Static',1,1378),(203,'Analysis Type','Static',1,1379),(204,'Analysis Type','Static',1,1380),(205,'Analysis Type','Static',1,1381),(206,'Analysis Type','Static',1,1382),(207,'Analysis Type','Static',1,1383),(208,'Analysis Type','Static',1,1384),(209,'Analysis Type','Static',1,1385),(210,'Analysis Type','Static',1,1386),(211,'Analysis Type','Static',1,1387),(212,'Analysis Type','Static',1,1388),(213,'Analysis Type','Static',1,1389),(214,'Analysis Type','Static',1,1390),(215,'Analysis Type','Static',1,1391),(216,'Analysis Type','Static',1,1392),(217,'Analysis Type','Static',1,1393),(218,'Analysis Type','Static',1,1394),(219,'Analysis Type','Static',1,1395),(220,'Analysis Type','Static',1,1396),(221,'Analysis Type','Static',1,1397),(222,'Analysis Type','Static',1,1398),(223,'Analysis Type','Static',1,1399),(224,'Analysis Type','Static',1,1400),(225,'Analysis Type','Static',1,1401),(226,'Analysis Type','Static',1,1402),(227,'Analysis Type','Static',1,1403),(228,'Analysis Type','Static',1,1404),(229,'Analysis Type','Static',1,1405),(230,'Analysis Type','Static',1,1406),(231,'Analysis Type','Static',1,1407),(232,'Analysis Type','Static',1,1408),(233,'Analysis Type','Static',1,1409),(234,'Analysis Type','Static',1,1410),(235,'Analysis Type','Static',1,1411),(236,'Analysis Type','Static',1,1412),(237,'Analysis Type','Static',1,1413),(238,'Analysis Type','Static',1,1414),(239,'Analysis Type','Static',1,1415),(240,'Analysis Type','Static',1,1416),(241,'Analysis Type','Static',1,1434),(242,'Analysis Type','Static',1,1435),(243,'Analysis Type','Static',1,1449),(244,'Analysis Type','Static',1,1450),(245,'Analysis Type','Static',1,1451),(246,'Analysis Type','Static',1,1452),(247,'Analysis Type','Static',1,1453),(248,'Analysis Type','Static',1,1454),(249,'Analysis Type','Static',1,1455),(250,'Analysis Type','Static',1,1456),(251,'Analysis Type','Static',1,1457),(252,'Analysis Type','Static',1,1458),(253,'Analysis Type','Static',1,1459),(254,'Analysis Type','Static',1,1460),(255,'Analysis Type','Static',1,1461),(256,'Analysis Type','Static',1,1462),(257,'Analysis Type','Static',1,1463),(258,'Analysis Type','Static',1,1464),(259,'Analysis Type','Static',1,1465),(260,'Analysis Type','Static',1,1466),(261,'Analysis Type','Static',1,1467),(262,'Analysis Type','Static',1,1468),(263,'Analysis Type','Static',1,1469),(264,'Analysis Type','Static',1,1470),(265,'Analysis Type','Static',1,1471),(266,'Analysis Type','Static',1,1472),(267,'Analysis Type','Static',1,1473),(268,'Analysis Type','Static',1,1474),(269,'Analysis Type','Static',1,1475),(270,'Analysis Type','Static',1,1476),(271,'Analysis Type','Static',1,1477),(272,'Analysis Type','Static',1,1478),(273,'Analysis Type','Static',1,1479),(274,'Analysis Type','Static',1,1480),(275,'Analysis Type','Static',1,1481),(276,'Analysis Type','Static',1,1482),(277,'Analysis Type','Static',1,1483),(278,'Analysis Type','Static',1,1484),(279,'Analysis Type','Static',1,1485),(280,'Analysis Type','Static',1,1486),(281,'Analysis Type','Static',1,1487),(282,'Analysis Type','Static',1,1488),(283,'Analysis Type','Static',1,1489),(284,'Analysis Type','Static',1,1490),(285,'Analysis Type','Static',1,1491),(286,'Analysis Type','Static',1,1492),(287,'Analysis Type','Static',1,1493),(288,'Analysis Type','Static',1,1494),(289,'Analysis Type','Static',1,1495),(290,'Analysis Type','Static',1,1496),(291,'Analysis Type','Static',1,1497),(292,'Analysis Type','Static',1,1498),(293,'Analysis Type','Static',1,1499),(294,'Analysis Type','Static',1,1500),(295,'Analysis Type','Static',1,1501),(296,'Analysis Type','Static',1,1502),(297,'Analysis Type','Static',1,1503),(298,'Analysis Type','Static',1,1504),(299,'Analysis Type','Static',1,1505),(300,'Analysis Type','Static',1,1506),(301,'Analysis Type','Static',1,1507),(302,'Analysis Type','Static',1,1508),(303,'Analysis Type','Static',1,1509),(304,'Analysis Type','Static',1,1510),(305,'Analysis Type','Static',1,1511),(306,'Analysis Type','Static',1,1512),(307,'Analysis Type','Static',1,1513),(308,'Analysis Type','Static',1,1514),(309,'Analysis Type','Static',1,1515),(310,'Analysis Type','Static',1,1516),(311,'Analysis Type','Static',1,1517),(312,'Analysis Type','Static',1,1518),(313,'Analysis Type','Static',1,1519),(314,'Analysis Type','Static',1,1521),(315,'Analysis Type','Static',1,1522),(316,'Analysis Type','Static',1,1523),(317,'Analysis Type','Static',1,1524),(318,'Analysis Type','Static',1,1525),(319,'Analysis Type','Static',1,1526),(320,'Analysis Type','Static',1,1527),(321,'Analysis Type','Static',1,1528),(322,'Analysis Type','Static',1,1529),(323,'Analysis Type','Static',1,1530),(324,'Analysis Type','Static',1,1531),(325,'Analysis Type','Static',1,1532),(326,'Analysis Type','Static',1,1533),(327,'Analysis Type','Static',1,1534),(328,'Analysis Type','Static',1,1535),(329,'Analysis Type','Static',1,1536),(330,'Analysis Type','Static',1,1537),(331,'Analysis Type','Static',1,1538),(332,'Analysis Type','Static',1,1539),(333,'Analysis Type','Static',1,1540),(334,'Analysis Type','Static',1,1541),(335,'Analysis Type','Static',1,1542),(336,'Analysis Type','Static',1,1543),(337,'Analysis Type','Static',1,1544),(338,'Analysis Type','Static',1,1545),(339,'Analysis Type','Static',1,1546),(340,'Analysis Type','Static',1,1547),(341,'Analysis Type','Static',1,1548),(342,'Analysis Type','Static',1,1549),(343,'Analysis Type','Static',1,1550),(344,'Analysis Type','Static',1,1551),(345,'Analysis Type','Static',1,1552),(346,'Analysis Type','Static',1,1553),(347,'Analysis Type','Static',1,1554),(348,'Analysis Type','Static',1,1555),(349,'Analysis Type','Static',1,1556),(350,'Analysis Type','Static',1,1557),(351,'Analysis Type','Static',1,1558),(352,'Analysis Type','Static',1,1559),(353,'Analysis Type','Static',1,1560),(354,'Analysis Type','Static',1,1561),(355,'Analysis Type','Static',1,1562),(356,'Analysis Type','Static',1,1563),(357,'Analysis Type','Static',1,1564),(358,'Analysis Type','Static',1,1565),(359,'Analysis Type','Static',1,1566),(360,'Analysis Type','Static',1,1567),(361,'Analysis Type','Static',1,1568),(362,'Analysis Type','Static',1,1569),(363,'Analysis Type','Static',1,1570),(364,'Analysis Type','Static',1,1571),(365,'Analysis Type','Static',1,1572),(366,'Analysis Type','Static',1,1573),(367,'Analysis Type','Static',1,1574),(368,'Analysis Type','Static',1,1575),(369,'Analysis Type','Static',1,1576),(370,'Analysis Type','Static',1,1577),(371,'Analysis Type','Static',1,1578),(372,'Analysis Type','Static',1,1579),(373,'Analysis Type','Static',1,1580),(374,'Analysis Type','Static',1,1581),(375,'Analysis Type','Static',1,1582),(376,'Analysis Type','Static',1,1583),(377,'Analysis Type','Static',1,1584),(378,'Analysis Type','Static',1,1585),(379,'Analysis Type','Static',1,1586),(380,'Analysis Type','Static',1,1587),(381,'Analysis Type','Static',1,1588),(382,'Analysis Type','Static',1,1589),(383,'Analysis Type','Static',1,1590),(384,'Analysis Type','Static',1,1591),(385,'Analysis Type','Static',1,1592),(386,'Analysis Type','Static',1,1593),(387,'Analysis Type','Static',1,1594),(388,'Analysis Type','Static',1,1595),(389,'Analysis Type','Static',1,1598),(390,'Analysis Type','Static',1,1599),(391,'Analysis Type','Static',1,1600),(392,'Analysis Type','Static',1,1601),(393,'Analysis Type','Static',1,1602),(394,'Analysis Type','Static',1,1603),(395,'Analysis Type','Static',1,1604),(396,'Analysis Type','Static',1,1605),(397,'Analysis Type','Static',1,1606),(398,'Analysis Type','Static',1,1607),(399,'Analysis Type','Static',1,1608),(400,'Analysis Type','Static',1,1609),(401,'Analysis Type','Static',1,1610),(402,'Analysis Type','Static',1,1611),(403,'Analysis Type','Static',1,1612),(404,'Analysis Type','Static',1,1613),(405,'Analysis Type','Static',1,1614),(406,'Analysis Type','Static',1,1615),(407,'Analysis Type','Static',1,1616),(408,'Analysis Type','Static',1,1617),(409,'Analysis Type','Static',1,1618),(410,'Analysis Type','Static',1,1619),(411,'Analysis Type','Static',1,1620),(412,'Analysis Type','Static',1,1621),(413,'Analysis Type','Static',1,1622),(414,'Analysis Type','Static',1,1623),(415,'Analysis Type','Static',1,1624),(416,'Analysis Type','Static',1,1625),(417,'Analysis Type','Static',1,1626),(418,'Analysis Type','Static',1,1627),(419,'Analysis Type','Static',1,1628),(420,'Analysis Type','Static',1,1629),(421,'Analysis Type','Static',1,1630),(422,'Analysis Type','Static',1,1631),(423,'Analysis Type','Static',1,1632),(424,'Analysis Type','Static',1,1633),(425,'Analysis Type','Static',1,1634),(426,'Analysis Type','Static',1,1635),(427,'Analysis Type','Static',1,1636),(428,'Analysis Type','Static',1,1637),(429,'Analysis Type','Static',1,1638),(430,'Analysis Type','Static',1,1639),(431,'Analysis Type','Static',1,1640),(432,'Analysis Type','Static',1,1641),(433,'Analysis Type','Static',1,1642),(434,'Analysis Type','Static',1,1643),(435,'Analysis Type','Static',1,1644),(436,'Analysis Type','Static',1,1645),(437,'Analysis Type','Static',1,1646),(438,'Analysis Type','Static',1,1647),(439,'Analysis Type','Static',1,1648),(440,'Analysis Type','Static',1,1649),(441,'Analysis Type','Static',1,1650),(442,'Analysis Type','Static',1,1651),(443,'Analysis Type','Static',1,1652),(444,'Analysis Type','Static',1,1653),(445,'Analysis Type','Static',1,1654),(446,'Analysis Type','Static',1,1655),(447,'Analysis Type','Static',1,1656),(448,'Analysis Type','Static',1,1657),(449,'Analysis Type','Static',1,1658),(450,'Analysis Type','Static',1,1659),(451,'Analysis Type','Static',1,1660),(452,'Analysis Type','Static',1,1661),(453,'Analysis Type','Static',1,1662),(454,'Analysis Type','Static',1,1663),(455,'Analysis Type','Static',1,1664),(456,'Analysis Type','Static',1,1665),(457,'Analysis Type','Static',1,1666),(458,'Analysis Type','Static',1,1667),(459,'Analysis Type','Static',1,1668),(460,'Analysis Type','Static',1,1669),(461,'Analysis Type','Static',1,1670),(462,'Analysis Type','Static',1,1671),(463,'Analysis Type','Static',1,1672),(464,'Analysis Type','Static',1,1673),(465,'Analysis Type','Static',1,1674),(466,'Analysis Type','Static',1,1675),(467,'Analysis Type','Static',1,1676),(468,'Analysis Type','Static',1,1677),(469,'Analysis Type','Static',1,1678),(470,'Analysis Type','Static',1,1679),(471,'Analysis Type','Static',1,1680),(472,'Analysis Type','Static',1,1681),(473,'Analysis Type','Static',1,1682),(474,'Analysis Type','Static',1,1683),(475,'Analysis Type','Static',1,1684),(476,'Analysis Type','Static',1,1685),(477,'Analysis Type','Static',1,1686),(478,'Analysis Type','Static',1,1687),(479,'Analysis Type','Static',1,1688),(480,'Analysis Type','Static',1,1689),(481,'Analysis Type','Static',1,1690),(482,'Analysis Type','Static',1,1691),(483,'Analysis Type','Static',1,1692),(484,'Analysis Type','Static',1,1693),(485,'Analysis Type','Static',1,1694),(486,'Analysis Type','Static',1,1695),(487,'Analysis Type','Static',1,1696),(488,'Analysis Type','Static',1,1697),(489,'Analysis Type','Static',1,1698),(490,'Analysis Type','Static',1,1699),(491,'Analysis Type','Static',1,1700),(492,'Analysis Type','Static',1,1701),(493,'Analysis Type','Static',1,1702),(494,'Analysis Type','Static',1,1703),(495,'Analysis Type','Static',1,1704),(496,'Analysis Type','Static',1,1705),(497,'Analysis Type','Static',1,1706),(498,'Analysis Type','Static',1,1707),(499,'Analysis Type','Static',1,1708),(500,'Analysis Type','Static',1,1709),(501,'Analysis Type','Static',1,1710),(502,'Analysis Type','Static',1,1711),(503,'Analysis Type','Static',1,1712),(504,'Analysis Type','Static',1,1713),(505,'Analysis Type','Static',1,1714),(506,'Analysis Type','Static',1,1715),(507,'Analysis Type','Static',1,1716),(508,'Analysis Type','Static',1,1717),(509,'Analysis Type','Static',1,1718),(510,'Analysis Type','Static',1,1719),(511,'Analysis Type','Static',1,1720),(512,'Analysis Type','Static',1,1721),(513,'Analysis Type','Static',1,1722),(514,'Analysis Type','Static',1,1723),(515,'Analysis Type','Static',1,1724),(516,'Analysis Type','Static',1,1725),(517,'Analysis Type','Static',1,1726),(518,'Analysis Type','Static',1,1727),(519,'Analysis Type','Static',1,1728),(520,'Analysis Type','Static',1,1729),(521,'Analysis Type','Static',1,1730),(522,'Analysis Type','Static',1,1731),(523,'Analysis Type','Static',1,1732),(524,'Analysis Type','Static',1,1733),(525,'Analysis Type','Static',1,1734),(526,'Analysis Type','Static',1,1735),(527,'Analysis Type','Static',1,1736),(528,'Analysis Type','Static',1,1737),(529,'Analysis Type','Static',1,1738),(530,'Analysis Type','Static',1,1739),(531,'Analysis Type','Static',1,1740),(532,'Analysis Type','Static',1,1741),(533,'Analysis Type','Static',1,1742),(534,'Analysis Type','Static',1,1743),(535,'Analysis Type','Static',1,1744),(536,'Analysis Type','Static',1,1745),(537,'Analysis Type','Static',1,1746),(538,'Analysis Type','Static',1,1747),(539,'Analysis Type','Static',1,1748),(540,'Analysis Type','Static',1,1749),(541,'Analysis Type','Static',1,1750),(542,'Analysis Type','Static',1,1751),(543,'Analysis Type','Static',1,1752),(544,'Analysis Type','Static',1,1753),(545,'Analysis Type','Static',1,1754),(546,'Analysis Type','Static',1,1755),(547,'Analysis Type','Static',1,1756),(548,'Analysis Type','Static',1,1758),(549,'Analysis Type','Static',1,1759),(550,'Analysis Type','Static',1,1760),(551,'Analysis Type','Static',1,1761),(552,'Analysis Type','Static',1,1762),(553,'Analysis Type','Static',1,1763),(554,'Analysis Type','Static',1,1764),(555,'Analysis Type','Static',1,1765),(556,'Analysis Type','Static',1,1766),(557,'Analysis Type','Static',1,1767),(558,'Analysis Type','Static',1,1768),(559,'Analysis Type','Static',1,1769),(560,'Analysis Type','Static',1,1770),(561,'Analysis Type','Static',1,1771),(562,'Analysis Type','Static',1,1772),(563,'Analysis Type','Static',1,1773),(564,'Analysis Type','Static',1,1774),(565,'Analysis Type','Static',1,1775),(566,'Analysis Type','Static',1,1776),(567,'Analysis Type','Static',1,1777),(568,'Analysis Type','Static',1,1778),(569,'Analysis Type','Static',1,1779),(570,'Analysis Type','Static',1,1780),(571,'Analysis Type','Static',1,1781),(572,'Analysis Type','Static',1,1782),(573,'Analysis Type','Static',1,1783),(574,'Analysis Type','Static',1,1784),(575,'Analysis Type','Static',1,1785),(576,'Analysis Type','Static',1,1786),(577,'Analysis Type','Static',1,1787),(578,'Analysis Type','Static',1,1792),(579,'Analysis Type','Static',1,1793),(580,'Analysis Type','Static',1,1799);

INSERT INTO dbo.gpAnnotationProperty
(idAnnotationProperty,
name,
value,
idProperty,
idAnnotation
) VALUES
(581,'Analysis Type','Static',1,1800),(582,'Analysis Type','Static',1,1801),(583,'Analysis Type','Static',1,1802),(584,'Analysis Type','Static',1,1803),(585,'Analysis Type','Static',1,1805),(586,'Analysis Type','Static',1,1806),(587,'Analysis Type','Static',1,1807),(588,'Analysis Type','Static',1,1808),(589,'Analysis Type','Static',1,1809),(590,'Analysis Type','Static',1,1812),(591,'Analysis Type','Static',1,1813),(592,'Analysis Type','Static',1,1814),(593,'Analysis Type','Static',1,1816),(594,'Analysis Type','Static',1,1817),(595,'Analysis Type','Static',1,1819),(596,'Analysis Type','Static',1,1820),(597,'Analysis Type','Static',1,1821),(598,'Analysis Type','Static',1,1822),(599,'Analysis Type','Static',1,1823),(600,'Analysis Type','Static',1,1824),(601,'Analysis Type','Static',1,1825),(602,'Analysis Type','Static',1,1826),(603,'Analysis Type','Static',1,1827),(604,'Analysis Type','Static',1,1828),(605,'Analysis Type','Static',1,1829),(606,'Analysis Type','Static',1,1830),(607,'Analysis Type','Static',1,1831),(608,'Analysis Type','Static',1,1832),(609,'Analysis Type','Static',1,1833),(610,'Analysis Type','Static',1,1834),(611,'Analysis Type','Static',1,1835),(612,'Analysis Type','Static',1,1837),(613,'Analysis Type','Static',1,1838),(614,'Analysis Type','Static',1,1839),(615,'Analysis Type','Static',1,1840),(616,'Analysis Type','Static',1,1841),(617,'Analysis Type','Static',1,1842),(618,'Analysis Type','Static',1,1850),(619,'Analysis Type','Static',1,1861),(620,'Analysis Type','Static',1,1862),(621,'Analysis Type','Static',1,1863),(622,'Analysis Type','Static',1,1864),(623,'Analysis Type','Static',1,1865),(624,'Analysis Type','Static',1,1869),(625,'Analysis Type','Static',1,1870),(626,'Analysis Type','Static',1,1871),(627,'Analysis Type','Static',1,1872),(628,'Analysis Type','Static',1,1873),(629,'Analysis Type','Static',1,1874),(630,'Analysis Type','Static',1,1875),(631,'Analysis Type','Static',1,1876),(632,'Analysis Type','Static',1,1877),(633,'Analysis Type','Static',1,1878),(634,'Analysis Type','Static',1,1879),(635,'Analysis Type','Static',1,1880),(636,'Analysis Type','Static',1,1881),(637,'Analysis Type','Static',1,1882),(638,'Analysis Type','Static',1,1883),(639,'Analysis Type','Static',1,1884),(640,'Analysis Type','Static',1,1885),(641,'Analysis Type','Static',1,1886),(642,'Analysis Type','Static',1,1887),(643,'Analysis Type','Static',1,1888),(644,'Analysis Type','Static',1,1889),(645,'Analysis Type','Static',1,1890),(646,'Analysis Type','Static',1,1891),(647,'Analysis Type','Static',1,1893),(648,'Analysis Type','Static',1,1894),(649,'Analysis Type','Static',1,1895),(650,'Analysis Type','Static',1,1896),(651,'Analysis Type','Static',1,1897),(652,'Analysis Type','Static',1,1898),(653,'Analysis Type','Static',1,1899),(654,'Analysis Type','Static',1,1900),(655,'Analysis Type','Static',1,1901),(656,'Analysis Type','Static',1,1902),(657,'Analysis Type','Static',1,1903),(658,'Analysis Type','Static',1,1904),(659,'Analysis Type','Static',1,1905),(660,'Analysis Type','Static',1,1921),(661,'Analysis Type','Static',1,1922),(662,'Analysis Type','Static',1,1923),(663,'Analysis Type','Static',1,1924),(664,'Analysis Type','Static',1,1925),(665,'Analysis Type','Static',1,1926),(666,'Analysis Type','Static',1,1927),(667,'Analysis Type','Static',1,1928),(668,'Analysis Type','Static',1,1929),(669,'Analysis Type','Static',1,1930),(670,'Analysis Type','Static',1,1931),(671,'Analysis Type','Static',1,1933),(672,'Analysis Type','Static',1,1934),(673,'Analysis Type','Static',1,1935),(674,'Analysis Type','Static',1,1936),(675,'Analysis Type','Static',1,1937),(676,'Analysis Type','Static',1,1938),(677,'Analysis Type','Static',1,1939),(678,'Analysis Type','Static',1,1940),(679,'Analysis Type','Static',1,1941),(680,'Analysis Type','Static',1,1942),(681,'Analysis Type','Static',1,1943),(682,'Analysis Type','Static',1,1944),(683,'Analysis Type','Static',1,1945),(684,'Analysis Type','Static',1,1946),(685,'Analysis Type','Static',1,1947),(686,'Analysis Type','Static',1,1948),(687,'Analysis Type','Static',1,1949),(688,'Analysis Type','Static',1,1950),(689,'Analysis Type','Static',1,1966),(690,'Analysis Type','Static',1,1967),(691,'Analysis Type','Static',1,1968),(692,'Analysis Type','Static',1,1969),(693,'Analysis Type','Static',1,1970),(694,'Analysis Type','Static',1,1971),(695,'Analysis Type','Static',1,1972),(696,'Analysis Type','Static',1,1973),(697,'Analysis Type','Static',1,1974),(698,'Analysis Type','Static',1,1975),(699,'Analysis Type','Static',1,1976),(700,'Analysis Type','Static',1,1977),(701,'Analysis Type','Static',1,1978),(702,'Analysis Type','Static',1,1979),(703,'Analysis Type','Static',1,1980),(704,'Analysis Type','Static',1,1981),(705,'Analysis Type','Static',1,1982),(706,'Analysis Type','Static',1,1983),(707,'Analysis Type','Static',1,1984),(708,'Analysis Type','Static',1,1985),(709,'Analysis Type','Static',1,1986),(710,'Analysis Type','Static',1,1987),(711,'Analysis Type','Static',1,1988),(712,'Analysis Type','Static',1,1989),(713,'Analysis Type','Static',1,1990),(714,'Analysis Type','Static',1,1991),(715,'Analysis Type','Static',1,1992),(716,'Analysis Type','Static',1,1993),(717,'Analysis Type','Static',1,1994),(718,'Analysis Type','Static',1,1995),(719,'Analysis Type','Static',1,1996),(720,'Analysis Type','Static',1,1997),(721,'Analysis Type','Static',1,1998),(722,'Analysis Type','Static',1,1999),(723,'Analysis Type','Static',1,2000),(724,'Analysis Type','Static',1,2001),(725,'Analysis Type','Static',1,2002),(726,'Analysis Type','Static',1,2003),(727,'Analysis Type','Static',1,2004),(728,'Analysis Type','Static',1,2005),(729,'Analysis Type','Static',1,2006),(730,'Analysis Type','Static',1,2007),(731,'Analysis Type','Static',1,2008),(732,'Analysis Type','Static',1,2009),(733,'Analysis Type','Static',1,2010),(734,'Analysis Type','Static',1,2011),(735,'Analysis Type','Static',1,2012),(736,'Analysis Type','Static',1,2013),(737,'Analysis Type','Static',1,2014),(738,'Analysis Type','Static',1,2015),(739,'Analysis Type','Static',1,2016),(740,'Analysis Type','Static',1,2017),(741,'Analysis Type','Static',1,2018),(742,'Analysis Type','Static',1,2019),(743,'Analysis Type','Static',1,2020),(744,'Analysis Type','Static',1,2021),(745,'Analysis Type','Static',1,2022),(746,'Analysis Type','Static',1,2023),(747,'Analysis Type','Static',1,2024),(748,'Analysis Type','Static',1,2025),(749,'Analysis Type','Static',1,2026),(750,'Analysis Type','Static',1,2027),(751,'Analysis Type','Static',1,2028),(752,'Analysis Type','Static',1,2029),(753,'Analysis Type','Static',1,2030),(754,'Analysis Type','Static',1,2031),(755,'Analysis Type','Static',1,2032),(756,'Analysis Type','Static',1,2033),(757,'Analysis Type','Static',1,2034),(758,'Analysis Type','Static',1,2035),(759,'Analysis Type','Static',1,2036),(760,'Analysis Type','Static',1,2037),(761,'Analysis Type','Static',1,2038),(762,'Analysis Type','Static',1,2039),(763,'Analysis Type','Static',1,2040),(764,'Analysis Type','Static',1,2041),(765,'Analysis Type','Static',1,2042),(766,'Analysis Type','Static',1,2043),(767,'Analysis Type','Static',1,2044),(768,'Analysis Type','Static',1,2045),(769,'Analysis Type','Static',1,2046),(770,'Analysis Type','Static',1,2047),(771,'Analysis Type','Static',1,2048),(772,'Analysis Type','Static',1,2049),(773,'Analysis Type','Static',1,2050),(774,'Analysis Type','Static',1,2051),(775,'Analysis Type','Static',1,2052),(776,'Analysis Type','Static',1,2074),(777,'Analysis Type','Static',1,2075),(778,'Analysis Type','Static',1,2076),(779,'Analysis Type','Static',1,2077),(780,'Analysis Type','Static',1,2078),(781,'Analysis Type','Static',1,2079),(782,'Analysis Type','Static',1,2080),(783,'Analysis Type','Static',1,2081),(784,'Analysis Type','Static',1,2082),(785,'Analysis Type','Static',1,2083),(786,'Analysis Type','Static',1,2084),(787,'Analysis Type','Static',1,2085),(788,'Analysis Type','Static',1,2086),(789,'Analysis Type','Static',1,2087),(790,'Analysis Type','Static',1,2088),(791,'Analysis Type','Static',1,2089),(792,'Analysis Type','Static',1,2090),(793,'Analysis Type','Static',1,2091),(794,'Analysis Type','Static',1,2092),(795,'Analysis Type','Static',1,2093),(796,'Analysis Type','Static',1,2094),(797,'Analysis Type','Static',1,2095),(798,'Analysis Type','Static',1,2096),(799,'Analysis Type','Static',1,2097),(800,'Analysis Type','Static',1,2098),(801,'Analysis Type','Static',1,2099),(802,'Analysis Type','Static',1,2100),(803,'Analysis Type','Static',1,2101),(804,'Analysis Type','Static',1,2102),(805,'Analysis Type','Static',1,2103),(806,'Analysis Type','Static',1,2104),(807,'Analysis Type','Static',1,2105),(808,'Analysis Type','Static',1,2106),(809,'Analysis Type','Static',1,2107),(810,'Analysis Type','Static',1,2108),(811,'Analysis Type','Static',1,2109),(812,'Analysis Type','Static',1,2110),(813,'Analysis Type','Static',1,2111),(814,'Analysis Type','Static',1,2112),(815,'Analysis Type','Static',1,2113),(816,'Analysis Type','Static',1,2114),(817,'Analysis Type','Static',1,2115),(818,'Analysis Type','Static',1,2116),(819,'Analysis Type','Static',1,2117),(820,'Analysis Type','Static',1,2118),(821,'Analysis Type','Static',1,2119),(822,'Analysis Type','Static',1,2120),(823,'Analysis Type','Static',1,2121),(824,'Analysis Type','Static',1,2122),(825,'Analysis Type','Static',1,2123),(826,'Analysis Type','Static',1,2124),(827,'Analysis Type','Static',1,2125),(828,'Analysis Type','Static',1,2126),(829,'Analysis Type','Static',1,2127),(830,'Analysis Type','Static',1,2128),(831,'Analysis Type','Static',1,2129),(832,'Analysis Type','Static',1,2130),(833,'Analysis Type','Static',1,2131),(834,'Analysis Type','Static',1,2132),(835,'Analysis Type','Static',1,2134),(836,'Analysis Type','Static',1,2135),(837,'Analysis Type','Static',1,2136),(838,'Analysis Type','Static',1,2137),(839,'Analysis Type','Static',1,2138),(840,'Analysis Type','Static',1,2139),(841,'Analysis Type','Static',1,2141),(842,'Analysis Type','Static',1,2142),(843,'Analysis Type','Static',1,2143),(844,'Analysis Type','Static',1,2144),(845,'Analysis Type','Static',1,2145),(846,'Analysis Type','Static',1,2146),(847,'Analysis Type','Static',1,2147),(848,'Analysis Type','Static',1,2148),(849,'Analysis Type','Static',1,2149),(850,'Analysis Type','Static',1,2151),(851,'Analysis Type','Static',1,2152),(852,'Analysis Type','Static',1,2153),(853,'Analysis Type','Static',1,2154),(854,'Analysis Type','Static',1,2160),(855,'Analysis Type','Static',1,2161),(856,'Analysis Type','Static',1,2169),(857,'Analysis Type','Static',1,2170),(858,'Analysis Type','Static',1,2171),(859,'Analysis Type','Static',1,2172),(860,'Analysis Type','Static',1,2173),(861,'Analysis Type','Static',1,2174),(862,'Analysis Type','Static',1,2175),(863,'Analysis Type','Static',1,2176),(864,'Analysis Type','Static',1,2177),(865,'Analysis Type','Static',1,2178),(866,'Analysis Type','Static',1,2179),(867,'Analysis Type','Static',1,2180),(868,'Analysis Type','Static',1,2181),(869,'Analysis Type','Static',1,2182),(870,'Analysis Type','Static',1,2183),(871,'Analysis Type','Static',1,2211),(872,'Analysis Type','Static',1,2212),(873,'Analysis Type','Static',1,2213),(874,'Analysis Type','Static',1,2214),(875,'Analysis Type','Static',1,2215),(876,'Analysis Type','Static',1,2216),(877,'Analysis Type','Static',1,2217),(878,'Analysis Type','Static',1,2218),(879,'Analysis Type','Static',1,2219),(880,'Analysis Type','Static',1,2220),(881,'Analysis Type','Static',1,2221),(882,'Analysis Type','Static',1,2222),(883,'Analysis Type','Static',1,2223),(884,'Analysis Type','Static',1,2224),(885,'Analysis Type','Static',1,2225),(886,'Analysis Type','Static',1,2226),(887,'Analysis Type','Static',1,2228),(888,'Analysis Type','Static',1,2229),(889,'Analysis Type','Static',1,2230),(890,'Analysis Type','Static',1,2231),(891,'Analysis Type','Static',1,2236),(892,'Analysis Type','Static',1,2239),(893,'Analysis Type','Static',1,2240),(894,'Analysis Type','Static',1,2241),(895,'Analysis Type','Static',1,2263),(896,'Analysis Type','Static',1,2264),(897,'Analysis Type','Static',1,2265),(898,'Analysis Type','Static',1,2273),(899,'Analysis Type','Static',1,2274),(900,'Analysis Type','Static',1,2275),(901,'Analysis Type','Static',1,2276),(902,'Analysis Type','Static',1,2277),(903,'Analysis Type','Static',1,2278),(904,'Analysis Type','Static',1,2279),(905,'Analysis Type','Static',1,2280),(906,'Analysis Type','Static',1,2281),(907,'Analysis Type','Static',1,2282),(908,'Analysis Type','Static',1,2283),(909,'Analysis Type','Static',1,2284),(910,'Analysis Type','Static',1,2298),(911,'Analysis Type','Static',1,2299),(912,'Analysis Type','Static',1,2350),(913,'Analysis Type','Static',1,2353),(914,'Analysis Type','Static',1,2354),(915,'Analysis Type','Static',1,2355),(916,'Analysis Type','Static',1,2356),(917,'Analysis Type','Static',1,2357),(918,'Analysis Type','Static',1,2358),(919,'Analysis Type','Static',1,2359),(920,'Analysis Type','Static',1,2360),(921,'Analysis Type','Static',1,2361),(922,'Analysis Type','Static',1,2362),(923,'Analysis Type','Static',1,2363),(924,'Analysis Type','Static',1,2364),(925,'Analysis Type','Static',1,2365),(926,'Analysis Type','Static',1,2366),(927,'Analysis Type','Static',1,2367),(928,'Analysis Type','Static',1,2368),(930,'Analysis Type','Static',1,2377),(939,'Analysis Type','Static',1,2387),(940,'Analysis Type','Static',1,2388),(941,'Analysis Type','Static',1,2389),(942,'Analysis Type','Static',1,2390),(943,'Analysis Type','Static',1,2391),(944,'Analysis Type','Static',1,2392),(945,'Analysis Type','Static',1,2393),(946,'Analysis Type','Static',1,2394),(947,'Analysis Type','Static',1,2395),(948,'Analysis Type','Static',1,2396),(949,'Analysis Type','Static',1,2397),(950,'Analysis Type','Static',1,2398),(951,'Analysis Type','Static',1,2401),(952,'Analysis Type','Static',1,2402),(953,'Analysis Type','Static',1,2403),(954,'Analysis Type','Static',1,2404),(955,'Analysis Type','Static',1,2405),(956,'Analysis Type','Static',1,2406),(957,'Analysis Type','Static',1,2407),(958,'Analysis Type','Static',1,2408),(959,'Analysis Type','Static',1,2409),(960,'Analysis Type','Static',1,2410),(961,'Analysis Type','Static',1,2411),(962,'Analysis Type','Static',1,2412),(963,'Analysis Type','Static',1,2413),(964,'Analysis Type','Static',1,2414),(965,'Analysis Type','Static',1,2415),(966,'Analysis Type','Static',1,2416),(967,'Analysis Type','Static',1,2417),(968,'Analysis Type','Static',1,2418),(969,'Analysis Type','Static',1,2419),(970,'Analysis Type','Static',1,2420),(971,'Analysis Type','Static',1,2421),(972,'Analysis Type','Static',1,2422),(973,'Analysis Type','Static',1,2423),(974,'Analysis Type','Static',1,2426),(975,'Analysis Type','Static',1,2427),(976,'Analysis Type','Static',1,2428),(977,'Analysis Type','Static',1,2429),(978,'Analysis Type','Static',1,2430),(979,'Analysis Type','Static',1,2431),(980,'Analysis Type','Static',1,2432),(981,'Analysis Type','Static',1,2433),(982,'Analysis Type','Static',1,2435),(983,'Analysis Type','Static',1,2436),(984,'Analysis Type','Static',1,2437),(985,'Analysis Type','Static',1,2438),(986,'Analysis Type','Static',1,2439),(987,'Analysis Type','Static',1,2440),(988,'Analysis Type','Static',1,2441),(989,'Analysis Type','Static',1,2442),(990,'Analysis Type','Static',1,2443),(996,'Analysis Type','Static',1,2453),(1001,'Analysis Type','Static',1,2459),(1002,'Analysis Type','Static',1,2460),(1003,'Analysis Type','Static',1,2461),(1004,'Analysis Type','Static',1,2462),(1005,'Analysis Type','Static',1,2463),(1006,'Analysis Type','Static',1,2464),(1007,'Analysis Type','Static',1,2465),(1008,'Analysis Type','Static',1,2466),(1009,'Analysis Type','Static',1,2467),(1010,'Analysis Type','Static',1,2468),(1011,'Analysis Type','Static',1,2469),(1012,'Analysis Type','Static',1,2470),(1013,'Analysis Type','Static',1,2471),(1014,'Analysis Type','Static',1,2472),(1015,'Analysis Type','Static',1,2473),(1016,'Analysis Type','Static',1,2474),(1017,'Analysis Type','Static',1,2475),(1018,'Analysis Type','Static',1,2476),(1019,'Analysis Type','Static',1,2477),(1020,'Analysis Type','Static',1,2478),(1021,'Analysis Type','Static',1,2495),(1022,'Analysis Type','Static',1,2496),(1023,'Analysis Type','Static',1,2497),(1024,'Analysis Type','Static',1,2498),(1025,'Analysis Type','Static',1,2499),(1026,'Analysis Type','Static',1,2501),(1029,'Analysis Type','Static',1,2593),(1030,'Analysis Type','Static',1,2594),(1031,'Analysis Type','Static',1,2595),(1032,'Analysis Type','Static',1,2596),(1033,'Analysis Type','Static',1,2779),(1034,'Analysis Type','Static',1,2780),(1035,'Analysis Type','Static',1,2781),(1036,'Analysis Type','Static',1,2782),(1037,'Analysis Type','Static',1,2783),(1038,'Analysis Type','Static',1,2784),(1039,'Analysis Type','Static',1,2785),(1040,'Analysis Type','Static',1,2786),(1041,'Analysis Type','Static',1,2787),(1042,'Analysis Type','Static',1,2788),(1043,'Analysis Type','Static',1,2789),(1044,'Analysis Type','Static',1,2790),(1045,'Analysis Type','Static',1,2791),(1046,'Analysis Type','Static',1,2792),(1047,'Analysis Type','Static',1,2795),(1048,'Analysis Type','Static',1,2797),(1049,'Analysis Type','Static',1,2798),(1050,'Analysis Type','Static',1,2799),(1051,'Analysis Type','Static',1,2800),(1052,'Analysis Type','Static',1,2801),(1053,'Analysis Type','Static',1,2802),(1054,'Analysis Type','Static',1,2816),(1055,'Analysis Type','Static',1,2839),(1056,'Analysis Type','Static',1,2840),(1057,'Analysis Type','Static',1,2841),(1058,'Analysis Type','Static',1,2842),(1059,'Analysis Type','Static',1,2843),(1060,'Analysis Type','Static',1,2844),(1061,'Analysis Type','Static',1,2847),(1062,'Analysis Type','Static',1,2849),(1063,'Analysis Type','Static',1,2850),(1064,'Analysis Type','Static',1,2851),(1065,'Analysis Type','Static',1,2852),(1066,'Analysis Type','Static',1,2853),(1067,'Analysis Type','Static',1,2854),(1068,'Analysis Type','Static',1,2855),(1069,'Analysis Type','Static',1,2856),(1070,'Analysis Type','Static',1,2857),(1071,'Analysis Type','Static',1,2908),(1072,'Analysis Type','Static',1,2946),(1073,'Analysis Type','Static',1,2947),(1074,'Analysis Type','Static',1,2952),(1075,'Analysis Type','Static',1,2953),(1076,'Analysis Type','Static',1,2954),(1077,'Analysis Type','Static',1,2962),(1078,'Analysis Type','Static',1,2978),(1079,'Analysis Type','Static',1,2979),(1080,'Analysis Type','Static',1,2980),(1081,'Analysis Type','Static',1,2987),(1082,'Analysis Type','Static',1,2988),(1083,'Analysis Type','Static',1,3014),(1084,'Analysis Type','Static',1,3015),(1085,'Analysis Type','Static',1,3016),(1086,'Analysis Type','Static',1,3017),(1087,'Analysis Type','Static',1,3018),(1088,'Analysis Type','Static',1,3019),(1089,'Analysis Type','Static',1,3076),(1090,'Analysis Type','Static',1,3079),(1091,'Analysis Type','Static',1,3080),(1092,'Analysis Type','Static',1,3102),(1093,'Analysis Type','Static',1,3103),(1094,'Analysis Type','Static',1,3104),(1095,'Analysis Type','Static',1,3250),(1096,'Analysis Type','Static',1,3251),(1097,'Analysis Type','Static',1,3252),(1098,'Analysis Type','Static',1,3253),(1099,'Analysis Type','Static',1,3254),(1100,'Analysis Type','Static',1,3257),(1101,'Analysis Type','Static',1,3258),(1102,'Analysis Type','Static',1,3259),(1103,'Analysis Type','Static',1,3260),(1104,'Analysis Type','Static',1,3261),(1105,'Analysis Type','Static',1,3262),(1106,'Analysis Type','Static',1,3264),(1107,'Analysis Type','Static',1,3265),(1108,'Analysis Type','Static',1,3266),(1109,'Analysis Type','Static',1,3267),(1110,'Analysis Type','Static',1,3269),(1111,'Analysis Type','Static',1,3270),(1112,'Analysis Type','Static',1,3271),(1113,'Analysis Type','Static',1,3272),(1114,'Analysis Type','Static',1,3275),(1115,'Analysis Type','Static',1,3276),(1116,'Analysis Type','Static',1,3286),(1117,'Analysis Type','Static',1,3287),(1118,'Analysis Type','Static',1,3288),(1119,'Analysis Type','Static',1,3289),(1120,'Analysis Type','Static',1,3290),(1121,'Analysis Type','Static',1,3291),(1122,'Analysis Type','Static',1,3292),(1123,'Analysis Type','Static',1,3293),(1124,'Analysis Type','Static',1,3294),(1125,'Analysis Type','Static',1,3295),(1126,'Analysis Type','Static',1,3296),(1127,'Analysis Type','Static',1,3297),(1128,'Analysis Type','Static',1,3298),(1129,'Analysis Type','Static',1,3299),(1130,'Analysis Type','Static',1,3300),(1131,'Analysis Type','Static',1,3301),(1132,'Analysis Type','Static',1,3302),(1133,'Analysis Type','Static',1,3303),(1134,'Analysis Type','Static',1,3304),(1135,'Analysis Type','Static',1,3305),(1136,'Analysis Type','Static',1,3306),(1137,'Analysis Type','Static',1,3307),(1138,'Analysis Type','Static',1,3308),(1139,'Analysis Type','Static',1,3309),(1140,'Analysis Type','Static',1,3310),(1141,'Analysis Type','Static',1,3311),(1142,'Analysis Type','Static',1,3312),(1143,'Analysis Type','Static',1,3313),(1144,'Analysis Type','Static',1,3314),(1145,'Analysis Type','Static',1,3315),(1146,'Analysis Type','Static',1,3316),(1147,'Analysis Type','Static',1,3317),(1148,'Analysis Type','Static',1,3324),(1149,'Analysis Type','Static',1,3325),(1150,'Analysis Type','Static',1,3326),(1151,'Analysis Type','Static',1,3327),(1152,'Analysis Type','Static',1,3328),(1153,'Analysis Type','Static',1,3329),(1154,'Analysis Type','Static',1,3330),(1155,'Analysis Type','Static',1,3331),(1156,'Analysis Type','Static',1,3332),(1157,'Analysis Type','Static',1,3333),(1158,'Analysis Type','Static',1,3334),(1159,'Analysis Type','Static',1,3335),(1160,'Analysis Type','Static',1,3336),(1161,'Analysis Type','Static',1,3337),(1162,'Analysis Type','Static',1,3338),(1163,'Analysis Type','Static',1,3339),(1164,'Analysis Type','Static',1,3340),(1165,'Analysis Type','Static',1,3341),(1166,'Analysis Type','Static',1,3342),(1167,'Analysis Type','Static',1,3343),(1168,'Analysis Type','Static',1,3344),(1169,'Analysis Type','Static',1,3345),(1170,'Analysis Type','Static',1,3346),(1171,'Analysis Type','Static',1,3347),(1172,'Analysis Type','Static',1,3348),(1173,'Analysis Type','Static',1,3349),(1174,'Analysis Type','Static',1,3350),(1175,'Analysis Type','Static',1,3351),(1176,'Analysis Type','Static',1,3352),(1177,'Analysis Type','Static',1,3353),(1178,'Analysis Type','Static',1,3354),(1179,'Analysis Type','Static',1,3355),(1180,'Analysis Type','Static',1,3356),(1181,'Analysis Type','Static',1,3357),(1182,'Analysis Type','Static',1,3358),(1183,'Analysis Type','Static',1,3359),(1184,'Analysis Type','Static',1,3360),(1185,'Analysis Type','Static',1,3361),(1186,'Analysis Type','Static',1,3362),(1187,'Analysis Type','Static',1,3363),(1188,'Analysis Type','Static',1,3364),(1189,'Analysis Type','Static',1,3365),(1190,'Analysis Type','Static',1,3366),(1191,'Analysis Type','Static',1,3367),(1192,'Analysis Type','Static',1,3368),(1193,'Analysis Type','Static',1,3369),(1194,'Analysis Type','Static',1,3370),(1195,'Analysis Type','Static',1,3371),(1196,'Analysis Type','Static',1,3381),(1197,'Analysis Type','Static',1,3382),(1198,'Analysis Type','Static',1,3409),(1199,'Analysis Type','Static',1,3410),(1200,'Analysis Type','Static',1,3411),(1201,'Analysis Type','Static',1,3412),(1202,'Analysis Type','Static',1,3413),(1203,'Analysis Type','Static',1,3414),(1204,'Analysis Type','Static',1,3430),(1205,'Analysis Type','Static',1,3431),(1206,'Analysis Type','Static',1,3432),(1207,'Analysis Type','Static',1,3433),(1208,'Analysis Type','Static',1,3434),(1209,'Analysis Type','Static',1,3435),(1210,'Analysis Type','Static',1,3439),(1211,'Analysis Type','Static',1,3440),(1212,'Analysis Type','Static',1,3443),(1213,'Analysis Type','Static',1,3444),(1214,'Analysis Type','Static',1,3445),(1215,'Analysis Type','Static',1,3461),(1224,'Analysis Type','Static',1,3477),(1225,'Analysis Type','Static',1,3478),(1226,'Analysis Type','Static',1,3479),(1227,'Analysis Type','Static',1,3480),(1228,'Analysis Type','Static',1,3481),(1229,'Analysis Type','Static',1,3482),(1230,'Analysis Type','Static',1,3483),(1231,'Analysis Type','Static',1,3484),(1232,'Analysis Type','Static',1,3486),(1233,'Analysis Type','Static',1,3487),(1234,'Analysis Type','Static',1,3554),(1235,'Analysis Type','Static',1,3555),(1236,'Analysis Type','Static',1,3556),(1237,'Analysis Type','Static',1,3557),(1238,'Analysis Type','Static',1,3558),(1239,'Analysis Type','Static',1,3559),(1240,'Analysis Type','Static',1,3560),(1241,'Analysis Type','Static',1,3561),(1246,'Analysis Type','Static',1,3566),(1247,'Analysis Type','Static',1,3567),(1248,'Analysis Type','Static',1,3568),(1249,'Analysis Type','Static',1,3569),(1250,'Analysis Type','Static',1,3570),(1251,'Analysis Type','Static',1,3571),(1252,'Analysis Type','Static',1,3572),(1253,'Analysis Type','Static',1,3573),(1254,'Analysis Type','Static',1,3574),(1255,'Analysis Type','Static',1,3575),(1256,'Analysis Type','Static',1,3576),(1257,'Analysis Type','Static',1,3577),(1258,'Analysis Type','Static',1,3578),(1259,'Analysis Type','Static',1,3579),(1260,'Analysis Type','Static',1,3580),(1261,'Analysis Type','Static',1,3581),(1262,'Analysis Type','Static',1,3582),(1263,'Analysis Type','Static',1,3583),(1264,'Analysis Type','Static',1,3584),(1265,'Analysis Type','Static',1,3585),(1266,'Analysis Type','Static',1,3586),(1267,'Analysis Type','Static',1,3587),(1268,'Analysis Type','Static',1,3588),(1269,'Analysis Type','Static',1,3589),(1270,'Analysis Type','Static',1,3596),(1271,'Analysis Type','Static',1,3597),(1272,'Analysis Type','Static',1,3598),(1273,'Analysis Type','Static',1,3599),(1274,'Analysis Type','Static',1,3600),(1275,'Analysis Type','Static',1,3601),(1276,'Analysis Type','Static',1,3602),(1277,'Analysis Type','Static',1,3603),(1278,'Analysis Type','Static',1,3604),(1279,'Analysis Type','Static',1,3605),(1280,'Analysis Type','Static',1,3606),(1281,'Analysis Type','Static',1,3607),(1282,'Analysis Type','Static',1,3608),(1283,'Analysis Type','Static',1,3609),(1284,'Analysis Type','Static',1,3610),(1285,'Analysis Type','Static',1,3611),(1286,'Analysis Type','Static',1,3612),(1287,'Analysis Type','Static',1,3613),(1288,'Analysis Type','Static',1,3614),(1289,'Analysis Type','Static',1,3615),(1290,'Analysis Type','Static',1,3616),(1291,'Analysis Type','Static',1,3617),(1292,'Analysis Type','Static',1,3618),(1293,'Analysis Type','Static',1,3619),(1294,'Experiment Platform','Illumina Genome Analyzer',2,823),(1295,'Experiment Platform','Illumina Genome Analyzer',2,881),(1296,'Experiment Platform','Illumina Genome Analyzer',2,882),(1297,'Experiment Platform','Illumina Genome Analyzer',2,883),(1298,'Experiment Platform','Illumina Genome Analyzer',2,1262),(1299,'Experiment Platform','Illumina Genome Analyzer',2,1329),(1300,'Experiment Platform','Illumina Genome Analyzer',2,1330),(1301,'Experiment Platform','Illumina Genome Analyzer',2,1331),(1302,'Experiment Platform','Illumina Genome Analyzer',2,1332),(1303,'Experiment Platform','Illumina Genome Analyzer',2,1333),(1304,'Experiment Platform','Illumina Genome Analyzer',2,1334),(1305,'Experiment Platform','Illumina Genome Analyzer',2,1335),(1306,'Experiment Platform','Illumina Genome Analyzer',2,1336),(1307,'Experiment Platform','Illumina Genome Analyzer',2,1337),(1308,'Experiment Platform','Illumina Genome Analyzer',2,1338),(1309,'Experiment Platform','Illumina Genome Analyzer',2,1339),(1310,'Experiment Platform','Illumina Genome Analyzer',2,1340),(1311,'Experiment Platform','Illumina Genome Analyzer',2,1354),(1312,'Experiment Platform','Illumina Genome Analyzer',2,1355),(1313,'Experiment Platform','Illumina Genome Analyzer',2,1356),(1314,'Experiment Platform','Illumina Genome Analyzer',2,1357),(1315,'Experiment Platform','Illumina Genome Analyzer',2,1358),(1316,'Experiment Platform','Illumina Genome Analyzer',2,1359),(1317,'Experiment Platform','Illumina Genome Analyzer',2,1360),(1318,'Experiment Platform','Illumina Genome Analyzer',2,1361),(1319,'Experiment Platform','Illumina Genome Analyzer',2,1362),(1320,'Experiment Platform','Illumina Genome Analyzer',2,1363),(1321,'Experiment Platform','Illumina Genome Analyzer',2,1364),(1322,'Experiment Platform','Illumina Genome Analyzer',2,1365),(1323,'Experiment Platform','Illumina Genome Analyzer',2,1366),(1324,'Experiment Platform','Illumina Genome Analyzer',2,1367),(1325,'Experiment Platform','Illumina Genome Analyzer',2,1368),(1326,'Experiment Platform','Illumina Genome Analyzer',2,1369),(1327,'Experiment Platform','Illumina Genome Analyzer',2,1370),(1328,'Experiment Platform','Illumina Genome Analyzer',2,1371),(1329,'Experiment Platform','Illumina Genome Analyzer',2,1372),(1330,'Experiment Platform','Illumina Genome Analyzer',2,1373),(1331,'Experiment Platform','Illumina Genome Analyzer',2,1374),(1332,'Experiment Platform','Illumina Genome Analyzer',2,1375),(1333,'Experiment Platform','Illumina Genome Analyzer',2,1376),(1334,'Experiment Platform','Illumina Genome Analyzer',2,1377),(1335,'Experiment Platform','Illumina Genome Analyzer',2,1378),(1336,'Experiment Platform','Illumina Genome Analyzer',2,1379),(1337,'Experiment Platform','Illumina Genome Analyzer',2,1380),(1338,'Experiment Platform','Illumina Genome Analyzer',2,1381),(1339,'Experiment Platform','Illumina Genome Analyzer',2,1382),(1340,'Experiment Platform','Illumina Genome Analyzer',2,1383),(1341,'Experiment Platform','Illumina Genome Analyzer',2,1384),(1342,'Experiment Platform','Illumina Genome Analyzer',2,1385),(1343,'Experiment Platform','Illumina Genome Analyzer',2,1386),(1344,'Experiment Platform','Illumina Genome Analyzer',2,1387),(1345,'Experiment Platform','Illumina Genome Analyzer',2,1388),(1346,'Experiment Platform','Illumina Genome Analyzer',2,1389),(1347,'Experiment Platform','Illumina Genome Analyzer',2,1390),(1348,'Experiment Platform','Illumina Genome Analyzer',2,1391),(1349,'Experiment Platform','Illumina Genome Analyzer',2,1392),(1350,'Experiment Platform','Illumina Genome Analyzer',2,1393),(1351,'Experiment Platform','Illumina Genome Analyzer',2,1394),(1352,'Experiment Platform','Illumina Genome Analyzer',2,1395),(1353,'Experiment Platform','Illumina Genome Analyzer',2,1396),(1354,'Experiment Platform','Illumina Genome Analyzer',2,1397),(1355,'Experiment Platform','Illumina Genome Analyzer',2,1398),(1356,'Experiment Platform','Illumina Genome Analyzer',2,1399),(1357,'Experiment Platform','Illumina Genome Analyzer',2,1400),(1358,'Experiment Platform','Illumina Genome Analyzer',2,1401),(1359,'Experiment Platform','Illumina Genome Analyzer',2,1402),(1360,'Experiment Platform','Illumina Genome Analyzer',2,1403),(1361,'Experiment Platform','Illumina Genome Analyzer',2,1404),(1362,'Experiment Platform','Illumina Genome Analyzer',2,1405),(1363,'Experiment Platform','Illumina Genome Analyzer',2,1406);

INSERT INTO dbo.gpAnnotationProperty
(idAnnotationProperty,
name,
value,
idProperty,
idAnnotation
) VALUES
(1364,'Experiment Platform','Illumina Genome Analyzer',2,1407),(1365,'Experiment Platform','Illumina Genome Analyzer',2,1408),(1366,'Experiment Platform','Illumina Genome Analyzer',2,1409),(1367,'Experiment Platform','Illumina Genome Analyzer',2,1410),(1368,'Experiment Platform','Illumina Genome Analyzer',2,1411),(1369,'Experiment Platform','Illumina Genome Analyzer',2,1412),(1370,'Experiment Platform','Illumina Genome Analyzer',2,1413),(1371,'Experiment Platform','Illumina Genome Analyzer',2,1414),(1372,'Experiment Platform','Illumina Genome Analyzer',2,1415),(1373,'Experiment Platform','Illumina Genome Analyzer',2,1416),(1374,'Experiment Platform','Illumina Genome Analyzer',2,1424),(1375,'Experiment Platform','Illumina Genome Analyzer',2,1425),(1376,'Experiment Platform','Illumina Genome Analyzer',2,1426),(1377,'Experiment Platform','Illumina Genome Analyzer',2,1427),(1378,'Experiment Platform','Illumina Genome Analyzer',2,1428),(1379,'Experiment Platform','Illumina Genome Analyzer',2,1429),(1380,'Experiment Platform','Illumina Genome Analyzer',2,1430),(1381,'Experiment Platform','Illumina Genome Analyzer',2,1431),(1382,'Experiment Platform','Illumina Genome Analyzer',2,1432),(1383,'Experiment Platform','Illumina Genome Analyzer',2,1433),(1384,'Experiment Platform','Illumina Genome Analyzer',2,1434),(1385,'Experiment Platform','Illumina Genome Analyzer',2,1435),(1386,'Experiment Platform','Illumina Genome Analyzer',2,1449),(1387,'Experiment Platform','Illumina Genome Analyzer',2,1450),(1388,'Experiment Platform','Illumina Genome Analyzer',2,1451),(1389,'Experiment Platform','Illumina Genome Analyzer',2,1452),(1390,'Experiment Platform','Illumina Genome Analyzer',2,1453),(1391,'Experiment Platform','Illumina Genome Analyzer',2,1454),(1392,'Experiment Platform','Illumina Genome Analyzer',2,1455),(1393,'Experiment Platform','Illumina Genome Analyzer',2,1456),(1394,'Experiment Platform','Illumina Genome Analyzer',2,1457),(1395,'Experiment Platform','Illumina Genome Analyzer',2,1458),(1396,'Experiment Platform','Illumina Genome Analyzer',2,1459),(1397,'Experiment Platform','Illumina Genome Analyzer',2,1460),(1398,'Experiment Platform','Illumina Genome Analyzer',2,1461),(1399,'Experiment Platform','Illumina Genome Analyzer',2,1462),(1400,'Experiment Platform','Illumina Genome Analyzer',2,1463),(1401,'Experiment Platform','Illumina Genome Analyzer',2,1464),(1402,'Experiment Platform','Illumina Genome Analyzer',2,1465),(1403,'Experiment Platform','Illumina Genome Analyzer',2,1466),(1404,'Experiment Platform','Illumina Genome Analyzer',2,1467),(1405,'Experiment Platform','Illumina Genome Analyzer',2,1468),(1406,'Experiment Platform','Illumina Genome Analyzer',2,1469),(1407,'Experiment Platform','Illumina Genome Analyzer',2,1470),(1408,'Experiment Platform','Illumina Genome Analyzer',2,1471),(1409,'Experiment Platform','Illumina Genome Analyzer',2,1472),(1410,'Experiment Platform','Illumina Genome Analyzer',2,1473),(1411,'Experiment Platform','Illumina Genome Analyzer',2,1474),(1412,'Experiment Platform','Illumina Genome Analyzer',2,1475),(1413,'Experiment Platform','Illumina Genome Analyzer',2,1476),(1414,'Experiment Platform','Illumina Genome Analyzer',2,1477),(1415,'Experiment Platform','Illumina Genome Analyzer',2,1478),(1416,'Experiment Platform','Illumina Genome Analyzer',2,1479),(1417,'Experiment Platform','Illumina Genome Analyzer',2,1480),(1418,'Experiment Platform','Illumina Genome Analyzer',2,1481),(1419,'Experiment Platform','Illumina Genome Analyzer',2,1482),(1420,'Experiment Platform','Illumina Genome Analyzer',2,1483),(1421,'Experiment Platform','Illumina Genome Analyzer',2,1484),(1422,'Experiment Platform','Illumina Genome Analyzer',2,1485),(1423,'Experiment Platform','Illumina Genome Analyzer',2,1486),(1424,'Experiment Platform','Illumina Genome Analyzer',2,1487),(1425,'Experiment Platform','Illumina Genome Analyzer',2,1488),(1426,'Experiment Platform','Illumina Genome Analyzer',2,1489),(1427,'Experiment Platform','Illumina Genome Analyzer',2,1490),(1428,'Experiment Platform','Illumina Genome Analyzer',2,1491),(1429,'Experiment Platform','Illumina Genome Analyzer',2,1492),(1430,'Experiment Platform','Illumina Genome Analyzer',2,1493),(1431,'Experiment Platform','Illumina Genome Analyzer',2,1494),(1432,'Experiment Platform','Illumina Genome Analyzer',2,1495),(1433,'Experiment Platform','Illumina Genome Analyzer',2,1496),(1434,'Experiment Platform','Illumina Genome Analyzer',2,1497),(1435,'Experiment Platform','Illumina Genome Analyzer',2,1498),(1436,'Experiment Platform','Illumina Genome Analyzer',2,1499),(1437,'Experiment Platform','Illumina Genome Analyzer',2,1500),(1438,'Experiment Platform','Illumina Genome Analyzer',2,1501),(1439,'Experiment Platform','Illumina Genome Analyzer',2,1502),(1440,'Experiment Platform','Illumina Genome Analyzer',2,1503),(1441,'Experiment Platform','Illumina Genome Analyzer',2,1504),(1442,'Experiment Platform','Illumina Genome Analyzer',2,1505),(1443,'Experiment Platform','Illumina Genome Analyzer',2,1506),(1444,'Experiment Platform','Illumina Genome Analyzer',2,1507),(1445,'Experiment Platform','Illumina Genome Analyzer',2,1508),(1446,'Experiment Platform','Illumina Genome Analyzer',2,1509),(1447,'Experiment Platform','Illumina Genome Analyzer',2,1510),(1448,'Experiment Platform','Illumina Genome Analyzer',2,1511),(1449,'Experiment Platform','Illumina Genome Analyzer',2,1512),(1450,'Experiment Platform','Illumina Genome Analyzer',2,1513),(1451,'Experiment Platform','Illumina Genome Analyzer',2,1514),(1452,'Experiment Platform','Illumina Genome Analyzer',2,1515),(1453,'Experiment Platform','Illumina Genome Analyzer',2,1516),(1454,'Experiment Platform','Illumina Genome Analyzer',2,1517),(1455,'Experiment Platform','Illumina Genome Analyzer',2,1518),(1456,'Experiment Platform','Illumina Genome Analyzer',2,1519),(1457,'Experiment Platform','Illumina Genome Analyzer',2,1521),(1458,'Experiment Platform','Illumina Genome Analyzer',2,1522),(1459,'Experiment Platform','Illumina Genome Analyzer',2,1523),(1460,'Experiment Platform','Illumina Genome Analyzer',2,1524),(1461,'Experiment Platform','Illumina Genome Analyzer',2,1525),(1462,'Experiment Platform','Illumina Genome Analyzer',2,1526),(1463,'Experiment Platform','Illumina Genome Analyzer',2,1527),(1464,'Experiment Platform','Illumina Genome Analyzer',2,1528),(1465,'Experiment Platform','Illumina Genome Analyzer',2,1529),(1466,'Experiment Platform','Illumina Genome Analyzer',2,1530),(1467,'Experiment Platform','Illumina Genome Analyzer',2,1531),(1468,'Experiment Platform','Illumina Genome Analyzer',2,1532),(1469,'Experiment Platform','Illumina Genome Analyzer',2,1533),(1470,'Experiment Platform','Illumina Genome Analyzer',2,1534),(1471,'Experiment Platform','Illumina Genome Analyzer',2,1535),(1472,'Experiment Platform','Illumina Genome Analyzer',2,1536),(1473,'Experiment Platform','Illumina Genome Analyzer',2,1537),(1474,'Experiment Platform','Illumina Genome Analyzer',2,1538),(1475,'Experiment Platform','Illumina Genome Analyzer',2,1539),(1476,'Experiment Platform','Illumina Genome Analyzer',2,1540),(1477,'Experiment Platform','Illumina Genome Analyzer',2,1541),(1478,'Experiment Platform','Illumina Genome Analyzer',2,1542),(1479,'Experiment Platform','Illumina Genome Analyzer',2,1543),(1480,'Experiment Platform','Illumina Genome Analyzer',2,1544),(1481,'Experiment Platform','Illumina Genome Analyzer',2,1545),(1482,'Experiment Platform','Illumina Genome Analyzer',2,1546),(1483,'Experiment Platform','Illumina Genome Analyzer',2,1547),(1484,'Experiment Platform','Illumina Genome Analyzer',2,1548),(1485,'Experiment Platform','Illumina Genome Analyzer',2,1549),(1486,'Experiment Platform','Illumina Genome Analyzer',2,1550),(1487,'Experiment Platform','Illumina Genome Analyzer',2,1551),(1488,'Experiment Platform','Illumina Genome Analyzer',2,1552),(1489,'Experiment Platform','Illumina Genome Analyzer',2,1553),(1490,'Experiment Platform','Illumina Genome Analyzer',2,1554),(1491,'Experiment Platform','Illumina Genome Analyzer',2,1555),(1492,'Experiment Platform','Illumina Genome Analyzer',2,1556),(1493,'Experiment Platform','Illumina Genome Analyzer',2,1557),(1494,'Experiment Platform','Illumina Genome Analyzer',2,1558),(1495,'Experiment Platform','Illumina Genome Analyzer',2,1559),(1496,'Experiment Platform','Illumina Genome Analyzer',2,1560),(1497,'Experiment Platform','Illumina Genome Analyzer',2,1561),(1498,'Experiment Platform','Illumina Genome Analyzer',2,1562),(1499,'Experiment Platform','Illumina Genome Analyzer',2,1563),(1500,'Experiment Platform','Illumina Genome Analyzer',2,1564),(1501,'Experiment Platform','Illumina Genome Analyzer',2,1565),(1502,'Experiment Platform','Illumina Genome Analyzer',2,1566),(1503,'Experiment Platform','Illumina Genome Analyzer',2,1567),(1504,'Experiment Platform','Illumina Genome Analyzer',2,1568),(1505,'Experiment Platform','Illumina Genome Analyzer',2,1569),(1506,'Experiment Platform','Illumina Genome Analyzer',2,1570),(1507,'Experiment Platform','Illumina Genome Analyzer',2,1571),(1508,'Experiment Platform','Illumina Genome Analyzer',2,1572),(1509,'Experiment Platform','Illumina Genome Analyzer',2,1573),(1510,'Experiment Platform','Illumina Genome Analyzer',2,1574),(1511,'Experiment Platform','Illumina Genome Analyzer',2,1575),(1512,'Experiment Platform','Illumina Genome Analyzer',2,1576),(1513,'Experiment Platform','Illumina Genome Analyzer',2,1577),(1514,'Experiment Platform','Illumina Genome Analyzer',2,1578),(1515,'Experiment Platform','Illumina Genome Analyzer',2,1579),(1516,'Experiment Platform','Illumina Genome Analyzer',2,1580),(1517,'Experiment Platform','Illumina Genome Analyzer',2,1581),(1518,'Experiment Platform','Illumina Genome Analyzer',2,1582),(1519,'Experiment Platform','Illumina Genome Analyzer',2,1583),(1520,'Experiment Platform','Illumina Genome Analyzer',2,1584),(1521,'Experiment Platform','Illumina Genome Analyzer',2,1585),(1522,'Experiment Platform','Illumina Genome Analyzer',2,1586),(1523,'Experiment Platform','Illumina Genome Analyzer',2,1587),(1524,'Experiment Platform','Illumina Genome Analyzer',2,1588),(1525,'Experiment Platform','Illumina Genome Analyzer',2,1589),(1526,'Experiment Platform','Illumina Genome Analyzer',2,1590),(1527,'Experiment Platform','Illumina Genome Analyzer',2,1591),(1528,'Experiment Platform','Illumina Genome Analyzer',2,1592),(1529,'Experiment Platform','Illumina Genome Analyzer',2,1593),(1530,'Experiment Platform','Illumina Genome Analyzer',2,1594),(1531,'Experiment Platform','Illumina Genome Analyzer',2,1595),(1532,'Experiment Platform','Illumina Genome Analyzer',2,1598),(1533,'Experiment Platform','Illumina Genome Analyzer',2,1599),(1534,'Experiment Platform','Illumina Genome Analyzer',2,1600),(1535,'Experiment Platform','Illumina Genome Analyzer',2,1601),(1536,'Experiment Platform','Illumina Genome Analyzer',2,1602),(1537,'Experiment Platform','Illumina Genome Analyzer',2,1603),(1538,'Experiment Platform','Illumina Genome Analyzer',2,1604),(1539,'Experiment Platform','Illumina Genome Analyzer',2,1605),(1540,'Experiment Platform','Illumina Genome Analyzer',2,1606),(1541,'Experiment Platform','Illumina Genome Analyzer',2,1607),(1542,'Experiment Platform','Illumina Genome Analyzer',2,1608),(1543,'Experiment Platform','Illumina Genome Analyzer',2,1609),(1544,'Experiment Platform','Illumina Genome Analyzer',2,1610),(1545,'Experiment Platform','Illumina Genome Analyzer',2,1611),(1546,'Experiment Platform','Illumina Genome Analyzer',2,1612),(1547,'Experiment Platform','Illumina Genome Analyzer',2,1613),(1548,'Experiment Platform','Illumina Genome Analyzer',2,1614),(1549,'Experiment Platform','Illumina Genome Analyzer',2,1615),(1550,'Experiment Platform','Illumina Genome Analyzer',2,1616),(1551,'Experiment Platform','Illumina Genome Analyzer',2,1617),(1552,'Experiment Platform','Illumina Genome Analyzer',2,1618),(1553,'Experiment Platform','Illumina Genome Analyzer',2,1619),(1554,'Experiment Platform','Illumina Genome Analyzer',2,1620),(1555,'Experiment Platform','Illumina Genome Analyzer',2,1621),(1556,'Experiment Platform','Illumina Genome Analyzer',2,1622),(1557,'Experiment Platform','Illumina Genome Analyzer',2,1623),(1558,'Experiment Platform','Illumina Genome Analyzer',2,1624),(1559,'Experiment Platform','Illumina Genome Analyzer',2,1625),(1560,'Experiment Platform','Illumina Genome Analyzer',2,1626),(1561,'Experiment Platform','Illumina Genome Analyzer',2,1627),(1562,'Experiment Platform','Illumina Genome Analyzer',2,1628),(1563,'Experiment Platform','Illumina Genome Analyzer',2,1629),(1564,'Experiment Platform','Illumina Genome Analyzer',2,1630),(1565,'Experiment Platform','Illumina Genome Analyzer',2,1631),(1566,'Experiment Platform','Illumina Genome Analyzer',2,1632),(1567,'Experiment Platform','Illumina Genome Analyzer',2,1633),(1568,'Experiment Platform','Illumina Genome Analyzer',2,1634),(1569,'Experiment Platform','Illumina Genome Analyzer',2,1635),(1570,'Experiment Platform','Illumina Genome Analyzer',2,1636),(1571,'Experiment Platform','Illumina Genome Analyzer',2,1637),(1572,'Experiment Platform','Illumina Genome Analyzer',2,1638),(1573,'Experiment Platform','Illumina Genome Analyzer',2,1639),(1574,'Experiment Platform','Illumina Genome Analyzer',2,1640),(1575,'Experiment Platform','Illumina Genome Analyzer',2,1641),(1576,'Experiment Platform','Illumina Genome Analyzer',2,1642),(1577,'Experiment Platform','Illumina Genome Analyzer',2,1643),(1578,'Experiment Platform','Illumina Genome Analyzer',2,1644),(1579,'Experiment Platform','Illumina Genome Analyzer',2,1645),(1580,'Experiment Platform','Illumina Genome Analyzer',2,1646),(1581,'Experiment Platform','Illumina Genome Analyzer',2,1647),(1582,'Experiment Platform','Illumina Genome Analyzer',2,1648),(1583,'Experiment Platform','Illumina Genome Analyzer',2,1649),(1584,'Experiment Platform','Illumina Genome Analyzer',2,1650),(1585,'Experiment Platform','Illumina Genome Analyzer',2,1651),(1586,'Experiment Platform','Illumina Genome Analyzer',2,1652),(1587,'Experiment Platform','Illumina Genome Analyzer',2,1653),(1588,'Experiment Platform','Illumina Genome Analyzer',2,1654),(1589,'Experiment Platform','Illumina Genome Analyzer',2,1655),(1590,'Experiment Platform','Illumina Genome Analyzer',2,1656),(1591,'Experiment Platform','Illumina Genome Analyzer',2,1657),(1592,'Experiment Platform','Illumina Genome Analyzer',2,1658),(1593,'Experiment Platform','Illumina Genome Analyzer',2,1659),(1594,'Experiment Platform','Illumina Genome Analyzer',2,1660),(1595,'Experiment Platform','Illumina Genome Analyzer',2,1661);

INSERT INTO dbo.gpAnnotationProperty
(idAnnotationProperty,
name,
value,
idProperty,
idAnnotation
) VALUES
(1596,'Experiment Platform','Illumina Genome Analyzer',2,1662),(1597,'Experiment Platform','Illumina Genome Analyzer',2,1663),(1598,'Experiment Platform','Illumina Genome Analyzer',2,1664),(1599,'Experiment Platform','Illumina Genome Analyzer',2,1665),(1600,'Experiment Platform','Illumina Genome Analyzer',2,1666),(1601,'Experiment Platform','Illumina Genome Analyzer',2,1667),(1602,'Experiment Platform','Illumina Genome Analyzer',2,1668),(1603,'Experiment Platform','Illumina Genome Analyzer',2,1669),(1604,'Experiment Platform','Illumina Genome Analyzer',2,1670),(1605,'Experiment Platform','Illumina Genome Analyzer',2,1671),(1606,'Experiment Platform','Illumina Genome Analyzer',2,1672),(1607,'Experiment Platform','Illumina Genome Analyzer',2,1673),(1608,'Experiment Platform','Illumina Genome Analyzer',2,1674),(1609,'Experiment Platform','Illumina Genome Analyzer',2,1675),(1610,'Experiment Platform','Illumina Genome Analyzer',2,1676),(1611,'Experiment Platform','Illumina Genome Analyzer',2,1677),(1612,'Experiment Platform','Illumina Genome Analyzer',2,1678),(1613,'Experiment Platform','Illumina Genome Analyzer',2,1679),(1614,'Experiment Platform','Illumina Genome Analyzer',2,1680),(1615,'Experiment Platform','Illumina Genome Analyzer',2,1681),(1616,'Experiment Platform','Illumina Genome Analyzer',2,1682),(1617,'Experiment Platform','Illumina Genome Analyzer',2,1683),(1618,'Experiment Platform','Illumina Genome Analyzer',2,1684),(1619,'Experiment Platform','Illumina Genome Analyzer',2,1685),(1620,'Experiment Platform','Illumina Genome Analyzer',2,1686),(1621,'Experiment Platform','Illumina Genome Analyzer',2,1687),(1622,'Experiment Platform','Illumina Genome Analyzer',2,1688),(1623,'Experiment Platform','Illumina Genome Analyzer',2,1689),(1624,'Experiment Platform','Illumina Genome Analyzer',2,1690),(1625,'Experiment Platform','Illumina Genome Analyzer',2,1691),(1626,'Experiment Platform','Illumina Genome Analyzer',2,1692),(1627,'Experiment Platform','Illumina Genome Analyzer',2,1693),(1628,'Experiment Platform','Illumina Genome Analyzer',2,1694),(1629,'Experiment Platform','Illumina Genome Analyzer',2,1695),(1630,'Experiment Platform','Illumina Genome Analyzer',2,1696),(1631,'Experiment Platform','Illumina Genome Analyzer',2,1697),(1632,'Experiment Platform','Illumina Genome Analyzer',2,1698),(1633,'Experiment Platform','Illumina Genome Analyzer',2,1699),(1634,'Experiment Platform','Illumina Genome Analyzer',2,1700),(1635,'Experiment Platform','Illumina Genome Analyzer',2,1701),(1636,'Experiment Platform','Illumina Genome Analyzer',2,1702),(1637,'Experiment Platform','Illumina Genome Analyzer',2,1703),(1638,'Experiment Platform','Illumina Genome Analyzer',2,1704),(1639,'Experiment Platform','Illumina Genome Analyzer',2,1705),(1640,'Experiment Platform','Illumina Genome Analyzer',2,1706),(1641,'Experiment Platform','Illumina Genome Analyzer',2,1707),(1642,'Experiment Platform','Illumina Genome Analyzer',2,1708),(1643,'Experiment Platform','Illumina Genome Analyzer',2,1709),(1644,'Experiment Platform','Illumina Genome Analyzer',2,1710),(1645,'Experiment Platform','Illumina Genome Analyzer',2,1711),(1646,'Experiment Platform','Illumina Genome Analyzer',2,1712),(1647,'Experiment Platform','Illumina Genome Analyzer',2,1713),(1648,'Experiment Platform','Illumina Genome Analyzer',2,1714),(1649,'Experiment Platform','Illumina Genome Analyzer',2,1715),(1650,'Experiment Platform','Illumina Genome Analyzer',2,1716),(1651,'Experiment Platform','Illumina Genome Analyzer',2,1717),(1652,'Experiment Platform','Illumina Genome Analyzer',2,1718),(1653,'Experiment Platform','Illumina Genome Analyzer',2,1719),(1654,'Experiment Platform','Illumina Genome Analyzer',2,1720),(1655,'Experiment Platform','Illumina Genome Analyzer',2,1721),(1656,'Experiment Platform','Illumina Genome Analyzer',2,1722),(1657,'Experiment Platform','Illumina Genome Analyzer',2,1723),(1658,'Experiment Platform','Illumina Genome Analyzer',2,1724),(1659,'Experiment Platform','Illumina Genome Analyzer',2,1725),(1660,'Experiment Platform','Illumina Genome Analyzer',2,1726),(1661,'Experiment Platform','Illumina Genome Analyzer',2,1727),(1662,'Experiment Platform','Illumina Genome Analyzer',2,1728),(1663,'Experiment Platform','Illumina Genome Analyzer',2,1729),(1664,'Experiment Platform','Illumina Genome Analyzer',2,1730),(1665,'Experiment Platform','Illumina Genome Analyzer',2,1731),(1666,'Experiment Platform','Illumina Genome Analyzer',2,1732),(1667,'Experiment Platform','Illumina Genome Analyzer',2,1733),(1668,'Experiment Platform','Illumina Genome Analyzer',2,1734),(1669,'Experiment Platform','Illumina Genome Analyzer',2,1735),(1670,'Experiment Platform','Illumina Genome Analyzer',2,1736),(1671,'Experiment Platform','Illumina Genome Analyzer',2,1737),(1672,'Experiment Platform','Illumina Genome Analyzer',2,1738),(1673,'Experiment Platform','Illumina Genome Analyzer',2,1739),(1674,'Experiment Platform','Illumina Genome Analyzer',2,1740),(1675,'Experiment Platform','Illumina Genome Analyzer',2,1741),(1676,'Experiment Platform','Illumina Genome Analyzer',2,1742),(1677,'Experiment Platform','Illumina Genome Analyzer',2,1743),(1678,'Experiment Platform','Illumina Genome Analyzer',2,1744),(1679,'Experiment Platform','Illumina Genome Analyzer',2,1745),(1680,'Experiment Platform','Illumina Genome Analyzer',2,1746),(1681,'Experiment Platform','Illumina Genome Analyzer',2,1747),(1682,'Experiment Platform','Illumina Genome Analyzer',2,1748),(1683,'Experiment Platform','Illumina Genome Analyzer',2,1749),(1684,'Experiment Platform','Illumina Genome Analyzer',2,1750),(1685,'Experiment Platform','Illumina Genome Analyzer',2,1751),(1686,'Experiment Platform','Illumina Genome Analyzer',2,1752),(1687,'Experiment Platform','Illumina Genome Analyzer',2,1753),(1688,'Experiment Platform','Illumina Genome Analyzer',2,1754),(1689,'Experiment Platform','Illumina Genome Analyzer',2,1755),(1690,'Experiment Platform','Illumina Genome Analyzer',2,1756),(1691,'Experiment Platform','Illumina Genome Analyzer',2,1758),(1692,'Experiment Platform','Illumina Genome Analyzer',2,1759),(1693,'Experiment Platform','Illumina Genome Analyzer',2,1760),(1694,'Experiment Platform','Illumina Genome Analyzer',2,1761),(1695,'Experiment Platform','Illumina Genome Analyzer',2,1762),(1696,'Experiment Platform','Illumina Genome Analyzer',2,1763),(1697,'Experiment Platform','Illumina Genome Analyzer',2,1764),(1698,'Experiment Platform','Illumina Genome Analyzer',2,1792),(1699,'Experiment Platform','Illumina Genome Analyzer',2,1793),(1700,'Experiment Platform','Illumina Genome Analyzer',2,1799),(1701,'Experiment Platform','Illumina Genome Analyzer',2,1800),(1702,'Experiment Platform','Illumina Genome Analyzer',2,1801),(1703,'Experiment Platform','Illumina Genome Analyzer',2,1802),(1704,'Experiment Platform','Illumina Genome Analyzer',2,1803),(1705,'Experiment Platform','Illumina Genome Analyzer',2,1805),(1706,'Experiment Platform','Illumina Genome Analyzer',2,1806),(1707,'Experiment Platform','Illumina Genome Analyzer',2,1807),(1708,'Experiment Platform','Illumina Genome Analyzer',2,1808),(1709,'Experiment Platform','Illumina Genome Analyzer',2,1809),(1710,'Experiment Platform','Illumina Genome Analyzer',2,1812),(1711,'Experiment Platform','Illumina Genome Analyzer',2,1813),(1712,'Experiment Platform','Illumina Genome Analyzer',2,1814),(1713,'Experiment Platform','Illumina Genome Analyzer',2,1815),(1714,'Experiment Platform','Illumina Genome Analyzer',2,1816),(1715,'Experiment Platform','Illumina Genome Analyzer',2,1817),(1716,'Experiment Platform','Illumina Genome Analyzer',2,1818),(1717,'Experiment Platform','Illumina Genome Analyzer',2,1819),(1718,'Experiment Platform','Illumina Genome Analyzer',2,1820),(1719,'Experiment Platform','Illumina Genome Analyzer',2,1821),(1720,'Experiment Platform','Illumina Genome Analyzer',2,1822),(1721,'Experiment Platform','Illumina Genome Analyzer',2,1823),(1722,'Experiment Platform','Illumina Genome Analyzer',2,1824),(1723,'Experiment Platform','Illumina Genome Analyzer',2,1825),(1724,'Experiment Platform','Illumina Genome Analyzer',2,1826),(1725,'Experiment Platform','Illumina Genome Analyzer',2,1827),(1726,'Experiment Platform','Illumina Genome Analyzer',2,1828),(1727,'Experiment Platform','Illumina Genome Analyzer',2,1829),(1728,'Experiment Platform','Illumina Genome Analyzer',2,1830),(1729,'Experiment Platform','Illumina Genome Analyzer',2,1831),(1730,'Experiment Platform','Illumina Genome Analyzer',2,1832),(1731,'Experiment Platform','Illumina Genome Analyzer',2,1833),(1732,'Experiment Platform','Illumina Genome Analyzer',2,1834),(1733,'Experiment Platform','Illumina Genome Analyzer',2,1835),(1734,'Experiment Platform','Illumina Genome Analyzer',2,1837),(1735,'Experiment Platform','Illumina Genome Analyzer',2,1838),(1736,'Experiment Platform','Illumina Genome Analyzer',2,1839),(1737,'Experiment Platform','Illumina Genome Analyzer',2,1840),(1738,'Experiment Platform','Illumina Genome Analyzer',2,1841),(1739,'Experiment Platform','Illumina Genome Analyzer',2,1842),(1740,'Experiment Platform','Illumina Genome Analyzer',2,1861),(1741,'Experiment Platform','Illumina Genome Analyzer',2,1862),(1742,'Experiment Platform','Illumina Genome Analyzer',2,1863),(1743,'Experiment Platform','Illumina Genome Analyzer',2,1864),(1744,'Experiment Platform','Illumina Genome Analyzer',2,1865),(1745,'Experiment Platform','Illumina Genome Analyzer',2,1866),(1746,'Experiment Platform','Illumina Genome Analyzer',2,1867),(1747,'Experiment Platform','Illumina Genome Analyzer',2,1868),(1748,'Experiment Platform','Illumina Genome Analyzer',2,1869),(1749,'Experiment Platform','Illumina Genome Analyzer',2,1870),(1750,'Experiment Platform','Illumina Genome Analyzer',2,1871),(1751,'Experiment Platform','Illumina Genome Analyzer',2,1872),(1752,'Experiment Platform','Illumina Genome Analyzer',2,1873),(1753,'Experiment Platform','Illumina Genome Analyzer',2,1874),(1754,'Experiment Platform','Illumina Genome Analyzer',2,1875),(1755,'Experiment Platform','Illumina Genome Analyzer',2,1876),(1756,'Experiment Platform','Illumina Genome Analyzer',2,1877),(1757,'Experiment Platform','Illumina Genome Analyzer',2,1878),(1758,'Experiment Platform','Illumina Genome Analyzer',2,1879),(1759,'Experiment Platform','Illumina Genome Analyzer',2,1880),(1760,'Experiment Platform','Illumina Genome Analyzer',2,1881),(1761,'Experiment Platform','Illumina Genome Analyzer',2,1882),(1762,'Experiment Platform','Illumina Genome Analyzer',2,1883),(1763,'Experiment Platform','Illumina Genome Analyzer',2,1884),(1764,'Experiment Platform','Illumina Genome Analyzer',2,1885),(1765,'Experiment Platform','Illumina Genome Analyzer',2,1886),(1766,'Experiment Platform','Illumina Genome Analyzer',2,1887),(1767,'Experiment Platform','Illumina Genome Analyzer',2,1888),(1768,'Experiment Platform','Illumina Genome Analyzer',2,1889),(1769,'Experiment Platform','Illumina Genome Analyzer',2,1890),(1770,'Experiment Platform','Illumina Genome Analyzer',2,1891),(1771,'Experiment Platform','Illumina Genome Analyzer',2,1893),(1772,'Experiment Platform','Illumina Genome Analyzer',2,1894),(1773,'Experiment Platform','Illumina Genome Analyzer',2,1895),(1774,'Experiment Platform','Illumina Genome Analyzer',2,1896),(1775,'Experiment Platform','Illumina Genome Analyzer',2,1897),(1776,'Experiment Platform','Illumina Genome Analyzer',2,1898),(1777,'Experiment Platform','Illumina Genome Analyzer',2,1899),(1778,'Experiment Platform','Illumina Genome Analyzer',2,1900),(1779,'Experiment Platform','Illumina Genome Analyzer',2,1901),(1780,'Experiment Platform','Illumina Genome Analyzer',2,1902),(1781,'Experiment Platform','Illumina Genome Analyzer',2,1903),(1782,'Experiment Platform','Illumina Genome Analyzer',2,1904),(1783,'Experiment Platform','Illumina Genome Analyzer',2,1905),(1784,'Experiment Platform','Illumina Genome Analyzer',2,1921),(1785,'Experiment Platform','Illumina Genome Analyzer',2,1922),(1786,'Experiment Platform','Illumina Genome Analyzer',2,1923),(1787,'Experiment Platform','Illumina Genome Analyzer',2,1924),(1788,'Experiment Platform','Illumina Genome Analyzer',2,1925),(1789,'Experiment Platform','Illumina Genome Analyzer',2,1926),(1790,'Experiment Platform','Illumina Genome Analyzer',2,1927),(1791,'Experiment Platform','Illumina Genome Analyzer',2,1928),(1792,'Experiment Platform','Illumina Genome Analyzer',2,1929),(1793,'Experiment Platform','Illumina Genome Analyzer',2,1930),(1794,'Experiment Platform','Illumina Genome Analyzer',2,1931),(1795,'Experiment Platform','Illumina Genome Analyzer',2,1933),(1796,'Experiment Platform','Illumina Genome Analyzer',2,1934),(1797,'Experiment Platform','Illumina Genome Analyzer',2,1935),(1798,'Experiment Platform','Illumina Genome Analyzer',2,1936),(1799,'Experiment Platform','Illumina Genome Analyzer',2,1937),(1800,'Experiment Platform','Illumina Genome Analyzer',2,1938),(1801,'Experiment Platform','Illumina Genome Analyzer',2,1939),(1802,'Experiment Platform','Illumina Genome Analyzer',2,1940),(1803,'Experiment Platform','Illumina Genome Analyzer',2,1941),(1804,'Experiment Platform','Illumina Genome Analyzer',2,1942),(1805,'Experiment Platform','Illumina Genome Analyzer',2,1943),(1806,'Experiment Platform','Illumina Genome Analyzer',2,1944),(1807,'Experiment Platform','Illumina Genome Analyzer',2,1945),(1808,'Experiment Platform','Illumina Genome Analyzer',2,1946),(1809,'Experiment Platform','Illumina Genome Analyzer',2,1947),(1810,'Experiment Platform','Illumina Genome Analyzer',2,1948),(1811,'Experiment Platform','Illumina Genome Analyzer',2,1949),(1812,'Experiment Platform','Illumina Genome Analyzer',2,1950),(1813,'Experiment Platform','Illumina Genome Analyzer',2,1966),(1814,'Experiment Platform','Illumina Genome Analyzer',2,1967),(1815,'Experiment Platform','Illumina Genome Analyzer',2,1968),(1816,'Experiment Platform','Illumina Genome Analyzer',2,1969),(1817,'Experiment Platform','Illumina Genome Analyzer',2,1970),(1818,'Experiment Platform','Illumina Genome Analyzer',2,1971),(1819,'Experiment Platform','Illumina Genome Analyzer',2,1972),(1820,'Experiment Platform','Illumina Genome Analyzer',2,1973),(1821,'Experiment Platform','Illumina Genome Analyzer',2,1974),(1822,'Experiment Platform','Illumina Genome Analyzer',2,1975),(1823,'Experiment Platform','Illumina Genome Analyzer',2,1976),(1824,'Experiment Platform','Illumina Genome Analyzer',2,1977),(1825,'Experiment Platform','Illumina Genome Analyzer',2,1978),(1826,'Experiment Platform','Illumina Genome Analyzer',2,1979),(1827,'Experiment Platform','Illumina Genome Analyzer',2,1980),(1828,'Experiment Platform','Illumina Genome Analyzer',2,1981),(1829,'Experiment Platform','Illumina Genome Analyzer',2,1982),(1830,'Experiment Platform','Illumina Genome Analyzer',2,1983),(1831,'Experiment Platform','Illumina Genome Analyzer',2,1984),(1832,'Experiment Platform','Illumina Genome Analyzer',2,1985),(1833,'Experiment Platform','Illumina Genome Analyzer',2,1986),(1834,'Experiment Platform','Illumina Genome Analyzer',2,1987),(1835,'Experiment Platform','Illumina Genome Analyzer',2,1988),(1836,'Experiment Platform','Illumina Genome Analyzer',2,1989),(1837,'Experiment Platform','Illumina Genome Analyzer',2,1990),(1838,'Experiment Platform','Illumina Genome Analyzer',2,1991),(1839,'Experiment Platform','Illumina Genome Analyzer',2,1992),(1840,'Experiment Platform','Illumina Genome Analyzer',2,1993),(1841,'Experiment Platform','Illumina Genome Analyzer',2,1994),(1842,'Experiment Platform','Illumina Genome Analyzer',2,1995),(1843,'Experiment Platform','Illumina Genome Analyzer',2,1996),(1844,'Experiment Platform','Illumina Genome Analyzer',2,1997),(1845,'Experiment Platform','Illumina Genome Analyzer',2,1998),(1846,'Experiment Platform','Illumina Genome Analyzer',2,1999),(1847,'Experiment Platform','Illumina Genome Analyzer',2,2000),(1848,'Experiment Platform','Illumina Genome Analyzer',2,2001),(1849,'Experiment Platform','Illumina Genome Analyzer',2,2002),(1850,'Experiment Platform','Illumina Genome Analyzer',2,2003),(1851,'Experiment Platform','Illumina Genome Analyzer',2,2004),(1852,'Experiment Platform','Illumina Genome Analyzer',2,2005),(1853,'Experiment Platform','Illumina Genome Analyzer',2,2006),(1854,'Experiment Platform','Illumina Genome Analyzer',2,2007),(1855,'Experiment Platform','Illumina Genome Analyzer',2,2008),(1856,'Experiment Platform','Illumina Genome Analyzer',2,2009);

INSERT INTO dbo.gpAnnotationProperty
(idAnnotationProperty,
name,
value,
idProperty,
idAnnotation
) VALUES
(1857,'Experiment Platform','Illumina Genome Analyzer',2,2010),(1858,'Experiment Platform','Illumina Genome Analyzer',2,2011),(1859,'Experiment Platform','Illumina Genome Analyzer',2,2012),(1860,'Experiment Platform','Illumina Genome Analyzer',2,2013),(1861,'Experiment Platform','Illumina Genome Analyzer',2,2014),(1862,'Experiment Platform','Illumina Genome Analyzer',2,2015),(1863,'Experiment Platform','Illumina Genome Analyzer',2,2016),(1864,'Experiment Platform','Illumina Genome Analyzer',2,2017),(1865,'Experiment Platform','Illumina Genome Analyzer',2,2018),(1866,'Experiment Platform','Illumina Genome Analyzer',2,2019),(1867,'Experiment Platform','Illumina Genome Analyzer',2,2020),(1868,'Experiment Platform','Illumina Genome Analyzer',2,2021),(1869,'Experiment Platform','Illumina Genome Analyzer',2,2022),(1870,'Experiment Platform','Illumina Genome Analyzer',2,2023),(1871,'Experiment Platform','Illumina Genome Analyzer',2,2024),(1872,'Experiment Platform','Illumina Genome Analyzer',2,2025),(1873,'Experiment Platform','Illumina Genome Analyzer',2,2026),(1874,'Experiment Platform','Illumina Genome Analyzer',2,2027),(1875,'Experiment Platform','Illumina Genome Analyzer',2,2028),(1876,'Experiment Platform','Illumina Genome Analyzer',2,2029),(1877,'Experiment Platform','Illumina Genome Analyzer',2,2030),(1878,'Experiment Platform','Illumina Genome Analyzer',2,2031),(1879,'Experiment Platform','Illumina Genome Analyzer',2,2032),(1880,'Experiment Platform','Illumina Genome Analyzer',2,2033),(1881,'Experiment Platform','Illumina Genome Analyzer',2,2034),(1882,'Experiment Platform','Illumina Genome Analyzer',2,2035),(1883,'Experiment Platform','Illumina Genome Analyzer',2,2036),(1884,'Experiment Platform','Illumina Genome Analyzer',2,2037),(1885,'Experiment Platform','Illumina Genome Analyzer',2,2038),(1886,'Experiment Platform','Illumina Genome Analyzer',2,2039),(1887,'Experiment Platform','Illumina Genome Analyzer',2,2040),(1888,'Experiment Platform','Illumina Genome Analyzer',2,2041),(1889,'Experiment Platform','Illumina Genome Analyzer',2,2042),(1890,'Experiment Platform','Illumina Genome Analyzer',2,2043),(1891,'Experiment Platform','Illumina Genome Analyzer',2,2044),(1892,'Experiment Platform','Illumina Genome Analyzer',2,2045),(1893,'Experiment Platform','Illumina Genome Analyzer',2,2046),(1894,'Experiment Platform','Illumina Genome Analyzer',2,2047),(1895,'Experiment Platform','Illumina Genome Analyzer',2,2048),(1896,'Experiment Platform','Illumina Genome Analyzer',2,2049),(1897,'Experiment Platform','Illumina Genome Analyzer',2,2050),(1898,'Experiment Platform','Illumina Genome Analyzer',2,2051),(1899,'Experiment Platform','Illumina Genome Analyzer',2,2052),(1900,'Experiment Platform','Illumina Genome Analyzer',2,2074),(1901,'Experiment Platform','Illumina Genome Analyzer',2,2075),(1902,'Experiment Platform','Illumina Genome Analyzer',2,2076),(1903,'Experiment Platform','Illumina Genome Analyzer',2,2077),(1904,'Experiment Platform','Illumina Genome Analyzer',2,2078),(1905,'Experiment Platform','Illumina Genome Analyzer',2,2079),(1906,'Experiment Platform','Illumina Genome Analyzer',2,2080),(1907,'Experiment Platform','Illumina Genome Analyzer',2,2081),(1908,'Experiment Platform','Illumina Genome Analyzer',2,2082),(1909,'Experiment Platform','Illumina Genome Analyzer',2,2083),(1910,'Experiment Platform','Illumina Genome Analyzer',2,2084),(1911,'Experiment Platform','Illumina Genome Analyzer',2,2085),(1912,'Experiment Platform','Illumina Genome Analyzer',2,2086),(1913,'Experiment Platform','Illumina Genome Analyzer',2,2087),(1914,'Experiment Platform','Illumina Genome Analyzer',2,2088),(1915,'Experiment Platform','Illumina Genome Analyzer',2,2089),(1916,'Experiment Platform','Illumina Genome Analyzer',2,2090),(1917,'Experiment Platform','Illumina Genome Analyzer',2,2091),(1918,'Experiment Platform','Illumina Genome Analyzer',2,2092),(1919,'Experiment Platform','Illumina Genome Analyzer',2,2093),(1920,'Experiment Platform','Illumina Genome Analyzer',2,2094),(1921,'Experiment Platform','Illumina Genome Analyzer',2,2095),(1922,'Experiment Platform','Illumina Genome Analyzer',2,2096),(1923,'Experiment Platform','Illumina Genome Analyzer',2,2097),(1924,'Experiment Platform','Illumina Genome Analyzer',2,2098),(1925,'Experiment Platform','Illumina Genome Analyzer',2,2099),(1926,'Experiment Platform','Illumina Genome Analyzer',2,2100),(1927,'Experiment Platform','Illumina Genome Analyzer',2,2101),(1928,'Experiment Platform','Illumina Genome Analyzer',2,2102),(1929,'Experiment Platform','Illumina Genome Analyzer',2,2103),(1930,'Experiment Platform','Illumina Genome Analyzer',2,2104),(1931,'Experiment Platform','Illumina Genome Analyzer',2,2105),(1932,'Experiment Platform','Illumina Genome Analyzer',2,2106),(1933,'Experiment Platform','Illumina Genome Analyzer',2,2107),(1934,'Experiment Platform','Illumina Genome Analyzer',2,2108),(1935,'Experiment Platform','Illumina Genome Analyzer',2,2109),(1936,'Experiment Platform','Illumina Genome Analyzer',2,2110),(1937,'Experiment Platform','Illumina Genome Analyzer',2,2111),(1938,'Experiment Platform','Illumina Genome Analyzer',2,2112),(1939,'Experiment Platform','Illumina Genome Analyzer',2,2113),(1940,'Experiment Platform','Illumina Genome Analyzer',2,2114),(1941,'Experiment Platform','Illumina Genome Analyzer',2,2115),(1942,'Experiment Platform','Illumina Genome Analyzer',2,2116),(1943,'Experiment Platform','Illumina Genome Analyzer',2,2117),(1944,'Experiment Platform','Illumina Genome Analyzer',2,2118),(1945,'Experiment Platform','Illumina Genome Analyzer',2,2119),(1946,'Experiment Platform','Illumina Genome Analyzer',2,2120),(1947,'Experiment Platform','Illumina Genome Analyzer',2,2121),(1948,'Experiment Platform','Illumina Genome Analyzer',2,2122),(1949,'Experiment Platform','Illumina Genome Analyzer',2,2123),(1950,'Experiment Platform','Illumina Genome Analyzer',2,2124),(1951,'Experiment Platform','Illumina Genome Analyzer',2,2125),(1952,'Experiment Platform','Illumina Genome Analyzer',2,2126),(1953,'Experiment Platform','Illumina Genome Analyzer',2,2127),(1954,'Experiment Platform','Illumina Genome Analyzer',2,2128),(1955,'Experiment Platform','Illumina Genome Analyzer',2,2129),(1956,'Experiment Platform','Illumina Genome Analyzer',2,2130),(1957,'Experiment Platform','Illumina Genome Analyzer',2,2131),(1958,'Experiment Platform','Illumina Genome Analyzer',2,2132),(1959,'Experiment Platform','Illumina Genome Analyzer',2,2134),(1960,'Experiment Platform','Illumina Genome Analyzer',2,2135),(1961,'Experiment Platform','Illumina Genome Analyzer',2,2136),(1962,'Experiment Platform','Illumina Genome Analyzer',2,2137),(1963,'Experiment Platform','Illumina Genome Analyzer',2,2138),(1964,'Experiment Platform','Illumina Genome Analyzer',2,2139),(1965,'Experiment Platform','Illumina Genome Analyzer',2,2141),(1966,'Experiment Platform','Illumina Genome Analyzer',2,2142),(1967,'Experiment Platform','Illumina Genome Analyzer',2,2143),(1968,'Experiment Platform','Illumina Genome Analyzer',2,2144),(1969,'Experiment Platform','Illumina Genome Analyzer',2,2145),(1970,'Experiment Platform','Illumina Genome Analyzer',2,2146),(1971,'Experiment Platform','Illumina Genome Analyzer',2,2147),(1972,'Experiment Platform','Illumina Genome Analyzer',2,2148),(1973,'Experiment Platform','Illumina Genome Analyzer',2,2149),(1974,'Experiment Platform','Illumina Genome Analyzer',2,2150),(1975,'Experiment Platform','Illumina Genome Analyzer',2,2151),(1976,'Experiment Platform','Illumina Genome Analyzer',2,2152),(1977,'Experiment Platform','Illumina Genome Analyzer',2,2153),(1978,'Experiment Platform','Illumina Genome Analyzer',2,2154),(1979,'Experiment Platform','Illumina Genome Analyzer',2,2155),(1980,'Experiment Platform','Illumina Genome Analyzer',2,2160),(1981,'Experiment Platform','Illumina Genome Analyzer',2,2161),(1982,'Experiment Platform','Illumina Genome Analyzer',2,2169),(1983,'Experiment Platform','Illumina Genome Analyzer',2,2170),(1984,'Experiment Platform','Illumina Genome Analyzer',2,2171),(1985,'Experiment Platform','Illumina Genome Analyzer',2,2172),(1986,'Experiment Platform','Illumina Genome Analyzer',2,2173),(1987,'Experiment Platform','Illumina Genome Analyzer',2,2174),(1988,'Experiment Platform','Illumina Genome Analyzer',2,2175),(1989,'Experiment Platform','Illumina Genome Analyzer',2,2176),(1990,'Experiment Platform','Illumina Genome Analyzer',2,2177),(1991,'Experiment Platform','Illumina Genome Analyzer',2,2178),(1992,'Experiment Platform','Illumina Genome Analyzer',2,2179),(1993,'Experiment Platform','Illumina Genome Analyzer',2,2180),(1994,'Experiment Platform','Illumina Genome Analyzer',2,2181),(1995,'Experiment Platform','Illumina Genome Analyzer',2,2182),(1996,'Experiment Platform','Illumina Genome Analyzer',2,2183),(1997,'Experiment Platform','Illumina Genome Analyzer',2,2211),(1998,'Experiment Platform','Illumina Genome Analyzer',2,2212),(1999,'Experiment Platform','Illumina Genome Analyzer',2,2213),(2000,'Experiment Platform','Illumina Genome Analyzer',2,2214),(2001,'Experiment Platform','Illumina Genome Analyzer',2,2215),(2002,'Experiment Platform','Illumina Genome Analyzer',2,2216),(2003,'Experiment Platform','Illumina Genome Analyzer',2,2217),(2004,'Experiment Platform','Illumina Genome Analyzer',2,2218),(2005,'Experiment Platform','Illumina Genome Analyzer',2,2219),(2006,'Experiment Platform','Illumina Genome Analyzer',2,2220),(2007,'Experiment Platform','Illumina Genome Analyzer',2,2221),(2008,'Experiment Platform','Illumina Genome Analyzer',2,2222),(2009,'Experiment Platform','Illumina Genome Analyzer',2,2223),(2010,'Experiment Platform','Illumina Genome Analyzer',2,2224),(2011,'Experiment Platform','Illumina Genome Analyzer',2,2225),(2012,'Experiment Platform','Illumina Genome Analyzer',2,2226),(2013,'Experiment Platform','Illumina Genome Analyzer',2,2228),(2014,'Experiment Platform','Illumina Genome Analyzer',2,2229),(2015,'Experiment Platform','Illumina Genome Analyzer',2,2230),(2016,'Experiment Platform','Illumina Genome Analyzer',2,2231),(2017,'Experiment Platform','Illumina Genome Analyzer',2,2234),(2018,'Experiment Platform','Illumina Genome Analyzer',2,2235),(2019,'Experiment Platform','Illumina Genome Analyzer',2,2236),(2020,'Experiment Platform','Illumina Genome Analyzer',2,2239),(2021,'Experiment Platform','Illumina Genome Analyzer',2,2240),(2022,'Experiment Platform','Illumina Genome Analyzer',2,2241),(2023,'Experiment Platform','Illumina Genome Analyzer',2,2263),(2024,'Experiment Platform','Illumina Genome Analyzer',2,2264),(2025,'Experiment Platform','Illumina Genome Analyzer',2,2265),(2026,'Experiment Platform','Illumina Genome Analyzer',2,2273),(2027,'Experiment Platform','Illumina Genome Analyzer',2,2274),(2028,'Experiment Platform','Illumina Genome Analyzer',2,2275),(2029,'Experiment Platform','Illumina Genome Analyzer',2,2276),(2030,'Experiment Platform','Illumina Genome Analyzer',2,2277),(2031,'Experiment Platform','Illumina Genome Analyzer',2,2278),(2032,'Experiment Platform','Illumina Genome Analyzer',2,2279),(2033,'Experiment Platform','Illumina Genome Analyzer',2,2280),(2034,'Experiment Platform','Illumina Genome Analyzer',2,2281),(2035,'Experiment Platform','Illumina Genome Analyzer',2,2282),(2036,'Experiment Platform','Illumina Genome Analyzer',2,2283),(2037,'Experiment Platform','Illumina Genome Analyzer',2,2284),(2038,'Experiment Platform','Illumina Genome Analyzer',2,2298),(2039,'Experiment Platform','Illumina Genome Analyzer',2,2299),(2040,'Experiment Platform','Illumina Genome Analyzer',2,2350),(2041,'Experiment Platform','Illumina Genome Analyzer',2,2353),(2042,'Experiment Platform','Illumina Genome Analyzer',2,2354),(2043,'Experiment Platform','Illumina Genome Analyzer',2,2355),(2044,'Experiment Platform','Illumina Genome Analyzer',2,2356),(2045,'Experiment Platform','Illumina Genome Analyzer',2,2357),(2046,'Experiment Platform','Illumina Genome Analyzer',2,2358),(2047,'Experiment Platform','Illumina Genome Analyzer',2,2359),(2048,'Experiment Platform','Illumina Genome Analyzer',2,2360),(2049,'Experiment Platform','Illumina Genome Analyzer',2,2361),(2050,'Experiment Platform','Illumina Genome Analyzer',2,2362),(2051,'Experiment Platform','Illumina Genome Analyzer',2,2363),(2052,'Experiment Platform','Illumina Genome Analyzer',2,2364),(2053,'Experiment Platform','Illumina Genome Analyzer',2,2365),(2054,'Experiment Platform','Illumina Genome Analyzer',2,2366),(2055,'Experiment Platform','Illumina Genome Analyzer',2,2367),(2056,'Experiment Platform','Illumina Genome Analyzer',2,2368),(2058,'Experiment Platform','Illumina Genome Analyzer',2,2377),(2067,'Experiment Platform','Illumina Genome Analyzer',2,2387),(2068,'Experiment Platform','Illumina Genome Analyzer',2,2388),(2069,'Experiment Platform','Illumina Genome Analyzer',2,2389),(2070,'Experiment Platform','Illumina Genome Analyzer',2,2390),(2071,'Experiment Platform','Illumina Genome Analyzer',2,2391),(2072,'Experiment Platform','Illumina Genome Analyzer',2,2392),(2073,'Experiment Platform','Illumina Genome Analyzer',2,2393),(2074,'Experiment Platform','Illumina Genome Analyzer',2,2394),(2075,'Experiment Platform','Illumina Genome Analyzer',2,2395),(2076,'Experiment Platform','Illumina Genome Analyzer',2,2396),(2077,'Experiment Platform','Illumina Genome Analyzer',2,2397),(2078,'Experiment Platform','Illumina Genome Analyzer',2,2398),(2079,'Experiment Platform','Illumina Genome Analyzer',2,2401),(2080,'Experiment Platform','Illumina Genome Analyzer',2,2402),(2081,'Experiment Platform','Illumina Genome Analyzer',2,2403),(2082,'Experiment Platform','Illumina Genome Analyzer',2,2404),(2083,'Experiment Platform','Illumina Genome Analyzer',2,2405),(2084,'Experiment Platform','Illumina Genome Analyzer',2,2406),(2085,'Experiment Platform','Illumina Genome Analyzer',2,2407),(2086,'Experiment Platform','Illumina Genome Analyzer',2,2408),(2087,'Experiment Platform','Illumina Genome Analyzer',2,2409),(2088,'Experiment Platform','Illumina Genome Analyzer',2,2410),(2089,'Experiment Platform','Illumina Genome Analyzer',2,2411),(2090,'Experiment Platform','Illumina Genome Analyzer',2,2412),(2091,'Experiment Platform','Illumina Genome Analyzer',2,2413),(2092,'Experiment Platform','Illumina Genome Analyzer',2,2414),(2093,'Experiment Platform','Illumina Genome Analyzer',2,2415),(2094,'Experiment Platform','Illumina Genome Analyzer',2,2416),(2095,'Experiment Platform','Illumina Genome Analyzer',2,2417),(2096,'Experiment Platform','Illumina Genome Analyzer',2,2418),(2097,'Experiment Platform','Illumina Genome Analyzer',2,2419),(2098,'Experiment Platform','Illumina Genome Analyzer',2,2420),(2099,'Experiment Platform','Illumina Genome Analyzer',2,2421),(2100,'Experiment Platform','Illumina Genome Analyzer',2,2422),(2101,'Experiment Platform','Illumina Genome Analyzer',2,2423),(2102,'Experiment Platform','Illumina Genome Analyzer',2,2426),(2103,'Experiment Platform','Illumina Genome Analyzer',2,2427),(2104,'Experiment Platform','Illumina Genome Analyzer',2,2428),(2105,'Experiment Platform','Illumina Genome Analyzer',2,2429),(2106,'Experiment Platform','Illumina Genome Analyzer',2,2430),(2107,'Experiment Platform','Illumina Genome Analyzer',2,2431),(2108,'Experiment Platform','Illumina Genome Analyzer',2,2432),(2109,'Experiment Platform','Illumina Genome Analyzer',2,2433),(2110,'Experiment Platform','Illumina Genome Analyzer',2,2435),(2111,'Experiment Platform','Illumina Genome Analyzer',2,2436),(2112,'Experiment Platform','Illumina Genome Analyzer',2,2437),(2113,'Experiment Platform','Illumina Genome Analyzer',2,2438),(2114,'Experiment Platform','Illumina Genome Analyzer',2,2439),(2115,'Experiment Platform','Illumina Genome Analyzer',2,2440),(2116,'Experiment Platform','Illumina Genome Analyzer',2,2441),(2117,'Experiment Platform','Illumina Genome Analyzer',2,2442),(2118,'Experiment Platform','Illumina Genome Analyzer',2,2443),(2128,'Experiment Platform','Illumina Genome Analyzer',2,2459),(2129,'Experiment Platform','Illumina Genome Analyzer',2,2460),(2130,'Experiment Platform','Illumina Genome Analyzer',2,2461),(2131,'Experiment Platform','Illumina Genome Analyzer',2,2462),(2132,'Experiment Platform','Illumina Genome Analyzer',2,2463),(2133,'Experiment Platform','Illumina Genome Analyzer',2,2464),(2134,'Experiment Platform','Illumina Genome Analyzer',2,2465),(2135,'Experiment Platform','Illumina Genome Analyzer',2,2466),(2136,'Experiment Platform','Illumina Genome Analyzer',2,2467),(2137,'Experiment Platform','Illumina Genome Analyzer',2,2468),(2138,'Experiment Platform','Illumina Genome Analyzer',2,2469),(2139,'Experiment Platform','Illumina Genome Analyzer',2,2470),(2140,'Experiment Platform','Illumina Genome Analyzer',2,2471),(2141,'Experiment Platform','Illumina Genome Analyzer',2,2472),(2142,'Experiment Platform','Illumina Genome Analyzer',2,2473),(2143,'Experiment Platform','Illumina Genome Analyzer',2,2474),(2144,'Experiment Platform','Illumina Genome Analyzer',2,2475),(2145,'Experiment Platform','Illumina Genome Analyzer',2,2476),(2146,'Experiment Platform','Illumina Genome Analyzer',2,2477),(2147,'Experiment Platform','Illumina Genome Analyzer',2,2478),(2148,'Experiment Platform','Illumina Genome Analyzer',2,2495),(2149,'Experiment Platform','Illumina Genome Analyzer',2,2496),(2150,'Experiment Platform','Illumina Genome Analyzer',2,2497),(2151,'Experiment Platform','Illumina Genome Analyzer',2,2498),(2152,'Experiment Platform','Illumina Genome Analyzer',2,2499),(2153,'Experiment Platform','Illumina Genome Analyzer',2,2501),(2156,'Experiment Platform','Illumina Genome Analyzer',2,2536),(2157,'Experiment Platform','Illumina Genome Analyzer',2,2562),(2158,'Experiment Platform','Illumina Genome Analyzer',2,2592),(2159,'Experiment Platform','Illumina Genome Analyzer',2,2631),(2160,'Experiment Platform','Illumina Genome Analyzer',2,2640),(2161,'Experiment Platform','Illumina Genome Analyzer',2,2779),(2162,'Experiment Platform','Illumina Genome Analyzer',2,2780),(2163,'Experiment Platform','Illumina Genome Analyzer',2,2781),(2164,'Experiment Platform','Illumina Genome Analyzer',2,2782),(2165,'Experiment Platform','Illumina Genome Analyzer',2,2783),(2166,'Experiment Platform','Illumina Genome Analyzer',2,2784),(2167,'Experiment Platform','Illumina Genome Analyzer',2,2785),(2168,'Experiment Platform','Illumina Genome Analyzer',2,2786),(2169,'Experiment Platform','Illumina Genome Analyzer',2,2787),(2170,'Experiment Platform','Illumina Genome Analyzer',2,2788),(2171,'Experiment Platform','Illumina Genome Analyzer',2,2789),(2172,'Experiment Platform','Illumina Genome Analyzer',2,2790),(2173,'Experiment Platform','Illumina Genome Analyzer',2,2791),(2174,'Experiment Platform','Illumina Genome Analyzer',2,2792),(2175,'Experiment Platform','Illumina Genome Analyzer',2,2795),(2176,'Experiment Platform','Illumina Genome Analyzer',2,2797),(2177,'Experiment Platform','Illumina Genome Analyzer',2,2798),(2178,'Experiment Platform','Illumina Genome Analyzer',2,2799),(2179,'Experiment Platform','Illumina Genome Analyzer',2,2800),(2180,'Experiment Platform','Illumina Genome Analyzer',2,2801),(2181,'Experiment Platform','Illumina Genome Analyzer',2,2802),(2182,'Experiment Platform','Illumina Genome Analyzer',2,2816),(2183,'Experiment Platform','Illumina Genome Analyzer',2,2839),(2184,'Experiment Platform','Illumina Genome Analyzer',2,2840),(2185,'Experiment Platform','Illumina Genome Analyzer',2,2841),(2186,'Experiment Platform','Illumina Genome Analyzer',2,2842),(2187,'Experiment Platform','Illumina Genome Analyzer',2,2843),(2188,'Experiment Platform','Illumina Genome Analyzer',2,2844),(2189,'Experiment Platform','Illumina Genome Analyzer',2,2847),(2190,'Experiment Platform','Illumina Genome Analyzer',2,2849),(2191,'Experiment Platform','Illumina Genome Analyzer',2,2850),(2192,'Experiment Platform','Illumina Genome Analyzer',2,2851),(2193,'Experiment Platform','Illumina Genome Analyzer',2,2852),(2194,'Experiment Platform','Illumina Genome Analyzer',2,2853),(2195,'Experiment Platform','Illumina Genome Analyzer',2,2854),(2196,'Experiment Platform','Illumina Genome Analyzer',2,2855),(2197,'Experiment Platform','Illumina Genome Analyzer',2,2856),(2198,'Experiment Platform','Illumina Genome Analyzer',2,2857),(2199,'Experiment Platform','Illumina Genome Analyzer',2,2882),(2200,'Experiment Platform','Illumina Genome Analyzer',2,2892),(2201,'Experiment Platform','Illumina Genome Analyzer',2,2893),(2202,'Experiment Platform','Illumina Genome Analyzer',2,2896),(2203,'Experiment Platform','Illumina Genome Analyzer',2,2897),(2204,'Experiment Platform','Illumina Genome Analyzer',2,2898),(2205,'Experiment Platform','Illumina Genome Analyzer',2,2899),(2206,'Experiment Platform','Illumina Genome Analyzer',2,2908),(2207,'Experiment Platform','Illumina Genome Analyzer',2,2946),(2208,'Experiment Platform','Illumina Genome Analyzer',2,2947),(2209,'Experiment Platform','Illumina Genome Analyzer',2,2952),(2210,'Experiment Platform','Illumina Genome Analyzer',2,2953),(2211,'Experiment Platform','Illumina Genome Analyzer',2,2954),(2212,'Experiment Platform','Illumina Genome Analyzer',2,2962),(2213,'Experiment Platform','Illumina Genome Analyzer',2,2978),(2214,'Experiment Platform','Illumina Genome Analyzer',2,2979),(2215,'Experiment Platform','Illumina Genome Analyzer',2,2980),(2216,'Experiment Platform','Illumina Genome Analyzer',2,2987),(2217,'Experiment Platform','Illumina Genome Analyzer',2,2988),(2218,'Experiment Platform','Illumina Genome Analyzer',2,3014),(2219,'Experiment Platform','Illumina Genome Analyzer',2,3015),(2220,'Experiment Platform','Illumina Genome Analyzer',2,3016),(2221,'Experiment Platform','Illumina Genome Analyzer',2,3017),(2222,'Experiment Platform','Illumina Genome Analyzer',2,3018),(2223,'Experiment Platform','Illumina Genome Analyzer',2,3019),(2224,'Experiment Platform','Illumina Genome Analyzer',2,3076),(2225,'Experiment Platform','Illumina Genome Analyzer',2,3079),(2226,'Experiment Platform','Illumina Genome Analyzer',2,3080),(2227,'Experiment Platform','Illumina Genome Analyzer',2,3102),(2228,'Experiment Platform','Illumina Genome Analyzer',2,3103),(2229,'Experiment Platform','Illumina Genome Analyzer',2,3104),(2230,'Experiment Platform','Illumina Genome Analyzer',2,3115),(2231,'Experiment Platform','Illumina Genome Analyzer',2,3116),(2232,'Experiment Platform','Illumina Genome Analyzer',2,3117),(2233,'Experiment Platform','Illumina Genome Analyzer',2,3118),(2234,'Experiment Platform','Illumina Genome Analyzer',2,3119),(2235,'Experiment Platform','Illumina Genome Analyzer',2,3120),(2236,'Experiment Platform','Illumina Genome Analyzer',2,3121),(2237,'Experiment Platform','Illumina Genome Analyzer',2,3122),(2238,'Experiment Platform','Illumina Genome Analyzer',2,3123),(2239,'Experiment Platform','Illumina Genome Analyzer',2,3124),(2240,'Experiment Platform','Illumina Genome Analyzer',2,3125),(2241,'Experiment Platform','Illumina Genome Analyzer',2,3126),(2242,'Experiment Platform','Illumina Genome Analyzer',2,3127),(2243,'Experiment Platform','Illumina Genome Analyzer',2,3128),(2244,'Experiment Platform','Illumina Genome Analyzer',2,3129),(2245,'Experiment Platform','Illumina Genome Analyzer',2,3130),(2246,'Experiment Platform','Illumina Genome Analyzer',2,3131),(2247,'Experiment Platform','Illumina Genome Analyzer',2,3132),(2248,'Experiment Platform','Illumina Genome Analyzer',2,3133),(2249,'Experiment Platform','Illumina Genome Analyzer',2,3134),(2250,'Experiment Platform','Illumina Genome Analyzer',2,3135),(2251,'Experiment Platform','Illumina Genome Analyzer',2,3136),(2252,'Experiment Platform','Illumina Genome Analyzer',2,3137),(2253,'Experiment Platform','Illumina Genome Analyzer',2,3138),(2254,'Experiment Platform','Illumina Genome Analyzer',2,3139),(2255,'Experiment Platform','Illumina Genome Analyzer',2,3140),(2256,'Experiment Platform','Illumina Genome Analyzer',2,3141),(2257,'Experiment Platform','Illumina Genome Analyzer',2,3142),(2258,'Experiment Platform','Illumina Genome Analyzer',2,3143),(2259,'Experiment Platform','Illumina Genome Analyzer',2,3144),(2260,'Experiment Platform','Illumina Genome Analyzer',2,3145),(2261,'Experiment Platform','Illumina Genome Analyzer',2,3146),(2262,'Experiment Platform','Illumina Genome Analyzer',2,3147),(2263,'Experiment Platform','Illumina Genome Analyzer',2,3148),(2264,'Experiment Platform','Illumina Genome Analyzer',2,3149),(2265,'Experiment Platform','Illumina Genome Analyzer',2,3150),(2266,'Experiment Platform','Illumina Genome Analyzer',2,3151),(2267,'Experiment Platform','Illumina Genome Analyzer',2,3152),(2268,'Experiment Platform','Illumina Genome Analyzer',2,3153),(2269,'Experiment Platform','Illumina Genome Analyzer',2,3154),(2270,'Experiment Platform','Illumina Genome Analyzer',2,3155),(2271,'Experiment Platform','Illumina Genome Analyzer',2,3156),(2272,'Experiment Platform','Illumina Genome Analyzer',2,3157),(2273,'Experiment Platform','Illumina Genome Analyzer',2,3158),(2274,'Experiment Platform','Illumina Genome Analyzer',2,3159),(2275,'Experiment Platform','Illumina Genome Analyzer',2,3160),(2276,'Experiment Platform','Illumina Genome Analyzer',2,3161),(2277,'Experiment Platform','Illumina Genome Analyzer',2,3162),(2278,'Experiment Platform','Illumina Genome Analyzer',2,3163),(2279,'Experiment Platform','Illumina Genome Analyzer',2,3164),(2280,'Experiment Platform','Illumina Genome Analyzer',2,3165),(2281,'Experiment Platform','Illumina Genome Analyzer',2,3166),(2282,'Experiment Platform','Illumina Genome Analyzer',2,3167),(2283,'Experiment Platform','Illumina Genome Analyzer',2,3168),(2284,'Experiment Platform','Illumina Genome Analyzer',2,3169),(2285,'Experiment Platform','Illumina Genome Analyzer',2,3170),(2286,'Experiment Platform','Illumina Genome Analyzer',2,3171),(2287,'Experiment Platform','Illumina Genome Analyzer',2,3172),(2288,'Experiment Platform','Illumina Genome Analyzer',2,3173),(2289,'Experiment Platform','Illumina Genome Analyzer',2,3174),(2290,'Experiment Platform','Illumina Genome Analyzer',2,3175),(2291,'Experiment Platform','Illumina Genome Analyzer',2,3176),(2292,'Experiment Platform','Illumina Genome Analyzer',2,3177),(2293,'Experiment Platform','Illumina Genome Analyzer',2,3178),(2294,'Experiment Platform','Illumina Genome Analyzer',2,3179),(2295,'Experiment Platform','Illumina Genome Analyzer',2,3180),(2296,'Experiment Platform','Illumina Genome Analyzer',2,3181),(2297,'Experiment Platform','Illumina Genome Analyzer',2,3182),(2298,'Experiment Platform','Illumina Genome Analyzer',2,3183),(2299,'Experiment Platform','Illumina Genome Analyzer',2,3184),(2300,'Experiment Platform','Illumina Genome Analyzer',2,3185),(2301,'Experiment Platform','Illumina Genome Analyzer',2,3186),(2302,'Experiment Platform','Illumina Genome Analyzer',2,3187),(2303,'Experiment Platform','Illumina Genome Analyzer',2,3188),(2304,'Experiment Platform','Illumina Genome Analyzer',2,3189),(2305,'Experiment Platform','Illumina Genome Analyzer',2,3190),(2306,'Experiment Platform','Illumina Genome Analyzer',2,3191),(2307,'Experiment Platform','Illumina Genome Analyzer',2,3192),(2308,'Experiment Platform','Illumina Genome Analyzer',2,3193),(2309,'Experiment Platform','Illumina Genome Analyzer',2,3194),(2310,'Experiment Platform','Illumina Genome Analyzer',2,3195),(2311,'Experiment Platform','Illumina Genome Analyzer',2,3196),(2312,'Experiment Platform','Illumina Genome Analyzer',2,3197),(2313,'Experiment Platform','Illumina Genome Analyzer',2,3198),(2314,'Experiment Platform','Illumina Genome Analyzer',2,3199),(2315,'Experiment Platform','Illumina Genome Analyzer',2,3200),(2316,'Experiment Platform','Illumina Genome Analyzer',2,3201),(2317,'Experiment Platform','Illumina Genome Analyzer',2,3202),(2318,'Experiment Platform','Illumina Genome Analyzer',2,3203),(2319,'Experiment Platform','Illumina Genome Analyzer',2,3204);

INSERT INTO dbo.gpAnnotationProperty
(idAnnotationProperty,
name,
value,
idProperty,
idAnnotation
) VALUES
(2320,'Experiment Platform','Illumina Genome Analyzer',2,3205),(2321,'Experiment Platform','Illumina Genome Analyzer',2,3206),(2322,'Experiment Platform','Illumina Genome Analyzer',2,3207),(2323,'Experiment Platform','Illumina Genome Analyzer',2,3208),(2324,'Experiment Platform','Illumina Genome Analyzer',2,3209),(2325,'Experiment Platform','Illumina Genome Analyzer',2,3210),(2326,'Experiment Platform','Illumina Genome Analyzer',2,3211),(2327,'Experiment Platform','Illumina Genome Analyzer',2,3212),(2328,'Experiment Platform','Illumina Genome Analyzer',2,3213),(2329,'Experiment Platform','Illumina Genome Analyzer',2,3214),(2330,'Experiment Platform','Illumina Genome Analyzer',2,3215),(2331,'Experiment Platform','Illumina Genome Analyzer',2,3216),(2332,'Experiment Platform','Illumina Genome Analyzer',2,3217),(2333,'Experiment Platform','Illumina Genome Analyzer',2,3218),(2334,'Experiment Platform','Illumina Genome Analyzer',2,3219),(2335,'Experiment Platform','Illumina Genome Analyzer',2,3220),(2336,'Experiment Platform','Illumina Genome Analyzer',2,3221),(2337,'Experiment Platform','Illumina Genome Analyzer',2,3222),(2338,'Experiment Platform','Illumina Genome Analyzer',2,3223),(2339,'Experiment Platform','Illumina Genome Analyzer',2,3224),(2340,'Experiment Platform','Illumina Genome Analyzer',2,3225),(2341,'Experiment Platform','Illumina Genome Analyzer',2,3226),(2342,'Experiment Platform','Illumina Genome Analyzer',2,3227),(2343,'Experiment Platform','Illumina Genome Analyzer',2,3228),(2344,'Experiment Platform','Illumina Genome Analyzer',2,3229),(2345,'Experiment Platform','Illumina Genome Analyzer',2,3230),(2346,'Experiment Platform','Illumina Genome Analyzer',2,3231),(2347,'Experiment Platform','Illumina Genome Analyzer',2,3232),(2348,'Experiment Platform','Illumina Genome Analyzer',2,3233),(2349,'Experiment Platform','Illumina Genome Analyzer',2,3234),(2350,'Experiment Platform','Illumina Genome Analyzer',2,3235),(2351,'Experiment Platform','Illumina Genome Analyzer',2,3236),(2352,'Experiment Platform','Illumina Genome Analyzer',2,3237),(2353,'Experiment Platform','Illumina Genome Analyzer',2,3250),(2354,'Experiment Platform','Illumina Genome Analyzer',2,3251),(2355,'Experiment Platform','Illumina Genome Analyzer',2,3252),(2356,'Experiment Platform','Illumina Genome Analyzer',2,3253),(2357,'Experiment Platform','Illumina Genome Analyzer',2,3254),(2358,'Experiment Platform','Illumina Genome Analyzer',2,3257),(2359,'Experiment Platform','Illumina Genome Analyzer',2,3258),(2360,'Experiment Platform','Illumina Genome Analyzer',2,3259),(2361,'Experiment Platform','Illumina Genome Analyzer',2,3260),(2362,'Experiment Platform','Illumina Genome Analyzer',2,3261),(2363,'Experiment Platform','Illumina Genome Analyzer',2,3262),(2364,'Experiment Platform','Illumina Genome Analyzer',2,3263),(2365,'Experiment Platform','Illumina Genome Analyzer',2,3264),(2366,'Experiment Platform','Illumina Genome Analyzer',2,3265),(2367,'Experiment Platform','Illumina Genome Analyzer',2,3266),(2368,'Experiment Platform','Illumina Genome Analyzer',2,3267),(2369,'Experiment Platform','Illumina Genome Analyzer',2,3268),(2370,'Experiment Platform','Illumina Genome Analyzer',2,3269),(2371,'Experiment Platform','Illumina Genome Analyzer',2,3270),(2372,'Experiment Platform','Illumina Genome Analyzer',2,3271),(2373,'Experiment Platform','Illumina Genome Analyzer',2,3272),(2374,'Experiment Platform','Illumina Genome Analyzer',2,3275),(2375,'Experiment Platform','Illumina Genome Analyzer',2,3276),(2376,'Experiment Platform','Illumina Genome Analyzer',2,3286),(2377,'Experiment Platform','Illumina Genome Analyzer',2,3287),(2378,'Experiment Platform','Illumina Genome Analyzer',2,3288),(2379,'Experiment Platform','Illumina Genome Analyzer',2,3289),(2380,'Experiment Platform','Illumina Genome Analyzer',2,3290),(2381,'Experiment Platform','Illumina Genome Analyzer',2,3291),(2382,'Experiment Platform','Illumina Genome Analyzer',2,3292),(2383,'Experiment Platform','Illumina Genome Analyzer',2,3293),(2384,'Experiment Platform','Illumina Genome Analyzer',2,3294),(2385,'Experiment Platform','Illumina Genome Analyzer',2,3295),(2386,'Experiment Platform','Illumina Genome Analyzer',2,3296),(2387,'Experiment Platform','Illumina Genome Analyzer',2,3297),(2388,'Experiment Platform','Illumina Genome Analyzer',2,3298),(2389,'Experiment Platform','Illumina Genome Analyzer',2,3299),(2390,'Experiment Platform','Illumina Genome Analyzer',2,3300),(2391,'Experiment Platform','Illumina Genome Analyzer',2,3301),(2392,'Experiment Platform','Illumina Genome Analyzer',2,3302),(2393,'Experiment Platform','Illumina Genome Analyzer',2,3303),(2394,'Experiment Platform','Illumina Genome Analyzer',2,3304),(2395,'Experiment Platform','Illumina Genome Analyzer',2,3305),(2396,'Experiment Platform','Illumina Genome Analyzer',2,3306),(2397,'Experiment Platform','Illumina Genome Analyzer',2,3307),(2398,'Experiment Platform','Illumina Genome Analyzer',2,3308),(2399,'Experiment Platform','Illumina Genome Analyzer',2,3309),(2400,'Experiment Platform','Illumina Genome Analyzer',2,3310),(2401,'Experiment Platform','Illumina Genome Analyzer',2,3311),(2402,'Experiment Platform','Illumina Genome Analyzer',2,3312),(2403,'Experiment Platform','Illumina Genome Analyzer',2,3313),(2404,'Experiment Platform','Illumina Genome Analyzer',2,3314),(2405,'Experiment Platform','Illumina Genome Analyzer',2,3315),(2406,'Experiment Platform','Illumina Genome Analyzer',2,3316),(2407,'Experiment Platform','Illumina Genome Analyzer',2,3317),(2408,'Experiment Platform','Illumina Genome Analyzer',2,3324),(2409,'Experiment Platform','Illumina Genome Analyzer',2,3325),(2410,'Experiment Platform','Illumina Genome Analyzer',2,3326),(2411,'Experiment Platform','Illumina Genome Analyzer',2,3327),(2412,'Experiment Platform','Illumina Genome Analyzer',2,3328),(2413,'Experiment Platform','Illumina Genome Analyzer',2,3329),(2414,'Experiment Platform','Illumina Genome Analyzer',2,3330),(2415,'Experiment Platform','Illumina Genome Analyzer',2,3331),(2416,'Experiment Platform','Illumina Genome Analyzer',2,3332),(2417,'Experiment Platform','Illumina Genome Analyzer',2,3333),(2418,'Experiment Platform','Illumina Genome Analyzer',2,3334),(2419,'Experiment Platform','Illumina Genome Analyzer',2,3335),(2420,'Experiment Platform','Illumina Genome Analyzer',2,3336),(2421,'Experiment Platform','Illumina Genome Analyzer',2,3337),(2422,'Experiment Platform','Illumina Genome Analyzer',2,3338),(2423,'Experiment Platform','Illumina Genome Analyzer',2,3339),(2424,'Experiment Platform','Illumina Genome Analyzer',2,3340),(2425,'Experiment Platform','Illumina Genome Analyzer',2,3341),(2426,'Experiment Platform','Illumina Genome Analyzer',2,3342),(2427,'Experiment Platform','Illumina Genome Analyzer',2,3343),(2428,'Experiment Platform','Illumina Genome Analyzer',2,3344),(2429,'Experiment Platform','Illumina Genome Analyzer',2,3345),(2430,'Experiment Platform','Illumina Genome Analyzer',2,3346),(2431,'Experiment Platform','Illumina Genome Analyzer',2,3347),(2432,'Experiment Platform','Illumina Genome Analyzer',2,3348),(2433,'Experiment Platform','Illumina Genome Analyzer',2,3349),(2434,'Experiment Platform','Illumina Genome Analyzer',2,3350),(2435,'Experiment Platform','Illumina Genome Analyzer',2,3351),(2436,'Experiment Platform','Illumina Genome Analyzer',2,3352);

INSERT INTO dbo.gpAnnotationProperty
(idAnnotationProperty,
name,
value,
idProperty,
idAnnotation
) VALUES
(2437,'Experiment Platform','Illumina Genome Analyzer',2,3353),(2438,'Experiment Platform','Illumina Genome Analyzer',2,3354),(2439,'Experiment Platform','Illumina Genome Analyzer',2,3355),(2440,'Experiment Platform','Illumina Genome Analyzer',2,3356),(2441,'Experiment Platform','Illumina Genome Analyzer',2,3357),(2442,'Experiment Platform','Illumina Genome Analyzer',2,3358),(2443,'Experiment Platform','Illumina Genome Analyzer',2,3359),(2444,'Experiment Platform','Illumina Genome Analyzer',2,3360),(2445,'Experiment Platform','Illumina Genome Analyzer',2,3361),(2446,'Experiment Platform','Illumina Genome Analyzer',2,3362),(2447,'Experiment Platform','Illumina Genome Analyzer',2,3363),(2448,'Experiment Platform','Illumina Genome Analyzer',2,3364),(2449,'Experiment Platform','Illumina Genome Analyzer',2,3365),(2450,'Experiment Platform','Illumina Genome Analyzer',2,3366),(2451,'Experiment Platform','Illumina Genome Analyzer',2,3367),(2452,'Experiment Platform','Illumina Genome Analyzer',2,3368),(2453,'Experiment Platform','Illumina Genome Analyzer',2,3369),(2454,'Experiment Platform','Illumina Genome Analyzer',2,3370),(2455,'Experiment Platform','Illumina Genome Analyzer',2,3371),(2456,'Experiment Platform','Illumina Genome Analyzer',2,3381),(2457,'Experiment Platform','Illumina Genome Analyzer',2,3382),(2458,'Experiment Platform','Illumina Genome Analyzer',2,3409),(2459,'Experiment Platform','Illumina Genome Analyzer',2,3410),(2460,'Experiment Platform','Illumina Genome Analyzer',2,3411),(2461,'Experiment Platform','Illumina Genome Analyzer',2,3412),(2462,'Experiment Platform','Illumina Genome Analyzer',2,3413),(2463,'Experiment Platform','Illumina Genome Analyzer',2,3414),(2464,'Experiment Platform','Illumina Genome Analyzer',2,3420),(2465,'Experiment Platform','Illumina Genome Analyzer',2,3421),(2466,'Experiment Platform','Illumina Genome Analyzer',2,3422),(2467,'Experiment Platform','Illumina Genome Analyzer',2,3423),(2468,'Experiment Platform','Illumina Genome Analyzer',2,3424),(2469,'Experiment Platform','Illumina Genome Analyzer',2,3425),(2470,'Experiment Platform','Illumina Genome Analyzer',2,3430),(2471,'Experiment Platform','Illumina Genome Analyzer',2,3431),(2472,'Experiment Platform','Illumina Genome Analyzer',2,3432),(2473,'Experiment Platform','Illumina Genome Analyzer',2,3433),(2474,'Experiment Platform','Illumina Genome Analyzer',2,3434),(2475,'Experiment Platform','Illumina Genome Analyzer',2,3435),(2476,'Experiment Platform','Illumina Genome Analyzer',2,3439),(2477,'Experiment Platform','Illumina Genome Analyzer',2,3440),(2478,'Experiment Platform','Illumina Genome Analyzer',2,3443),(2479,'Experiment Platform','Illumina Genome Analyzer',2,3444),(2480,'Experiment Platform','Illumina Genome Analyzer',2,3445),(2481,'Experiment Platform','Illumina Genome Analyzer',2,3446),(2482,'Experiment Platform','Illumina Genome Analyzer',2,3447),(2483,'Experiment Platform','Illumina Genome Analyzer',2,3448),(2484,'Experiment Platform','Illumina Genome Analyzer',2,3449),(2485,'Experiment Platform','Illumina Genome Analyzer',2,3450),(2486,'Experiment Platform','Illumina Genome Analyzer',2,3451),(2487,'Experiment Platform','Illumina Genome Analyzer',2,3452),(2488,'Experiment Platform','Illumina Genome Analyzer',2,3453),(2489,'Experiment Platform','Illumina Genome Analyzer',2,3461),(2498,'Experiment Platform','Illumina Genome Analyzer',2,3477),(2499,'Experiment Platform','Illumina Genome Analyzer',2,3478),(2500,'Experiment Platform','Illumina Genome Analyzer',2,3479),(2501,'Experiment Platform','Illumina Genome Analyzer',2,3480),(2502,'Experiment Platform','Illumina Genome Analyzer',2,3481),(2503,'Experiment Platform','Illumina Genome Analyzer',2,3482),(2504,'Experiment Platform','Illumina Genome Analyzer',2,3483),(2505,'Experiment Platform','Illumina Genome Analyzer',2,3484),(2506,'Experiment Platform','Illumina Genome Analyzer',2,3486),(2507,'Experiment Platform','Illumina Genome Analyzer',2,3487),(2508,'Experiment Platform','Illumina Genome Analyzer',2,3527),(2509,'Experiment Platform','Illumina Genome Analyzer',2,3531),(2510,'Experiment Platform','Illumina Genome Analyzer',2,3554),(2511,'Experiment Platform','Illumina Genome Analyzer',2,3555),(2512,'Experiment Platform','Illumina Genome Analyzer',2,3556),(2513,'Experiment Platform','Illumina Genome Analyzer',2,3557),(2514,'Experiment Platform','Illumina Genome Analyzer',2,3558),(2515,'Experiment Platform','Illumina Genome Analyzer',2,3559),(2516,'Experiment Platform','Illumina Genome Analyzer',2,3560),(2517,'Experiment Platform','Illumina Genome Analyzer',2,3561),(2522,'Experiment Platform','Illumina Genome Analyzer',2,3566),(2523,'Experiment Platform','Illumina Genome Analyzer',2,3567),(2524,'Experiment Platform','Illumina Genome Analyzer',2,3568),(2525,'Experiment Platform','Illumina Genome Analyzer',2,3569),(2526,'Experiment Platform','Illumina Genome Analyzer',2,3570),(2527,'Experiment Platform','Illumina Genome Analyzer',2,3571),(2528,'Experiment Platform','Illumina Genome Analyzer',2,3572),(2529,'Experiment Platform','Illumina Genome Analyzer',2,3573),(2530,'Experiment Platform','Illumina Genome Analyzer',2,3574),(2531,'Experiment Platform','Illumina Genome Analyzer',2,3575),(2532,'Experiment Platform','Illumina Genome Analyzer',2,3576),(2533,'Experiment Platform','Illumina Genome Analyzer',2,3577),(2534,'Experiment Platform','Illumina Genome Analyzer',2,3578),(2535,'Experiment Platform','Illumina Genome Analyzer',2,3579),(2536,'Experiment Platform','Illumina Genome Analyzer',2,3580),(2537,'Experiment Platform','Illumina Genome Analyzer',2,3581),(2538,'Experiment Platform','Illumina Genome Analyzer',2,3582),(2539,'Experiment Platform','Illumina Genome Analyzer',2,3583),(2540,'Experiment Platform','Illumina Genome Analyzer',2,3584),(2541,'Experiment Platform','Illumina Genome Analyzer',2,3585),(2542,'Experiment Platform','Illumina Genome Analyzer',2,3586),(2543,'Experiment Platform','Illumina Genome Analyzer',2,3587),(2544,'Experiment Platform','Illumina Genome Analyzer',2,3588),(2545,'Experiment Platform','Illumina Genome Analyzer',2,3589),(2546,'Experiment Platform','Illumina Genome Analyzer',2,3596),(2547,'Experiment Platform','Illumina Genome Analyzer',2,3597),(2548,'Experiment Platform','Illumina Genome Analyzer',2,3598),(2549,'Experiment Platform','Illumina Genome Analyzer',2,3599),(2550,'Experiment Platform','Illumina Genome Analyzer',2,3600),(2551,'Experiment Platform','Illumina Genome Analyzer',2,3601),(2552,'Experiment Platform','Illumina Genome Analyzer',2,3602),(2553,'Experiment Platform','Illumina Genome Analyzer',2,3603),(2554,'Experiment Platform','Illumina Genome Analyzer',2,3604),(2555,'Experiment Platform','Illumina Genome Analyzer',2,3605),(2556,'Experiment Platform','Illumina Genome Analyzer',2,3606),(2557,'Experiment Platform','Illumina Genome Analyzer',2,3607),(2558,'Experiment Platform','Illumina Genome Analyzer',2,3608),(2559,'Experiment Platform','Illumina Genome Analyzer',2,3609),(2560,'Experiment Platform','Illumina Genome Analyzer',2,3610),(2561,'Experiment Platform','Illumina Genome Analyzer',2,3611),(2562,'Experiment Platform','Illumina Genome Analyzer',2,3612),(2563,'Experiment Platform','Illumina Genome Analyzer',2,3613),(2564,'Experiment Platform','Illumina Genome Analyzer',2,3614),(2565,'Experiment Platform','Illumina Genome Analyzer',2,3615),(2566,'Experiment Platform','Illumina Genome Analyzer',2,3616),(2567,'Experiment Platform','Illumina Genome Analyzer',2,3617),(2568,'Experiment Platform','Illumina Genome Analyzer',2,3618),(2569,'Experiment Platform','Illumina Genome Analyzer',2,3619),(2570,'Experiment Platform','Agilent Microarray',2,1285),(2571,'Experiment Platform','Agilent Microarray',2,1286),(2572,'Experiment Platform','Agilent Microarray',2,1287),(2573,'Experiment Platform','Agilent Microarray',2,1288),(2574,'Experiment Platform','Agilent Microarray',2,1294),(2575,'Experiment Platform','Agilent Microarray',2,1295),(2576,'Experiment Platform','Agilent Microarray',2,1296),(2577,'Experiment Platform','Agilent Microarray',2,1297),(2578,'Experiment Platform','Agilent Microarray',2,1298),(2579,'Experiment Platform','Agilent Microarray',2,1299),(2580,'Experiment Platform','Agilent Microarray',2,1300),(2581,'Experiment Platform','Agilent Microarray',2,1301),(2582,'Experiment Platform','Agilent Microarray',2,1850),(2583,'Experiment Platform','Agilent Microarray',2,2453),(2584,'Experiment Platform','Agilent Microarray',2,3042),(2585,'Experiment Platform','Applied Biosystems SOLiD',2,1765),(2586,'Experiment Platform','Applied Biosystems SOLiD',2,1766),(2587,'Experiment Platform','Applied Biosystems SOLiD',2,1767),(2588,'Experiment Platform','Applied Biosystems SOLiD',2,1768),(2589,'Experiment Platform','Applied Biosystems SOLiD',2,1769),(2590,'Experiment Platform','Applied Biosystems SOLiD',2,1770),(2591,'Experiment Platform','Applied Biosystems SOLiD',2,1771),(2592,'Experiment Platform','Applied Biosystems SOLiD',2,1772),(2593,'Experiment Platform','Applied Biosystems SOLiD',2,1773),(2594,'Experiment Platform','Applied Biosystems SOLiD',2,1774),(2595,'Experiment Platform','Applied Biosystems SOLiD',2,1775),(2596,'Experiment Platform','Applied Biosystems SOLiD',2,1776),(2597,'Experiment Platform','Applied Biosystems SOLiD',2,1777),(2598,'Experiment Platform','Applied Biosystems SOLiD',2,1778),(2599,'Experiment Platform','Applied Biosystems SOLiD',2,1779),(2600,'Experiment Platform','Applied Biosystems SOLiD',2,1780),(2601,'Experiment Platform','Applied Biosystems SOLiD',2,1781),(2602,'Experiment Platform','Applied Biosystems SOLiD',2,1782),(2603,'Experiment Platform','Applied Biosystems SOLiD',2,1783),(2604,'Experiment Platform','Applied Biosystems SOLiD',2,1784),(2605,'Experiment Platform','Applied Biosystems SOLiD',2,1785),(2606,'Experiment Platform','Applied Biosystems SOLiD',2,1786),(2607,'Experiment Platform','Applied Biosystems SOLiD',2,1787),(2608,'Experiment Platform','Applied Biosystems SOLiD',2,2593),(2609,'Experiment Platform','Applied Biosystems SOLiD',2,2594),(2610,'Experiment Platform','Applied Biosystems SOLiD',2,2595),(2611,'Experiment Platform','Applied Biosystems SOLiD',2,2596),(2612,'Experiment Platform','Applied Biosystems SOLiD',2,2597),(2613,'Experiment Platform','Applied Biosystems SOLiD',2,2598),(2614,'Experiment Platform','Applied Biosystems SOLiD',2,2599),(2615,'Experiment Platform','Applied Biosystems SOLiD',2,2600),(2616,'Experiment Platform','Applied Biosystems SOLiD',2,2617),(2617,'Experiment Platform','Applied Biosystems SOLiD',2,2618),(2618,'Experiment Method','chIP-Seq',3,881),(2619,'Experiment Method','chIP-Seq',3,882),(2620,'Experiment Method','chIP-Seq',3,883),(2621,'Experiment Method','chIP-Seq',3,1262),(2622,'Experiment Method','chIP-Seq',3,1329),(2623,'Experiment Method','chIP-Seq',3,1330),(2624,'Experiment Method','chIP-Seq',3,1331),(2625,'Experiment Method','chIP-Seq',3,1332),(2626,'Experiment Method','chIP-Seq',3,1333),(2627,'Experiment Method','chIP-Seq',3,1334),(2628,'Experiment Method','chIP-Seq',3,1335),(2629,'Experiment Method','chIP-Seq',3,1336),(2630,'Experiment Method','chIP-Seq',3,1337),(2631,'Experiment Method','chIP-Seq',3,1338),(2632,'Experiment Method','chIP-Seq',3,1339),(2633,'Experiment Method','chIP-Seq',3,1340),(2634,'Experiment Method','chIP-Seq',3,1354),(2635,'Experiment Method','chIP-Seq',3,1355),(2636,'Experiment Method','chIP-Seq',3,1356),(2637,'Experiment Method','chIP-Seq',3,1357),(2638,'Experiment Method','chIP-Seq',3,1358),(2639,'Experiment Method','chIP-Seq',3,1359),(2640,'Experiment Method','chIP-Seq',3,1360),(2641,'Experiment Method','chIP-Seq',3,1361),(2642,'Experiment Method','chIP-Seq',3,1362),(2643,'Experiment Method','chIP-Seq',3,1363),(2644,'Experiment Method','chIP-Seq',3,1364),(2645,'Experiment Method','chIP-Seq',3,1365),(2646,'Experiment Method','chIP-Seq',3,1366),(2647,'Experiment Method','chIP-Seq',3,1367),(2648,'Experiment Method','chIP-Seq',3,1368),(2649,'Experiment Method','chIP-Seq',3,1369),(2650,'Experiment Method','chIP-Seq',3,1370),(2651,'Experiment Method','chIP-Seq',3,1371),(2652,'Experiment Method','chIP-Seq',3,1372),(2653,'Experiment Method','chIP-Seq',3,1373),(2654,'Experiment Method','chIP-Seq',3,1374),(2655,'Experiment Method','chIP-Seq',3,1375),(2656,'Experiment Method','chIP-Seq',3,1376),(2657,'Experiment Method','chIP-Seq',3,1377),(2658,'Experiment Method','chIP-Seq',3,1378),(2659,'Experiment Method','chIP-Seq',3,1379),(2660,'Experiment Method','chIP-Seq',3,1380),(2661,'Experiment Method','chIP-Seq',3,1381),(2662,'Experiment Method','chIP-Seq',3,1382),(2663,'Experiment Method','chIP-Seq',3,1383),(2664,'Experiment Method','chIP-Seq',3,1384),(2665,'Experiment Method','chIP-Seq',3,1385),(2666,'Experiment Method','chIP-Seq',3,1386),(2667,'Experiment Method','chIP-Seq',3,1387),(2668,'Experiment Method','chIP-Seq',3,1388),(2669,'Experiment Method','chIP-Seq',3,1389),(2670,'Experiment Method','chIP-Seq',3,1390),(2671,'Experiment Method','chIP-Seq',3,1391),(2672,'Experiment Method','chIP-Seq',3,1392),(2673,'Experiment Method','chIP-Seq',3,1393),(2674,'Experiment Method','chIP-Seq',3,1394),(2675,'Experiment Method','chIP-Seq',3,1395),(2676,'Experiment Method','chIP-Seq',3,1396),(2677,'Experiment Method','chIP-Seq',3,1397),(2678,'Experiment Method','chIP-Seq',3,1398),(2679,'Experiment Method','chIP-Seq',3,1399),(2680,'Experiment Method','chIP-Seq',3,1400),(2681,'Experiment Method','chIP-Seq',3,1401),(2682,'Experiment Method','chIP-Seq',3,1402),(2683,'Experiment Method','chIP-Seq',3,1403),(2684,'Experiment Method','chIP-Seq',3,1404),(2685,'Experiment Method','chIP-Seq',3,1405),(2686,'Experiment Method','chIP-Seq',3,1406),(2687,'Experiment Method','chIP-Seq',3,1407),(2688,'Experiment Method','chIP-Seq',3,1408),(2689,'Experiment Method','chIP-Seq',3,1409),(2690,'Experiment Method','chIP-Seq',3,1410),(2691,'Experiment Method','chIP-Seq',3,1411),(2692,'Experiment Method','chIP-Seq',3,1412),(2693,'Experiment Method','chIP-Seq',3,1413),(2694,'Experiment Method','chIP-Seq',3,1414),(2695,'Experiment Method','chIP-Seq',3,1415),(2696,'Experiment Method','chIP-Seq',3,1416),(2697,'Experiment Method','chIP-Seq',3,1449),(2698,'Experiment Method','chIP-Seq',3,1450),(2699,'Experiment Method','chIP-Seq',3,1451),(2700,'Experiment Method','chIP-Seq',3,1452),(2701,'Experiment Method','chIP-Seq',3,1453),(2702,'Experiment Method','chIP-Seq',3,1454),(2703,'Experiment Method','chIP-Seq',3,1455),(2704,'Experiment Method','chIP-Seq',3,1456),(2705,'Experiment Method','chIP-Seq',3,1457),(2706,'Experiment Method','chIP-Seq',3,1458),(2707,'Experiment Method','chIP-Seq',3,1459),(2708,'Experiment Method','chIP-Seq',3,1460),(2709,'Experiment Method','chIP-Seq',3,1461),(2710,'Experiment Method','chIP-Seq',3,1462),(2711,'Experiment Method','chIP-Seq',3,1463),(2712,'Experiment Method','chIP-Seq',3,1464),(2713,'Experiment Method','chIP-Seq',3,1465),(2714,'Experiment Method','chIP-Seq',3,1466),(2715,'Experiment Method','chIP-Seq',3,1467),(2716,'Experiment Method','chIP-Seq',3,1468),(2717,'Experiment Method','chIP-Seq',3,1469),(2718,'Experiment Method','chIP-Seq',3,1470),(2719,'Experiment Method','chIP-Seq',3,1471),(2720,'Experiment Method','chIP-Seq',3,1472),(2721,'Experiment Method','chIP-Seq',3,1473),(2722,'Experiment Method','chIP-Seq',3,1474),(2723,'Experiment Method','chIP-Seq',3,1475),(2724,'Experiment Method','chIP-Seq',3,1476),(2725,'Experiment Method','chIP-Seq',3,1477),(2726,'Experiment Method','chIP-Seq',3,1478),(2727,'Experiment Method','chIP-Seq',3,1479),(2728,'Experiment Method','chIP-Seq',3,1480),(2729,'Experiment Method','chIP-Seq',3,1481),(2730,'Experiment Method','chIP-Seq',3,1482),(2731,'Experiment Method','chIP-Seq',3,1483),(2732,'Experiment Method','chIP-Seq',3,1484),(2733,'Experiment Method','chIP-Seq',3,1485),(2734,'Experiment Method','chIP-Seq',3,1486),(2735,'Experiment Method','chIP-Seq',3,1487),(2736,'Experiment Method','chIP-Seq',3,1488),(2737,'Experiment Method','chIP-Seq',3,1489),(2738,'Experiment Method','chIP-Seq',3,1490),(2739,'Experiment Method','chIP-Seq',3,1491),(2740,'Experiment Method','chIP-Seq',3,1492),(2741,'Experiment Method','chIP-Seq',3,1493),(2742,'Experiment Method','chIP-Seq',3,1494),(2743,'Experiment Method','chIP-Seq',3,1495),(2744,'Experiment Method','chIP-Seq',3,1496),(2745,'Experiment Method','chIP-Seq',3,1497),(2746,'Experiment Method','chIP-Seq',3,1498),(2747,'Experiment Method','chIP-Seq',3,1499),(2748,'Experiment Method','chIP-Seq',3,1500),(2749,'Experiment Method','chIP-Seq',3,1501),(2750,'Experiment Method','chIP-Seq',3,1502),(2751,'Experiment Method','chIP-Seq',3,1503),(2752,'Experiment Method','chIP-Seq',3,1504),(2753,'Experiment Method','chIP-Seq',3,1505),(2754,'Experiment Method','chIP-Seq',3,1506),(2755,'Experiment Method','chIP-Seq',3,1507),(2756,'Experiment Method','chIP-Seq',3,1508),(2757,'Experiment Method','chIP-Seq',3,1509),(2758,'Experiment Method','chIP-Seq',3,1510),(2759,'Experiment Method','chIP-Seq',3,1511),(2760,'Experiment Method','chIP-Seq',3,1512),(2761,'Experiment Method','chIP-Seq',3,1513),(2762,'Experiment Method','chIP-Seq',3,1514),(2763,'Experiment Method','chIP-Seq',3,1515),(2764,'Experiment Method','chIP-Seq',3,1516),(2765,'Experiment Method','chIP-Seq',3,1517),(2766,'Experiment Method','chIP-Seq',3,1518),(2767,'Experiment Method','chIP-Seq',3,1519),(2768,'Experiment Method','chIP-Seq',3,1598),(2769,'Experiment Method','chIP-Seq',3,1599),(2770,'Experiment Method','chIP-Seq',3,1600),(2771,'Experiment Method','chIP-Seq',3,1601),(2772,'Experiment Method','chIP-Seq',3,1602),(2773,'Experiment Method','chIP-Seq',3,1603),(2774,'Experiment Method','chIP-Seq',3,1604),(2775,'Experiment Method','chIP-Seq',3,1605),(2776,'Experiment Method','chIP-Seq',3,1606),(2777,'Experiment Method','chIP-Seq',3,1607),(2778,'Experiment Method','chIP-Seq',3,1608),(2779,'Experiment Method','chIP-Seq',3,1609),(2780,'Experiment Method','chIP-Seq',3,1610),(2781,'Experiment Method','chIP-Seq',3,1611),(2782,'Experiment Method','chIP-Seq',3,1612),(2783,'Experiment Method','chIP-Seq',3,1613),(2784,'Experiment Method','chIP-Seq',3,1614),(2785,'Experiment Method','chIP-Seq',3,1615),(2786,'Experiment Method','chIP-Seq',3,1616),(2787,'Experiment Method','chIP-Seq',3,1617),(2788,'Experiment Method','chIP-Seq',3,1618),(2789,'Experiment Method','chIP-Seq',3,1619),(2790,'Experiment Method','chIP-Seq',3,1620),(2791,'Experiment Method','chIP-Seq',3,1621),(2792,'Experiment Method','chIP-Seq',3,1622),(2793,'Experiment Method','chIP-Seq',3,1623),(2794,'Experiment Method','chIP-Seq',3,1624),(2795,'Experiment Method','chIP-Seq',3,1625),(2796,'Experiment Method','chIP-Seq',3,1626),(2797,'Experiment Method','chIP-Seq',3,1627),(2798,'Experiment Method','chIP-Seq',3,1628),(2799,'Experiment Method','chIP-Seq',3,1629),(2800,'Experiment Method','chIP-Seq',3,1630),(2801,'Experiment Method','chIP-Seq',3,1631),(2802,'Experiment Method','chIP-Seq',3,1632),(2803,'Experiment Method','chIP-Seq',3,1633),(2804,'Experiment Method','chIP-Seq',3,1634),(2805,'Experiment Method','chIP-Seq',3,1635),(2806,'Experiment Method','chIP-Seq',3,1636),(2807,'Experiment Method','chIP-Seq',3,1637),(2808,'Experiment Method','chIP-Seq',3,1638),(2809,'Experiment Method','chIP-Seq',3,1639),(2810,'Experiment Method','chIP-Seq',3,1640),(2811,'Experiment Method','chIP-Seq',3,1641),(2812,'Experiment Method','chIP-Seq',3,1642),(2813,'Experiment Method','chIP-Seq',3,1643),(2814,'Experiment Method','chIP-Seq',3,1644),(2815,'Experiment Method','chIP-Seq',3,1645),(2816,'Experiment Method','chIP-Seq',3,1646),(2817,'Experiment Method','chIP-Seq',3,1647),(2818,'Experiment Method','chIP-Seq',3,1648);

INSERT INTO dbo.gpAnnotationProperty
(idAnnotationProperty,
name,
value,
idProperty,
idAnnotation
) VALUES(2819,'Experiment Method','chIP-Seq',3,1649),(2820,'Experiment Method','chIP-Seq',3,1650),(2821,'Experiment Method','chIP-Seq',3,1651),(2822,'Experiment Method','chIP-Seq',3,1652),(2823,'Experiment Method','chIP-Seq',3,1653),(2824,'Experiment Method','chIP-Seq',3,1654),(2825,'Experiment Method','chIP-Seq',3,1655),(2826,'Experiment Method','chIP-Seq',3,1656),(2827,'Experiment Method','chIP-Seq',3,1657),(2828,'Experiment Method','chIP-Seq',3,1658),(2829,'Experiment Method','chIP-Seq',3,1659),(2830,'Experiment Method','chIP-Seq',3,1660),(2831,'Experiment Method','chIP-Seq',3,1661),(2832,'Experiment Method','chIP-Seq',3,1662),(2833,'Experiment Method','chIP-Seq',3,1663),(2834,'Experiment Method','chIP-Seq',3,1664),(2835,'Experiment Method','chIP-Seq',3,1665),(2836,'Experiment Method','chIP-Seq',3,1666),(2837,'Experiment Method','chIP-Seq',3,1667),(2838,'Experiment Method','chIP-Seq',3,1668),(2839,'Experiment Method','chIP-Seq',3,1669),(2840,'Experiment Method','chIP-Seq',3,1670),(2841,'Experiment Method','chIP-Seq',3,1671),(2842,'Experiment Method','chIP-Seq',3,1672),(2843,'Experiment Method','chIP-Seq',3,1673),(2844,'Experiment Method','chIP-Seq',3,1674),(2845,'Experiment Method','chIP-Seq',3,1675),(2846,'Experiment Method','chIP-Seq',3,1676),(2847,'Experiment Method','chIP-Seq',3,1677),(2848,'Experiment Method','chIP-Seq',3,1678),(2849,'Experiment Method','chIP-Seq',3,1679),(2850,'Experiment Method','chIP-Seq',3,1680),(2851,'Experiment Method','chIP-Seq',3,1681),(2852,'Experiment Method','chIP-Seq',3,1682),(2853,'Experiment Method','chIP-Seq',3,1683),(2854,'Experiment Method','chIP-Seq',3,1684),(2855,'Experiment Method','chIP-Seq',3,1685),(2856,'Experiment Method','chIP-Seq',3,1686),(2857,'Experiment Method','chIP-Seq',3,1687),(2858,'Experiment Method','chIP-Seq',3,1688),(2859,'Experiment Method','chIP-Seq',3,1689),(2860,'Experiment Method','chIP-Seq',3,1690),(2861,'Experiment Method','chIP-Seq',3,1691),(2862,'Experiment Method','chIP-Seq',3,1692),(2863,'Experiment Method','chIP-Seq',3,1693),(2864,'Experiment Method','chIP-Seq',3,1694),(2865,'Experiment Method','chIP-Seq',3,1695),(2866,'Experiment Method','chIP-Seq',3,1696),(2867,'Experiment Method','chIP-Seq',3,1697),(2868,'Experiment Method','chIP-Seq',3,1698),(2869,'Experiment Method','chIP-Seq',3,1699),(2870,'Experiment Method','chIP-Seq',3,1700),(2871,'Experiment Method','chIP-Seq',3,1701),(2872,'Experiment Method','chIP-Seq',3,1702),(2873,'Experiment Method','chIP-Seq',3,1703),(2874,'Experiment Method','chIP-Seq',3,1704),(2875,'Experiment Method','chIP-Seq',3,1705),(2876,'Experiment Method','chIP-Seq',3,1706),(2877,'Experiment Method','chIP-Seq',3,1707),(2878,'Experiment Method','chIP-Seq',3,1708),(2879,'Experiment Method','chIP-Seq',3,1709),(2880,'Experiment Method','chIP-Seq',3,1710),(2881,'Experiment Method','chIP-Seq',3,1711),(2882,'Experiment Method','chIP-Seq',3,1712),(2883,'Experiment Method','chIP-Seq',3,1713),(2884,'Experiment Method','chIP-Seq',3,1714),(2885,'Experiment Method','chIP-Seq',3,1715),(2886,'Experiment Method','chIP-Seq',3,1716),(2887,'Experiment Method','chIP-Seq',3,1717),(2888,'Experiment Method','chIP-Seq',3,1718),(2889,'Experiment Method','chIP-Seq',3,1719),(2890,'Experiment Method','chIP-Seq',3,1720),(2891,'Experiment Method','chIP-Seq',3,1721),(2892,'Experiment Method','chIP-Seq',3,1722),(2893,'Experiment Method','chIP-Seq',3,1723),(2894,'Experiment Method','chIP-Seq',3,1724),(2895,'Experiment Method','chIP-Seq',3,1725),(2896,'Experiment Method','chIP-Seq',3,1726),(2897,'Experiment Method','chIP-Seq',3,1727),(2898,'Experiment Method','chIP-Seq',3,1728),(2899,'Experiment Method','chIP-Seq',3,1729),(2900,'Experiment Method','chIP-Seq',3,1730),(2901,'Experiment Method','chIP-Seq',3,1731),(2902,'Experiment Method','chIP-Seq',3,1732),(2903,'Experiment Method','chIP-Seq',3,1733),(2904,'Experiment Method','chIP-Seq',3,1734),(2905,'Experiment Method','chIP-Seq',3,1735),(2906,'Experiment Method','chIP-Seq',3,1736),(2907,'Experiment Method','chIP-Seq',3,1737),(2908,'Experiment Method','chIP-Seq',3,1738),(2909,'Experiment Method','chIP-Seq',3,1739),(2910,'Experiment Method','chIP-Seq',3,1740),(2911,'Experiment Method','chIP-Seq',3,1741),(2912,'Experiment Method','chIP-Seq',3,1742),(2913,'Experiment Method','chIP-Seq',3,1743),(2914,'Experiment Method','chIP-Seq',3,1744),(2915,'Experiment Method','chIP-Seq',3,1745),(2916,'Experiment Method','chIP-Seq',3,1746),(2917,'Experiment Method','chIP-Seq',3,1747),(2918,'Experiment Method','chIP-Seq',3,1748),(2919,'Experiment Method','chIP-Seq',3,1749),(2920,'Experiment Method','chIP-Seq',3,1750),(2921,'Experiment Method','chIP-Seq',3,1751),(2922,'Experiment Method','chIP-Seq',3,1752),(2923,'Experiment Method','chIP-Seq',3,1753),(2924,'Experiment Method','chIP-Seq',3,1754),(2925,'Experiment Method','chIP-Seq',3,1755),(2926,'Experiment Method','chIP-Seq',3,1756),(2927,'Experiment Method','chIP-Seq',3,1808),(2928,'Experiment Method','chIP-Seq',3,1809),(2929,'Experiment Method','chIP-Seq',3,1812),(2930,'Experiment Method','chIP-Seq',3,1813),(2931,'Experiment Method','chIP-Seq',3,1814),(2932,'Experiment Method','chIP-Seq',3,1815),(2933,'Experiment Method','chIP-Seq',3,1816),(2934,'Experiment Method','chIP-Seq',3,1817),(2935,'Experiment Method','chIP-Seq',3,1818),(2936,'Experiment Method','chIP-Seq',3,1819),(2937,'Experiment Method','chIP-Seq',3,1820),(2938,'Experiment Method','chIP-Seq',3,1821),(2939,'Experiment Method','chIP-Seq',3,1822),(2940,'Experiment Method','chIP-Seq',3,1823),(2941,'Experiment Method','chIP-Seq',3,1824),(2942,'Experiment Method','chIP-Seq',3,1825),(2943,'Experiment Method','chIP-Seq',3,1826),(2944,'Experiment Method','chIP-Seq',3,1827),(2945,'Experiment Method','chIP-Seq',3,1828),(2946,'Experiment Method','chIP-Seq',3,1829),(2947,'Experiment Method','chIP-Seq',3,1830),(2948,'Experiment Method','chIP-Seq',3,1831),(2949,'Experiment Method','chIP-Seq',3,1832),(2950,'Experiment Method','chIP-Seq',3,1833),(2951,'Experiment Method','chIP-Seq',3,1834),(2952,'Experiment Method','chIP-Seq',3,1835),(2953,'Experiment Method','chIP-Seq',3,1886),(2954,'Experiment Method','chIP-Seq',3,1887),(2955,'Experiment Method','chIP-Seq',3,1888),(2956,'Experiment Method','chIP-Seq',3,1889),(2957,'Experiment Method','chIP-Seq',3,1890),(2958,'Experiment Method','chIP-Seq',3,1891),(2959,'Experiment Method','chIP-Seq',3,1894),(2960,'Experiment Method','chIP-Seq',3,1895),(2961,'Experiment Method','chIP-Seq',3,1896),(2962,'Experiment Method','chIP-Seq',3,1897),(2963,'Experiment Method','chIP-Seq',3,1898),(2964,'Experiment Method','chIP-Seq',3,1899),(2965,'Experiment Method','chIP-Seq',3,1900),(2966,'Experiment Method','chIP-Seq',3,1901),(2967,'Experiment Method','chIP-Seq',3,1902),(2968,'Experiment Method','chIP-Seq',3,1903),(2969,'Experiment Method','chIP-Seq',3,1904),(2970,'Experiment Method','chIP-Seq',3,1905),(2971,'Experiment Method','chIP-Seq',3,1921),(2972,'Experiment Method','chIP-Seq',3,1922),(2973,'Experiment Method','chIP-Seq',3,1923),(2974,'Experiment Method','chIP-Seq',3,1924),(2975,'Experiment Method','chIP-Seq',3,1925),(2976,'Experiment Method','chIP-Seq',3,1926),(2977,'Experiment Method','chIP-Seq',3,1927),(2978,'Experiment Method','chIP-Seq',3,1928),(2979,'Experiment Method','chIP-Seq',3,1929),(2980,'Experiment Method','chIP-Seq',3,1933),(2981,'Experiment Method','chIP-Seq',3,1934),(2982,'Experiment Method','chIP-Seq',3,1935),(2983,'Experiment Method','chIP-Seq',3,1936),(2984,'Experiment Method','chIP-Seq',3,1937),(2985,'Experiment Method','chIP-Seq',3,1938),(2986,'Experiment Method','chIP-Seq',3,1939),(2987,'Experiment Method','chIP-Seq',3,1940),(2988,'Experiment Method','chIP-Seq',3,1941),(2989,'Experiment Method','chIP-Seq',3,1942),(2990,'Experiment Method','chIP-Seq',3,1943),(2991,'Experiment Method','chIP-Seq',3,1944),(2992,'Experiment Method','chIP-Seq',3,1945),(2993,'Experiment Method','chIP-Seq',3,1946),(2994,'Experiment Method','chIP-Seq',3,1947),(2995,'Experiment Method','chIP-Seq',3,1948),(2996,'Experiment Method','chIP-Seq',3,1949),(2997,'Experiment Method','chIP-Seq',3,1950),(2998,'Experiment Method','chIP-Seq',3,2134),(2999,'Experiment Method','chIP-Seq',3,2135),(3000,'Experiment Method','chIP-Seq',3,2136),(3001,'Experiment Method','chIP-Seq',3,2137),(3002,'Experiment Method','chIP-Seq',3,2138),(3003,'Experiment Method','chIP-Seq',3,2139),(3004,'Experiment Method','chIP-Seq',3,2141),(3005,'Experiment Method','chIP-Seq',3,2144),(3006,'Experiment Method','chIP-Seq',3,2145),(3007,'Experiment Method','chIP-Seq',3,2146),(3008,'Experiment Method','chIP-Seq',3,2147),(3009,'Experiment Method','chIP-Seq',3,2148),(3010,'Experiment Method','chIP-Seq',3,2149),(3011,'Experiment Method','chIP-Seq',3,2150),(3012,'Experiment Method','chIP-Seq',3,2151),(3013,'Experiment Method','chIP-Seq',3,2152),(3014,'Experiment Method','chIP-Seq',3,2153),(3015,'Experiment Method','chIP-Seq',3,2154),(3016,'Experiment Method','chIP-Seq',3,2155),(3017,'Experiment Method','chIP-Seq',3,2160),(3018,'Experiment Method','chIP-Seq',3,2161),(3019,'Experiment Method','chIP-Seq',3,2211),(3020,'Experiment Method','chIP-Seq',3,2212),(3021,'Experiment Method','chIP-Seq',3,2213),(3022,'Experiment Method','chIP-Seq',3,2214),(3023,'Experiment Method','chIP-Seq',3,2215),(3024,'Experiment Method','chIP-Seq',3,2216),(3025,'Experiment Method','chIP-Seq',3,2217),(3026,'Experiment Method','chIP-Seq',3,2218),(3027,'Experiment Method','chIP-Seq',3,2219),(3028,'Experiment Method','chIP-Seq',3,2220),(3029,'Experiment Method','chIP-Seq',3,2221),(3030,'Experiment Method','chIP-Seq',3,2222),(3031,'Experiment Method','chIP-Seq',3,2223),(3032,'Experiment Method','chIP-Seq',3,2224),(3033,'Experiment Method','chIP-Seq',3,2225),(3034,'Experiment Method','chIP-Seq',3,2226),(3035,'Experiment Method','chIP-Seq',3,2228),(3036,'Experiment Method','chIP-Seq',3,2229),(3037,'Experiment Method','chIP-Seq',3,2230),(3038,'Experiment Method','chIP-Seq',3,2231),(3039,'Experiment Method','chIP-Seq',3,2263),(3040,'Experiment Method','chIP-Seq',3,2264),(3041,'Experiment Method','chIP-Seq',3,2265),(3042,'Experiment Method','chIP-Seq',3,2365),(3043,'Experiment Method','chIP-Seq',3,2368),(3045,'Experiment Method','chIP-Seq',3,2377),(3054,'Experiment Method','chIP-Seq',3,2387),(3055,'Experiment Method','chIP-Seq',3,2388),(3056,'Experiment Method','chIP-Seq',3,2389),(3057,'Experiment Method','chIP-Seq',3,2390),(3058,'Experiment Method','chIP-Seq',3,2391),(3059,'Experiment Method','chIP-Seq',3,2392),(3060,'Experiment Method','chIP-Seq',3,2393),(3061,'Experiment Method','chIP-Seq',3,2394),(3062,'Experiment Method','chIP-Seq',3,2395),(3063,'Experiment Method','chIP-Seq',3,2396),(3064,'Experiment Method','chIP-Seq',3,2397),(3065,'Experiment Method','chIP-Seq',3,2398),(3066,'Experiment Method','chIP-Seq',3,2423),(3067,'Experiment Method','chIP-Seq',3,2426),(3068,'Experiment Method','chIP-Seq',3,2427),(3069,'Experiment Method','chIP-Seq',3,2428),(3070,'Experiment Method','chIP-Seq',3,2429),(3071,'Experiment Method','chIP-Seq',3,2430),(3072,'Experiment Method','chIP-Seq',3,2431),(3073,'Experiment Method','chIP-Seq',3,2432),(3074,'Experiment Method','chIP-Seq',3,2433),(3075,'Experiment Method','chIP-Seq',3,2435),(3076,'Experiment Method','chIP-Seq',3,2436),(3077,'Experiment Method','chIP-Seq',3,2437),(3078,'Experiment Method','chIP-Seq',3,2438),(3079,'Experiment Method','chIP-Seq',3,2439),(3080,'Experiment Method','chIP-Seq',3,2440),(3081,'Experiment Method','chIP-Seq',3,2441),(3082,'Experiment Method','chIP-Seq',3,2442),(3083,'Experiment Method','chIP-Seq',3,2443),(3084,'Experiment Method','chIP-Seq',3,2468),(3085,'Experiment Method','chIP-Seq',3,2469),(3086,'Experiment Method','chIP-Seq',3,2470),(3087,'Experiment Method','chIP-Seq',3,2471),(3088,'Experiment Method','chIP-Seq',3,2472),(3089,'Experiment Method','chIP-Seq',3,2473),(3090,'Experiment Method','chIP-Seq',3,2474),(3091,'Experiment Method','chIP-Seq',3,2475),(3092,'Experiment Method','chIP-Seq',3,2476),(3093,'Experiment Method','chIP-Seq',3,2477),(3094,'Experiment Method','chIP-Seq',3,2478),(3095,'Experiment Method','chIP-Seq',3,2495),(3096,'Experiment Method','chIP-Seq',3,2496),(3097,'Experiment Method','chIP-Seq',3,2497),(3098,'Experiment Method','chIP-Seq',3,2498),(3099,'Experiment Method','chIP-Seq',3,2499),(3100,'Experiment Method','chIP-Seq',3,2501),(3103,'Experiment Method','chIP-Seq',3,2787),(3104,'Experiment Method','chIP-Seq',3,2788),(3105,'Experiment Method','chIP-Seq',3,2789),(3106,'Experiment Method','chIP-Seq',3,2790),(3107,'Experiment Method','chIP-Seq',3,2791),(3108,'Experiment Method','chIP-Seq',3,2792),(3109,'Experiment Method','chIP-Seq',3,2795),(3110,'Experiment Method','chIP-Seq',3,2797),(3111,'Experiment Method','chIP-Seq',3,2798),(3112,'Experiment Method','chIP-Seq',3,2799),(3113,'Experiment Method','chIP-Seq',3,2800),(3114,'Experiment Method','chIP-Seq',3,2801),(3115,'Experiment Method','chIP-Seq',3,2802),(3116,'Experiment Method','chIP-Seq',3,2816),(3117,'Experiment Method','chIP-Seq',3,2978),(3118,'Experiment Method','chIP-Seq',3,2979),(3119,'Experiment Method','chIP-Seq',3,2980),(3120,'Experiment Method','chIP-Seq',3,2987),(3121,'Experiment Method','chIP-Seq',3,2988),(3122,'Experiment Method','chIP-Seq',3,3076),(3123,'Experiment Method','chIP-Seq',3,3079),(3124,'Experiment Method','chIP-Seq',3,3080),(3125,'Experiment Method','chIP-Seq',3,3102),(3126,'Experiment Method','chIP-Seq',3,3103),(3127,'Experiment Method','chIP-Seq',3,3104),(3128,'Experiment Method','chIP-Seq',3,3258),(3129,'Experiment Method','chIP-Seq',3,3259),(3130,'Experiment Method','chIP-Seq',3,3260),(3131,'Experiment Method','chIP-Seq',3,3261),(3132,'Experiment Method','chIP-Seq',3,3262),(3133,'Experiment Method','chIP-Seq',3,3263),(3134,'Experiment Method','chIP-Seq',3,3264),(3135,'Experiment Method','chIP-Seq',3,3265),(3136,'Experiment Method','chIP-Seq',3,3266),(3137,'Experiment Method','chIP-Seq',3,3267),(3138,'Experiment Method','chIP-Seq',3,3268),(3139,'Experiment Method','chIP-Seq',3,3269),(3140,'Experiment Method','chIP-Seq',3,3270),(3141,'Experiment Method','chIP-Seq',3,3271),(3142,'Experiment Method','chIP-Seq',3,3272),(3143,'Experiment Method','chIP-Seq',3,3430),(3144,'Experiment Method','chIP-Seq',3,3431),(3145,'Experiment Method','chIP-Seq',3,3432),(3146,'Experiment Method','chIP-Seq',3,3433),(3147,'Experiment Method','chIP-Seq',3,3434),(3148,'Experiment Method','chIP-Seq',3,3435),(3149,'Experiment Method','chIP-Seq',3,3439),(3150,'Experiment Method','chIP-Seq',3,3440),(3151,'Experiment Method','chIP-Seq',3,3443),(3152,'Experiment Method','chIP-Seq',3,3444),(3153,'Experiment Method','chIP-Seq',3,3445),(3154,'Experiment Method','chIP-Seq',3,3477),(3155,'Experiment Method','chIP-Seq',3,3478),(3156,'Experiment Method','chIP-Seq',3,3479),(3157,'Experiment Method','chIP-Seq',3,3480),(3158,'Experiment Method','chIP-Seq',3,3481),(3159,'Experiment Method','chIP-Seq',3,3482),(3160,'Experiment Method','chIP-Seq',3,3483),(3161,'Experiment Method','chIP-chip',3,1285),(3162,'Experiment Method','chIP-chip',3,1286),(3163,'Experiment Method','chIP-chip',3,1287),(3164,'Experiment Method','chIP-chip',3,1288),(3165,'Experiment Method','chIP-chip',3,1294),(3166,'Experiment Method','chIP-chip',3,1295),(3167,'Experiment Method','chIP-chip',3,1296),(3168,'Experiment Method','chIP-chip',3,1297),(3169,'Experiment Method','chIP-chip',3,1298),(3170,'Experiment Method','chIP-chip',3,1299),(3171,'Experiment Method','chIP-chip',3,1300),(3172,'Experiment Method','chIP-chip',3,1301),(3173,'Experiment Method','chIP-chip',3,1850),(3174,'Experiment Method','chIP-chip',3,3042),(3175,'Experiment Method','Read Counts/ Coverage',3,1874),(3176,'Experiment Method','Read Counts/ Coverage',3,1875),(3177,'Experiment Method','Read Counts/ Coverage',3,1876),(3178,'Experiment Method','Read Counts/ Coverage',3,1877),(3179,'Experiment Method','Read Counts/ Coverage',3,1878),(3180,'Experiment Method','RNA-Seq',3,1430),(3181,'Experiment Method','RNA-Seq',3,1431),(3182,'Experiment Method','RNA-Seq',3,1432),(3183,'Experiment Method','RNA-Seq',3,1433),(3184,'Experiment Method','RNA-Seq',3,1434),(3185,'Experiment Method','RNA-Seq',3,1435),(3186,'Experiment Method','RNA-Seq',3,1792),(3187,'Experiment Method','RNA-Seq',3,1793),(3188,'Experiment Method','RNA-Seq',3,1799),(3189,'Experiment Method','RNA-Seq',3,1800),(3190,'Experiment Method','RNA-Seq',3,1801),(3191,'Experiment Method','RNA-Seq',3,1802),(3192,'Experiment Method','RNA-Seq',3,1803),(3193,'Experiment Method','RNA-Seq',3,1805),(3194,'Experiment Method','RNA-Seq',3,1806),(3195,'Experiment Method','RNA-Seq',3,1807),(3196,'Experiment Method','RNA-Seq',3,1837),(3197,'Experiment Method','RNA-Seq',3,1838),(3198,'Experiment Method','RNA-Seq',3,1839),(3199,'Experiment Method','RNA-Seq',3,1840),(3200,'Experiment Method','RNA-Seq',3,1841),(3201,'Experiment Method','RNA-Seq',3,1842),(3202,'Experiment Method','RNA-Seq',3,1861),(3203,'Experiment Method','RNA-Seq',3,1862),(3204,'Experiment Method','RNA-Seq',3,1863),(3205,'Experiment Method','RNA-Seq',3,1864),(3206,'Experiment Method','RNA-Seq',3,1865),(3207,'Experiment Method','RNA-Seq',3,1866),(3208,'Experiment Method','RNA-Seq',3,1867),(3209,'Experiment Method','RNA-Seq',3,1868),(3210,'Experiment Method','RNA-Seq',3,1873),(3211,'Experiment Method','RNA-Seq',3,1879),(3212,'Experiment Method','RNA-Seq',3,1880),(3213,'Experiment Method','RNA-Seq',3,1881),(3214,'Experiment Method','RNA-Seq',3,1882),(3215,'Experiment Method','RNA-Seq',3,1883),(3216,'Experiment Method','RNA-Seq',3,1884),(3217,'Experiment Method','RNA-Seq',3,1885),(3218,'Experiment Method','RNA-Seq',3,1893),(3219,'Experiment Method','RNA-Seq',3,1930),(3220,'Experiment Method','RNA-Seq',3,1931),(3221,'Experiment Method','RNA-Seq',3,2132),(3222,'Experiment Method','RNA-Seq',3,2142),(3223,'Experiment Method','RNA-Seq',3,2143),(3224,'Experiment Method','RNA-Seq',3,2169),(3225,'Experiment Method','RNA-Seq',3,2170),(3226,'Experiment Method','RNA-Seq',3,2234),(3227,'Experiment Method','RNA-Seq',3,2235),(3228,'Experiment Method','RNA-Seq',3,2236),(3229,'Experiment Method','RNA-Seq',3,2239),(3230,'Experiment Method','RNA-Seq',3,2240),(3231,'Experiment Method','RNA-Seq',3,2241),(3232,'Experiment Method','RNA-Seq',3,2357),(3233,'Experiment Method','RNA-Seq',3,2358),(3234,'Experiment Method','RNA-Seq',3,2359),(3235,'Experiment Method','RNA-Seq',3,2360),(3236,'Experiment Method','RNA-Seq',3,2361),(3237,'Experiment Method','RNA-Seq',3,2362),(3238,'Experiment Method','RNA-Seq',3,2363),(3239,'Experiment Method','RNA-Seq',3,2364),(3240,'Experiment Method','RNA-Seq',3,2366),(3241,'Experiment Method','RNA-Seq',3,2367),(3242,'Experiment Method','RNA-Seq',3,2401),(3243,'Experiment Method','RNA-Seq',3,2402),(3244,'Experiment Method','RNA-Seq',3,2403),(3245,'Experiment Method','RNA-Seq',3,2404),(3246,'Experiment Method','RNA-Seq',3,2405),(3247,'Experiment Method','RNA-Seq',3,2406),(3248,'Experiment Method','RNA-Seq',3,2407),(3249,'Experiment Method','RNA-Seq',3,2408),(3250,'Experiment Method','RNA-Seq',3,2409),(3251,'Experiment Method','RNA-Seq',3,2410),(3252,'Experiment Method','RNA-Seq',3,2411),(3253,'Experiment Method','RNA-Seq',3,2412),(3254,'Experiment Method','RNA-Seq',3,2413),(3255,'Experiment Method','RNA-Seq',3,2414),(3256,'Experiment Method','RNA-Seq',3,2415),(3257,'Experiment Method','RNA-Seq',3,2416),(3258,'Experiment Method','RNA-Seq',3,2417),(3259,'Experiment Method','RNA-Seq',3,2418),(3260,'Experiment Method','RNA-Seq',3,2419),(3261,'Experiment Method','RNA-Seq',3,2420),(3262,'Experiment Method','RNA-Seq',3,2421),(3263,'Experiment Method','RNA-Seq',3,2422),(3264,'Experiment Method','RNA-Seq',3,2562),(3265,'Experiment Method','RNA-Seq',3,2592),(3266,'Experiment Method','RNA-Seq',3,2593),(3267,'Experiment Method','RNA-Seq',3,2594),(3268,'Experiment Method','RNA-Seq',3,2595),(3269,'Experiment Method','RNA-Seq',3,2596),(3270,'Experiment Method','RNA-Seq',3,2597),(3271,'Experiment Method','RNA-Seq',3,2598),(3272,'Experiment Method','RNA-Seq',3,2599),(3273,'Experiment Method','RNA-Seq',3,2600),(3274,'Experiment Method','RNA-Seq',3,2617),(3275,'Experiment Method','RNA-Seq',3,2618),(3276,'Experiment Method','RNA-Seq',3,2631),(3277,'Experiment Method','RNA-Seq',3,2640),(3278,'Experiment Method','RNA-Seq',3,2783),(3279,'Experiment Method','RNA-Seq',3,2784),(3280,'Experiment Method','RNA-Seq',3,2785),(3281,'Experiment Method','RNA-Seq',3,2786),(3282,'Experiment Method','RNA-Seq',3,2892),(3283,'Experiment Method','RNA-Seq',3,2893),(3284,'Experiment Method','RNA-Seq',3,2896),(3285,'Experiment Method','RNA-Seq',3,2897),(3286,'Experiment Method','RNA-Seq',3,2898),(3287,'Experiment Method','RNA-Seq',3,2899),(3288,'Experiment Method','RNA-Seq',3,2947),(3289,'Experiment Method','RNA-Seq',3,3381),(3290,'Experiment Method','RNA-Seq',3,3382),(3291,'Experiment Method','RNA-Seq',3,3420),(3292,'Experiment Method','RNA-Seq',3,3421),(3293,'Experiment Method','RNA-Seq',3,3422),(3294,'Experiment Method','RNA-Seq',3,3423),(3295,'Experiment Method','RNA-Seq',3,3424),(3296,'Experiment Method','RNA-Seq',3,3425),(3297,'Experiment Method','RNA-Seq',3,3461),(3298,'Experiment Method','RNA-Seq',3,3484),(3299,'Experiment Method','RNA-Seq',3,3527),(3300,'Experiment Method','RNA-Seq',3,3531),(3301,'Experiment Method','Digital DNaseI',3,1521),(3302,'Experiment Method','Digital DNaseI',3,1522),(3303,'Experiment Method','Digital DNaseI',3,1523),(3304,'Experiment Method','Digital DNaseI',3,1524),(3305,'Experiment Method','Digital DNaseI',3,1525),(3306,'Experiment Method','Digital DNaseI',3,1526),(3307,'Experiment Method','Digital DNaseI',3,1527),(3308,'Experiment Method','Digital DNaseI',3,1528),(3309,'Experiment Method','Digital DNaseI',3,1529),(3310,'Experiment Method','Digital DNaseI',3,1530),(3311,'Experiment Method','Digital DNaseI',3,1531),(3312,'Experiment Method','Digital DNaseI',3,1532),(3313,'Experiment Method','Digital DNaseI',3,1533),(3314,'Experiment Method','Digital DNaseI',3,1534),(3315,'Experiment Method','Digital DNaseI',3,1535),(3316,'Experiment Method','Digital DNaseI',3,1536),(3317,'Experiment Method','Digital DNaseI',3,1537),(3318,'Experiment Method','Digital DNaseI',3,1538),(3319,'Experiment Method','Digital DNaseI',3,1539),(3320,'Experiment Method','Digital DNaseI',3,1540),(3321,'Experiment Method','Digital DNaseI',3,1541),(3322,'Experiment Method','Digital DNaseI',3,1542),(3323,'Experiment Method','Digital DNaseI',3,1543),(3324,'Experiment Method','Digital DNaseI',3,1544),(3325,'Experiment Method','Digital DNaseI',3,1545),(3326,'Experiment Method','Digital DNaseI',3,1546),(3327,'Experiment Method','Digital DNaseI',3,1547),(3328,'Experiment Method','Digital DNaseI',3,1548),(3329,'Experiment Method','Digital DNaseI',3,1549),(3330,'Experiment Method','Digital DNaseI',3,1550),(3331,'Experiment Method','Digital DNaseI',3,1551),(3332,'Experiment Method','Digital DNaseI',3,1552),(3333,'Experiment Method','Digital DNaseI',3,1553),(3334,'Experiment Method','Digital DNaseI',3,1554),(3335,'Experiment Method','Digital DNaseI',3,1555),(3336,'Experiment Method','Digital DNaseI',3,1556),(3337,'Experiment Method','Digital DNaseI',3,1557),(3338,'Experiment Method','Digital DNaseI',3,1558),(3339,'Experiment Method','Digital DNaseI',3,1559),(3340,'Experiment Method','Digital DNaseI',3,1560),(3341,'Experiment Method','Digital DNaseI',3,1561),(3342,'Experiment Method','Digital DNaseI',3,1562),(3343,'Experiment Method','Digital DNaseI',3,1563),(3344,'Experiment Method','Digital DNaseI',3,1564),(3345,'Experiment Method','Digital DNaseI',3,1565),(3346,'Experiment Method','Digital DNaseI',3,1566),(3347,'Experiment Method','Digital DNaseI',3,1567),(3348,'Experiment Method','Digital DNaseI',3,1568),(3349,'Experiment Method','Digital DNaseI',3,1569),(3350,'Experiment Method','Digital DNaseI',3,1570),(3351,'Experiment Method','Digital DNaseI',3,1571),(3352,'Experiment Method','Digital DNaseI',3,1572),(3353,'Experiment Method','Digital DNaseI',3,1573),(3354,'Experiment Method','Digital DNaseI',3,1574),(3355,'Experiment Method','Digital DNaseI',3,1575),(3356,'Experiment Method','Digital DNaseI',3,1576),(3357,'Experiment Method','Digital DNaseI',3,1577),(3358,'Experiment Method','Digital DNaseI',3,1578),(3359,'Experiment Method','Digital DNaseI',3,1579),(3360,'Experiment Method','Digital DNaseI',3,1580),(3361,'Experiment Method','Digital DNaseI',3,1581),(3362,'Experiment Method','Digital DNaseI',3,1582),(3363,'Experiment Method','Digital DNaseI',3,1583),(3364,'Experiment Method','Digital DNaseI',3,1584),(3365,'Experiment Method','Digital DNaseI',3,1585),(3366,'Experiment Method','Digital DNaseI',3,1586),(3367,'Experiment Method','Digital DNaseI',3,1587),(3368,'Experiment Method','Digital DNaseI',3,1588),(3369,'Experiment Method','Digital DNaseI',3,1589),(3370,'Experiment Method','Digital DNaseI',3,1590),(3371,'Experiment Method','Digital DNaseI',3,1591),(3372,'Experiment Method','Digital DNaseI',3,1592),(3373,'Experiment Method','Digital DNaseI',3,1593),(3374,'Experiment Method','Digital DNaseI',3,1594),(3375,'Experiment Method','Digital DNaseI',3,1595),(3376,'Experiment Method','Digital DNaseI',3,2908),(3377,'Experiment Method','RIP-Seq',3,1758),(3378,'Experiment Method','RIP-Seq',3,1759),(3379,'Experiment Method','RIP-Seq',3,1760),(3380,'Experiment Method','RIP-Seq',3,1761),(3381,'Experiment Method','RIP-Seq',3,1762),(3382,'Experiment Method','RIP-Seq',3,1763),(3383,'Experiment Method','RIP-Seq',3,1764),(3384,'Experiment Method','CAGE',3,1765),(3385,'Experiment Method','CAGE',3,1766),(3386,'Experiment Method','CAGE',3,1767),(3387,'Experiment Method','CAGE',3,1768),(3388,'Experiment Method','CAGE',3,1769),(3389,'Experiment Method','CAGE',3,1770),(3390,'Experiment Method','CAGE',3,1771),(3391,'Experiment Method','CAGE',3,1772),(3392,'Experiment Method','CAGE',3,1773),(3393,'Experiment Method','CAGE',3,1774),(3394,'Experiment Method','CAGE',3,1775),(3395,'Experiment Method','CAGE',3,1776),(3396,'Experiment Method','CAGE',3,1777),(3397,'Experiment Method','CAGE',3,1778),(3398,'Experiment Method','CAGE',3,1779),(3399,'Experiment Method','CAGE',3,1780),(3400,'Experiment Method','CAGE',3,1781),(3401,'Experiment Method','CAGE',3,1782),(3402,'Experiment Method','CAGE',3,1783),(3403,'Experiment Method','CAGE',3,1784),(3404,'Experiment Method','CAGE',3,1785),(3405,'Experiment Method','CAGE',3,1786);

INSERT INTO dbo.gpAnnotationProperty
(idAnnotationProperty,
name,
value,
idProperty,
idAnnotation
) VALUES
(3406,'Experiment Method','CAGE',3,1787),(3407,'Experiment Method','Cap-Seq',3,823),(3408,'Experiment Method','Cap-Seq',3,1869),(3409,'Experiment Method','Cap-Seq',3,1870),(3410,'Experiment Method','Cap-Seq',3,1871),(3411,'Experiment Method','Cap-Seq',3,1872),(3412,'Experiment Method','Cap-Seq',3,1966),(3413,'Experiment Method','Cap-Seq',3,1967),(3414,'Experiment Method','Cap-Seq',3,1968),(3415,'Experiment Method','Cap-Seq',3,1969),(3416,'Experiment Method','Cap-Seq',3,1970),(3417,'Experiment Method','Cap-Seq',3,1971),(3418,'Experiment Method','Cap-Seq',3,1972),(3419,'Experiment Method','Cap-Seq',3,1973),(3420,'Experiment Method','Cap-Seq',3,1974),(3421,'Experiment Method','Cap-Seq',3,1975),(3422,'Experiment Method','Cap-Seq',3,1976),(3423,'Experiment Method','Cap-Seq',3,1977),(3424,'Experiment Method','Cap-Seq',3,1978),(3425,'Experiment Method','Cap-Seq',3,1979),(3426,'Experiment Method','Cap-Seq',3,1980),(3427,'Experiment Method','Cap-Seq',3,1981),(3428,'Experiment Method','Cap-Seq',3,1982),(3429,'Experiment Method','Cap-Seq',3,1983),(3430,'Experiment Method','Cap-Seq',3,1984),(3431,'Experiment Method','Cap-Seq',3,1985),(3432,'Experiment Method','Cap-Seq',3,1986),(3433,'Experiment Method','Cap-Seq',3,1987),(3434,'Experiment Method','Cap-Seq',3,1988),(3435,'Experiment Method','Cap-Seq',3,1989),(3436,'Experiment Method','Cap-Seq',3,1990),(3437,'Experiment Method','Cap-Seq',3,1991),(3438,'Experiment Method','Cap-Seq',3,1992),(3439,'Experiment Method','Cap-Seq',3,1993),(3440,'Experiment Method','Cap-Seq',3,1994),(3441,'Experiment Method','Cap-Seq',3,1995),(3442,'Experiment Method','Cap-Seq',3,1996),(3443,'Experiment Method','Cap-Seq',3,1997),(3444,'Experiment Method','Cap-Seq',3,1998),(3445,'Experiment Method','Cap-Seq',3,1999),(3446,'Experiment Method','Cap-Seq',3,2000),(3447,'Experiment Method','Cap-Seq',3,2001),(3448,'Experiment Method','Cap-Seq',3,2002),(3449,'Experiment Method','Cap-Seq',3,2003),(3450,'Experiment Method','Cap-Seq',3,2004),(3451,'Experiment Method','Cap-Seq',3,2005),(3452,'Experiment Method','Cap-Seq',3,2006),(3453,'Experiment Method','Cap-Seq',3,2007),(3454,'Experiment Method','Cap-Seq',3,2008),(3455,'Experiment Method','Cap-Seq',3,2009),(3456,'Experiment Method','Cap-Seq',3,2010),(3457,'Experiment Method','Cap-Seq',3,2011),(3458,'Experiment Method','Cap-Seq',3,2012),(3459,'Experiment Method','Cap-Seq',3,2013),(3460,'Experiment Method','Cap-Seq',3,2014),(3461,'Experiment Method','Cap-Seq',3,2015),(3462,'Experiment Method','Cap-Seq',3,2016),(3463,'Experiment Method','Cap-Seq',3,2017),(3464,'Experiment Method','Cap-Seq',3,2018),(3465,'Experiment Method','Cap-Seq',3,2019),(3466,'Experiment Method','Cap-Seq',3,2020),(3467,'Experiment Method','Cap-Seq',3,2021),(3468,'Experiment Method','Cap-Seq',3,2022),(3469,'Experiment Method','Cap-Seq',3,2023),(3470,'Experiment Method','Cap-Seq',3,2024),(3471,'Experiment Method','Cap-Seq',3,2025),(3472,'Experiment Method','Cap-Seq',3,2026),(3473,'Experiment Method','Cap-Seq',3,2027),(3474,'Experiment Method','Cap-Seq',3,2028),(3475,'Experiment Method','Cap-Seq',3,2029),(3476,'Experiment Method','Cap-Seq',3,2030),(3477,'Experiment Method','Cap-Seq',3,2031),(3478,'Experiment Method','Cap-Seq',3,2032),(3479,'Experiment Method','Cap-Seq',3,2033),(3480,'Experiment Method','Cap-Seq',3,2034),(3481,'Experiment Method','Cap-Seq',3,2035),(3482,'Experiment Method','Cap-Seq',3,2036),(3483,'Experiment Method','Cap-Seq',3,2037),(3484,'Experiment Method','Cap-Seq',3,2038),(3485,'Experiment Method','Cap-Seq',3,2039),(3486,'Experiment Method','Cap-Seq',3,2040),(3487,'Experiment Method','Cap-Seq',3,2041),(3488,'Experiment Method','Cap-Seq',3,2042),(3489,'Experiment Method','Cap-Seq',3,2043),(3490,'Experiment Method','Cap-Seq',3,2044),(3491,'Experiment Method','Cap-Seq',3,2045),(3492,'Experiment Method','Cap-Seq',3,2046),(3493,'Experiment Method','Cap-Seq',3,2047),(3494,'Experiment Method','Cap-Seq',3,2048),(3495,'Experiment Method','Cap-Seq',3,2049),(3496,'Experiment Method','Cap-Seq',3,2050),(3497,'Experiment Method','Cap-Seq',3,2051),(3498,'Experiment Method','Cap-Seq',3,2052),(3499,'Experiment Method','Cap-Seq',3,2074),(3500,'Experiment Method','Cap-Seq',3,2075),(3501,'Experiment Method','Cap-Seq',3,2076),(3502,'Experiment Method','Cap-Seq',3,2077),(3503,'Experiment Method','Cap-Seq',3,2078),(3504,'Experiment Method','Cap-Seq',3,2079),(3505,'Experiment Method','Cap-Seq',3,2080),(3506,'Experiment Method','Cap-Seq',3,2081),(3507,'Experiment Method','Cap-Seq',3,2082),(3508,'Experiment Method','Cap-Seq',3,2083),(3509,'Experiment Method','Cap-Seq',3,2084),(3510,'Experiment Method','Cap-Seq',3,2085),(3511,'Experiment Method','Cap-Seq',3,2086),(3512,'Experiment Method','Cap-Seq',3,2087),(3513,'Experiment Method','Cap-Seq',3,2088),(3514,'Experiment Method','Cap-Seq',3,2089),(3515,'Experiment Method','Cap-Seq',3,2090),(3516,'Experiment Method','Cap-Seq',3,2091),(3517,'Experiment Method','Cap-Seq',3,2092),(3518,'Experiment Method','Cap-Seq',3,2093),(3519,'Experiment Method','Cap-Seq',3,2094),(3520,'Experiment Method','Cap-Seq',3,2095),(3521,'Experiment Method','Cap-Seq',3,2096),(3522,'Experiment Method','Cap-Seq',3,2097),(3523,'Experiment Method','Cap-Seq',3,2098),(3524,'Experiment Method','Cap-Seq',3,2099),(3525,'Experiment Method','Cap-Seq',3,2100),(3526,'Experiment Method','Cap-Seq',3,2101),(3527,'Experiment Method','Cap-Seq',3,2102),(3528,'Experiment Method','Cap-Seq',3,2103),(3529,'Experiment Method','Cap-Seq',3,2104),(3530,'Experiment Method','Cap-Seq',3,2105),(3531,'Experiment Method','Cap-Seq',3,2106),(3532,'Experiment Method','Cap-Seq',3,2107),(3533,'Experiment Method','Cap-Seq',3,2108),(3534,'Experiment Method','Cap-Seq',3,2109),(3535,'Experiment Method','Cap-Seq',3,2110),(3536,'Experiment Method','Cap-Seq',3,2111),(3537,'Experiment Method','Cap-Seq',3,2112),(3538,'Experiment Method','Cap-Seq',3,2113),(3539,'Experiment Method','Cap-Seq',3,2114),(3540,'Experiment Method','Cap-Seq',3,2115),(3541,'Experiment Method','Cap-Seq',3,2116),(3542,'Experiment Method','Cap-Seq',3,2117),(3543,'Experiment Method','Cap-Seq',3,2118),(3544,'Experiment Method','Cap-Seq',3,2119),(3545,'Experiment Method','Cap-Seq',3,2120),(3546,'Experiment Method','Cap-Seq',3,2121),(3547,'Experiment Method','Cap-Seq',3,2122),(3548,'Experiment Method','Cap-Seq',3,2123),(3549,'Experiment Method','Cap-Seq',3,2124),(3550,'Experiment Method','Cap-Seq',3,2125),(3551,'Experiment Method','Cap-Seq',3,2126),(3552,'Experiment Method','Cap-Seq',3,2127),(3553,'Experiment Method','Cap-Seq',3,2128),(3554,'Experiment Method','Cap-Seq',3,2129),(3555,'Experiment Method','Cap-Seq',3,2130),(3556,'Experiment Method','Cap-Seq',3,2131),(3557,'Experiment Method','Cap-Seq',3,2298),(3558,'Experiment Method','Cap-Seq',3,2299),(3564,'Experiment Method','Cap-Seq',3,2453),(3569,'Experiment Method','Cap-Seq',3,2459),(3570,'Experiment Method','Cap-Seq',3,2460),(3571,'Experiment Method','Cap-Seq',3,2461),(3572,'Experiment Method','Cap-Seq',3,2462),(3573,'Experiment Method','Cap-Seq',3,2463),(3574,'Experiment Method','Cap-Seq',3,2464),(3575,'Experiment Method','Cap-Seq',3,2465),(3576,'Experiment Method','Cap-Seq',3,2466),(3577,'Experiment Method','Cap-Seq',3,2467),(3578,'Experiment Method','Cap-Seq',3,2536),(3579,'Experiment Method','Cap-Seq',3,2839),(3580,'Experiment Method','Cap-Seq',3,2840),(3581,'Experiment Method','Cap-Seq',3,2841),(3582,'Experiment Method','Cap-Seq',3,2842),(3583,'Experiment Method','Cap-Seq',3,2843),(3584,'Experiment Method','Cap-Seq',3,2844),(3585,'Experiment Method','Cap-Seq',3,2847),(3586,'Experiment Method','Cap-Seq',3,2849),(3587,'Experiment Method','Cap-Seq',3,2850),(3588,'Experiment Method','Cap-Seq',3,2851),(3589,'Experiment Method','Cap-Seq',3,2852),(3590,'Experiment Method','Cap-Seq',3,2853),(3591,'Experiment Method','Cap-Seq',3,2854),(3592,'Experiment Method','Cap-Seq',3,2855),(3593,'Experiment Method','Cap-Seq',3,2856),(3594,'Experiment Method','Cap-Seq',3,2857),(3595,'Experiment Method','Cap-Seq',3,3409),(3596,'Experiment Method','Cap-Seq',3,3410),(3597,'Experiment Method','Cap-Seq',3,3411),(3598,'Experiment Method','Cap-Seq',3,3412),(3599,'Experiment Method','Cap-Seq',3,3413),(3600,'Experiment Method','Cap-Seq',3,3414),(3601,'Experiment Method','Cap-Seq',3,3446),(3602,'Experiment Method','Cap-Seq',3,3447),(3603,'Experiment Method','Cap-Seq',3,3448),(3604,'Experiment Method','Cap-Seq',3,3449),(3605,'Experiment Method','Cap-Seq',3,3450),(3606,'Experiment Method','Cap-Seq',3,3451),(3607,'Experiment Method','Cap-Seq',3,3452),(3608,'Experiment Method','Cap-Seq',3,3453),(3617,'Experiment Method','Cap-Seq',3,3554),(3618,'Experiment Method','Cap-Seq',3,3555),(3619,'Experiment Method','Cap-Seq',3,3556),(3620,'Experiment Method','Cap-Seq',3,3557),(3621,'Experiment Method','Cap-Seq',3,3558),(3622,'Experiment Method','Cap-Seq',3,3559),(3623,'Experiment Method','Cap-Seq',3,3560),(3624,'Experiment Method','Cap-Seq',3,3561),(3629,'Experiment Method','Small RNA-Seq',3,2171),(3630,'Experiment Method','Small RNA-Seq',3,2172),(3631,'Experiment Method','Small RNA-Seq',3,2173),(3632,'Experiment Method','Small RNA-Seq',3,2174),(3633,'Experiment Method','Small RNA-Seq',3,2175),(3634,'Experiment Method','Small RNA-Seq',3,2176),(3635,'Experiment Method','Small RNA-Seq',3,2177),(3636,'Experiment Method','Small RNA-Seq',3,2178),(3637,'Experiment Method','Small RNA-Seq',3,2179),(3638,'Experiment Method','Small RNA-Seq',3,2180),(3639,'Experiment Method','Small RNA-Seq',3,2181),(3640,'Experiment Method','Small RNA-Seq',3,2182),(3641,'Experiment Method','Small RNA-Seq',3,2183),(3642,'Experiment Method','Small RNA-Seq',3,2779),(3643,'Experiment Method','Small RNA-Seq',3,2780),(3644,'Experiment Method','Small RNA-Seq',3,2781),(3645,'Experiment Method','Small RNA-Seq',3,2782),(3646,'Experiment Method','Bis-Seq',3,2273),(3647,'Experiment Method','Bis-Seq',3,2274),(3648,'Experiment Method','Bis-Seq',3,2275),(3649,'Experiment Method','Bis-Seq',3,2276),(3650,'Experiment Method','Bis-Seq',3,2277),(3651,'Experiment Method','Bis-Seq',3,2278),(3652,'Experiment Method','Bis-Seq',3,2279),(3653,'Experiment Method','Bis-Seq',3,2280),(3654,'Experiment Method','Bis-Seq',3,2281),(3655,'Experiment Method','Bis-Seq',3,2282),(3656,'Experiment Method','Bis-Seq',3,2283),(3657,'Experiment Method','Bis-Seq',3,2284),(3658,'Experiment Method','Bis-Seq',3,2350),(3659,'Experiment Method','Bis-Seq',3,2353),(3660,'Experiment Method','Bis-Seq',3,2354),(3661,'Experiment Method','Bis-Seq',3,2355),(3662,'Experiment Method','Bis-Seq',3,2356),(3663,'Experiment Method','Bis-Seq',3,3014),(3664,'Experiment Method','Bis-Seq',3,3015),(3665,'Experiment Method','Bis-Seq',3,3016),(3666,'Experiment Method','Bis-Seq',3,3017),(3667,'Experiment Method','Bis-Seq',3,3018),(3668,'Experiment Method','Bis-Seq',3,3019),(3669,'Experiment Method','Bis-Seq',3,3115),(3670,'Experiment Method','Bis-Seq',3,3116),(3671,'Experiment Method','Bis-Seq',3,3117),(3672,'Experiment Method','Bis-Seq',3,3118),(3673,'Experiment Method','Bis-Seq',3,3119),(3674,'Experiment Method','Bis-Seq',3,3120),(3675,'Experiment Method','Bis-Seq',3,3121),(3676,'Experiment Method','Bis-Seq',3,3122),(3677,'Experiment Method','Bis-Seq',3,3123),(3678,'Experiment Method','Bis-Seq',3,3124),(3679,'Experiment Method','Bis-Seq',3,3125),(3680,'Experiment Method','Bis-Seq',3,3126),(3681,'Experiment Method','Bis-Seq',3,3127),(3682,'Experiment Method','Bis-Seq',3,3128),(3683,'Experiment Method','Bis-Seq',3,3129),(3684,'Experiment Method','Bis-Seq',3,3130),(3685,'Experiment Method','Bis-Seq',3,3131),(3686,'Experiment Method','Bis-Seq',3,3132),(3687,'Experiment Method','Bis-Seq',3,3133),(3688,'Experiment Method','Bis-Seq',3,3134),(3689,'Experiment Method','Bis-Seq',3,3135),(3690,'Experiment Method','Bis-Seq',3,3136),(3691,'Experiment Method','Bis-Seq',3,3137),(3692,'Experiment Method','Bis-Seq',3,3138),(3693,'Experiment Method','Bis-Seq',3,3139),(3694,'Experiment Method','Bis-Seq',3,3140),(3695,'Experiment Method','Bis-Seq',3,3141),(3696,'Experiment Method','Bis-Seq',3,3142),(3697,'Experiment Method','Bis-Seq',3,3143),(3698,'Experiment Method','Bis-Seq',3,3144),(3699,'Experiment Method','Bis-Seq',3,3145),(3700,'Experiment Method','Bis-Seq',3,3146),(3701,'Experiment Method','Bis-Seq',3,3147),(3702,'Experiment Method','Bis-Seq',3,3148),(3703,'Experiment Method','Bis-Seq',3,3149),(3704,'Experiment Method','Bis-Seq',3,3150),(3705,'Experiment Method','Bis-Seq',3,3151),(3706,'Experiment Method','Bis-Seq',3,3152),(3707,'Experiment Method','Bis-Seq',3,3153),(3708,'Experiment Method','Bis-Seq',3,3154),(3709,'Experiment Method','Bis-Seq',3,3155),(3710,'Experiment Method','Bis-Seq',3,3156),(3711,'Experiment Method','Bis-Seq',3,3157),(3712,'Experiment Method','Bis-Seq',3,3158),(3713,'Experiment Method','Bis-Seq',3,3159),(3714,'Experiment Method','Bis-Seq',3,3160),(3715,'Experiment Method','Bis-Seq',3,3161),(3716,'Experiment Method','Bis-Seq',3,3162),(3717,'Experiment Method','Bis-Seq',3,3163),(3718,'Experiment Method','Bis-Seq',3,3164),(3719,'Experiment Method','Bis-Seq',3,3165),(3720,'Experiment Method','Bis-Seq',3,3166),(3721,'Experiment Method','Bis-Seq',3,3167),(3722,'Experiment Method','Bis-Seq',3,3168),(3723,'Experiment Method','Bis-Seq',3,3169),(3724,'Experiment Method','Bis-Seq',3,3170),(3725,'Experiment Method','Bis-Seq',3,3171),(3726,'Experiment Method','Bis-Seq',3,3172),(3727,'Experiment Method','Bis-Seq',3,3173),(3728,'Experiment Method','Bis-Seq',3,3174),(3729,'Experiment Method','Bis-Seq',3,3175),(3730,'Experiment Method','Bis-Seq',3,3176),(3731,'Experiment Method','Bis-Seq',3,3177),(3732,'Experiment Method','Bis-Seq',3,3178),(3733,'Experiment Method','Bis-Seq',3,3179),(3734,'Experiment Method','Bis-Seq',3,3180),(3735,'Experiment Method','Bis-Seq',3,3181),(3736,'Experiment Method','Bis-Seq',3,3182),(3737,'Experiment Method','Bis-Seq',3,3183),(3738,'Experiment Method','Bis-Seq',3,3184),(3739,'Experiment Method','Bis-Seq',3,3185),(3740,'Experiment Method','Bis-Seq',3,3186),(3741,'Experiment Method','Bis-Seq',3,3187),(3742,'Experiment Method','Bis-Seq',3,3188),(3743,'Experiment Method','Bis-Seq',3,3189),(3744,'Experiment Method','Bis-Seq',3,3190),(3745,'Experiment Method','Bis-Seq',3,3191),(3746,'Experiment Method','Bis-Seq',3,3192),(3747,'Experiment Method','Bis-Seq',3,3193),(3748,'Experiment Method','Bis-Seq',3,3194),(3749,'Experiment Method','Bis-Seq',3,3195),(3750,'Experiment Method','Bis-Seq',3,3196),(3751,'Experiment Method','Bis-Seq',3,3197),(3752,'Experiment Method','Bis-Seq',3,3198),(3753,'Experiment Method','Bis-Seq',3,3199),(3754,'Experiment Method','Bis-Seq',3,3200),(3755,'Experiment Method','Bis-Seq',3,3201),(3756,'Experiment Method','Bis-Seq',3,3202),(3757,'Experiment Method','Bis-Seq',3,3203),(3758,'Experiment Method','Bis-Seq',3,3204),(3759,'Experiment Method','Bis-Seq',3,3205),(3760,'Experiment Method','Bis-Seq',3,3206),(3761,'Experiment Method','Bis-Seq',3,3207),(3762,'Experiment Method','Bis-Seq',3,3208),(3763,'Experiment Method','Bis-Seq',3,3209),(3764,'Experiment Method','Bis-Seq',3,3210),(3765,'Experiment Method','Bis-Seq',3,3211),(3766,'Experiment Method','Bis-Seq',3,3212),(3767,'Experiment Method','Bis-Seq',3,3213),(3768,'Experiment Method','Bis-Seq',3,3214),(3769,'Experiment Method','Bis-Seq',3,3215),(3770,'Experiment Method','Bis-Seq',3,3216),(3771,'Experiment Method','Bis-Seq',3,3217),(3772,'Experiment Method','Bis-Seq',3,3218),(3773,'Experiment Method','Bis-Seq',3,3219),(3774,'Experiment Method','Bis-Seq',3,3220),(3775,'Experiment Method','Bis-Seq',3,3221),(3776,'Experiment Method','Bis-Seq',3,3222),(3777,'Experiment Method','Bis-Seq',3,3223),(3778,'Experiment Method','Bis-Seq',3,3224),(3779,'Experiment Method','Bis-Seq',3,3225),(3780,'Experiment Method','Bis-Seq',3,3226),(3781,'Experiment Method','Bis-Seq',3,3227),(3782,'Experiment Method','Bis-Seq',3,3228),(3783,'Experiment Method','Bis-Seq',3,3229),(3784,'Experiment Method','Bis-Seq',3,3230),(3785,'Experiment Method','Bis-Seq',3,3231),(3786,'Experiment Method','Bis-Seq',3,3232),(3787,'Experiment Method','Bis-Seq',3,3233),(3788,'Experiment Method','Bis-Seq',3,3234),(3789,'Experiment Method','Bis-Seq',3,3235),(3790,'Experiment Method','Bis-Seq',3,3236),(3791,'Experiment Method','Bis-Seq',3,3237),(3792,'Experiment Method','Bis-Seq',3,3250),(3793,'Experiment Method','Bis-Seq',3,3251),(3794,'Experiment Method','Bis-Seq',3,3252),(3795,'Experiment Method','Bis-Seq',3,3253),(3796,'Experiment Method','Bis-Seq',3,3254),(3797,'Experiment Method','Bis-Seq',3,3257),(3798,'Experiment Method','Bis-Seq',3,3275),(3799,'Experiment Method','Bis-Seq',3,3276),(3800,'Experiment Method','Bis-Seq',3,3286),(3801,'Experiment Method','Bis-Seq',3,3287),(3802,'Experiment Method','Bis-Seq',3,3288),(3803,'Experiment Method','Bis-Seq',3,3289),(3804,'Experiment Method','Bis-Seq',3,3290),(3805,'Experiment Method','Bis-Seq',3,3291),(3806,'Experiment Method','Bis-Seq',3,3292),(3807,'Experiment Method','Bis-Seq',3,3293),(3808,'Experiment Method','Bis-Seq',3,3294),(3809,'Experiment Method','Bis-Seq',3,3295),(3810,'Experiment Method','Bis-Seq',3,3296),(3811,'Experiment Method','Bis-Seq',3,3297),(3812,'Experiment Method','Bis-Seq',3,3298),(3813,'Experiment Method','Bis-Seq',3,3299),(3814,'Experiment Method','Bis-Seq',3,3300),(3815,'Experiment Method','Bis-Seq',3,3301),(3816,'Experiment Method','Bis-Seq',3,3302),(3817,'Experiment Method','Bis-Seq',3,3303),(3818,'Experiment Method','Bis-Seq',3,3304),(3819,'Experiment Method','Bis-Seq',3,3305),(3820,'Experiment Method','Bis-Seq',3,3306),(3821,'Experiment Method','Bis-Seq',3,3307),(3822,'Experiment Method','Bis-Seq',3,3308),(3823,'Experiment Method','Bis-Seq',3,3309),(3824,'Experiment Method','Bis-Seq',3,3310),(3825,'Experiment Method','Bis-Seq',3,3311),(3826,'Experiment Method','Bis-Seq',3,3312),(3827,'Experiment Method','Bis-Seq',3,3313),(3828,'Experiment Method','Bis-Seq',3,3314),(3829,'Experiment Method','Bis-Seq',3,3315),(3830,'Experiment Method','Bis-Seq',3,3316),(3831,'Experiment Method','Bis-Seq',3,3317),(3832,'Experiment Method','Bis-Seq',3,3324),(3833,'Experiment Method','Bis-Seq',3,3325),(3834,'Experiment Method','Bis-Seq',3,3326),(3835,'Experiment Method','Bis-Seq',3,3327),(3836,'Experiment Method','Bis-Seq',3,3328),(3837,'Experiment Method','Bis-Seq',3,3329),(3838,'Experiment Method','Bis-Seq',3,3330),(3839,'Experiment Method','Bis-Seq',3,3331),(3840,'Experiment Method','Bis-Seq',3,3332),(3841,'Experiment Method','Bis-Seq',3,3333),(3842,'Experiment Method','Bis-Seq',3,3334),(3843,'Experiment Method','Bis-Seq',3,3335),(3844,'Experiment Method','Bis-Seq',3,3336),(3845,'Experiment Method','Bis-Seq',3,3337),(3846,'Experiment Method','Bis-Seq',3,3338),(3847,'Experiment Method','Bis-Seq',3,3339),(3848,'Experiment Method','Bis-Seq',3,3340),(3849,'Experiment Method','Bis-Seq',3,3341),(3850,'Experiment Method','Bis-Seq',3,3342),(3851,'Experiment Method','Bis-Seq',3,3343),(3852,'Experiment Method','Bis-Seq',3,3344),(3853,'Experiment Method','Bis-Seq',3,3345),(3854,'Experiment Method','Bis-Seq',3,3346),(3855,'Experiment Method','Bis-Seq',3,3347),(3856,'Experiment Method','Bis-Seq',3,3348),(3857,'Experiment Method','Bis-Seq',3,3349),(3858,'Experiment Method','Bis-Seq',3,3350),(3859,'Experiment Method','Bis-Seq',3,3351),(3860,'Experiment Method','Bis-Seq',3,3352),(3861,'Experiment Method','Bis-Seq',3,3353),(3862,'Experiment Method','Bis-Seq',3,3354),(3863,'Experiment Method','Bis-Seq',3,3355),(3864,'Experiment Method','Bis-Seq',3,3356),(3865,'Experiment Method','Bis-Seq',3,3357),(3866,'Experiment Method','Bis-Seq',3,3358),(3867,'Experiment Method','Bis-Seq',3,3359),(3868,'Experiment Method','Bis-Seq',3,3360),(3869,'Experiment Method','Bis-Seq',3,3361),(3870,'Experiment Method','Bis-Seq',3,3362),(3871,'Experiment Method','Bis-Seq',3,3363),(3872,'Experiment Method','Bis-Seq',3,3364),(3873,'Experiment Method','Bis-Seq',3,3365),(3874,'Experiment Method','Bis-Seq',3,3366),(3875,'Experiment Method','Bis-Seq',3,3367),(3876,'Experiment Method','Bis-Seq',3,3368),(3877,'Experiment Method','Bis-Seq',3,3369),(3878,'Experiment Method','Bis-Seq',3,3370),(3879,'Experiment Method','Bis-Seq',3,3371),(3880,'Experiment Method','Bis-Seq',3,3566),(3881,'Experiment Method','Bis-Seq',3,3567),(3882,'Experiment Method','Bis-Seq',3,3568),(3883,'Experiment Method','Bis-Seq',3,3569),(3884,'Experiment Method','Bis-Seq',3,3570),(3885,'Experiment Method','Bis-Seq',3,3571),(3886,'Experiment Method','Bis-Seq',3,3572),(3887,'Experiment Method','Bis-Seq',3,3573),(3888,'Experiment Method','Bis-Seq',3,3574),(3889,'Experiment Method','Bis-Seq',3,3575),(3890,'Experiment Method','Bis-Seq',3,3576),(3891,'Experiment Method','Bis-Seq',3,3577),(3892,'Experiment Method','Bis-Seq',3,3578),(3893,'Experiment Method','Bis-Seq',3,3579),(3894,'Experiment Method','Bis-Seq',3,3580),(3895,'Experiment Method','Bis-Seq',3,3581),(3896,'Experiment Method','Bis-Seq',3,3582),(3897,'Experiment Method','Bis-Seq',3,3583),(3898,'Experiment Method','Bis-Seq',3,3584),(3899,'Experiment Method','Bis-Seq',3,3585),(3900,'Experiment Method','Bis-Seq',3,3586),(3901,'Experiment Method','Bis-Seq',3,3587),(3902,'Experiment Method','Bis-Seq',3,3588),(3903,'Experiment Method','Bis-Seq',3,3589),(3904,'Experiment Method','Bis-Seq',3,3596),(3905,'Experiment Method','Bis-Seq',3,3597),(3906,'Experiment Method','Bis-Seq',3,3598),(3907,'Experiment Method','Bis-Seq',3,3599),(3908,'Experiment Method','Bis-Seq',3,3600),(3909,'Experiment Method','Bis-Seq',3,3601),(3910,'Experiment Method','Bis-Seq',3,3602),(3911,'Experiment Method','Bis-Seq',3,3603),(3912,'Experiment Method','Bis-Seq',3,3604),(3913,'Experiment Method','Bis-Seq',3,3605),(3914,'Experiment Method','Bis-Seq',3,3606),(3915,'Experiment Method','Bis-Seq',3,3607),(3916,'Experiment Method','Bis-Seq',3,3608),(3917,'Experiment Method','Bis-Seq',3,3609),(3918,'Experiment Method','Bis-Seq',3,3610),(3919,'Experiment Method','Bis-Seq',3,3611),(3920,'Experiment Method','Bis-Seq',3,3612),(3921,'Experiment Method','Bis-Seq',3,3613),(3922,'Experiment Method','Bis-Seq',3,3614),(3923,'Experiment Method','Bis-Seq',3,3615),(3924,'Experiment Method','Bis-Seq',3,3616),(3925,'Experiment Method','Bis-Seq',3,3617),(3926,'Experiment Method','Bis-Seq',3,3618),(3927,'Experiment Method','Bis-Seq',3,3619),(3928,'Experiment Method','Whole Genome Sequencing',3,2946),(3929,'Experiment Method','Whole Genome Sequencing',3,2952),(3930,'Experiment Method','Whole Genome Sequencing',3,2953),(3931,'Experiment Method','Whole Genome Sequencing',3,2954),(3932,'Experiment Method','Whole Genome Sequencing',3,2962),(3933,'Experiment Method','Whole Genome Sequencing',3,3486),(3934,'Experiment Method','Whole Genome Sequencing',3,3487),(3938,'Analysis Type','Static',1,3620),(3939,'Experiment Method','chIP-Seq',3,3620),(3940,'Experiment Platform','Illumina Genome Analyzer',2,3620),(3941,'Analysis Type','Static',1,3621),(3942,'Experiment Method','chIP-Seq',3,3621),(3943,'Experiment Platform','Illumina Genome Analyzer',2,3621),(3944,'Analysis Type','Static',1,3622),(3945,'Experiment Method','chIP-Seq',3,3622),(3946,'Experiment Platform','Illumina Genome Analyzer',2,3622),(3947,'Analysis Type','Static',1,3623),(3948,'Experiment Method','chIP-Seq',3,3623),(3949,'Experiment Platform','',2,3623),(3950,'Analysis Type','Static',1,3624),(3951,'Experiment Method','chIP-Seq',3,3624),(3952,'Experiment Platform','Illumina Genome Analyzer',2,3624),(3953,'Analysis Type','',1,3625),(3954,'Experiment Method','',3,3625),(3955,'Experiment Platform','',2,3625),(3956,'Analysis Type','',1,3626),(3957,'Experiment Method','',3,3626),(3958,'Experiment Platform','',2,3626),(3959,'Analysis Type','',1,3627),(3960,'Experiment Method','',3,3627),(3961,'Experiment Platform','',2,3627),(3962,'Analysis Type','',1,3628),(3963,'Experiment Method','',3,3628),(3964,'Experiment Platform','',2,3628),(3965,'Analysis Type','',1,3629),(3966,'Experiment Method','',3,3629),(3967,'Experiment Platform','',2,3629),(3971,'Analysis Type','',1,3631),(3972,'Experiment Method','',3,3631),(3973,'Experiment Platform','',2,3631),(3974,'Analysis Type','',1,3632),(3975,'Experiment Method','',3,3632),(3976,'Experiment Platform','',2,3632),(3977,'Analysis Type','',1,3633),(3978,'Experiment Method','',3,3633),(3979,'Experiment Platform','',2,3633),(3983,'Analysis Type','',1,3635),(3984,'Experiment Method','',3,3635),(3985,'Experiment Platform','',2,3635),(3986,'Analysis Type','',1,3636),(3987,'Experiment Method','',3,3636),(3988,'Experiment Platform','',2,3636),(3989,'Analysis Type','',1,3637),(3990,'Experiment Method','',3,3637),(3991,'Experiment Platform','',2,3637),(3992,'Analysis Type','',1,3638),(3993,'Experiment Method','',3,3638),(3994,'Experiment Platform','',2,3638),(3995,'Analysis Type','',1,2494),(3996,'Experiment Method','',3,2494),(3997,'Experiment Platform','',2,2494),(3998,'Analysis Type','',1,3639),(3999,'Experiment Method','',3,3639),(4000,'Experiment Platform','',2,3639),(4001,'Analysis Type','Static',1,3641),(4002,'Experiment Method','RNA-Seq',3,3641),(4003,'Experiment Platform','Illumina Genome Analyzer',2,3641),(4004,'Analysis Type','Static',1,3642),(4005,'Experiment Method','RNA-Seq',3,3642),(4006,'Experiment Platform','Illumina Genome Analyzer',2,3642),(4007,'Analysis Type','Static',1,3643),(4008,'Experiment Method','RNA-Seq',3,3643),(4009,'Experiment Platform','Illumina Genome Analyzer',2,3643),(4016,'Analysis Type','',1,3651),(4017,'Experiment Method','',3,3651),(4018,'Experiment Platform','',2,3651),(4019,'Analysis Type','',1,3650),(4020,'Experiment Method','',3,3650),(4021,'Experiment Platform','',2,3650),(4022,'Analysis Type','',1,3649),(4023,'Experiment Method','',3,3649),(4024,'Experiment Platform','',2,3649),(4025,'Analysis Type','',1,3648),(4026,'Experiment Method','',3,3648),(4027,'Experiment Platform','',2,3648),(4031,'Analysis Type','',1,3653),(4032,'Experiment Method','',3,3653),(4033,'Experiment Platform','',2,3653),(4040,'Analysis Type','',1,3001),(4041,'Experiment Method','',3,3001),(4042,'Experiment Platform','',2,3001),(4058,'Analysis Type','Static',1,3678),(4059,'Experiment Method','',3,3678),(4060,'Experiment Platform','Illumina Genome Analyzer',2,3678),(4061,'Analysis Type','',1,3679),(4062,'Experiment Method','',3,3679),(4063,'Experiment Platform','',2,3679),(4064,'Analysis Type','',1,3680),(4065,'Experiment Method','',3,3680),(4066,'Experiment Platform','',2,3680),(4067,'Analysis Type','Static',1,3681),(4068,'Experiment Method','',3,3681),(4069,'Experiment Platform','Illumina Genome Analyzer',2,3681),(4070,'Analysis Type','Static',1,3682),(4071,'Experiment Method','',3,3682),(4072,'Experiment Platform','Illumina Genome Analyzer',2,3682),(4073,'Analysis Type','Static',1,3683),(4074,'Experiment Method','',3,3683),(4075,'Experiment Platform','Illumina Genome Analyzer',2,3683),(4076,'Analysis Type','Static',1,3684),(4077,'Experiment Method','',3,3684),(4078,'Experiment Platform','Illumina Genome Analyzer',2,3684),(4079,'Analysis Type','',1,3685),(4080,'Experiment Method','',3,3685),(4081,'Experiment Platform','',2,3685),(4082,'Analysis Type','',1,3686),(4083,'Experiment Method','',3,3686),(4084,'Experiment Platform','',2,3686),(4085,'Analysis Type','',1,3700),(4086,'Experiment Method','',3,3700),(4087,'Experiment Platform','',2,3700),(4088,'Analysis Type','',1,3698),(4089,'Experiment Method','',3,3698),(4090,'Experiment Platform','',2,3698),(4091,'Analysis Type','',1,3699),(4092,'Experiment Method','',3,3699),(4093,'Experiment Platform','',2,3699),(4094,'Analysis Type','',1,3704),(4095,'Experiment Method','',3,3704),(4096,'Experiment Platform','',2,3704),(4097,'Analysis Type','',1,3703),(4098,'Experiment Method','',3,3703),(4099,'Experiment Platform','',2,3703),(4100,'Analysis Type','',1,3706),(4101,'Experiment Method','',3,3706),(4102,'Experiment Platform','',2,3706),(4103,'Analysis Type','',1,3705),(4104,'Experiment Method','',3,3705),(4105,'Experiment Platform','',2,3705),(4106,'Analysis Type','',1,3707),(4107,'Experiment Method','',3,3707),(4108,'Experiment Platform','',2,3707),(4109,'Analysis Type','',1,3710),(4110,'Experiment Method','',3,3710),(4111,'Experiment Platform','',2,3710),(4112,'Analysis Type','',1,3709),(4113,'Experiment Method','',3,3709),(4114,'Experiment Platform','',2,3709),(4115,'Analysis Type','',1,3708),(4116,'Experiment Method','',3,3708),(4117,'Experiment Platform','',2,3708),(4118,'Analysis Type','',1,3712),(4119,'Experiment Method','',3,3712),(4120,'Experiment Platform','',2,3712),(4145,'Analysis Type','',1,3725),(4146,'Experiment Method','',3,3725),(4147,'Experiment Platform','',2,3725),(4148,'Analysis Type','Static',1,3726),(4149,'Experiment Method','RNA-Seq',3,3726),(4150,'Experiment Platform','Illumina Genome Analyzer',2,3726),(4151,'Analysis Type','',1,3727),(4152,'Experiment Method','',3,3727),(4153,'Experiment Platform','',2,3727),(4154,'Analysis Type','Static',1,3728),(4155,'Experiment Method','RNA-Seq',3,3728),(4156,'Experiment Platform','Illumina Genome Analyzer',2,3728),(4157,'Analysis Type','',1,3729),(4158,'Experiment Method','',3,3729),(4159,'Experiment Platform','',2,3729),(4160,'Analysis Type','',1,3730),(4161,'Experiment Method','',3,3730),(4162,'Experiment Platform','',2,3730),(4163,'Analysis Type','',1,3731),(4164,'Experiment Method','',3,3731),(4165,'Experiment Platform','',2,3731),(4166,'Analysis Type','',1,3732),(4167,'Experiment Method','',3,3732),(4168,'Experiment Platform','',2,3732),(4169,'Analysis Type','',1,3733),(4170,'Experiment Method','',3,3733),(4171,'Experiment Platform','',2,3733),(4172,'Analysis Type','',1,3734),(4173,'Experiment Method','',3,3734),(4174,'Experiment Platform','',2,3734),(4175,'Analysis Type','',1,3735),(4176,'Experiment Method','',3,3735),(4177,'Experiment Platform','',2,3735),(4178,'Analysis Type','',1,3736),(4179,'Experiment Method','',3,3736),(4180,'Experiment Platform','',2,3736),(4181,'Analysis Type','',1,3737),(4182,'Experiment Method','',3,3737),(4183,'Experiment Platform','',2,3737),(4184,'Analysis Type','Dynamic/Differential',1,3738),(4185,'Experiment Method','RNA-Seq',3,3738),(4186,'Experiment Platform','Illumina Genome Analyzer',2,3738),(4193,'Analysis Type','',1,3741),(4194,'Experiment Method','',3,3741),(4195,'Experiment Platform','',2,3741),(4202,'Analysis Type','',1,3744),(4203,'Experiment Method','',3,3744),(4204,'Experiment Platform','',2,3744),(4205,'Analysis Type','Static',1,3751),(4206,'Experiment Method','Bis-Seq',3,3751),(4207,'Experiment Platform','Illumina Genome Analyzer',2,3751),(4208,'Analysis Type','Static',1,3754),(4209,'Experiment Method','chIP-Seq',3,3754),(4210,'Experiment Platform','Illumina Genome Analyzer',2,3754),(4211,'Analysis Type','Static',1,3755),(4212,'Experiment Method','chIP-Seq',3,3755),(4213,'Experiment Platform','Illumina Genome Analyzer',2,3755),(4214,'Analysis Type','Static',1,3757),(4215,'Experiment Method','chIP-Seq',3,3757),(4216,'Experiment Platform','Illumina Genome Analyzer',2,3757),(4221,'Analysis Type','Static',1,3764),(4222,'Analysis URL Links','',4,3764),(4223,'Experiment Method','Bis-Seq',3,3764),(4224,'Experiment Platform','Illumina Genome Analyzer',2,3764),(4225,'Analysis Type','Static',1,3765),(4226,'Analysis URL Links','',4,3765),(4227,'Experiment Method','Bis-Seq',3,3765),(4228,'Experiment Platform','Illumina Genome Analyzer',2,3765),(4229,'Analysis Type','Static',1,3766),(4230,'Analysis URL Links','',4,3766),(4231,'Experiment Method','Bis-Seq',3,3766),(4232,'Experiment Platform','Illumina Genome Analyzer',2,3766),(4233,'Analysis Type','Static',1,3767),(4234,'Analysis URL Links','',4,3767),(4235,'Experiment Method','Bis-Seq',3,3767),(4236,'Experiment Platform','Illumina Genome Analyzer',2,3767),(4237,'Analysis Type','',1,3),(4238,'Analysis URL Links','',4,3),(4239,'Experiment Method','',3,3),(4240,'Experiment Platform','',2,3),(4241,'Analysis URL Links','',4,2219),(4242,'Analysis URL Links','',4,2221),(4243,'Analysis URL Links','',4,2220),(4244,'Analysis URL Links','',4,2816),(4245,'Analysis URL Links','',4,2213),(4246,'Analysis URL Links','',4,2214),(4247,'Analysis URL Links','',4,2217),(4248,'Analysis URL Links','',4,2216),(4249,'Analysis URL Links','',4,2212),(4250,'Analysis URL Links','',4,2211),(4251,'Analysis Type','Static',1,3776),(4252,'Analysis URL Links','',4,3776),(4253,'Experiment Method','chIP-Seq',3,3776),(4254,'Experiment Platform','Illumina Genome Analyzer',2,3776),(4259,'Analysis Type','',1,3779),(4260,'Analysis URL Links','',4,3779),(4261,'Experiment Method','',3,3779),(4262,'Experiment Platform','',2,3779),(4263,'Analysis Type','Static',1,3780),(4264,'Analysis URL Links','',4,3780),(4265,'Experiment Method','Read Counts/ Coverage',3,3780),(4266,'Experiment Platform','Illumina Genome Analyzer',2,3780),(4267,'Analysis Type','Static',1,3781),(4268,'Analysis URL Links','',4,3781),(4269,'Experiment Method','Read Counts/ Coverage',3,3781),(4270,'Experiment Platform','Illumina Genome Analyzer',2,3781),(4271,'Analysis Type','Static',1,3782),(4272,'Analysis URL Links','',4,3782),(4273,'Experiment Method','Read Counts/ Coverage',3,3782),(4274,'Experiment Platform','Illumina Genome Analyzer',2,3782),(4275,'Analysis Type','Static',1,3783),(4276,'Analysis URL Links','',4,3783),(4277,'Experiment Method','Read Counts/ Coverage',3,3783),(4278,'Experiment Platform','Illumina Genome Analyzer',2,3783),(4279,'Analysis Type','Static',1,3784),(4280,'Analysis URL Links','',4,3784),(4281,'Experiment Method','Read Counts/ Coverage',3,3784),(4282,'Experiment Platform','Illumina Genome Analyzer',2,3784),(4283,'Analysis Type','Static',1,3785),(4284,'Analysis URL Links','',4,3785),(4285,'Experiment Method','Read Counts/ Coverage',3,3785),(4286,'Experiment Platform','Illumina Genome Analyzer',2,3785),(4287,'Analysis Type','Static',1,3786),(4288,'Analysis URL Links','',4,3786),(4289,'Experiment Method','Read Counts/ Coverage',3,3786),(4290,'Experiment Platform','Illumina Genome Analyzer',2,3786),(4291,'Analysis Type','Static',1,3787),(4292,'Analysis URL Links','',4,3787),(4293,'Experiment Method','Read Counts/ Coverage',3,3787),(4294,'Experiment Platform','Illumina Genome Analyzer',2,3787),(4295,'Analysis Type','Static',1,3788),(4296,'Analysis URL Links','',4,3788),(4297,'Experiment Method','Read Counts/ Coverage',3,3788),(4298,'Experiment Platform','Illumina Genome Analyzer',2,3788),(4299,'Analysis Type','Static',1,3789),(4300,'Analysis URL Links','',4,3789),(4301,'Experiment Method','Read Counts/ Coverage',3,3789),(4302,'Experiment Platform','Illumina Genome Analyzer',2,3789),(4303,'Analysis Type','Static',1,3790),(4304,'Analysis URL Links','',4,3790),(4305,'Experiment Method','Read Counts/ Coverage',3,3790),(4306,'Experiment Platform','Illumina Genome Analyzer',2,3790),(4307,'Analysis Type','Static',1,3791),(4308,'Analysis URL Links','',4,3791),(4309,'Experiment Method','Read Counts/ Coverage',3,3791),(4310,'Experiment Platform','Illumina Genome Analyzer',2,3791),(4311,'Analysis Type','Static',1,3792),(4312,'Analysis URL Links','',4,3792),(4313,'Experiment Method','Read Counts/ Coverage',3,3792),(4314,'Experiment Platform','Illumina Genome Analyzer',2,3792),(4315,'Analysis Type','Static',1,3793),(4316,'Analysis URL Links','',4,3793),(4317,'Experiment Method','Read Counts/ Coverage',3,3793),(4318,'Experiment Platform','Illumina Genome Analyzer',2,3793),(4319,'Analysis Type','',1,3794),(4320,'Analysis URL Links','',4,3794),(4321,'Experiment Method','',3,3794),(4322,'Experiment Platform','',2,3794),(4323,'Analysis Type','Static',1,3805),(4324,'Analysis URL Links','',4,3805),(4325,'Experiment Method','SNP Microarray',3,3805),(4326,'Experiment Platform','Affymetrix Microarray',2,3805),(4327,'Analysis Type','Static',1,3806),(4328,'Analysis URL Links','',4,3806),(4329,'Experiment Method','SNP Microarray',3,3806),(4330,'Experiment Platform','Affymetrix Microarray',2,3806),(4331,'Analysis Type','Static',1,3807),(4332,'Analysis URL Links','',4,3807),(4333,'Experiment Method','SNP Microarray',3,3807),(4334,'Experiment Platform','Affymetrix Microarray',2,3807),(4335,'Analysis Type','Static',1,3808),(4336,'Analysis URL Links','',4,3808),(4337,'Experiment Method','SNP Microarray',3,3808),(4338,'Experiment Platform','Affymetrix Microarray',2,3808),(4339,'Analysis Type','Static',1,3809),(4340,'Analysis URL Links','',4,3809),(4341,'Experiment Method','SNP Microarray',3,3809),(4342,'Experiment Platform','Affymetrix Microarray',2,3809),(4343,'Analysis Type','Static',1,3810),(4344,'Analysis URL Links','',4,3810),(4345,'Experiment Method','SNP Microarray',3,3810),(4346,'Experiment Platform','Affymetrix Microarray',2,3810),(4347,'Analysis Type','Static',1,3811),(4348,'Analysis URL Links','',4,3811),(4349,'Experiment Method','SNP Microarray',3,3811),(4350,'Experiment Platform','Affymetrix Microarray',2,3811),(4351,'Analysis Type','Static',1,3812),(4352,'Analysis URL Links','',4,3812),(4353,'Experiment Method','SNP Microarray',3,3812),(4354,'Experiment Platform','Affymetrix Microarray',2,3812),(4355,'Analysis Type','Static',1,3813),(4356,'Analysis URL Links','',4,3813),(4357,'Experiment Method','SNP Microarray',3,3813),(4358,'Experiment Platform','Affymetrix Microarray',2,3813),(4359,'Analysis Type','Static',1,3814),(4360,'Analysis URL Links','',4,3814),(4361,'Experiment Method','SNP Microarray',3,3814),(4362,'Experiment Platform','Affymetrix Microarray',2,3814),(4363,'Analysis Type','Static',1,3815),(4364,'Analysis URL Links','',4,3815),(4365,'Experiment Method','SNP Microarray',3,3815),(4366,'Experiment Platform','Affymetrix Microarray',2,3815),(4367,'Analysis Type','Static',1,3816),(4368,'Analysis URL Links','',4,3816),(4369,'Experiment Method','SNP Microarray',3,3816),(4370,'Experiment Platform','Affymetrix Microarray',2,3816),(4371,'Analysis Type','Static',1,3817),(4372,'Analysis URL Links','',4,3817),(4373,'Experiment Method','SNP Microarray',3,3817),(4374,'Experiment Platform','Affymetrix Microarray',2,3817),(4375,'Analysis Type','Static',1,3818),(4376,'Analysis URL Links','',4,3818),(4377,'Experiment Method','SNP Microarray',3,3818),(4378,'Experiment Platform','Affymetrix Microarray',2,3818),(4379,'Analysis Type','Static',1,3819),(4380,'Analysis URL Links','',4,3819),(4381,'Experiment Method','SNP Microarray',3,3819),(4382,'Experiment Platform','Affymetrix Microarray',2,3819),(4383,'Analysis Type','Static',1,3820),(4384,'Analysis URL Links','',4,3820),(4385,'Experiment Method','SNP Microarray',3,3820),(4386,'Experiment Platform','Affymetrix Microarray',2,3820),(4387,'Analysis Type','Static',1,3821),(4388,'Analysis URL Links','',4,3821),(4389,'Experiment Method','SNP Microarray',3,3821),(4390,'Experiment Platform','Affymetrix Microarray',2,3821),(4391,'Analysis Type','Static',1,3822),(4392,'Analysis URL Links','',4,3822),(4393,'Experiment Method','SNP Microarray',3,3822),(4394,'Experiment Platform','Affymetrix Microarray',2,3822),(4395,'Analysis Type','Static',1,3823),(4396,'Analysis URL Links','',4,3823),(4397,'Experiment Method','SNP Microarray',3,3823),(4398,'Experiment Platform','Affymetrix Microarray',2,3823),(4399,'Analysis Type','Static',1,3824),(4400,'Analysis URL Links','',4,3824),(4401,'Experiment Method','SNP Microarray',3,3824),(4402,'Experiment Platform','Affymetrix Microarray',2,3824),(4403,'Analysis Type','Static',1,3825),(4404,'Analysis URL Links','',4,3825),(4405,'Experiment Method','SNP Microarray',3,3825),(4406,'Experiment Platform','Affymetrix Microarray',2,3825),(4407,'Analysis Type','Static',1,3826),(4408,'Analysis URL Links','',4,3826),(4409,'Experiment Method','SNP Microarray',3,3826),(4410,'Experiment Platform','Affymetrix Microarray',2,3826),(4411,'Analysis Type','Static',1,3827),(4412,'Analysis URL Links','',4,3827),(4413,'Experiment Method','SNP Microarray',3,3827),(4414,'Experiment Platform','Affymetrix Microarray',2,3827),(4415,'Analysis Type','Static',1,3828),(4416,'Analysis URL Links','',4,3828),(4417,'Experiment Method','SNP Microarray',3,3828),(4418,'Experiment Platform','Affymetrix Microarray',2,3828),(4419,'Analysis Type','Static',1,3829),(4420,'Analysis URL Links','',4,3829),(4421,'Experiment Method','SNP Microarray',3,3829),(4422,'Experiment Platform','Affymetrix Microarray',2,3829),(4423,'Analysis Type','Static',1,3830),(4424,'Analysis URL Links','',4,3830),(4425,'Experiment Method','SNP Microarray',3,3830),(4426,'Experiment Platform','Affymetrix Microarray',2,3830),(4427,'Analysis Type','Static',1,3831),(4428,'Analysis URL Links','',4,3831),(4429,'Experiment Method','SNP Microarray',3,3831),(4430,'Experiment Platform','Affymetrix Microarray',2,3831),(4431,'Analysis Type','Static',1,3832),(4432,'Analysis URL Links','',4,3832),(4433,'Experiment Method','SNP Microarray',3,3832),(4434,'Experiment Platform','Affymetrix Microarray',2,3832),(4435,'Analysis Type','Static',1,3833),(4436,'Analysis URL Links','',4,3833),(4437,'Experiment Method','SNP Microarray',3,3833),(4438,'Experiment Platform','Affymetrix Microarray',2,3833),(4439,'Analysis Type','Static',1,3834),(4440,'Analysis URL Links','',4,3834),(4441,'Experiment Method','SNP Microarray',3,3834),(4442,'Experiment Platform','Affymetrix Microarray',2,3834),(4443,'Analysis Type','Static',1,3835),(4444,'Analysis URL Links','',4,3835),(4445,'Experiment Method','SNP Microarray',3,3835);


INSERT INTO dbo.gpAnnotationProperty
(idAnnotationProperty,
name,
value,
idProperty,
idAnnotation
) VALUES
(4446,'Experiment Platform','Affymetrix Microarray',2,3835),(4447,'Analysis Type','Static',1,3836),(4448,'Analysis URL Links','',4,3836),(4449,'Experiment Method','SNP Microarray',3,3836),(4450,'Experiment Platform','Affymetrix Microarray',2,3836),(4451,'Analysis Type','Static',1,3837),(4452,'Analysis URL Links','',4,3837),(4453,'Experiment Method','SNP Microarray',3,3837),(4454,'Experiment Platform','Affymetrix Microarray',2,3837),(4455,'Analysis Type','Static',1,3838),(4456,'Analysis URL Links','',4,3838),(4457,'Experiment Method','SNP Microarray',3,3838),(4458,'Experiment Platform','Affymetrix Microarray',2,3838),(4459,'Analysis Type','Static',1,3839),(4460,'Analysis URL Links','',4,3839),(4461,'Experiment Method','SNP Microarray',3,3839),(4462,'Experiment Platform','Affymetrix Microarray',2,3839),(4463,'Analysis Type','Static',1,3840),(4464,'Analysis URL Links','',4,3840),(4465,'Experiment Method','SNP Microarray',3,3840),(4466,'Experiment Platform','Affymetrix Microarray',2,3840),(4467,'Analysis Type','Static',1,3841),(4468,'Analysis URL Links','',4,3841),(4469,'Experiment Method','SNP Microarray',3,3841),(4470,'Experiment Platform','Affymetrix Microarray',2,3841),(4471,'Analysis Type','Static',1,3842),(4472,'Analysis URL Links','',4,3842),(4473,'Experiment Method','SNP Microarray',3,3842),(4474,'Experiment Platform','Affymetrix Microarray',2,3842),(4475,'Analysis Type','Static',1,3843),(4476,'Analysis URL Links','',4,3843),(4477,'Experiment Method','SNP Microarray',3,3843),(4478,'Experiment Platform','Affymetrix Microarray',2,3843),(4479,'Analysis Type','Static',1,3844),(4480,'Analysis URL Links','',4,3844),(4481,'Experiment Method','SNP Microarray',3,3844),(4482,'Experiment Platform','Affymetrix Microarray',2,3844),(4483,'Analysis Type','Static',1,3845),(4484,'Analysis URL Links','',4,3845),(4485,'Experiment Method','SNP Microarray',3,3845),(4486,'Experiment Platform','Affymetrix Microarray',2,3845),(4487,'Analysis Type','Static',1,3846),(4488,'Analysis URL Links','',4,3846),(4489,'Experiment Method','SNP Microarray',3,3846),(4490,'Experiment Platform','Affymetrix Microarray',2,3846),(4491,'Analysis Type','Static',1,3847),(4492,'Analysis URL Links','',4,3847),(4493,'Experiment Method','SNP Microarray',3,3847),(4494,'Experiment Platform','Affymetrix Microarray',2,3847),(4495,'Analysis Type','Static',1,3848),(4496,'Analysis URL Links','',4,3848),(4497,'Experiment Method','SNP Microarray',3,3848),(4498,'Experiment Platform','Affymetrix Microarray',2,3848),(4499,'Analysis Type','Static',1,3849),(4500,'Analysis URL Links','',4,3849),(4501,'Experiment Method','SNP Microarray',3,3849),(4502,'Experiment Platform','Affymetrix Microarray',2,3849),(4503,'Analysis Type','Static',1,3850),(4504,'Analysis URL Links','',4,3850),(4505,'Experiment Method','SNP Microarray',3,3850),(4506,'Experiment Platform','Affymetrix Microarray',2,3850),(4507,'Analysis Type','Static',1,3851),(4508,'Analysis URL Links','',4,3851),(4509,'Experiment Method','SNP Microarray',3,3851),(4510,'Experiment Platform','Affymetrix Microarray',2,3851),(4511,'Analysis Type','Static',1,3852),(4512,'Analysis URL Links','',4,3852),(4513,'Experiment Method','SNP Microarray',3,3852),(4514,'Experiment Platform','Affymetrix Microarray',2,3852),(4515,'Analysis Type','Static',1,3853),(4516,'Analysis URL Links','',4,3853),(4517,'Experiment Method','SNP Microarray',3,3853),(4518,'Experiment Platform','Affymetrix Microarray',2,3853),(4519,'Analysis Type','Static',1,3854),(4520,'Analysis URL Links','',4,3854),(4521,'Experiment Method','SNP Microarray',3,3854),(4522,'Experiment Platform','Affymetrix Microarray',2,3854),(4523,'Analysis Type','Static',1,3855),(4524,'Analysis URL Links','',4,3855),(4525,'Experiment Method','SNP Microarray',3,3855),(4526,'Experiment Platform','Affymetrix Microarray',2,3855),(4527,'Analysis Type','Static',1,3856),(4528,'Analysis URL Links','',4,3856),(4529,'Experiment Method','SNP Microarray',3,3856),(4530,'Experiment Platform','Affymetrix Microarray',2,3856),(4531,'Analysis Type','Static',1,3857),(4532,'Analysis URL Links','',4,3857),(4533,'Experiment Method','SNP Microarray',3,3857),(4534,'Experiment Platform','Affymetrix Microarray',2,3857),(4535,'Analysis Type','Static',1,3858),(4536,'Analysis URL Links','',4,3858),(4537,'Experiment Method','SNP Microarray',3,3858),(4538,'Experiment Platform','Affymetrix Microarray',2,3858),(4539,'Analysis Type','Static',1,3859),(4540,'Analysis URL Links','',4,3859),(4541,'Experiment Method','SNP Microarray',3,3859),(4542,'Experiment Platform','Affymetrix Microarray',2,3859),(4543,'Analysis Type','Static',1,3860),(4544,'Analysis URL Links','',4,3860),(4545,'Experiment Method','SNP Microarray',3,3860),(4546,'Experiment Platform','Affymetrix Microarray',2,3860),(4547,'Analysis Type','Static',1,3861),(4548,'Analysis URL Links','',4,3861),(4549,'Experiment Method','SNP Microarray',3,3861),(4550,'Experiment Platform','Affymetrix Microarray',2,3861),(4551,'Analysis Type','Static',1,3862),(4552,'Analysis URL Links','',4,3862),(4553,'Experiment Method','SNP Microarray',3,3862),(4554,'Experiment Platform','Affymetrix Microarray',2,3862),(4555,'Analysis Type','Static',1,3863),(4556,'Analysis URL Links','',4,3863),(4557,'Experiment Method','SNP Microarray',3,3863),(4558,'Experiment Platform','Affymetrix Microarray',2,3863),(4559,'Analysis Type','Static',1,3864),(4560,'Analysis URL Links','',4,3864),(4561,'Experiment Method','SNP Microarray',3,3864),(4562,'Experiment Platform','Affymetrix Microarray',2,3864),(4563,'Analysis Type','Static',1,3865),(4564,'Analysis URL Links','',4,3865),(4565,'Experiment Method','SNP Microarray',3,3865),(4566,'Experiment Platform','Affymetrix Microarray',2,3865),(4567,'Analysis Type','Static',1,3866),(4568,'Analysis URL Links','',4,3866),(4569,'Experiment Method','SNP Microarray',3,3866),(4570,'Experiment Platform','Affymetrix Microarray',2,3866),(4571,'Analysis Type','Static',1,3867),(4572,'Analysis URL Links','',4,3867),(4573,'Experiment Method','SNP Microarray',3,3867),(4574,'Experiment Platform','Affymetrix Microarray',2,3867),(4575,'Analysis Type','Static',1,3868),(4576,'Analysis URL Links','',4,3868),(4577,'Experiment Method','SNP Microarray',3,3868),(4578,'Experiment Platform','Affymetrix Microarray',2,3868),(4579,'Analysis Type','Static',1,3869),(4580,'Analysis URL Links','',4,3869),(4581,'Experiment Method','SNP Microarray',3,3869),(4582,'Experiment Platform','Affymetrix Microarray',2,3869),(4583,'Analysis Type','Static',1,3870),(4584,'Analysis URL Links','',4,3870),(4585,'Experiment Method','SNP Microarray',3,3870),(4586,'Experiment Platform','Affymetrix Microarray',2,3870),(4587,'Analysis Type','Static',1,3871),(4588,'Analysis URL Links','',4,3871),(4589,'Experiment Method','SNP Microarray',3,3871),(4590,'Experiment Platform','Affymetrix Microarray',2,3871),(4591,'Analysis Type','Static',1,3872),(4592,'Analysis URL Links','',4,3872),(4593,'Experiment Method','SNP Microarray',3,3872),(4594,'Experiment Platform','Affymetrix Microarray',2,3872),(4595,'Analysis Type','Static',1,3873),(4596,'Analysis URL Links','',4,3873),(4597,'Experiment Method','SNP Microarray',3,3873),(4598,'Experiment Platform','Affymetrix Microarray',2,3873),(4599,'Analysis Type','Static',1,3874),(4600,'Analysis URL Links','',4,3874),(4601,'Experiment Method','SNP Microarray',3,3874),(4602,'Experiment Platform','Affymetrix Microarray',2,3874),(4603,'Analysis Type','Static',1,3875),(4604,'Analysis URL Links','',4,3875),(4605,'Experiment Method','SNP Microarray',3,3875),(4606,'Experiment Platform','Affymetrix Microarray',2,3875),(4607,'Analysis Type','Static',1,3876),(4608,'Analysis URL Links','',4,3876),(4609,'Experiment Method','SNP Microarray',3,3876),(4610,'Experiment Platform','Affymetrix Microarray',2,3876),(4611,'Analysis Type','Static',1,3877),(4612,'Analysis URL Links','',4,3877),(4613,'Experiment Method','SNP Microarray',3,3877),(4614,'Experiment Platform','Affymetrix Microarray',2,3877),(4615,'Analysis Type','Static',1,3878),(4616,'Analysis URL Links','',4,3878),(4617,'Experiment Method','SNP Microarray',3,3878),(4618,'Experiment Platform','Affymetrix Microarray',2,3878),(4619,'Analysis Type','Static',1,3879),(4620,'Analysis URL Links','',4,3879),(4621,'Experiment Method','SNP Microarray',3,3879),(4622,'Experiment Platform','Affymetrix Microarray',2,3879),(4623,'Analysis Type','Static',1,3880),(4624,'Analysis URL Links','',4,3880),(4625,'Experiment Method','SNP Microarray',3,3880),(4626,'Experiment Platform','Affymetrix Microarray',2,3880),(4627,'Analysis Type','Static',1,3881),(4628,'Analysis URL Links','',4,3881),(4629,'Experiment Method','SNP Microarray',3,3881),(4630,'Experiment Platform','Affymetrix Microarray',2,3881),(4631,'Analysis Type','Static',1,3882),(4632,'Analysis URL Links','',4,3882),(4633,'Experiment Method','SNP Microarray',3,3882),(4634,'Experiment Platform','Affymetrix Microarray',2,3882),(4635,'Analysis Type','Static',1,3883),(4636,'Analysis URL Links','',4,3883),(4637,'Experiment Method','SNP Microarray',3,3883),(4638,'Experiment Platform','Affymetrix Microarray',2,3883),(4639,'Analysis Type','Static',1,3884),(4640,'Analysis URL Links','',4,3884),(4641,'Experiment Method','SNP Microarray',3,3884),(4642,'Experiment Platform','Affymetrix Microarray',2,3884),(4643,'Analysis Type','Static',1,3885),(4644,'Analysis URL Links','',4,3885),(4645,'Experiment Method','SNP Microarray',3,3885),(4646,'Experiment Platform','Affymetrix Microarray',2,3885),(4647,'Analysis Type','Static',1,3886),(4648,'Analysis URL Links','',4,3886),(4649,'Experiment Method','SNP Microarray',3,3886),(4650,'Experiment Platform','Affymetrix Microarray',2,3886),(4651,'Analysis Type','Static',1,3887),(4652,'Analysis URL Links','',4,3887),(4653,'Experiment Method','SNP Microarray',3,3887),(4654,'Experiment Platform','Affymetrix Microarray',2,3887),(4655,'Analysis Type','Static',1,3888),(4656,'Analysis URL Links','',4,3888),(4657,'Experiment Method','SNP Microarray',3,3888),(4658,'Experiment Platform','Affymetrix Microarray',2,3888),(4659,'Analysis Type','Static',1,3889),(4660,'Analysis URL Links','',4,3889),(4661,'Experiment Method','SNP Microarray',3,3889),(4662,'Experiment Platform','Affymetrix Microarray',2,3889),(4663,'Analysis Type','Static',1,3890),(4664,'Analysis URL Links','',4,3890),(4665,'Experiment Method','SNP Microarray',3,3890),(4666,'Experiment Platform','Affymetrix Microarray',2,3890),(4667,'Analysis Type','Static',1,3891),(4668,'Analysis URL Links','',4,3891),(4669,'Experiment Method','SNP Microarray',3,3891),(4670,'Experiment Platform','Affymetrix Microarray',2,3891),(4671,'Analysis Type','Static',1,3892),(4672,'Analysis URL Links','',4,3892),(4673,'Experiment Method','SNP Microarray',3,3892),(4674,'Experiment Platform','Affymetrix Microarray',2,3892),(4675,'Analysis Type','Static',1,3893),(4676,'Analysis URL Links','',4,3893),(4677,'Experiment Method','SNP Microarray',3,3893),(4678,'Experiment Platform','Affymetrix Microarray',2,3893),(4679,'Analysis Type','Static',1,3894),(4680,'Analysis URL Links','',4,3894),(4681,'Experiment Method','SNP Microarray',3,3894),(4682,'Experiment Platform','Affymetrix Microarray',2,3894),(4683,'Analysis Type','Static',1,3895),(4684,'Analysis URL Links','',4,3895),(4685,'Experiment Method','SNP Microarray',3,3895),(4686,'Experiment Platform','Affymetrix Microarray',2,3895),(4687,'Analysis Type','Static',1,3896),(4688,'Analysis URL Links','',4,3896),(4689,'Experiment Method','SNP Microarray',3,3896),(4690,'Experiment Platform','Affymetrix Microarray',2,3896),(4691,'Analysis Type','Static',1,3897),(4692,'Analysis URL Links','',4,3897),(4693,'Experiment Method','SNP Microarray',3,3897),(4694,'Experiment Platform','Affymetrix Microarray',2,3897),(4695,'Analysis Type','Static',1,3898),(4696,'Analysis URL Links','',4,3898),(4697,'Experiment Method','SNP Microarray',3,3898),(4698,'Experiment Platform','Affymetrix Microarray',2,3898),(4699,'Analysis Type','Static',1,3899),(4700,'Analysis URL Links','',4,3899),(4701,'Experiment Method','SNP Microarray',3,3899),(4702,'Experiment Platform','Affymetrix Microarray',2,3899),(4703,'Analysis Type','Static',1,3901),(4704,'Analysis URL Links','',4,3901),(4705,'Experiment Method','SNP Microarray',3,3901),(4706,'Experiment Platform','Affymetrix Microarray',2,3901),(4707,'Analysis Type','Static',1,3902),(4708,'Analysis URL Links','',4,3902),(4709,'Experiment Method','SNP Microarray',3,3902),(4710,'Experiment Platform','Affymetrix Microarray',2,3902),(4711,'Analysis Type','Static',1,3903),(4712,'Analysis URL Links','',4,3903),(4713,'Experiment Method','SNP Microarray',3,3903),(4714,'Experiment Platform','Affymetrix Microarray',2,3903),(4719,'Analysis Type','',1,3905),(4720,'Analysis URL Links','',4,3905),(4721,'Experiment Method','',3,3905),(4722,'Experiment Platform','',2,3905),(4723,'Analysis Type','',1,3931),(4724,'Analysis URL Links','',4,3931),(4725,'Experiment Method','',3,3931),(4726,'Experiment Platform','',2,3931),(4727,'Analysis Type','',1,3932),(4728,'Analysis URL Links','',4,3932),(4729,'Experiment Method','',3,3932),(4730,'Experiment Platform','',2,3932),(4731,'Analysis Type','',1,3934),(4732,'Analysis URL Links','',4,3934),(4733,'Experiment Method','',3,3934),(4734,'Experiment Platform','',2,3934),(4747,'Analysis Type','',1,3935),(4748,'Analysis URL Links','',4,3935),(4749,'Experiment Method','',3,3935),(4750,'Experiment Platform','',2,3935),(4751,'Analysis Type','',1,3940),(4752,'Analysis URL Links','',4,3940),(4753,'Experiment Method','',3,3940),(4754,'Experiment Platform','',2,3940),(4755,'Analysis Type','',1,3942),(4756,'Analysis URL Links','',4,3942),(4757,'Experiment Method','',3,3942),(4758,'Experiment Platform','',2,3942),(4759,'Analysis Type','',1,3943),(4760,'Analysis URL Links','',4,3943),(4761,'Experiment Method','',3,3943),(4762,'Experiment Platform','',2,3943),(4763,'Analysis Type','',1,3941),(4764,'Analysis URL Links','',4,3941),(4765,'Experiment Method','',3,3941),(4766,'Experiment Platform','',2,3941),(4767,'Analysis Type','',1,3948),(4768,'Analysis URL Links','',4,3948),(4769,'Experiment Method','',3,3948),(4770,'Experiment Platform','',2,3948),(4771,'Analysis Type','',1,3949),(4772,'Analysis URL Links','',4,3949),(4773,'Experiment Method','',3,3949),(4774,'Experiment Platform','',2,3949),(4791,'Analysis Type','',1,3958),(4792,'Analysis URL Links','',4,3958),(4793,'Experiment Method','',3,3958),(4794,'Experiment Platform','',2,3958),(4795,'Analysis Type','',1,3959),(4796,'Analysis URL Links','',4,3959),(4797,'Experiment Method','',3,3959),(4798,'Experiment Platform','',2,3959),(4799,'Analysis Type','',1,3960),(4800,'Analysis URL Links','',4,3960),(4801,'Experiment Method','',3,3960),(4802,'Experiment Platform','',2,3960),(4803,'Analysis Type','',1,3961),(4804,'Analysis URL Links','',4,3961),(4805,'Experiment Method','',3,3961),(4806,'Experiment Platform','',2,3961),(4807,'Analysis Type','',1,3962),(4808,'Analysis URL Links','',4,3962),(4809,'Experiment Method','',3,3962),(4810,'Experiment Platform','',2,3962),(4811,'Analysis Type','',1,3963),(4812,'Analysis URL Links','',4,3963),(4813,'Experiment Method','',3,3963),(4814,'Experiment Platform','',2,3963),(4815,'Analysis Type','',1,3964),(4816,'Analysis URL Links','',4,3964),(4817,'Experiment Method','',3,3964),(4818,'Experiment Platform','',2,3964),(4819,'Analysis Type','',1,3965),(4820,'Analysis URL Links','',4,3965),(4821,'Experiment Method','',3,3965),(4822,'Experiment Platform','',2,3965),(4835,'Analysis Type','',1,3972),(4836,'Analysis URL Links','',4,3972),(4837,'Experiment Method','',3,3972),(4838,'Experiment Platform','',2,3972),(4839,'Analysis Type','',1,3973),(4840,'Analysis URL Links','',4,3973),(4841,'Experiment Method','',3,3973),(4842,'Experiment Platform','',2,3973),(4843,'Analysis Type','',1,3975),(4844,'Analysis URL Links','',4,3975),(4845,'Experiment Method','',3,3975),(4846,'Experiment Platform','',2,3975),(4855,'Analysis Type','',1,3985),(4856,'Analysis URL Links','',4,3985),(4857,'Experiment Method','',3,3985),(4858,'Experiment Platform','',2,3985),(4859,'Analysis Type','',1,3984),(4860,'Analysis URL Links','',4,3984),(4861,'Experiment Method','',3,3984),(4862,'Experiment Platform','',2,3984),(4863,'Analysis Type','',1,3986),(4864,'Analysis URL Links','',4,3986),(4865,'Experiment Method','',3,3986),(4866,'Experiment Platform','',2,3986),(4871,'Analysis Type','',1,4000),(4872,'Analysis URL Links','',4,4000),(4873,'Experiment Method','',3,4000),(4874,'Experiment Platform','',2,4000),(4875,'Analysis Type','',1,2349),(4876,'Analysis URL Links','',4,2349),(4877,'Experiment Method','',3,2349),(4878,'Experiment Platform','',2,2349),(4879,'Analysis Type','',1,2328),(4880,'Analysis URL Links','',4,2328),(4881,'Experiment Method','',3,2328),(4882,'Experiment Platform','',2,2328),(4883,'Analysis Type','Static',1,3999),(4884,'Analysis URL Links','',4,3999),(4885,'Experiment Method','RNA-Seq',3,3999),(4886,'Experiment Platform','Illumina Genome Analyzer',2,3999),(4887,'Analysis Type','',1,4002),(4888,'Analysis URL Links','',4,4002),(4889,'Experiment Method','',3,4002),(4890,'Experiment Platform','',2,4002),(4891,'Analysis URL Links','',4,3596),(4892,'Analysis URL Links','',4,3597),(4893,'Analysis URL Links','',4,3599),(4894,'Analysis URL Links','',4,3600),(4895,'Analysis URL Links','',4,3602),(4896,'Analysis URL Links','',4,3603),(4897,'Analysis URL Links','',4,3598),(4898,'Analysis URL Links','',4,3601),(4899,'Analysis URL Links','',4,3604),(4900,'Analysis URL Links','',4,3605),(4901,'Analysis URL Links','',4,3606),(4902,'Analysis URL Links','',4,3608),(4903,'Analysis URL Links','',4,3609),(4904,'Analysis URL Links','',4,3611),(4905,'Analysis URL Links','',4,3612),(4906,'Analysis URL Links','',4,3614),(4907,'Analysis URL Links','',4,3615),(4908,'Analysis URL Links','',4,3617),(4909,'Analysis URL Links','',4,3618),(4910,'Analysis URL Links','',4,3613),(4911,'Analysis URL Links','',4,3616),(4912,'Analysis URL Links','',4,3619),(4913,'Analysis URL Links','',4,3607),(4914,'Analysis URL Links','',4,3610),(4919,'Analysis Type','Static',1,4138),(4920,'Analysis URL Links','',4,4138),(4921,'Experiment Method','Whole Genome Sequencing',3,4138),(4922,'Experiment Platform','Illumina Genome Analyzer',2,4138),(4923,'Analysis Type','',1,4139),(4924,'Analysis URL Links','',4,4139),(4925,'Experiment Method','',3,4139),(4926,'Experiment Platform','',2,4139),(4927,'Analysis Type','',1,4140),(4928,'Analysis URL Links','',4,4140),(4929,'Experiment Method','',3,4140),(4930,'Experiment Platform','',2,4140),(4931,'Analysis Type','',1,4141),(4932,'Analysis URL Links','',4,4141),(4933,'Experiment Method','',3,4141),(4934,'Experiment Platform','',2,4141),(4935,'Analysis Type','',1,4142),(4936,'Analysis URL Links','',4,4142),(4937,'Experiment Method','',3,4142),(4938,'Experiment Platform','',2,4142),(4939,'Analysis Type','',1,4143),(4940,'Analysis URL Links','',4,4143),(4941,'Experiment Method','',3,4143),(4942,'Experiment Platform','',2,4143),(4947,'Analysis Type','Static',1,4157),(4948,'Analysis URL Links','',4,4157),(4949,'Experiment Method','RNA-Seq',3,4157),(4950,'Experiment Platform','Illumina Genome Analyzer',2,4157),(4951,'Analysis Type','Static',1,4158),(4952,'Analysis URL Links','',4,4158),(4953,'Experiment Method','RNA-Seq',3,4158),(4954,'Experiment Platform','Illumina Genome Analyzer',2,4158),(5047,'Analysis Type','',1,4196),(5048,'Analysis URL Links','',4,4196),(5049,'Experiment Method','',3,4196),(5050,'Experiment Platform','',2,4196),(5051,'Analysis Type','',1,4203),(5052,'Analysis URL Links','',4,4203),(5053,'Experiment Method','',3,4203),(5054,'Experiment Platform','',2,4203),(5055,'Analysis Type','',1,4204),(5056,'Analysis URL Links','',4,4204),(5057,'Experiment Method','',3,4204),(5058,'Experiment Platform','',2,4204),(5059,'Analysis Type','',1,4205),(5060,'Analysis URL Links','',4,4205),(5061,'Experiment Method','',3,4205),(5062,'Experiment Platform','',2,4205),(5063,'Analysis Type','',1,4197),(5064,'Analysis URL Links','',4,4197),(5065,'Experiment Method','',3,4197),(5066,'Experiment Platform','',2,4197),(5067,'Analysis Type','',1,4198),(5068,'Analysis URL Links','',4,4198),(5069,'Experiment Method','',3,4198),(5070,'Experiment Platform','',2,4198),(5071,'Analysis Type','',1,4147),(5072,'Analysis URL Links','',4,4147),(5073,'Experiment Method','',3,4147),(5074,'Experiment Platform','',2,4147),(5075,'Analysis Type','',1,4199),(5076,'Analysis URL Links','',4,4199),(5077,'Experiment Method','',3,4199),(5078,'Experiment Platform','',2,4199),(5079,'Analysis Type','',1,4148),(5080,'Analysis URL Links','',4,4148),(5081,'Experiment Method','',3,4148),(5082,'Experiment Platform','',2,4148),(5083,'Analysis Type','',1,4200),(5084,'Analysis URL Links','',4,4200),(5085,'Experiment Method','',3,4200),(5086,'Experiment Platform','',2,4200),(5087,'Analysis Type','',1,4201),(5088,'Analysis URL Links','',4,4201),(5089,'Experiment Method','',3,4201),(5090,'Experiment Platform','',2,4201),(5091,'Analysis Type','',1,4202),(5092,'Analysis URL Links','',4,4202),(5093,'Experiment Method','',3,4202),(5094,'Experiment Platform','',2,4202),(5095,'Analysis Type','',1,4224),(5096,'Analysis URL Links','',4,4224),(5097,'Experiment Method','',3,4224),(5098,'Experiment Platform','',2,4224),(5099,'Analysis Type','',1,4226),(5100,'Analysis URL Links','',4,4226),(5101,'Experiment Method','',3,4226),(5102,'Experiment Platform','',2,4226),(5103,'Analysis Type','',1,4222),(5104,'Analysis URL Links','',4,4222),(5105,'Experiment Method','',3,4222),(5106,'Experiment Platform','',2,4222),(5107,'Analysis Type','',1,4228),(5108,'Analysis URL Links','',4,4228),(5109,'Experiment Method','',3,4228),(5110,'Experiment Platform','',2,4228),(5235,'Analysis Type','Static',1,4278),(5236,'Analysis URL Links','',4,4278),(5237,'Experiment Method','',3,4278),(5238,'Experiment Platform','',2,4278),(5239,'Analysis Type','Static',1,4279),(5240,'Analysis URL Links','',4,4279),(5241,'Experiment Method','',3,4279),(5242,'Experiment Platform','',2,4279),(5243,'Analysis Type','',1,4280),(5244,'Analysis URL Links','',4,4280),(5245,'Experiment Method','',3,4280),(5246,'Experiment Platform','',2,4280),(5247,'Analysis Type','',1,4281),(5248,'Analysis URL Links','',4,4281),(5249,'Experiment Method','',3,4281),(5250,'Experiment Platform','',2,4281),(5251,'Analysis Type','',1,4282),(5252,'Analysis URL Links','',4,4282),(5253,'Experiment Method','',3,4282),(5254,'Experiment Platform','',2,4282),(5255,'Analysis Type','',1,4284),(5256,'Analysis URL Links','',4,4284),(5257,'Experiment Method','',3,4284),(5258,'Experiment Platform','',2,4284),(5259,'Analysis Type','',1,4285),(5260,'Analysis URL Links','',4,4285),(5261,'Experiment Method','',3,4285),(5262,'Experiment Platform','',2,4285),(5263,'Analysis Type','',1,4286),(5264,'Analysis URL Links','',4,4286),(5265,'Experiment Method','',3,4286),(5266,'Experiment Platform','',2,4286),(5267,'Analysis Type','',1,4271),(5268,'Analysis URL Links','',4,4271),(5269,'Experiment Method','',3,4271),(5270,'Experiment Platform','',2,4271),(5271,'Analysis Type','',1,4302),(5272,'Analysis URL Links','',4,4302),(5273,'Experiment Method','',3,4302),(5274,'Experiment Platform','',2,4302),(5275,'Analysis Type','',1,4308),(5276,'Analysis URL Links','',4,4308),(5277,'Experiment Method','',3,4308),(5278,'Experiment Platform','',2,4308),(5279,'Analysis Type','',1,4263),(5280,'Analysis URL Links','',4,4263),(5281,'Experiment Method','',3,4263),(5282,'Experiment Platform','',2,4263),(5283,'Analysis Type','',1,4258),(5284,'Analysis URL Links','',4,4258),(5285,'Experiment Method','',3,4258),(5286,'Experiment Platform','',2,4258),(5287,'Analysis Type','',1,4259),(5288,'Analysis URL Links','',4,4259),(5289,'Experiment Method','',3,4259),(5290,'Experiment Platform','',2,4259),(5291,'Analysis Type','',1,4261),(5292,'Analysis URL Links','',4,4261),(5293,'Experiment Method','',3,4261),(5294,'Experiment Platform','',2,4261),(5295,'Analysis Type','',1,4260),(5296,'Analysis URL Links','',4,4260),(5297,'Experiment Method','',3,4260),(5298,'Experiment Platform','',2,4260),(5299,'Analysis Type','',1,4262),(5300,'Analysis URL Links','',4,4262),(5301,'Experiment Method','',3,4262),(5302,'Experiment Platform','',2,4262),(5303,'Analysis Type','',1,4264),(5304,'Analysis URL Links','',4,4264),(5305,'Experiment Method','',3,4264),(5306,'Experiment Platform','',2,4264),(5307,'Analysis Type','',1,4265),(5308,'Analysis URL Links','',4,4265),(5309,'Experiment Method','',3,4265),(5310,'Experiment Platform','',2,4265),(5311,'Analysis Type','',1,4266),(5312,'Analysis URL Links','',4,4266),(5313,'Experiment Method','',3,4266),(5314,'Experiment Platform','',2,4266),(5315,'Analysis Type','',1,4269),(5316,'Analysis URL Links','',4,4269),(5317,'Experiment Method','',3,4269),(5318,'Experiment Platform','',2,4269),(5319,'Analysis Type','',1,4267),(5320,'Analysis URL Links','',4,4267),(5321,'Experiment Method','',3,4267),(5322,'Experiment Platform','',2,4267),(5323,'Analysis Type','',1,4268),(5324,'Analysis URL Links','',4,4268),(5325,'Experiment Method','',3,4268),(5326,'Experiment Platform','',2,4268),(5327,'Analysis Type','',1,3800),(5328,'Analysis URL Links','',4,3800),(5329,'Experiment Method','',3,3800),(5330,'Experiment Platform','',2,3800),(5331,'Analysis Type','',1,3799),(5332,'Analysis URL Links','',4,3799),(5333,'Experiment Method','',3,3799),(5334,'Experiment Platform','',2,3799),(5335,'Analysis Type','',1,3798),(5336,'Analysis URL Links','',4,3798),(5337,'Experiment Method','',3,3798),(5338,'Experiment Platform','',2,3798),(5339,'Analysis Type','',1,3801),(5340,'Analysis URL Links','',4,3801),(5341,'Experiment Method','',3,3801),(5342,'Experiment Platform','',2,3801),(5343,'Analysis Type','',1,4353),(5344,'Analysis URL Links','',4,4353),(5345,'Experiment Method','',3,4353),(5346,'Experiment Platform','',2,4353),(5347,'Analysis Type','',1,4355),(5348,'Analysis URL Links','',4,4355),(5349,'Experiment Method','',3,4355),(5350,'Experiment Platform','',2,4355),(5351,'Analysis Type','',1,4361),(5352,'Analysis URL Links','',4,4361),(5353,'Experiment Method','',3,4361),(5354,'Experiment Platform','',2,4361),(5355,'Analysis Type','',1,4362),(5356,'Analysis URL Links','',4,4362),(5357,'Experiment Method','',3,4362),(5358,'Experiment Platform','',2,4362),(5359,'Analysis Type','',1,4364),(5360,'Analysis URL Links','',4,4364),(5361,'Experiment Method','',3,4364),(5362,'Experiment Platform','',2,4364),(5403,'Analysis Type','',1,4388),(5404,'Analysis URL Links','',4,4388),(5405,'Experiment Method','',3,4388),(5406,'Experiment Platform','',2,4388),(5407,'Analysis Type','',1,4389),(5408,'Analysis URL Links','',4,4389),(5409,'Experiment Method','',3,4389),(5410,'Experiment Platform','',2,4389),(5411,'Analysis Type','',1,4395),(5412,'Analysis URL Links','',4,4395),(5413,'Experiment Method','',3,4395),(5414,'Experiment Platform','',2,4395),(5415,'Analysis Type','',1,4396),(5416,'Analysis URL Links','',4,4396),(5417,'Experiment Method','',3,4396),(5418,'Experiment Platform','',2,4396),(5419,'Analysis Type','',1,4397),(5420,'Analysis URL Links','',4,4397),(5421,'Experiment Method','',3,4397),(5422,'Experiment Platform','',2,4397),(5423,'Analysis Type','',1,4399),(5424,'Analysis URL Links','',4,4399),(5425,'Experiment Method','',3,4399),(5426,'Experiment Platform','',2,4399),(5427,'Analysis Type','',1,4410),(5428,'Analysis URL Links','',4,4410),(5429,'Experiment Method','',3,4410),(5430,'Experiment Platform','',2,4410),(5431,'Analysis Type','',1,4413),(5432,'Analysis URL Links','',4,4413),(5433,'Experiment Method','',3,4413),(5434,'Experiment Platform','',2,4413),(5435,'Analysis Type','',1,4415),(5436,'Analysis URL Links','',4,4415),(5437,'Experiment Method','',3,4415),(5438,'Experiment Platform','',2,4415),(5443,'Analysis Type','',1,4414),(5444,'Analysis URL Links','',4,4414),(5445,'Experiment Method','',3,4414),(5446,'Experiment Platform','',2,4414),(5451,'Analysis Type','',1,4422),(5452,'Analysis URL Links','',4,4422),(5453,'Experiment Method','',3,4422),(5454,'Experiment Platform','',2,4422),(5455,'Analysis Type','',1,4423),(5456,'Analysis URL Links','',4,4423),(5457,'Experiment Method','',3,4423),(5458,'Experiment Platform','',2,4423),(5459,'Analysis Type','',1,4424),(5460,'Analysis URL Links','',4,4424),(5461,'Experiment Method','',3,4424),(5462,'Experiment Platform','',2,4424);
ALTER TABLE dbo.gpAnnotationProperty CHECK CONSTRAINT all;
SET IDENTITY_INSERT dbo.gpAnnotationProperty OFF;


--
-- Table structure for table dbo.gpAnnotationPropertyOption --

IF  EXISTS (SELECT * FROM sys.objects WHERE object_id = OBJECT_ID(N'dbo.gpAnnotationPropertyOption') AND type in (N'U')) DROP TABLE dbo.gpAnnotationPropertyOption;


CREATE TABLE dbo.gpAnnotationPropertyOption (
  idAnnotationProperty int  NOT NULL,
  idPropertyOption int  NOT NULL,
CONSTRAINT [PK_AnnotationPropertyOption] PRIMARY KEY CLUSTERED 
(
	[idAnnotationProperty] ASC,
	[idPropertyOption] ASC
),
  
  CONSTRAINT FK_AnnotationPropertyOption_AnnotationProperty FOREIGN KEY (idAnnotationProperty) REFERENCES dbo.gpAnnotationProperty (idAnnotationProperty) ON DELETE NO ACTION ON UPDATE NO ACTION,
  CONSTRAINT FK_AnnotationPropertyOption_PropertyOption FOREIGN KEY (idPropertyOption) REFERENCES dbo.gpPropertyOption (idPropertyOption) ON DELETE NO ACTION ON UPDATE NO ACTION
)  ;


--
-- Dumping data for table dbo.gpAnnotationPropertyOption --


ALTER TABLE dbo.gpAnnotationPropertyOption NOCHECK CONSTRAINT all;
INSERT INTO dbo.gpAnnotationPropertyOption (idAnnotationProperty, idPropertyOption) VALUES (2179,5),(2180,5),(2181,5),(2182,5),(2183,5),(2184,5),(2185,5),(2186,5),(2187,5),(2188,5),(2189,5),(2190,5),(2191,5),(2192,5),(2193,5),(2194,5),(2195,5),(2196,5),(2197,5),(2198,5),(2199,5),(2200,5),(2201,5),(2202,5),(2203,5),(2204,5),(2205,5),(2206,5),(2207,5),(2208,5),(2209,5),(2210,5),(2211,5),(2212,5),(2213,5),(2214,5),(2215,5),(2216,5),(2217,5),(2218,5),(2219,5),(2220,5),(2221,5),(2222,5),(2223,5),(2224,5),(2225,5),(2226,5),(2227,5),(2228,5),(2229,5),(2230,5),(2231,5),(2232,5),(2233,5),(2234,5),(2235,5),(2236,5),(2237,5),(2238,5),(2239,5),(2240,5),(2241,5),(2242,5),(2243,5),(2244,5),(2245,5),(2246,5),(2247,5),(2248,5),(2249,5),(2250,5),(2251,5),(2252,5),(2253,5),(2254,5),(2255,5),(2256,5),(2257,5),(2258,5),(2259,5),(2260,5),(2261,5),(2262,5),(2263,5),(2264,5),(2265,5),(2266,5),(2267,5),(2268,5),(2269,5),(2270,5),(2271,5),(2272,5),(2273,5),(2274,5),(2275,5),(2276,5),(2277,5),(2278,5),(2279,5),(2280,5),(2281,5),(2282,5),(2283,5),(2284,5),(2285,5),(2286,5),(2287,5),(2288,5),(2289,5),(2290,5),(2291,5),(2292,5),(2293,5),(2294,5),(2295,5),(2296,5),(2297,5),(2298,5),(2299,5),(2300,5),(2301,5),(2302,5),(2303,5),(2304,5),(2305,5),(2306,5),(2307,5),(2308,5),(2309,5),(2310,5),(2311,5),(2312,5),(2313,5),(2314,5),(2315,5),(2316,5),(2317,5),(2318,5),(2319,5),(2320,5),(2321,5),(2322,5),(2323,5),(2324,5),(2325,5),(2326,5),(2327,5),(2328,5),(2329,5),(2330,5),(2331,5),(2332,5),(2333,5),(2334,5),(2335,5),(2336,5),(2337,5),(2338,5),(2339,5),(2340,5),(2341,5),(2342,5),(2343,5),(2344,5),(2345,5),(2346,5),(2347,5),(2348,5),(2349,5),(2350,5),(2351,5),(2352,5),(2353,5),(2354,5),(2355,5),(2356,5),(2357,5),(2358,5),(2359,5),(2360,5),(2361,5),(2362,5),(2363,5),(2364,5),(2365,5),(2366,5),(2367,5),(2368,5),(2369,5),(2370,5),(2371,5),(2372,5),(2373,5),(2374,5),(2375,5),(2376,5),(2377,5),(2378,5),(2379,5),(2380,5),(2381,5),(2382,5),(2383,5),(2384,5),(2385,5),(2386,5),(2387,5),(2388,5),(2389,5),(2390,5),(2391,5),(2392,5),(2393,5),(2394,5),(2395,5),(2396,5),(2397,5),(2398,5),(2399,5),(2400,5),(2401,5),(2402,5),(2403,5),(2404,5),(2405,5),(2406,5),(2407,5),(2408,5),(2409,5),(2410,5),(2411,5),(2412,5),(2413,5),(2414,5),(2415,5),(2416,5),(2417,5),(2418,5),(2419,5),(2420,5),(2421,5),(2422,5),(2423,5),(2424,5),(2425,5),(2426,5),(2427,5),(2428,5),(2429,5),(2430,5),(2431,5),(2432,5),(2433,5),(2434,5),(2435,5),(2436,5),(2437,5),(2438,5),(2439,5),(2440,5),(2441,5),(2442,5),(2443,5),(2444,5),(2445,5),(2446,5),(2447,5),(2448,5),(2449,5),(2450,5),(2451,5),(2452,5),(2453,5),(2454,5),(2455,5),(2456,5),(2457,5),(2458,5),(2459,5),(2460,5),(2461,5),(2462,5),(2463,5),(2464,5),(2465,5),(2466,5),(2467,5),(2468,5),(2469,5),(2470,5),(2471,5),(2472,5),(2473,5),(2474,5),(2475,5),(2476,5),(2477,5),(2478,5),(2479,5),(2480,5),(2481,5),(2482,5),(2483,5),(2484,5),(2485,5),(2486,5),(2487,5),(2488,5),(2489,5),(2498,5),(2499,5),(2500,5),(2501,5),(2502,5),(2503,5),(2504,5),(2505,5),(2506,5),(2507,5),(2508,5),(2509,5),(2510,5),(2511,5),(2512,5),(2513,5),(2514,5),(2515,5),(2516,5),(2517,5),(2522,5),(2523,5),(2524,5),(2525,5),(2526,5),(2527,5),(2528,5),(2529,5),(2530,5),(2531,5),(2532,5),(2533,5),(2534,5),(2535,5),(2536,5),(2537,5),(2538,5),(2539,5),(2540,5),(2541,5),(2542,5),(2543,5),(2544,5),(2545,5),(2546,5),(2547,5),(2548,5),(2549,5),(2550,5),(2551,5),(2552,5),(2553,5),(2554,5),(2555,5),(2556,5),(2557,5),(2558,5),(2559,5),(2560,5),(2561,5),(2562,5),(2563,5),(2564,5),(2565,5),(2566,5),(2567,5),(2568,5),(2569,5),(3940,5),(3943,5),(3946,5),(3952,5),(4003,5),(4006,5),(4009,5),(4060,5),(4069,5),(4072,5),(4075,5),(4078,5),(4150,5),(4156,5),(4186,5),(4207,5),(4210,5),(4213,5),(4216,5),(4224,5),(4228,5),(4232,5),(4236,5),(4254,5),(4266,5),(4270,5),(4274,5),(4278,5),(4282,5),(4286,5),(4290,5),(4294,5),(4298,5),(4302,5),(4306,5),(4310,5),(4314,5),(4318,5),(4886,5),(4922,5),(4950,5),(4954,5),(2570,6),(2571,6),(2572,6),(2573,6),(2574,6),(2575,6),(2576,6),(2577,6),(2578,6),(2579,6),(2580,6),(2581,6),(2582,6),(2583,6),(2584,6),(4326,7),(4330,7),(4334,7),(4338,7),(4342,7),(4346,7),(4350,7),(4354,7),(4358,7),(4362,7),(4366,7),(4370,7),(4374,7),(4378,7),(4382,7),(4386,7),(4390,7),(4394,7),(4398,7),(4402,7),(4406,7),(4410,7),(4414,7),(4418,7),(4422,7),(4426,7),(4430,7),(4434,7),(4438,7),(4442,7),(4446,7),(4450,7),(4454,7),(4458,7),(4462,7),(4466,7),(4470,7),(4474,7),(4478,7),(4482,7),(4486,7),(4490,7),(4494,7),(4498,7),(4502,7),(4506,7),(4510,7),(4514,7),(4518,7),(4522,7),(4526,7),(4530,7),(4534,7),(4538,7),(4542,7),(4546,7),(4550,7),(4554,7),(4558,7),(4562,7),(4566,7),(4570,7),(4574,7),(4578,7),(4582,7),(4586,7),(4590,7),(4594,7),(4598,7),(4602,7),(4606,7),(4610,7),(4614,7),(4618,7),(4622,7),(4626,7),(4630,7),(4634,7),(4638,7),(4642,7),(4646,7),(4650,7),(4654,7),(4658,7),(4662,7),(4666,7),(4670,7),(4674,7),(4678,7),(4682,7),(4686,7),(4690,7),(4694,7),(4698,7),(4702,7),(4706,7),(4710,7),(4714,7),(2585,9),(2586,9),(2587,9),(2588,9),(2589,9),(2590,9),(2591,9),(2592,9),(2593,9),(2594,9),(2595,9),(2596,9),(2597,9),(2598,9),(2599,9),(2600,9),(2601,9),(2602,9),(2603,9),(2604,9),(2605,9),(2606,9),(2607,9),(2608,9),(2609,9),(2610,9),(2611,9),(2612,9),(2613,9),(2614,9),(2615,9),(2616,9),(2617,9),(3954,11),(3957,11),(3960,11),(3963,11),(3966,11),(3972,11),(3975,11),(3978,11),(3984,11),(3987,11),(3990,11),(3993,11),(3996,11),(3999,11),(4017,11),(4020,11),(4023,11),(4026,11),(4032,11),(4041,11),(4059,11),(4062,11),(4065,11),(4068,11),(4071,11),(4074,11),(4077,11),(4080,11),(4083,11),(4086,11),(4089,11),(4092,11),(4095,11),(4098,11),(4101,11),(4104,11),(4107,11),(4110,11),(4113,11),(4116,11),(4119,11),(4146,11),(4152,11),(4158,11),(4161,11),(4164,11),(4167,11),(4170,11),(4173,11),(4176,11),(4179,11),(4182,11),(4194,11),(4203,11),(4239,11),(4261,11),(4321,11),(4721,11),(4725,11),(4729,11),(4733,11),(4749,11),(4753,11),(4757,11),(4761,11),(4765,11),(4769,11),(4773,11),(4793,11),(4797,11),(4801,11),(4805,11),(4809,11),(4813,11),(4817,11),(4821,11),(4837,11),(4841,11),(4845,11),(4857,11),(4861,11),(4865,11),(4873,11),(4877,11),(4881,11),(4889,11),(4925,11),(4929,11),(4933,11),(4937,11),(4941,11),(5049,11),(5053,11),(5057,11),(5061,11),(5065,11),(5069,11),(5073,11),(5077,11),(5081,11),(5085,11),(5089,11),(5093,11),(5097,11),(5101,11),(5105,11),(5109,11),(5237,11),(5241,11),(5245,11),(5249,11),(5253,11),(5257,11),(5261,11),(5265,11),(5269,11),(5273,11),(5277,11),(5281,11),(5285,11),(5289,11),(5293,11),(5297,11),(5301,11),(5305,11),(5309,11),(5313,11),(5317,11),(5321,11),(5325,11),(5329,11),(5333,11),(5337,11),(5341,11),(5345,11),(5349,11),(5353,11),(5357,11),(5361,11),(5405,11),(5409,11),(5413,11),(5417,11),(5421,11),(5425,11),(5429,11),(5433,11),(5437,11),(5445,11),(5453,11),(5457,11),(5461,11),(2618,12),(2619,12),(2620,12),(2621,12),(2622,12),(2623,12),(2624,12),(2625,12),(2626,12),(2627,12),(2628,12),(2629,12),(2630,12),(2631,12),(2632,12),(2633,12),(2634,12),(2635,12),(2636,12),(2637,12),(2638,12),(2639,12),(2640,12),(2641,12),(2642,12),(2643,12),(2644,12),(2645,12),(2646,12),(2647,12),(2648,12),(2649,12),(2650,12),(2651,12),(2652,12),(2653,12),(2654,12),(2655,12),(2656,12),(2657,12),(2658,12),(2659,12),(2660,12),(2661,12),(2662,12),(2663,12),(2664,12),(2665,12),(2666,12),(2667,12),(2668,12),(2669,12),(2670,12),(2671,12),(2672,12),(2673,12),(2674,12),(2675,12),(2676,12),(2677,12),(2678,12),(2679,12),(2680,12),(2681,12),(2682,12),(2683,12),(2684,12),(2685,12),(2686,12),(2687,12),(2688,12),(2689,12),(2690,12),(2691,12),(2692,12),(2693,12),(2694,12),(2695,12),(2696,12),(2697,12),(2698,12),(2699,12),(2700,12),(2701,12),(2702,12),(2703,12),(2704,12),(2705,12),(2706,12),(2707,12),(2708,12),(2709,12),(2710,12),(2711,12),(2712,12),(2713,12),(2714,12),(2715,12),(2716,12),(2717,12),(2718,12),(2719,12),(2720,12),(2721,12),(2722,12),(2723,12),(2724,12),(2725,12),(2726,12),(2727,12),(2728,12),(2729,12),(2730,12),(2731,12),(2732,12),(2733,12),(2734,12),(2735,12),(2736,12),(2737,12),(2738,12),(2739,12),(2740,12),(2741,12),(2742,12),(2743,12),(2744,12),(2745,12),(2746,12),(2747,12),(2748,12),(2749,12),(2750,12),(2751,12),(2752,12),(2753,12),(2754,12),(2755,12),(2756,12),(2757,12),(2758,12),(2759,12),(2760,12),(2761,12),(2762,12),(2763,12),(2764,12),(2765,12),(2766,12),(2767,12),(2768,12),(2769,12),(2770,12),(2771,12),(2772,12),(2773,12),(2774,12),(2775,12),(2776,12),(2777,12),(2778,12),(2779,12),(2780,12),(2781,12),(2782,12),(2783,12),(2784,12),(2785,12),(2786,12),(2787,12),(2788,12),(2789,12),(2790,12),(2791,12),(2792,12),(2793,12),(2794,12),(2795,12),(2796,12),(2797,12),(2798,12),(2799,12),(2800,12),(2801,12),(2802,12),(2803,12),(2804,12),(2805,12),(2806,12),(2807,12),(2808,12),(2809,12),(2810,12),(2811,12),(2812,12),(2813,12),(2814,12),(2815,12),(2816,12),(2817,12),(2818,12),(2819,12),(2820,12),(2821,12),(2822,12),(2823,12),(2824,12),(2825,12),(2826,12),(2827,12),(2828,12),(2829,12),(2830,12),(2831,12),(2832,12),(2833,12),(2834,12),(2835,12),(2836,12),(2837,12),(2838,12),(2839,12),(2840,12),(2841,12),(2842,12),(2843,12),(2844,12),(2845,12),(2846,12),(2847,12),(2848,12),(2849,12),(2850,12),(2851,12),(2852,12),(2853,12),(2854,12),(2855,12),(2856,12),(2857,12),(2858,12),(2859,12),(2860,12),(2861,12),(2862,12),(2863,12),(2864,12),(2865,12),(2866,12),(2867,12),(2868,12),(2869,12),(2870,12),(2871,12),(2872,12),(2873,12),(2874,12),(2875,12),(2876,12),(2877,12),(2878,12),(2879,12),(2880,12),(2881,12),(2882,12),(2883,12),(2884,12),(2885,12),(2886,12),(2887,12),(2888,12),(2889,12);
INSERT INTO dbo.gpAnnotationPropertyOption (idAnnotationProperty, idPropertyOption) VALUES (2890,12),(2891,12),(2892,12),(2893,12),(2894,12),(2895,12),(2896,12),(2897,12),(2898,12),(2899,12),(2900,12),(2901,12),(2902,12),(2903,12),(2904,12),(2905,12),(2906,12),(2907,12),(2908,12),(2909,12),(2910,12),(2911,12),(2912,12),(2913,12),(2914,12),(2915,12),(2916,12),(2917,12),(2918,12),(2919,12),(2920,12),(2921,12),(2922,12),(2923,12),(2924,12),(2925,12),(2926,12),(2927,12),(2928,12),(2929,12),(2930,12),(2931,12),(2932,12),(2933,12),(2934,12),(2935,12),(2936,12),(2937,12),(2938,12),(2939,12),(2940,12),(2941,12),(2942,12),(2943,12),(2944,12),(2945,12),(2946,12),(2947,12),(2948,12),(2949,12),(2950,12),(2951,12),(2952,12),(2953,12),(2954,12),(2955,12),(2956,12),(2957,12),(2958,12),(2959,12),(2960,12),(2961,12),(2962,12),(2963,12),(2964,12),(2965,12),(2966,12),(2967,12),(2968,12),(2969,12),(2970,12),(2971,12),(2972,12),(2973,12),(2974,12),(2975,12),(2976,12),(2977,12),(2978,12),(2979,12),(2980,12),(2981,12),(2982,12);
INSERT INTO dbo.gpAnnotationPropertyOption (idAnnotationProperty, idPropertyOption) VALUES (2983,12),(2984,12),(2985,12),(2986,12),(2987,12),(2988,12),(2989,12),(2990,12),(2991,12),(2992,12),(2993,12),(2994,12),(2995,12),(2996,12),(2997,12),(2998,12),(2999,12),(3000,12),(3001,12),(3002,12),(3003,12),(3004,12),(3005,12),(3006,12),(3007,12),(3008,12),(3009,12),(3010,12),(3011,12),(3012,12),(3013,12),(3014,12),(3015,12),(3016,12),(3017,12),(3018,12),(3019,12),(3020,12),(3021,12),(3022,12),(3023,12),(3024,12),(3025,12),(3026,12),(3027,12),(3028,12),(3029,12),(3030,12),(3031,12),(3032,12),(3033,12),(3034,12),(3035,12),(3036,12),(3037,12),(3038,12),(3039,12),(3040,12),(3041,12),(3042,12),(3043,12),(3045,12),(3054,12),(3055,12),(3056,12),(3057,12),(3058,12),(3059,12),(3060,12),(3061,12),(3062,12),(3063,12),(3064,12),(3065,12),(3066,12),(3067,12),(3068,12),(3069,12),(3070,12),(3071,12),(3072,12),(3073,12),(3074,12),(3075,12),(3076,12),(3077,12),(3078,12),(3079,12),(3080,12),(3081,12),(3082,12),(3083,12),(3084,12),(3085,12),(3086,12),(3087,12),(3088,12),(3089,12),(3090,12),(3091,12),(3092,12),(3093,12),(3094,12),(3095,12),(3096,12),(3097,12),(3098,12),(3099,12),(3100,12),(3103,12),(3104,12),(3105,12),(3106,12),(3107,12),(3108,12),(3109,12),(3110,12),(3111,12),(3112,12),(3113,12),(3114,12),(3115,12),(3116,12),(3117,12),(3118,12),(3119,12),(3120,12),(3121,12),(3122,12),(3123,12),(3124,12),(3125,12),(3126,12),(3127,12),(3128,12),(3129,12),(3130,12),(3131,12),(3132,12),(3133,12),(3134,12),(3135,12),(3136,12),(3137,12),(3138,12),(3139,12),(3140,12),(3141,12),(3142,12),(3143,12),(3144,12),(3145,12),(3146,12),(3147,12),(3148,12),(3149,12),(3150,12),(3151,12),(3152,12),(3153,12),(3154,12),(3155,12),(3156,12),(3157,12),(3158,12),(3159,12),(3160,12),(3939,12),(3942,12),(3945,12),(3948,12),(3951,12),(4209,12),(4212,12),(4215,12),(4253,12),(3161,13),(3162,13),(3163,13),(3164,13),(3165,13),(3166,13),(3167,13),(3168,13),(3169,13),(3170,13),(3171,13),(3172,13),(3173,13),(3174,13),(4325,15),(4329,15),(4333,15),(4337,15),(4341,15),(4345,15),(4349,15),(4353,15),(4357,15),(4361,15),(4365,15),(4369,15),(4373,15),(4377,15),(4381,15),(4385,15),(4389,15),(4393,15),(4397,15),(4401,15),(4405,15),(4409,15),(4413,15),(4417,15),(4421,15),(4425,15),(4429,15),(4433,15),(4437,15),(4441,15),(4445,15),(4449,15),(4453,15),(4457,15),(4461,15),(4465,15),(4469,15),(4473,15),(4477,15),(4481,15),(4485,15),(4489,15),(4493,15),(4497,15),(4501,15),(4505,15),(4509,15),(4513,15),(4517,15),(4521,15),(4525,15),(4529,15),(4533,15),(4537,15),(4541,15),(4545,15),(4549,15),(4553,15),(4557,15),(4561,15),(4565,15),(4569,15),(4573,15),(4577,15),(4581,15),(4585,15),(4589,15),(4593,15),(4597,15),(4601,15),(4605,15),(4609,15),(4613,15),(4617,15),(4621,15),(4625,15),(4629,15),(4633,15),(4637,15),(4641,15),(4645,15),(4649,15),(4653,15),(4657,15),(4661,15),(4665,15),(4669,15),(4673,15),(4677,15),(4681,15),(4685,15),(4689,15),(4693,15),(4697,15),(4701,15),(4705,15),(4709,15),(4713,15),(3175,20),(3176,20),(3177,20),(3178,20),(3179,20),(4265,20),(4269,20),(4273,20),(4277,20),(4281,20),(4285,20),(4289,20),(4293,20),(4297,20),(4301,20),(4305,20),(4309,20),(4313,20),(4317,20),(3180,21),(3181,21),(3182,21),(3183,21),(3184,21),(3185,21),(3186,21),(3187,21),(3188,21),(3189,21),(3190,21),(3191,21),(3192,21),(3193,21),(3194,21),(3195,21),(3196,21),(3197,21),(3198,21),(3199,21),(3200,21),(3201,21),(3202,21),(3203,21),(3204,21),(3205,21),(3206,21),(3207,21),(3208,21),(3209,21),(3210,21),(3211,21),(3212,21),(3213,21),(3214,21),(3215,21),(3216,21),(3217,21),(3218,21),(3219,21),(3220,21),(3221,21),(3222,21),(3223,21),(3224,21),(3225,21),(3226,21),(3227,21),(3228,21),(3229,21),(3230,21),(3231,21),(3232,21),(3233,21),(3234,21),(3235,21),(3236,21),(3237,21),(3238,21),(3239,21),(3240,21),(3241,21),(3242,21),(3243,21),(3244,21),(3245,21),(3246,21),(3247,21),(3248,21),(3249,21),(3250,21),(3251,21),(3252,21),(3253,21),(3254,21),(3255,21),(3256,21),(3257,21),(3258,21),(3259,21),(3260,21),(3261,21),(3262,21),(3263,21),(3264,21),(3265,21),(3266,21),(3267,21),(3268,21),(3269,21),(3270,21),(3271,21),(3272,21),(3273,21),(3274,21),(3275,21),(3276,21),(3277,21),(3278,21),(3279,21),(3280,21),(3281,21),(3282,21),(3283,21),(3284,21),(3285,21),(3286,21),(3287,21),(3288,21),(3289,21),(3290,21),(3291,21),(3292,21),(3293,21),(3294,21),(3295,21),(3296,21),(3297,21),(3298,21),(3299,21),(3300,21),(4002,21),(4005,21),(4008,21),(4149,21),(4155,21),(4185,21),(4885,21),(4949,21),(4953,21),(3301,22),(3302,22),(3303,22),(3304,22),(3305,22),(3306,22),(3307,22),(3308,22),(3309,22),(3310,22),(3311,22),(3312,22),(3313,22),(3314,22),(3315,22),(3316,22),(3317,22),(3318,22),(3319,22),(3320,22),(3321,22),(3322,22),(3323,22),(3324,22),(3325,22),(3326,22),(3327,22),(3328,22),(3329,22),(3330,22),(3331,22),(3332,22),(3333,22),(3334,22),(3335,22),(3336,22),(3337,22),(3338,22),(3339,22),(3340,22),(3341,22),(3342,22),(3343,22),(3344,22),(3345,22),(3346,22),(3347,22),(3348,22),(3349,22),(3350,22),(3351,22),(3352,22),(3353,22),(3354,22),(3355,22),(3356,22),(3357,22),(3358,22),(3359,22),(3360,22),(3361,22),(3362,22),(3363,22),(3364,22),(3365,22),(3366,22),(3367,22),(3368,22),(3369,22),(3370,22),(3371,22),(3372,22),(3373,22),(3374,22),(3375,22),(3376,22),(3377,23),(3378,23),(3379,23),(3380,23),(3381,23),(3382,23),(3383,23),(3384,24),(3385,24),(3386,24),(3387,24),(3388,24),(3389,24),(3390,24),(3391,24),(3392,24),(3393,24),(3394,24),(3395,24),(3396,24),(3397,24),(3398,24),(3399,24),(3400,24),(3401,24),(3402,24),(3403,24),(3404,24),(3405,24),(3406,24),(3407,25),(3408,25),(3409,25),(3410,25),(3411,25),(3412,25),(3413,25),(3414,25),(3415,25),(3416,25),(3417,25),(3418,25),(3419,25),(3420,25),(3421,25),(3422,25),(3423,25),(3424,25),(3425,25),(3426,25),(3427,25),(3428,25),(3429,25),(3430,25),(3431,25),(3432,25),(3433,25),(3434,25),(3435,25),(3436,25),(3437,25),(3438,25),(3439,25),(3440,25),(3441,25),(3442,25),(3443,25),(3444,25),(3445,25),(3446,25),(3447,25),(3448,25),(3449,25),(3450,25),(3451,25),(3452,25),(3453,25),(3454,25),(3455,25),(3456,25),(3457,25),(3458,25),(3459,25),(3460,25),(3461,25),(3462,25),(3463,25),(3464,25),(3465,25),(3466,25),(3467,25),(3468,25),(3469,25),(3470,25),(3471,25),(3472,25),(3473,25),(3474,25),(3475,25),(3476,25),(3477,25),(3478,25),(3479,25),(3480,25),(3481,25),(3482,25),(3483,25),(3484,25),(3485,25),(3486,25),(3487,25),(3488,25),(3489,25),(3490,25),(3491,25),(3492,25),(3493,25),(3494,25),(3495,25),(3496,25),(3497,25),(3498,25),(3499,25),(3500,25),(3501,25),(3502,25),(3503,25),(3504,25),(3505,25),(3506,25),(3507,25),(3508,25),(3509,25),(3510,25),(3511,25),(3512,25),(3513,25),(3514,25),(3515,25),(3516,25),(3517,25),(3518,25),(3519,25),(3520,25),(3521,25),(3522,25),(3523,25),(3524,25),(3525,25),(3526,25),(3527,25),(3528,25),(3529,25),(3530,25),(3531,25),(3532,25),(3533,25),(3534,25),(3535,25),(3536,25),(3537,25),(3538,25),(3539,25),(3540,25),(3541,25),(3542,25),(3543,25),(3544,25),(3545,25),(3546,25),(3547,25),(3548,25),(3549,25),(3550,25),(3551,25),(3552,25),(3553,25),(3554,25),(3555,25),(3556,25),(3557,25),(3558,25),(3564,25),(3569,25),(3570,25),(3571,25),(3572,25),(3573,25),(3574,25),(3575,25),(3576,25),(3577,25),(3578,25),(3579,25),(3580,25),(3581,25),(3582,25),(3583,25),(3584,25),(3585,25),(3586,25),(3587,25),(3588,25),(3589,25),(3590,25),(3591,25),(3592,25),(3593,25),(3594,25),(3595,25),(3596,25),(3597,25),(3598,25),(3599,25),(3600,25),(3601,25),(3602,25),(3603,25),(3604,25),(3605,25),(3606,25),(3607,25),(3608,25),(3617,25),(3618,25),(3619,25),(3620,25),(3621,25),(3622,25),(3623,25),(3624,25),(3629,26),(3630,26),(3631,26),(3632,26),(3633,26),(3634,26),(3635,26),(3636,26),(3637,26),(3638,26),(3639,26),(3640,26),(3641,26),(3642,26),(3643,26),(3644,26),(3645,26),(3646,27),(3647,27),(3648,27),(3649,27),(3650,27),(3651,27),(3652,27),(3653,27),(3654,27),(3655,27),(3656,27),(3657,27),(3658,27),(3659,27),(3660,27),(3661,27),(3662,27),(3663,27),(3664,27),(3665,27),(3666,27),(3667,27),(3668,27),(3669,27),(3670,27),(3671,27),(3672,27),(3673,27),(3674,27),(3675,27),(3676,27),(3677,27),(3678,27),(3679,27),(3680,27),(3681,27),(3682,27),(3683,27),(3684,27),(3685,27),(3686,27),(3687,27),(3688,27),(3689,27),(3690,27),(3691,27),(3692,27),(3693,27),(3694,27),(3695,27),(3696,27),(3697,27),(3698,27),(3699,27),(3700,27),(3701,27),(3702,27),(3703,27),(3704,27),(3705,27),(3706,27),(3707,27),(3708,27),(3709,27),(3710,27),(3711,27),(3712,27),(3713,27),(3714,27),(3715,27),(3716,27),(3717,27),(3718,27),(3719,27),(3720,27),(3721,27),(3722,27),(3723,27),(3724,27),(3725,27),(3726,27),(3727,27),(3728,27),(3729,27),(3730,27),(3731,27),(3732,27),(3733,27),(3734,27),(3735,27),(3736,27),(3737,27),(3738,27),(3739,27),(3740,27),(3741,27),(3742,27),(3743,27),(3744,27),(3745,27),(3746,27),(3747,27),(3748,27),(3749,27),(3750,27),(3751,27),(3752,27),(3753,27),(3754,27),(3755,27),(3756,27),(3757,27),(3758,27),(3759,27),(3760,27),(3761,27),(3762,27),(3763,27),(3764,27),(3765,27),(3766,27),(3767,27),(3768,27),(3769,27),(3770,27),(3771,27),(3772,27),(3773,27),(3774,27),(3775,27),(3776,27),(3777,27),(3778,27),(3779,27),(3780,27),(3781,27),(3782,27),(3783,27),(3784,27),(3785,27),(3786,27),(3787,27),(3788,27),(3789,27),(3790,27),(3791,27),(3792,27),(3793,27),(3794,27),(3795,27),(3796,27),(3797,27),(3798,27),(3799,27),(3800,27),(3801,27),(3802,27),(3803,27),(3804,27),(3805,27),(3806,27),(3807,27),(3808,27),(3809,27),(3810,27),(3811,27),(3812,27),(3813,27),(3814,27),(3815,27),(3816,27),(3817,27),(3818,27),(3819,27),(3820,27),(3821,27),(3822,27),(3823,27),(3824,27),(3825,27),(3826,27),(3827,27),(3828,27),(3829,27),(3830,27),(3831,27),(3832,27),(3833,27),(3834,27),(3835,27),(3836,27),(3837,27),(3838,27),(3839,27),(3840,27),(3841,27),(3842,27),(3843,27),(3844,27),(3845,27),(3846,27),(3847,27),(3848,27),(3849,27),(3850,27),(3851,27),(3852,27),(3853,27),(3854,27),(3855,27),(3856,27),(3857,27),(3858,27),(3859,27),(3860,27),(3861,27),(3862,27),(3863,27);
INSERT INTO dbo.gpAnnotationPropertyOption (idAnnotationProperty, idPropertyOption) VALUES (3864,27),(3865,27),(3866,27),(3867,27),(3868,27),(3869,27),(3870,27),(3871,27),(3872,27),(3873,27),(3874,27),(3875,27),(3876,27),(3877,27),(3878,27),(3879,27),(3880,27),(3881,27),(3882,27),(3883,27),(3884,27),(3885,27),(3886,27),(3887,27),(3888,27),(3889,27),(3890,27),(3891,27),(3892,27),(3893,27),(3894,27),(3895,27),(3896,27),(3897,27),(3898,27),(3899,27),(3900,27),(3901,27),(3902,27),(3903,27),(3904,27),(3905,27),(3906,27),(3907,27),(3908,27),(3909,27),(3910,27),(3911,27),(3912,27),(3913,27),(3914,27),(3915,27),(3916,27),(3917,27),(3918,27),(3919,27),(3920,27),(3921,27),(3922,27),(3923,27),(3924,27),(3925,27),(3926,27),(3927,27),(4206,27),(4223,27),(4227,27),(4231,27),(4235,27),(3928,28),(3929,28),(3930,28),(3931,28),(3932,28),(3933,28),(3934,28),(4921,28);
INSERT INTO dbo.gpAnnotationPropertyOption (idAnnotationProperty, idPropertyOption) VALUES (3953,1),(3956,1),(3959,1),(3962,1),(3965,1),(3971,1),(3974,1),(3977,1),(3983,1),(3986,1),(3989,1),(3992,1),(3995,1),(3998,1),(4016,1),(4019,1),(4022,1),(4025,1),(4031,1),(4040,1),(4061,1),(4064,1),(4079,1),(4082,1),(4085,1),(4088,1),(4091,1),(4094,1),(4097,1),(4100,1),(4103,1),(4106,1),(4109,1),(4112,1),(4115,1),(4118,1),(4145,1),(4151,1),(4157,1),(4160,1),(4163,1),(4166,1),(4169,1),(4172,1),(4175,1),(4178,1),(4181,1),(4193,1),(4202,1),(4237,1),(4259,1),(4319,1),(4719,1),(4723,1),(4727,1),(4731,1),(4747,1),(4751,1),(4755,1),(4759,1),(4763,1),(4767,1),(4771,1),(4791,1),(4795,1),(4799,1),(4803,1),(4807,1),(4811,1),(4815,1),(4819,1),(4835,1),(4839,1),(4843,1),(4855,1),(4859,1),(4863,1),(4871,1),(4875,1),(4879,1),(4887,1),(4923,1),(4927,1),(4931,1),(4935,1),(4939,1),(5047,1),(5051,1),(5055,1),(5059,1),(5063,1),(5067,1),(5071,1),(5075,1),(5079,1),(5083,1),(5087,1),(5091,1),(5095,1),(5099,1),(5103,1),(5107,1),(5243,1),(5247,1),(5251,1),(5255,1),(5259,1),(5263,1),(5267,1),(5271,1),(5275,1),(5279,1),(5283,1),(5287,1),(5291,1),(5295,1),(5299,1),(5303,1),(5307,1),(5311,1),(5315,1),(5319,1),(5323,1),(5327,1),(5331,1),(5335,1),(5339,1),(5343,1),(5347,1),(5351,1),(5355,1),(5359,1),(5403,1),(5407,1),(5411,1),(5415,1),(5419,1),(5423,1),(5427,1),(5431,1),(5435,1),(5443,1),(5451,1),(5455,1),(5459,1),(1,2),(2,2),(3,2),(4,2),(5,2),(6,2),(7,2),(8,2),(9,2),(10,2),(11,2),(12,2),(13,2),(14,2),(15,2),(16,2),(17,2),(18,2),(19,2),(20,2),(21,2),(22,2),(23,2),(24,2),(25,2),(26,2),(27,2),(28,2),(29,2),(30,2),(31,2),(32,2),(33,2),(34,2),(35,2),(36,2),(37,2),(38,2),(39,2),(40,2),(41,2),(42,2),(43,2),(44,2),(45,2),(46,2),(47,2),(48,2),(49,2),(50,2),(51,2),(52,2),(53,2),(54,2),(55,2),(56,2),(57,2),(58,2),(59,2),(60,2),(61,2),(62,2),(63,2),(64,2),(65,2),(66,2),(67,2),(68,2),(69,2),(70,2),(71,2),(72,2),(73,2),(74,2),(75,2),(76,2),(77,2),(78,2),(79,2),(80,2),(81,2),(82,2),(83,2),(84,2),(85,2),(86,2),(87,2),(88,2),(89,2),(90,2),(91,2),(92,2),(93,2),(94,2),(95,2),(96,2),(97,2),(98,2),(99,2),(100,2),(101,2),(102,2),(103,2),(104,2),(105,2),(106,2),(107,2),(108,2),(109,2),(110,2),(111,2),(112,2),(113,2),(114,2),(115,2),(116,2),(117,2),(118,2),(119,2),(120,2),(121,2),(122,2),(123,2),(124,2),(125,2),(126,2),(127,2),(128,2),(129,2),(130,2),(131,2),(132,2),(133,2),(134,2),(135,2),(136,2),(137,2),(138,2),(139,2),(140,2),(141,2),(142,2),(143,2),(144,2),(145,2),(146,2),(147,2),(148,2),(149,2),(4184,2),(150,3),(151,3),(152,3),(153,3),(154,3),(155,3),(156,3),(157,3),(158,3),(159,3),(160,3),(161,3),(162,3),(163,3),(164,3),(165,3),(166,3),(167,3),(168,3),(169,3),(170,3),(171,3),(172,3),(173,3),(174,3),(175,3),(176,3),(177,3),(178,3),(179,3),(180,3),(181,3),(182,3),(183,3),(184,3),(185,3),(186,3),(187,3),(188,3),(189,3),(190,3),(191,3),(192,3),(193,3),(194,3),(195,3),(196,3),(197,3),(198,3),(199,3),(200,3),(201,3),(202,3),(203,3),(204,3),(205,3),(206,3),(207,3),(208,3),(209,3),(210,3),(211,3),(212,3),(213,3),(214,3),(215,3),(216,3),(217,3),(218,3),(219,3),(220,3),(221,3),(222,3),(223,3),(224,3),(225,3),(226,3),(227,3),(228,3),(229,3),(230,3),(231,3),(232,3),(233,3),(234,3),(235,3),(236,3),(237,3),(238,3),(239,3),(240,3),(241,3),(242,3),(243,3),(244,3),(245,3),(246,3),(247,3),(248,3),(249,3),(250,3),(251,3),(252,3),(253,3),(254,3),(255,3),(256,3),(257,3),(258,3),(259,3),(260,3),(261,3),(262,3),(263,3),(264,3),(265,3),(266,3),(267,3),(268,3),(269,3),(270,3),(271,3),(272,3),(273,3),(274,3),(275,3),(276,3),(277,3),(278,3),(279,3),(280,3),(281,3),(282,3),(283,3),(284,3),(285,3),(286,3),(287,3),(288,3),(289,3),(290,3),(291,3),(292,3),(293,3),(294,3),(295,3),(296,3),(297,3),(298,3),(299,3),(300,3),(301,3),(302,3),(303,3),(304,3),(305,3),(306,3),(307,3),(308,3),(309,3),(310,3),(311,3),(312,3),(313,3),(314,3),(315,3),(316,3),(317,3),(318,3),(319,3),(320,3),(321,3),(322,3),(323,3),(324,3),(325,3),(326,3),(327,3),(328,3),(329,3),(330,3),(331,3),(332,3),(333,3),(334,3),(335,3),(336,3),(337,3),(338,3),(339,3),(340,3),(341,3),(342,3),(343,3),(344,3),(345,3),(346,3),(347,3),(348,3),(349,3),(350,3),(351,3),(352,3),(353,3),(354,3),(355,3),(356,3),(357,3),(358,3),(359,3),(360,3),(361,3),(362,3),(363,3),(364,3),(365,3),(366,3),(367,3),(368,3),(369,3),(370,3),(371,3),(372,3),(373,3),(374,3),(375,3),(376,3),(377,3),(378,3),(379,3),(380,3),(381,3),(382,3),(383,3),(384,3),(385,3),(386,3),(387,3),(388,3),(389,3),(390,3),(391,3),(392,3),(393,3),(394,3),(395,3),(396,3),(397,3),(398,3),(399,3),(400,3),(401,3),(402,3),(403,3),(404,3),(405,3),(406,3),(407,3),(408,3),(409,3),(410,3),(411,3),(412,3),(413,3),(414,3),(415,3),(416,3),(417,3),(418,3),(419,3),(420,3),(421,3),(422,3),(423,3),(424,3),(425,3),(426,3),(427,3),(428,3),(429,3),(430,3),(431,3),(432,3),(433,3),(434,3),(435,3),(436,3),(437,3),(438,3),(439,3),(440,3),(441,3),(442,3),(443,3),(444,3),(445,3),(446,3),(447,3),(448,3),(449,3),(450,3),(451,3),(452,3),(453,3),(454,3),(455,3),(456,3),(457,3),(458,3),(459,3),(460,3),(461,3),(462,3),(463,3),(464,3),(465,3),(466,3),(467,3),(468,3),(469,3),(470,3),(471,3),(472,3),(473,3),(474,3),(475,3),(476,3),(477,3),(478,3),(479,3),(480,3),(481,3),(482,3),(483,3),(484,3),(485,3),(486,3),(487,3),(488,3),(489,3),(490,3),(491,3),(492,3),(493,3),(494,3),(495,3),(496,3),(497,3),(498,3),(499,3),(500,3),(501,3),(502,3),(503,3),(504,3),(505,3),(506,3),(507,3),(508,3),(509,3),(510,3),(511,3),(512,3),(513,3),(514,3),(515,3),(516,3),(517,3),(518,3),(519,3),(520,3),(521,3),(522,3),(523,3),(524,3),(525,3),(526,3),(527,3),(528,3),(529,3),(530,3),(531,3),(532,3),(533,3),(534,3),(535,3),(536,3),(537,3),(538,3),(539,3),(540,3),(541,3),(542,3),(543,3),(544,3),(545,3),(546,3),(547,3),(548,3),(549,3),(550,3),(551,3),(552,3),(553,3),(554,3),(555,3),(556,3),(557,3),(558,3),(559,3),(560,3),(561,3),(562,3),(563,3),(564,3),(565,3),(566,3),(567,3),(568,3),(569,3),(570,3),(571,3),(572,3),(573,3),(574,3),(575,3),(576,3),(577,3),(578,3),(579,3),(580,3),(581,3),(582,3),(583,3),(584,3),(585,3),(586,3),(587,3),(588,3),(589,3),(590,3),(591,3),(592,3),(593,3),(594,3),(595,3),(596,3),(597,3),(598,3),(599,3),(600,3),(601,3),(602,3),(603,3),(604,3),(605,3),(606,3),(607,3),(608,3),(609,3),(610,3),(611,3),(612,3),(613,3),(614,3),(615,3),(616,3),(617,3),(618,3),(619,3),(620,3),(621,3),(622,3),(623,3),(624,3),(625,3),(626,3),(627,3),(628,3),(629,3),(630,3),(631,3),(632,3),(633,3),(634,3),(635,3),(636,3),(637,3),(638,3),(639,3),(640,3),(641,3),(642,3),(643,3),(644,3),(645,3),(646,3),(647,3),(648,3),(649,3),(650,3),(651,3),(652,3),(653,3),(654,3),(655,3),(656,3),(657,3),(658,3),(659,3),(660,3),(661,3),(662,3),(663,3),(664,3),(665,3),(666,3),(667,3),(668,3),(669,3),(670,3),(671,3),(672,3),(673,3),(674,3);
INSERT INTO dbo.gpAnnotationPropertyOption (idAnnotationProperty, idPropertyOption) VALUES (675,3),(676,3),(677,3),(678,3),(679,3),(680,3),(681,3),(682,3),(683,3),(684,3),(685,3),(686,3),(687,3),(688,3),(689,3),(690,3),(691,3),(692,3),(693,3),(694,3),(695,3),(696,3),(697,3),(698,3),(699,3),(700,3),(701,3),(702,3),(703,3),(704,3),(705,3),(706,3),(707,3),(708,3),(709,3),(710,3),(711,3),(712,3),(713,3),(714,3),(715,3),(716,3),(717,3),(718,3),(719,3),(720,3),(721,3),(722,3),(723,3),(724,3),(725,3),(726,3),(727,3),(728,3),(729,3),(730,3),(731,3),(732,3),(733,3),(734,3),(735,3),(736,3),(737,3),(738,3),(739,3),(740,3),(741,3),(742,3),(743,3),(744,3),(745,3),(746,3),(747,3),(748,3),(749,3),(750,3),(751,3),(752,3),(753,3),(754,3),(755,3),(756,3),(757,3),(758,3),(759,3),(760,3),(761,3),(762,3),(763,3),(764,3),(765,3),(766,3),(767,3),(768,3),(769,3),(770,3),(771,3),(772,3),(773,3),(774,3),(775,3),(776,3),(777,3),(778,3),(779,3),(780,3),(781,3),(782,3),(783,3),(784,3),(785,3),(786,3),(787,3),(788,3),(789,3),(790,3),(791,3),(792,3),(793,3),(794,3),(795,3),(796,3),(797,3),(798,3),(799,3),(800,3),(801,3),(802,3),(803,3),(804,3),(805,3),(806,3),(807,3),(808,3),(809,3),(810,3),(811,3),(812,3),(813,3),(814,3),(815,3),(816,3),(817,3),(818,3),(819,3),(820,3),(821,3),(822,3),(823,3),(824,3),(825,3),(826,3),(827,3),(828,3),(829,3),(830,3),(831,3),(832,3),(833,3),(834,3),(835,3),(836,3),(837,3),(838,3),(839,3),(840,3),(841,3),(842,3),(843,3),(844,3),(845,3),(846,3),(847,3),(848,3),(849,3),(850,3),(851,3),(852,3),(853,3),(854,3),(855,3),(856,3),(857,3),(858,3),(859,3),(860,3),(861,3),(862,3),(863,3),(864,3),(865,3),(866,3),(867,3),(868,3),(869,3),(870,3),(871,3),(872,3),(873,3),(874,3),(875,3),(876,3),(877,3),(878,3),(879,3),(880,3),(881,3),(882,3),(883,3),(884,3),(885,3),(886,3),(887,3),(888,3),(889,3),(890,3),(891,3),(892,3),(893,3),(894,3),(895,3),(896,3),(897,3),(898,3),(899,3),(900,3),(901,3),(902,3),(903,3),(904,3),(905,3),(906,3),(907,3),(908,3),(909,3),(910,3),(911,3),(912,3),(913,3),(914,3),(915,3),(916,3),(917,3),(918,3),(919,3),(920,3),(921,3),(922,3),(923,3),(924,3),(925,3),(926,3),(927,3),(928,3),(930,3),(939,3),(940,3),(941,3),(942,3),(943,3),(944,3),(945,3),(946,3),(947,3),(948,3),(949,3),(950,3),(951,3),(952,3),(953,3),(954,3),(955,3),(956,3),(957,3),(958,3),(959,3),(960,3),(961,3),(962,3),(963,3),(964,3),(965,3),(966,3),(967,3),(968,3),(969,3),(970,3),(971,3),(972,3),(973,3),(974,3),(975,3),(976,3),(977,3),(978,3),(979,3),(980,3),(981,3),(982,3),(983,3),(984,3),(985,3),(986,3),(987,3),(988,3),(989,3),(990,3),(996,3),(1001,3),(1002,3),(1003,3),(1004,3),(1005,3),(1006,3),(1007,3),(1008,3),(1009,3),(1010,3),(1011,3),(1012,3),(1013,3),(1014,3),(1015,3),(1016,3),(1017,3),(1018,3),(1019,3),(1020,3),(1021,3),(1022,3),(1023,3);
INSERT INTO dbo.gpAnnotationPropertyOption (idAnnotationProperty, idPropertyOption) VALUES (1024,3),(1025,3),(1026,3),(1029,3),(1030,3),(1031,3),(1032,3),(1033,3),(1034,3),(1035,3),(1036,3),(1037,3),(1038,3),(1039,3),(1040,3),(1041,3),(1042,3),(1043,3),(1044,3),(1045,3),(1046,3),(1047,3),(1048,3),(1049,3),(1050,3),(1051,3),(1052,3),(1053,3),(1054,3),(1055,3),(1056,3),(1057,3),(1058,3),(1059,3),(1060,3),(1061,3),(1062,3),(1063,3),(1064,3),(1065,3),(1066,3),(1067,3),(1068,3),(1069,3),(1070,3),(1071,3),(1072,3),(1073,3),(1074,3),(1075,3),(1076,3),(1077,3),(1078,3),(1079,3),(1080,3),(1081,3),(1082,3),(1083,3),(1084,3),(1085,3),(1086,3),(1087,3),(1088,3),(1089,3),(1090,3),(1091,3),(1092,3),(1093,3),(1094,3),(1095,3),(1096,3),(1097,3),(1098,3),(1099,3),(1100,3),(1101,3),(1102,3),(1103,3),(1104,3),(1105,3),(1106,3),(1107,3),(1108,3),(1109,3),(1110,3),(1111,3),(1112,3),(1113,3),(1114,3),(1115,3),(1116,3),(1117,3),(1118,3),(1119,3),(1120,3),(1121,3),(1122,3),(1123,3),(1124,3),(1125,3),(1126,3),(1127,3),(1128,3),(1129,3),(1130,3),(1131,3),(1132,3),(1133,3),(1134,3),(1135,3),(1136,3),(1137,3),(1138,3),(1139,3),(1140,3),(1141,3),(1142,3),(1143,3),(1144,3),(1145,3),(1146,3),(1147,3),(1148,3),(1149,3),(1150,3),(1151,3),(1152,3),(1153,3),(1154,3),(1155,3),(1156,3),(1157,3),(1158,3),(1159,3),(1160,3),(1161,3),(1162,3),(1163,3),(1164,3),(1165,3),(1166,3),(1167,3),(1168,3),(1169,3),(1170,3),(1171,3),(1172,3),(1173,3),(1174,3),(1175,3),(1176,3),(1177,3),(1178,3),(1179,3),(1180,3),(1181,3),(1182,3),(1183,3),(1184,3),(1185,3),(1186,3),(1187,3),(1188,3),(1189,3),(1190,3),(1191,3),(1192,3),(1193,3),(1194,3),(1195,3),(1196,3),(1197,3),(1198,3),(1199,3),(1200,3),(1201,3),(1202,3),(1203,3),(1204,3),(1205,3),(1206,3),(1207,3),(1208,3),(1209,3),(1210,3),(1211,3),(1212,3),(1213,3),(1214,3),(1215,3),(1224,3),(1225,3),(1226,3),(1227,3),(1228,3),(1229,3),(1230,3),(1231,3),(1232,3),(1233,3),(1234,3);
INSERT INTO dbo.gpAnnotationPropertyOption (idAnnotationProperty, idPropertyOption) VALUES (1235,3),(1236,3),(1237,3),(1238,3),(1239,3),(1240,3),(1241,3),(1246,3),(1247,3),(1248,3),(1249,3),(1250,3),(1251,3),(1252,3),(1253,3),(1254,3),(1255,3),(1256,3),(1257,3),(1258,3),(1259,3),(1260,3),(1261,3),(1262,3),(1263,3),(1264,3),(1265,3),(1266,3),(1267,3),(1268,3),(1269,3),(1270,3),(1271,3),(1272,3),(1273,3),(1274,3),(1275,3),(1276,3),(1277,3),(1278,3),(1279,3),(1280,3),(1281,3),(1282,3),(1283,3),(1284,3),(1285,3),(1286,3),(1287,3),(1288,3),(1289,3),(1290,3),(1291,3),(1292,3),(1293,3),(3938,3),(3941,3),(3944,3),(3947,3),(3950,3),(4001,3),(4004,3),(4007,3),(4058,3),(4067,3),(4070,3),(4073,3),(4076,3),(4148,3),(4154,3),(4205,3),(4208,3),(4211,3),(4214,3),(4221,3),(4225,3),(4229,3),(4233,3),(4251,3),(4263,3),(4267,3),(4271,3),(4275,3),(4279,3),(4283,3),(4287,3),(4291,3),(4295,3),(4299,3),(4303,3),(4307,3),(4311,3),(4315,3),(4323,3),(4327,3),(4331,3),(4335,3),(4339,3),(4343,3),(4347,3),(4351,3),(4355,3),(4359,3),(4363,3),(4367,3),(4371,3),(4375,3),(4379,3),(4383,3),(4387,3),(4391,3),(4395,3),(4399,3),(4403,3),(4407,3),(4411,3),(4415,3),(4419,3),(4423,3),(4427,3),(4431,3),(4435,3),(4439,3),(4443,3),(4447,3),(4451,3),(4455,3),(4459,3),(4463,3),(4467,3),(4471,3),(4475,3),(4479,3),(4483,3),(4487,3),(4491,3),(4495,3),(4499,3),(4503,3),(4507,3),(4511,3),(4515,3),(4519,3),(4523,3),(4527,3),(4531,3),(4535,3),(4539,3),(4543,3),(4547,3),(4551,3),(4555,3),(4559,3),(4563,3),(4567,3),(4571,3),(4575,3),(4579,3),(4583,3),(4587,3),(4591,3),(4595,3),(4599,3),(4603,3),(4607,3),(4611,3),(4615,3),(4619,3),(4623,3),(4627,3),(4631,3),(4635,3),(4639,3),(4643,3),(4647,3),(4651,3),(4655,3),(4659,3),(4663,3),(4667,3),(4671,3),(4675,3),(4679,3),(4683,3),(4687,3),(4691,3),(4695,3),(4699,3),(4703,3),(4707,3),(4711,3),(4883,3),(4919,3),(4947,3),(4951,3),(5235,3),(5239,3),(3949,4),(3955,4),(3958,4),(3961,4),(3964,4),(3967,4),(3973,4),(3976,4),(3979,4),(3985,4),(3988,4),(3991,4),(3994,4),(3997,4),(4000,4),(4018,4),(4021,4),(4024,4),(4027,4),(4033,4),(4042,4),(4063,4),(4066,4),(4081,4),(4084,4),(4087,4),(4090,4),(4093,4),(4096,4),(4099,4),(4102,4),(4105,4),(4108,4),(4111,4),(4114,4),(4117,4),(4120,4),(4147,4),(4153,4),(4159,4),(4162,4),(4165,4),(4168,4),(4171,4),(4174,4),(4177,4),(4180,4),(4183,4),(4195,4),(4204,4),(4240,4),(4262,4),(4322,4),(4722,4),(4726,4),(4730,4),(4734,4),(4750,4);
INSERT INTO dbo.gpAnnotationPropertyOption (idAnnotationProperty, idPropertyOption) VALUES (4754,4),(4758,4),(4762,4),(4766,4),(4770,4),(4774,4),(4794,4),(4798,4),(4802,4),(4806,4),(4810,4),(4814,4),(4818,4),(4822,4),(4838,4),(4842,4),(4846,4),(4858,4),(4862,4),(4866,4),(4874,4),(4878,4),(4882,4),(4890,4),(4926,4),(4930,4),(4934,4),(4938,4),(4942,4),(5050,4),(5054,4),(5058,4),(5062,4),(5066,4),(5070,4),(5074,4),(5078,4),(5082,4),(5086,4),(5090,4),(5094,4),(5098,4),(5102,4),(5106,4),(5110,4),(5238,4),(5242,4),(5246,4),(5250,4),(5254,4),(5258,4),(5262,4),(5266,4),(5270,4),(5274,4),(5278,4),(5282,4),(5286,4),(5290,4),(5294,4),(5298,4),(5302,4),(5306,4),(5310,4),(5314,4),(5318,4),(5322,4),(5326,4),(5330,4),(5334,4),(5338,4),(5342,4),(5346,4),(5350,4),(5354,4),(5358,4),(5362,4),(5406,4),(5410,4),(5414,4),(5418,4),(5422,4),(5426,4),(5430,4),(5434,4),(5438,4),(5446,4),(5454,4),(5458,4),(5462,4),(1294,5),(1295,5),(1296,5),(1297,5),(1298,5),(1299,5),(1300,5),(1301,5),(1302,5),(1303,5),(1304,5),(1305,5),(1306,5),(1307,5),(1308,5),(1309,5),(1310,5),(1311,5),(1312,5),(1313,5),(1314,5),(1315,5),(1316,5),(1317,5),(1318,5),(1319,5),(1320,5),(1321,5),(1322,5),(1323,5),(1324,5),(1325,5),(1326,5),(1327,5),(1328,5),(1329,5),(1330,5),(1331,5),(1332,5),(1333,5),(1334,5),(1335,5),(1336,5),(1337,5),(1338,5),(1339,5),(1340,5),(1341,5),(1342,5),(1343,5),(1344,5),(1345,5),(1346,5),(1347,5),(1348,5),(1349,5),(1350,5),(1351,5),(1352,5),(1353,5),(1354,5),(1355,5),(1356,5),(1357,5),(1358,5),(1359,5),(1360,5),(1361,5),(1362,5),(1363,5),(1364,5),(1365,5),(1366,5),(1367,5),(1368,5),(1369,5),(1370,5),(1371,5),(1372,5),(1373,5),(1374,5),(1375,5),(1376,5),(1377,5),(1378,5),(1379,5),(1380,5),(1381,5),(1382,5),(1383,5),(1384,5),(1385,5),(1386,5),(1387,5),(1388,5),(1389,5),(1390,5),(1391,5),(1392,5),(1393,5),(1394,5),(1395,5),(1396,5),(1397,5),(1398,5),(1399,5),(1400,5),(1401,5),(1402,5),(1403,5),(1404,5),(1405,5),(1406,5),(1407,5),(1408,5),(1409,5),(1410,5),(1411,5),(1412,5),(1413,5),(1414,5),(1415,5),(1416,5),(1417,5),(1418,5),(1419,5),(1420,5),(1421,5),(1422,5),(1423,5),(1424,5),(1425,5),(1426,5),(1427,5),(1428,5),(1429,5),(1430,5),(1431,5),(1432,5),(1433,5),(1434,5),(1435,5),(1436,5),(1437,5),(1438,5),(1439,5),(1440,5),(1441,5),(1442,5),(1443,5),(1444,5),(1445,5),(1446,5),(1447,5),(1448,5),(1449,5),(1450,5),(1451,5),(1452,5),(1453,5),(1454,5),(1455,5),(1456,5),(1457,5),(1458,5),(1459,5),(1460,5),(1461,5),(1462,5),(1463,5),(1464,5),(1465,5),(1466,5),(1467,5),(1468,5),(1469,5),(1470,5),(1471,5),(1472,5),(1473,5),(1474,5),(1475,5),(1476,5),(1477,5),(1478,5),(1479,5),(1480,5),(1481,5),(1482,5),(1483,5),(1484,5),(1485,5),(1486,5),(1487,5),(1488,5),(1489,5),(1490,5),(1491,5),(1492,5),(1493,5),(1494,5),(1495,5),(1496,5),(1497,5),(1498,5),(1499,5),(1500,5),(1501,5),(1502,5),(1503,5),(1504,5),(1505,5),(1506,5),(1507,5),(1508,5),(1509,5),(1510,5),(1511,5),(1512,5),(1513,5),(1514,5),(1515,5),(1516,5),(1517,5),(1518,5),(1519,5),(1520,5),(1521,5),(1522,5),(1523,5),(1524,5),(1525,5),(1526,5),(1527,5),(1528,5),(1529,5),(1530,5),(1531,5),(1532,5),(1533,5),(1534,5),(1535,5),(1536,5),(1537,5),(1538,5),(1539,5),(1540,5),(1541,5),(1542,5),(1543,5),(1544,5),(1545,5),(1546,5),(1547,5),(1548,5),(1549,5),(1550,5),(1551,5),(1552,5),(1553,5),(1554,5),(1555,5),(1556,5),(1557,5),(1558,5),(1559,5),(1560,5),(1561,5),(1562,5),(1563,5),(1564,5),(1565,5),(1566,5),(1567,5),(1568,5),(1569,5),(1570,5),(1571,5),(1572,5),(1573,5),(1574,5),(1575,5),(1576,5),(1577,5),(1578,5),(1579,5),(1580,5),(1581,5),(1582,5),(1583,5),(1584,5),(1585,5),(1586,5),(1587,5),(1588,5),(1589,5),(1590,5),(1591,5),(1592,5),(1593,5),(1594,5),(1595,5),(1596,5),(1597,5),(1598,5),(1599,5),(1600,5),(1601,5),(1602,5),(1603,5),(1604,5),(1605,5),(1606,5),(1607,5),(1608,5),(1609,5),(1610,5),(1611,5),(1612,5),(1613,5),(1614,5),(1615,5),(1616,5),(1617,5),(1618,5),(1619,5),(1620,5),(1621,5),(1622,5),(1623,5),(1624,5),(1625,5),(1626,5),(1627,5),(1628,5),(1629,5),(1630,5),(1631,5),(1632,5),(1633,5),(1634,5),(1635,5),(1636,5),(1637,5),(1638,5),(1639,5),(1640,5),(1641,5),(1642,5),(1643,5),(1644,5),(1645,5),(1646,5),(1647,5),(1648,5),(1649,5),(1650,5),(1651,5),(1652,5),(1653,5),(1654,5),(1655,5),(1656,5),(1657,5),(1658,5),(1659,5),(1660,5),(1661,5),(1662,5),(1663,5),(1664,5),(1665,5),(1666,5),(1667,5),(1668,5),(1669,5),(1670,5),(1671,5),(1672,5),(1673,5),(1674,5),(1675,5),(1676,5),(1677,5),(1678,5),(1679,5),(1680,5),(1681,5),(1682,5),(1683,5),(1684,5),(1685,5),(1686,5),(1687,5),(1688,5),(1689,5),(1690,5),(1691,5),(1692,5),(1693,5),(1694,5),(1695,5),(1696,5),(1697,5),(1698,5),(1699,5),(1700,5),(1701,5),(1702,5),(1703,5),(1704,5),(1705,5),(1706,5),(1707,5),(1708,5),(1709,5),(1710,5),(1711,5),(1712,5),(1713,5),(1714,5),(1715,5),(1716,5),(1717,5),(1718,5),(1719,5),(1720,5),(1721,5),(1722,5),(1723,5),(1724,5),(1725,5),(1726,5),(1727,5),(1728,5),(1729,5),(1730,5),(1731,5),(1732,5),(1733,5),(1734,5),(1735,5),(1736,5),(1737,5);
INSERT INTO dbo.gpAnnotationPropertyOption (idAnnotationProperty, idPropertyOption) VALUES (1738,5),(1739,5),(1740,5),(1741,5),(1742,5),(1743,5),(1744,5),(1745,5),(1746,5),(1747,5),(1748,5),(1749,5),(1750,5),(1751,5),(1752,5),(1753,5),(1754,5),(1755,5),(1756,5),(1757,5),(1758,5),(1759,5),(1760,5),(1761,5),(1762,5),(1763,5),(1764,5),(1765,5),(1766,5),(1767,5),(1768,5),(1769,5),(1770,5),(1771,5),(1772,5),(1773,5),(1774,5),(1775,5),(1776,5),(1777,5),(1778,5),(1779,5),(1780,5),(1781,5),(1782,5),(1783,5),(1784,5),(1785,5),(1786,5),(1787,5),(1788,5),(1789,5),(1790,5),(1791,5),(1792,5),(1793,5),(1794,5),(1795,5),(1796,5),(1797,5),(1798,5),(1799,5),(1800,5),(1801,5),(1802,5),(1803,5),(1804,5),(1805,5),(1806,5),(1807,5),(1808,5),(1809,5),(1810,5),(1811,5),(1812,5),(1813,5),(1814,5),(1815,5),(1816,5),(1817,5),(1818,5),(1819,5),(1820,5),(1821,5),(1822,5),(1823,5),(1824,5),(1825,5),(1826,5),(1827,5),(1828,5),(1829,5),(1830,5),(1831,5),(1832,5),(1833,5),(1834,5),(1835,5),(1836,5),(1837,5),(1838,5),(1839,5),(1840,5),(1841,5),(1842,5),(1843,5),(1844,5),(1845,5),(1846,5),(1847,5),(1848,5),(1849,5),(1850,5),(1851,5),(1852,5),(1853,5),(1854,5),(1855,5),(1856,5),(1857,5),(1858,5),(1859,5),(1860,5),(1861,5),(1862,5),(1863,5),(1864,5),(1865,5),(1866,5),(1867,5),(1868,5),(1869,5),(1870,5),(1871,5),(1872,5),(1873,5),(1874,5),(1875,5),(1876,5),(1877,5),(1878,5),(1879,5),(1880,5),(1881,5),(1882,5),(1883,5),(1884,5),(1885,5),(1886,5),(1887,5),(1888,5),(1889,5),(1890,5),(1891,5),(1892,5),(1893,5),(1894,5),(1895,5),(1896,5),(1897,5),(1898,5),(1899,5),(1900,5),(1901,5),(1902,5),(1903,5),(1904,5),(1905,5),(1906,5),(1907,5),(1908,5),(1909,5),(1910,5),(1911,5),(1912,5),(1913,5),(1914,5),(1915,5),(1916,5),(1917,5),(1918,5),(1919,5),(1920,5),(1921,5),(1922,5),(1923,5),(1924,5),(1925,5),(1926,5),(1927,5),(1928,5),(1929,5),(1930,5),(1931,5),(1932,5),(1933,5),(1934,5),(1935,5),(1936,5),(1937,5),(1938,5),(1939,5),(1940,5),(1941,5),(1942,5),(1943,5),(1944,5),(1945,5),(1946,5),(1947,5),(1948,5),(1949,5),(1950,5),(1951,5),(1952,5),(1953,5),(1954,5),(1955,5),(1956,5),(1957,5),(1958,5),(1959,5),(1960,5),(1961,5),(1962,5),(1963,5),(1964,5),(1965,5),(1966,5),(1967,5),(1968,5),(1969,5),(1970,5),(1971,5),(1972,5),(1973,5),(1974,5),(1975,5),(1976,5),(1977,5),(1978,5),(1979,5),(1980,5),(1981,5),(1982,5),(1983,5),(1984,5),(1985,5),(1986,5),(1987,5),(1988,5),(1989,5),(1990,5),(1991,5),(1992,5),(1993,5),(1994,5),(1995,5),(1996,5),(1997,5),(1998,5),(1999,5),(2000,5),(2001,5),(2002,5),(2003,5),(2004,5),(2005,5),(2006,5),(2007,5),(2008,5),(2009,5),(2010,5),(2011,5),(2012,5),(2013,5),(2014,5),(2015,5),(2016,5),(2017,5),(2018,5),(2019,5),(2020,5),(2021,5),(2022,5),(2023,5),(2024,5),(2025,5),(2026,5),(2027,5),(2028,5),(2029,5),(2030,5),(2031,5),(2032,5),(2033,5),(2034,5),(2035,5),(2036,5),(2037,5),(2038,5),(2039,5),(2040,5),(2041,5),(2042,5),(2043,5),(2044,5),(2045,5),(2046,5),(2047,5),(2048,5),(2049,5),(2050,5),(2051,5),(2052,5),(2053,5),(2054,5),(2055,5),(2056,5),(2058,5),(2067,5),(2068,5),(2069,5),(2070,5),(2071,5),(2072,5),(2073,5),(2074,5),(2075,5),(2076,5),(2077,5),(2078,5),(2079,5),(2080,5),(2081,5),(2082,5),(2083,5),(2084,5),(2085,5),(2086,5),(2087,5),(2088,5),(2089,5),(2090,5),(2091,5),(2092,5),(2093,5),(2094,5),(2095,5),(2096,5),(2097,5),(2098,5),(2099,5),(2100,5),(2101,5),(2102,5),(2103,5),(2104,5),(2105,5),(2106,5),(2107,5),(2108,5),(2109,5),(2110,5),(2111,5),(2112,5),(2113,5),(2114,5),(2115,5),(2116,5),(2117,5),(2118,5),(2128,5),(2129,5),(2130,5),(2131,5),(2132,5),(2133,5),(2134,5),(2135,5),(2136,5),(2137,5),(2138,5),(2139,5),(2140,5),(2141,5),(2142,5),(2143,5),(2144,5),(2145,5),(2146,5),(2147,5),(2148,5),(2149,5),(2150,5),(2151,5),(2152,5),(2153,5),(2156,5),(2157,5),(2158,5),(2159,5),(2160,5),(2161,5),(2162,5),(2163,5),(2164,5),(2165,5),(2166,5),(2167,5),(2168,5),(2169,5),(2170,5),(2171,5),(2172,5),(2173,5),(2174,5),(2175,5),(2176,5),(2177,5),(2178,5);
ALTER TABLE dbo.gpAnnotationPropertyOption CHECK CONSTRAINT all;



--
-- Table structure for table dbo.gpAnnotationPropertyValue --

IF  EXISTS (SELECT * FROM sys.objects WHERE object_id = OBJECT_ID(N'dbo.gpAnnotationPropertyValue') AND type in (N'U')) DROP TABLE dbo.gpAnnotationPropertyValue;


CREATE TABLE dbo.gpAnnotationPropertyValue (
  idAnnotationPropertyValue int  NOT NULL identity,
  value varchar(200)  NULL,
  idAnnotationProperty int  NOT NULL,
CONSTRAINT [PK_AnnotationPropertyValue] PRIMARY KEY CLUSTERED 
(
	[idAnnotationPropertyValue] ASC
),
  
  CONSTRAINT FK_AnnotationPropertyValue_AnnotationProperty FOREIGN KEY (idAnnotationProperty) REFERENCES dbo.gpAnnotationProperty (idAnnotationProperty) ON DELETE NO ACTION ON UPDATE NO ACTION
)  ;


--
-- Dumping data for table dbo.gpAnnotationPropertyValue --


ALTER TABLE dbo.gpAnnotationPropertyValue NOCHECK CONSTRAINT all;
SET IDENTITY_INSERT dbo.gpAnnotationPropertyValue ON;
INSERT INTO dbo.gpAnnotationPropertyValue
( idAnnotationPropertyValue,
value,
idAnnotationProperty
) VALUES (2,'https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A235',4222),(3,'https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A235',4226),(4,'https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A235',4230),(5,'https://hci-as3.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A235',4234),(6,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4324),(7,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4328),(8,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4332),(9,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4336),(10,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4340),(11,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4344),(12,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4348),(13,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4352),(14,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4356),(15,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4360),(16,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4364),(17,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4368),(18,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4372),(19,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4376),(20,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4380),(21,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4384),(22,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4388),(23,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4392),(24,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4396),(25,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4400),(26,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4404),(27,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4408),(28,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4412),(29,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4416),(30,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4420),(31,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4424),(32,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4428),(33,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4432),(34,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4436),(35,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4440),(36,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4444),(37,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4448),(38,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4452),(39,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4456),(40,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4460),(41,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4464),(42,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4468),(43,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4472),(44,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4476),(45,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4480),(46,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4484),(47,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4488),(48,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4492),(49,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4496),(50,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4500),(51,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4504),(52,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4508),(53,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4512),(54,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4516),(55,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4520),(56,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4524),(57,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4528),(58,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4532),(59,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4536),(60,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4540),(61,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4544),(62,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4548),(63,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4552),(64,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4556),(65,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4560),(66,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4564),(67,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4568),(68,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4572),(69,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4576),(70,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4580),(71,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4584),(72,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4588),(73,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4592),(74,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4596),(75,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4600),(76,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4604),(77,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4608),(78,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4612),(79,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4616),(80,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4620),(81,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4624),(82,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4628),(83,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4632),(84,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4636),(85,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4640),(86,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4644),(87,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4648),(88,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4652),(89,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4656),(90,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4660),(91,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4664),(92,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4668),(93,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4672),(94,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4676),(95,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4680),(96,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4684),(97,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4688),(98,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4692),(99,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4696),(100,'https://bioserver.hci.utah.edu/gnomex/gnomexFlex.jsp?launchWindow=AnalysisDetail&analysisNumber=A237',4700),(101,'',4704),(102,'',4708),(103,'',4712);
ALTER TABLE dbo.gpAnnotationPropertyValue CHECK CONSTRAINT all;
SET IDENTITY_INSERT dbo.gpAnnotationPropertyValue OFF;


--
-- Table structure for table dbo.gpAnnotationToAnnotationGrouping --

IF  EXISTS (SELECT * FROM sys.objects WHERE object_id = OBJECT_ID(N'dbo.gpAnnotationToAnnotationGrouping') AND type in (N'U')) DROP TABLE dbo.gpAnnotationToAnnotationGrouping;


CREATE TABLE dbo.gpAnnotationToAnnotationGrouping (
  idAnnotation int  NOT NULL,
  idAnnotationGrouping int  NOT NULL,
CONSTRAINT [PK_AnnotationToAnnGrouping] PRIMARY KEY CLUSTERED 
(
	[idAnnotation] ASC,
	[idAnnotationGrouping] ASC
),
  
  CONSTRAINT FK_AnnotationToAnnotationGrouping_AnnotationGrouping FOREIGN KEY (idAnnotationGrouping) REFERENCES dbo.gpAnnotationGrouping (idAnnotationGrouping),
  CONSTRAINT FK_AnnotationToGrouping_Annotation FOREIGN KEY (idAnnotation) REFERENCES dbo.gpAnnotation (idAnnotation)
) ;


--
-- Dumping data for table dbo.gpAnnotationToAnnotationGrouping --


ALTER TABLE dbo.gpAnnotationToAnnotationGrouping NOCHECK CONSTRAINT all;
SET IDENTITY_INSERT dbo.gpAnnotationToAnnotationGrouping ON;
INSERT INTO dbo.gpAnnotationToAnnotationGrouping (idAnnotation, idAnnotationGrouping) VALUES 
(2,1204),
(3,1204),
(61,2701),
(1305,2701),
(1306,2701),
(1307,2701),
(1844,3405),
(1961,3405),
(1962,3405),
(1963,3405),
(3534,3405),
(3667,3405),
(2295,3806),
(2296,3806),
(2297,3806),
(63,3900),
(117,5401),
(118,5401),
(119,5401),
(120,5401),
(16,6305),
(17,6305),
(18,6305),
(19,6305),
(20,6305),
(21,6305),
(22,6305),
(23,6305),
(24,6305),
(25,6305),
(26,6305),
(27,6305),
(41,6307),
(42,6307),
(43,6307),
(44,6307),
(45,6307),
(46,6307),
(47,6307),
(48,6307),
(49,6307),
(50,6307),
(51,6307),
(52,6307),
(28,6315),
(29,6315),
(31,6315),
(32,6315),
(33,6315),
(34,6315),
(4,6316),
(5,6316),
(6,6316),
(7,6316),
(8,6317),
(9,6317),
(10,6317),
(11,6317),
(12,6318),
(13,6318),
(14,6318),
(15,6318),
(35,6319),
(36,6319),
(37,6319),
(38,6319),
(39,6319),
(40,6319),
(53,6320),
(54,6320),
(55,6321),
(56,6321),
(58,6323),
(59,6323),
(64,6327),
(65,6327),
(66,6327),
(67,6327),
(68,6328),
(69,6328),
(70,6328),
(71,6328),
(72,6329),
(73,6329),
(74,6329),
(75,6329),
(76,6330),
(77,6330),
(78,6330),
(79,6330),
(84,6331),
(85,6331),
(86,6331),
(87,6331),
(80,6332),
(81,6332),
(82,6332),
(83,6332),
(88,6333),
(89,6333),
(90,6333),
(91,6333),
(92,6337),
(93,6337),
(94,6338),
(95,6338),
(96,6338),
(97,6338),
(98,6338),
(99,6338),
(100,6339),
(101,6339),
(102,6342),
(103,6342),
(114,6342),
(105,6346),
(106,6346),
(107,6346),
(108,6349),
(109,6349),
(110,6350),
(111,6350),
(112,6350),
(113,6350),
(116,6351),
(121,6353),
(122,6353),
(123,6353),
(124,6355),
(125,6355),
(126,6355),
(127,6355),
(128,6355),
(129,6355),
(130,6355),
(131,6355),
(132,6355),
(133,6355),
(134,6355),
(135,6355),
(136,6357),
(137,6357),
(138,6357),
(139,6357),
(140,6358),
(141,6358),
(142,6358),
(143,6358),
(144,6358),
(145,6358),
(146,6358),
(147,6358),
(148,6359),
(149,6359),
(150,6359),
(151,6359),
(152,6360),
(153,6360),
(154,6360),
(155,6360),
(156,6360),
(157,6360),
(158,6360),
(159,6360),
(160,6363),
(161,6363),
(162,6363),
(163,6363),
(164,6363),
(165,6363),
(166,6365),
(167,6365),
(168,6366),
(169,6366),
(170,6366),
(171,6366),
(172,6366),
(173,6366),
(174,6368),
(175,6368),
(176,6368),
(177,6368),
(178,6368),
(179,6368),
(234,6370),
(235,6370),
(236,6370),
(237,6371),
(1345,6371),
(241,6375),
(242,6375),
(243,6375),
(244,6375),
(245,6375),
(246,6375),
(247,6375),
(248,6375),
(249,6375),
(250,6375),
(251,6375),
(252,6376),
(253,6376),
(254,6376),
(255,6376),
(256,6376),
(257,6376),
(258,6376),
(259,6376),
(260,6376),
(261,6376),
(262,6376),
(263,6378),
(264,6378),
(265,6378),
(266,6378),
(267,6378),
(268,6378),
(269,6379),
(270,6379),
(271,6379),
(272,6379),
(273,6379),
(274,6379),
(275,6383),
(276,6383),
(277,6383),
(278,6383),
(279,6383),
(280,6383),
(281,6383),
(282,6383),
(283,6383),
(284,6383),
(285,6383),
(286,6383),
(287,6383),
(288,6383),
(289,6383),
(315,6384),
(316,6384),
(317,6384),
(318,6384),
(319,6384),
(320,6384),
(321,6384),
(322,6384),
(323,6384),
(324,6384),
(325,6384),
(326,6384),
(327,6384),
(328,6384),
(329,6384),
(330,6384),
(331,6384),
(332,6384),
(333,6384),
(334,6384),
(335,6384),
(336,6384),
(337,6384),
(338,6384),
(339,6384),
(340,6384),
(341,6384),
(342,6384),
(343,6384),
(344,6384),
(345,6384),
(346,6384),
(347,6384),
(348,6384),
(349,6384),
(350,6384),
(351,6384),
(352,6384),
(353,6384),
(354,6384),
(355,6384),
(356,6384),
(357,6384),
(358,6384),
(359,6384),
(360,6384),
(361,6385),
(362,6385),
(363,6385),
(364,6385),
(365,6385),
(366,6385),
(367,6385),
(368,6385),
(369,6385),
(370,6385),
(371,6385),
(372,6385),
(373,6385),
(374,6385),
(375,6385),
(376,6385),
(377,6385),
(378,6385),
(379,6385),
(380,6385),
(381,6385),
(382,6385),
(383,6385),
(384,6385),
(385,6385),
(386,6385),
(387,6385),
(388,6385),
(389,6385),
(390,6385),
(391,6385),
(392,6385),
(393,6385),
(394,6385),
(395,6385),
(397,6385),
(398,6385),
(399,6385),
(400,6385),
(401,6385),
(402,6385),
(403,6385),
(404,6385),
(405,6385),
(406,6385),
(407,6385),
(290,6386),
(291,6386),
(292,6386),
(293,6386),
(294,6386),
(296,6388),
(297,6388),
(298,6388),
(299,6388),
(301,6388),
(302,6388),
(303,6388),
(304,6388),
(305,6389),
(306,6389),
(307,6389),
(308,6389),
(309,6389),
(310,6389),
(311,6389),
(312,6389),
(313,6389),
(314,6389),
(409,6391),
(410,6391),
(411,6392),
(412,6392),
(429,6397),
(430,6397),
(413,6398),
(414,6398),
(415,6399),
(416,6399),
(417,6401),
(418,6401),
(419,6402),
(420,6402),
(421,6404),
(422,6404),
(423,6405),
(424,6405),
(425,6406),
(426,6406),
(427,6407),
(428,6408),
(431,6410),
(432,6410),
(433,6410),
(434,6410),
(435,6410),
(436,6410),
(437,6410),
(438,6410),
(439,6410),
(440,6410),
(441,6410),
(442,6410),
(443,6411),
(444,6411),
(445,6411),
(446,6411),
(447,6411),
(448,6411),
(449,6411),
(450,6411),
(461,6415),
(462,6415),
(463,6415),
(464,6415),
(451,6417),
(452,6417),
(453,6417),
(454,6417),
(470,6420),
(471,6420),
(472,6420),
(473,6420),
(482,6422),
(483,6422),
(484,6422),
(485,6422),
(486,6423),
(487,6423),
(488,6423),
(489,6423),
(490,6424),
(491,6424),
(492,6424),
(493,6424),
(455,6426),
(456,6426),
(457,6427),
(458,6427),
(459,6428),
(460,6428),
(466,6429),
(467,6429),
(468,6430),
(469,6430),
(474,6432),
(475,6432),
(476,6433),
(477,6433),
(478,6435),
(479,6435),
(480,6436),
(481,6436),
(494,6437),
(495,6437),
(496,6438),
(497,6438),
(498,6440),
(499,6440),
(500,6440),
(501,6440),
(502,6440),
(503,6440),
(504,6440),
(505,6440),
(506,6440),
(507,6440),
(508,6440),
(509,6440),
(510,6440),
(511,6440),
(512,6440),
(513,6440),
(514,6440),
(515,6440),
(516,6440),
(517,6440),
(518,6440),
(519,6440),
(520,6440),
(521,6441),
(522,6441),
(523,6441),
(524,6441),
(525,6441),
(526,6441),
(527,6441),
(528,6441),
(529,6441),
(530,6441),
(531,6441),
(532,6441),
(533,6441),
(534,6441),
(535,6441),
(536,6441),
(537,6441),
(538,6441),
(539,6441),
(540,6441),
(541,6441);
INSERT INTO dbo.gpAnnotationToAnnotationGrouping (idAnnotation, idAnnotationGrouping) VALUES 
(542,6441),
(543,6441),
(544,6442),
(545,6442),
(546,6442),
(547,6442),
(548,6442),
(549,6442),
(550,6442),
(551,6442),
(552,6442),
(553,6442),
(554,6442),
(555,6442),
(556,6442),
(557,6442),
(558,6442),
(559,6442),
(560,6442),
(561,6442),
(562,6442),
(563,6442),
(564,6442),
(565,6442),
(566,6442),
(567,6444),
(568,6444),
(569,6444),
(570,6444),
(571,6444),
(572,6444),
(573,6444),
(574,6444),
(575,6444),
(576,6444),
(577,6444),
(578,6444),
(579,6444),
(580,6444),
(581,6444),
(582,6444),
(583,6445),
(584,6445),
(585,6445),
(586,6445),
(587,6445),
(588,6445),
(589,6445),
(590,6445),
(591,6445),
(592,6445),
(593,6445),
(594,6445),
(595,6445),
(596,6445),
(597,6445),
(598,6445),
(599,6449),
(600,6449),
(601,6449),
(602,6450),
(603,6450),
(604,6450),
(605,6451),
(606,6451),
(607,6451),
(608,6451),
(609,6451),
(610,6452),
(611,6452),
(612,6452),
(613,6452),
(614,6452),
(615,6452),
(616,6452),
(624,6457),
(625,6457),
(626,6457),
(627,6458),
(628,6458),
(629,6458),
(630,6459),
(631,6459),
(632,6459),
(633,6460),
(634,6460),
(635,6460),
(636,6460),
(637,6460),
(638,6460),
(639,6461),
(640,6461),
(641,6462),
(642,6462),
(643,6463),
(644,6463),
(645,6464),
(646,6464),
(647,6464),
(648,6464),
(649,6465),
(650,6465),
(651,6467),
(652,6467),
(653,6467),
(654,6467),
(655,6470),
(656,6470),
(657,6470),
(658,6471),
(659,6471),
(660,6471),
(661,6472),
(662,6472),
(663,6472),
(664,6472),
(665,6473),
(666,6473),
(667,6473),
(668,6474),
(670,6474),
(671,6474),
(669,6475),
(759,6479),
(760,6479),
(761,6479),
(762,6479),
(765,6489),
(766,6489),
(767,6489),
(768,6489),
(769,6489),
(770,6489),
(771,6489),
(772,6489),
(773,6489),
(774,6489),
(775,6489),
(776,6489),
(777,6489),
(778,6489),
(779,6489),
(780,6489),
(781,6489),
(782,6489),
(783,6489),
(784,6489),
(785,6489),
(786,6489),
(787,6489),
(788,6489),
(789,6489),
(790,6489),
(791,6489),
(792,6489),
(793,6489),
(794,6490),
(795,6490),
(796,6490),
(797,6490),
(798,6490),
(799,6490),
(800,6490),
(801,6490),
(802,6490),
(803,6490),
(804,6490),
(805,6490),
(806,6490),
(807,6490),
(808,6490),
(809,6490),
(810,6490),
(811,6490),
(812,6490),
(813,6490),
(814,6490),
(815,6490),
(816,6490),
(817,6490),
(818,6490),
(819,6490),
(820,6490),
(821,6490),
(822,6490),
(1966,6491),
(1967,6491),
(1968,6491),
(1969,6491),
(1970,6491),
(1971,6491),
(1972,6491),
(1973,6491),
(1974,6491),
(1975,6491),
(1976,6491),
(1977,6491),
(1978,6491),
(1979,6491),
(1980,6491),
(1981,6491),
(1982,6491),
(1983,6491),
(1984,6491),
(1985,6491),
(1986,6491),
(1987,6491),
(1988,6491),
(1989,6491),
(1990,6491),
(1991,6491),
(1992,6491),
(1993,6491),
(1994,6491),
(1995,6492),
(1996,6492),
(1997,6492),
(1998,6492),
(1999,6492),
(2000,6492),
(2001,6492),
(2002,6492),
(2003,6492),
(2004,6492),
(2005,6492),
(2006,6492),
(2007,6492),
(2008,6492),
(2009,6492),
(2010,6492),
(2011,6492),
(2012,6492),
(2013,6492),
(2014,6492),
(2015,6492),
(2016,6492),
(2017,6492),
(2018,6492),
(2019,6492),
(2020,6492),
(2021,6492),
(2022,6492),
(2023,6492),
(2162,6493),
(2163,6493),
(1341,6505),
(1158,6506),
(1159,6509),
(1160,6509),
(1161,6509),
(1162,6510),
(1163,6511),
(1164,6511),
(1165,6511),
(1166,6515),
(1167,6516),
(1168,6517),
(1169,6517),
(1170,6517),
(1171,6517),
(1259,6527),
(1260,6527),
(1262,6527),
(1263,6529),
(1264,6529),
(1265,6530),
(1266,6530),
(1267,6530),
(1268,6530),
(1269,6531),
(1270,6531),
(1271,6531),
(1272,6531),
(2140,6531),
(1273,6532),
(1274,6532),
(1275,6532),
(1276,6533),
(1277,6533),
(1278,6533),
(1285,6541),
(1286,6541),
(1287,6541),
(1288,6541),
(1289,6541),
(1290,6541),
(1291,6541),
(1292,6541),
(1293,6541),
(1294,6543),
(1295,6543),
(1296,6543),
(1297,6543),
(1298,6543),
(1299,6543),
(1300,6543),
(1301,6543),
(1302,6567),
(1303,6567),
(1329,6599),
(1330,6599),
(1331,6599),
(1332,6599),
(1333,6600),
(1334,6600),
(1335,6600),
(1336,6600),
(1337,6601),
(1338,6601),
(1339,6601),
(1340,6601),
(2377,6601),
(2387,6601),
(3620,6601),
(3621,6601),
(3622,6601),
(4361,6601),
(4362,6601),
(4364,6601),
(1354,6604),
(1355,6604),
(1356,6604),
(1357,6604),
(1358,6604),
(1359,6604),
(1360,6604),
(1361,6604),
(1362,6604),
(1363,6604),
(1364,6604),
(1365,6604),
(1366,6604),
(1367,6604),
(1369,6604),
(1370,6604),
(1371,6604),
(1372,6604),
(1373,6604),
(1374,6604),
(1375,6604),
(1376,6604),
(1377,6604),
(1378,6604),
(1379,6604),
(1380,6604),
(1381,6604),
(1382,6604),
(1383,6604),
(1384,6604),
(1385,6604),
(1386,6604),
(1387,6604),
(1388,6604),
(1389,6604),
(1390,6604),
(1391,6604),
(1392,6604),
(1393,6604),
(1394,6604),
(1395,6604),
(1396,6604),
(1397,6604),
(1398,6604),
(1399,6604),
(1400,6604),
(1401,6604),
(1402,6604),
(1403,6604),
(1404,6604),
(1405,6604),
(1406,6604),
(1407,6604),
(1408,6604),
(1409,6604),
(1410,6604),
(1411,6604),
(1412,6604),
(1413,6604),
(1414,6604),
(1415,6604),
(1416,6604),
(1677,6605),
(1678,6605),
(1679,6605),
(1680,6605),
(1681,6605),
(1682,6605),
(1683,6605),
(1684,6605),
(1685,6605),
(1686,6605),
(1687,6605),
(1688,6605),
(1689,6605),
(1690,6605),
(1691,6605),
(1692,6605),
(1693,6605),
(1694,6605),
(1695,6605),
(1696,6605),
(1697,6605),
(1698,6605),
(1699,6605),
(1700,6605),
(1701,6605),
(1702,6605),
(1703,6605),
(1704,6605),
(1705,6605),
(1706,6605),
(1707,6605),
(1708,6605),
(1709,6605),
(1710,6605),
(1711,6605),
(1712,6605),
(1713,6605),
(1714,6605),
(1715,6605),
(1716,6605),
(1717,6605),
(1718,6605),
(1719,6605),
(1720,6605),
(1721,6605),
(1722,6605),
(1723,6605),
(1724,6605),
(1725,6605),
(1726,6605),
(1727,6605),
(1728,6605),
(1729,6605),
(1730,6605),
(1731,6605),
(1732,6605),
(1733,6605),
(1734,6605),
(1735,6605),
(1736,6605),
(1737,6605),
(1738,6605),
(1739,6605),
(1740,6605),
(1741,6605),
(1742,6605),
(1743,6605),
(1744,6605),
(1745,6605),
(1746,6605),
(1747,6605),
(1748,6605),
(1749,6605);

INSERT INTO dbo.gpAnnotationToAnnotationGrouping (idAnnotation, idAnnotationGrouping) VALUES 
(1750,6605),
(1751,6605),
(1752,6605),
(1753,6605),
(1754,6605),
(1755,6605),
(1756,6605),
(1424,6609),
(1425,6609),
(1426,6609),
(1427,6609),
(1428,6609),
(1429,6609),
(1430,6610),
(1431,6610),
(1432,6611),
(1433,6611),
(2234,6611),
(1434,6612),
(1435,6612),
(1837,6612),
(1838,6612),
(1839,6612),
(1840,6612),
(1841,6612),
(1893,6612),
(1932,6612),
(1954,6612),
(1958,6612),
(1959,6612),
(1960,6612),
(2185,6612),
(2289,6612),
(2306,6612),
(2307,6612),
(2328,6612),
(2349,6612),
(2896,6612),
(2897,6612),
(2898,6612),
(2899,6612),
(3418,6612),
(3419,6612),
(3420,6612),
(3421,6612),
(3422,6612),
(3423,6612),
(3424,6612),
(3425,6612),
(3527,6612),
(3531,6612),
(3643,6612),
(3999,6612),
(4000,6612),
(1449,6614),
(1450,6614),
(1451,6614),
(1452,6614),
(1453,6614),
(1454,6614),
(1455,6614),
(1456,6614),
(1457,6614),
(1458,6614),
(1459,6614),
(1460,6614),
(1461,6614),
(1462,6614),
(1463,6614),
(1464,6614),
(1465,6614),
(1466,6614),
(1467,6614),
(1468,6614),
(1469,6614),
(1470,6614),
(1471,6614),
(1472,6614),
(1473,6614),
(1474,6614),
(1475,6614),
(1476,6614),
(1477,6614),
(1478,6614),
(1479,6614),
(1480,6614),
(1481,6614),
(1482,6614),
(1483,6614),
(1484,6614),
(1485,6614),
(1486,6614),
(1487,6614),
(1488,6614),
(1489,6614),
(1490,6614),
(1491,6614),
(1492,6614),
(1493,6614),
(1494,6614),
(1495,6614),
(1496,6614),
(1497,6614),
(1498,6614),
(1499,6614),
(1500,6614),
(1501,6614),
(1502,6614),
(1503,6614),
(1504,6614),
(1505,6614),
(1506,6614),
(1507,6614),
(1508,6614),
(1509,6614),
(1510,6614),
(1511,6614),
(1512,6614),
(1513,6614),
(1514,6614),
(1515,6614),
(1516,6614),
(1517,6614),
(1518,6614),
(1519,6614),
(1598,6615),
(1599,6615),
(1600,6615),
(1601,6615),
(1602,6615),
(1603,6615),
(1604,6615),
(1605,6615),
(1606,6615),
(1607,6615),
(1608,6615),
(1609,6615),
(1610,6615),
(1611,6615),
(1612,6615),
(1613,6615),
(1614,6615),
(1615,6615),
(1616,6615),
(1617,6615),
(1618,6615),
(1619,6615),
(1620,6615),
(1621,6615),
(1622,6615),
(1623,6615),
(1624,6615),
(1625,6615),
(1626,6615),
(1627,6615),
(1628,6615),
(1629,6615),
(1630,6615),
(1631,6615),
(1632,6615),
(1633,6615),
(1634,6615),
(1635,6615),
(1636,6615),
(1637,6615),
(1638,6615),
(1639,6615),
(1640,6615),
(1641,6615),
(1642,6615),
(1643,6615),
(1644,6615),
(1645,6615),
(1646,6615),
(1647,6615),
(1648,6615),
(1649,6615),
(1650,6615),
(1651,6615),
(1652,6615),
(1653,6615),
(1654,6615),
(1655,6615),
(1656,6615),
(1657,6615),
(1658,6615),
(1659,6615),
(1660,6615),
(1661,6615),
(1662,6615),
(1663,6615),
(1664,6615),
(1665,6615),
(1666,6615),
(1667,6615),
(1668,6615),
(1669,6615),
(1670,6615),
(1671,6615),
(1672,6615),
(1673,6615),
(1674,6615),
(1675,6615),
(1676,6615),
(1546,6620),
(1547,6620),
(1548,6620),
(1549,6620),
(1550,6620),
(1551,6620),
(1552,6620),
(1553,6620),
(1554,6620),
(1555,6620),
(1556,6620),
(1557,6620),
(1558,6620),
(1559,6620),
(1560,6620),
(1561,6620),
(1562,6620),
(1563,6620),
(1564,6620),
(1565,6620),
(1566,6620),
(1567,6620),
(1568,6620),
(1569,6620),
(1570,6620),
(1521,6621),
(1522,6621),
(1523,6621),
(1524,6621),
(1525,6621),
(1526,6621),
(1527,6621),
(1528,6621),
(1529,6621),
(1530,6621),
(1531,6621),
(1532,6621),
(1533,6621),
(1534,6621),
(1535,6621),
(1536,6621),
(1537,6621),
(1538,6621),
(1539,6621),
(1540,6621),
(1541,6621),
(1542,6621),
(1543,6621),
(1544,6621),
(1545,6621),
(1571,6622),
(1572,6622),
(1573,6622),
(1574,6622),
(1575,6622),
(1576,6622),
(1577,6622),
(1578,6622),
(1579,6622),
(1580,6622),
(1581,6622),
(1582,6622),
(1583,6622),
(1584,6622),
(1585,6622),
(1586,6622),
(1587,6622),
(1588,6622),
(1589,6622),
(1590,6622),
(1591,6622),
(1592,6622),
(1593,6622),
(1594,6622),
(1595,6622),
(1758,6624),
(1759,6624),
(1760,6624),
(1761,6624),
(1762,6624),
(1763,6624),
(1764,6624),
(1765,6625),
(1766,6625),
(1767,6625),
(1768,6625),
(1769,6625),
(1770,6625),
(1771,6625),
(1772,6625),
(1773,6625);

INSERT INTO dbo.gpAnnotationToAnnotationGrouping (idAnnotation, idAnnotationGrouping) VALUES 
(1774,6625),
(1775,6625),
(1776,6625),
(1777,6625),
(1778,6625),
(1779,6625),
(1780,6625),
(1781,6625),
(1782,6625),
(1783,6625),
(1784,6625),
(1785,6625),
(1786,6625),
(1787,6625),
(1792,6626),
(1793,6626),
(3948,6626),
(3949,6626),
(1799,6639),
(1800,6639),
(1801,6639),
(1802,6639),
(1803,6639),
(1806,6639),
(1807,6639),
(1805,6641),
(1808,6643),
(1809,6643),
(1810,6643),
(1811,6643),
(1812,6645),
(1813,6645),
(1814,6645),
(1815,6645),
(1816,6646),
(1817,6646),
(1818,6646),
(1819,6646),
(1827,6647),
(1828,6647),
(1829,6647),
(1820,6648),
(1821,6648),
(1823,6648),
(1822,6649),
(1824,6649),
(1825,6649),
(1826,6649),
(1830,6652),
(1831,6652),
(1832,6652),
(1833,6654),
(1834,6654),
(1835,6655),
(1842,6657),
(2200,6657),
(2235,6657),
(2562,6657),
(2592,6657),
(2631,6657),
(2640,6657),
(2882,6657),
(3641,6657),
(3642,6657),
(1894,6658),
(1898,6658),
(1902,6658),
(1933,6658),
(1934,6658),
(1850,6659),
(1896,6659),
(1900,6659),
(1904,6659),
(1935,6659),
(1936,6659),
(1937,6659),
(1895,6660),
(1899,6660),
(1903,6660),
(1947,6660),
(1948,6660),
(1897,6661),
(1901,6661),
(1905,6661),
(1949,6661),
(1950,6661),
(1921,6664),
(1924,6664),
(1925,6664),
(1938,6664),
(1939,6664),
(1940,6664),
(1922,6665),
(1926,6665),
(1927,6665),
(1941,6665),
(1942,6665),
(1943,6665),
(1923,6666),
(1928,6666),
(1929,6666),
(1944,6666),
(1945,6666),
(1946,6666),
(1852,6668),
(1853,6668),
(1854,6669),
(1855,6669),
(1856,6669),
(1857,6669),
(2298,6671),
(2299,6671),
(1859,6672),
(1860,6672),
(2453,6672),
(1865,6674),
(1861,6675),
(1862,6675),
(1863,6675),
(1864,6675),
(1866,6676),
(1867,6676),
(1868,6676),
(1869,6678),
(1870,6678),
(1871,6678),
(1872,6678),
(1873,6680),
(1874,6680),
(1875,6680),
(1876,6680),
(1877,6680),
(1878,6680),
(1879,6684),
(1880,6684),
(1881,6684),
(1882,6684),
(1883,6684),
(1884,6684),
(1885,6684),
(1886,6687),
(1887,6687),
(1888,6688),
(1889,6688),
(1890,6688),
(1891,6689),
(1930,6691),
(1931,6692),
(2024,6693),
(2025,6693),
(2026,6693),
(2027,6693),
(2028,6693),
(2029,6693),
(2030,6693),
(2031,6693),
(2032,6693),
(2033,6693),
(2034,6693),
(2035,6693),
(2036,6693),
(2037,6693),
(2038,6693),
(2039,6693),
(2040,6693),
(2041,6693),
(2042,6693),
(2043,6693),
(2044,6693),
(2045,6693),
(2046,6693),
(2047,6693),
(2048,6693),
(2049,6693),
(2050,6693),
(2051,6693),
(2052,6693),
(2074,6694),
(2075,6694),
(2076,6694),
(2077,6694),
(2078,6694),
(2079,6694),
(2080,6694),
(2081,6694),
(2082,6694),
(2083,6694),
(2084,6694),
(2085,6694),
(2086,6694),
(2087,6694),
(2088,6694),
(2089,6694),
(2090,6694),
(2091,6694),
(2092,6694),
(2093,6694),
(2094,6694),
(2095,6694),
(2096,6694),
(2097,6694),
(2098,6694),
(2099,6694),
(2100,6694),
(2101,6694),
(2102,6694),
(823,6695),
(2103,6695),
(2104,6695),
(2105,6695),
(2106,6695),
(2107,6695),
(2108,6695),
(2109,6695),
(2110,6695),
(2111,6695),
(2112,6695),
(2113,6695),
(2114,6695),
(2115,6695),
(2116,6695),
(2117,6695),
(2118,6695),
(2119,6695),
(2120,6695),
(2121,6695),
(2122,6695),
(2123,6695),
(2124,6695),
(2125,6695),
(2126,6695),
(2127,6695),
(2128,6695),
(2129,6695),
(2130,6695),
(2131,6695),
(2132,6697),
(2142,6697),
(2143,6697),
(2134,6701),
(2135,6701),
(2136,6701),
(2137,6701),
(2138,6702),
(2141,6702),
(2139,6703),
(2144,6706),
(2145,6706),
(2146,6706),
(2147,6706),
(2148,6706),
(2149,6706),
(2150,6708),
(2151,6708),
(2152,6708),
(2153,6709),
(2154,6709),
(2155,6709),
(2160,6712),
(2161,6712),
(881,6713),
(882,6713),
(883,6713),
(2167,6717),
(2168,6717),
(2169,6720),
(2170,6720),
(2171,6720),
(2172,6720),
(2173,6722),
(2174,6722),
(2175,6722),
(2176,6722),
(2177,6722),
(2178,6722),
(2179,6722),
(2180,6722),
(2181,6722),
(2182,6722),
(2183,6722),
(2188,6723),
(2193,6723),
(2194,6723),
(2211,6726),
(2212,6726),
(2213,6726),
(2214,6726),
(2215,6726),
(2216,6726),
(2217,6726),
(2218,6726),
(2219,6726),
(2220,6726),
(2221,6726),
(2222,6726),
(2223,6726),
(2224,6726),
(2225,6726),
(2226,6726),
(2228,6726),
(2229,6726),
(2230,6726),
(2231,6726),
(2365,6726),
(2368,6726),
(2816,6726),
(2236,6727),
(2239,6727),
(2263,6732),
(2264,6732),
(2265,6732),
(2267,6733),
(2269,6735),
(2285,6735),
(2286,6735),
(2268,6736),
(3779,6736),
(3934,6736),
(3935,6736),
(2272,6739),
(2287,6739),
(2288,6739),
(2271,6740),
(2273,6743),
(2274,6743),
(2275,6743),
(2276,6743),
(2277,6745),
(2278,6745),
(2279,6746),
(2280,6746),
(2281,6747),
(2282,6747),
(2283,6748),
(2284,6748);


INSERT INTO dbo.gpAnnotationToAnnotationGrouping (idAnnotation, idAnnotationGrouping) VALUES 
(2290,6749),
(2291,6749),
(2292,6749),
(2293,6749),
(2294,6749),
(2350,6751),
(2353,6752),
(2354,6752),
(2355,6753),
(2356,6753),
(2357,6754),
(2358,6754),
(2359,6754),
(2360,6754),
(2361,6755),
(2362,6755),
(2363,6755),
(2364,6755),
(2366,6757),
(2367,6758),
(2371,6759),
(2372,6759),
(2373,6759),
(2374,6759),
(2375,6759),
(2376,6759),
(2388,6762),
(2389,6762),
(2390,6762),
(2391,6762),
(2392,6762),
(2393,6762),
(2394,6762),
(2395,6762),
(2396,6762),
(2397,6762),
(2398,6762),
(2423,6764),
(2426,6764),
(2427,6764),
(2428,6764),
(2429,6764),
(2430,6764),
(2431,6764),
(2432,6764),
(2433,6764),
(2435,6765),
(2436,6765),
(2437,6765),
(2438,6765),
(2439,6765),
(2440,6765),
(2441,6765),
(2442,6765),
(2443,6765),
(2401,6766),
(2402,6766),
(2403,6766),
(2404,6766),
(2405,6766),
(2406,6766),
(2407,6766),
(2408,6766),
(2409,6766),
(2410,6766),
(2411,6766),
(2412,6766),
(2413,6766),
(2414,6766),
(2415,6766),
(2416,6766),
(2417,6766),
(2418,6766),
(2419,6766),
(2420,6766),
(2421,6766),
(2422,6766),
(2444,6767),
(2446,6767),
(2459,6776),
(2460,6776),
(2461,6776),
(2465,6778),
(2466,6778),
(2467,6778),
(2462,6779),
(2463,6779),
(2464,6779),
(2468,6784),
(2469,6784),
(2470,6785),
(2471,6785),
(2472,6785),
(2473,6786),
(2474,6786),
(2475,6787),
(2476,6787),
(2477,6787),
(2478,6788),
(2502,6791),
(2503,6791),
(2504,6791),
(2505,6791),
(2511,6792),
(2513,6792),
(2516,6792),
(2518,6792),
(2488,6793),
(2527,6794),
(2528,6794),
(2529,6794),
(2530,6794),
(2494,6796),
(2495,6798),
(2496,6798),
(2497,6798),
(2498,6799),
(2499,6799),
(2501,6799),
(2525,6807),
(2531,6808),
(2532,6808),
(2533,6808),
(2534,6808),
(2536,6813),
(2601,6815),
(2602,6815),
(2603,6815),
(2604,6815),
(2605,6816),
(2606,6816),
(2607,6816),
(2608,6816),
(2609,6817),
(2610,6817),
(2611,6817),
(2612,6817),
(2597,6822),
(2598,6822),
(2593,6823),
(2594,6823),
(2595,6823),
(2596,6823),
(2599,6824),
(2600,6824),
(2613,6825),
(2614,6825),
(2615,6825),
(2616,6825),
(2617,6827),
(2618,6827),
(2778,6828),
(3476,6830),
(2779,6832),
(2780,6832),
(2781,6832),
(2782,6832),
(2783,6833),
(2784,6833),
(2785,6833),
(2786,6833),
(2787,6835),
(2788,6835),
(2789,6835),
(2790,6835),
(2791,6835),
(2795,6836),
(2796,6836),
(2797,6836),
(2798,6836),
(2799,6836),
(2800,6836),
(2801,6836),
(2792,6837),
(2793,6837),
(2794,6837),
(2802,6838),
(2821,6844),
(2822,6844),
(2823,6844),
(2824,6844),
(2825,6844),
(2826,6844),
(2827,6846),
(2828,6846),
(2829,6846),
(2830,6846),
(2831,6846),
(2832,6846),
(2833,6847),
(2834,6847),
(2835,6847),
(2836,6847),
(2837,6847),
(2838,6847),
(3484,6848),
(2817,6849),
(2818,6849),
(2819,6849),
(2820,6849),
(2840,6851),
(2842,6851),
(2844,6851),
(2849,6851),
(2851,6851),
(2852,6851),
(2853,6851),
(2854,6851),
(3635,6851),
(3636,6851),
(3637,6851),
(3638,6851),
(2845,6853),
(2846,6854),
(2858,6855),
(2859,6855),
(2860,6855),
(2883,6859),
(2886,6859),
(2887,6859),
(2888,6859),
(2889,6859),
(2890,6859),
(2891,6859),
(2892,6860),
(2893,6860),
(2895,6864),
(2902,6869),
(2903,6869),
(2904,6869),
(2905,6870),
(2906,6870),
(2907,6870),
(2908,6873),
(2909,6873),
(2967,6873),
(2910,6874),
(2911,6874),
(2912,6874),
(2913,6874),
(2917,6875),
(2918,6875),
(2919,6875),
(2920,6875),
(2921,6875),
(2922,6875),
(2923,6875),
(2924,6875),
(2925,6875),
(2926,6875),
(2927,6875),
(2928,6875),
(2929,6875),
(2930,6875),
(2931,6875),
(2932,6875),
(2933,6875),
(2934,6875),
(2935,6875),
(2936,6875),
(2937,6875),
(2938,6875),
(2939,6875),
(2940,6875),
(2941,6878),
(2942,6879),
(3623,6880),
(3624,6880),
(2943,6881),
(2944,6881),
(2946,6883),
(2952,6883),
(2953,6883),
(2954,6883),
(2962,6883),
(2947,6889),
(2949,6889),
(2950,6889),
(2957,6890),
(2958,6890),
(2959,6890),
(2960,6890),
(3041,6892),
(3042,6892),
(2968,6894),
(2969,6894),
(3101,6894),
(2970,6895),
(2978,6900),
(2979,6900),
(2980,6900),
(2981,6901),
(2982,6901),
(2983,6901),
(2987,6904),
(2988,6904),
(2989,6905),
(3077,6908),
(3078,6908),
(3076,6909),
(3079,6909),
(3080,6909),
(3081,6910),
(3014,6912),
(3015,6912),
(3016,6912),
(3017,6912),
(3018,6912),
(3019,6912),
(3596,6913),
(3597,6913),
(3599,6913),
(3600,6913),
(3602,6913),
(3603,6913),
(3598,6914),
(3601,6914),
(3604,6914),
(3043,6916),
(3044,6916),
(3052,6917),
(3053,6917),
(3056,6918),
(3057,6918),
(3047,6919),
(3048,6919),
(3045,6920),
(3046,6920),
(3054,6921),
(3055,6921),
(3062,6922),
(3063,6922),
(3060,6923),
(3061,6923),
(3082,6928),
(3083,6928),
(3098,6932),
(3099,6932),
(3100,6932),
(3102,6934),
(3103,6934),
(3104,6934),
(3105,6935),
(3106,6935),
(3107,6935),
(3108,6935),
(3109,6935),
(3110,6935),
(3111,6936),
(3112,6936),
(3115,6940),
(3116,6940),
(3117,6940),
(3118,6940),
(3119,6940),
(3120,6940),
(3121,6940),
(3122,6940),
(3123,6940),
(3124,6940),
(3125,6940),
(3126,6940),
(3127,6940),
(3128,6940),
(3129,6940),
(3130,6940),
(3131,6940),
(3132,6940),
(3133,6940),
(3134,6940),
(3135,6940),
(3136,6940),
(3137,6940),
(3138,6940),
(3139,6940),
(3140,6940),
(3141,6940),
(3142,6940),
(3143,6940),
(3144,6940),
(3145,6941),
(3146,6941),
(3147,6941),
(3148,6941),
(3149,6941),
(3150,6941),
(3151,6941),
(3152,6941),
(3153,6941),
(3154,6941),
(3155,6941),
(3156,6941),
(3157,6941),
(3158,6941),
(3159,6941),
(3160,6941),
(3161,6941),
(3162,6941),
(3163,6941),
(3164,6941),
(3165,6941),
(3166,6941),
(3167,6941),
(3168,6941),
(3169,6941),
(3170,6941),
(3171,6941),
(3172,6941),
(3173,6941),
(3174,6941),
(3175,6941),
(3176,6942),
(3177,6942),
(3178,6942),
(3179,6942),
(3180,6942),
(3181,6942),
(3182,6942),
(3183,6942),
(3184,6942),
(3185,6942),
(3186,6942),
(3187,6942),
(3188,6942),
(3189,6942),
(3190,6942),
(3191,6942),
(3192,6942),
(3193,6942),
(3194,6942),
(3195,6942),
(3196,6942),
(3197,6942),
(3198,6942),
(3199,6942),
(3200,6942),
(3201,6942),
(3202,6942),
(3203,6942),
(3204,6942),
(3205,6942),
(3206,6942),
(3207,6943),
(3208,6943),
(3209,6943),
(3210,6943),
(3211,6943),
(3212,6943),
(3213,6943),
(3214,6943),
(3215,6943),
(3216,6943),
(3217,6943),
(3218,6943),
(3219,6943),
(3220,6943),
(3221,6943),
(3222,6943),
(3223,6943),
(3224,6943),
(3225,6943),
(3226,6943),
(3227,6943),
(3228,6943),
(3229,6943),
(3230,6943),
(3231,6943),
(3232,6943),
(3233,6943),
(3234,6943),
(3235,6943),
(3236,6943),
(3237,6943),
(3607,6944),
(3610,6944),
(3611,6946),
(3612,6946),
(3614,6946),
(3615,6946),
(3617,6946),
(3618,6946),
(3250,6947),
(3251,6947),
(3252,6947),
(3253,6947),
(3613,6948),
(3616,6948),
(3619,6948),
(3605,6949),
(3606,6949),
(3608,6949),
(3609,6949),
(3254,6950),
(3257,6950),
(3275,6950),
(3276,6950),
(3258,6953),
(3259,6953),
(3260,6953),
(3261,6954),
(3262,6954),
(3263,6954),
(3264,6954),
(3265,6954),
(3266,6954),
(3267,6954),
(3268,6954),
(3269,6955),
(3270,6955),
(3271,6955),
(3272,6955),
(3286,6958),
(3287,6958),
(3288,6958),
(3289,6958),
(3290,6967),
(3291,6967),
(3292,6967),
(3293,6967),
(3294,6968),
(3295,6968),
(3296,6968),
(3297,6968),
(3298,6971),
(3299,6971),
(3300,6971),
(3301,6971),
(3302,6972),
(3303,6972),
(3304,6972),
(3305,6972),
(3306,6975),
(3307,6975),
(3308,6975),
(3309,6975),
(3310,6976),
(3311,6976),
(3312,6976),
(3313,6976),
(3314,6977),
(3315,6977),
(3316,6977),
(3317,6977),
(3318,6980),
(3319,6980),
(3320,6980),
(3321,6980),
(3322,6980),
(3324,6983),
(3326,6983),
(3328,6983),
(3333,6983),
(3335,6983),
(3337,6983),
(3325,6984),
(3327,6984),
(3329,6984),
(3334,6984),
(3336,6984),
(3338,6984),
(3330,6985),
(3331,6985),
(3332,6985),
(3339,6985),
(3340,6985),
(3341,6985),
(3342,6988),
(3344,6988),
(3348,6988),
(3350,6988),
(3343,6989),
(3345,6989),
(3349,6989),
(3351,6989),
(3346,6990),
(3347,6990),
(3352,6990),
(3353,6990),
(3354,6993),
(3356,6993),
(3358,6993),
(3363,6993),
(3365,6993),
(3367,6993),
(3355,6994),
(3357,6994),
(3359,6994),
(3364,6994),
(3366,6994),
(3368,6994),
(3360,6995),
(3361,6995),
(3362,6995),
(3369,6995),
(3370,6995),
(3371,6995),
(3372,7001),
(3373,7001),
(3374,7001),
(3375,7001),
(3376,7001),
(3380,7002),
(3381,7002),
(3382,7002),
(3383,7005),
(3385,7008),
(3386,7008),
(3387,7008),
(3388,7008),
(3389,7008),
(3390,7008),
(3391,7008),
(3392,7008),
(3393,7009),
(3394,7009),
(3395,7009),
(3396,7009),
(3397,7009),
(3398,7009),
(3399,7009),
(3400,7009),
(3401,7010),
(3402,7010),
(3403,7010),
(3404,7010),
(3405,7010),
(3406,7010),
(3407,7010),
(3408,7010),
(3409,7011),
(3410,7011),
(3411,7011),
(3412,7011),
(3413,7011),
(3414,7011),
(3711,7012),
(3712,7012),
(3430,7018),
(3431,7018),
(3432,7018),
(3433,7018),
(3434,7018),
(3439,7018),
(3440,7018),
(3443,7018),
(3444,7018),
(3445,7018),
(3435,7021),
(3446,7022),
(3447,7022),
(3448,7022),
(3449,7022),
(3450,7022),
(3451,7022),
(3452,7022),
(3453,7022),
(3461,7025),
(3462,7031),
(3463,7031),
(3464,7031),
(3465,7031),
(3466,7031),
(3467,7031),
(3477,7034),
(3478,7034),
(3479,7034),
(3480,7034),
(3481,7036),
(3482,7036),
(3483,7036),
(3486,7038),
(3487,7038),
(3552,7038),
(3532,7040),
(3533,7040),
(3537,7043),
(3538,7043),
(3539,7043);


INSERT INTO dbo.gpAnnotationToAnnotationGrouping (idAnnotation, idAnnotationGrouping) VALUES 
(3540,7043),
(3541,7044),
(3542,7044),
(3543,7044),
(3544,7044),
(3545,7044),
(3546,7046),
(3547,7046),
(3548,7046),
(3549,7046),
(2839,7049),
(2841,7049),
(2843,7049),
(2847,7049),
(2850,7049),
(2855,7049),
(2856,7049),
(2857,7049),
(3629,7049),
(3631,7049),
(3632,7049),
(3633,7049),
(3554,7050),
(3555,7050),
(3556,7050),
(3557,7050),
(3558,7050),
(3559,7050),
(3560,7050),
(3561,7050),
(3625,7050),
(3626,7050),
(3627,7050),
(3628,7050),
(3581,7057),
(3582,7057),
(3583,7057),
(3584,7057),
(3585,7057),
(3586,7057),
(3587,7058),
(3588,7058),
(3589,7058),
(3572,7059),
(3573,7059),
(3574,7059),
(3575,7059),
(3576,7059),
(3577,7059),
(3578,7060),
(3579,7060),
(3580,7060),
(3566,7061),
(3567,7061),
(3568,7061),
(3569,7061),
(3570,7062),
(3571,7062),
(3981,7063),
(3644,7064),
(3645,7064),
(3648,7066),
(3649,7066),
(3650,7066),
(3651,7066),
(3653,7067),
(3802,7068),
(3803,7068),
(3804,7068),
(3660,7072),
(3661,7072),
(3662,7072),
(3663,7072),
(3664,7072),
(3665,7072),
(3666,7072),
(3677,7072),
(3678,7076),
(3679,7076),
(3680,7076),
(3681,7076),
(3682,7076),
(3683,7076),
(3684,7076),
(3685,7076),
(3686,7076),
(3695,7077),
(3696,7077),
(3697,7077),
(3698,7077),
(3699,7077),
(3700,7077),
(3701,7079),
(3702,7079),
(3703,7079),
(3704,7079),
(3705,7081),
(3706,7081),
(3707,7081),
(3708,7081),
(3709,7081),
(3710,7081),
(3958,7083),
(3959,7083),
(3960,7083),
(3961,7083),
(3962,7083),
(3963,7083),
(3964,7083),
(3965,7083),
(3972,7083),
(3973,7083),
(3974,7083),
(3975,7083),
(3976,7083),
(4417,7083),
(4418,7083),
(4419,7083),
(4420,7083),
(3001,7084),
(3002,7084),
(3725,7084),
(3738,7087),
(3726,7088),
(3727,7088),
(3728,7089),
(3729,7089),
(3730,7089),
(3731,7089),
(3732,7089),
(3733,7089),
(3734,7089),
(3735,7089),
(3736,7089),
(3737,7089),
(2240,7090),
(2241,7090),
(3744,7090),
(3745,7090),
(3747,7090),
(3749,7090),
(3750,7090),
(3639,7091),
(3751,7092),
(3755,7092),
(3756,7092),
(3757,7092),
(3758,7092),
(3759,7092),
(3768,7092),
(3769,7092),
(3771,7092),
(3772,7092),
(3773,7092),
(3764,7093),
(3765,7093),
(3766,7093),
(3767,7093),
(3776,7094),
(3777,7094),
(3778,7094),
(3754,7095),
(3774,7095),
(3775,7095),
(3780,7096),
(3781,7096),
(3782,7096),
(3783,7096),
(3789,7096),
(3790,7096),
(3791,7096),
(3784,7097),
(3785,7097),
(3786,7097),
(3787,7097),
(3788,7097),
(3792,7097),
(3793,7097),
(3805,7099),
(3806,7099),
(3807,7099),
(3808,7099),
(3809,7099),
(3810,7099),
(3811,7099),
(3812,7099),
(3813,7099),
(3814,7099),
(3815,7099),
(3816,7099),
(3817,7099),
(3818,7099),
(3819,7099),
(3820,7099),
(3821,7099),
(3822,7099),
(3823,7099),
(3824,7099),
(3825,7099),
(3826,7099),
(3827,7099),
(3828,7099),
(3829,7099),
(3830,7099),
(3831,7099),
(3832,7099),
(3833,7099),
(3834,7099),
(3835,7099),
(3836,7099),
(3837,7099),
(3838,7099),
(3839,7099),
(3840,7099),
(3841,7099),
(3842,7099),
(3843,7099),
(3844,7099),
(3845,7099),
(3846,7099),
(3847,7099),
(3848,7099),
(3849,7099),
(3850,7099),
(3851,7099),
(3852,7099),
(3853,7099),
(3854,7099),
(3855,7099),
(3856,7099),
(3857,7099),
(3858,7099),
(3859,7099),
(3860,7099),
(3861,7099),
(3862,7099),
(3863,7099),
(3864,7099),
(3865,7099),
(3866,7099),
(3867,7099),
(3868,7099),
(3869,7099),
(3870,7099),
(3871,7099),
(3872,7099),
(3873,7099),
(3874,7099),
(3875,7099),
(3876,7099),
(3877,7099),
(3878,7099),
(3879,7099),
(3880,7099),
(3881,7099),
(3882,7099),
(3883,7099),
(3884,7099),
(3885,7099),
(3886,7099),
(3887,7099),
(3888,7099),
(3889,7099),
(3890,7099),
(3891,7099),
(3892,7099),
(3893,7099),
(3894,7099),
(3895,7099),
(3896,7099),
(3897,7099),
(3898,7099),
(3899,7099),
(3905,7099),
(3906,7099),
(3907,7099),
(3908,7099),
(3909,7099),
(3910,7099),
(3911,7099),
(3912,7099),
(3913,7099),
(3914,7099),
(3915,7099),
(3916,7099),
(3917,7099),
(3918,7099),
(3919,7099),
(3920,7099),
(3921,7099),
(3922,7099),
(3923,7099),
(3924,7099),
(3925,7099),
(3926,7099),
(3927,7099),
(3928,7099),
(3929,7099),
(3930,7099),
(3806,7101),
(3807,7101),
(3808,7101),
(3809,7101),
(3810,7101),
(3811,7101),
(3812,7101),
(3813,7101),
(3814,7101),
(3815,7101),
(3816,7101),
(3817,7101),
(3818,7101),
(3819,7101),
(3820,7101),
(3821,7101),
(3822,7101),
(3823,7102),
(3824,7102),
(3825,7102),
(3826,7102),
(3827,7102),
(3828,7102),
(3829,7102),
(3830,7102),
(3831,7102),
(3832,7102),
(3833,7102),
(3834,7102),
(3835,7102),
(3836,7102),
(3837,7102),
(3838,7102),
(3839,7102),
(3840,7104),
(3841,7104),
(3842,7104),
(3843,7104),
(3844,7104),
(3845,7104),
(3846,7104),
(3847,7104),
(3848,7104),
(3849,7104),
(3850,7104),
(3851,7104),
(3852,7104),
(3853,7104),
(3854,7104),
(3855,7104),
(3856,7104),
(3857,7104),
(3858,7104),
(3859,7104),
(3860,7104),
(3861,7104),
(3862,7104),
(3863,7104),
(3864,7104),
(3865,7104),
(3866,7104),
(3867,7104),
(3868,7104),
(3869,7104),
(3870,7105),
(3871,7105),
(3872,7105),
(3873,7105),
(3874,7105),
(3875,7105),
(3876,7105),
(3877,7105),
(3878,7105),
(3879,7105),
(3880,7105),
(3881,7105),
(3882,7105),
(3883,7105),
(3884,7105),
(3885,7105),
(3886,7105),
(3887,7105),
(3888,7105),
(3889,7105),
(3890,7105),
(3891,7105),
(3892,7105),
(3893,7105),
(3894,7105),
(3895,7105),
(3896,7105),
(3897,7105),
(3898,7105),
(3899,7105),
(3901,7108),
(3902,7108),
(3903,7108),
(3905,7109),
(3906,7109),
(3907,7109),
(3908,7109),
(3909,7109),
(3910,7109),
(3911,7109),
(3912,7109),
(3913,7109),
(3914,7109),
(3915,7109),
(3916,7109),
(3917,7109),
(3918,7109),
(3919,7109),
(3920,7109),
(3921,7109),
(3922,7109),
(3923,7110),
(3924,7110),
(3925,7110),
(3926,7110),
(3927,7110),
(3928,7110),
(3929,7110),
(3930,7110),
(3941,7112),
(3942,7112),
(3943,7112),
(3940,7113),
(3944,7113),
(3945,7113),
(3982,7117),
(4387,7117),
(3984,7118),
(3985,7118),
(3986,7118),
(3987,7118),
(3988,7118),
(3989,7118),
(3990,7118),
(4002,7121),
(4003,7121),
(4004,7121),
(4005,7121),
(4006,7121),
(4007,7121),
(4008,7121),
(4009,7121),
(4082,7162),
(4084,7162),
(4083,7163),
(4085,7163),
(4086,7164),
(4087,7164),
(4099,7164),
(4114,7164),
(4088,7165),
(4089,7165),
(4109,7165),
(4110,7165),
(4119,7165),
(4090,7166),
(4091,7166),
(4092,7166),
(4103,7166),
(4104,7166),
(4120,7166),
(4121,7166),
(4093,7167),
(4094,7167),
(4095,7167),
(4111,7167),
(4112,7167),
(4096,7168),
(4097,7168),
(4098,7168),
(4113,7168),
(4100,7169),
(4105,7169),
(4106,7169),
(4115,7169),
(4116,7169),
(4102,7170),
(4107,7170),
(4108,7170),
(4117,7170),
(4118,7170),
(4101,7171),
(4122,7176),
(4123,7176),
(4124,7176),
(4125,7176),
(4126,7176),
(4127,7176),
(4128,7176),
(4129,7176),
(4130,7176),
(4131,7176),
(4132,7176),
(4133,7176),
(4134,7176),
(4135,7176),
(4136,7176),
(4137,7176),
(4138,7183),
(4139,7183),
(4140,7183),
(4141,7184),
(4142,7184),
(4143,7184),
(4144,7185),
(4145,7185),
(4146,7185),
(4147,7189),
(4148,7189),
(4196,7189),
(4197,7189),
(4198,7189),
(4199,7189),
(4200,7189),
(4201,7189),
(4202,7189),
(4203,7189),
(4204,7189),
(4205,7189),
(4149,7191),
(4150,7191),
(4354,7191),
(4151,7192),
(4152,7192),
(4153,7192),
(4154,7192),
(4157,7194),
(4158,7194),
(4166,7197),
(4167,7198),
(4168,7199),
(4169,7200),
(4171,7201),
(4172,7202),
(4173,7203),
(4174,7204),
(4175,7205),
(4176,7206),
(4271,7216),
(4301,7216),
(4302,7216),
(4303,7216),
(4304,7216),
(4305,7216),
(4306,7217),
(4307,7217),
(4308,7217),
(4309,7217),
(4310,7217),
(4311,7217),
(4312,7217),
(4207,7248),
(4208,7248),
(4209,7248),
(4214,7248),
(4215,7248),
(4216,7248),
(4217,7248),
(4222,7248),
(4223,7248),
(4224,7248),
(4225,7248),
(4226,7248),
(4210,7249),
(4211,7249),
(4320,7249),
(4317,7250),
(4318,7250),
(4322,7250),
(4400,7251),
(4401,7251),
(4402,7251),
(4286,7255),
(4287,7255),
(4288,7255),
(4289,7255),
(4290,7255),
(4291,7255),
(4292,7255),
(4293,7255),
(4294,7255),
(4295,7255),
(4296,7255),
(4297,7255),
(4298,7255),
(4299,7255),
(4300,7255),
(4275,7264),
(4276,7264),
(4284,7267),
(4285,7267),
(4282,7268),
(4283,7268),
(4278,7269),
(4279,7269),
(4280,7270),
(4281,7270),
(4315,7274),
(4316,7274),
(4321,7274),
(4377,7275),
(4378,7275),
(4379,7275),
(4380,7275),
(4383,7275),
(4384,7275),
(4345,7276),
(4346,7276),
(4347,7276),
(4348,7276),
(4349,7276),
(4350,7276),
(4351,7276),
(4352,7276),
(4353,7276),
(4228,7277),
(4342,7277),
(4343,7277),
(4344,7277),
(3741,7278),
(4388,7278),
(4399,7278),
(3794,7279),
(3795,7279),
(3796,7279),
(3797,7279),
(3931,7279),
(3798,7280),
(3799,7280),
(3800,7280),
(3801,7280),
(3932,7280),
(4389,7280),
(4390,7280),
(4391,7280),
(4392,7280),
(4393,7280),
(4319,7281),
(4338,7281),
(4339,7281),
(4340,7281),
(4341,7281),
(4355,7281),
(4394,7281),
(4395,7281),
(4396,7281),
(4397,7281),
(4398,7281),
(4357,7282),
(4358,7282),
(4359,7282),
(4360,7282),
(4403,7288),
(4404,7288),
(4405,7288),
(4406,7288),
(4408,7292),
(4409,7292),
(4415,7292),
(4264,7299),
(4265,7299),
(4266,7299),
(4267,7299),
(4268,7299),
(4269,7299),
(4258,7300),
(4259,7300),
(4260,7300),
(4261,7300),
(4262,7300),
(4263,7300),
(4410,7303),
(4412,7304),
(4413,7305),
(4414,7305),
(4422,7308),
(4423,7308),
(4428,7308),
(4424,7309),
(4425,7309),
(4426,7309),
(4427,7309);
ALTER TABLE dbo.gpAnnotationToAnnotationGrouping CHECK CONSTRAINT all;
SET IDENTITY_INSERT dbo.gpAnnotationToAnnotationGrouping OFF;



--
-- Table structure for table dbo.gpUnloadAnnotation --

IF  EXISTS (SELECT * FROM sys.objects WHERE object_id = OBJECT_ID(N'dbo.gpUnloadAnnotation') AND type in (N'U')) DROP TABLE dbo.gpUnloadAnnotation;


CREATE TABLE dbo.gpUnloadAnnotation (
  idUnloadAnnotation int  NOT NULL identity,
  typeName varchar(2000) NOT NULL,
  idUser int   NULL,
  idGenomeVersion int  NOT NULL,
CONSTRAINT [PK_UnloadAnnotation] PRIMARY KEY CLUSTERED 
(
	[idUnloadAnnotation] ASC
),
  
  
)  ;


--
-- Dumping data for table dbo.gpUnloadAnnotation --


ALTER TABLE dbo.gpUnloadAnnotation NOCHECK CONSTRAINT all;
SET IDENTITY_INSERT dbo.gpUnloadAnnotation ON;
INSERT INTO dbo.gpUnloadAnnotation (
idUnloadAnnotation,
typeName,
idUser,
idGenomeVersion
) VALUES (174,'GenBank',3,6304),(175,'Genes',3,6304),(471,'Ying/tony test',4,6300);
ALTER TABLE dbo.gpUnloadAnnotation CHECK CONSTRAINT all;
SET IDENTITY_INSERT dbo.gpUnloadAnnotation OFF;


