Index: src/java/net/sf/samtools/SAMTextReader.java
===================================================================
--- src/java/net/sf/samtools/SAMTextReader.java	(revision 973)
+++ src/java/net/sf/samtools/SAMTextReader.java	(working copy)
@@ -37,7 +37,7 @@
 /**
  * Internal class for reading SAM text files.
  */
-class SAMTextReader extends SAMFileReader.ReaderImplementation {
+public class SAMTextReader extends SAMFileReader.ReaderImplementation {
     // From SAM specification
     private static final int QNAME_COL = 0;
     private static final int FLAG_COL = 1;
@@ -69,6 +69,13 @@
      */
     private SAMFileReader mParentReader;
 
+    public SAMTextReader(BufferedLineReader mReader, SAMFileHeader mFileHeader, final SAMFileReader.ValidationStringency validationStringency) {
+	this.mReader = mReader;
+        this.validationStringency = validationStringency;
+	this.mFileHeader = mFileHeader;
+	advanceLine();
+    }
+	
     /**
      * Prepare to read a SAM text file.
      * @param stream Need not be buffered, as this class provides buffered reading.
