
-- ----------------------------------------------------------------------
-- SQL data bulk transfer script generated by the MySQL Migration Toolkit
-- ----------------------------------------------------------------------

-- Disable foreign key checks
SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0;


INSERT INTO `gnomex`.`AnalysisType`(`idAnalysisType`, `analysisType`, `isActive`, `idAppUser`)
VALUES (1, 'ChIP-Seq analysis', 'Y', NULL),
  (2, 'TiMAT2 Tiling Analysis', 'Y', 566),
  (3, 'RNA-Seq, Differential', 'Y', 566),
  (4, 'Viral sequence screen', 'Y', 273),
  (5, 'small RNA-Seq', 'Y', 566),
  (6, 'aCGH', 'Y', 566),
  (7, 'Digital Gene Expression (DiGE)', 'Y', 566),
  (8, 'Microarray Design', 'Y', 566),
  (9, 'Alignment', 'Y', 566),
  (10, 'Cap-Seq', 'Y', 566),
  (11, 'SNP/INDEL', 'Y', 566);

INSERT INTO `gnomex`.`AppUser`(`idAppUser`, `lastName`, `firstName`, `uNID`, `email`, `phone`, `department`, `institute`, `jobTitle`, `isActive`, `codeUserPermissionKind`, `userNameExternal`, `passwordExternal`)
VALUES (1, '', 'admin', null, null, null, NULL, NULL, NULL, 'Y', 'ADMIN', 'admin', '0DPiKuNIrrVmD8IUCuw1hQxNqZc=');

INSERT INTO `gnomex`.`ArrayCoordinate`(`idArrayCoordinate`, `name`, `x`, `y`, `idSlideDesign`)
VALUES  (2,'1_1',1,1,54),
 (3,'1_2',1,2,54),
 (16,'1',1,1,3),
 (17,'1',1,1,4),
 (18,'1',1,1,5),
 (19,'1',1,1,6),
 (21,'1',1,1,8),
 (24,'1',1,1,11),
 (25,'1',1,1,12),
 (27,'1',1,1,14),
 (31,'1',1,1,18),
 (38,'1',1,1,25),
 (44,'1',1,1,31),
 (48,'1',1,1,36),
 (72,'1',1,1,115),
 (74,'1',1,1,118),
 (75,'1_3',1,3,128),
 (76,'1_1',1,1,128),
 (77,'1_4',1,4,128),
 (78,'1_2',1,2,128),
 (79,'1_3',1,3,17),
 (80,'1_1',1,1,17),
 (81,'1_4',1,4,17),
 (82,'1_2',1,2,17),
 (85,'1_3',1,3,131),
 (86,'1_1',1,1,131),
 (87,'1_4',1,4,131),
 (88,'1_2',1,2,131),
 (97,'1_3',1,3,15),
 (98,'1_1',1,1,15),
 (99,'1_4',1,4,15),
 (100,'1_2',1,2,15),
 (102,'1_1',1,1,32),
 (103,'1_2',1,2,32),
 (104,'1_3',1,3,54),
 (105,'1_4',1,4,54),
 (106,'1_3',1,3,26),
 (107,'1_1',1,1,26),
 (108,'1_4',1,4,26),
 (109,'1_2',1,2,26),
 (110,'1_3',1,3,132),
 (111,'1_1',1,1,132),
 (112,'1_4',1,4,132),
 (113,'1_2',1,2,132),
 (114,'1_3',1,3,133),
 (115,'1_1',1,1,133),
 (116,'1_4',1,4,133),
 (117,'1_2',1,2,133),
 (122,'1_3',1,3,135),
 (123,'1_1',1,1,135),
 (124,'1_4',1,4,135),
 (125,'1_2',1,2,135),
 (126,'1_3',1,3,136),
 (127,'1_1',1,1,136),
 (128,'1_4',1,4,136),
 (129,'1_2',1,2,136),
 (130,'1_1',1,1,117),
 (131,'1_2',1,2,117),
 (132,'1_3',1,3,139),
 (133,'1_1',1,1,139),
 (134,'1_4',1,4,139),
 (135,'1_2',1,2,139),
 (140,'1_3',1,3,33),
 (141,'1_1',1,1,33),
 (142,'1_4',1,4,33),
 (143,'1_2',1,2,33),
 (157,'1_3',1,3,153),
 (158,'1_4',1,4,153),
 (159,'1_1',1,1,153),
 (160,'1_2',1,2,153),
 (177,'1_3',1,3,164),
 (178,'1_4',1,4,164),
 (179,'1_1',1,1,164),
 (180,'1_2',1,2,164),
 (182,'2_4',2,4,167),
 (183,'2_1',2,1,167),
 (184,'1_3',1,3,167),
 (185,'2_3',2,3,167),
 (186,'1_4',1,4,167),
 (187,'1_1',1,1,167),
 (188,'2_2',2,2,167),
 (189,'1_2',1,2,167),
 (190,'2_4',2,4,165),
 (191,'2_1',2,1,165),
 (192,'1_3',1,3,165),
 (193,'2_3',2,3,165),
 (194,'1_4',1,4,165),
 (195,'1_1',1,1,165),
 (196,'2_2',2,2,165),
 (197,'1_2',1,2,165),
 (198,'1_1',1,1,168),
 (199,'1_3',1,3,168),
 (200,'1_4',1,4,168),
 (201,'1_2',1,2,168),
 (208,'1_1',1,1,175),
 (209,'1_3',1,3,175),
 (210,'1_4',1,4,175),
 (211,'1_2',1,2,175),
 (212,'1_1',1,1,177),
 (213,'2_1',2,1,177),
 (214,'1_3',1,3,177),
 (215,'1_4',1,4,177),
 (216,'1_2',1,2,177),
 (217,'2_2',2,2,177),
 (218,'2_4',2,4,177),
 (219,'2_3',2,3,177);





INSERT INTO `gnomex`.`BillingChargeKind`(`codeBillingChargeKind`, `billingChargeKind`, `isActive`)
VALUES ('PRODUCT', 'Product', 'Y'),
  ('SERVICE', 'Service', 'Y');


INSERT INTO `gnomex`.`BillingPeriod`(`idBillingPeriod`, `billingPeriod`, `startDate`, `endDate`, `isActive`)
VALUES (1, 'Jan 2009', '2009-01-01 00:00:00', '2009-01-31 00:00:00', 'Y'),
  (2, 'Feb 2009', '2009-02-01 00:00:00', '2009-02-28 00:00:00', 'Y'),
  (3, 'Mar 2009', '2009-03-01 00:00:00', '2009-03-31 00:00:00', 'Y'),
  (4, 'Apr 2009', '2009-04-01 00:00:00', '2009-04-30 00:00:00', 'Y'),
  (5, 'May 2009', '2009-05-01 00:00:00', '2009-05-31 00:00:00', 'Y'),
  (6, 'Jun 2009', '2009-06-01 00:00:00', '2009-06-30 00:00:00', 'Y'),
  (7, 'Jul 2009', '2009-07-01 00:00:00', '2009-07-31 00:00:00', 'Y'),
  (8, 'Aug 2009', '2009-08-01 00:00:00', '2009-08-31 00:00:00', 'Y'),
  (9, 'Sep 2009', '2009-09-01 00:00:00', '2009-09-30 00:00:00', 'Y'),
  (10, 'Oct 2009', '2009-10-01 00:00:00', '2009-10-31 00:00:00', 'Y'),
  (11, 'Nov 2009', '2009-11-01 00:00:00', '2009-11-30 00:00:00', 'Y'),
  (12, 'Dec 2009', '2009-12-01 00:00:00', '2009-12-31 00:00:00', 'Y'),
  (13, 'Jan 2010', '2010-01-01 00:00:00', '2010-01-31 00:00:00', 'Y'),
  (14, 'Feb 2010', '2010-02-01 00:00:00', '2010-02-28 00:00:00', 'Y'),
  (15, 'Mar 2010', '2010-03-01 00:00:00', '2010-03-31 00:00:00', 'Y'),
  (16, 'Apr 2010', '2010-04-01 00:00:00', '2010-04-30 00:00:00', 'Y'),
  (17, 'May 2010', '2010-05-01 00:00:00', '2010-05-31 00:00:00', 'Y'),
  (18, 'Jun 2010', '2010-06-01 00:00:00', '2010-06-30 00:00:00', 'Y'),
  (19, 'Jul 2010', '2010-07-01 00:00:00', '2010-07-31 00:00:00', 'Y'),
  (20, 'Aug 2010', '2010-08-01 00:00:00', '2010-08-31 00:00:00', 'Y'),
  (21, 'Sep 2010', '2010-09-01 00:00:00', '2010-09-30 00:00:00', 'Y'),
  (22, 'Oct 2010', '2010-10-01 00:00:00', '2010-10-31 00:00:00', 'Y'),
  (23, 'Nov 2010', '2010-11-01 00:00:00', '2010-11-30 00:00:00', 'Y'),
  (24, 'Dec 2010', '2010-12-01 00:00:00', '2010-12-31 00:00:00', 'Y'),
  (25, 'Jan 2011', '2011-01-01 00:00:00', '2011-01-31 00:00:00', 'Y'),
  (26, 'Feb 2011', '2011-02-01 00:00:00', '2011-02-28 00:00:00', 'Y'),
  (27, 'Mar 2011', '2011-03-01 00:00:00', '2011-03-31 00:00:00', 'Y'),
  (28, 'Apr 2011', '2011-04-01 00:00:00', '2011-04-30 00:00:00', 'Y'),
  (29, 'May 2011', '2011-05-01 00:00:00', '2011-05-31 00:00:00', 'Y'),
  (30, 'Jun 2011', '2011-06-01 00:00:00', '2011-06-30 00:00:00', 'Y'),
  (31, 'Jul 2011', '2011-07-01 00:00:00', '2011-07-31 00:00:00', 'Y'),
  (32, 'Aug 2011', '2011-08-01 00:00:00', '2011-08-31 00:00:00', 'Y'),
  (33, 'Sep 2011', '2011-09-01 00:00:00', '2011-09-30 00:00:00', 'Y'),
  (34, 'Oct 2011', '2011-10-01 00:00:00', '2011-10-31 00:00:00', 'Y'),
  (35, 'Nov 2011', '2011-11-01 00:00:00', '2011-11-30 00:00:00', 'Y'),
  (36, 'Dec 2011', '2011-12-01 00:00:00', '2011-12-31 00:00:00', 'Y'),
  (37, 'Dec 2008', '2008-12-01 00:00:00', '2008-12-31 00:00:00', 'Y'),
  (38, 'Nov 2008', '2008-11-01 00:00:00', '2008-11-30 00:00:00', 'Y'),
  (39, 'Oct 2008', '2008-10-01 00:00:00', '2008-10-31 00:00:00', 'Y'),
  (40, 'Sep 2008', '2008-09-01 00:00:00', '2008-09-30 00:00:00', 'Y');



INSERT INTO `gnomex`.`BillingSlideProductClass`(`idBillingSlideProductClass`, `billingSlideProductClass`, `isActive`)
VALUES (1, 'Agilent 244K microarray', 'Y'),
  (2, 'Agilent 105K microarray', 'Y'),
  (3, 'Agilent 44K microarray', 'Y'),
  (4, 'Agilent 15K microarray', 'Y'),
  (5, 'Affymetrix 250K SNP microarray', 'Y'),
  (6, 'Affymetrix 5.0 SNP microarray', 'Y'),
  (7, 'Affymetrix Gene Expression microarray', 'Y'),
  (8, 'Affymetrix 6.0 SNP microarray', 'Y'),
  (9, 'Affymetrix Gene Expression (rat) microarray', 'Y'),
  (10, 'Affymetrix Gene Expression (zebrafish) microarray', 'Y'),
  (11, 'Agilent miRNA 15K microarray', 'Y'),
  (12, 'Agilent 1M microarray', 'Y'),
  (13, 'Agilent 180K microarray', 'Y');

INSERT INTO `gnomex`.`BillingSlideServiceClass`(`idBillingSlideServiceClass`, `billingSlideServiceClass`, `isActive`)
VALUES (1, 'Agilent Gene Expression 44K', 'Y'),
  (2, 'Aglient CGH 244K', 'Y'),
  (3, 'Agilent CGH 105K', 'Y'),
  (4, 'Agilent CGH 44K', 'Y'),
  (5, 'Agilent CGH 15K', 'Y'),
  (6, 'Agilent ChIP-on-chip 244K', 'Y'),
  (7, 'Agilent ChIP-on-chip 105K', 'Y'),
  (8, 'Agilent ChIP-on-chip 44K', 'Y'),
  (9, 'Agilent ChIP-on-chip 15K', 'Y'),
  (10, 'Agilent HybMap 244K', 'Y'),
  (11, 'Agilent HybMap 44K', 'Y'),
  (12, 'Affymetrix Gene Expression', 'Y'),
  (13, 'Affymetrix SNP 250K', 'Y'),
  (14, 'Affymetrix SNP 5.0/6.0', 'Y'),
  (15, 'Agilent Zebrafish Custom', 'Y'),
  (16, 'Agilent 15K miRNA', 'Y'),
  (17, 'Agilent Gene Expression 244K', 'Y'),
  (18, 'Agilent HybMap 1M', 'Y'),
  (19, 'Agilent CGH 180K', 'Y');

INSERT INTO `gnomex`.`BillingStatus`(`codeBillingStatus`, `billingStatus`, `isActive`)
VALUES ('APPROVED', 'Approved', 'Y'),
  ('APPROVEDEX', 'Approved (External)', 'Y'),
  ('COMPLETE', 'Completed', 'Y'),
  ('NEW', 'New', 'Y'),
  ('PENDING', 'Pending', 'Y');

INSERT INTO `gnomex`.`PriceSheet`(`idPriceSheet`, `name`, `description`, `isActive`)
VALUES 
  (1, 'Sample Quality', 'Sample Quality Price Sheet', 'Y'),
  (2, 'Agilent Microarray', 'Agilent Microarray Price Sheet', 'Y'),
  (3, 'Affymetrix Microarray', 'Affymetrix Microarray Price Sheet', 'Y'),
  (4, 'Illumina GAIIx Sequencing', 'Illumina GAIIx Sequencing Price Sheet', 'Y'),
  (5, 'Illumina HiSeq 2000 Sequencing', 'Illumina HiSeq 2000 Sequencing Price Sheet', 'Y');

INSERT INTO `gnomex`.`PriceSheetRequestCategory`(`idPriceSheet`, `codeRequestCategory`)
VALUES (1, 'QC'),
  (2, 'AGIL'),
  (2, 'AGIL1'),
  (3, 'AFFY'),
  (4, 'SOLEXA'),
  (5, 'HISEQ');

INSERT INTO `gnomex`.`PriceCategory`(`idPriceCategory`, `name`, `description`, `pluginClassName`, `codeBillingChargeKind`, `dictionaryClassNameFilter1`, `dictionaryClassNameFilter2`, `isActive`)
VALUES (1, 'Sample Quality', 'Sample Quality', 'hci.gnomex.billing.SampleQualityPlugin', 'SERVICE', 'hci.gnomex.model.Application', 'hci.gnomex.model.BioanalyzerChipType', 'Y'),
  (2, 'Microarray Labeling Reaction', 'Agilent Microarray Labeling Reaction', 'hci.gnomex.billing.LabelingReactionPlugin', 'SERVICE', 'hci.gnomex.model.Application', NULL, 'Y'),
  (3, 'Microarray Analysis', 'Agilent Microarray Analysis', 'hci.gnomex.billing.MicroarrayAnalysisPlugin', 'SERVICE', 'hci.gnomex.model.BillingSlideServiceClass', NULL, 'Y'),
  (4, 'Microarray', 'Agilent Microarray', 'hci.gnomex.billing.MicroarrayPlugin', 'PRODUCT', 'hci.gnomex.model.BillingSlideProductClass', NULL, 'Y'),
  (20, 'Microarray Labeling Reaction', 'Affymetrix Microarray Labeling Reaction', 'hci.gnomex.billing.LabelingReactionPlugin', 'SERVICE', 'hci.gnomex.model.BillingSlideServiceClass', NULL, 'Y'),
  (30, 'Microarray Analysis', 'Affymetrix Microarray Analysis', 'hci.gnomex.billing.MicroarrayAnalysisPlugin', 'SERVICE', 'hci.gnomex.model.BillingSlideServiceClass', NULL, 'Y'),
  (40, 'Microarray', 'Affymetrix Microarray', 'hci.gnomex.billing.MicroarrayPlugin', 'PRODUCT', 'hci.gnomex.model.BillingSlideProductClass', NULL, 'Y'),
  (5, 'Illumina Library Preparation', 'Illumina Library Preparation', 'hci.gnomex.billing.IlluminaLibPrepPlugin', 'SERVICE', 'hci.gnomex.model.Application', NULL, 'Y'),
  (7, 'Illumina GA Sequencing', 'Illumina GA Sequencing', 'hci.gnomex.billing.IlluminaSeqPlugin', 'SERVICE', 'hci.gnomex.model.SeqRunType', 'hci.gnomex.model.NumberSequencingCycles', 'Y'),
  (8, 'Sample Quality (Illumina Lib)', 'Sample Quality (Illumina Lib)', 'hci.gnomex.billing.IlluminaLibSampleQualityPlugin', 'SERVICE',  'hci.gnomex.model.Application', 'hci.gnomex.model.BioanalyzerChipType', 'Y'),
  (9, 'Miscellaneous', 'Agilent Miscellaneous', NULL, 'SERVICE', NULL, NULL, 'Y'),
  (10, 'Array Capture', 'Illumina Array Capture', NULL, 'SERVICE', NULL, NULL, 'Y'),
  (11, 'Illumina HiSeq Sequencing', 'Illumina HiSeq Sequencing', 'hci.gnomex.billing.IlluminaSeqPlugin', 'SERVICE', 'hci.gnomex.model.SeqRunType', 'hci.gnomex.model.NumberSequencingCycles', 'Y');

INSERT INTO `gnomex`.`PriceSheetPriceCategory`(`idPriceSheet`, `idPriceCategory`, `sortOrder`)
VALUES
  (1, 1, 1),
  (2, 1, 1),
  (2, 2, 2),
  (2, 3, 3),
  (2, 4, 4),
  (2, 9, 5),
  (3, 1, 1),
  (3, 20, 2),
  (3, 30, 3),
  (3, 40, 4),
  (4, 1, 1),
  (4, 5, 2),
  (4, 8, 3),
  (4, 7, 5),
  (4, 10, 6),
  (5, 1, 1),
  (5, 5, 2),
  (5, 8, 3),
  (5, 11, 5),
  (5, 10, 6);

INSERT INTO `gnomex`.`Price` (`idPrice`,`name`,`description`,`unitPrice`,`unitPriceExternal`,`idPriceCategory`,`isActive`) VALUES
 (1,'Bioanalyzer RNA Nano','sample quality','0.00',NULL,1,'Y'),
 (2,'Bioanalyzer RNA Pico','sample quality','0.00',NULL,1,'Y'),
 (3,'Bioanalyzer DNA 1000','sample quality','0.00',NULL,1,'Y'),
 (4,'Bioanalyzer Small RNA','sample quality','0.00',NULL,1,'Y'),
 (5,'Agilent Gene Expression','agilent labeling','0.00',NULL,2,'Y'),
 (6,'Agilent CGH','agilent labeling','0.00',NULL,2,'Y'),
 (7,'Agilent ChIP-on-chip','agilent labeling','0.00',NULL,2,'Y'),
 (8,'Agilent DNA Methylation','agilent labeling','0.00',NULL,2,'Y'),
 (9,'Agilent HybMap','agilent labeling','0.00',NULL,2,'Y'),
 (10,'Agilent SNP','agilent labeling','0.00',NULL,2,'Y'),
 (11,'Agilent miRNA','agilent labeling','0.00',NULL,2,'Y'),
 (12,'Agilent Exon','agilent labeling','0.00',NULL,2,'Y'),
 (13,'Agilent Gene Expression 44K','agilent microarray analysis','0.00',NULL,3,'Y'),
 (14,'Aglient CGH 244K','agilent microarray analysis','0.00',NULL,3,'Y'),
 (15,'Agilent CGH 105K','agilent microarray analysis','0.00',NULL,3,'Y'),
 (16,'Agilent CGH 44K','agilent microarray analysis','0.00',NULL,3,'Y'),
 (17,'Agilent CGH 15K','agilent microarray analysis','0.00',NULL,3,'Y'),
 (18,'Agilent ChIP-on-chip 244K','agilent microarray analysis','0.00',NULL,3,'Y'),
 (19,'Agilent ChIP-on-chip 105K','agilent microarray analysis','0.00',NULL,3,'Y'),
 (20,'Agilent ChIP-on-chip 44K','agilent microarray analysis','0.00',NULL,3,'Y'),
 (21,'Agilent ChIP-on-chip 15K','agilent microarray analysis','0.00',NULL,3,'Y'),
 (22,'Agilent HybMap 244K','agilent microarray analysis','0.00',NULL,3,'Y'),
 (23,'Agilent HybMap 44K','agilent microarray analysis','0.00',NULL,3,'Y'),
 (24,'Affymetrix Gene Expression','affy microarray analysis','0.00',NULL,30,'Y'),
 (25,'Affymetrix SNP','affy microarray analysis','0.00',NULL,30,'Y'),
 (26,'Agilent 244K Microarray','agilent microarray','0.00',NULL,4,'Y'),
 (27,'Agilent 105K Microarray','agilent microarray','0.00',NULL,4,'Y'),
 (28,'Agilent 44K Microarray','agilent microarray','0.00',NULL,4,'Y'),
 (29,'Agilent 15K Microarray','agilent microarray','0.00',NULL,4,'Y'),
 (30,'Affymetrix SNP 250K Microarray','affymetrix microarray','0.00',NULL,40,'Y'),
 (31,'Genomic DNA','illumina library prep','0.00',NULL,5,'Y'),
 (32,'ChIP-Seq','illumina library prep','0.00',NULL,5,'Y'),
 (33,'Small RNA','illumina library prep','0.00',NULL,5,'Y'),
 (34,'Directional mRNA seq','illumina library prep','0.00',NULL,5,'Y'),
 (35,'18 Cycle SR','illumina ga sequencing','0.00',NULL,7,'Y'),
 (36,'26 Cycle SR','illumina ga sequencing','0.00',NULL,7,'Y'),
 (37,'36 Cycle SR','illumina ga sequencing','0.00',NULL,7,'Y'),
 (38,'18 Cycle PE','illumina ga sequencing','0.00',NULL,7,'Y'),
 (39,'26 Cycle PE','illumina ga sequencing','0.00',NULL,7,'Y'),
 (40,'36 Cycle PE','illumina ga sequencing','0.00',NULL,7,'Y'),
 (41,'Affymetrix Gene Expression Labeling','affy labeling','0.00',NULL,20,'Y'),
 (42,'Affymetrix 250K SNP Labeling','affy labeling','0.00',NULL,20,'Y'),
 (43,'Qubit PicoGreen Assay','sample quality','0.00',NULL,1,'Y'),
 (44,'gDNA gel','sample quality','0.00',NULL,1,'Y'),
 (45,'Agilent HybMap 15K','agilent microarray analysis','0.00',NULL,3,'Y'),
 (46,'Slide Re-use Preparation','agilent miscellaneous','0.00',NULL,9,'Y'),
 (47,'Bioanalyzer RNA Nano','sample quality (illumina lib)','0.00',NULL,8,'Y'),
 (48,'Bioanalyzer RNA Pico','sample quality (illumina lib)','0.00',NULL,8,'Y'),
 (49,'Bioanalyzer DNA 1000','sample quality (illumina lib)','0.00',NULL,8,'Y'),
 (50,'Bioanalyzer Small RNA','sample quality (illumina lib)','0.00',NULL,8,'Y'),
 (51,'Qubit PicoGreen Assay','sample quality (illumina lib)','0.00',NULL,8,'Y'),
 (52,'gDNA gel','sample quality (illumina lib)','0.00',NULL,8,'Y'),
 (53,'Agilent Zebrafish Custom','agilent microarray analysis','0.00',NULL,3,'Y'),
 (54,'Agilent 15K miRNA','agilent microarray analysis','0.00',NULL,3,'Y'),
 (55,'Affymetrix 5.0/6.0 SNP Labeling','affy labeling','0.00',NULL,20,'Y'),
 (57,'Affymetrix SNP 5.0 Microarray','affymetrix microarray','0.00',NULL,40,'Y'),
 (58,'Affymetrix Gene Expression Microarray','affymetrix microarray','0.00',NULL,40,'Y'),
 (59,'Agilent cDNA synthesis/Agilent ChIP-on-chip Labeling','agilent labeling','0.00',NULL,2,'Y'),
 (62,'Targeted Genomic DNA Sequencing','illumina library prep','0.00',NULL,5,'Y'),
 (63,'mRNA-Seq','illumina library prep','0.00',NULL,5,'Y'),
 (64,'Agilent Gene Expression 244K','agilent microarray analysis','0.00',NULL,3,'Y'),
 (65,'Affymetrix SNP 6.0 Microarray','affymetrix microarray','0.00',NULL,40,'Y'),
 (66,'Affymetrix Gene Expression (rat) Microarray','affymetrix microarray','0.00',NULL,40,'Y'),
 (67,'Affymetrix Gene Expression (zebrafish) Microarray','affymetrix microarray','0.00',NULL,40,'Y'),
 (68,'Agilent miRNA 15K','agilent microarray','0.00',NULL,4,'Y'),
 (69,'Agilent 1M microarray','agilent microarray','0.00',NULL,4,'Y'),
 (70,'Agilent HybMap 1M','agilent microarray analysis','0.00',NULL,3,'Y'),
 (71,'Agilent HQ Gene Expression','agilent labeling','0.00',NULL,2,'Y'),
 (72,'Agilent 1X244K','illumina array capture','0.00',NULL,10,'Y'),
 (73,'Agilent 2x105K','illumina array capture','0.00',NULL,10,'Y'),
 (74,'Agilent 4x44K','illumina array capture','0.00',NULL,10,'Y'),
 (75,'Agilent 180K microarray','agilent microarray','0.00',NULL,4,'Y'),
 (76,'Agilent CGH 180K','agilent microarray analysis','0.00',NULL,3,'Y'),
 (77,'76 Cycle SR','illumina ga sequencing','0.00',NULL,7,'Y'),
 (78,'76 Cycle PE','illumina ga sequencing','0.00',NULL,7,'Y');

INSERT INTO `gnomex`.`PriceCriteria` (`idPriceCriteria`,`filter1`,`filter2`,`idPrice`) VALUES 
 (1,'BIOAN','RNANANO',1),
 (2,'BIOAN','RNAPICO',2),
 (3,'BIOAN','DNA1000',3),
 (4,'BIOAN','SMALLRNA',4),
 (5,'EXP',NULL,5),
 (6,'CGH',NULL,6),
 (7,'CHIP',NULL,7),
 (8,'METH',NULL,8),
 (9,'WTRANSCRP',NULL,9),
 (10,'SNP',NULL,10),
 (11,'MIRNA',NULL,11),
 (12,'EXON',NULL,12),
 (13,'1',NULL,13),
 (14,'2',NULL,14),
 (15,'3',NULL,15),
 (16,'4',NULL,16),
 (17,'5',NULL,17),
 (18,'6',NULL,18),
 (19,'7',NULL,19),
 (20,'8',NULL,20),
 (21,'9',NULL,21),
 (22,'10',NULL,22),
 (23,'11',NULL,23),
 (24,'12',NULL,24),
 (25,'13',NULL,25),
 (26,'1',NULL,26),
 (27,'2',NULL,27),
 (28,'3',NULL,28),
 (29,'4',NULL,29),
 (30,'5',NULL,30),
 (31,'DNASEQ',NULL,31),
 (32,'CHIPSEQ',NULL,32),
 (33,'SMRNASEQ',NULL,33),
 (34,'DMRNASEQ',NULL,34),
 (35,'3','1',35),
 (36,'3','2',36),
 (37,'3','3',37),
 (38,'4','1',38),
 (39,'4','2',39),
 (40,'4','3',40),
 (41,'12',NULL,41),
 (42,'13',NULL,42),
 (43,'QUBIT',NULL,43),
 (44,'DNAGEL',NULL,44),
 (45,'15',NULL,45),
 (46,NULL,NULL,46),
 (47,'BIOAN','RNANANO',47),
 (48,'BIOAN','RNAPICO',48),
 (49,'BIOAN','DNA1000',49),
 (50,'BIOAN','SMALLRNA',50),
 (51,'QUBIT',NULL,51),
 (52,'DNAGEL',NULL,52),
 (53,'15',NULL,53),
 (54,'16',NULL,54),
 (55,'14',NULL,55),
 (57,'6',NULL,57),
 (58,'7',NULL,58),
 (59,NULL,NULL,59),
 (62,'TDNASEQ',NULL,62),
 (63,'MRNASEQ',NULL,63),
 (64,'17',NULL,64),
 (65,'8',NULL,65),
 (66,'9',NULL,66),
 (67,'10',NULL,67),
 (68,'11',NULL,68),
 (69,'12',NULL,69),
 (70,'18',NULL,70),
 (71,NULL,NULL,71),
 (72,NULL,NULL,72),
 (73,NULL,NULL,73),
 (74,NULL,NULL,74),
 (75,'13',NULL,75),
 (76,'19',NULL,76),
 (77,'3','5',77),
 (78,'4','5',78),
 (79,'14',NULL,25),
 (80,'MONNUCSEQ',NULL,32),
 (81,'DNAMETHSEQ',NULL,31),
 (82,'TSCRPTSEQ',NULL,63);



INSERT INTO `gnomex`.`BioanalyzerChipType`(`codeBioanalyzerChipType`, `bioanalyzerChipType`, `concentrationRange`, `maxSampleBufferStrength`, `costPerSample`, `sampleWellsPerChip`, `isActive`, `codeConcentrationUnit`)
VALUES ('DNA1000', 'DNA 1000 Chip', '5 to 50 ng/µl', '250 mM KCl/NaCl', 10.00, 12, 'Y', 'ng/ul'),
  ('RNANANO', 'RNA Nano Chip', '25 to 500 ng/µl', '10mM TE', 7.00, 12, 'Y', 'ng/ul'),
  ('RNAPICO', 'RNA Pico Chip', '200 to 5000 pg/µl', '10 mM TE', 8.00, 11, 'Y', 'pg/ul'),
  ('SMALLRNA', 'Small RNA Chip', '1-20 ng/µl', NULL, NULL, 11, 'Y', 'pg/ul');

INSERT INTO `gnomex`.`ConcentrationUnit`(`codeConcentrationUnit`, `concentrationUnit`, `mageOntologyCode`, `mageOntologyDefinition`, `isActive`)
VALUES ('ng/ul', 'ng/ul', NULL, NULL, 'Y'),
  ('pg/ul', 'pg/ul', NULL, NULL, 'Y');

INSERT INTO `gnomex`.`ExperimentDesign`(`codeExperimentDesign`, `experimentDesign`, `mageOntologyCode`, `mageOntologyDefinition`, `isActive`, `idAppUser`)
VALUES ('BINDSITE', 'Binding Site Identification', 'binding_site_identification_design', NULL, 'Y', NULL),
  ('CGH', 'Comparative Genomic Hybridization', 'comparative_genome_hybridization_design', NULL, 'Y', NULL),
  ('CHROM', 'Chromatin Marks / Modifications', NULL, NULL, 'Y', NULL),
  ('DIFFEXPR', 'Differential Expression', NULL, NULL, 'Y', NULL),
  ('DIFFSPLICE', 'Differential Splicing', NULL, NULL, 'Y', NULL),
  ('EXPR', 'Expression', 'is_espressed_design', NULL, 'Y', NULL),
  ('OTHER', 'Other', NULL, NULL, 'Y', NULL),
  ('SEQENRICH', 'Sequence Enrichment', NULL, NULL, 'Y', NULL),
  ('TRANSCRIPT', 'Transcript Identification', 'transcript_identification_design', NULL, 'Y', NULL);


INSERT INTO `gnomex`.`ExperimentFactor`(`codeExperimentFactor`, `experimentFactor`, `mageOntologyCode`, `mageOntologyDefinition`, `isActive`, `idAppUser`)
VALUES ('AGE', 'Age / Developmental Stage', 'age', NULL, 'Y', NULL),
  ('CELLLINE', 'Cell Line / Strain', 'cell_line', NULL, 'Y', NULL),
  ('CELLTYPE', 'Cell Type', 'cell_type', NULL, 'Y', NULL),
  ('CLININFO', 'Clinical Information', 'clinical_information', NULL, 'Y', NULL),
  ('COMPOUND', 'Compound', NULL, NULL, 'Y', NULL),
  ('DISSTATE', 'Disease State / Stage / Tumor Grade', 'distease_state', NULL, 'Y', NULL),
  ('DOSE', 'Dose', 'dose', NULL, 'Y', NULL),
  ('GENMOD', 'Genetic Modification', NULL, NULL, 'Y', NULL),
  ('GENOTYPE', 'Genotype', 'genotype', NULL, 'Y', NULL),
  ('GROWTHCOND', 'Growth Conditions', NULL, NULL, 'Y', NULL),
  ('INDIV', 'Individual', 'individual', NULL, 'Y', NULL),
  ('ORGPART', 'Organ / Tissue', 'organism_part', NULL, 'Y', NULL),
  ('OTHER', 'Other', NULL, NULL, 'Y', NULL),
  ('SEX', 'Sex', 'sex', NULL, 'Y', NULL),
  ('TEMP', 'Temperature', NULL, NULL, 'Y', NULL),
  ('TIME', 'Time Course', 'sampling_time_point', NULL, 'Y', NULL);




INSERT INTO `gnomex`.`FundingAgency`(`idFundingAgency`, `fundingAgency`, `isPeerReviewedFunding`, `isActive`)
VALUES (4, 'NIH', 'Y', 'Y'),
  (5, 'Howard Hughes Foundation', 'Y', 'Y'),
  (6, 'American Cancer Society (ACS)\r\n', 'Y', 'Y'),
  (7, 'Centers for Disease Control and Prevention (CDC)\r\n', 'Y', 'Y'),
  (8, 'Food and Drug Administration (FDA)\r\n', 'Y', 'Y'),
  (9, 'National Science Foundation\r\n', 'Y', 'Y'),
  (10, 'US Army Department of Defense (DOD) (ovarian, breast, prostate programs)', 'Y', 'Y'),
  (11, 'Susan G. Komen Breast Cancer Foundation\r\n', 'Y', 'Y'),
  (12, 'Environmental Protection Agency', 'Y', 'Y'),
  (13, 'Leukemia and Lymphoma Society', 'Y', 'Y'),
  (14, 'Veterans Administration (central office, excluding local/regional/block grants)\r\n', 'Y', 'Y'),
  (15, 'Agency for Healthcare Research and Quality (AHRQ)\r\n', 'Y', 'Y'),
  (16, 'American Foundation for AIDS Research (AFAR)\r\n', 'Y', 'Y'),
  (17, 'American Institute for Cancer Research (AICR)\r\n', 'Y', 'Y'),
  (18, 'Cancer Research Foundation of America', 'Y', 'Y'),
  (19, 'Multiple Myeloma Research Foundation', 'Y', 'Y'),
  (20, 'National Institute for Occupational Safefy and Health (NIOSH)', 'Y', 'Y'),
  (21, 'University of California-Wide Breast Cancer Research Program', 'Y', 'Y'),
  (22, 'Other', 'N', 'Y');



/*!40000 ALTER TABLE `GenomeBuild` DISABLE KEYS */;
INSERT INTO `GenomeBuild` (`idGenomeBuild`,genomeBuildName, `das2Name`,`idOrganism`,`buildDate`,`coordURI`,`coordVersion`,`coordSource`,`coordTestRange`,`coordAuthority`,`ucscName`) VALUES 
 (400,'A_thaliana_Jan_2004','A_thaliana_Jan_2004', 400,'2004-01-01 00:00:00',NULL,NULL,NULL,NULL,NULL,NULL),
 (401,'TAIR8; A_thaliana_Mar_2008','A_thaliana_Mar_2008', 400,'2008-03-01 00:00:00',NULL,NULL,NULL,NULL,NULL,NULL),
 (501,'B_taurus_Oct_2007','B_taurus_Oct_2007',500,'2007-10-01 00:00:00',NULL,NULL,NULL,NULL,NULL,'bosTau4'),
 (1202,'C_elegans_Jan_2007','C_elegans_Jan_2007',1200,'2007-01-01 00:00:00','http://www.wormbase.org/genome/C_elegans/WS180/','180','Chromosome','','WS','ce4'),
 (1204,'ce6, C_elegans_May_2008, WS190','C_elegans_May_2008',1200,'2008-05-01 00:00:00',NULL,NULL,NULL,NULL,NULL,'ce6'),
 (1500,'C_familiaris_May_2005','C_familiaris_May_2005',1500,'2005-05-01 00:00:00',NULL,NULL,NULL,NULL,NULL,NULL),
 (1900,'D_melanogaster_Apr_2004','D_melanogaster_Apr_2004',1900,'2004-04-01 00:00:00',NULL,NULL,NULL,NULL,NULL,'dm2'),
 (1901,'D_melanogaster_Apr_2006','D_melanogaster_Apr_2006',1900,'2006-04-01 00:00:00',NULL,NULL,NULL,NULL,NULL,'dm3'),
 (2700,'D_rerio_Mar_2006','D_rerio_Mar_2006',2700,'2006-03-01 00:00:00',NULL,NULL,NULL,NULL,NULL,'danRer6'),
 (2701,'Zv7, D_rerio_Jul_2007','D_rerio_Jul_2007',2700,'2007-07-01 00:00:00','http://zfin.org/genome/D_rerio/Zv7/','Zv7','Chromosome',NULL,'ZFISH_7','danRer7'),
 (2702,'Zv8, D_rerio_Jun_2008','D_rerio_Jun_2008',2700,'2008-06-01 00:00:00','http://zfin.org/genome/D_rerio/Zv8/','Zv8','Chromosome',NULL,'ZFISH_8','danRer8'),
 (2900,'E_coli_Oct_2007','E_coli_Oct_2007',2900,'2007-10-01 00:00:00','','','','','',NULL),
 (3100,'G_gallus_May_2006','G_gallus_May_2006',3100,'2006-05-01 00:00:00',NULL,NULL,NULL,NULL,NULL,'galGal3'),
 (3404,'H_sapiens_May_2004','H_sapiens_May_2004',3400,'2004-05-01 00:00:00','http://www.ncbi.nlm.nih.gov/genome/H_sapiens/B35.1/','35','Chromosome',NULL,'NCBI','hg17'),
 (3405,'hg18; H_sapiens_Mar_2006; NCBI Build 36.1','H_sapiens_Mar_2006',3400,'2006-03-01 00:00:00','http://www.ncbi.nlm.nih.gov/genome/H_sapiens/B36.1/','36','Chromosome','','NCBI','hg18'),
 (3500,'M_mulatta_Jan_2006','M_mulatta_Jan_2006',3500,'2006-01-01 00:00:00',NULL,NULL,NULL,NULL,NULL,NULL),
 (3600,'M_truncatula_Aug_2007','M_truncatula_Aug_2007',3600,'2007-08-01 00:00:00',NULL,NULL,NULL,NULL,NULL,NULL),
 (3804,'mm8, M_musculus_Mar_2006, NCBI Build 36','M_musculus_Mar_2006',3800,'2006-03-01 00:00:00',NULL,NULL,NULL,NULL,NULL,'mm8'),
 (3806,'mm9, M_musculus_Jul_2007, NCBI Build 37','M_musculus_Jul_2007',3800,'2007-07-01 00:00:00',NULL,NULL,NULL,NULL,NULL,'mm9'),
 (3900,'M_abscessus_Mar_2008','M_abscessus_Mar_2008',3900,'2008-03-01 00:00:00',NULL,NULL,NULL,NULL,NULL,NULL),
 (4000,'M_smegmatis_Mar_2008','M_smegmatis_Mar_2008',4000,'2008-03-01 00:00:00','','','','','',NULL),
 (4100,'M_tuberculosis_Sep_2008','M_tuberculosis_Sep_2008',4100,'2008-09-01 00:00:00','','','','','',NULL),
 (4302,'O_sativa_Jun_2009','O_sativa_Jun_2009',4300,'2009-06-01 00:00:00',NULL,NULL,NULL,NULL,NULL,NULL),
 (4800,'P_falciparum_Jul_2007','P_falciparum_Jul_2007',4800,'2007-07-01 00:00:00',NULL,NULL,NULL,NULL,NULL,NULL),
 (5000,'P_trichocarpa_Jun_2004','P_trichocarpa_Jun_2004',5000,'2004-06-01 00:00:00',NULL,NULL,NULL,NULL,NULL,NULL),
 (5202,'R_norvegicus_Nov_2004','R_norvegicus_Nov_2004',5200,'2004-11-01 00:00:00',NULL,NULL,NULL,NULL,NULL,'rn4'),
 (5306,'S_cerevisiae_Apr_2008','S_cerevisiae_Apr_2008',5300,'2008-04-01 00:00:00',NULL,NULL,NULL,NULL,NULL,'sacCer2'),
 (5307,'S_cerevisiae_20070113','S_cerevisiae_Jan_2007',5300,'2007-01-01 00:00:00',NULL,NULL,NULL,NULL,NULL,''),
 (5400,'S_pombe_Sep_2004','S_pombe_Sep_2004',5400,'2004-09-01 00:00:00',NULL,NULL,NULL,NULL,NULL,NULL),
 (5401,'S_pombe_Apr_2007','S_pombe_Apr_2007',5400,'2007-04-01 00:00:00','http://www.sanger.ac.uk/Projects/S_pombe/Apr_2007','Apr_2007','Chromosome','','Sanger',NULL),
 (5500,'S_glossinidius_Jan_2006','S_glossinidius_Jan_2006',5500,'2006-01-01 00:00:00','ftp://ftp.ncbi.nih.gov/genomes/Bacteria/Sodalis_glossinidius_morsitans/Jan_2006','Jan_2006','Chromosome',NULL,'NCBI',NULL),
 (6000,'T_nigroviridis_Feb_2004','T_nigroviridis_Feb_2004',6000,'2004-02-01 00:00:00','','','','','',NULL),
 (6100,'V_vinifera_Apr_2007','V_vinifera_Apr_2007',6100,'2007-04-01 00:00:00',NULL,NULL,NULL,NULL,NULL,NULL),
 (6201,'X_tropicalis_Aug_2005','X_tropicalis_Aug_2005',6200,'2005-08-01 00:00:00','','','','','','xenTro2'),
 (6202,'hg19; H_sapiens_Feb_2009; GRCh37','H_sapiens_Feb_2009',3400,'2009-02-01 00:00:00','','','','','','hg19');



INSERT INTO `gnomex`.`Label`(`idLabel`, `label`, `isActive`)
VALUES (1, 'Cy3', 'Y'),
  (2, 'Cy5', 'Y');


INSERT INTO `gnomex`.`LabelingReactionSize`(`codeLabelingReactionSize`, `labelingReactionSize`, `isActive`, `sortOrder`)
VALUES ('STD', 'Standard', 'Y', 1),
  ('LRG', 'Large', 'Y', 2);

INSERT INTO `gnomex`.`ApplicationTheme`(`idApplicationTheme`, `applicationTheme`, `isActive`)
VALUES (1, 'DNA Sequencing', 'Y'),
  (2, 'RNA Sequencing', 'Y'),
  (3, 'Microarray', 'Y'),
  (4, 'Sample Quality', 'Y');

INSERT INTO `gnomex`.`Application`(`codeApplication`, `application`, `isActive`, `idApplicationTheme`, `sortOrder`)
VALUES ('CGH', 'CGH', 'Y', 3, NULL),
  ('CHIP', 'ChiP-on-chip', 'Y', 3, NULL),
  ('EXON', 'Exon', 'Y', 3, NULL),
  ('EXP', 'Gene Expression', 'Y', 3, NULL),
  ('METH', 'DNA Methylation', 'Y', 3, NULL),
  ('MIRNA', 'miRNA', 'Y', 3, NULL),
  ('SNP', 'SNP', 'Y', 3, NULL),
  ('WTRANSCRP', 'HybMap', 'Y', 3, NULL),
  ('CHIPSEQ', 'ChIP-Seq', 'Y', 1, 4),
  ('DMRNASEQ', 'Directional mRNA Sequencing', 'Y', 2, 2),
  ('DNASEQ', 'Genomic DNA Sequencing', 'Y', 1, 1),
  ('TDNASEQ', 'Targeted Genomic Resequencing', 'Y', 1, 3),
  ('MRNASEQ', 'mRNA Sequencing', 'Y', 2, 1),
  ('SMRNASEQ', 'Small mRNA Sequencing', 'Y', 2, 4),
  ('BIOAN', 'Bioanalyzer', 'Y', 4, NULL),
  ('QUBIT', 'Qubit picoGreen', 'Y', 4, NULL),
  ('DNAGEL', 'DNA Gel', 'Y', 4, NULL),
  ('DNAMETHSEQ', 'DNA Methylation Analysis', 'Y', 1, 2),
  ('MONNUCSEQ', 'Mononucleosome sequencing', 'Y', 1, 5),
  ('TSCRPTSEQ', '3.	Transcriptome sequencing (with DSN treatment of library)', 'Y', 2, 3);


INSERT INTO `gnomex`.`NumberSequencingCycles`(`idNumberSequencingCycles`, `numberSequencingCycles`, `isActive`)
VALUES (1, 18, 'Y'),
  (2, 26, 'N'),
  (3, 36, 'Y'),
  (4, 50, 'Y'),
  (5, 76, 'N'),
  (6, 101, 'Y');

INSERT INTO `gnomex`.`NumberSequencingCyclesAllowed`(`idNumberSequencingCyclesAllowed`, `idNumberSequencingCycles`, `codeRequestCategory`, idSeqRunType, name)
VALUES 
  (1, 1, 'SOLEXA', 3, '18 cycle single-end reads'),
  (2, 3, 'SOLEXA', 3, '36 cycle single-end reads'),
  (3, 4, 'HISEQ', 3, '50 cycle single-end reads'),
  (4, 6, 'HISEQ', 3,  '101 cycle single-end reads'),
  (5, 1, 'SOLEXA', 4, '18 cycle paired-end reads'),
  (6, 3, 'SOLEXA', 4, '36 cycle paired-end reads'),
  (7, 4, 'HISEQ', 4, '50 cycle paired-end reads'),
  (8, 6, 'HISEQ', 4, '101 cycle paired-end reads');

INSERT INTO `gnomex`.`OligoBarcodeScheme`(`idOligoBarcodeScheme`, `oligoBarcodeScheme`, `description`, `isActive`)
VALUES (1, 'GAIIx 4 sequence tag scheme', 'GAII 4 sequence tag scheme, allows for 2-4 samples per flowcell channel', 'Y'),
       (2, 'HiSeq 12 sequence tag scheme', 'HiSeq 12 sequence tag scheme, allows for 2-12 samples per flowcell channel', 'Y');

INSERT INTO `gnomex`.`OligoBarcodeSchemeAllowed`(`idOligoBarcodeSchemeAllowed`, `idOligoBarcodeScheme`, `codeRequestCategory`)
VALUES (1, 1, 'SOLEXA'),
       (2, 2, 'HISEQ');

INSERT INTO `gnomex`.`OligoBarcode`(`idOligoBarcode`, `name`, `barcodeSequence`, `idOligoBarcodeScheme`, `isActive`, `sortOrder`)
VALUES (1,  'GAII Tag 1', 'AAT', 1, 'Y', 1),
       (2,  'GAII Tag 2', 'CCT', 1, 'Y', 2),
       (3,  'GAII Tag 3', 'GGT', 1, 'Y', 3),
       (4,  'GAII Tag 4', 'TTT', 1, 'Y', 4),
	   (5,  'HiSeq Tag 1', 'CGTGAT', 2, 'Y', 1),
	   (6,	'HiSeq Tag 2', 'ACATCG', 2, 'Y', 2),
	   (7,	'HiSeq Tag 3', 'GCCTAA', 2, 'Y', 3),
	   (8,	'HiSeq Tag 4', 'TGGTCA', 2, 'Y', 4),
	   (9,	'HiSeq Tag 5', 'CACTGT', 2, 'Y', 5),
	   (10,	'HiSeq Tag 6', 'ATTGGC', 2, 'Y', 6),
	   (11,	'HiSeq Tag 7', 'GATCTG', 2, 'Y', 7),
	   (12,	'HiSeq Tag 8', 'TCAAGT', 2, 'Y', 8),
	   (13,	'HiSeq Tag 9', 'CTGATC', 2, 'Y', 9),
	   (14,	'HiSeq Tag 10', 'AAGCTA', 2, 'Y', 10),
	   (15,	'HiSeq Tag 11', 'GTAGCC', 2, 'Y', 11),
	   (16,	'HiSeq Tag 12', 'TACAAG', 2, 'Y', 12);

  
/*!40000 ALTER TABLE `Organism` DISABLE KEYS */;
INSERT INTO gnomex.`Organism` (`idOrganism`,`organism`,`sortOrder`,`binomialName`,`NCBITaxID`,`idAppUser`,`das2Name`) VALUES 
 (100,'Lizard',NULL,'Anolis carolinensis',NULL,NULL,'A_carolinensis'),
 (200,'A.gambiae',NULL,'Anopheles gambiae',NULL,NULL,'A_gambiae'),
 (300,'A_mellifera',NULL,'Apis mellifera','',NULL,'A_mellifera'),
 (400,'A. thaliana',NULL,'Arabidopsis thaliana',NULL,NULL,'A_thaliana'),
 (500,'Cow',NULL,'Bos taurus',NULL,NULL,'B_taurus'),
 (600,'Lancet',NULL,'Branchiostoma floridae',NULL,NULL,'B_floridae'),
 (700,'Marmoset',NULL,'Callithrix jacchus',NULL,NULL,'C_jacchus'),
 (800,'Guinea Pig',NULL,'Cavia porcellus',NULL,NULL,'C_porcellus'),
 (900,'C. intestinalis',NULL,'Ciona intestinalis',NULL,NULL,'C_intestinalis'),
 (1000,'C. brenneri',NULL,'Caenorhabditis brenneri',NULL,NULL,'C_brenneri'),
 (1100,'C. briggsae',NULL,'Caenorhabditis briggsae',NULL,NULL,'C_briggsae'),
 (1200,'C. elegans',NULL,'Caenorhabditis elegans','6239',NULL,'C_elegans'),
 (1300,'C. japonica',NULL,'Caenorhabditis japonica',NULL,NULL,'C_japonica'),
 (1400,'C. remanei',NULL,'Caenorhabditis remanei',NULL,NULL,'C_remanei'),
 (1500,'Dog',NULL,'Canis lupus familiaris',NULL,NULL,'C_familiaris'),
 (1600,'D. ananassae',NULL,'Drosophila ananassae',NULL,NULL,'D_ananassae'),
 (1700,'D. erecta',NULL,'Drosophila erecta',NULL,NULL,'D_erecta'),
 (1800,'D. grimshawi',NULL,'Drosophila grimshawi',NULL,NULL,'D_grimshawi'),
 (1900,'Drosophila melanogaster',NULL,'Drosophila melanogaster',NULL,NULL,'D_melanogaster'),
 (2000,'Drosophila mojavensis',NULL,'Drosophila mojavensis',NULL,NULL,'D_mojavensis'),
 (2100,'Drosophila persimilis',NULL,'Drosophila persimilis',NULL,NULL,'D_persimilis'),
 (2200, 'Drosophila pseudoobscura',NULL,'Drosophila pseudoobscura',NULL,NULL,'D_pseudoobscura'),
 (2300,'Drosophila sechellia',NULL,'Drosophila sechellia',NULL,NULL,'D_sechellia'),
 (2400,'Drosophila simulans',NULL,'Drosophila simulans',NULL,NULL,'D_simulans'),
 (2500,'Drosophila virilis',NULL,'Drosophila virilis',NULL,NULL,'D_virilis'),
 (2600,'Drosophila yakuba',NULL,'Drosophila yakuba',NULL,NULL,'D_yakuba'),
 (2700,'Zebrafish',NULL,'Danio rerio','7955',NULL,'D_rerio'),
 (2800,'Horse',NULL,'Equus caballus',NULL,NULL,'E_caballus'),
 (2900,'E. coli',NULL,'Escherichia coli',NULL,NULL,'E_coli'),
 (3000,'Cat',NULL,'Felis catus',NULL,NULL,'F_catus'),
 (3100,'Chicken',NULL,'Gallus gallus',NULL,NULL,'G_gallus'),
 (3200,'Stickleback G_aculeatus',NULL,'Gasterosteus aculeatus','',NULL,'G_aculeatus'),
 (3300,'Soybean',NULL,'Glycine max',NULL,NULL,'G_max'),
 (3400,'Human',NULL,'Homo sapiens','9606',NULL,'H_sapiens'),
 (3500,'Rhesus',NULL,'Macaca mulatta',NULL,NULL,'M_mulatta'),
 (3600,'Barrel Medic',NULL,'Medicago truncatula',NULL,NULL,'M_truncatula'),
 (3700,'Opossum',NULL,'Monodelphis domestica',NULL,NULL,'M_domestica'),
 (3800,'Mouse',NULL,'Mus musculus','10090',NULL,'M_musculus'),
 (3900,'M. abcessus',NULL,'Mycobacterium abscessus',NULL,NULL,'M_abscessus'),
 (4000,'M. smegmatis',NULL,'Mycobacterium smegmatis',NULL,NULL,'M_smegmatis'),
 (4100,'M. tuberculosis',NULL,'Mycobacterium tuberculosis',NULL,NULL,'M_tuberculosis'),
 (4200,'Platypus',NULL,'Ornithorhynchus anatinus',NULL,NULL,'O_anatinus'),
 (4300,'Rice',NULL,'Oryza sativa',NULL,NULL,'O_sativa'),
 (4400,'Medaka',NULL,'Oryzias latipes',NULL,NULL,'O_latipes'),
 (4500,'O. lucimarinus',NULL,'Ostreococcus lucimarinus',NULL,NULL,'O_lucimarinus_Apr_2007'),
 (4600,'Chimp',NULL,'Pan troglodytes',NULL,NULL,'P_troglodytes'),
 (4700,'Lamprey',NULL,'Petromyzon marinus',NULL,NULL,'P_marinus'),
 (4800,'P. falciparum',NULL,'Plasmodium falciparum',NULL,NULL,'P_falciparum'),
 (4900,'Orangutan',NULL,'Pongo pygmaeus abelii',NULL,NULL,'P_abelii'),
 (5000,'Black Cottonwood',NULL,'Populus trichocarpa',NULL,NULL,'P_trichocarpa'),
 (5100,'P. pacificus',NULL,'Pristionchus pacificus',NULL,NULL,'P_pacificus'),
 (5200,'Rat',NULL,'Rattus norvegicus',NULL,NULL,'R_norvegicus'),
 (5300,'Yeast',NULL,'Saccharomyces cerevisiae',NULL,NULL,'S_cerevisiae'),
 (5400,'Fission Yeast',NULL,'Schizosaccharomyces pombe','4896',NULL,'S_pombe'),
 (5500,'S. glossinidius',NULL,'Sodalis glossinidius','343509',NULL,'S_glossinidius'),
 (5600,'Sorghum',NULL,'Sorghum bicolor','',NULL,'S_bicolor'),
 (5700,'Purple Sea Urchin',NULL,'Strongylocentrotus purpuratus',NULL,NULL,'S_purpuratus'),
 (5800,'Zebra Finch',NULL,'Taeniopygia guttata','',NULL,'T_guttata'),
 (5900,'Fugu',NULL,'Takifugu rubripes','',NULL,'T_rubripes'),
 (6000,'Tetraodon',NULL,'Tetraodon nigroviridis','',NULL,'T_nigroviridis'),
 (6100,'Common Grape Vine',NULL,'Vitis vinifera',NULL,NULL,'V_vinifera'),
 (6200,'Pipid Frog',NULL,'Xenopus tropicalis','',NULL,'X_tropicalis');
/*!40000 ALTER TABLE `Organism` ENABLE KEYS */;
  

INSERT INTO `gnomex`.`PropertyDictionary` (`idPropertyDictionary`,`propertyName`,`propertyValue`,`propertyDescription`, `forServerOnly`) VALUES
 (1,'workauth_instructions','If billing account is inactive, please download the work authorization form below.<br> Forward the completed form to the [ENTER FACILITY NAME HERE] ([ENTER ROOM AND PHONE NUMBER HERE])','The instructions about work authorization that appear on the Experiment Submit Request page', 'N'),
 (2,'qc_instructions','Terms:  Bioanalyzer chips will be run when sufficient samples have been collected to load all wells of a chip.  Expedited service can be obtained if the researcher is willing to accept charges for any remaining empty wells on a chip.  Call [ENTER CORE FACILITY INFO HERE] for details.','The instructions regarding sample quality experiments.  These instructions about on Sample Tab of Experiment Submit Request window.', 'N'),
 (3,'workauth1_url','doc/blank_microarray_core_work_authorization_form.pdf','The URL to the work authorization document (pdf).', 'N'),
 (4,'workauth1_name','Work authorization (internal customers)','The name of the link', 'N'),
 (5,'contact_name_software_bugs','Bionformatics Core','Software org or contact', 'N'),
 (6,'contact_email_software_bugs','first.last@location.edu','Email address for reporting software bugs', 'N'),
 (7,'contact_name_core_facility','','Core facility\'s contact name', 'N'),
 (8,'contact_email_core_facility','first.last@somwhere.edu','Core facility\'s email adress', 'N'),
 (9,'help_url','http://www.hci.utah.edu/internal/CSG/PublicDocs/SourceDocs/HCIApps/GNomEx/GNomEx.htm','The URL to the GNomEx help documentation', 'N'),
 (10,'pricing_url','','The URL containing pricing info.', 'N'),
 (18,'core_facility_name','Microarray and Genomic Analysis Core Facility','The name of the core facility', 'N'),
 (19,'university_user_authentication','N','Always set to \'N\' for open source GNomEx.  Indicates that authentication is performed against local user name and passwords, stored in GNomEx database.', 'N'),
 (20,'generate_general_ledger_interface','N','Always set to \'N\' for open source GNomEx.  Indicates that GL interface file cannot be generated.', 'N'),
 (21,'contact_email_software_tester','softwaredeveloper.name@somewhere.edu','Email address of the software tester.  Test will receive auto emails (on test server) when this email is entered as user\'s email or group contact\'s email.', 'N'),
 (22,'contact_phone_core_facility','','The phone number of the core facility.', 'N'),
 (23,'flowcell_directory_flag','FC','Always set to \'FC\'.  Indicates the prefix of the flowcell directory name.', 'N'),
 (24,'qc_directory','bioanalysis','The name of the QC folder under the experiment folder.  Place all sample quality (bioanalysis, spectrophotometer reports, etc.) in this directory', 'N'),
 (25,'production_server','','The DNS name or IP address of the production app server, example:  myprodserver.hci.utah.edu', 'N'),
 (26,'lucene_index_directory','/home/gnomex/luceneIndex','The file directory for storing lucene index files.', 'Y'),
 (27,'lucene_experiment_index_directory','/home/gnomex/luceneIndex/Experiment','The file directory for storing lucene index files on experiment data.', 'Y'),
 (28,'lucene_analysis_index_directory','/home/gnomex/luceneIndex/Analysis','The file directory for storing lucene index files on analysis data.', 'Y'),
 (29,'lucene_protocol_index_directory','/home/gnomex/luceneIndex/Protocol','The file directory for storing lucene index files on protocol data.', 'Y'),
 (30,'experiment_read_directory','/home/gnomex/experimentData/','The file directory where experiment results are stored. NOTE: must end in /', 'Y'),
 (31,'experiment_write_directory','/home/gnomex/experimentData/','The file directory where experiment results are stored. NOTE: must end in /', 'Y'),
 (32,'analysis_read_directory','/home/gnomex/analysisData/','The file directory where analysis files are stored. NOTE: must end in /', 'Y'),
 (33,'analysis_write_directory','/home/gnomex/analysisData/','The file directory where analysis files are stored. NOTE: must end in /', 'Y'),
 (34,'flowcell_directory','/home/gnomex/flowCellData/','The file directory where sequencing pipeline results for a flowcell are stored. NOTE: must end in /', 'Y'),
 (35,'flowcell_test_directory','/home/gnomex/flowCellData/','The test file directory where sequencing pipeline results for a flowcell are stored. NOTE: must end in /', 'Y'),
 (36,'billing_core_facility_business_unit','','Always blank for open source GNomEx', 'Y'),
 (37,'billing_core_facility_account','','Always blank for open source GNomEx', 'Y'),
 (38,'billing_core_facility_fund','','Always blank for open source GNomEx', 'Y'),
 (39,'billing_core_facility_org','','Always blank for open source GNomEx', 'Y'),
 (40,'billing_core_facility_activity','','Always blank for open source GNomEx', 'Y'),
 (41,'temp_directory','/temp','The temp directory where temp files are created', 'Y'),
 (42,'illumina_barcoding_note','','Illumina sample barcoding instructions', 'N'),
 (43,'workauth2_url','doc/blank_microarray_core_work_authorization_form.pdf','The URL to an alternalte work authorization document.', 'N'),
 (44,'workauth2_name','Work authorization (internal customers)','The name of the link', 'N'),
 (45,'contact_email_core_facility_workauth','first.last@somwhere.edu','Core facility\'s email adresses to for notification when work authorization submitted', 'N'),
 (46, 'experiment_submission_default_mode', 'INTERNAL', 'INTERNAL if new experiment request to core facility, EXTERNAL if external experiment submission', 'N'), 
 (47, 'bst_linkage_supported', 'N', 'Set to N.  Only HCI installation supports linkage from GNomEx to Biospecimen tracking system.', 'Y'), 
 (48, 'fdt_supported', 'N', 'Flag indicating if FDT is supported.', 'N'),
 (49, 'http_port', '80', 'The port number that non-secure (http) runs on.', 'Y'),
 (50, 'fdt_directory', '', 'The fdt staging directory (file path) that is accessible from the fdt server.', 'N'),
 (51, 'fdt_directory_gnomex', '', 'The fdt staging directory (file path) that is accessible from gnomex application server.', 'N'),
 (52, 'fdt_client_codebase', '', 'The URL that the fdtClient.jar is served from.', 'N'),
 (53, 'fdt_server_name', '', 'The server name that the fdt server is running on', 'N'),
 (54, 'fdt_file_daemon_task_dir', '', 'Location for fdtfilemonitor task files (these contain info on source/target folders for moving files after completion of uploads).', 'N'),
 (55, 'fdt_user', 'fdt', 'User account that fdt server runs under.', 'N'),
 (56, 'fdt_group', 'fdtsecurity', 'Unix group for fdt user account.', 'N'),
 (57, 'usage_user_visibility', 'none', 'Permission level for non-admin gnomex users to view lab usage data. [none, masked, full]', 'N'),
 (58, 'external_data_sharing_site', 'N', 'Indicates if this gnomex installation is for external data sharting site, which limits options on welcome screen', 'N'),
 (59, 'site_title', '', 'The server name that appears next to the GNomEx title on the right top corner of the app', 'N'),
 (60, 'site_welcome_message', '', 'Welcome message that appears on the GNomEx home/welcome page', 'N'),
 (61, 'show_usage_on_startup', 'N', 'Y/N - Show usage charts on startup rather than home (welcome) view', 'N'),
 (62, 'billing_core_facility_po_account', '', 'Used in GL interface (internal use only)', 'Y'),
 (63, 'billing_po_account', '', 'Used in GL interface (internal use only)', 'Y'),
 (64, 'bst_linkage_gnomex_url', '', 'The url used in BST to launch gnomex when samples are inked (internal use only)', 'Y'),
 (65, 'contact_email_core_facility_workauth_reminder', 'first.last@somwhere.edu', 'The email addresses to send email about pending work authorizations', 'Y'),
 (66, 'dataset_privacy_expiration', '0', 'When a new dataset is created, this value will determine the default number of months until its visibility is automatically changed to public. If the value is <= 0 then the feature is disabled.', 'N'),
 (67, 'dataset_privacy_expiration_warning', '0', 'This value determines the number of days before dataset_privacy_expiration that a warning message is sent out. If the value is <= 0 then the feature is disabled.', 'N'),
 (68, 'fdt_jar_location', 'http://monalisa.cern.ch/FDT/', 'The public website for obtaining the fdt jar ', 'Y'),
 (69, 'fdt_help_url', '', 'The URL that contains help information on using Fast Data Transfer', 'N'),
 (70,'contact_email_bioinformatics','first.last@somwhere.edu','Bioinformatics core facility\'s email adress', 'N'),
 (71,'reply_email_core_facility_workauth_reminder','first.last@somewhere.edu','Email reply address for pending work authorizations reminder messages', 'Y'),
 (72,'dataset_privacy_expiration_reply_email','first.last@somewhere.edu','Reply address for dataset privacy expiration reminder emails', 'Y'),
 (73,'datatrack_read_directory','/home/gnomex/dataTrackData/','The file directory where data track files are stored that are not linked to an analysis file. NOTE: must end in /', 'Y'),
 (74,'datatrack_write_directory','/home/gnomex/dataTrackData/','The file directory where analysis files are stored that are not linked to an analysis file NOTE: must end in /', 'Y'),
 (75,'sequence_alignment_server_url','','Location of web service for submitting files for sequence alignment.', 'N'),
 (76,'sequence_alignment_supported','N','True if sequence alignment feature supported.', 'N'),
 (77,'lucene_datatrack_index_directory','/home/gnomex/luceneIndex/DataTrack','The file directory for storing lucene index files on data track data.', 'Y');


INSERT INTO `gnomex`.`ProtocolType`(`codeProtocolType`, `protocolType`, `isActive`)
VALUES ('IACUC', 'IACUC', 'Y'),
  ('IRB', 'IRB', 'Y');

INSERT INTO `gnomex`.`QualityControlStep`(`codeQualityControlStep`, `qualityControlStep`, `mageOntologyCode`, `mageOntologyDefinition`, `isActive`)
VALUES ('BIOREPL', 'Biological Replicate', 'biological_replicate', NULL, 'Y'),
  ('DYESWAP', 'Dye Swap', 'dye_swap_quality_control', NULL, 'Y'),
  ('NORTHBLOT', 'Northern Blotting', NULL, NULL, 'Y'),
  ('OTHER', 'Other Quality Control Step Taken (specify)', NULL, NULL, 'Y'),
  ('OTHERVALID', 'Other Validation (specify)', NULL, NULL, 'Y'),
  ('PCR', 'PCR', NULL, NULL, 'Y'),
  ('PEER', 'Peer Review', 'peer_review_quality_control', NULL, 'Y'),
  ('QUALPCR', 'Qualitative PCR', NULL, NULL, 'Y'),
  ('REVTPCR', 'Reverse Transcription PCR', 'reverse_transcription_PCR_quality_control', NULL, 'Y'),
  ('SPIKE', 'Spike', 'spike_quality_control', NULL, 'Y'),
  ('TECHREPL', 'Technical Replicate', 'technical_replicate', NULL, 'Y');


INSERT INTO `gnomex`.`RequestCategory`(`codeRequestCategory`, `requestCategory`, `idVendor`, `isActive`, `numberOfChannels`, `notes`, `icon`, `type`, isInternal, isExternal)
VALUES ('AFFY', 'Affymetrix Microarray', 2, 'Y', 1, 'Gene expression, SNP analysis', 'assets/microarray_chip.png', 'MICROARRAY', 'Y', 'Y'),
  ('AGIL', 'Agilent 2-color Microarray', 1, 'Y', 2, 'Gene expression, CGH, ChIP-on-chip', 'assets/microarray_small.png', 'MICROARRAY', 'Y', 'Y'),
  ('INHOUSE', 'In-house Spotted Microarray', 4, 'N', 2, null, null, 'MICROARRAY', 'Y', 'Y'),
  ('NIMBLE', 'NimbleGen Microarray', 6, 'N', 2, null, null, 'MICROARRAY', 'Y', 'Y'),
  ('OTHER', 'Microarray (Other)', NULL, 'N', NULL, null, null, 'MICROARRAY', 'Y', 'Y'),
  ('QC', 'Sample Quality', NULL, 'Y', NULL, 'RNA NanoChip, RNA PicoChip, DNA 1000 chip, Qubit picoGreen, gDNA gel', 'assets/chart_line.png', 'QC', 'Y', 'Y'),
  ('SOLEXA', 'Illumina GAIIx Sequencing', 7, 'Y', NULL, 'Genomic seq, mRNA seq, directional mRNAseq, ChIP-seq, small RNA seq, targeted genomic seq (capture/release)', 'assets/DNA_diag.png', 'ILLUMINA', 'Y', 'Y'),
  ('HISEQ', 'Illumina HiSeq 2000 Sequencing', 7, 'Y', NULL, 'Genomic seq, mRNA seq, directional mRNAseq, ChIP-seq, small RNA seq, targeted genomic seq (capture/release)', 'assets/DNA_diag_lightening.png', 'ILLUMINA', 'Y', 'Y'),
  ('AGIL1', 'Agilent 1-color Microarray', 1, 'Y', 1, 'Gene expression; miRNA', 'assets/microarray_small_single_color.png', 'MICROARRAY', 'Y', 'Y');

INSERT INTO `gnomex`.`RequestCategoryApplication`(`codeRequestCategory`, `codeApplication`)
VALUES ('AFFY', 'CHIP'),
  ('AFFY', 'EXON'),
  ('AFFY', 'EXP'),
  ('AFFY', 'SNP'),
  ('AGIL', 'CGH'),
  ('AGIL', 'CHIP'),
  ('AGIL', 'EXP'),
  ('AGIL1', 'EXP'),
  ('AGIL1', 'MIRNA'),
  ('AGIL1', 'WTRANSCRP'),
  ('INHOUSE', 'EXP'),
  ('NIMBLE', 'EXP'),
  ('NIMBLE', 'METH'),
  ('QC', 'BIOAN'),
  ('QC', 'QUBIT'),
  ('SOLEXA', 'CHIPSEQ' ),
  ('SOLEXA', 'DMRNASEQ'),
  ('SOLEXA', 'DNASEQ'),
  ('SOLEXA', 'TDNASEQ'),
  ('SOLEXA', 'MRNASEQ'),
  ('SOLEXA', 'SMRNASEQ'),
  ('SOLEXA', 'DNAMETHSEQ'),
  ('SOLEXA', 'MONNUCSEQ'),
  ('SOLEXA', 'TSCRPTSEQ'),
  ('HISEQ', 'CHIPSEQ' ),
  ('HISEQ', 'DMRNASEQ'),
  ('HISEQ', 'DNASEQ'),
  ('HISEQ', 'TDNASEQ'),
  ('HISEQ', 'MRNASEQ'),
  ('HISEQ', 'SMRNASEQ'),
  ('HISEQ', 'DNAMETHSEQ'),
  ('HISEQ', 'MONNUCSEQ'),
  ('HISEQ', 'TSCRPTSEQ');

 
-- Insert entries into PropertyType
INSERT INTO gnomex.PropertyType (codePropertyType, name) values ('TEXT',        'Text');
INSERT INTO gnomex.PropertyType (codePropertyType, name) values ('URL',         'URL');
INSERT INTO gnomex.PropertyType (codePropertyType, name) values ('CHECK',       'Checkbox');
INSERT INTO gnomex.PropertyType (codePropertyType, name) values ('OPTION',      'Option (Single selection)');
INSERT INTO gnomex.PropertyType (codePropertyType, name) values ('MOPTION',     'Option (Multiple selection)');
 


INSERT INTO `gnomex`.`Property`(`idProperty`, `name`, `mageOntologyCode`, `mageOntologyDefinition`, `isActive`, `idAppUser`, codePropertyType, isRequired)
VALUES (1, 'Age / Developmental Stage', 'age', NULL, 'Y', NULL, 'TEXT', 'N'),
  (2, 'Cell Line / Strain', 'cell_line', NULL, 'Y', NULL, 'TEXT', 'N'),
  (3, 'Cell Type', 'cell_type', NULL, 'Y', NULL, 'TEXT', 'N'),
  (4, 'Clinical Information', 'clinical_information', NULL, 'Y', NULL, 'TEXT', 'N'),
  (5, 'Compound', NULL, NULL, 'Y', NULL, 'TEXT', 'N'),
  (6, 'Disease State / Stage / Tumor Grade', 'distease_state', NULL, 'Y', NULL, 'TEXT', 'N'),
  (7, 'Dose', 'dose', NULL, 'Y', NULL, 'TEXT', 'N'),
  (8, 'Genetic Modification', NULL, NULL, 'Y', NULL, 'TEXT', 'N'),
  (9, 'Genotype', 'genotype', NULL, 'Y', NULL, 'TEXT', 'N'),
  (10, 'Growth Conditions', NULL, NULL, 'Y', NULL, 'TEXT', 'N'),
  (11, 'Individual', 'individual', NULL, 'Y', NULL, 'TEXT', 'N'),
  (12, 'Organ / Tissue', 'organism_part', NULL, 'Y', NULL, 'TEXT', 'N'),
  (13, 'Other', NULL, NULL, 'Y', NULL, 'TEXT', 'N'),
  (14, 'Sex', 'sex', NULL, 'Y', NULL, 'TEXT', 'N'),
  (15, 'Temperature', NULL, NULL, 'Y', NULL, 'TEXT', 'N'),
  (16, 'Time Course', 'sampling_time_point', NULL, 'Y', NULL, 'TEXT', 'N');

INSERT INTO `gnomex`.`SamplePrepMethod`(`idSamplePrepMethod`, `samplePrepMethod`, `isActive`)
VALUES (1, 'Qiagen RNeasy mini kit', 'Y'),
  (2, 'Trizol Reagent', 'Y'),
  (3, 'Agilent total RNA isolation mini kit', 'Y'),
  (4, 'RiboAmp', 'N'),
  (5, 'MessageAmp', 'N'),
  (6, 'Phenol/chloroform', 'Y'),
  (20, 'Gentra Puregene DNA Purification Kit', 'Y'),
  (21, 'Qiagen DNeasy tissue kit', 'Y'),
  (22, 'Other', 'Y'),
  (23, 'Agilent plant RNA isolation mini kit', 'Y'),
  (24, 'Ambion Tri Reagent', 'Y'),
  (25, 'Ambion RNAqueous kit', 'Y'),
  (26, 'Promega SV Total RNA Isolation kit', 'Y'),
  (27, 'Qiagen RNeasy Plant kit', 'Y'),
  (28, 'Ambion mirVana miRNA isolation kit', 'Y'),
  (29, 'Qiagen RNeasy Micro Kit', 'Y'),
  (30, 'Qiagen Genomic-tip', 'Y'),
  (41, 'Qiagen miRNeasy Mini Kit', 'Y');

INSERT INTO `gnomex`.`SampleSource`(`idSampleSource`, `sampleSource`, `isActive`)
VALUES (1, 'Cell Line', 'Y'),
  (2, 'Organ', 'Y'),
  (3, 'Whole Organism', 'Y'),
  (4, 'Tissue', 'Y'),
  (5, 'Other (specify in notes)', 'Y'),
  (6, 'Blood', 'Y');

INSERT INTO `gnomex`.`SampleType`(`idSampleType`, `sampleType`, `sortOrder`, `isActive`)
VALUES (1, 'DNA', 3, 'Y'),
  (2, 'aRNA', 6, 'N'),
  (3, 'miRNA', 8, 'Y'),
  (4, 'mRNA', 7, 'Y'),
  (5, 'Total RNA (eukaryote)', 1, 'Y'),
  (6, 'FFPE DNA', 5, 'Y'),
  (7, 'chIP DNA', 4, 'Y'),
  (8, 'polyA RNA', NULL, 'N'),
  (9, 'Total RNA (prokaryote)', 2, 'Y'),
  (10, 'cDNA', 5, 'Y'),
  (11, 'cap RNA', 6, 'Y'),
  (12, 'small RNA', 10, 'Y');

INSERT INTO `gnomex`.`SampleTypeApplication`(`idSampleTypeApplication`, `idSampleType`, `codeApplication`, `isActive`)
VALUES (2, 5, 'EXP',  'Y'),
  (3, 4, 'EXP',  'Y'),
  (4, 3, 'EXP', 'Y'),
  (5, 2, 'EXP',  'N'),
  (6, 1, 'CGH', 'Y'),
  (7, 6, 'CGH', 'Y'),
  (8, 7, 'CHIP',  'Y'),
  (9, 1, 'METH',  'Y'),
  (10, 7, 'METH', 'Y'),
  (11, 1, 'SNP',  'Y'),
  (12, 5, 'WTRANSCRP', 'Y'),
  (13, 5, 'MIRNA',  'Y'),
  (14, 7, 'CHIPSEQ',  'Y'),
  (15, 4, 'DMRNASEQ',  'Y'),
  (16, 1, 'DNASEQ',  'Y'),
  (17, 1, 'TDNASEQ', 'Y'),
  (18, 4, 'MRNASEQ', 'Y'),
  (19, 12, 'SMRNASEQ',  'Y'),
  (20, 1, 'DNAMETHSEQ',  'Y'),
  (21, 1, 'MONNUCSEQ',  'Y'),
  (22, 4, 'TSCRPTSEQ',  'Y'),
  (23, 9, 'MRNASEQ', 'Y'),
  (24, 9, 'TSCRPTSEQ',  'Y');

INSERT INTO `gnomex`.`SampleTypeRequestCategory`(`idSampleTypeRequestCategory`, `idSampleType`, `codeRequestCategory`)
VALUES (1, 1, 'AGIL'),
  (2, 2, 'AGIL'),
  (3, 3, 'AGIL'),
  (4, 4, 'AGIL'),
  (5, 5, 'AGIL'),
  (6, 6, 'AGIL'),
  (7, 7, 'AGIL'),
  (8, 8, 'AGIL'),
  (9, 9, 'AGIL'),
  (10, 1, 'QC'),
  (11, 2, 'QC'),
  (12, 3, 'QC'),
  (13, 4, 'QC'),
  (14, 5, 'QC'),
  (15, 6, 'QC'),
  (16, 7, 'QC'),
  (17, 8, 'QC'),
  (18, 9, 'QC'),
  (19, 1, 'AFFY'),
  (20, 2, 'AFFY'),
  (21, 3, 'AFFY'),
  (22, 4, 'AFFY'),
  (23, 5, 'AFFY'),
  (24, 6, 'AFFY'),
  (25, 7, 'AFFY'),
  (26, 8, 'AFFY'),
  (27, 9, 'AFFY'),
  (28, 1, 'SOLEXA'),
  (29, 5, 'SOLEXA'),
  (30, 9, 'SOLEXA'),
  (31, 7, 'SOLEXA'),
  (32, 4, 'SOLEXA'),
  (33, 9, 'SOLEXA'),
  (34, 10, 'SOLEXA'),
  (35, 11, 'SOLEXA'),
  (36, 12, 'SOLEXA'),
  (37, 2, 'AGIL1'),
  (38, 3, 'AGIL1'),
  (39, 4, 'AGIL1'),
  (40, 5, 'AGIL1'),
  (41, 6, 'AGIL1'),
  (42, 7, 'AGIL1'),
  (43, 8, 'AGIL1'),
  (44, 9, 'AGIL1'),
  (45, 1, 'HISEQ'),
  (46, 5, 'HISEQ'),
  (47, 9, 'HISEQ'),
  (48, 7, 'HISEQ'),
  (49, 4, 'HISEQ'),
  (50, 9, 'HISEQ'),
  (51, 10, 'HISEQ'),
  (52, 11, 'HISEQ'),
  (53, 12, 'HISEQ');


INSERT INTO `gnomex`.`SeqLibProtocol`(`idSeqLibProtocol`, `seqLibProtocol`,`description`,`url`, `isActive`)
VALUES  (31, 'Illumina Genomic DNA Sample Prep Kit', 'Illumina Genomic DNA Sample Prep Kit', null, 'Y'),
  (32, 'Illumina DGE-Tag Profiling NlaIII Sample Prep Kit', 'Illumina DGE-Tag Profiling NlaIII Sample Prep Kit', null, 'N'),
  (33, 'Illumina DGE-Tag Profiling DpnII Sample Prep Kit', 'Illumina DGE-Tag Profiling DpnII Sample Prep Kit', null, 'N'),
  (34, 'Illumina ChIP-Seq prep kit', 'Illumina ChIP-Seq prep kit', null, 'Y'),
  (35, 'Illumina Small RNA Sample Prep Kit', 'Illumina Small RNA Sample Prep Kit', null, 'Y'),
  (39, 'Illumina mRNA-Seq Sample Prep Kit', 'Illumina mRNA-Seq Sample Prep Kit', null, 'Y');

INSERT INTO `gnomex`.`SeqLibProtocolApplication`(`idSeqLibProtocol`, `codeApplication`)
VALUES (31, 'DNASEQ'),
  (32, 'MRNASEQ'),
  (33, 'MRNASEQ'),
  (34,'CHIPSEQ'),
  (35, 'SMRNASEQ'),
  (39, 'MRNASEQ'),
  (31, 'DNAMETHSEQ'),
  (31, 'MONNUCSEQ'),
  (39, 'TSCRPTSEQ');
  
-- TREATMENTS are replaced by Applications (for now)
-- INSERT INTO `gnomex`.`SeqLibTreatment`(`idSeqLibTreatment`, `seqLibTreatment`, `isActive`)
-- VALUES (1, 'Mononucleosome targeted sequencing', 'Y'),
--  (2, 'Bisulfite treatment', 'Y'),
--  (3, 'DSN (Duplex-specific nuclease) treatment of library', 'Y');

INSERT INTO `gnomex`.`SeqRunType`(`idSeqRunType`, `seqRunType`, `isActive`, `sortOrder`)
VALUES (1, 'Sequencing / chIP Sequencing', 'N', NULL),
  (2, 'Digital Gene Expression', 'N', NULL),
  (3, 'Single-end reads', 'Y',1),
  (4, 'Paired-end reads', 'Y',2);



INSERT INTO `gnomex`.`SequencingControl`(`idSequencingControl`, `sequencingControl`, `isActive`, `idAppUser`)
VALUES (1, 'PhiX Control', 'Y', NULL);

INSERT INTO `gnomex`.`SequencingPlatform`(`codeSequencingPlatform`, `sequencingPlatform`, `isActive`)
VALUES ('GAIIX', 'Illumina GAIIx Sequencing', 'Y'),
   ('HISEQ', 'Illumina HiSeq 2000 Sequencing', 'Y');

INSERT INTO `gnomex`.`SlideDesign`(`idSlideDesign`, `name`, `slideDesignProtocolName`, `idSlideProduct`, `accessionNumberArrayExpress`, `isActive`)
VALUES (3,'Whole human genome microarray (44K)','012391',2,'A-AGIL-28','N'),
 (4,'Human promoter 1 ChIP-on-chip (44K)',NULL,3,NULL,'N'),
 (5,'Human promoter 2 ChIP-on-chip (44K)',NULL,3,NULL,'Y'),
 (6,'Human 1A GE microarray (22K)',NULL,5,NULL,'N'),
 (8,'Human genome CGH 44B (4x44K)','013282',7,NULL,'Y'),
 (11,'Human Genome CGH (1x185K)',NULL,10,NULL,'N'),
 (12,'Arabidopsis 3 oligo microarray (44K)',NULL,11,'A-AGIL-12','N'),
 (14,'C. elegans Gene Expression (22K)','012795',13,NULL,'N'),
 (15,'Canine Gene Expression v2 (4x44K)','021193',14,NULL,'Y'),
 (17,'Chicken Gene Expression (4x44k)','015068',16,NULL,'Y'),
 (18,'Whole mouse genome microarray (44K)',NULL,17,NULL,'N'),
 (25,'Rat oligo microarray (22K)',NULL,24,NULL,'N'),
 (26,'Whole rat genome microarray (44K)','014879',25,NULL,'Y'),
 (31,'Xenopus oligo expression microarray (22K)','013665',30,NULL,'N'),
 (32,'Yeast V2 oligo expression microarray (11K)','013884',31,NULL,'N'),
 (33,'S. cerevisiae ChIP array(4x44K)','014810',32,NULL,'Y'),
 (36,'Zebrafish oligo expression microarray (22K)',NULL,35,'A-AGIL-ZEBRAFISH','N'),
 (54,'Whole human genome microarray (4x44K)','014850',54,'A-AGIL-28','Y'),
 (114,'Yeast Whole Genome ChIP-on-chip microarray (244K)','14741',110,NULL,'Y'),
 (115,'Human Genome CGH 244A (1x244K)','014693',111,NULL,'Y'),
 (117,'Human Genome CGH 105A (2x105K)','014698',113,NULL,'Y'),
 (118,'Affymetrix GeneChip Human Genome U133 Plus 2.0',NULL,114,NULL,'Y'),
 (126,'Human GeneChip Mapping 250K Nsp Array',NULL,123,NULL,'Y'),
 (128,'Arabidopsis Gene Expression v3 (4x44K)','015059',124,NULL,'Y'),
 (129,'Arabidopsis Whole Genome ChIP-on-chip slide 1','014798',125,NULL,'Y'),
 (130,'Arabidopsis Whole Genome ChIP-on-chip Slide 2','014799',125,NULL,'Y'),
 (131,'C. elegans Gene Expression (4x44K)','015061',126,NULL,'Y'),
 (132,'Zebrafish oligo microarray (4x44K)','015064',127,NULL,'N'),
 (133,'Xenopus oligo microarray (4x44K)','015066',128,NULL,'Y'),
 (135,'Sodalis tiling microarray (4x44K)','015651',130,NULL,'Y'),
 (136,'Yeast GE microarray (4x44K)','015072',131,NULL,'Y'),
 (137,'Human Promoter ChIP-on-chip slide 1 (1x244K)','014706',132,NULL,'Y'),
 (138,'Human Promoter ChIP-on-chip slide 2 (1x244K)','014707',132,NULL,'Y'),
 (139,'Whole Mouse Genome Microarray (4x44K)','014868',133,NULL,'Y'),
 (142,'Canine SNP Array (520431)','520431',135,NULL,'Y'),
 (143,'Mouse Genome 430 2.0 Array','900496',136,NULL,'Y'),
 (144,'Human CpG Island (1x244K)','014791',137,NULL,'Y'),
 (148,'Mouse Promoter 1 ChIP-on-Chip (244K)','014716',141,NULL,'Y'),
 (149,'Mouse Promoter 2 ChIP-on-Chip (244K)','014717',141,NULL,'Y'),
 (151,'Human ENCODE ChIP-on-chip (1x244K)','014792',143,NULL,'Y'),
 (153,'Drosophila Gene Expression (4x44K)','018972',145,NULL,'Y'),
 (155,'Affymetrix Human SNP 5.0 Array',NULL,147,NULL,'Y'),
 (164,'Zebrafish Gene Expression v2 (4x44K)','19161',155,NULL,'Y'),
 (165,'Mouse miRNA microarray (8x15K)','019119',156,NULL,'Y'),
 (167,'Human miRNA v2 (8x15K)','016436',158,NULL,'Y'),
 (168,'Human CGH (4x180K)','022060',159,NULL,'Y'),
 (170,'Affymetrix Human SNP 6.0 Array',NULL,161,NULL,'Y'),
 (175,'Porcine Gene Expression Microarray','020109',166,NULL,'Y'),
 (177,'Mouse miRNA microarray v2 (8x15K)','021828',168,NULL,'Y'),
 (179,'Human miRNA v3 (8x15K)','021827',170,NULL,'Y');


INSERT INTO `gnomex`.`SlideProduct`(`idSlideProduct`, `name`, `catalogNumber`, `isCustom`, `idLab`, `codeApplication`, `idVendor`, `idOrganism`, `arraysPerSlide`, `slidesInSet`, `isSlideSet`, `isActive`, `idBillingSlideProductClass`, `idBillingSlideServiceClass`)
VALUES  (2,'Whole human genome expression array (44K)',NULL,'N',NULL,'EXP',1,1,1,1,'N','N',NULL,NULL),
 (3,'Human promoter ChIP-on-chip Set (2 slides) (44K)',NULL,'N',NULL,'CHIP',1,1,1,2,'Y','N',3,8),
 (5,'Human 1A GE microarray (22K)',NULL,'N',NULL,'EXP',1,1,1,1,'N','N',NULL,NULL),
 (7,'Human genome CGH 44B (4x44K)',NULL,'N',NULL,'CGH',1,1,1,1,'N','Y',3,4),
 (10,'Human Genome CGH (1x185K)',NULL,'N',NULL,'CGH',1,1,1,1,'N','N',NULL,NULL),
 (11,'Arabidopsis 3 oligo expression array (44K)',NULL,'N',NULL,'EXP',1,11,1,1,'N','N',NULL,NULL),
 (13,'C. elegans Gene Expression (22K)',NULL,'N',NULL,'EXP',1,6,1,1,'N','N',NULL,NULL),
 (14,'Canine Gene Expression v2 (4x44K)',NULL,'N',NULL,'EXP',1,15,4,1,'N','Y',3,1),
 (16,'Chicken Gene Expression (4x44k)',NULL,'N',NULL,'EXP',1,13,4,1,'N','Y',3,1),
 (17,'Whole mouse genome expression array (44K)',NULL,'N',NULL,'EXP',1,3,1,1,'N','N',NULL,NULL),
 (24,'Rat oligo expression array (22K)',NULL,'N',NULL,'EXP',1,10,1,1,'N','N',NULL,NULL),
 (25,'Whole rat genome expression array (44K)',NULL,'N',NULL,'EXP',1,10,4,1,'N','Y',3,1),
 (30,'Xenopus oligo expression microarray (22K)',NULL,'N',NULL,'EXP',1,8,1,1,'N','N',NULL,NULL),
 (31,'S. cerevisiae V2 oligo expression array (11K)',NULL,'N',NULL,'EXP',1,2,2,1,'N','N',NULL,NULL),
 (32,'S. cerevisiae ChIP array(4x44K)',NULL,'N',NULL,'CHIP',1,2,4,1,'N','Y',3,8),
 (35,'Zebrafish oligo expression array (22K)',NULL,'N',NULL,'EXP',1,9,1,1,'N','N',NULL,NULL),
 (54,'Whole human genome expression array (44K) (4 pack)',NULL,'N',NULL,'EXP',1,1,4,1,'N','Y',3,1),
 (110,'Yeast Whole Genome ChIP-on-chip microarray (244K)',NULL,'N',NULL,'CHIP',1,2,1,1,'N','Y',1,6),
 (111,'Human Genome CGH 244A (1x244K)',NULL,'N',NULL,'CGH',1,1,1,1,'N','Y',1,2),
 (113,'Human Genome CGH 105A (2x105K)',NULL,'N',NULL,'CGH',1,1,2,1,'N','Y',2,3),
 (114,'Affymetrix GeneChip Human Genome U133 Plus 2.0',NULL,'N',NULL,'EXP',2,1,1,1,'N','Y',7,12),
 (123,'Human GeneChip Mapping 250K Nsp Array',NULL,'N',NULL,'SNP',2,1,1,1,'N','Y',5,13),
 (124,'Arabidopsis Gene Expression v3 (4x44K)',NULL,'N',NULL,'EXP',1,11,4,1,'N','Y',3,1),
 (125,'Arabidopsis Whole Genome ChIP-on-chip Set (244K)',NULL,'N',NULL,'CHIP',1,11,1,2,'Y','Y',1,6),
 (126,'C. elegans Gene Expression (4x44K)',NULL,'N',NULL,'EXP',1,6,4,1,'N','Y',3,1),
 (127,'Zebrafish oligo microarray (4x44K)',NULL,'N',NULL,'EXP',1,9,4,1,'N','N',3,1),
 (128,'Xenopus oligo microarray (4x44K)',NULL,'N',NULL,'EXP',1,8,4,1,'N','Y',3,1),
 (130,'Sodalis tiling microarray (4x44K)',NULL,'N',NULL,'EXP',1,22,4,1,'N','Y',NULL,NULL),
 (131,'Yeast GE microarray (4x44K)',NULL,'N',NULL,'EXP',1,2,4,1,'N','Y',3,1),
 (132,'Human Promoter ChIP-on-chip 244K Set',NULL,'N',NULL,'CHIP',1,1,1,2,'Y','Y',1,6),
 (133,'Whole Mouse Genome Microarray (4x44K)',NULL,'N',NULL,'EXP',1,3,4,1,'N','Y',3,1),
 (135,'Canine SNP Array (520431)',NULL,'N',NULL,NULL,NULL,NULL,1,1,'N','Y',5,13),
 (136,'Mouse Genome 430 2.0 Array',NULL,'N',NULL,NULL,2,3,1,1,'N','Y',7,12),
 (137,'Human CpG Island (1x244K)',NULL,'N',NULL,NULL,1,1,1,1,'N','Y',1,6),
 (141,'Mouse Promoter ChIP-on-chip Set (244K)',NULL,'N',NULL,NULL,1,3,1,2,'Y','Y',1,6),
 (143,'Human ENCODE ChIP-on-chip (1x244K)',NULL,'N',NULL,NULL,1,1,1,1,'N','Y',1,6),
 (145,'Drosophila Gene Expression (4x44K)',NULL,'N',NULL,NULL,1,4,4,1,'N','Y',3,1),
 (147,'Affymetrix Human SNP 5.0 Array','901069','N',NULL,NULL,2,1,1,1,'N','Y',6,13),
 (155,'Zebrafish Gene Expression v2 (4x44K)',NULL,'N',NULL,NULL,1,9,4,1,'N','Y',3,1),
 (156,'Mouse miRNA microarray (8x15K)',NULL,'N',NULL,NULL,1,3,8,1,'N','Y',11,16),
 (158,'Human miRNA v2 (8x15K)',NULL,'N',NULL,NULL,1,1,8,1,'N','Y',11,16),
 (159,'Human CGH (4x180K)',NULL,'N',NULL,NULL,1,1,4,1,'N','Y',13,19),
 (161,'Affymetrix Human SNP 6.0 Array',NULL,'N',NULL,NULL,2,1,1,1,'N','Y',8,14),
 (166,'Porcine Gene Expression Microarray',NULL,'N',NULL,NULL,1,23,4,1,'N','Y',3,1),
 (168,'Mouse miRNA microarray v2 (8x15K)',NULL,'N',NULL,NULL,1,3,8,1,'N','Y',11,16),
 (170,'Human miRNA v3 (8x15K)',NULL,'N',NULL,NULL,1,1,1,1,'N','Y',11,16);


INSERT INTO `gnomex`.`SlideProductApplication`(`idSlideProduct`, `codeApplication`)
VALUES  (7,'CGH'),
 (10,'CGH'),
 (111,'CGH'),
 (113,'CGH'),
 (159,'CGH'),
 (3,'CHIP'),
 (32,'CHIP'),
 (110,'CHIP'),
 (125,'CHIP'),
 (132,'CHIP'),
 (137,'CHIP'),
 (141,'CHIP'),
 (143,'CHIP'),
 (2,'EXP'),
 (5,'EXP'),
 (11,'EXP'),
 (13,'EXP'),
 (14,'EXP'),
 (16,'EXP'),
 (17,'EXP'),
 (24,'EXP'),
 (25,'EXP'),
 (30,'EXP'),
 (31,'EXP'),
 (35,'EXP'),
 (54,'EXP'),
 (114,'EXP'),
 (124,'EXP'),
 (126,'EXP'),
 (127,'EXP'),
 (128,'EXP'),
 (130,'EXP'),
 (131,'EXP'),
 (133,'EXP'),
 (136,'EXP'),
 (145,'EXP'),
 (155,'EXP'),
 (166,'EXP'),
 (132,'METH'),
 (137,'METH'),
 (156,'MIRNA'),
 (158,'MIRNA'),
 (168,'MIRNA'),
 (170,'MIRNA'),
 (123,'SNP'),
 (135,'SNP'),
 (147,'SNP'),
 (161,'SNP'),
 (143,'WTRANSCRP');


INSERT INTO `gnomex`.`SlideSource`(`codeSlideSource`, `slideSource`, `isActive`)
VALUES ('CLIENT', 'New microarray purchased by lab', 'Y'),
  ('CORE', 'New microarray purchased from core facility', 'Y'),
  ('REUSE', 'Strip and reuse existing microarray', 'Y');

INSERT INTO `gnomex`.`State`(`codeState`, `state`, `isActive`)
VALUES ('AK', 'Alaska', 'Y'),
  ('AL', 'Alabama', 'Y'),
  ('AR', 'Arkansas', 'Y'),
  ('AZ', 'Arizona', 'Y'),
  ('CA', 'California', 'Y'),
  ('CO', 'Colorado', 'Y'),
  ('CT', 'Conneticut', 'Y'),
  ('DE', 'Deleware', 'Y'),
  ('FL', 'Florida', 'Y'),
  ('GA', 'Georgia', 'Y'),
  ('HI', 'Hawaii', 'Y'),
  ('IA', 'Iowa', 'Y'),
  ('ID', 'Idaho', 'Y'),
  ('IL', 'Illinois', 'Y'),
  ('IN', 'Indiana', 'Y'),
  ('KS', 'Kansas', 'Y'),
  ('KY', 'Kentucky', 'Y'),
  ('LA', 'Lousiana', 'Y'),
  ('MA', 'Massachusetts', 'Y'),
  ('MD', 'Maryland', 'Y'),
  ('ME', 'Maine', 'Y'),
  ('MI', 'Michigan', 'Y'),
  ('MN', 'Minnesota', 'Y'),
  ('MO', 'Missouri', 'Y'),
  ('MS', 'Mississippi', 'Y'),
  ('MT', 'Montana', 'Y'),
  ('NC', 'North Carolina', 'Y'),
  ('ND', 'North Dakota', 'Y'),
  ('NE', 'Nebraska', 'Y'),
  ('NH', 'New Hampshire', 'Y'),
  ('NJ', 'New Jersey', 'Y'),
  ('NM', 'New Mexico', 'Y'),
  ('NV', 'Nevada', 'Y'),
  ('NY', 'New York', 'Y'),
  ('OH', 'Ohio', 'Y'),
  ('OK', 'Oklahoma', 'Y'),
  ('OR', 'Oregon', 'Y'),
  ('PA', 'Pennsylvania', 'Y'),
  ('RI', 'Rhode Island', 'Y'),
  ('SC', 'South Carolina', 'Y'),
  ('SD', 'South Dakota', 'Y'),
  ('TN', 'Tennessee', 'Y'),
  ('TX', 'Texas', 'Y'),
  ('UT', 'Utah', 'Y'),
  ('VA', 'Virginia', 'Y'),
  ('VT', 'Vermont', 'Y'),
  ('WA', 'Washington', 'Y'),
  ('WI', 'Wisconsin', 'Y'),
  ('WV', 'West Virginia', 'Y'),
  ('WY', 'Wyoming', 'Y');

INSERT INTO `gnomex`.`Step`(`codeStep`, `step`, `isActive`)
VALUES ('COMPLETE', 'Complete', 'Y'),
  ('EXT', 'Scan/Feature Extraction', 'Y'),
  ('HYB', 'Hybridization', 'Y'),
  ('LABEL', 'Labeling', 'Y'),
  ('QC', 'Quality Control', 'Y'),
  ('SEQASSEM', 'Illumina GAIIx Cluster Gen', 'Y'),
  ('SEQPIPE', 'Illumina GAIIx Data Pipeline', 'Y'),
  ('SEQPREP', 'Illumina GAIIx Library Prep', 'Y'),
  ('SEQQC', 'Illumina GAIIx Quality Control', 'Y'),
  ('SEQRUN', 'Illumina GAIIx Sequencing Run', 'Y'),
  ('HSEQASSEM', 'Illumina GAIIx Cluster Gen', 'Y'),
  ('HSEQPIPE', 'Illumina HiSeq 2000 Data Pipeline', 'Y'),
  ('HSEQPREP', 'Illumina HiSeq 2000 Library Prep', 'Y'),
  ('HSEQQC', 'Illumina HiSeq 2000 Quality Control', 'Y'),
  ('HSEQRUN', 'Illumina HiSeq 2000 Sequencing Run', 'Y'),
  ('STOP', 'Stop/Failed', 'Y');


INSERT INTO `gnomex`.`UserPermissionKind`(`codeUserPermissionKind`, `userPermissionKind`, `isActive`)
VALUES ('ADMIN', 'Admin', 'Y'),
  ('BILLING', 'Billing', 'Y'),
  ('LAB', 'Lab', 'Y'),
  ('USER', 'User', 'Y');

INSERT INTO `gnomex`.`Vendor`(`idVendor`, `vendorName`, `description`, `isActive`)
VALUES (1, 'Agilent', 'Agilent', 'Y'),
  (2, 'Affymetrix', 'Affymetrix', 'Y'),
  (4, 'In House', 'In House', 'N'),
  (5, 'Invitrogen', 'Invitrogen', 'N'),
  (6, 'NimbleGen', 'NimbleGen', 'N'),
  (7, 'Illumina', 'Illumina', 'Y');

INSERT INTO `gnomex`.`Visibility`(`codeVisibility`, `visibility`)
VALUES ('OWNER', 'Owner'),
  ('MEM', 'Members'),
  ('INST', 'Institution'),
  ('PUBLIC', 'Public');
  



-- Re-enable foreign key checks
SET FOREIGN_KEY_CHECKS=@OLD_FOREIGN_KEY_CHECKS;

-- End of script